{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "07cjmUohEssy"
      },
      "source": [
        "Установка HMMER"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VxNHzLqLAk-o",
        "outputId": "107ac34d-3096-46f7-c73d-68b983416cf9"
      },
      "outputs": [
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "--2025-06-18 08:09:22--  http://eddylab.org/software/hmmer/hmmer-3.3.2.tar.gz\n",
            "Resolving eddylab.org (eddylab.org)... 96.126.110.11, 2600:3c03::f03c:91ff:fec8:383c\n",
            "Connecting to eddylab.org (eddylab.org)|96.126.110.11|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 18213049 (17M) [application/x-gzip]\n",
            "Saving to: ‘hmmer-3.3.2.tar.gz’\n",
            "\n",
            "     0K .......... .......... .......... .......... ..........  0%  407K 44s\n",
            "    50K .......... .......... .......... .......... ..........  0%  812K 33s\n",
            "   100K .......... .......... .......... .......... ..........  0% 45.0M 22s\n",
            "   150K .......... .......... .......... .......... ..........  1%  304M 16s\n",
            "   200K .......... .......... .......... .......... ..........  1%  820K 17s\n",
            "   250K .......... .......... .......... .......... ..........  1% 55.4M 14s\n",
            "   300K .......... .......... .......... .......... ..........  1%  302M 12s\n",
            "   350K .......... .......... .......... .......... ..........  2%  206M 11s\n",
            "   400K .......... .......... .......... .......... ..........  2%  836K 12s\n",
            "   450K .......... .......... .......... .......... ..........  2%  115M 11s\n",
            "   500K .......... .......... .......... .......... ..........  3%  130M 10s\n",
            "   550K .......... .......... .......... .......... ..........  3%  164M 9s\n",
            "   600K .......... .......... .......... .......... ..........  3%  121M 8s\n",
            "   650K .......... .......... .......... .......... ..........  3%  358M 8s\n",
            "   700K .......... .......... .......... .......... ..........  4%  355M 7s\n",
            "   750K .......... .......... .......... .......... ..........  4%  316M 7s\n",
            "   800K .......... .......... .......... .......... ..........  4%  311M 6s\n",
            "   850K .......... .......... .......... .......... ..........  5%  842K 7s\n",
            "   900K .......... .......... .......... .......... ..........  5%  228M 7s\n",
            "   950K .......... .......... .......... .......... ..........  5%  145M 6s\n",
            "  1000K .......... .......... .......... .......... ..........  5%  152M 6s\n",
            "  1050K .......... .......... .......... .......... ..........  6%  141M 6s\n",
            "  1100K .......... .......... .......... .......... ..........  6%  126M 5s\n",
            "  1150K .......... .......... .......... .......... ..........  6%  224M 5s\n",
            "  1200K .......... .......... .......... .......... ..........  7%  384M 5s\n",
            "  1250K .......... .......... .......... .......... ..........  7%  302M 5s\n",
            "  1300K .......... .......... .......... .......... ..........  7%  320M 5s\n",
            "  1350K .......... .......... .......... .......... ..........  7%  288M 4s\n",
            "  1400K .......... .......... .......... .......... ..........  8%  309M 4s\n",
            "  1450K .......... .......... .......... .......... ..........  8%  266M 4s\n",
            "  1500K .......... .......... .......... .......... ..........  8%  371M 4s\n",
            "  1550K .......... .......... .......... .......... ..........  8%  380M 4s\n",
            "  1600K .......... .......... .......... .......... ..........  9%  384M 4s\n",
            "  1650K .......... .......... .......... .......... ..........  9%  327M 4s\n",
            "  1700K .......... .......... .......... .......... ..........  9%  309M 3s\n",
            "  1750K .......... .......... .......... .......... .......... 10%  865K 4s\n",
            "  1800K .......... .......... .......... .......... .......... 10% 58.9M 4s\n",
            "  1850K .......... .......... .......... .......... .......... 10% 50.2M 4s\n",
            "  1900K .......... .......... .......... .......... .......... 10% 50.8M 4s\n",
            "  1950K .......... .......... .......... .......... .......... 11%  208M 3s\n",
            "  2000K .......... .......... .......... .......... .......... 11%  199M 3s\n",
            "  2050K .......... .......... .......... .......... .......... 11%  188M 3s\n",
            "  2100K .......... .......... .......... .......... .......... 12%  240M 3s\n",
            "  2150K .......... .......... .......... .......... .......... 12%  213M 3s\n",
            "  2200K .......... .......... .......... .......... .......... 12%  204M 3s\n",
            "  2250K .......... .......... .......... .......... .......... 12%  134M 3s\n",
            "  2300K .......... .......... .......... .......... .......... 13%  143M 3s\n",
            "  2350K .......... .......... .......... .......... .......... 13% 88.8M 3s\n",
            "  2400K .......... .......... .......... .......... .......... 13%  139M 3s\n",
            "  2450K .......... .......... .......... .......... .......... 14% 80.8M 3s\n",
            "  2500K .......... .......... .......... .......... .......... 14%  116M 3s\n",
            "  2550K .......... .......... .......... .......... .......... 14%  346M 3s\n",
            "  2600K .......... .......... .......... .......... .......... 14% 68.6M 3s\n",
            "  2650K .......... .......... .......... .......... .......... 15% 96.2M 2s\n",
            "  2700K .......... .......... .......... .......... .......... 15%  318M 2s\n",
            "  2750K .......... .......... .......... .......... .......... 15% 78.3M 2s\n",
            "  2800K .......... .......... .......... .......... .......... 16% 68.0M 2s\n",
            "  2850K .......... .......... .......... .......... .......... 16%  302M 2s\n",
            "  2900K .......... .......... .......... .......... .......... 16% 55.4M 2s\n",
            "  2950K .......... .......... .......... .......... .......... 16%  197M 2s\n",
            "  3000K .......... .......... .......... .......... .......... 17%  295M 2s\n",
            "  3050K .......... .......... .......... .......... .......... 17% 15.8M 2s\n",
            "  3100K .......... .......... .......... .......... .......... 17% 71.7M 2s\n",
            "  3150K .......... .......... .......... .......... .......... 17% 88.6M 2s\n",
            "  3200K .......... .......... .......... .......... .......... 18%  281M 2s\n",
            "  3250K .......... .......... .......... .......... .......... 18% 53.8M 2s\n",
            "  3300K .......... .......... .......... .......... .......... 18%  196M 2s\n",
            "  3350K .......... .......... .......... .......... .......... 19%  368M 2s\n",
            "  3400K .......... .......... .......... .......... .......... 19%  377M 2s\n",
            "  3450K .......... .......... .......... .......... .......... 19%  283M 2s\n",
            "  3500K .......... .......... .......... .......... .......... 19% 87.5M 2s\n",
            "  3550K .......... .......... .......... .......... .......... 20% 63.9M 2s\n",
            "  3600K .......... .......... .......... .......... .......... 20% 1.13M 2s\n",
            "  3650K .......... .......... .......... .......... .......... 20% 93.8M 2s\n",
            "  3700K .......... .......... .......... .......... .......... 21%  239M 2s\n",
            "  3750K .......... .......... .......... .......... .......... 21%  166M 2s\n",
            "  3800K .......... .......... .......... .......... .......... 21%  115M 2s\n",
            "  3850K .......... .......... .......... .......... .......... 21%  149M 2s\n",
            "  3900K .......... .......... .......... .......... .......... 22% 53.2M 2s\n",
            "  3950K .......... .......... .......... .......... .......... 22%  166M 2s\n",
            "  4000K .......... .......... .......... .......... .......... 22% 99.3M 2s\n",
            "  4050K .......... .......... .......... .......... .......... 23%  273M 2s\n",
            "  4100K .......... .......... .......... .......... .......... 23%  319M 2s\n",
            "  4150K .......... .......... .......... .......... .......... 23%  351M 2s\n",
            "  4200K .......... .......... .......... .......... .......... 23%  298M 2s\n",
            "  4250K .......... .......... .......... .......... .......... 24%  327M 2s\n",
            "  4300K .......... .......... .......... .......... .......... 24% 61.6M 2s\n",
            "  4350K .......... .......... .......... .......... .......... 24% 59.1M 2s\n",
            "  4400K .......... .......... .......... .......... .......... 25% 54.4M 1s\n",
            "  4450K .......... .......... .......... .......... .......... 25% 64.4M 1s\n",
            "  4500K .......... .......... .......... .......... .......... 25% 72.1M 1s\n",
            "  4550K .......... .......... .......... .......... .......... 25% 70.3M 1s\n",
            "  4600K .......... .......... .......... .......... .......... 26% 29.9M 1s\n",
            "  4650K .......... .......... .......... .......... .......... 26% 69.8M 1s\n",
            "  4700K .......... .......... .......... .......... .......... 26% 27.4M 1s\n",
            "  4750K .......... .......... .......... .......... .......... 26% 32.6M 1s\n",
            "  4800K .......... .......... .......... .......... .......... 27% 33.4M 1s\n",
            "  4850K .......... .......... .......... .......... .......... 27%  343M 1s\n",
            "  4900K .......... .......... .......... .......... .......... 27%  305M 1s\n",
            "  4950K .......... .......... .......... .......... .......... 28%  350M 1s\n",
            "  5000K .......... .......... .......... .......... .......... 28%  278M 1s\n",
            "  5050K .......... .......... .......... .......... .......... 28%  328M 1s\n",
            "  5100K .......... .......... .......... .......... .......... 28%  346M 1s\n",
            "  5150K .......... .......... .......... .......... .......... 29% 10.4M 1s\n",
            "  5200K .......... .......... .......... .......... .......... 29% 55.0M 1s\n",
            "  5250K .......... .......... .......... .......... .......... 29% 73.8M 1s\n",
            "  5300K .......... .......... .......... .......... .......... 30% 62.1M 1s\n",
            "  5350K .......... .......... .......... .......... .......... 30% 71.2M 1s\n",
            "  5400K .......... .......... .......... .......... .......... 30% 58.4M 1s\n",
            "  5450K .......... .......... .......... .......... .......... 30% 69.7M 1s\n",
            "  5500K .......... .......... .......... .......... .......... 31% 64.6M 1s\n",
            "  5550K .......... .......... .......... .......... .......... 31% 67.3M 1s\n",
            "  5600K .......... .......... .......... .......... .......... 31%  127M 1s\n",
            "  5650K .......... .......... .......... .......... .......... 32%  349M 1s\n",
            "  5700K .......... .......... .......... .......... .......... 32% 17.6M 1s\n",
            "  5750K .......... .......... .......... .......... .......... 32% 63.5M 1s\n",
            "  5800K .......... .......... .......... .......... .......... 32%  205M 1s\n",
            "  5850K .......... .......... .......... .......... .......... 33%  347M 1s\n",
            "  5900K .......... .......... .......... .......... .......... 33% 1.47M 1s\n",
            "  5950K .......... .......... .......... .......... .......... 33%  152M 1s\n",
            "  6000K .......... .......... .......... .......... .......... 34% 70.4M 1s\n",
            "  6050K .......... .......... .......... .......... .......... 34%  224M 1s\n",
            "  6100K .......... .......... .......... .......... .......... 34%  861K 1s\n",
            "  6150K .......... .......... .......... .......... .......... 34%  188M 1s\n",
            "  6200K .......... .......... .......... .......... .......... 35%  202M 1s\n",
            "  6250K .......... .......... .......... .......... .......... 35%  258M 1s\n",
            "  6300K .......... .......... .......... .......... .......... 35%  230M 1s\n",
            "  6350K .......... .......... .......... .......... .......... 35%  320M 1s\n",
            "  6400K .......... .......... .......... .......... .......... 36%  185M 1s\n",
            "  6450K .......... .......... .......... .......... .......... 36%  344M 1s\n",
            "  6500K .......... .......... .......... .......... .......... 36%  269M 1s\n",
            "  6550K .......... .......... .......... .......... .......... 37%  252M 1s\n",
            "  6600K .......... .......... .......... .......... .......... 37%  209M 1s\n",
            "  6650K .......... .......... .......... .......... .......... 37%  217M 1s\n",
            "  6700K .......... .......... .......... .......... .......... 37%  266M 1s\n",
            "  6750K .......... .......... .......... .......... .......... 38%  281M 1s\n",
            "  6800K .......... .......... .......... .......... .......... 38%  280M 1s\n",
            "  6850K .......... .......... .......... .......... .......... 38% 83.0M 1s\n",
            "  6900K .......... .......... .......... .......... .......... 39% 53.1M 1s\n",
            "  6950K .......... .......... .......... .......... .......... 39% 64.8M 1s\n",
            "  7000K .......... .......... .......... .......... .......... 39%  287M 1s\n",
            "  7050K .......... .......... .......... .......... .......... 39%  367M 1s\n",
            "  7100K .......... .......... .......... .......... .......... 40%  364M 1s\n",
            "  7150K .......... .......... .......... .......... .......... 40%  149M 1s\n",
            "  7200K .......... .......... .......... .......... .......... 40% 53.2M 1s\n",
            "  7250K .......... .......... .......... .......... .......... 41% 55.9M 1s\n",
            "  7300K .......... .......... .......... .......... .......... 41% 58.2M 1s\n",
            "  7350K .......... .......... .......... .......... .......... 41% 60.0M 1s\n",
            "  7400K .......... .......... .......... .......... .......... 41% 90.9M 1s\n",
            "  7450K .......... .......... .......... .......... .......... 42% 84.5M 1s\n",
            "  7500K .......... .......... .......... .......... .......... 42% 84.1M 1s\n",
            "  7550K .......... .......... .......... .......... .......... 42% 46.5M 1s\n",
            "  7600K .......... .......... .......... .......... .......... 43%  153M 1s\n",
            "  7650K .......... .......... .......... .......... .......... 43%  355M 1s\n",
            "  7700K .......... .......... .......... .......... .......... 43% 31.0M 1s\n",
            "  7750K .......... .......... .......... .......... .......... 43% 41.8M 1s\n",
            "  7800K .......... .......... .......... .......... .......... 44% 58.2M 1s\n",
            "  7850K .......... .......... .......... .......... .......... 44% 45.3M 1s\n",
            "  7900K .......... .......... .......... .......... .......... 44%  340M 1s\n",
            "  7950K .......... .......... .......... .......... .......... 44% 33.9M 1s\n",
            "  8000K .......... .......... .......... .......... .......... 45% 49.9M 1s\n",
            "  8050K .......... .......... .......... .......... .......... 45% 90.9M 1s\n",
            "  8100K .......... .......... .......... .......... .......... 45%  368M 1s\n",
            "  8150K .......... .......... .......... .......... .......... 46%  375M 1s\n",
            "  8200K .......... .......... .......... .......... .......... 46% 25.0M 1s\n",
            "  8250K .......... .......... .......... .......... .......... 46% 43.7M 1s\n",
            "  8300K .......... .......... .......... .......... .......... 46% 42.6M 1s\n",
            "  8350K .......... .......... .......... .......... .......... 47%  317M 1s\n",
            "  8400K .......... .......... .......... .......... .......... 47% 39.1M 1s\n",
            "  8450K .......... .......... .......... .......... .......... 47% 39.2M 1s\n",
            "  8500K .......... .......... .......... .......... .......... 48% 99.2M 1s\n",
            "  8550K .......... .......... .......... .......... .......... 48% 55.6M 1s\n",
            "  8600K .......... .......... .......... .......... .......... 48% 33.1M 1s\n",
            "  8650K .......... .......... .......... .......... .......... 48%  123M 1s\n",
            "  8700K .......... .......... .......... .......... .......... 49% 45.0M 1s\n",
            "  8750K .......... .......... .......... .......... .......... 49% 45.9M 1s\n",
            "  8800K .......... .......... .......... .......... .......... 49% 42.8M 1s\n",
            "  8850K .......... .......... .......... .......... .......... 50%  199M 1s\n",
            "  8900K .......... .......... .......... .......... .......... 50% 1.80M 1s\n",
            "  8950K .......... .......... .......... .......... .......... 50%  197M 1s\n",
            "  9000K .......... .......... .......... .......... .......... 50%  138M 1s\n",
            "  9050K .......... .......... .......... .......... .......... 51%  141M 1s\n",
            "  9100K .......... .......... .......... .......... .......... 51%  131M 1s\n",
            "  9150K .......... .......... .......... .......... .......... 51% 65.7M 1s\n",
            "  9200K .......... .......... .......... .......... .......... 52%  131M 1s\n",
            "  9250K .......... .......... .......... .......... .......... 52% 53.9M 1s\n",
            "  9300K .......... .......... .......... .......... .......... 52%  276M 1s\n",
            "  9350K .......... .......... .......... .......... .......... 52%  362M 1s\n",
            "  9400K .......... .......... .......... .......... .......... 53% 68.6M 1s\n",
            "  9450K .......... .......... .......... .......... .......... 53% 83.0M 1s\n",
            "  9500K .......... .......... .......... .......... .......... 53%  287M 1s\n",
            "  9550K .......... .......... .......... .......... .......... 53%  340M 1s\n",
            "  9600K .......... .......... .......... .......... .......... 54%  356M 1s\n",
            "  9650K .......... .......... .......... .......... .......... 54%  153M 1s\n",
            "  9700K .......... .......... .......... .......... .......... 54% 57.6M 1s\n",
            "  9750K .......... .......... .......... .......... .......... 55% 65.5M 1s\n",
            "  9800K .......... .......... .......... .......... .......... 55% 65.4M 1s\n",
            "  9850K .......... .......... .......... .......... .......... 55% 65.6M 1s\n",
            "  9900K .......... .......... .......... .......... .......... 55%  104M 1s\n",
            "  9950K .......... .......... .......... .......... .......... 56%  138M 1s\n",
            " 10000K .......... .......... .......... .......... .......... 56%  343M 1s\n",
            " 10050K .......... .......... .......... .......... .......... 56% 70.8M 1s\n",
            " 10100K .......... .......... .......... .......... .......... 57% 66.9M 1s\n",
            " 10150K .......... .......... .......... .......... .......... 57%  357M 1s\n",
            " 10200K .......... .......... .......... .......... .......... 57%  308M 1s\n",
            " 10250K .......... .......... .......... .......... .......... 57% 67.0M 1s\n",
            " 10300K .......... .......... .......... .......... .......... 58% 26.8M 0s\n",
            " 10350K .......... .......... .......... .......... .......... 58% 71.1M 0s\n",
            " 10400K .......... .......... .......... .......... .......... 58% 97.9M 0s\n",
            " 10450K .......... .......... .......... .......... .......... 59% 23.5M 0s\n",
            " 10500K .......... .......... .......... .......... .......... 59%  302M 0s\n",
            " 10550K .......... .......... .......... .......... .......... 59%  315M 0s\n",
            " 10600K .......... .......... .......... .......... .......... 59% 85.3M 0s\n",
            " 10650K .......... .......... .......... .......... .......... 60%  469K 1s\n",
            " 10700K .......... .......... .......... .......... .......... 60%  331M 1s\n",
            " 10750K .......... .......... .......... .......... .......... 60%  358M 1s\n",
            " 10800K .......... .......... .......... .......... .......... 61%  368M 1s\n",
            " 10850K .......... .......... .......... .......... .......... 61%  235M 1s\n",
            " 10900K .......... .......... .......... .......... .......... 61% 75.8M 1s\n",
            " 10950K .......... .......... .......... .......... .......... 61%  352M 0s\n",
            " 11000K .......... .......... .......... .......... .......... 62%  343M 0s\n",
            " 11050K .......... .......... .......... .......... .......... 62%  250M 0s\n",
            " 11100K .......... .......... .......... .......... .......... 62%  367M 0s\n",
            " 11150K .......... .......... .......... .......... .......... 62%  322M 0s\n",
            " 11200K .......... .......... .......... .......... .......... 63%  361M 0s\n",
            " 11250K .......... .......... .......... .......... .......... 63%  315M 0s\n",
            " 11300K .......... .......... .......... .......... .......... 63%  347M 0s\n",
            " 11350K .......... .......... .......... .......... .......... 64%  321M 0s\n",
            " 11400K .......... .......... .......... .......... .......... 64%  371M 0s\n",
            " 11450K .......... .......... .......... .......... .......... 64%  318M 0s\n",
            " 11500K .......... .......... .......... .......... .......... 64%  324M 0s\n",
            " 11550K .......... .......... .......... .......... .......... 65%  351M 0s\n",
            " 11600K .......... .......... .......... .......... .......... 65%  365M 0s\n",
            " 11650K .......... .......... .......... .......... .......... 65%  316M 0s\n",
            " 11700K .......... .......... .......... .......... .......... 66%  114M 0s\n",
            " 11750K .......... .......... .......... .......... .......... 66% 71.5M 0s\n",
            " 11800K .......... .......... .......... .......... .......... 66% 73.4M 0s\n",
            " 11850K .......... .......... .......... .......... .......... 66% 65.0M 0s\n",
            " 11900K .......... .......... .......... .......... .......... 67% 75.1M 0s\n",
            " 11950K .......... .......... .......... .......... .......... 67% 43.0M 0s\n",
            " 12000K .......... .......... .......... .......... .......... 67% 45.3M 0s\n",
            " 12050K .......... .......... .......... .......... .......... 68% 29.2M 0s\n",
            " 12100K .......... .......... .......... .......... .......... 68% 15.7M 0s\n",
            " 12150K .......... .......... .......... .......... .......... 68% 73.1M 0s\n",
            " 12200K .......... .......... .......... .......... .......... 68% 71.5M 0s\n",
            " 12250K .......... .......... .......... .......... .......... 69% 40.9M 0s\n",
            " 12300K .......... .......... .......... .......... .......... 69% 18.4M 0s\n",
            " 12350K .......... .......... .......... .......... .......... 69% 40.0M 0s\n",
            " 12400K .......... .......... .......... .......... .......... 69% 96.9M 0s\n",
            " 12450K .......... .......... .......... .......... .......... 70%  294M 0s\n",
            " 12500K .......... .......... .......... .......... .......... 70% 49.2M 0s\n",
            " 12550K .......... .......... .......... .......... .......... 70% 60.9M 0s\n",
            " 12600K .......... .......... .......... .......... .......... 71% 82.3M 0s\n",
            " 12650K .......... .......... .......... .......... .......... 71%  295M 0s\n",
            " 12700K .......... .......... .......... .......... .......... 71%  356M 0s\n",
            " 12750K .......... .......... .......... .......... .......... 71%  314M 0s\n",
            " 12800K .......... .......... .......... .......... .......... 72%  332M 0s\n",
            " 12850K .......... .......... .......... .......... .......... 72%  313M 0s\n",
            " 12900K .......... .......... .......... .......... .......... 72%  338M 0s\n",
            " 12950K .......... .......... .......... .......... .......... 73%  352M 0s\n",
            " 13000K .......... .......... .......... .......... .......... 73%  374M 0s\n",
            " 13050K .......... .......... .......... .......... .......... 73%  309M 0s\n",
            " 13100K .......... .......... .......... .......... .......... 73%  322M 0s\n",
            " 13150K .......... .......... .......... .......... .......... 74%  372M 0s\n",
            " 13200K .......... .......... .......... .......... .......... 74%  348M 0s\n",
            " 13250K .......... .......... .......... .......... .......... 74%  283M 0s\n",
            " 13300K .......... .......... .......... .......... .......... 75%  370M 0s\n",
            " 13350K .......... .......... .......... .......... .......... 75%  367M 0s\n",
            " 13400K .......... .......... .......... .......... .......... 75%  348M 0s\n",
            " 13450K .......... .......... .......... .......... .......... 75% 42.0M 0s\n",
            " 13500K .......... .......... .......... .......... .......... 76% 33.6M 0s\n",
            " 13550K .......... .......... .......... .......... .......... 76% 35.6M 0s\n",
            " 13600K .......... .......... .......... .......... .......... 76% 26.4M 0s\n",
            " 13650K .......... .......... .......... .......... .......... 77% 41.3M 0s\n",
            " 13700K .......... .......... .......... .......... .......... 77% 12.6M 0s\n",
            " 13750K .......... .......... .......... .......... .......... 77% 63.4M 0s\n",
            " 13800K .......... .......... .......... .......... .......... 77% 57.4M 0s\n",
            " 13850K .......... .......... .......... .......... .......... 78% 59.0M 0s\n",
            " 13900K .......... .......... .......... .......... .......... 78% 70.2M 0s\n",
            " 13950K .......... .......... .......... .......... .......... 78% 2.20M 0s\n",
            " 14000K .......... .......... .......... .......... .......... 78% 88.7M 0s\n",
            " 14050K .......... .......... .......... .......... .......... 79% 46.7M 0s\n",
            " 14100K .......... .......... .......... .......... .......... 79% 39.1M 0s\n",
            " 14150K .......... .......... .......... .......... .......... 79% 70.0M 0s\n",
            " 14200K .......... .......... .......... .......... .......... 80% 62.6M 0s\n",
            " 14250K .......... .......... .......... .......... .......... 80% 74.3M 0s\n",
            " 14300K .......... .......... .......... .......... .......... 80% 73.2M 0s\n",
            " 14350K .......... .......... .......... .......... .......... 80% 70.9M 0s\n",
            " 14400K .......... .......... .......... .......... .......... 81% 63.2M 0s\n",
            " 14450K .......... .......... .......... .......... .......... 81% 73.2M 0s\n",
            " 14500K .......... .......... .......... .......... .......... 81%  132M 0s\n",
            " 14550K .......... .......... .......... .......... .......... 82%  298M 0s\n",
            " 14600K .......... .......... .......... .......... .......... 82%  291M 0s\n",
            " 14650K .......... .......... .......... .......... .......... 82%  347M 0s\n",
            " 14700K .......... .......... .......... .......... .......... 82%  339M 0s\n",
            " 14750K .......... .......... .......... .......... .......... 83%  310M 0s\n",
            " 14800K .......... .......... .......... .......... .......... 83%  311M 0s\n",
            " 14850K .......... .......... .......... .......... .......... 83%  359M 0s\n",
            " 14900K .......... .......... .......... .......... .......... 84%  324M 0s\n",
            " 14950K .......... .......... .......... .......... .......... 84%  365M 0s\n",
            " 15000K .......... .......... .......... .......... .......... 84%  321M 0s\n",
            " 15050K .......... .......... .......... .......... .......... 84%  369M 0s\n",
            " 15100K .......... .......... .......... .......... .......... 85%  289M 0s\n",
            " 15150K .......... .......... .......... .......... .......... 85%  341M 0s\n",
            " 15200K .......... .......... .......... .......... .......... 85%  308M 0s\n",
            " 15250K .......... .......... .......... .......... .......... 86%  332M 0s\n",
            " 15300K .......... .......... .......... .......... .......... 86%  365M 0s\n",
            " 15350K .......... .......... .......... .......... .......... 86%  356M 0s\n",
            " 15400K .......... .......... .......... .......... .......... 86%  319M 0s\n",
            " 15450K .......... .......... .......... .......... .......... 87% 92.1M 0s\n",
            " 15500K .......... .......... .......... .......... .......... 87% 67.9M 0s\n",
            " 15550K .......... .......... .......... .......... .......... 87% 71.4M 0s\n",
            " 15600K .......... .......... .......... .......... .......... 87% 58.5M 0s\n",
            " 15650K .......... .......... .......... .......... .......... 88% 69.5M 0s\n",
            " 15700K .......... .......... .......... .......... .......... 88% 69.3M 0s\n",
            " 15750K .......... .......... .......... .......... .......... 88% 68.5M 0s\n",
            " 15800K .......... .......... .......... .......... .......... 89% 62.0M 0s\n",
            " 15850K .......... .......... .......... .......... .......... 89% 3.32M 0s\n",
            " 15900K .......... .......... .......... .......... .......... 89%  197M 0s\n",
            " 15950K .......... .......... .......... .......... .......... 89%  163M 0s\n",
            " 16000K .......... .......... .......... .......... .......... 90%  341M 0s\n",
            " 16050K .......... .......... .......... .......... .......... 90%  332M 0s\n",
            " 16100K .......... .......... .......... .......... .......... 90% 41.0M 0s\n",
            " 16150K .......... .......... .......... .......... .......... 91% 61.4M 0s\n",
            " 16200K .......... .......... .......... .......... .......... 91% 73.8M 0s\n",
            " 16250K .......... .......... .......... .......... .......... 91% 72.3M 0s\n",
            " 16300K .......... .......... .......... .......... .......... 91% 74.1M 0s\n",
            " 16350K .......... .......... .......... .......... .......... 92% 65.2M 0s\n",
            " 16400K .......... .......... .......... .......... .......... 92% 68.5M 0s\n",
            " 16450K .......... .......... .......... .......... .......... 92% 72.9M 0s\n",
            " 16500K .......... .......... .......... .......... .......... 93% 75.7M 0s\n",
            " 16550K .......... .......... .......... .......... .......... 93% 83.6M 0s\n",
            " 16600K .......... .......... .......... .......... .......... 93%  338M 0s\n",
            " 16650K .......... .......... .......... .......... .......... 93%  344M 0s\n",
            " 16700K .......... .......... .......... .......... .......... 94%  363M 0s\n",
            " 16750K .......... .......... .......... .......... .......... 94%  277M 0s\n",
            " 16800K .......... .......... .......... .......... .......... 94%  334M 0s\n",
            " 16850K .......... .......... .......... .......... .......... 95% 2.29M 0s\n",
            " 16900K .......... .......... .......... .......... .......... 95%  253M 0s\n",
            " 16950K .......... .......... .......... .......... .......... 95%  350M 0s\n",
            " 17000K .......... .......... .......... .......... .......... 95%  140M 0s\n",
            " 17050K .......... .......... .......... .......... .......... 96%  160M 0s\n",
            " 17100K .......... .......... .......... .......... .......... 96%  194M 0s\n",
            " 17150K .......... .......... .......... .......... .......... 96%  171M 0s\n",
            " 17200K .......... .......... .......... .......... .......... 96%  245M 0s\n",
            " 17250K .......... .......... .......... .......... .......... 97%  315M 0s\n",
            " 17300K .......... .......... .......... .......... .......... 97%  353M 0s\n",
            " 17350K .......... .......... .......... .......... .......... 97%  365M 0s\n",
            " 17400K .......... .......... .......... .......... .......... 98%  314M 0s\n",
            " 17450K .......... .......... .......... .......... .......... 98%  344M 0s\n",
            " 17500K .......... .......... .......... .......... .......... 98%  262M 0s\n",
            " 17550K .......... .......... .......... .......... .......... 98%  289M 0s\n",
            " 17600K .......... .......... .......... .......... .......... 99%  281M 0s\n",
            " 17650K .......... .......... .......... .......... .......... 99%  328M 0s\n",
            " 17700K .......... .......... .......... .......... .......... 99%  284M 0s\n",
            " 17750K .......... .......... .......... ......               100%  324M=0.9s\n",
            "\n",
            "2025-06-18 08:09:23 (18.7 MB/s) - ‘hmmer-3.3.2.tar.gz’ saved [18213049/18213049]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "wget http://eddylab.org/software/hmmer/hmmer-3.3.2.tar.gz\n",
        "tar xf hmmer-3.3.2.tar.gz\n",
        "cd hmmer-3.3.2"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "rAv0w_ccEyLH",
        "outputId": "38efdd1a-46e0-484a-b5ee-0685b7a0dbdf"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "configure: Configuring HMMER3 for your system.\n",
            "checking build system type... x86_64-pc-linux-gnu\n",
            "checking host system type... x86_64-pc-linux-gnu\n",
            "checking whether to compile using MPI... no\n",
            "checking for gcc... gcc\n",
            "checking whether the C compiler works... yes\n",
            "checking for C compiler default output file name... a.out\n",
            "checking for suffix of executables... \n",
            "checking whether we are cross compiling... no\n",
            "checking for suffix of object files... o\n",
            "checking whether we are using the GNU C compiler... yes\n",
            "checking whether gcc accepts -g... yes\n",
            "checking for gcc option to accept ISO C89... none needed\n",
            "checking for gcc option to accept ISO C99... none needed\n",
            "checking for gcc option to accept ISO Standard C... (cached) none needed\n",
            "checking how to run the C preprocessor... gcc -E\n",
            "checking for a BSD-compatible install... /usr/bin/install -c\n",
            "checking for strip... strip\n",
            "checking for ranlib... ranlib\n",
            "checking for ar... /usr/bin/ar\n",
            "checking whether ln -s works... yes\n",
            "checking for a sed that does not truncate output... /usr/bin/sed\n",
            "checking for grep that handles long lines and -e... /usr/bin/grep\n",
            "checking for egrep... /usr/bin/grep -E\n",
            "checking whether gcc is Clang... no\n",
            "checking whether pthreads work with -pthread... yes\n",
            "checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE\n",
            "checking whether more special flags are required for pthreads... no\n",
            "checking for PTHREAD_PRIO_INHERIT... yes\n",
            "checking whether gcc can compile our Altivec/VMX code... no\n",
            "checking whether gcc can compile our SSE code... yes\n",
            "configure: Activating Intel/AMD SSE vector DP implementation\n",
            "checking whether _MM_SET_FLUSH_ZERO_MODE is supported... yes\n",
            "checking compiler support --start-group... yes\n",
            "checking for gzip... /usr/bin/gzip\n",
            "checking for python3... yes\n",
            "checking for library containing clock_gettime... none required\n",
            "checking for ANSI C header files... yes\n",
            "checking for sys/types.h... yes\n",
            "checking for sys/stat.h... yes\n",
            "checking for stdlib.h... yes\n",
            "checking for string.h... yes\n",
            "checking for memory.h... yes\n",
            "checking for strings.h... yes\n",
            "checking for inttypes.h... yes\n",
            "checking for stdint.h... yes\n",
            "checking for unistd.h... yes\n",
            "checking endian.h usability... yes\n",
            "checking endian.h presence... yes\n",
            "checking for endian.h... yes\n",
            "checking for inttypes.h... (cached) yes\n",
            "checking for stdint.h... (cached) yes\n",
            "checking for unistd.h... (cached) yes\n",
            "checking for sys/types.h... (cached) yes\n",
            "checking netinet/in.h usability... yes\n",
            "checking netinet/in.h presence... yes\n",
            "checking for netinet/in.h... yes\n",
            "checking sys/param.h usability... yes\n",
            "checking sys/param.h presence... yes\n",
            "checking for sys/param.h... yes\n",
            "checking for sys/sysctl.h... no\n",
            "checking for uint8_t... yes\n",
            "checking for uint16_t... yes\n",
            "checking for uint32_t... yes\n",
            "checking for uint64_t... yes\n",
            "checking for off_t... yes\n",
            "checking whether byte ordering is bigendian... no\n",
            "checking for __attribute__((noreturn))... yes\n",
            "checking for __attribute__((format))... yes\n",
            "checking for mkstemp... yes\n",
            "checking for popen... yes\n",
            "checking for putenv... yes\n",
            "checking for strcasecmp... yes\n",
            "checking for strsep... yes\n",
            "checking for times... yes\n",
            "checking for getpid... yes\n",
            "checking for sysctl... no\n",
            "checking for sysconf... yes\n",
            "checking for getcwd... yes\n",
            "checking for chmod... yes\n",
            "checking for stat... yes\n",
            "checking for fstat... yes\n",
            "checking for erfc... no\n",
            "checking for library containing ntohs... none required\n",
            "checking for library containing ntohl... none required\n",
            "checking for library containing htons... none required\n",
            "checking for library containing htonl... none required\n",
            "checking for library containing socket... none required\n",
            "checking for library containing inet_pton... none required\n",
            "checking for _LARGEFILE_SOURCE value needed for large files... no\n",
            "checking for special C compiler options needed for large files... no\n",
            "checking for _FILE_OFFSET_BITS value needed for large files... no\n",
            "configure: creating ./config.status\n",
            "config.status: creating Makefile\n",
            "config.status: creating src/Makefile\n",
            "config.status: creating testsuite/Makefile\n",
            "config.status: creating profmark/Makefile\n",
            "config.status: creating src/impl_sse/Makefile\n",
            "config.status: creating documentation/Makefile\n",
            "config.status: creating documentation/man/Makefile\n",
            "config.status: creating documentation/userguide/Makefile\n",
            "config.status: creating documentation/userguide/inclusions/Makefile\n",
            "config.status: creating libdivsufsort/Makefile\n",
            "config.status: creating easel/Makefile\n",
            "config.status: creating easel/miniapps/Makefile\n",
            "config.status: creating easel/testsuite/Makefile\n",
            "config.status: creating easel/documentation/Makefile\n",
            "config.status: creating documentation/userguide/titlepage.tex\n",
            "config.status: creating documentation/userguide/titlepage_daemon.tex\n",
            "config.status: creating documentation/userguide/copyright.tex\n",
            "config.status: creating documentation/man/alimask.man\n",
            "config.status: creating documentation/man/hmmalign.man\n",
            "config.status: creating documentation/man/hmmbuild.man\n",
            "config.status: creating documentation/man/hmmc2.man\n",
            "config.status: creating documentation/man/hmmconvert.man\n",
            "config.status: creating documentation/man/hmmemit.man\n",
            "config.status: creating documentation/man/hmmer.man\n",
            "config.status: creating documentation/man/hmmfetch.man\n",
            "config.status: creating documentation/man/hmmlogo.man\n",
            "config.status: creating documentation/man/hmmpgmd.man\n",
            "config.status: creating documentation/man/hmmpgmd_shard.man\n",
            "config.status: creating documentation/man/hmmpress.man\n",
            "config.status: creating documentation/man/hmmscan.man\n",
            "config.status: creating documentation/man/hmmsearch.man\n",
            "config.status: creating documentation/man/hmmsim.man\n",
            "config.status: creating documentation/man/hmmstat.man\n",
            "config.status: creating documentation/man/jackhmmer.man\n",
            "config.status: creating documentation/man/makehmmerdb.man\n",
            "config.status: creating documentation/man/nhmmer.man\n",
            "config.status: creating documentation/man/nhmmscan.man\n",
            "config.status: creating documentation/man/phmmer.man\n",
            "config.status: creating easel/miniapps/esl-afetch.man\n",
            "config.status: creating easel/miniapps/esl-alimanip.man\n",
            "config.status: creating easel/miniapps/esl-alimap.man\n",
            "config.status: creating easel/miniapps/esl-alimask.man\n",
            "config.status: creating easel/miniapps/esl-alimerge.man\n",
            "config.status: creating easel/miniapps/esl-alipid.man\n",
            "config.status: creating easel/miniapps/esl-alirev.man\n",
            "config.status: creating easel/miniapps/esl-alistat.man\n",
            "config.status: creating easel/miniapps/esl-compalign.man\n",
            "config.status: creating easel/miniapps/esl-compstruct.man\n",
            "config.status: creating easel/miniapps/esl-construct.man\n",
            "config.status: creating easel/miniapps/esl-histplot.man\n",
            "config.status: creating easel/miniapps/esl-mask.man\n",
            "config.status: creating easel/miniapps/esl-mixdchlet.man\n",
            "config.status: creating easel/miniapps/esl-reformat.man\n",
            "config.status: creating easel/miniapps/esl-selectn.man\n",
            "config.status: creating easel/miniapps/esl-seqrange.man\n",
            "config.status: creating easel/miniapps/esl-seqstat.man\n",
            "config.status: creating easel/miniapps/esl-sfetch.man\n",
            "config.status: creating easel/miniapps/esl-shuffle.man\n",
            "config.status: creating easel/miniapps/esl-ssdraw.man\n",
            "config.status: creating easel/miniapps/esl-translate.man\n",
            "config.status: creating easel/miniapps/esl-weight.man\n",
            "config.status: creating easel/decoy_config.h\n",
            "config.status: creating src/p7_config.h\n",
            "config.status: creating easel/esl_config.h\n",
            "config.status: creating libdivsufsort/divsufsort.h\n",
            "config.status: linking src/impl_sse to src/impl\n",
            "\n",
            "\n",
            "HMMER configuration:\n",
            "   compiler:             gcc -O3   -pthread \n",
            "   host:                 x86_64-pc-linux-gnu\n",
            "   linker:               \n",
            "   libraries:              \n",
            "   DP implementation:    sse\n",
            "\n",
            "Now do 'make'  to build HMMER, and optionally:\n",
            "       'make check'  to run self tests,\n",
            "       'make install'  to install programs and man pages,\n",
            "       '(cd easel; make install)'  to install Easel tools.\n",
            "\n",
            "     SUBDIR easel\n",
            "     CC easel.o\n",
            "     CC esl_alloc.o\n",
            "     CC esl_alphabet.o\n",
            "     CC esl_arr2.o\n",
            "     CC esl_arr3.o\n",
            "     CC esl_bitfield.o\n",
            "     CC esl_buffer.o\n",
            "     CC esl_cluster.o\n",
            "     CC esl_composition.o\n",
            "     CC esl_cpu.o\n",
            "     CC esl_dirichlet.o\n",
            "     CC esl_distance.o\n",
            "     CC esl_dmatrix.o\n",
            "     CC esl_dsqdata.o\n",
            "     CC esl_exponential.o\n",
            "     CC esl_fileparser.o\n",
            "     CC esl_gamma.o\n",
            "     CC esl_gencode.o\n",
            "     CC esl_getopts.o\n",
            "     CC esl_gev.o\n",
            "     CC esl_graph.o\n",
            "     CC esl_gumbel.o\n",
            "     CC esl_heap.o\n",
            "     CC esl_histogram.o\n",
            "     CC esl_hmm.o\n",
            "     CC esl_huffman.o\n",
            "     CC esl_hyperexp.o\n",
            "     CC esl_json.o\n",
            "     CC esl_keyhash.o\n",
            "     CC esl_matrixops.o\n",
            "     CC esl_mem.o\n",
            "     CC esl_minimizer.o\n",
            "     CC esl_mixdchlet.o\n",
            "     CC esl_mixgev.o\n",
            "     CC esl_mpi.o\n",
            "     CC esl_msa.o\n",
            "     CC esl_msacluster.o\n",
            "     CC esl_msafile.o\n",
            "     CC esl_msafile2.o\n",
            "     CC esl_msafile_a2m.o\n",
            "     CC esl_msafile_afa.o\n",
            "     CC esl_msafile_clustal.o\n",
            "     CC esl_msafile_phylip.o\n",
            "     CC esl_msafile_psiblast.o\n",
            "     CC esl_msafile_selex.o\n",
            "     CC esl_msafile_stockholm.o\n",
            "     CC esl_msashuffle.o\n",
            "     CC esl_msaweight.o\n",
            "     CC esl_normal.o\n",
            "     CC esl_paml.o\n",
            "     CC esl_quicksort.o\n",
            "     CC esl_random.o\n",
            "     CC esl_rand64.o\n",
            "     CC esl_randomseq.o\n",
            "     CC esl_ratematrix.o\n",
            "     CC esl_recorder.o\n",
            "     CC esl_red_black.o\n",
            "     CC esl_regexp.o\n",
            "     CC esl_rootfinder.o\n",
            "     CC esl_scorematrix.o\n",
            "     CC esl_sq.o\n",
            "     CC esl_sqio.o\n",
            "     CC esl_sqio_ascii.o\n",
            "     CC esl_sqio_ncbi.o\n",
            "     CC esl_ssi.o\n",
            "     CC esl_stack.o\n",
            "     CC esl_stats.o\n",
            "     CC esl_stopwatch.o\n",
            "     CC esl_stretchexp.o\n",
            "     CC esl_subcmd.o\n",
            "     CC esl_threads.o\n",
            "     CC esl_tree.o\n",
            "     CC esl_varint.o\n",
            "     CC esl_vectorops.o\n",
            "     CC esl_weibull.o\n",
            "     CC esl_workqueue.o\n",
            "     CC esl_wuss.o\n",
            "     CC esl_sse.o\n",
            "     CC esl_avx.o\n",
            "     CC esl_avx512.o\n",
            "     CC esl_neon.o\n",
            "     CC esl_vmx.o\n",
            "     AR libeasel.a\n",
            "     SUBDIR miniapps\n",
            "     GEN esl-afetch\n",
            "     GEN esl-alimanip\n",
            "     GEN esl-alimap\n",
            "     GEN esl-alimask\n",
            "     GEN esl-alimerge\n",
            "     GEN esl-alipid\n",
            "     GEN esl-alirev\n",
            "     GEN esl-alistat\n",
            "     GEN esl-compalign\n",
            "     GEN esl-compstruct\n",
            "     GEN esl-construct\n",
            "     GEN esl-histplot\n",
            "     GEN esl-mask\n",
            "     GEN esl-mixdchlet\n",
            "     GEN esl-reformat\n",
            "     GEN esl-selectn\n",
            "     GEN esl-seqrange\n",
            "     GEN esl-seqstat\n",
            "     GEN esl-sfetch\n",
            "     GEN esl-shuffle\n",
            "     GEN esl-ssdraw\n",
            "     GEN esl-translate\n",
            "     GEN esl-weight\n",
            "     CC cmd_alistat.o\n",
            "     CC cmd_downsample.o\n",
            "     CC cmd_filter.o\n",
            "     CC cmd_index.o\n",
            "     GEN easel\n",
            "     SUBDIR libdivsufsort\n",
            "     CC divsufsort.o\n",
            "     AR libdivsufsort.a\n",
            "     SUBDIR src\n",
            "     CC alimask.o\n",
            "     CC build.o\n",
            "     CC cachedb.o\n",
            "     CC cachedb_shard.o\n",
            "     CC emit.o\n",
            "     CC errors.o\n",
            "     CC evalues.o\n",
            "     CC eweight.o\n",
            "     CC generic_decoding.o\n",
            "     CC generic_fwdback.o\n",
            "     CC generic_fwdback_chk.o\n",
            "     CC generic_fwdback_banded.o\n",
            "     CC generic_null2.o\n",
            "     CC generic_msv.o\n",
            "     CC generic_optacc.o\n",
            "     CC generic_stotrace.o\n",
            "     CC generic_viterbi.o\n",
            "     CC generic_vtrace.o\n",
            "     CC h2_io.o\n",
            "     CC heatmap.o\n",
            "     CC hmmlogo.o\n",
            "     CC hmmdmstr.o\n",
            "     CC hmmdmstr_shard.o\n",
            "     CC hmmd_search_status.o\n",
            "     CC hmmdwrkr.o\n",
            "     CC hmmdwrkr_shard.o\n",
            "     CC hmmdutils.o\n",
            "     CC hmmer.o\n",
            "     CC logsum.o\n",
            "     CC modelconfig.o\n",
            "     CC modelstats.o\n",
            "     CC mpisupport.o\n",
            "     CC seqmodel.o\n",
            "     CC tracealign.o\n",
            "     CC p7_alidisplay.o\n",
            "     CC p7_bg.o\n",
            "     CC p7_builder.o\n",
            "     CC p7_domain.o\n",
            "     CC p7_domaindef.o\n",
            "     CC p7_gbands.o\n",
            "     CC p7_gmx.o\n",
            "     CC p7_gmxb.o\n",
            "     CC p7_gmxchk.o\n",
            "     CC p7_hit.o\n",
            "     CC p7_hmm.o\n",
            "     CC p7_hmmcache.o\n",
            "     CC p7_hmmd_search_stats.o\n",
            "     CC p7_hmmfile.o\n",
            "     CC p7_hmmwindow.o\n",
            "     CC p7_pipeline.o\n",
            "     CC p7_prior.o\n",
            "     CC p7_profile.o\n",
            "     CC p7_spensemble.o\n",
            "     CC p7_tophits.o\n",
            "     CC p7_trace.o\n",
            "     CC p7_scoredata.o\n",
            "     CC hmmpgmd2msa.o\n",
            "     CC fm_alphabet.o\n",
            "     CC fm_general.o\n",
            "     CC fm_sse.o\n",
            "     CC fm_ssv.o\n",
            "     AR libhmmer-src.stamp\n",
            "     SUBDIR impl_sse\n",
            "     CC decoding.o\n",
            "     CC fwdback.o\n",
            "     CC io.o\n",
            "     CC ssvfilter.o\n",
            "     CC msvfilter.o\n",
            "     CC null2.o\n",
            "     CC optacc.o\n",
            "     CC stotrace.o\n",
            "     CC vitfilter.o\n",
            "     CC p7_omx.o\n",
            "     CC p7_oprofile.o\n",
            "     CC mpi.o\n",
            "     AR libhmmer-impl.stamp\n",
            "     GEN alimask\n",
            "     CC hmmalign.o\n",
            "     GEN hmmalign\n",
            "     CC hmmbuild.o\n",
            "     GEN hmmbuild\n",
            "     CC hmmconvert.o\n",
            "     GEN hmmconvert\n",
            "     CC hmmemit.o\n",
            "     GEN hmmemit\n",
            "     CC hmmfetch.o\n",
            "     GEN hmmfetch\n",
            "     GEN hmmlogo\n",
            "     CC hmmpgmd.o\n",
            "     GEN hmmpgmd\n",
            "     CC hmmpgmd_shard.o\n",
            "     GEN hmmpgmd_shard\n",
            "     CC hmmpress.o\n",
            "     GEN hmmpress\n",
            "     CC hmmscan.o\n",
            "     GEN hmmscan\n",
            "     CC hmmsearch.o\n",
            "     GEN hmmsearch\n",
            "     CC hmmsim.o\n",
            "     GEN hmmsim\n",
            "     CC hmmstat.o\n",
            "     GEN hmmstat\n",
            "     CC jackhmmer.o\n",
            "     GEN jackhmmer\n",
            "     CC phmmer.o\n",
            "     GEN phmmer\n",
            "     CC nhmmer.o\n",
            "     GEN nhmmer\n",
            "     CC nhmmscan.o\n",
            "     GEN nhmmscan\n",
            "     CC makehmmerdb.o\n",
            "     GEN makehmmerdb\n",
            "     CC hmmc2.o\n",
            "     GEN hmmc2\n",
            "     CC hmmerfm-exactmatch.o\n",
            "     GEN hmmerfm-exactmatch\n",
            "     SUBDIR profmark\n",
            "     CC create-profmark.o\n",
            "     GEN create-profmark\n",
            "     CC rocplot.o\n",
            "     GEN rocplot\n",
            "     SUBDIR easel\n",
            "     SUBDIR miniapps\n",
            "     SUBDIR libdivsufsort\n",
            "     SUBDIR src\n",
            "     SUBDIR impl_sse\n",
            "     SUBDIR profmark\n",
            "/usr/bin/install -c -d /usr/local/bin\n",
            "/usr/bin/install -c -d /usr/local/share/man/man1\n",
            "make -C src install\n",
            "make[1]: Entering directory '/content/hmmer-3.3.2/src'\n",
            "for file in alimask hmmalign hmmbuild hmmconvert hmmemit hmmfetch hmmlogo hmmpgmd hmmpgmd_shard hmmpress hmmscan hmmsearch hmmsim hmmstat jackhmmer phmmer nhmmer nhmmscan makehmmerdb; do \\\n",
            "   /usr/bin/install -c -m 0755 $file /usr/local/bin/ ;\\\n",
            "done\n",
            "make[1]: Leaving directory '/content/hmmer-3.3.2/src'\n",
            "make -C documentation install\n",
            "make[1]: Entering directory '/content/hmmer-3.3.2/documentation'\n",
            "make -C man install\n",
            "make[2]: Entering directory '/content/hmmer-3.3.2/documentation/man'\n",
            "for file in hmmer hmmalign hmmbuild hmmc2 hmmconvert hmmemit hmmfetch hmmlogo hmmpgmd hmmpgmd_shard hmmpress hmmscan hmmsearch hmmsim hmmstat jackhmmer makehmmerdb phmmer nhmmer nhmmscan alimask; do \\\n",
            "   /usr/bin/install -c -m 0664 $file.man /usr/local/share/man/man1/${file}.1 ;\\\n",
            "done\n",
            "make[2]: Leaving directory '/content/hmmer-3.3.2/documentation/man'\n",
            "make[1]: Leaving directory '/content/hmmer-3.3.2/documentation'\n"
          ]
        },
        {
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "In file included from hmmbuild.c:13:\n",
            "In function ‘usual_master’,\n",
            "    inlined from ‘main’ at hmmbuild.c:463:7:\n",
            "../easel/easel.h:164:17: warning: argument 1 range [18446744022169944064, 18446744073709551592] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]\n",
            "  164 |     if ( ((p) = malloc(size)) == NULL)  { \\\n",
            "      |                 ^~~~~~~~~~~~\n",
            "hmmbuild.c:561:3: note: in expansion of macro ‘ESL_ALLOC’\n",
            "  561 |   ESL_ALLOC(info, sizeof(*info) * infocnt);\n",
            "      |   ^~~~~~~~~\n",
            "In file included from hmmbuild.c:6:\n",
            "hmmbuild.c: In function ‘main’:\n",
            "/usr/include/stdlib.h:540:14: note: in a call to allocation function ‘malloc’ declared here\n",
            "  540 | extern void *malloc (size_t __size) __THROW __attribute_malloc__\n",
            "      |              ^~~~~~\n",
            "In file included from hmmscan.c:14:\n",
            "In function ‘serial_master’,\n",
            "    inlined from ‘main’ at hmmscan.c:302:16:\n",
            "../easel/easel.h:164:17: warning: argument 1 range [18446743987810205696, 18446744073709551576] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]\n",
            "  164 |     if ( ((p) = malloc(size)) == NULL)  { \\\n",
            "      |                 ^~~~~~~~~~~~\n",
            "hmmscan.c:402:3: note: in expansion of macro ‘ESL_ALLOC’\n",
            "  402 |   ESL_ALLOC(info, sizeof(*info) * infocnt);\n",
            "      |   ^~~~~~~~~\n",
            "In file included from hmmscan.c:11:\n",
            "hmmscan.c: In function ‘main’:\n",
            "/usr/include/stdlib.h:540:14: note: in a call to allocation function ‘malloc’ declared here\n",
            "  540 | extern void *malloc (size_t __size) __THROW __attribute_malloc__\n",
            "      |              ^~~~~~\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "cd hmmer-3.3.2\n",
        "./configure\n",
        "make\n",
        "make install"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "CHfFSP-zF9vd"
      },
      "source": [
        "Скачиваем PFAM - базу данных по семействам белковых доменов\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YniMHw1xGCsL",
        "outputId": "f66952b9-7a4d-4502-bbc5-3046cd198787"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "--2025-06-18 08:10:52--  https://ftp.ebi.ac.uk/pub/databases/Pfam/releases/Pfam37.0/Pfam-A.hmm.gz\n",
            "Resolving ftp.ebi.ac.uk (ftp.ebi.ac.uk)... 193.62.193.165\n",
            "Connecting to ftp.ebi.ac.uk (ftp.ebi.ac.uk)|193.62.193.165|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 310495632 (296M) [application/x-gzip]\n",
            "Saving to: ‘Pfam-A.hmm.gz’\n",
            "\n",
            "Pfam-A.hmm.gz       100%[===================>] 296.11M  26.3MB/s    in 11s     \n",
            "\n",
            "2025-06-18 08:11:03 (27.7 MB/s) - ‘Pfam-A.hmm.gz’ saved [310495632/310495632]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!wget https://ftp.ebi.ac.uk/pub/databases/Pfam/releases/Pfam37.0/Pfam-A.hmm.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "2BlKbi35GsMf"
      },
      "outputs": [],
      "source": [
        "!gunzip Pfam-A.hmm.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Hqmgtbl6GwMc",
        "outputId": "fdc2bd7a-9297-4197-db09-7f7e95511aa7"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Working...    done.\n",
            "Pressed and indexed 21979 HMMs (21979 names and 21979 accessions).\n",
            "Models pressed into binary file:   Pfam-A.hmm.h3m\n",
            "SSI index for binary model file:   Pfam-A.hmm.h3i\n",
            "Profiles (MSV part) pressed into:  Pfam-A.hmm.h3f\n",
            "Profiles (remainder) pressed into: Pfam-A.hmm.h3p\n"
          ]
        }
      ],
      "source": [
        "!hmmpress Pfam-A.hmm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xuzDPwGdG0Nd",
        "outputId": "0fb71f53-53c5-43ec-c244-0be60ef21fef"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "HMMER3/f [3.3 | Nov 2019]\n",
            "NAME  1-cysPrx_C\n",
            "ACC   PF10417.14\n",
            "DESC  C-terminal domain of 1-Cys peroxiredoxin\n",
            "LENG  41\n",
            "ALPH  amino\n",
            "RF    no\n",
            "MM    no\n",
            "CONS  yes\n",
            "CS    yes\n",
            "MAP   yes\n",
            "DATE  Tue Jan 16 15:37:30 2024\n",
            "NSEQ  30\n",
            "EFFN  14.996338\n",
            "CKSUM 976910065\n",
            "GA    21.1 21.1;\n",
            "TC    21.1 21.1;\n",
            "NC    21 21;\n",
            "BM    hmmbuild HMM.ann SEED.ann\n",
            "SM    hmmsearch -Z 81514348 --cpu 4 -E 1000 HMM pfamseq\n",
            "STATS LOCAL MSV       -7.4395  0.71945\n",
            "STATS LOCAL VITERBI   -8.0185  0.71945\n",
            "STATS LOCAL FORWARD   -4.2047  0.71945\n",
            "HMM          A        C        D        E        F        G        H        I        K        L        M        N        P        Q        R        S        T        V        W        Y   \n",
            "            m->m     m->i     m->d     i->m     i->i     d->m     d->d\n",
            "  COMPO   2.29717  4.29094  2.87032  2.60484  3.85805  2.62286  3.73621  3.28244  2.52338  2.88709  3.88580  3.17233  2.60640  2.85725  3.35628  2.97677  2.80986  2.63009  3.48311  3.83163\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00261  6.34472  7.06707  0.61958  0.77255  0.00000        *\n",
            "      1   0.32482  6.00141  6.59672  6.58185  6.87107  2.23382  7.03985  6.37523  6.46319  6.07536  3.11282  5.75618  5.69608  6.50561  6.40211  2.42119  4.68877  5.43330  8.22447  7.22950      1 A - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00261  6.34472  7.06707  0.61958  0.77255  0.48576  0.95510\n",
            "      2   4.32709  5.66403  6.30484  5.69240  1.83222  5.53036  5.83602  2.03362  5.48587  0.87842  2.58510  5.69861  5.82311  5.54118  5.46484  4.85454  4.55127  2.49513  3.17854  3.56121      2 l - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00261  6.34472  7.06707  0.61958  0.77255  0.48576  0.95510\n",
            "      3   4.83170  7.06042  3.32161  4.40078  6.63215  5.34865  3.38725  5.93782  1.95428  5.26404  6.12468  4.81793  5.71814  0.42668  2.73248  4.77197  4.97039  5.58659  7.22621  6.08077      3 Q - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00261  6.34472  7.06707  0.61958  0.77255  0.48576  0.95510\n",
            "      4   2.45398  5.16871  5.70770  3.18612  1.71617  4.95187  2.54271  3.60119  4.88974  1.69719  3.17550  5.10001  5.31598  5.02910  4.91320  4.26128  2.32712  2.18100  5.78964  2.45488      4 l - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00261  6.34472  7.06707  0.61958  0.77255  0.48576  0.95510\n"
          ]
        }
      ],
      "source": [
        "!head -n 40 Pfam-A.hmm"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "5RECNZVaG1JP"
      },
      "source": [
        "Статистика"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "U8ewXBVhG5G5",
        "outputId": "a90271e5-1fa6-48ea-8239-21dbf8c079d5"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "# hmmstat :: display summary statistics for a profile file\n",
            "# HMMER 3.3.2 (Nov 2020); http://hmmer.org/\n",
            "# Copyright (C) 2020 Howard Hughes Medical Institute.\n",
            "# Freely distributed under the BSD open source license.\n",
            "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
            "#\n",
            "# idx  name                 accession        nseq eff_nseq      M relent   info p relE compKL\n",
            "# ---- -------------------- ------------ -------- -------- ------ ------ ------ ------ ------\n",
            "1      1-cysPrx_C           PF10417.14         30    15.00     41   1.34   1.28   1.26   0.08\n",
            "2      10_blade             PF21734.2          14     1.62    345   0.59   0.55   0.54   0.05\n",
            "3      117-like_vir         PF21578.2           1     0.38    101   0.59   0.54   0.51   0.05\n",
            "4      120_Rick_ant         PF12574.13          2     0.45    240   0.59   0.60   0.52   0.02\n",
            "5      12TM_1               PF09847.14          6     2.00    448   0.59   0.62   0.51   0.08\n",
            "6      14-3-3               PF00244.25        137     2.45    223   0.59   0.59   0.53   0.05\n",
            "7      17kDa_Anti_2         PF16998.10          5     0.96    116   0.59   0.56   0.52   0.03\n",
            "8      2-Hacid_dh           PF00389.35         74     5.70    134   0.59   0.62   0.44   0.03\n",
            "9      2-Hacid_dh_C         PF02826.24        128     3.57    178   0.59   0.61   0.54   0.01\n",
            "10     2-oxoacid_dh         PF00198.28         65     3.20    232   0.59   0.61   0.54   0.02\n",
            "11     2-oxogl_dehyd_N      PF16078.10        182    12.14     41   1.34   1.23   1.33   0.13\n",
            "12     2-ph_phosp           PF04029.19        114     2.94    226   0.59   0.61   0.52   0.02\n"
          ]
        }
      ],
      "source": [
        "!hmmstat Pfam-A.hmm.h3m | head -n 20"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "KTIDBi6uG_dv"
      },
      "source": [
        "Индексирование"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "A9dx9FWBG8nw",
        "outputId": "8e4408fc-22ed-4f5a-da7e-0102f230549b"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Working...    done.\n",
            "Indexed 21979 HMMs (21979 names and 21979 accessions).\n",
            "SSI index written to file Pfam-A.hmm.h3m.ssi\n"
          ]
        }
      ],
      "source": [
        "!hmmfetch --index Pfam-A.hmm.h3m"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "KuLzSWdz03Bl",
        "outputId": "b4defcd1-936d-4d4d-adc2-85fbcc8fe383"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "# hmmfetch :: retrieve profile HMM(s) from a file\n",
            "# Easel 0.48 (Nov 2020)\n",
            "# Copyright (C) 2020 Howard Hughes Medical Institute.\n",
            "# Freely distributed under the BSD open source license.\n",
            "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
            "Usage: hmmfetch [options] <hmmfile> <key>         (retrieves HMM named <key>)\n",
            "Usage: hmmfetch [options] -f <hmmfile> <keyfile>  (retrieves all HMMs in <keyfile>)\n",
            "Usage: hmmfetch [options] --index <hmmfile>       (indexes <hmmfile>)\n",
            "\n",
            "Options:\n",
            "  -h      : help; show brief info on version and usage\n",
            "  -f      : second cmdline arg is a file of names to retrieve\n",
            "  -o <f>  : output HMM to file <f> instead of stdout\n",
            "  -O      : output HMM to file named <key>\n",
            "  --index : index the <hmmfile>, creating <hmmfile>.ssi\n"
          ]
        }
      ],
      "source": [
        "!hmmfetch -h"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "IHNxStfZHDme"
      },
      "source": [
        "Скачиваем протеом"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5Jw0B1JqHFyP",
        "outputId": "832e77db-b2e9-430e-b4b6-9ba7d9c4d130"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "--2025-06-18 09:07:20--  https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/013/758/885/GCF_013758885.1_VT_AalbS3_pri_1.0/GCF_013758885.1_VT_AalbS3_pri_1.0_protein.faa.gz\n",
            "Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.13, 130.14.250.31, 130.14.250.7, ...\n",
            "Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.13|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 5261735 (5.0M) [application/x-gzip]\n",
            "Saving to: ‘GCF_013758885.1_VT_AalbS3_pri_1.0_protein.faa.gz’\n",
            "\n",
            "GCF_013758885.1_VT_ 100%[===================>]   5.02M  5.65MB/s    in 0.9s    \n",
            "\n",
            "2025-06-18 09:07:22 (5.65 MB/s) - ‘GCF_013758885.1_VT_AalbS3_pri_1.0_protein.faa.gz’ saved [5261735/5261735]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!wget https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/013/758/885/GCF_013758885.1_VT_AalbS3_pri_1.0/GCF_013758885.1_VT_AalbS3_pri_1.0_protein.faa.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "C7RuAmZZKuCm"
      },
      "outputs": [],
      "source": [
        "!gunzip GCF_013758885.1_VT_AalbS3_pri_1.0_protein.faa.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MOeot53YHIPB",
        "outputId": "3474d317-60ba-4243-efb8-35665892c7b7"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[1;30;43mВыходные данные были обрезаны до нескольких последних строк (5000).\u001b[0m\n",
            "16980  QSregVF              PF13652.11          6     0.55    112   0.59   0.58   0.52   0.02\n",
            "16981  QSregVF_b            PF12843.12         71     1.30     66   0.85   0.83   0.80   0.03\n",
            "16982  Qua1                 PF16274.10          9     1.61     52   1.06   1.08   1.01   0.06\n",
            "16983  Quaking_NLS          PF16551.10          3     1.63     30   1.79   1.74   1.76   0.22\n",
            "16984  QueC                 PF06508.18         27     1.37    210   0.59   0.57   0.52   0.02\n",
            "16985  QueF                 PF14489.11         18     1.39     81   0.70   0.67   0.66   0.02\n",
            "16986  QueF_N               PF14819.11         55     0.89    111   0.59   0.58   0.54   0.02\n",
            "16987  QueG_DUF1730         PF08331.15        298     3.58     77   0.73   0.72   0.63   0.04\n",
            "16988  QueH                 PF02677.19        223     1.35    175   0.59   0.51   0.53   0.05\n",
            "16989  Questin_oxidase      PF14027.11        194     4.54    340   0.59   0.57   0.44   0.02\n",
            "16990  QueT                 PF06177.16        164     3.25    141   0.59   0.62   0.45   0.10\n",
            "16991  Queuosine_synth      PF02547.20        517     1.74    322   0.59   0.59   0.48   0.02\n",
            "16992  QVR                  PF17064.10        754     2.98     86   0.66   0.48   0.51   0.19\n",
            "16993  QWRF                 PF04484.17         30     2.61    302   0.59   0.61   0.47   0.04\n",
            "16994  Q_salvage            PF10343.14        107     1.60    285   0.59   0.56   0.51   0.03\n",
            "16995  R-HINP1I             PF11463.13          2     0.46    208   0.59   0.57   0.51   0.03\n",
            "16996  R2K_2                PF18299.6          38     2.71    147   0.59   0.57   0.51   0.02\n",
            "16997  R2K_3                PF14243.11         36     4.67    169   0.59   0.58   0.42   0.01\n",
            "16998  R3H                  PF01424.27        568    20.55     60   0.93   0.93   0.89   0.05\n",
            "16999  R3H-assoc            PF13902.11         46     5.41    127   0.59   0.56   0.47   0.08\n",
            "17000  RA                   PF00788.28         39     9.10     93   0.61   0.59   0.54   0.03\n",
            "17001  Rab15_effector       PF15208.11          9     0.65    240   0.59   0.58   0.52   0.01\n",
            "17002  Rab3-GAP_cat_C       PF19533.4           6     0.67    203   0.59   0.61   0.52   0.02\n",
            "17003  Rab3-GTPase_cat      PF13890.11         39     1.51    158   0.59   0.55   0.52   0.04\n",
            "17004  RAB3GAP2_C           PF14656.11         19     1.85    596   0.59   0.58   0.52   0.03\n",
            "17005  RAB3GAP2_N           PF14655.11         34     2.61    419   0.59   0.58   0.50   0.02\n",
            "17006  Rab5-bind            PF09311.16          4     0.79    307   0.59   0.61   0.54   0.08\n",
            "17007  Rabaptin             PF03528.20          3     0.52    486   0.59   0.60   0.52   0.04\n",
            "17008  RabGAP-TBC           PF00566.24         68     6.50    216   0.59   0.54   0.50   0.04\n",
            "17009  RabGGT_insert        PF07711.16          7     0.71    103   0.59   0.53   0.53   0.03\n",
            "17010  Rab_bind             PF16704.10          3     0.78     66   0.85   0.88   0.80   0.08\n",
            "17011  Rab_eff_C            PF04698.17          7     0.68    717   0.59   0.60   0.52   0.03\n",
            "17012  Rac1                 PF09632.15          2     0.62    297   0.59   0.61   0.53   0.03\n",
            "17013  RACo_C_ter           PF14574.11        120     1.80    261   0.59   0.61   0.53   0.02\n",
            "17014  RACo_linker          PF17650.6         170     6.01     86   0.66   0.68   0.57   0.03\n",
            "17015  Raco_middle          PF17651.6          91     1.77    163   0.59   0.60   0.55   0.02\n",
            "17016  RAC_head             PF16717.10         67     2.20     98   0.59   0.62   0.53   0.10\n",
            "17017  Rad1                 PF02144.21         23     2.43    257   0.59   0.59   0.50   0.03\n",
            "17018  Rad10                PF03834.19        103     2.07    115   0.59   0.57   0.52   0.04\n",
            "17019  Rad17                PF03215.20          3     0.83    186   0.59   0.58   0.51   0.02\n",
            "17020  Rad21_Rec8           PF04824.21         14     5.16     55   1.01   1.05   0.99   0.04\n",
            "17021  Rad21_Rec8_N         PF04825.18        100     2.08    110   0.59   0.58   0.50   0.03\n",
            "17022  Rad26-like_C         PF21046.2          27     2.57     65   0.86   0.92   0.82   0.03\n",
            "17023  Rad26-like_helical_rpts PF12331.13         37     2.37    215   0.59   0.60   0.52   0.03\n",
            "17024  Rad26-like_N         PF21048.2          24     4.61     50   1.11   1.11   1.09   0.08\n",
            "17025  Rad33                PF08730.15         12     0.88    165   0.59   0.58   0.52   0.04\n",
            "17026  Rad4                 PF03835.20         25     3.88    148   0.59   0.54   0.49   0.05\n",
            "17027  Rad50_zn_hook        PF04423.19         48    17.91     52   1.07   1.04   1.03   0.13\n",
            "17028  Rad51                PF08423.16         19     0.65    244   0.59   0.60   0.53   0.01\n",
            "17029  RAD51D_N             PF21794.2          22     2.08     60   0.93   0.97   0.90   0.07\n",
            "17030  RAD51_interact       PF15696.10         16     3.72     39   1.40   1.47   1.38   0.15\n",
            "17031  Rad52_Rad22          PF04098.20         33     2.61    153   0.59   0.58   0.52   0.03\n",
            "17032  Rad54_N              PF08658.15         79     1.85    179   0.59   0.59   0.48   0.04\n",
            "17033  Rad60-SLD            PF11976.13         32     5.63     72   0.78   0.80   0.76   0.06\n",
            "17034  Rad60-SLD_2          PF13881.11         20     1.52    111   0.59   0.57   0.51   0.03\n",
            "17035  Rad9                 PF04139.18          8     1.39    253   0.59   0.58   0.51   0.03\n",
            "17036  Rad9_Rad53_bind      PF08605.15          7     1.13    129   0.59   0.56   0.50   0.02\n",
            "17037  RadC                 PF04002.20        667     3.30    121   0.59   0.60   0.53   0.03\n",
            "17038  Radial_spoke         PF04712.17         32     1.82    493   0.59   0.57   0.50   0.02\n",
            "17039  Radial_spoke_3       PF06098.16         40     1.97    229   0.59   0.60   0.56   0.05\n",
            "17040  Radical_SAM          PF04055.26        480    31.81    166   0.59   0.57   0.48   0.02\n",
            "17041  Radical_SAM_2        PF19238.4          84     0.99    151   0.59   0.57   0.54   0.03\n",
            "17042  Radical_SAM_C        PF16199.10        431     5.97     85   0.67   0.65   0.62   0.03\n",
            "17043  Radical_SAM_N        PF08497.15        244     1.11    301   0.59   0.58   0.52   0.01\n",
            "17044  Radical_SAM_N2       PF19864.4         641     1.82    151   0.59   0.59   0.55   0.01\n",
            "17045  RAE1_2_domI_C        PF22603.1          20     1.31    129   0.59   0.58   0.52   0.03\n",
            "17046  Raf1_HTH             PF18579.6          56     3.35     61   0.92   0.90   0.90   0.04\n",
            "17047  Raf1_N               PF18578.6          59     1.32    106   0.59   0.60   0.54   0.04\n",
            "17048  Raffinose_syn        PF05691.17          4     0.64    750   0.59   0.57   0.51   0.01\n",
            "17049  Raftlin              PF15250.11         13     0.88    459   0.59   0.58   0.52   0.01\n",
            "17050  RAG1                 PF12940.12          2     0.41    653   0.59   0.58   0.51   0.01\n",
            "17051  RAG1_imp_bd          PF12560.13         16     0.96    284   0.59   0.53   0.52   0.04\n",
            "17052  RAG2                 PF03089.19         11     0.59    339   0.59   0.57   0.52   0.01\n",
            "17053  RAG2_PHD             PF13341.11          7     0.56     78   0.72   0.60   0.66   0.04\n",
            "17054  Ragweed_pollen       PF03913.18          3     1.09     44   1.25   1.03   1.18   0.25\n",
            "17055  RAI1                 PF08652.16        154     3.69    333   0.59   0.55   0.49   0.02\n",
            "17056  RAI16-like           PF10257.14         19     2.42    358   0.59   0.61   0.51   0.03\n",
            "17057  Ral                  PF11058.13          2     0.64     66   0.85   0.78   0.79   0.06\n",
            "17058  RALF                 PF05498.16         49     3.42     65   0.86   0.76   0.76   0.10\n",
            "17059  RalF_SCD             PF18248.6           3     0.55    147   0.59   0.60   0.52   0.02\n",
            "17060  RALGAPB_N            PF20412.3          39     3.42    125   0.59   0.53   0.51   0.06\n",
            "17061  RAM                  PF15320.11         26     2.78     83   0.68   0.60   0.56   0.11\n",
            "17062  RAMA                 PF18755.6          76     5.14    107   0.59   0.59   0.50   0.02\n",
            "17063  RAMP                 PF04901.18         43     1.01    110   0.59   0.50   0.54   0.05\n",
            "17064  RAMP4                PF06624.17         54     4.22     60   0.93   0.96   0.88   0.05\n",
            "17065  RAMPs                PF03787.20        147    39.08    189   0.59   0.60   0.41   0.02\n",
            "17066  RamS                 PF19402.4          59     5.21     44   1.25   1.24   1.13   0.18\n",
            "17067  Ran-binding          PF05508.16         72     1.45    304   0.59   0.61   0.50   0.03\n",
            "17068  RanGAP1_C            PF07834.16         12     0.84    178   0.59   0.61   0.53   0.03\n",
            "17069  RANK_CRD_2           PF18278.6           4     1.14     41   1.33   1.13   1.29   0.17\n",
            "17070  Ran_BP1              PF00638.23          8     1.05    122   0.59   0.58   0.53   0.04\n",
            "17071  RAP                  PF08373.15         56    11.92     58   0.96   0.87   0.89   0.08\n",
            "17072  RAP-1                PF03085.20          4     0.88    244   0.59   0.56   0.53   0.04\n",
            "17073  Rap-GAP_dimer        PF21022.2          74     1.58    106   0.59   0.57   0.54   0.03\n",
            "17074  RAP1                 PF07218.16          2     0.55    793   0.59   0.58   0.52   0.03\n",
            "17075  Rap1-DNA-bind        PF09197.15         13     1.83    108   0.59   0.54   0.52   0.06\n",
            "17076  Rap1a                PF18602.6         112     7.60     90   0.63   0.58   0.52   0.04\n",
            "17077  Rap1_C               PF11626.13         70     4.81     84   0.68   0.65   0.59   0.03\n",
            "17078  Rap1_C_Sp            PF21411.2           2     1.39     40   1.37   1.46   1.33   0.07\n",
            "17079  RAP80_UIM            PF18282.6           2     0.91     57   0.98   1.02   0.93   0.13\n",
            "17080  RapA_C               PF12137.13         55     2.28    360   0.59   0.60   0.55   0.03\n",
            "17081  RapH_N               PF18801.6          35     3.05     62   0.90   0.86   0.88   0.09\n",
            "17082  Rapsyn_N             PF10579.14          2     0.71     80   0.71   0.68   0.65   0.02\n",
            "17083  Raptor_N             PF14538.11         53     3.20    151   0.59   0.54   0.51   0.02\n",
            "17084  RapZ-like_N          PF03668.20          4     0.77    158   0.59   0.62   0.53   0.02\n",
            "17085  Rap_GAP              PF02145.20        195     3.50    185   0.59   0.56   0.47   0.03\n",
            "17086  Ras                  PF00071.27         60     1.44    162   0.59   0.59   0.53   0.03\n",
            "17087  RasGAP               PF00616.24         50     5.40    207   0.59   0.59   0.51   0.03\n",
            "17088  RasGAP_C             PF03836.20        106     4.43    137   0.59   0.59   0.53   0.07\n",
            "17089  RasGEF               PF00617.24        580     5.97    181   0.59   0.56   0.46   0.04\n",
            "17090  RasGEF_N             PF00618.25         51     5.84    106   0.59   0.57   0.48   0.04\n",
            "17091  RasGEF_N_2           PF14663.11         68     3.47    108   0.59   0.57   0.53   0.04\n",
            "17092  RASSF8-10_RA         PF21712.2          29     1.72     79   0.72   0.69   0.67   0.04\n",
            "17093  Ras_bdg_2            PF14847.11         41     2.03    100   0.59   0.58   0.51   0.04\n",
            "17094  Rav1p_C              PF12234.13         57     1.53    640   0.59   0.58   0.52   0.03\n",
            "17095  RavJ_Peptidase       PF22469.1           1     0.39    202   0.60   0.58   0.52   0.01\n",
            "17096  RavZ_C               PF22225.1           1     0.40    101   0.60   0.62   0.52   0.03\n",
            "17097  RAWUL                PF16207.10         49     3.33     65   0.86   0.81   0.76   0.06\n",
            "17098  Rax2                 PF12768.12         67     2.56    254   0.59   0.57   0.51   0.02\n",
            "17099  Rax2_2               PF20842.3          61     1.41    146   0.59   0.54   0.51   0.02\n",
            "17100  Rax2_3               PF20843.3          64     1.36    161   0.59   0.54   0.51   0.03\n",
            "17101  RA_2                 PF21989.1         120     5.43     94   0.61   0.58   0.54   0.03\n",
            "17102  RA_3                 PF23010.1          37     1.62    118   0.59   0.56   0.53   0.01\n",
            "17103  RBB1NT               PF08169.16         21     1.15     94   0.61   0.60   0.54   0.03\n",
            "17104  RbcS                 PF12338.13         10     1.67     45   1.22   1.28   1.15   0.12\n",
            "17105  RbcX                 PF02341.20         26     0.98    111   0.59   0.59   0.53   0.04\n",
            "17106  RBD                  PF02196.20         45     5.69     69   0.81   0.83   0.78   0.04\n",
            "17107  RBD-FIP              PF09457.15          8     4.61     41   1.34   1.39   1.29   0.13\n",
            "17108  RBDV_coat            PF06593.16          1     0.39    274   0.58   0.59   0.51   0.02\n",
            "17109  RBFA                 PF02033.23        625     4.32    105   0.59   0.61   0.50   0.04\n",
            "17110  RBM1CTR              PF08081.16          9     1.94     45   1.22   1.17   1.17   0.23\n",
            "17111  RBM39linker          PF15519.11         83     3.73     81   0.70   0.70   0.61   0.04\n",
            "17112  RbmA-like_FnIII      PF22021.1           4     0.87    111   0.59   0.59   0.52   0.02\n",
            "17113  RbpA                 PF13397.11         47     1.43    105   0.59   0.58   0.53   0.03\n",
            "17114  RBP_receptor         PF14752.11         28     1.32    605   0.59   0.57   0.53   0.03\n",
            "17115  RBR                  PF17208.8          37     7.97     57   0.98   1.01   0.88   0.21\n",
            "17116  RbsD_FucU            PF05025.18        213     3.05    138   0.59   0.61   0.50   0.02\n",
            "17117  Rbsn                 PF11464.13         85    12.25     42   1.31   1.36   1.30   0.15\n",
            "17118  Rbx_binding          PF18113.6          38     4.66     71   0.79   0.79   0.76   0.02\n",
            "17119  RB_A                 PF01858.22         53     2.70    196   0.59   0.57   0.53   0.04\n",
            "17120  RB_B                 PF01857.25         70     1.80    144   0.59   0.55   0.51   0.05\n",
            "17121  Rb_C                 PF08934.15          5     0.56    152   0.59   0.59   0.52   0.02\n",
            "17122  RC-P840_PscD         PF10657.14          3     0.45    144   0.59   0.58   0.52   0.01\n",
            "17123  RCC1                 PF00415.23        101    16.84     50   1.11   1.07   1.02   0.06\n",
            "17124  RCC1_2               PF13540.11        167   167.00     30   1.56   1.47   1.54   0.19\n",
            "17125  RCC_reductase        PF06405.16         11     0.85    253   0.59   0.60   0.53   0.02\n",
            "17126  Rcd1                 PF04078.18         50     2.19    260   0.59   0.60   0.54   0.04\n",
            "17127  RCDG1                PF15725.10          3     0.65     83   0.68   0.74   0.60   0.05\n",
            "17128  Rce1-like            PF02517.21        608     8.62     93   0.61   0.61   0.52   0.10\n",
            "17129  RCF1-5-like_lid      PF21960.1          50     8.02     43   1.28   1.36   1.25   0.11\n",
            "17130  RcnB                 PF11776.13         56     8.64     51   1.09   1.01   1.06   0.10\n",
            "17131  RcpB                 PF16971.10          2     0.65    169   0.59   0.59   0.52   0.03\n",
            "17132  RcpC                 PF16976.10        106     3.92    114   0.59   0.66   0.52   0.04\n",
            "17133  RCR                  PF12273.13         48     1.94    117   0.59   0.53   0.41   0.04\n",
            "17134  RCS1                 PF07326.16         12     1.14    232   0.59   0.59   0.53   0.04\n",
            "17135  RcsC                 PF09456.15         11     1.60     91   0.63   0.62   0.58   0.03\n",
            "17136  RCSD                 PF05177.17          7     1.23    101   0.59   0.61   0.49   0.18\n",
            "17137  RcsD_ABL             PF16359.10         10     1.08    103   0.59   0.62   0.54   0.05\n",
            "17138  RcsF                 PF16358.10          7     0.79    110   0.59   0.58   0.52   0.03\n",
            "17139  RctB_central         PF11826.13          1     0.39    329   0.59   0.58   0.52   0.01\n",
            "17140  RD3                  PF14473.11         27     2.15    129   0.59   0.57   0.53   0.05\n",
            "17141  RDD                  PF06271.17         39     6.88    137   0.59   0.60   0.50   0.06\n",
            "17142  RDD1                 PF15828.10          8     1.08     77   0.73   0.74   0.67   0.05\n",
            "17143  RdDM_RDM1            PF09187.15         11     0.63    119   0.59   0.54   0.53   0.03\n",
            "17144  RdfA                 PF21811.2          94     1.87    195   0.59   0.58   0.54   0.05\n",
            "17145  RdgC                 PF04381.17         83     2.10    298   0.59   0.60   0.55   0.03\n",
            "17146  RDM                  PF11002.13         14     2.59     42   1.30   1.33   1.28   0.12\n",
            "17147  RdRP                 PF05183.17        239     4.04    582   0.59   0.57   0.43   0.01\n",
            "17148  RdRP_1               PF00680.25         19     5.56    450   0.59   0.56   0.50   0.01\n",
            "17149  RdRP_2               PF00978.26         36     3.04    440   0.59   0.57   0.53   0.02\n",
            "17150  RdRP_3               PF00998.28         32     2.79    486   0.59   0.56   0.53   0.02\n",
            "17151  RdRP_4               PF02123.21         28     6.28    465   0.59   0.56   0.47   0.02\n",
            "17152  RdRP_5               PF07925.16          3     0.56   1271   0.59   0.58   0.52   0.01\n",
            "17153  RDV-p3               PF09231.15          2     0.53    970   0.59   0.59   0.52   0.02\n",
            "17154  Rdx                  PF10262.14        209     4.93     74   0.76   0.71   0.67   0.03\n",
            "17155  Read-through         PF16814.10          1     0.36    182   0.59   0.57   0.52   0.01\n",
            "17156  REase-ARP            PF22558.1          20     5.47    270   0.59   0.56   0.43   0.03\n",
            "17157  Reb1_MybAD           PF21559.2           3     1.19     68   0.83   0.75   0.78   0.05\n",
            "17158  RebB                 PF11747.13         28     2.63     68   0.83   0.86   0.80   0.12\n",
            "17159  REC1                 PF18501.6           9     1.53    250   0.59   0.58   0.51   0.05\n",
            "17160  REC102               PF21736.2          37     4.15    197   0.59   0.56   0.50   0.04\n",
            "17161  REC104               PF17378.7           1     0.41    182   0.59   0.59   0.52   0.02\n",
            "17162  REC114-like          PF15165.11          6     0.72    236   0.59   0.58   0.51   0.01\n",
            "17163  RecA                 PF00154.26         14     0.58    262   0.59   0.62   0.52   0.02\n",
            "17164  RecA_C               PF21096.2         121     3.34     57   0.98   0.99   0.96   0.05\n",
            "17165  RecA_dep_nuc         PF16786.10          3     0.90    103   0.59   0.55   0.50   0.02\n",
            "17166  RecC_C               PF17946.6         107     6.49    202   0.59   0.61   0.47   0.03\n",
            "17167  RecD_N               PF21185.2         391     4.02    103   0.59   0.61   0.50   0.03\n",
            "17168  Receptor_2B4         PF11465.13          3     0.62    108   0.59   0.57   0.52   0.02\n",
            "17169  Receptor_IA-2        PF11548.13         19     1.71     89   0.64   0.67   0.60   0.04\n",
            "17170  Recep_L_domain       PF01030.29         48     3.57    112   0.59   0.58   0.49   0.04\n",
            "17171  RecG_dom3_C          PF19833.4           5     0.58     87   0.65   0.67   0.59   0.02\n",
            "17172  RecG_N               PF17190.9           7     1.45     89   0.64   0.66   0.59   0.07\n",
            "17173  RecG_wedge           PF17191.9           6     0.73    162   0.59   0.59   0.53   0.03\n",
            "17174  RecJ_C               PF22047.1           3     0.70    110   0.59   0.59   0.51   0.02\n",
            "17175  RecJ_C_Deinoc        PF20748.2          14     0.92    155   0.59   0.59   0.51   0.02\n",
            "17176  RecJ_OB              PF17768.6         472     5.37    107   0.59   0.58   0.53   0.01\n",
            "17177  RECK-like_dom        PF22955.1           1     0.89     37   1.47   1.31   1.42   0.20\n",
            "17178  RECK-like_N          PF22961.1          44     4.17     50   1.11   0.93   1.08   0.22\n",
            "17179  Recombinase          PF07508.18         65     7.88    102   0.59   0.56   0.50   0.04\n",
            "17180  RecO_C               PF02565.20         78    11.86    157   0.59   0.60   0.54   0.02\n",
            "17181  RecO_N               PF11967.13         81     6.27     80   0.71   0.73   0.65   0.03\n",
            "17182  RecO_N_2             PF13114.11         10     0.91     71   0.79   0.76   0.74   0.04\n",
            "17183  RecQ-1-like_HTH      PF21220.2          38     1.76     51   1.08   1.12   1.05   0.09\n",
            "17184  RecQ5                PF06959.16         10     0.84    203   0.59   0.60   0.52   0.05\n",
            "17185  RecQ_Zn_bind         PF16124.10        615     5.33     66   0.85   0.72   0.71   0.13\n",
            "17186  RecR_C               PF21175.2         596   596.00     24   2.13   2.24   2.13   0.12\n",
            "17187  RecR_HhH             PF21176.2         638     7.00     45   1.22   1.33   1.21   0.08\n",
            "17188  RecR_ZnF             PF02132.20        650   650.00     21   2.24   1.88   2.24   0.60\n",
            "17189  RecT                 PF03837.19         72     8.28    198   0.59   0.56   0.47   0.03\n",
            "17190  RecU                 PF03838.19        120     1.62    162   0.59   0.56   0.54   0.04\n",
            "17191  RecX_HTH1            PF21982.1         983    58.94     40   1.37   1.45   1.37   0.13\n",
            "17192  RecX_HTH2            PF02631.21       2387    50.27     42   1.31   1.36   1.30   0.10\n",
            "17193  RecX_HTH3            PF21981.1        2464   135.08     48   1.15   1.16   1.12   0.12\n",
            "17194  Red1                 PF07964.16         10     3.32    754   0.59   0.58   0.52   0.06\n",
            "17195  RedAm-like_C         PF21761.2         106     2.83    127   0.59   0.61   0.56   0.03\n",
            "17196  Redic1-like          PF15089.11          7     0.83    425   0.59   0.56   0.52   0.05\n",
            "17197  Redoxin              PF08534.15         51     7.21    148   0.59   0.59   0.46   0.01\n",
            "17198  Reductase_C          PF14759.11         55     2.72     85   0.67   0.65   0.63   0.03\n",
            "17199  RED_C                PF07807.16         11     0.71    112   0.59   0.56   0.52   0.06\n",
            "17200  RED_N                PF07808.18         23     1.24    234   0.59   0.59   0.51   0.06\n",
            "17201  Reeler               PF02014.21        107     2.90    131   0.59   0.52   0.44   0.03\n",
            "17202  Reelin_subrepeat-B   PF21471.2          67     2.56    127   0.59   0.54   0.50   0.02\n",
            "17203  REF                  PF05755.17         28     1.15    206   0.59   0.60   0.53   0.03\n",
            "17204  REGB_T4              PF10715.14          1     0.38    150   0.59   0.58   0.51   0.01\n",
            "17205  Regnase_1_C          PF18561.6           8     1.61     46   1.20   1.17   1.16   0.06\n",
            "17206  Regulator_TrmB       PF11495.13         21     4.11    233   0.59   0.61   0.53   0.02\n",
            "17207  Reg_prop             PF07494.16        659   659.00     24   1.04   0.98   1.00   0.09\n",
            "17208  REJ                  PF02010.20         23     4.81    452   0.59   0.57   0.49   0.02\n",
            "17209  Relaxase             PF03432.19        853     6.91    132   0.59   0.53   0.52   0.05\n",
            "17210  Relaxase_C           PF11083.13         15     2.85    125   0.59   0.63   0.55   0.09\n",
            "17211  Relaxase_M           PF20874.2          14     1.36     86   0.66   0.63   0.60   0.04\n",
            "17212  RelA_AH_RIS          PF19296.4          73     1.29    185   0.59   0.61   0.52   0.02\n",
            "17213  RelA_SpoT            PF04607.22         87     1.89    113   0.59   0.56   0.50   0.05\n",
            "17214  RelB                 PF04221.17          4     1.51     82   0.69   0.73   0.63   0.03\n",
            "17215  RelB-like            PF18506.6          21     5.19     47   1.17   1.24   1.16   0.12\n",
            "17216  RelB_leu_zip         PF16180.10          5     0.93     84   0.68   0.74   0.59   0.03\n",
            "17217  RelB_transactiv      PF16181.10          2     0.51    181   0.59   0.62   0.52   0.02\n",
            "17218  RelE                 PF06296.17         16     1.22    120   0.59   0.61   0.54   0.04\n",
            "17219  RELT                 PF12606.13         13     1.93     42   1.30   1.28   1.27   0.12\n",
            "17220  RemA-like            PF04025.17        110     3.19     73   0.77   0.82   0.73   0.06\n",
            "17221  Remorin_C            PF03763.18         85     3.09    107   0.59   0.57   0.56   0.15\n",
            "17222  Remorin_N            PF03766.18         21     3.23     54   1.03   1.12   0.93   0.13\n",
            "17223  Renin_r              PF07850.19         25     1.19     98   0.59   0.57   0.51   0.04\n",
            "17224  Reoviridae_Vp9       PF08978.16          2     0.58    280   0.59   0.60   0.52   0.02\n",
            "17225  Reovirus_bridge      PF20830.2           4     1.27    102   0.59   0.57   0.50   0.02\n",
            "17226  Reovirus_cap         PF00979.22          5     0.94    367   0.59   0.55   0.52   0.01\n",
            "17227  Reovirus_L2_4th      PF21062.2           3     0.75    110   0.59   0.61   0.53   0.02\n",
            "17228  Reovirus_L2_6th      PF21060.2           3     0.71     98   0.59   0.59   0.52   0.02\n",
            "17229  Reovirus_L2_7th      PF21061.2           3     0.61     99   0.59   0.58   0.52   0.02\n",
            "17230  Reovirus_L2_8th      PF06016.16          3     1.22     66   0.85   0.86   0.80   0.05\n",
            "17231  Reovirus_L2_GTase    PF21063.2           3     0.74    247   0.59   0.58   0.53   0.01\n",
            "17232  Reovirus_L2_MT1      PF21065.2           3     0.64    293   0.59   0.60   0.52   0.01\n",
            "17233  Reovirus_L2_MT2      PF21066.2           3     0.68    231   0.59   0.59   0.53   0.02\n",
            "17234  Reovirus_L2_N        PF21064.2           3     0.68    142   0.59   0.57   0.51   0.02\n",
            "17235  Reovirus_M2          PF05993.17          3     0.71    647   0.59   0.61   0.52   0.02\n",
            "17236  Reovirus_Mu2         PF07781.16          3     0.72    727   0.59   0.59   0.51   0.01\n",
            "17237  Reov_VP3_GTase       PF20790.2          11     2.08    264   0.59   0.54   0.53   0.04\n",
            "17238  Reov_VP3_MTase1      PF20831.2          11     1.97    226   0.59   0.59   0.50   0.03\n",
            "17239  Reov_VP3_MTase2      PF20832.2          11     2.80    203   0.59   0.57   0.51   0.03\n",
            "17240  Reo_P9               PF06043.16          1     0.39    334   0.59   0.59   0.52   0.02\n",
            "17241  Reo_sigma1           PF01664.21          1     0.39    219   0.59   0.58   0.52   0.01\n",
            "17242  Reo_sigmaC           PF04582.17          4     0.50    130   0.59   0.58   0.52   0.02\n",
            "17243  Reo_sigmaC_M         PF17750.6           4     1.57     40   1.37   1.38   1.33   0.11\n",
            "17244  Rep-A_N              PF04057.17         11     1.23    101   0.59   0.62   0.50   0.05\n",
            "17245  Rep3_C               PF21205.2         143    15.31     59   0.95   0.94   0.84   0.07\n",
            "17246  Rep3_N               PF01051.26        154     7.50    141   0.59   0.58   0.43   0.05\n",
            "17247  RepA1_leader         PF08048.17          5     3.20     25   2.13   2.04   2.07   0.26\n",
            "17248  RepA_C               PF04796.17          2     0.65    181   0.59   0.58   0.52   0.01\n",
            "17249  RepA_N               PF06970.16         11     1.59     76   0.74   0.72   0.71   0.04\n",
            "17250  REPA_OB_2            PF16900.10         95     1.46     98   0.59   0.60   0.53   0.04\n",
            "17251  RepB                 PF07506.16         15     4.64    185   0.59   0.62   0.51   0.03\n",
            "17252  RepB-RCR_reg         PF10723.14          3     0.75     81   0.70   0.70   0.63   0.05\n",
            "17253  RepB_C               PF21861.1          11     3.70     58   0.96   0.91   0.90   0.09\n",
            "17254  RepB_primase         PF16793.10          8     2.02    186   0.59   0.58   0.51   0.02\n",
            "17255  RepB_primase_C       PF22448.1          39     9.87     54   1.03   1.02   1.00   0.06\n",
            "17256  RepC                 PF06504.16          3     0.50    272   0.59   0.60   0.52   0.01\n",
            "17257  RepD-like_N          PF22477.1          10     1.07     87   0.65   0.62   0.59   0.04\n",
            "17258  RepL                 PF05732.16          3     0.75    164   0.59   0.60   0.52   0.03\n",
            "17259  Replicase            PF03090.22          6     1.22    130   0.59   0.56   0.51   0.02\n",
            "17260  Replic_Relax         PF13814.11         24     4.16    191   0.59   0.58   0.44   0.03\n",
            "17261  Replitron_C          PF21860.1           4     1.00     77   0.74   0.64   0.69   0.05\n",
            "17262  Replitron_HUH        PF21859.1           5     1.08    128   0.59   0.56   0.50   0.03\n",
            "17263  Repressor_Mnt        PF11423.13          2     2.00     28   1.84   1.85   1.81   0.16\n",
            "17264  Reprolysin           PF01421.24         16     1.27    200   0.59   0.54   0.53   0.03\n",
            "17265  Reprolysin_2         PF13574.11          6     1.60    195   0.59   0.57   0.45   0.02\n",
            "17266  Reprolysin_3         PF13582.11         19     3.36    123   0.59   0.56   0.46   0.04\n",
            "17267  Reprolysin_4         PF13583.11          4     1.50    203   0.59   0.58   0.49   0.03\n",
            "17268  Reprolysin_5         PF13688.11         46     5.14    196   0.59   0.54   0.43   0.04\n",
            "17269  RepSA                PF20199.3          88     2.62    438   0.59   0.55   0.48   0.03\n",
            "17270  Rep_1                PF01446.22         12     1.51    253   0.59   0.54   0.50   0.04\n",
            "17271  Rep_1B               PF17873.6           1     0.70     53   1.05   1.09   0.99   0.06\n",
            "17272  Rep_4                PF05797.16          1     0.38    335   0.59   0.57   0.51   0.01\n",
            "17273  Rep_fac-A_3          PF08661.16         17     3.09    105   0.59   0.59   0.51   0.03\n",
            "17274  Rep_fac-A_C          PF08646.15         28     1.43    146   0.59   0.52   0.53   0.05\n",
            "17275  Rep_fac_C            PF08542.16        300     8.38     88   0.65   0.67   0.59   0.03\n",
            "17276  Rep_N                PF08724.15          9     1.25    186   0.59   0.55   0.51   0.02\n",
            "17277  Rep_OBD              PF01719.22          6     1.10    121   0.59   0.54   0.51   0.04\n",
            "17278  Rep_Org_C            PF06926.16          3     0.64     95   0.60   0.58   0.54   0.05\n",
            "17279  Rep_trans            PF02486.24         25     3.19    201   0.59   0.56   0.51   0.03\n",
            "17280  Rer1                 PF03248.18        110     1.06    167   0.59   0.52   0.52   0.05\n",
            "17281  RES                  PF08808.16        175    13.40    157   0.59   0.57   0.42   0.02\n",
            "17282  ResB                 PF05140.19        162     4.25    486   0.59   0.59   0.42   0.01\n",
            "17283  ResIII               PF04851.20         52     4.13    164   0.59   0.58   0.47   0.03\n",
            "17284  Resistin             PF06954.16         28     0.86     88   0.65   0.52   0.58   0.12\n",
            "17285  Resolvase            PF00239.26        102     6.30    145   0.59   0.62   0.49   0.04\n",
            "17286  RESP18               PF14948.11         10     0.92    103   0.59   0.57   0.52   0.03\n",
            "17287  Response_reg         PF00072.29         52     3.82    111   0.59   0.62   0.54   0.03\n",
            "17288  Response_reg_2       PF19192.5          37     5.48    180   0.59   0.60   0.47   0.04\n",
            "17289  ResT-TelK_cat        PF16684.10         17     2.22    181   0.59   0.58   0.49   0.02\n",
            "17290  RestrictionMunI      PF11407.13          1     0.37    193   0.59   0.56   0.52   0.01\n",
            "17291  RestrictionSfiI      PF11487.13          2     0.71    292   0.59   0.58   0.51   0.01\n",
            "17292  REST_helical         PF20878.2          43     2.88     65   0.86   0.87   0.82   0.06\n",
            "17293  Ret2_MD              PF18528.6           2     0.42     93   0.61   0.59   0.54   0.02\n",
            "17294  RETICULATA-like      PF11891.13         38     1.58    177   0.59   0.59   0.53   0.02\n",
            "17295  Reticulon            PF02453.22        195     4.84    157   0.59   0.59   0.55   0.07\n",
            "17296  Retinal              PF15449.11         15     1.11   1292   0.59   0.59   0.53   0.04\n",
            "17297  Retinin_C            PF04527.17         14     1.46     63   0.89   0.91   0.80   0.14\n",
            "17298  Retrotrans_gag       PF03732.22         55     5.72     97   0.59   0.55   0.52   0.04\n",
            "17299  Retrotran_gag_2      PF14223.11         33     4.02    138   0.59   0.59   0.56   0.04\n",
            "17300  Retrotran_gag_3      PF14244.11         14     5.59     48   1.15   1.13   1.12   0.03\n",
            "17301  Retrotran_gag_4      PF17241.7          20     3.92    154   0.59   0.55   0.53   0.04\n",
            "17302  Retro_M              PF02813.19          1     0.45     86   0.67   0.70   0.60   0.02\n",
            "17303  RET_CLD1             PF17756.6          17     1.02    124   0.59   0.56   0.52   0.02\n",
            "17304  RET_CLD3             PF17812.6          13     0.83    116   0.59   0.58   0.53   0.03\n",
            "17305  RET_CLD4             PF17813.6          21     0.80    105   0.59   0.56   0.52   0.04\n",
            "17306  RET_CRD              PF22540.1          35     0.91    103   0.59   0.46   0.51   0.13\n",
            "17307  REV                  PF00424.23          2     0.65     94   0.61   0.62   0.53   0.04\n",
            "17308  REV1_C               PF16727.10        176     4.76     91   0.63   0.58   0.56   0.05\n",
            "17309  RexA                 PF15969.10          4     1.07    235   0.59   0.60   0.53   0.03\n",
            "17310  RexB                 PF15968.10          1     0.38    139   0.59   0.61   0.52   0.06\n",
            "17311  RE_AccI              PF09545.15          3     0.48    366   0.59   0.59   0.52   0.02\n",
            "17312  RE_Alw26IDE          PF09665.15          3     0.70    507   0.59   0.56   0.52   0.02\n",
            "17313  RE_AlwI              PF09491.15          5     1.55    441   0.59   0.58   0.50   0.03\n",
            "17314  RE_ApaLI             PF09499.15          3     0.50    189   0.59   0.55   0.52   0.03\n",
            "17315  RE_AspBHI_N          PF18062.6          10     1.25    185   0.59   0.55   0.51   0.02\n",
            "17316  RE_Bpu10I            PF09549.15          5     0.71    218   0.59   0.58   0.52   0.02\n",
            "17317  RE_BsaWI             PF18643.6          14     2.68    105   0.59   0.57   0.49   0.04\n",
            "17318  RE_Bsp6I             PF09504.15          1     0.36    179   0.60   0.56   0.52   0.02\n",
            "17319  RE_BstXI             PF09552.15          2     0.49    290   0.59   0.58   0.52   0.01\n",
            "17320  RE_CfrBI             PF09516.15          2     0.65    257   0.59   0.60   0.51   0.03\n",
            "17321  RE_Eco29kI           PF09517.15         12     0.89    164   0.59   0.56   0.52   0.01\n",
            "17322  RE_Eco47II           PF09553.15         14     1.04    202   0.59   0.56   0.53   0.04\n",
            "17323  RE_EcoO109I          PF14511.11         12     1.04    198   0.59   0.58   0.52   0.02\n",
            "17324  RE_endonuc           PF19778.4          23     1.31    108   0.59   0.56   0.50   0.03\n",
            "17325  RE_HaeII             PF09554.15          3     0.57    338   0.59   0.59   0.52   0.02\n",
            "17326  RE_HaeIII            PF09556.15          5     0.62    298   0.59   0.58   0.53   0.01\n",
            "17327  RE_HindIII           PF09518.15          4     0.96    285   0.59   0.59   0.52   0.03\n",
            "17328  RE_HindVP            PF09519.15          9     0.87    332   0.59   0.57   0.53   0.02\n",
            "17329  RE_HpaII             PF09561.15         16     1.52    359   0.59   0.58   0.53   0.03\n",
            "17330  RE_LlaJI             PF09563.15          3     0.76    365   0.59   0.56   0.51   0.03\n",
            "17331  RE_LlaMI             PF09562.15          2     0.54    266   0.59   0.53   0.50   0.03\n",
            "17332  RE_MamI              PF09567.15          3     0.81    183   0.59   0.56   0.50   0.02\n",
            "17333  RE_MjaI              PF09568.15         10     1.10    164   0.59   0.58   0.52   0.04\n",
            "17334  RE_NgoBV             PF09564.15          2     0.41    238   0.59   0.56   0.52   0.02\n",
            "17335  RE_NgoFVII           PF09565.15         11     1.73    162   0.59   0.57   0.51   0.03\n",
            "17336  RE_NgoFVII_C         PF20731.2          10     1.35    131   0.59   0.58   0.49   0.02\n",
            "17337  RE_NgoPII            PF09521.15          8     0.72    262   0.59   0.58   0.52   0.02\n",
            "17338  RE_R_Pab1            PF09522.15          2     0.57    119   0.59   0.59   0.52   0.04\n",
            "17339  RE_SacI              PF09566.15          5     1.13    276   0.59   0.60   0.49   0.02\n",
            "17340  RE_ScaI              PF09569.15          3     0.61    192   0.59   0.57   0.52   0.02\n",
            "17341  RE_SinI              PF09570.15          2     0.54    218   0.59   0.56   0.52   0.01\n",
            "17342  RE_TaqI              PF09573.15          3     0.56    229   0.59   0.55   0.51   0.03\n",
            "17343  RE_TdeIII            PF09520.15         14     1.87    242   0.59   0.57   0.51   0.03\n",
            "17344  RE_XamI              PF09572.15          5     0.82    256   0.59   0.59   0.51   0.01\n",
            "17345  RE_XcyI              PF09571.15          3     0.69    305   0.59   0.62   0.52   0.02\n",
            "17346  RF-1                 PF00472.25        718     2.82    116   0.59   0.59   0.48   0.07\n",
            "17347  RF3_C                PF16658.10        229     1.66    132   0.59   0.59   0.53   0.02\n",
            "17348  RFamide_26RFa        PF11109.13         14     1.08    131   0.59   0.60   0.53   0.02\n",
            "17349  RFC1                 PF08519.17        128     3.06    158   0.59   0.57   0.51   0.03\n",
            "17350  RFC_C                PF22534.1          16     0.77     93   0.62   0.61   0.55   0.04\n",
            "17351  Rft-1                PF04506.18         31     2.02    512   0.59   0.59   0.52   0.05\n",
            "17352  RFX1_trans_act       PF04589.18         14     1.04    160   0.59   0.59   0.49   0.06\n",
            "17353  RFX5_DNA_bdg         PF14621.11          9     0.82    219   0.59   0.62   0.53   0.06\n",
            "17354  RFX5_N               PF18326.6           3     0.99     59   0.95   0.98   0.90   0.05\n",
            "17355  RFXA_RFXANK_bdg      PF15289.11          3     0.66    122   0.59   0.55   0.51   0.05\n",
            "17356  RFX_DNA_binding      PF02257.20         58     2.12     79   0.72   0.66   0.67   0.04\n",
            "17357  RGCC                 PF15151.11          6     0.57    128   0.59   0.60   0.52   0.04\n",
            "17358  Rgg_C                PF21259.2          32     4.15    182   0.59   0.56   0.54   0.06\n",
            "17359  RGI1                 PF10843.13         10     0.55    194   0.59   0.55   0.52   0.03\n",
            "17360  RGI_lyase            PF18370.6          65     1.55     86   0.66   0.64   0.61   0.03\n",
            "17361  RGL11_C              PF21348.2          34     0.68    490   0.59   0.57   0.52   0.02\n",
            "17362  RGM_C                PF06534.18         28     1.16    171   0.59   0.56   0.52   0.03\n",
            "17363  RGM_N                PF06535.17         22     1.16    171   0.59   0.51   0.50   0.05\n",
            "17364  RGP                  PF03214.18         11     0.63    342   0.59   0.57   0.52   0.01\n",
            "17365  Rgp1                 PF08737.15         66     3.10    415   0.59   0.59   0.46   0.02\n",
            "17366  RgpF                 PF05045.17         12     1.23    503   0.59   0.56   0.52   0.02\n",
            "17367  RGS                  PF00615.24         51     3.67    117   0.59   0.55   0.52   0.05\n",
            "17368  RGS-like             PF09128.16          5     1.01    191   0.59   0.59   0.52   0.03\n",
            "17369  RGS12_us1            PF16613.10          6     0.64    118   0.59   0.60   0.52   0.06\n",
            "17370  RGS12_us2            PF16611.10          6     1.37     72   0.78   0.82   0.72   0.13\n",
            "17371  RGS12_usC            PF16612.10          9     1.05    146   0.59   0.60   0.52   0.06\n",
            "17372  RGS_DHEX             PF18148.6           4     0.63    100   0.59   0.53   0.52   0.04\n",
            "17373  RH1                  PF09744.14        140     6.09     65   0.86   0.93   0.83   0.07\n",
            "17374  RH3_dom              PF22180.1           1     0.61     66   0.85   0.87   0.78   0.06\n",
            "17375  Rh5                  PF18515.6           8     2.10    255   0.59   0.55   0.56   0.08\n",
            "17376  RHA1_ro05818_N       PF20859.2          19     2.02     58   0.96   1.06   0.92   0.11\n",
            "17377  RhaA                 PF06134.16          5     0.51    417   0.59   0.58   0.53   0.01\n",
            "17378  Rhabdo_glycop        PF00974.23          4     1.00    502   0.59   0.55   0.51   0.01\n",
            "17379  Rhabdo_M1            PF03342.18          1     0.41    227   0.60   0.61   0.52   0.02\n",
            "17380  Rhabdo_M2            PF04785.17          2     0.39    202   0.59   0.57   0.52   0.01\n",
            "17381  Rhabdo_matrix        PF03397.19          1     0.40    168   0.59   0.59   0.52   0.01\n",
            "17382  Rhabdo_ncap          PF00945.23          3     0.94    410   0.59   0.57   0.53   0.01\n",
            "17383  Rhabdo_ncap_2        PF03216.18         12     4.12    293   0.59   0.58   0.51   0.02\n",
            "17384  Rhabdo_NV            PF02484.20          1     0.37    111   0.59   0.57   0.51   0.02\n",
            "17385  RhaB_D2              PF21557.2           6     1.04    172   0.59   0.59   0.52   0.01\n",
            "17386  rhaM                 PF05336.18        218     2.74    105   0.59   0.51   0.49   0.05\n",
            "17387  Rhamnogal_lyase      PF06045.16          4     0.61    211   0.59   0.57   0.51   0.02\n",
            "17388  Rhamno_transf        PF11316.13         10     1.72    235   0.59   0.57   0.54   0.02\n",
            "17389  RhaT                 PF06379.17          2     0.38    344   0.59   0.58   0.51   0.04\n",
            "17390  RHD3_GTPase          PF05879.17          8     0.66    245   0.59   0.59   0.52   0.02\n",
            "17391  RHD_dimer            PF16179.10         41     1.69    103   0.59   0.55   0.53   0.02\n",
            "17392  RHD_DNA_bind         PF00554.27         21     1.57    169   0.59   0.57   0.51   0.03\n",
            "17393  RHG20_PH             PF22286.1          55     1.55     93   0.61   0.58   0.56   0.05\n",
            "17394  RhgB_N               PF09284.15         51     1.35    251   0.59   0.57   0.52   0.02\n",
            "17395  RHH_1                PF01402.26         30    30.00     39   1.15   1.26   1.14   0.14\n",
            "17396  RHH_3                PF12651.12         15     9.20     44   1.25   1.32   1.24   0.11\n",
            "17397  RHH_4                PF13467.11         96     3.11     67   0.84   0.86   0.76   0.05\n",
            "17398  RHH_5                PF07878.16         13     7.65     43   1.28   1.35   1.27   0.06\n",
            "17399  RHH_6                PF16762.10          5     0.92     77   0.73   0.77   0.67   0.04\n",
            "17400  RHH_7                PF16777.10          2     0.90     74   0.76   0.78   0.69   0.05\n",
            "17401  RHH_8                PF17723.6           6     0.76    119   0.59   0.64   0.53   0.04\n",
            "17402  RHH_9                PF19839.4           3     0.75     65   0.86   0.86   0.80   0.04\n",
            "17403  RhiE-like_linker     PF22336.1          19     2.67     67   0.84   0.87   0.79   0.05\n",
            "17404  RhiE_B               PF21887.1           2     0.43    120   0.59   0.59   0.52   0.01\n",
            "17405  RHIM                 PF12721.12         22    22.00     22   1.97   1.83   1.97   0.35\n",
            "17406  RHINO                PF15319.11         17     1.30    239   0.59   0.56   0.51   0.04\n",
            "17407  RhlB                 PF12300.13          2     0.53    181   0.59   0.64   0.51   0.06\n",
            "17408  Rhodanese            PF00581.25        139    12.87    102   0.59   0.57   0.45   0.01\n",
            "17409  Rhodanese_C          PF12368.13        126     2.70     66   0.85   0.67   0.74   0.19\n",
            "17410  RhodobacterPufX      PF11511.13          5     1.50     67   0.84   0.84   0.79   0.03\n",
            "17411  Rhodopsin_N          PF10413.14          5     1.23     36   1.51   1.40   1.47   0.14\n",
            "17412  RhoGAP               PF00620.32         82     4.31    148   0.59   0.59   0.54   0.03\n",
            "17413  RhoGAP-FF1           PF16512.10         21     1.55     80   0.71   0.66   0.67   0.07\n",
            "17414  RhoGAP_pG1_pG2       PF19518.4          19     1.03    698   0.59   0.59   0.50   0.02\n",
            "17415  RhoGEF               PF00621.25        154     5.85    180   0.59   0.57   0.46   0.05\n",
            "17416  RhoGEF67_u1          PF16615.10         13     4.52     47   1.17   1.18   1.11   0.19\n",
            "17417  RhoGEF67_u2          PF16614.10          4     0.47     99   0.59   0.59   0.52   0.03\n",
            "17418  Rhomboid             PF01694.27         58     4.19    143   0.59   0.58   0.52   0.09\n",
            "17419  Rhomboid_2           PF20401.3          13     0.59    194   0.59   0.57   0.53   0.04\n",
            "17420  Rhomboid_N           PF12122.13         13     1.84     86   0.66   0.62   0.61   0.04\n",
            "17421  RhopH3_C             PF22808.1          10     0.59    341   0.59   0.57   0.52   0.04\n",
            "17422  Rho_Binding          PF08912.16         32     2.93     69   0.81   0.85   0.77   0.15\n",
            "17423  Rho_GDI              PF02115.22         32     1.10    193   0.59   0.58   0.53   0.02\n",
            "17424  Rho_N                PF07498.17         93    19.56     43   1.28   1.37   1.26   0.14\n",
            "17425  Rho_RNA_bind         PF07497.17        211     2.26     73   0.77   0.78   0.71   0.02\n",
            "17426  RHS                  PF03527.19         15     6.30     38   1.43   1.31   1.42   0.10\n",
            "17427  RHSP                 PF07999.16          3     0.75    312   0.59   0.57   0.52   0.01\n",
            "17428  RHS_N                PF20445.3           3     0.66    127   0.59   0.54   0.51   0.02\n",
            "17429  RHS_repeat           PF05593.19        202   202.00     38   1.29   1.29   1.23   0.18\n",
            "17430  Rhv                  PF00073.25         49     5.28    171   0.59   0.53   0.51   0.03\n",
            "17431  Rhv_5                PF22663.1          10     2.28    216   0.59   0.56   0.47   0.02\n",
            "17432  RH_dom               PF13901.11         91     1.45    206   0.59   0.48   0.52   0.08\n",
            "17433  Rib                  PF08428.15         52     3.41     76   0.74   0.75   0.67   0.09\n",
            "17434  RIB43A               PF05914.17         57     2.87    372   0.59   0.57   0.56   0.11\n",
            "17435  RibD_C               PF01872.22         27     4.45    197   0.59   0.62   0.46   0.01\n",
            "17436  RIBIOP_C             PF04950.17        175     2.61    291   0.59   0.57   0.51   0.02\n",
            "17437  RibLong              PF18957.5          36     2.29     93   0.61   0.61   0.52   0.06\n",
            "17438  Ribonuclease         PF00545.25        293     4.09     98   0.59   0.54   0.43   0.04\n",
            "17439  Ribonuclease_3       PF00636.31        225     9.95    102   0.59   0.61   0.49   0.02\n",
            "17440  Ribonuclease_P       PF00825.23         82     4.19    109   0.59   0.60   0.52   0.06\n",
            "17441  Ribonuclease_T2      PF00445.23        187     2.94    188   0.59   0.49   0.42   0.05\n",
            "17442  Ribonucleas_3_2      PF11469.13          3     0.66    119   0.59   0.64   0.52   0.02\n",
            "17443  Ribonucleas_3_3      PF14622.11         36     2.28    128   0.59   0.62   0.51   0.03\n",
            "17444  Ribonuc_2-5A         PF06479.17        141     1.99    127   0.59   0.55   0.52   0.04\n",
            "17445  Ribonuc_L-PSP        PF01042.26         25     2.79    121   0.59   0.61   0.51   0.03\n",
            "17446  Ribonuc_P_40         PF08584.16         45     3.61    286   0.59   0.57   0.48   0.01\n",
            "17447  Ribonuc_red_2_N      PF08471.15         47     1.12    103   0.59   0.56   0.51   0.03\n",
            "17448  Ribonuc_red_lgC      PF02867.20         61     2.86    523   0.59   0.57   0.50   0.02\n",
            "17449  Ribonuc_red_lgN      PF00317.26        130    11.06     77   0.73   0.72   0.63   0.03\n",
            "17450  Ribonuc_red_sm       PF00268.26        243     4.29    281   0.59   0.55   0.53   0.04\n",
            "17451  Ribophorin_I         PF04597.19        156     3.35    437   0.59   0.57   0.50   0.02\n",
            "17452  Ribophorin_II        PF05817.19         31     2.66    632   0.59   0.61   0.51   0.03\n",
            "17453  Ribosomal_60s        PF00428.24         63     4.81     88   0.65   0.72   0.56   0.09\n",
            "17454  Ribosomal_bL9m_C     PF22078.1           9     0.94     86   0.66   0.64   0.60   0.03\n",
            "17455  Ribosomal_L1         PF00687.26        495     5.98    198   0.59   0.61   0.48   0.03\n",
            "17456  Ribosomal_L10        PF00466.25        102     6.19    100   0.59   0.62   0.53   0.05\n",
            "17457  Ribosomal_L11        PF00298.24        580     5.48     69   0.82   0.86   0.75   0.06\n",
            "17458  Ribosomal_L11_N      PF03946.19        236     6.15     58   0.96   0.99   0.92   0.04\n",
            "17459  Ribosomal_L12        PF00542.24        292     2.32     67   0.84   0.93   0.80   0.08\n",
            "17460  Ribosomal_L12_N      PF16320.10        437    11.72     48   1.15   1.30   1.05   0.14\n",
            "17461  Ribosomal_L13        PF00572.23         46     2.71    119   0.59   0.57   0.55   0.03\n",
            "17462  Ribosomal_L13e       PF01294.23         94     1.50    180   0.59   0.58   0.53   0.05\n",
            "17463  Ribosomal_L14        PF00238.24        262     1.56    122   0.59   0.61   0.52   0.05\n",
            "17464  Ribosomal_L14e       PF01929.22        127     3.86     75   0.75   0.74   0.72   0.08\n",
            "17465  Ribosomal_L15e       PF00827.22         79     1.26    191   0.59   0.57   0.54   0.04\n",
            "17466  Ribosomal_L16        PF00252.23        406     3.47    132   0.59   0.59   0.49   0.03\n",
            "17467  Ribosomal_L17        PF01196.24        504     2.64     97   0.59   0.61   0.50   0.05\n",
            "17468  Ribosomal_L18        PF17135.9          61     1.14    188   0.59   0.61   0.52   0.03\n",
            "17469  Ribosomal_L18A       PF01775.22         60     6.40     59   0.95   0.92   0.82   0.07\n",
            "17470  Ribosomal_L18p       PF00861.27         14     1.08    116   0.59   0.62   0.53   0.04\n",
            "17471  Ribosomal_L18_c      PF14204.11        132     8.57     56   0.99   1.00   0.92   0.18\n",
            "17472  Ribosomal_L19        PF01245.25        345     1.46    111   0.59   0.60   0.54   0.04\n",
            "17473  Ribosomal_L19e       PF01280.25        112     1.60    144   0.59   0.59   0.55   0.06\n",
            "17474  Ribosomal_L2         PF00181.28         24     3.25     77   0.73   0.73   0.66   0.03\n",
            "17475  Ribosomal_L20        PF00453.23        307     1.76    107   0.59   0.59   0.55   0.06\n",
            "17476  Ribosomal_L21e       PF01157.23        138     2.78    101   0.59   0.57   0.55   0.04\n",
            "17477  Ribosomal_L21p       PF00829.26        537     3.02     99   0.59   0.60   0.52   0.06\n",
            "17478  Ribosomal_L22        PF00237.24        645     5.20    104   0.59   0.60   0.53   0.05\n",
            "17479  Ribosomal_L22e       PF01776.22         88     1.61    110   0.59   0.57   0.53   0.05\n",
            "17480  Ribosomal_L23        PF00276.25        536     3.37     86   0.66   0.67   0.58   0.07\n",
            "17481  Ribosomal_L23eN      PF03939.18        106     7.00     51   1.09   1.10   1.06   0.19\n",
            "17482  ribosomal_L24        PF17136.9         381     7.14     65   0.86   0.85   0.76   0.07\n",
            "17483  Ribosomal_L24e       PF01246.25         89     3.18     66   0.85   0.76   0.82   0.07\n",
            "17484  Ribosomal_L25p       PF01386.24        544     5.67     82   0.69   0.71   0.52   0.03\n",
            "17485  Ribosomal_L26        PF16906.10         93     2.49     82   0.69   0.70   0.59   0.06\n",
            "17486  Ribosomal_L27        PF01016.24        321     1.74     79   0.72   0.73   0.66   0.05\n",
            "17487  Ribosomal_L27A       PF00828.24        431     3.44    123   0.59   0.62   0.39   0.04\n",
            "17488  Ribosomal_L27e       PF01777.23         78     2.70     85   0.67   0.63   0.62   0.09\n",
            "17489  Ribosomal_L27_C      PF18471.6          13     1.56    240   0.59   0.61   0.54   0.06\n",
            "17490  Ribosomal_L28        PF00830.24        376     8.88     60   0.93   0.89   0.86   0.09\n",
            "17491  Ribosomal_L28e       PF01778.22        198     4.18    115   0.59   0.57   0.49   0.05\n",
            "17492  Ribosomal_L29        PF00831.28        649    11.81     57   0.98   1.04   0.94   0.14\n",
            "17493  Ribosomal_L29e       PF01779.22         59     5.66     40   1.37   1.26   1.34   0.27\n",
            "17494  Ribosomal_L2_C       PF03947.23        339     1.88    126   0.59   0.60   0.51   0.04\n",
            "17495  Ribosomal_L3         PF00297.27          7     0.99    369   0.59   0.58   0.51   0.02\n",
            "17496  Ribosomal_L30        PF00327.25        700    19.72     51   1.09   1.13   1.05   0.09\n",
            "17497  Ribosomal_L30_N      PF08079.17        129     5.75     72   0.78   0.80   0.77   0.17\n",
            "17498  Ribosomal_L31        PF01197.23        431     3.32     65   0.86   0.78   0.78   0.05\n",
            "17499  Ribosomal_L31e       PF01198.24         79     2.71     82   0.69   0.67   0.66   0.06\n",
            "17500  Ribosomal_L32e       PF01655.23        135     2.22    108   0.59   0.57   0.55   0.04\n",
            "17501  Ribosomal_L32p       PF01783.28        104     8.46     56   0.99   0.92   0.93   0.13\n",
            "17502  Ribosomal_L33        PF00471.25        305     7.36     47   1.17   1.07   1.10   0.17\n",
            "17503  Ribosomal_L34        PF00468.22        245     5.26     44   1.25   1.24   1.23   0.21\n",
            "17504  Ribosomal_L34e       PF01199.23         85     2.10     94   0.61   0.57   0.56   0.06\n",
            "17505  Ribosomal_L35Ae      PF01247.23         85     1.47     95   0.60   0.59   0.54   0.03\n",
            "17506  Ribosomal_L35p       PF01632.24        495     7.68     61   0.92   0.91   0.88   0.16\n",
            "17507  Ribosomal_L36        PF00444.23        161     8.31     38   1.43   1.31   1.40   0.26\n",
            "17508  Ribosomal_L36e       PF01158.23         74     1.85     96   0.60   0.61   0.55   0.07\n",
            "17509  Ribosomal_L37        PF08561.15         75     2.35     89   0.64   0.62   0.53   0.06\n",
            "17510  Ribosomal_L37ae      PF01780.24         90     2.19     85   0.67   0.60   0.63   0.07\n",
            "17511  Ribosomal_L37e       PF01907.24         76     2.22     53   1.05   0.87   0.99   0.17\n",
            "17512  Ribosomal_L38e       PF01781.23         79     2.70     68   0.83   0.84   0.76   0.08\n",
            "17513  Ribosomal_L39        PF00832.25         68     6.27     42   1.31   1.21   1.29   0.25\n",
            "17514  Ribosomal_L4         PF00573.27        105     4.19    190   0.59   0.62   0.51   0.03\n",
            "17515  Ribosomal_L40e       PF01020.22         20     1.91     50   1.11   1.00   1.07   0.17\n",
            "17516  Ribosomal_L41        PF05162.18          5     2.98     25   2.13   2.02   2.10   0.82\n",
            "17517  Ribosomal_L44        PF00935.24         70     2.70     76   0.74   0.69   0.70   0.12\n",
            "17518  Ribosomal_L5         PF00281.24        440     7.21     57   0.98   1.00   0.94   0.06\n",
            "17519  Ribosomal_L50        PF10501.14         42     4.35    109   0.59   0.59   0.52   0.04\n",
            "17520  Ribosomal_L5e        PF17144.9          94     1.27    163   0.59   0.59   0.54   0.03\n",
            "17521  Ribosomal_L5_C       PF00673.26        353     2.95     94   0.61   0.60   0.52   0.02\n",
            "17522  Ribosomal_L6         PF00347.28         87     9.55     80   0.71   0.74   0.63   0.05\n",
            "17523  Ribosomal_L6e        PF01159.24        128     1.82    111   0.59   0.58   0.51   0.05\n",
            "17524  Ribosomal_L6e_N      PF03868.20         40     3.51     60   0.93   0.91   0.87   0.13\n",
            "17525  Ribosomal_L7Ae       PF01248.31         50     4.81     95   0.60   0.63   0.55   0.04\n",
            "17526  Ribosomal_L9_C       PF03948.19        708     4.35     86   0.66   0.71   0.60   0.06\n",
            "17527  Ribosomal_L9_N       PF01281.24        469    12.33     47   1.17   1.23   1.17   0.06\n",
            "17528  Ribosomal_S10        PF00338.27        323     5.64     98   0.59   0.58   0.55   0.04\n",
            "17529  Ribosomal_S11        PF00411.24          7     0.73    110   0.59   0.62   0.52   0.03\n",
            "17530  Ribosomal_S12        PF17487.7           5     0.59     87   0.65   0.60   0.58   0.11\n",
            "17531  Ribosomal_S13        PF00416.27         11     1.11    128   0.59   0.59   0.53   0.04\n",
            "17532  Ribosomal_S13_N      PF08069.17        105     2.91     60   0.93   0.91   0.90   0.04\n",
            "17533  Ribosomal_S14        PF00253.26        297     5.82     54   1.03   0.99   1.00   0.08\n",
            "17534  Ribosomal_S15        PF00312.27        550     4.84     81   0.70   0.69   0.67   0.06\n",
            "17535  Ribosomal_S16        PF00886.24        524     3.97     62   0.90   0.87   0.84   0.03\n",
            "17536  Ribosomal_S17        PF00366.25        486     5.34     68   0.83   0.79   0.80   0.08\n",
            "17537  Ribosomal_S17e       PF00833.23         39     1.25    122   0.59   0.59   0.53   0.05\n",
            "17538  Ribosomal_S17_N      PF16205.10         56     1.63     69   0.82   0.78   0.75   0.04\n",
            "17539  Ribosomal_S18        PF01084.25        315     8.16     52   1.07   1.07   1.05   0.06\n",
            "17540  Ribosomal_S19        PF00203.26        277     2.76     81   0.70   0.67   0.63   0.04\n",
            "17541  Ribosomal_S19e       PF01090.24        137     1.87    137   0.59   0.59   0.54   0.02\n",
            "17542  Ribosomal_S2         PF00318.25        337     2.25    216   0.59   0.58   0.54   0.02\n",
            "17543  Ribosomal_S20p       PF01649.23        564     4.26     83   0.68   0.72   0.65   0.15\n",
            "17544  Ribosomal_S21        PF01165.25        304    17.82     55   1.01   1.01   1.00   0.17\n",
            "17545  Ribosomal_S21e       PF01249.23         58     1.56     79   0.72   0.71   0.67   0.03\n",
            "17546  Ribosomal_S24e       PF01282.24        133     5.82     79   0.72   0.72   0.70   0.04\n",
            "17547  Ribosomal_S25        PF03297.20         62     2.73    101   0.59   0.61   0.54   0.07\n",
            "17548  Ribosomal_S26e       PF01283.24         32     1.00    108   0.59   0.55   0.53   0.07\n",
            "17549  Ribosomal_S27        PF01599.24         55     3.51     44   1.25   1.09   1.19   0.16\n",
            "17550  Ribosomal_S27e       PF01667.22          7     1.56     55   1.01   0.92   0.98   0.10\n",
            "17551  Ribosomal_S28e       PF01200.23         39     2.38     64   0.88   0.92   0.84   0.06\n",
            "17552  Ribosomal_S30        PF04758.19         30     2.55     58   0.96   0.95   0.89   0.14\n",
            "17553  Ribosomal_S30AE      PF02482.24        653     9.52     88   0.65   0.67   0.45   0.07\n",
            "17554  Ribosomal_S3Ae       PF01015.23        119     2.83    191   0.59   0.58   0.53   0.04\n",
            "17555  Ribosomal_S3_C       PF00189.25        211     2.57     83   0.68   0.71   0.64   0.05\n",
            "17556  Ribosomal_S4         PF00163.24        360     6.44     95   0.60   0.60   0.47   0.07\n",
            "17557  Ribosomal_S4e        PF00900.25        145     5.13     75   0.75   0.76   0.72   0.04\n",
            "17558  Ribosomal_S5         PF00333.25        317     6.07     65   0.86   0.90   0.85   0.07\n",
            "17559  Ribosomal_S5_C       PF03719.20        410     6.17     71   0.79   0.83   0.77   0.03\n",
            "17560  Ribosomal_S6         PF01250.22        649     5.00     90   0.63   0.63   0.60   0.03\n",
            "17561  Ribosomal_S6e        PF01092.24         93     1.92    125   0.59   0.61   0.54   0.03\n",
            "17562  Ribosomal_S7         PF00177.26        365     2.50    149   0.59   0.61   0.52   0.05\n",
            "17563  Ribosomal_S7e        PF01251.23         61     1.26    182   0.59   0.61   0.53   0.04\n",
            "17564  Ribosomal_S8         PF00410.24        366     1.54    126   0.59   0.60   0.50   0.03\n",
            "17565  Ribosomal_S8e        PF01201.27        214     2.77    133   0.59   0.61   0.49   0.05\n",
            "17566  Ribosomal_S9         PF00380.24        575     2.45    121   0.59   0.61   0.51   0.04\n",
            "17567  Ribosomal_TL5_C      PF14693.11        546     6.12     84   0.68   0.73   0.62   0.06\n",
            "17568  Ribosomal_uL24m-like PF22682.1          39     1.56    223   0.59   0.57   0.53   0.03\n",
            "17569  Ribosomal_uS5m_N     PF21251.2          18     0.90    120   0.59   0.58   0.52   0.04\n",
            "17570  Ribosom_S12_S23      PF00164.30         12     0.93    114   0.59   0.61   0.51   0.03\n",
            "17571  Ribosom_S30AE_C      PF16321.10        274     5.93     57   0.98   0.96   0.96   0.05\n",
            "17572  Ribos_L4_asso_C      PF14374.11         85     2.35     77   0.73   0.74   0.68   0.07\n",
            "17573  Ribo_biogen_C        PF04034.18        160     1.27    127   0.59   0.58   0.53   0.02\n",
            "17574  Ribul_P_3_epim       PF00834.24         12     1.12    198   0.59   0.59   0.53   0.02\n",
            "17575  Rib_5-P_isom_A       PF06026.19        405     2.77    173   0.59   0.62   0.51   0.02\n",
            "17576  Rib_hydrolayse       PF02267.22         26     1.27    241   0.59   0.52   0.53   0.04\n",
            "17577  Rib_mS39_PPR         PF22330.1          15     1.08    194   0.59   0.57   0.54   0.03\n",
            "17578  Rib_recp_KP_reg      PF05104.17         39     2.74    141   0.59   0.62   0.47   0.14\n",
            "17579  RIC1                 PF07064.18         64     2.57    250   0.59   0.58   0.50   0.03\n",
            "17580  RIC3                 PF15361.11         22     2.04    146   0.59   0.59   0.46   0.03\n",
            "17581  Ric8                 PF10165.14         86     3.46    447   0.59   0.61   0.46   0.02\n",
            "17582  RICH                 PF05062.17         25     2.73     82   0.69   0.71   0.62   0.11\n",
            "17583  RicinB_lectin_2      PF14200.11        190     9.37     89   0.64   0.58   0.50   0.07\n",
            "17584  Ricin_B_lectin       PF00652.27         85     5.74    126   0.59   0.50   0.45   0.07\n",
            "17585  RickCE_cat           PF22179.1           3     0.52    102   0.59   0.58   0.52   0.03\n",
            "17586  RickCE_N             PF22189.1           2     0.44    162   0.59   0.59   0.52   0.04\n",
            "17587  Rick_17kDa_Anti      PF05433.20         68    23.53     43   1.28   1.48   1.26   0.51\n",
            "17588  RICTOR_M             PF14666.11         92     3.26    104   0.59   0.59   0.54   0.04\n",
            "17589  RICTOR_N             PF14664.11         89     3.52    371   0.59   0.62   0.51   0.03\n",
            "17590  RICTOR_phospho       PF14665.11          7     0.69    108   0.59   0.61   0.51   0.03\n",
            "17591  RICTOR_V             PF14668.11         88     3.75     71   0.79   0.79   0.76   0.05\n",
            "17592  Rieske               PF00355.31         46     3.69     88   0.65   0.56   0.52   0.04\n",
            "17593  Rieske_2             PF13806.11         35     1.64    104   0.59   0.58   0.53   0.01\n",
            "17594  Rieske_3             PF18465.6          13     0.75     96   0.59   0.53   0.52   0.03\n",
            "17595  Rif1_N               PF12231.13         64     3.77    374   0.59   0.57   0.54   0.03\n",
            "17596  RIF2_ASCE            PF21449.2           2     0.54     93   0.61   0.62   0.55   0.06\n",
            "17597  RIF2_lid             PF21450.2           2     0.54     85   0.67   0.66   0.61   0.04\n",
            "17598  RIF5_SNase_1         PF18187.6           2     0.52    166   0.59   0.59   0.51   0.01\n",
            "17599  RIF5_SNase_2         PF18189.6           2     0.61    179   0.59   0.60   0.52   0.02\n",
            "17600  RIFIN                PF02009.21        112     1.54    327   0.59   0.58   0.51   0.05\n",
            "17601  RIG-I_C              PF18119.6          11     2.46    141   0.59   0.57   0.51   0.05\n",
            "17602  RIG-I_C-RD           PF11648.14         55     1.89    117   0.59   0.49   0.53   0.06\n",
            "17603  RIH_assoc            PF08454.16         61     2.54     99   0.59   0.55   0.53   0.07\n",
            "17604  RIIa                 PF02197.22         12     9.30     38   1.44   1.44   1.42   0.09\n",
            "17605  RII_binding_1        PF10522.14          6     6.00     19   1.39   1.50   1.38   0.24\n",
            "17606  RILP                 PF11461.13         26     2.80     69   0.81   0.85   0.78   0.09\n",
            "17607  RIM2a_ZnF            PF22601.1           6     0.83     52   1.07   0.89   1.00   0.20\n",
            "17608  RIMC1                PF17716.6           8     0.71    256   0.59   0.56   0.52   0.02\n",
            "17609  RimK                 PF08443.16          4     0.97    188   0.59   0.62   0.52   0.03\n",
            "17610  Rimk_N               PF18030.6         169     2.08     94   0.61   0.61   0.57   0.03\n",
            "17611  RimM                 PF01782.23         96     7.49     84   0.68   0.70   0.62   0.02\n",
            "17612  RimP_N               PF02576.22         28     3.23     73   0.77   0.81   0.71   0.03\n",
            "17613  RinB                 PF06116.17          2     1.27     51   1.08   1.14   1.03   0.09\n",
            "17614  RINGv                PF12906.12         21     2.02     47   1.17   0.88   1.09   0.31\n",
            "17615  RING_CBP-p300        PF06001.18         13     1.88     40   1.37   1.15   1.33   0.20\n",
            "17616  Ring_hydroxyl_A      PF00848.24         32     5.71    208   0.59   0.52   0.50   0.03\n",
            "17617  Ring_hydroxyl_B      PF00866.23         15     2.49    144   0.59   0.56   0.53   0.04\n",
            "17618  RINT1_TIP1           PF04437.18         56     3.20    514   0.59   0.58   0.49   0.03\n",
            "17619  RIO1                 PF01163.27         27     1.93    187   0.59   0.57   0.52   0.03\n",
            "17620  Rio2_N               PF09202.16         25     1.60     82   0.69   0.70   0.64   0.03\n",
            "17621  RIOX1_C_WH           PF21233.2          24     1.11    129   0.59   0.59   0.54   0.03\n",
            "17622  RIP                  PF00161.24         32     3.69    197   0.59   0.59   0.50   0.02\n",
            "17623  Ripply               PF14998.11         25     1.18     86   0.66   0.59   0.59   0.04\n",
            "17624  RIP_P131-like        PF22238.1           6     1.00     96   0.59   0.62   0.52   0.02\n",
            "17625  RisS_PPD             PF16524.10         23     1.27    107   0.59   0.59   0.54   0.03\n",
            "17626  Rit1_C               PF17184.9         103     1.17    271   0.59   0.55   0.49   0.02\n",
            "17627  RITA                 PF17066.10          9     0.95    268   0.59   0.58   0.52   0.04\n",
            "17628  RIX1                 PF08167.17         84     6.34    193   0.59   0.61   0.51   0.04\n",
            "17629  RKP_N                PF19322.4          29     0.99    137   0.59   0.62   0.53   0.03\n",
            "17630  RL                   PF17797.6          13     0.99     71   0.79   0.78   0.75   0.05\n",
            "17631  RL10P_insert         PF17777.6          21     2.97     71   0.79   0.85   0.75   0.03\n",
            "17632  RL11D                PF11088.13          2     0.57     99   0.59   0.55   0.51   0.04\n",
            "17633  RLAN                 PF14401.11         54     1.91    150   0.59   0.58   0.54   0.02\n",
            "17634  RlaP                 PF10127.14         20     2.45    249   0.59   0.58   0.53   0.02\n",
            "17635  RLI                  PF04068.20         65    19.22     35   1.55   1.41   1.54   0.20\n",
            "17636  RLIP76_Ral-bd        PF20924.2           2     1.01     49   1.13   1.12   1.08   0.12\n",
            "17637  RLL                  PF10036.14         40     3.02    253   0.59   0.60   0.49   0.04\n",
            "17638  RlmA_N               PF21302.2          64     1.82     44   1.25   1.12   1.22   0.10\n",
            "17639  RlmL_1st             PF22020.1         740     7.79     55   1.01   1.06   0.97   0.04\n",
            "17640  RlmM_FDX             PF18125.6           9     1.32     71   0.79   0.76   0.75   0.04\n",
            "17641  RLMM_N               PF21239.2          37     1.25     81   0.70   0.70   0.64   0.02\n",
            "17642  RlmN_N               PF21016.2         133     2.30     60   0.93   0.87   0.90   0.06\n",
            "17643  RloB                 PF13707.11         90     5.46    195   0.59   0.56   0.35   0.04\n",
            "17644  RM44_endonuclase     PF22935.1         152     2.73    139   0.59   0.61   0.54   0.03\n",
            "17645  RMC1_C               PF07035.17         18     2.25    248   0.59   0.60   0.52   0.04\n",
            "17646  RMC1_N               PF21029.2          85     2.90    119   0.59   0.57   0.53   0.04\n",
            "17647  RMD1-3               PF21033.2           1     0.39    235   0.59   0.59   0.52   0.02\n",
            "17648  RME-8_N              PF19432.4          33     1.16    818   0.59   0.58   0.52   0.02\n",
            "17649  RMF                  PF04957.17         21     1.37     54   1.03   0.94   0.98   0.12\n",
            "17650  RMI1_C               PF16099.10         20     2.09    143   0.59   0.58   0.53   0.03\n",
            "17651  RMI1_N_C             PF08585.17         95     3.64    146   0.59   0.61   0.45   0.03\n",
            "17652  RMI1_N_N             PF21000.2          95    53.60     49   1.13   1.12   1.03   0.10\n",
            "17653  RMI2                 PF16100.10         24     1.67    132   0.59   0.58   0.48   0.03\n",
            "17654  RmlD_sub_bind        PF04321.22         21     1.84    287   0.59   0.59   0.53   0.02\n",
            "17655  RMMBL                PF07521.17         81    15.91     63   0.89   0.88   0.82   0.03\n",
            "17656  RMNT_CmcI            PF04989.17         11     0.93    206   0.59   0.58   0.52   0.01\n",
            "17657  RMP                  PF14996.11         18     1.27    154   0.59   0.49   0.53   0.06\n",
            "17658  RMP1                 PF20945.2         136     1.93     95   0.60   0.56   0.52   0.05\n",
            "17659  RmuC                 PF02646.21        243     3.56    291   0.59   0.61   0.52   0.04\n",
            "17660  RNA12                PF10443.14         17     1.10    429   0.59   0.60   0.52   0.02\n",
            "17661  RnaseA               PF00074.25        133     2.16    121   0.59   0.47   0.51   0.08\n",
            "17662  RNAseD_HRDC_C        PF21293.2          30     1.57     66   0.85   0.82   0.80   0.05\n",
            "17663  RNaseH_C             PF09293.15          2     0.54    124   0.59   0.59   0.52   0.02\n",
            "17664  RNaseH_like          PF04308.17         46     1.62    145   0.59   0.59   0.53   0.03\n",
            "17665  RNaseH_pPIWI_RE      PF13032.11         38     6.39    288   0.59   0.56   0.46   0.03\n",
            "17666  RNase_3_N            PF18497.6          39     3.10     95   0.60   0.63   0.51   0.05\n",
            "17667  RNAse_A_bac          PF18431.6          24     2.37    113   0.59   0.58   0.52   0.03\n",
            "17668  RNase_E_G            PF10150.14        431     2.22    270   0.59   0.61   0.53   0.04\n",
            "17669  RNase_E_G_Thio       PF20833.2         560     2.21     86   0.66   0.65   0.63   0.04\n",
            "17670  RNase_H              PF00075.29         22     2.27    141   0.59   0.57   0.45   0.03\n",
            "17671  RNase_H2-Ydr279      PF09468.15         82     5.97    163   0.59   0.59   0.45   0.04\n",
            "17672  RNase_H2_suC         PF08615.16         18     2.96    133   0.59   0.56   0.47   0.02\n",
            "17673  RNase_HII            PF01351.23         11     2.16    199   0.59   0.61   0.52   0.02\n",
            "17674  RNase_H_2            PF13482.11         54     3.81    165   0.59   0.56   0.49   0.03\n",
            "17675  RNase_II_C_S1        PF18614.6          94    10.36     59   0.95   1.04   0.90   0.09\n",
            "17676  RNase_J_b_CASP       PF22505.1          35     1.03    126   0.59   0.62   0.54   0.03\n",
            "17677  RNase_J_C            PF17770.6          70     2.27    102   0.59   0.62   0.55   0.04\n",
            "17678  RNase_P-MRP_p29      PF01868.21        185     3.31     84   0.68   0.70   0.59   0.03\n",
            "17679  RNase_PH             PF01138.26        136     7.72    130   0.59   0.62   0.48   0.02\n",
            "17680  RNase_PH_C           PF03725.20         77    17.58     67   0.84   0.88   0.78   0.04\n",
            "17681  RNase_P_p30          PF01876.21         38     5.05    214   0.59   0.61   0.51   0.02\n",
            "17682  RNase_P_pop3         PF08228.16          5     1.18    158   0.59   0.60   0.50   0.03\n",
            "17683  RNase_P_Rpp14        PF01900.24        185     4.27    104   0.59   0.58   0.47   0.03\n",
            "17684  RNase_T              PF00929.29         40     7.12    164   0.59   0.58   0.50   0.02\n",
            "17685  RNase_Y_N            PF12072.13         42     1.87    201   0.59   0.63   0.55   0.10\n",
            "17686  RNase_Zc3h12a        PF11977.13         19     2.48    154   0.59   0.57   0.51   0.02\n",
            "17687  RNase_Zc3h12a_2      PF14626.11          3     0.48    122   0.59   0.57   0.51   0.02\n",
            "17688  RNA_bind             PF08675.16         10     0.97     79   0.72   0.65   0.66   0.03\n",
            "17689  RNA_binding          PF01877.22         82     4.04    120   0.59   0.61   0.50   0.04\n",
            "17690  RNA_capsid           PF03035.19         34     1.27    238   0.59   0.58   0.51   0.03\n",
            "17691  RNA_helicase         PF00910.27         99     4.59    105   0.59   0.55   0.51   0.02\n",
            "17692  RNA_helicase_helical PF21210.2           4     0.73    108   0.59   0.64   0.52   0.04\n",
            "17693  RNA_ligase           PF09414.15        132     7.93    171   0.59   0.58   0.30   0.02\n",
            "17694  RNA_lig_T4_1         PF09511.15        120     3.00    227   0.59   0.57   0.47   0.03\n",
            "17695  RNA_pol              PF00940.24        183     1.76    448   0.59   0.56   0.50   0.02\n",
            "17696  RNA_polI_A14         PF08203.16         24     2.15    117   0.59   0.62   0.50   0.05\n",
            "17697  RNA_polI_A34         PF08208.16         71     6.27    205   0.59   0.60   0.49   0.08\n",
            "17698  RNA_pol_3_Rpc31      PF11705.13         84     4.60    212   0.59   0.58   0.46   0.08\n",
            "17699  RNA_pol_A_bac        PF01000.31         59     3.50    119   0.59   0.60   0.47   0.03\n",
            "17700  RNA_pol_A_CTD        PF03118.20         53     4.05     67   0.84   0.89   0.81   0.07\n",
            "17701  RNA_pol_inhib        PF16857.10          8     2.14     47   1.17   1.18   1.12   0.06\n",
            "17702  RNA_pol_I_A49        PF06870.17        101     4.57    377   0.59   0.60   0.50   0.03\n",
            "17703  RNA_pol_L            PF01193.29        351     9.10     74   0.76   0.80   0.57   0.02\n",
            "17704  RNA_pol_L_2          PF13656.11        223     6.12     75   0.75   0.75   0.71   0.04\n",
            "17705  RNA_POL_M_15KD       PF02150.21          9     5.30     36   1.51   1.29   1.45   0.24\n",
            "17706  RNA_pol_N            PF01194.22         91     2.86     58   0.96   0.86   0.91   0.07\n",
            "17707  RNA_pol_Rbc25        PF08292.17         23     1.62    121   0.59   0.55   0.51   0.03\n",
            "17708  RNA_pol_Rpa2_4       PF06883.17        110     4.08     58   0.96   0.96   0.90   0.02\n",
            "17709  RNA_pol_Rpb1_1       PF04997.17         25     1.96    312   0.59   0.57   0.51   0.03\n",
            "17710  RNA_pol_Rpb1_2       PF00623.25         18     1.16    166   0.59   0.59   0.53   0.02\n",
            "17711  RNA_pol_Rpb1_3       PF04983.23        112     6.65    162   0.59   0.58   0.47   0.02\n",
            "17712  RNA_pol_Rpb1_4       PF05000.22        106     5.15    107   0.59   0.58   0.50   0.04\n",
            "17713  RNA_pol_Rpb1_5       PF04998.22         24     1.77    267   0.59   0.59   0.46   0.03\n",
            "17714  RNA_pol_Rpb1_6       PF04992.19        109     2.60    189   0.59   0.59   0.52   0.04\n",
            "17715  RNA_pol_Rpb1_7       PF04990.17         67     1.68    136   0.59   0.59   0.52   0.05\n",
            "17716  RNA_pol_Rpb1_R       PF05001.18         79    79.00     14   2.69   2.63   2.63   1.06\n",
            "17717  RNA_pol_Rpb2_1       PF04563.20         13     1.67    203   0.59   0.59   0.48   0.02\n",
            "17718  RNA_pol_Rpb2_2       PF04561.19         22     4.21    190   0.59   0.59   0.50   0.03\n",
            "17719  RNA_pol_Rpb2_3       PF04565.21         96     3.58     68   0.83   0.80   0.78   0.02\n",
            "17720  RNA_pol_Rpb2_4       PF04566.18        230     3.73     62   0.90   0.90   0.87   0.05\n",
            "17721  RNA_pol_Rpb2_45      PF10385.14        545     4.54     66   0.85   0.85   0.78   0.07\n",
            "17722  RNA_pol_Rpb2_5       PF04567.22        134     4.22     66   0.85   0.79   0.74   0.06\n",
            "17723  RNA_pol_Rpb2_6       PF00562.33         63     2.08    373   0.59   0.58   0.50   0.02\n",
            "17724  RNA_pol_Rpb2_7       PF04560.25         45     1.50     87   0.65   0.65   0.59   0.03\n",
            "17725  RNA_pol_Rpb4         PF03874.21         48     4.88    116   0.59   0.61   0.48   0.06\n",
            "17726  RNA_pol_Rpb5_C       PF01191.24        130     3.24     73   0.77   0.77   0.74   0.04\n",
            "17727  RNA_pol_Rpb5_N       PF03871.19        129     2.78     89   0.64   0.59   0.57   0.05\n",
            "17728  RNA_pol_Rpb6         PF01192.27        301    23.01     53   1.05   1.11   0.89   0.06\n",
            "17729  RNA_pol_Rpb8         PF03870.20        109     1.40    141   0.59   0.58   0.51   0.03\n",
            "17730  RNA_pol_RpbG         PF16992.10          7     1.14    119   0.59   0.59   0.52   0.03\n",
            "17731  RNA_pol_Rpc34        PF05158.17         14     2.05    332   0.59   0.59   0.51   0.02\n",
            "17732  RNA_pol_Rpc4         PF05132.19        156     4.74    143   0.59   0.60   0.50   0.03\n",
            "17733  RNA_pol_Rpc82        PF05645.18         29     5.94    249   0.59   0.59   0.44   0.03\n",
            "17734  RNA_pol_Rpo13        PF12136.13          7     4.84     43   1.28   1.34   1.23   0.19\n",
            "17735  RNA_replicase_B      PF03431.18          5     1.01    540   0.59   0.58   0.51   0.01\n",
            "17736  RNB                  PF00773.24         53     4.72    326   0.59   0.59   0.49   0.02\n",
            "17737  Rnf-Nqr              PF02508.19        568     2.99    174   0.59   0.62   0.52   0.09\n",
            "17738  RNF111_N             PF15303.11          8     0.68    279   0.59   0.60   0.50   0.05\n",
            "17739  RNF152_C             PF19325.4           6     0.69    136   0.59   0.59   0.50   0.02\n",
            "17740  RNF180_C             PF19332.4           9     0.72    101   0.59   0.51   0.52   0.04\n",
            "17741  RNF220               PF15926.10          6     0.93    249   0.59   0.57   0.50   0.04\n",
            "17742  RNF34L-like_3rd      PF22968.1           1     0.67     55   1.01   0.98   0.95   0.06\n",
            "17743  RnfC_N               PF13375.11         41     1.62    101   0.59   0.60   0.54   0.03\n",
            "17744  Rng_hyd_C            PF21274.2         114     3.08    107   0.59   0.61   0.50   0.03\n",
            "17745  RNHCP                PF12647.12         43     1.09     90   0.63   0.52   0.57   0.09\n",
            "17746  Rnk_N                PF14760.11         70    14.72     41   1.34   1.47   1.28   0.15\n",
            "17747  RnlA-toxin_DBD       PF19034.5           7     1.47    125   0.59   0.55   0.53   0.04\n",
            "17748  RnlA_toxin           PF15935.10          7     2.39     87   0.65   0.64   0.61   0.05\n",
            "17749  RnlA_toxin_N         PF19417.4          18     4.59     82   0.69   0.68   0.64   0.05\n",
            "17750  RnlB_antitoxin       PF15933.10          9     2.23     94   0.61   0.61   0.55   0.05\n",
            "17751  RNPP_C               PF18768.6           4     0.92    213   0.59   0.57   0.53   0.05\n",
            "17752  RNR-II_ins_dom       PF21995.1          12     0.65    107   0.59   0.60   0.52   0.01\n",
            "17753  RNR_Alpha            PF17975.6          31     1.00    101   0.59   0.54   0.53   0.03\n",
            "17754  RNR_inhib            PF08591.15         39     2.63    105   0.59   0.59   0.46   0.04\n",
            "17755  RNR_N                PF08343.15         98     2.17     82   0.69   0.65   0.65   0.08\n",
            "17756  RNR_OB1_NT           PF22896.1           9     1.12     99   0.59   0.60   0.52   0.04\n",
            "17757  Robl_LC7             PF03259.22         37     9.02     91   0.63   0.69   0.59   0.03\n",
            "17758  Roc                  PF08477.18        154     2.56    119   0.59   0.56   0.50   0.03\n",
            "17759  Rod-binding          PF10135.14        216    18.10     51   1.09   1.08   0.97   0.11\n",
            "17760  RodZ_C               PF13464.11        324    12.19     72   0.78   0.82   0.75   0.03\n",
            "17761  Rod_C                PF10493.14         13     1.24    551   0.59   0.58   0.52   0.03\n",
            "17762  Rod_cone_degen       PF15201.11          3     0.85     54   1.03   1.02   0.97   0.03\n",
            "17763  ROF                  PF07073.17         25     1.92     80   0.71   0.69   0.66   0.05\n",
            "17764  Rogdi_lz             PF10259.14         84     4.32    295   0.59   0.61   0.47   0.03\n",
            "17765  ROH1-like            PF05633.16          6     0.71    384   0.59   0.59   0.50   0.02\n",
            "17766  ROK                  PF00480.25         10     2.14    294   0.59   0.60   0.53   0.02\n",
            "17767  ROKNT                PF08067.16          6     1.49     41   1.34   1.31   1.25   0.20\n",
            "17768  RolB_RolC            PF02027.22         13     2.99    189   0.59   0.55   0.48   0.02\n",
            "17769  Rol_Rep_N            PF18106.6          14     1.58     91   0.63   0.58   0.56   0.03\n",
            "17770  Romo1                PF10247.14         54     3.29     66   0.85   0.85   0.82   0.13\n",
            "17771  rOmpB_passenger      PF12334.13          2     0.46    914   0.59   0.62   0.52   0.04\n",
            "17772  Rootletin            PF15035.11         24     2.58    188   0.59   0.59   0.51   0.10\n",
            "17773  Root_cap             PF06830.16         34     1.58     57   0.98   0.93   0.94   0.04\n",
            "17774  Rop                  PF01815.21          3     1.03     57   0.98   0.98   0.93   0.08\n",
            "17775  Rop-like             PF05082.18         44     2.65     64   0.88   0.89   0.84   0.08\n",
            "17776  Roquin_1_2-like_ROQ  PF21206.2           4     0.72     87   0.65   0.66   0.59   0.03\n",
            "17777  ROQ_II               PF18386.6           4     1.13     56   0.99   0.97   0.95   0.05\n",
            "17778  Rossmann-like        PF10727.14          4     0.65    131   0.59   0.65   0.52   0.03\n",
            "17779  Rost                 PF21534.2           1     0.35    275   0.59   0.54   0.51   0.05\n",
            "17780  ROS_MUCR             PF05443.16        133     1.82    123   0.59   0.59   0.52   0.03\n",
            "17781  Rot1                 PF10681.14         59     0.93    209   0.59   0.54   0.51   0.01\n",
            "17782  Rotamase             PF00639.26         35     3.63     97   0.59   0.60   0.52   0.03\n",
            "17783  Rotamase_2           PF13145.11         93    18.13    121   0.59   0.63   0.52   0.05\n",
            "17784  Rotamase_3           PF13616.11         15     2.21    116   0.59   0.60   0.50   0.03\n",
            "17785  Rotavirus_VP1        PF12289.13          2     0.53    315   0.59   0.59   0.52   0.03\n",
            "17786  Rotavirus_VP3        PF06929.16          2     0.55    729   0.59   0.56   0.51   0.03\n",
            "17787  Rotavirus_VP7        PF05868.16          3     0.52    249   0.59   0.55   0.52   0.03\n",
            "17788  Rota_Capsid_VP6      PF00980.22          2     0.54    396   0.59   0.58   0.53   0.02\n",
            "17789  Rota_NS26            PF01525.21          2     0.74    245   0.59   0.57   0.51   0.04\n",
            "17790  Rota_NS35_C          PF02509.19          2     0.57    173   0.59   0.56   0.52   0.02\n",
            "17791  Rota_NS35_N          PF21067.2           2     0.55    143   0.59   0.58   0.52   0.03\n",
            "17792  Rota_NS53            PF00981.22         14     0.71    488   0.59   0.53   0.53   0.03\n",
            "17793  Rota_NS6             PF04866.17          1     0.40     92   0.62   0.57   0.55   0.05\n",
            "17794  Rota_NSP3            PF01665.21          2     0.66    311   0.59   0.58   0.51   0.03\n",
            "17795  Rota_NSP4            PF01452.21          1     0.40    173   0.59   0.59   0.52   0.03\n",
            "17796  Rota_VP2             PF05087.17          2     0.52    890   0.59   0.58   0.52   0.02\n",
            "17797  Rota_VP4_MID         PF17477.7           2     0.52    238   0.59   0.57   0.51   0.02\n",
            "17798  Roughex              PF06020.16          2     0.56    379   0.59   0.60   0.51   0.02\n",
            "17799  RovC_DNA-bd          PF10074.14         43     3.31    107   0.59   0.61   0.52   0.04\n",
            "17800  Rox3                 PF08633.15         56     1.89    204   0.59   0.58   0.46   0.04\n",
            "17801  RoxA-like_Cyt-c      PF21419.2          21     1.69     39   1.40   1.33   1.36   0.09\n",
            "17802  ROXA-like_wH         PF20514.3          24     6.69    119   0.59   0.60   0.48   0.02\n",
            "17803  RP-C                 PF03428.18         30     2.22    175   0.59   0.62   0.54   0.03\n",
            "17804  RP-C_C               PF11800.13         27     1.49    102   0.59   0.60   0.54   0.02\n",
            "17805  RP1-2                PF12042.13         17     2.23    166   0.59   0.68   0.53   0.16\n",
            "17806  RP853                PF17542.7           2     0.44    317   0.59   0.60   0.52   0.04\n",
            "17807  RP854                PF17460.7           1     0.39    212   0.59   0.58   0.51   0.05\n",
            "17808  RPA                  PF10134.14         14     1.49    228   0.59   0.59   0.51   0.02\n",
            "17809  RPA43_OB             PF17875.6          64     4.13    115   0.59   0.57   0.49   0.02\n",
            "17810  RPAP1_C              PF08620.15         91     2.80     69   0.82   0.82   0.75   0.02\n",
            "17811  RPAP1_N              PF08621.15        110    10.83     46   1.20   1.26   1.19   0.19\n",
            "17812  RPAP2_Rtr1           PF04181.18        109     3.00     75   0.75   0.64   0.69   0.09\n",
            "17813  RPAP3_C              PF13877.11        110     7.64     92   0.62   0.62   0.54   0.04\n",
            "17814  RPA_C                PF08784.16         14     2.73    106   0.59   0.59   0.52   0.04\n",
            "17815  RPA_interact_C       PF14768.11         61     2.08     80   0.71   0.57   0.62   0.13\n",
            "17816  RPA_interact_M       PF14767.11         19     2.59     94   0.61   0.59   0.50   0.08\n",
            "17817  RPA_interact_N       PF14766.11         30    14.30     39   1.40   1.32   1.37   0.26\n",
            "17818  RPC3_helical         PF20912.2          79     6.85     64   0.88   0.93   0.81   0.11\n",
            "17819  RPC5                 PF04801.18         42     3.71    429   0.59   0.57   0.49   0.03\n",
            "17820  RPC5_C               PF19725.4          23     0.92    244   0.59   0.58   0.52   0.02\n",
            "17821  RPE65                PF03055.20        454     5.46    469   0.59   0.55   0.40   0.01\n",
            "17822  RPEL                 PF02755.20        114   114.00     24   1.91   2.01   1.91   0.28\n",
            "17823  Rpf1_C               PF11574.13         12     1.70     88   0.65   0.67   0.58   0.05\n",
            "17824  RPGR1_C              PF18111.6           6     1.03    165   0.59   0.60   0.51   0.02\n",
            "17825  RPM2                 PF08579.16          5     0.76    119   0.59   0.59   0.52   0.05\n",
            "17826  Rpn13_ADRM1_Pru      PF04683.18        124     2.26    102   0.59   0.54   0.49   0.03\n",
            "17827  RPN13_C              PF16550.10         47     2.57    112   0.59   0.63   0.51   0.04\n",
            "17828  RPN1_C               PF18051.6          37     1.87     54   1.03   1.05   0.99   0.02\n",
            "17829  RPN1_RPN2_N          PF17781.6          19     1.25    301   0.59   0.59   0.53   0.03\n",
            "17830  RPN2_C               PF18004.6          79     1.80    160   0.59   0.60   0.51   0.06\n",
            "17831  RPN2_N               PF21505.2          36     1.23    296   0.59   0.61   0.52   0.03\n",
            "17832  Rpn3_C               PF08375.16         59     2.60     66   0.85   0.81   0.78   0.08\n",
            "17833  RPN5_C               PF18098.6          36     6.29     33   1.64   1.59   1.63   0.23\n",
            "17834  RPN6_C_helix         PF18503.6          73    43.45     27   1.98   2.05   1.99   0.16\n",
            "17835  RPN6_N               PF18055.6         112     1.71    117   0.59   0.61   0.54   0.05\n",
            "17836  RPN7                 PF10602.14         53     4.40    174   0.59   0.59   0.55   0.04\n",
            "17837  RPN7_PSMD6_C         PF21154.2          65    20.90     23   2.31   2.36   2.28   0.32\n",
            "17838  Rpn9_C               PF18261.6          35     5.71     33   1.64   1.58   1.63   0.20\n",
            "17839  rpo132               PF12415.13          8     2.69     32   1.69   1.69   1.67   0.10\n",
            "17840  rpo30_N              PF12410.13          9     0.63    135   0.59   0.56   0.52   0.05\n",
            "17841  RPOC_hybrid          PF21668.2          18     1.15    102   0.59   0.65   0.52   0.03\n",
            "17842  RPOL_N               PF14700.11        165     4.59    334   0.59   0.58   0.50   0.03\n",
            "17843  RpoY                 PF07288.16         33     1.53     68   0.83   0.81   0.78   0.09\n",
            "17844  Rpp20                PF12328.13         37     2.68    118   0.59   0.61   0.49   0.05\n",
            "17845  Rpr2                 PF04032.21        255     4.85     90   0.63   0.55   0.51   0.09\n",
            "17846  RPS31                PF17067.10          1     0.40     99   0.59   0.60   0.52   0.03\n",
            "17847  RPT                  PF13446.11        155    13.66     59   0.95   0.98   0.87   0.06\n",
            "17848  RPW8                 PF05659.16         13     1.92    139   0.59   0.61   0.54   0.04\n",
            "17849  RQC                  PF09382.15         91     5.60    114   0.59   0.60   0.54   0.03\n",
            "17850  RraA-like            PF03737.20        297     3.97    152   0.59   0.61   0.47   0.03\n",
            "17851  RraB                 PF06877.16         67     6.45    102   0.59   0.57   0.54   0.03\n",
            "17852  RRF                  PF01765.24        695     3.61    164   0.59   0.62   0.53   0.05\n",
            "17853  Rrf2                 PF02082.25         95     4.47    132   0.59   0.63   0.55   0.02\n",
            "17854  RRF_GI               PF12614.13          8     0.68    126   0.59   0.59   0.52   0.04\n",
            "17855  RRG7                 PF10356.14         10     1.62    185   0.59   0.58   0.50   0.01\n",
            "17856  RRG8                 PF17068.10          4     1.01    277   0.59   0.60   0.52   0.03\n",
            "17857  RRM                  PF10378.14         13     2.61     52   1.07   1.04   1.02   0.12\n",
            "17858  RRM_1                PF00076.27         68    11.48     70   0.80   0.82   0.76   0.02\n",
            "17859  RRM_10               PF22976.1           1     0.39    105   0.59   0.58   0.52   0.02\n",
            "17860  RRM_2                PF04059.17          4     0.51     97   0.59   0.56   0.52   0.02\n",
            "17861  RRM_3                PF08777.16         10     1.51    102   0.59   0.60   0.51   0.04\n",
            "17862  RRM_4                PF10598.14         34     0.67     92   0.62   0.62   0.56   0.03\n",
            "17863  RRM_5                PF13893.11          6     1.08    125   0.59   0.58   0.52   0.02\n",
            "17864  RRM_7                PF16367.10         10     0.78     94   0.61   0.58   0.52   0.01\n",
            "17865  RRM_8                PF11835.13         10     0.81     89   0.64   0.63   0.58   0.04\n",
            "17866  RRM_9                PF18444.6          30     2.83     70   0.80   0.82   0.77   0.03\n",
            "17867  RRM_DME              PF15628.11         11     0.66    102   0.59   0.56   0.53   0.01\n",
            "17868  RRM_occluded         PF16842.10         42     3.02     79   0.72   0.76   0.67   0.03\n",
            "17869  RRM_Rrp7             PF17799.6           8     1.18    161   0.59   0.57   0.50   0.03\n",
            "17870  Rrn11                PF04090.17         10     1.19    192   0.59   0.55   0.51   0.03\n",
            "17871  RRN3                 PF05327.16         97     2.65    558   0.59   0.57   0.50   0.03\n",
            "17872  Rrn6_beta-prop       PF10214.14         44     4.41    375   0.59   0.57   0.51   0.02\n",
            "17873  Rrn6_HB              PF20640.2          45    12.46    192   0.59   0.60   0.43   0.03\n",
            "17874  Rrn6_K-rich          PF20639.2          44     5.28    131   0.59   0.57   0.40   0.11\n",
            "17875  Rrn7_cyclin_C        PF20645.2          25     4.40    163   0.59   0.56   0.49   0.03\n",
            "17876  Rrn7_cyclin_N        PF20644.2          18     2.87    126   0.59   0.58   0.49   0.03\n",
            "17877  RRN9                 PF10680.14         42     2.56     68   0.83   0.76   0.73   0.07\n",
            "17878  RrnaAD               PF00398.25         11     2.86    263   0.59   0.59   0.51   0.02\n",
            "17879  rRNA_methylase       PF06962.17        142     1.64    137   0.59   0.59   0.54   0.02\n",
            "17880  rRNA_processing      PF08524.16         26     2.07    144   0.59   0.58   0.51   0.15\n",
            "17881  RRP12_HEAT           PF08161.17        106     2.91    290   0.59   0.60   0.52   0.03\n",
            "17882  RRP14                PF15459.11         48     5.12     62   0.90   0.90   0.82   0.11\n",
            "17883  Rrp15p               PF07890.17        131     3.31    140   0.59   0.62   0.47   0.09\n",
            "17884  RRP36                PF06102.17        151     3.23    165   0.59   0.58   0.51   0.12\n",
            "17885  Rrp40_N              PF18311.6         263    15.52     40   1.37   1.40   1.24   0.07\n",
            "17886  RRP40_N_mamm         PF21261.2           1     0.47     75   0.76   0.79   0.69   0.05\n",
            "17887  RRP40_S1             PF21262.2         210     1.86     83   0.68   0.68   0.64   0.02\n",
            "17888  Rrp44_CSD1           PF17216.8           1     0.39    148   0.59   0.59   0.52   0.01\n",
            "17889  Rrp44_S1             PF17215.8          59     1.34     89   0.64   0.65   0.57   0.04\n",
            "17890  RRP4_S1              PF21266.2         155     2.30     73   0.77   0.77   0.74   0.04\n",
            "17891  RRP7                 PF12923.12         38     2.57    120   0.59   0.56   0.49   0.11\n",
            "17892  RRS1                 PF04939.17        135     2.15    165   0.59   0.57   0.50   0.06\n",
            "17893  RRT14                PF17075.10         11     1.52    196   0.59   0.62   0.51   0.07\n",
            "17894  RRXRR                PF14239.11         48     1.72    174   0.59   0.59   0.52   0.04\n",
            "17895  RR_TM4-6             PF06459.17         37     1.86    277   0.59   0.58   0.49   0.03\n",
            "17896  RS4NT                PF08071.17         40     6.02     37   1.47   1.38   1.45   0.19\n",
            "17897  Rsa3                 PF14615.11         62     6.54     46   1.20   1.22   1.18   0.08\n",
            "17898  RsaA_NTD             PF19198.5           7     0.78    185   0.59   0.60   0.51   0.03\n",
            "17899  RsbRD_N              PF14361.11         93     6.52    104   0.59   0.62   0.50   0.05\n",
            "17900  Rsbr_N               PF08678.15         12     1.45    130   0.59   0.56   0.54   0.04\n",
            "17901  RsbU_N               PF08673.15         22     2.48     75   0.75   0.77   0.71   0.05\n",
            "17902  RSB_motif            PF16294.10         19     1.71    102   0.59   0.58   0.51   0.10\n",
            "17903  Rsc14                PF08586.15         12     0.86     99   0.59   0.59   0.50   0.03\n",
            "17904  RSC4_Ig_like         PF22994.1         120     3.56    150   0.59   0.59   0.53   0.03\n",
            "17905  RSD-2                PF07547.18         17     2.17     84   0.68   0.63   0.60   0.05\n",
            "17906  RsdA_SigD_bd         PF16751.10         14     3.65     46   1.20   1.26   1.18   0.13\n",
            "17907  Rsd_AlgQ             PF04353.18         40     1.45    150   0.59   0.58   0.53   0.04\n",
            "17908  RseA_C               PF03873.18         29     5.38     55   1.01   1.01   0.92   0.08\n",
            "17909  RseA_N               PF03872.18         25     2.15     87   0.65   0.65   0.58   0.05\n",
            "17910  RseC_MucC            PF04246.17        148     4.08    130   0.59   0.58   0.51   0.04\n",
            "17911  RSF                  PF14876.11          7     1.49    217   0.59   0.54   0.52   0.05\n",
            "17912  RsfS                 PF02410.20        698     3.00     98   0.59   0.56   0.51   0.04\n",
            "17913  RsgA_GTPase          PF03193.21         59     1.81    174   0.59   0.62   0.53   0.02\n",
            "17914  RsgA_N               PF16745.10         49     3.10     54   1.03   1.03   0.98   0.06\n",
            "17915  RsgI_N               PF12791.12         41    14.25     53   1.05   1.07   0.96   0.05\n",
            "17916  RsiG                 PF22802.1         170     4.43     65   0.86   0.95   0.81   0.11\n",
            "17917  RskA_C               PF10099.14        150     8.76    135   0.59   0.64   0.46   0.05\n",
            "17918  RskA_N               PF22618.1         116     7.19     48   1.15   1.25   1.14   0.15\n",
            "17919  Rsm1                 PF08600.15         10     1.18     97   0.59   0.53   0.51   0.03\n",
            "17920  Rsm22                PF09243.15          5     0.97    275   0.59   0.56   0.51   0.02\n",
            "17921  RsmF-B_ferredox      PF22458.1         319     4.70     74   0.76   0.72   0.74   0.04\n",
            "17922  RsmF_methylt_CI      PF17126.10         82     4.52     61   0.92   0.94   0.86   0.02\n",
            "17923  RsmI_C               PF23016.1         379    24.34     46   1.20   1.35   1.19   0.16\n",
            "17924  RsmJ                 PF04378.18          5     0.78    245   0.59   0.59   0.53   0.01\n",
            "17925  RSN1_7TM             PF02714.20         75     3.95    274   0.59   0.57   0.55   0.07\n",
            "17926  RSN1_TM              PF13967.11        126     4.23    156   0.59   0.56   0.52   0.05\n",
            "17927  RSRP                 PF17069.10          3     0.65    299   0.59   0.58   0.48   0.08\n",
            "17928  RSS_P20              PF11757.13         11     4.01     94   0.61   0.60   0.56   0.06\n",
            "17929  RST                  PF12174.13         44     3.89     67   0.84   0.84   0.81   0.05\n",
            "17930  RSV_NS2              PF03113.19          1     0.38    124   0.59   0.56   0.51   0.03\n",
            "17931  RS_preATP-grasp-like PF22660.1          11     1.01     98   0.59   0.61   0.53   0.02\n",
            "17932  RTA1                 PF04479.18        235     3.02    208   0.59   0.58   0.51   0.06\n",
            "17933  RTBV_P12             PF06361.16          1     0.41    110   0.59   0.60   0.52   0.02\n",
            "17934  RTBV_P46             PF06216.16          1     0.37    389   0.59   0.56   0.51   0.01\n",
            "17935  RTC                  PF01137.26        146     2.84    221   0.59   0.61   0.48   0.01\n",
            "17936  RTC4                 PF14474.11         95     2.99    121   0.59   0.59   0.52   0.02\n",
            "17937  RtcB                 PF01139.22        226     2.07    426   0.59   0.58   0.50   0.02\n",
            "17938  RtcR                 PF06956.16         27     0.79    183   0.59   0.57   0.53   0.02\n",
            "17939  RTC_insert           PF05189.18        221     6.41    102   0.59   0.63   0.51   0.03\n",
            "17940  RteC                 PF09357.15         40     2.22    219   0.59   0.55   0.55   0.05\n",
            "17941  Rtf2                 PF04641.17         17     1.76    258   0.59   0.57   0.51   0.05\n",
            "17942  RTP                  PF02334.21          9     0.68    113   0.59   0.58   0.52   0.05\n",
            "17943  RTP1_C1              PF10363.14        112     2.86    112   0.59   0.62   0.53   0.03\n",
            "17944  RTP1_C2              PF10304.14         57    57.00     34   1.54   1.60   1.54   0.17\n",
            "17945  RTP801_C             PF07809.16         36     1.53    115   0.59   0.60   0.52   0.02\n",
            "17946  Rtt102p              PF09510.15          8     1.11    130   0.59   0.55   0.49   0.05\n",
            "17947  Rtt106_N             PF18215.6          18     4.35     45   1.22   1.25   1.18   0.07\n",
            "17948  RTT107_BRCT_5        PF16770.10         62     2.58    100   0.59   0.59   0.51   0.03\n",
            "17949  RTT107_BRCT_6        PF16771.10         20     1.32    114   0.59   0.56   0.51   0.05\n",
            "17950  RTTN_N               PF14726.11         25     2.06     97   0.59   0.62   0.54   0.05\n",
            "17951  Rttp106-like_middle  PF08512.17        180     4.29     91   0.63   0.61   0.54   0.03\n",
            "17952  RTX                  PF02382.20          5     1.06    311   0.59   0.63   0.52   0.03\n",
            "17953  RtxA                 PF07634.16         71    71.00     18   1.66   1.72   1.66   0.34\n",
            "17954  RtxA_C               PF21735.2          12    12.00     18   1.94   1.93   1.94   0.34\n",
            "17955  RTX_C                PF08339.15          4     0.76    132   0.59   0.60   0.50   0.04\n",
            "17956  RT_RNaseH            PF17917.6         107     4.02    105   0.59   0.56   0.53   0.03\n",
            "17957  RT_RNaseH_2          PF17919.6         143     4.87    100   0.59   0.57   0.56   0.02\n",
            "17958  Rua1_C               PF14616.11         32     1.82    122   0.59   0.48   0.44   0.06\n",
            "17959  RUBC_PIKBD           PF21054.2          27     2.38    212   0.59   0.58   0.51   0.05\n",
            "17960  Rubella_Capsid       PF05750.16          1     0.38    300   0.58   0.58   0.51   0.02\n",
            "17961  Rubella_E1           PF05748.16          1     0.36    496   0.59   0.55   0.51   0.02\n",
            "17962  Rubella_E2           PF05749.16          1     0.36    267   0.59   0.54   0.52   0.03\n",
            "17963  Rubis-subs-bind      PF09273.16        111     8.28    128   0.59   0.62   0.46   0.04\n",
            "17964  RuBisCO_activase_AAA_helical PF21228.2          44     0.92     96   0.60   0.60   0.54   0.03\n",
            "17965  RuBisCo_chap_C       PF18087.6          71     2.03    138   0.59   0.60   0.50   0.03\n",
            "17966  RuBisCO_large        PF00016.25         14     1.06    299   0.59   0.58   0.51   0.01\n",
            "17967  RuBisCO_large_N      PF02788.21         33     1.33    120   0.59   0.58   0.53   0.02\n",
            "17968  RuBisCO_small        PF00101.25         71     1.31     97   0.59   0.52   0.53   0.03\n",
            "17969  Rubi_NSP_C           PF12601.13          1     0.55     66   0.85   0.87   0.79   0.04\n",
            "17970  Rubredoxin           PF00301.25         22     1.67     47   1.17   0.97   1.14   0.15\n",
            "17971  Rubredoxin_2         PF18073.6          28     9.57     28   1.92   1.51   1.91   0.44\n",
            "17972  Rubredoxin_C         PF18267.6          30     3.34     70   0.80   0.83   0.76   0.02\n",
            "17973  Rubrerythrin         PF02915.22         38     4.48    137   0.59   0.58   0.55   0.07\n",
            "17974  RUBY_RBDX            PF21349.2         526     9.89     30   1.80   1.47   1.76   0.36\n",
            "17975  RUN                  PF02759.24         34     3.97    131   0.59   0.58   0.49   0.03\n",
            "17976  Runt                 PF00853.24         17     0.74    129   0.59   0.59   0.53   0.02\n",
            "17977  RunxI                PF08504.16          2     0.46     93   0.61   0.59   0.54   0.03\n",
            "17978  RusA                 PF05866.16         47     4.04    124   0.59   0.57   0.47   0.04\n",
            "17979  RuvA_C               PF07499.18         74    23.61     47   1.17   1.33   1.13   0.11\n",
            "17980  RuvA_N               PF01330.26        638     7.94     61   0.92   0.92   0.83   0.05\n",
            "17981  RuvB_C               PF05491.18         27     2.57     72   0.78   0.82   0.75   0.03\n",
            "17982  RuvB_N               PF05496.17         91     0.66    159   0.59   0.62   0.53   0.02\n",
            "17983  RuvC                 PF02075.22          5     0.83    148   0.59   0.62   0.53   0.03\n",
            "17984  RuvC_1               PF18516.6          10     0.90    244   0.59   0.58   0.50   0.04\n",
            "17985  RuvC_III             PF18541.6          43     2.64    179   0.59   0.56   0.48   0.05\n",
            "17986  RuvX                 PF03652.20        100     2.53    134   0.59   0.62   0.54   0.02\n",
            "17987  Rv0078B              PF18993.5           8     2.21     63   0.89   0.88   0.85   0.05\n",
            "17988  Rv2175c_C            PF18367.6          35     2.13     56   0.99   1.03   0.95   0.02\n",
            "17989  Rv2175c_wHTH         PF21531.2          87     4.05     57   0.98   1.05   0.95   0.05\n",
            "17990  Rv2179c-like         PF16473.10         25     2.53    177   0.59   0.55   0.48   0.03\n",
            "17991  Rv2466c-like         PF22234.1          16     0.66    180   0.59   0.58   0.52   0.01\n",
            "17992  Rv2525c_GlyHyd-like  PF08924.16         77     4.26    198   0.59   0.55   0.44   0.03\n",
            "17993  Rv2632c-like         PF08962.16         32     1.76     82   0.69   0.73   0.64   0.05\n",
            "17994  Rv2949c-like         PF01947.21         20     1.36    149   0.59   0.57   0.53   0.02\n",
            "17995  Rv2993c-like_N       PF10370.14        154    16.29     50   1.11   1.13   1.00   0.03\n",
            "17996  Rv3651-like_C        PF21043.2           7     0.64    118   0.59   0.58   0.53   0.02\n",
            "17997  Rv3651-like_middle   PF18621.6           8     0.65    110   0.59   0.59   0.53   0.02\n",
            "17998  Rv3651-like_N        PF18007.6          11     1.62     93   0.61   0.60   0.53   0.04\n",
            "17999  rve                  PF00665.31         65     8.45    102   0.59   0.54   0.52   0.02\n",
            "18000  rve_2                PF13333.11         13     2.73     56   0.99   0.91   0.92   0.08\n",
            "18001  rve_3                PF13683.11         43     4.72     67   0.84   0.77   0.81   0.04\n",
            "18002  RVP                  PF00077.25          8     1.83    101   0.59   0.59   0.51   0.01\n",
            "18003  RVP_2                PF08284.16          6     0.96    136   0.59   0.59   0.53   0.03\n",
            "18004  RVT_1                PF00078.32         69     5.93    205   0.59   0.57   0.44   0.02\n",
            "18005  RVT_2                PF07727.19          3     0.69    243   0.59   0.58   0.52   0.02\n",
            "18006  RVT_3                PF13456.11         24     4.29    123   0.59   0.59   0.55   0.03\n",
            "18007  RVT_connect          PF06815.18          2     0.43    102   0.59   0.51   0.51   0.05\n",
            "18008  RVT_N                PF13655.11         78     1.58     83   0.68   0.67   0.64   0.07\n",
            "18009  RVT_thumb            PF06817.19         14     2.15     66   0.85   0.86   0.81   0.05\n",
            "18010  RWD                  PF05773.27        114     8.12    112   0.59   0.58   0.48   0.04\n",
            "18011  RWP-RK               PF02042.20         75     4.75     49   1.13   1.09   1.10   0.06\n",
            "18012  RXLR                 PF16810.10         18     3.25    141   0.59   0.58   0.48   0.05\n",
            "18013  RXLR_WY              PF18634.6          46     6.16     51   1.09   1.11   1.06   0.07\n",
            "18014  RXT2_N               PF08595.16         58     2.23    144   0.59   0.60   0.50   0.04\n",
            "18015  Rxt3                 PF08642.15         70     1.74    127   0.59   0.56   0.50   0.02\n",
            "18016  Rx_N                 PF18052.6         419    11.26     93   0.61   0.63   0.54   0.06\n",
            "18017  RYDR_ITPR            PF01365.26         44     2.51    201   0.59   0.59   0.49   0.04\n",
            "18018  RYDR_Jsol            PF21119.2          22     1.17    415   0.59   0.58   0.52   0.03\n",
            "18019  RyR                  PF02026.21         87     2.14     91   0.63   0.58   0.59   0.03\n",
            "18020  Rz1                  PF06085.16          6     2.26     41   1.34   1.21   1.25   0.18\n",
            "18021  R_equi_Vir           PF05526.16          8     1.17    177   0.59   0.60   0.52   0.02\n",
            "18022  S-AdoMet_synt_C      PF02773.21        262     1.03    138   0.59   0.60   0.52   0.02\n",
            "18023  S-AdoMet_synt_M      PF02772.21        585     1.66    120   0.59   0.58   0.48   0.02\n",
            "18024  S-AdoMet_synt_N      PF00438.25        348     1.25     99   0.59   0.59   0.53   0.04\n",
            "18025  S-antigen            PF05756.16          1     0.41     92   0.63   0.62   0.55   0.05\n",
            "18026  S-layer              PF07752.16         38     1.65    259   0.59   0.58   0.51   0.04\n",
            "18027  S-l_SbsC_C           PF18316.6          18     4.24     86   0.66   0.69   0.57   0.06\n",
            "18028  S-methyl_trans       PF02574.21        596     4.69    283   0.59   0.58   0.39   0.02\n",
            "18029  S1                   PF00575.28         31    10.19     74   0.76   0.81   0.70   0.06\n",
            "18030  S1-like              PF14444.11         11     1.23     58   0.96   0.94   0.91   0.04\n",
            "18031  S1-P1_nuclease       PF02265.21         43     2.32    258   0.59   0.54   0.48   0.03\n",
            "18032  S100PBPR             PF15427.11         12     0.76    390   0.59   0.59   0.52   0.03\n",
            "18033  S10_plectin          PF03501.20         67     1.48     93   0.61   0.58   0.56   0.03\n",
            "18034  S1CSD-TOTE-1         PF22708.1          26     6.79     65   0.86   0.85   0.84   0.04\n",
            "18035  S1CSD-TOTE-2         PF22707.1          45     5.97     60   0.93   0.91   0.87   0.04\n",
            "18036  S1FA                 PF04689.18          6     0.73     66   0.85   0.89   0.80   0.05\n",
            "18037  S1_2                 PF13509.11         46     6.08     61   0.92   0.95   0.89   0.04\n",
            "18038  S4                   PF01479.30        116    50.44     48   1.15   1.20   1.13   0.07\n",
            "18039  s48_45               PF07422.18         66     3.46    110   0.59   0.49   0.42   0.08\n",
            "18040  S4_2                 PF13275.11         93     4.10     65   0.86   0.92   0.83   0.06\n",
            "18041  S6OS1                PF15676.10          7     0.91    557   0.59   0.57   0.53   0.04\n",
            "18042  S6PP                 PF05116.18         17     1.70    247   0.59   0.59   0.53   0.02\n",
            "18043  S6PP_C               PF08472.15         23     1.23    133   0.59   0.55   0.53   0.02\n",
            "18044  S8_pro-domain        PF16470.10        107     4.79     79   0.72   0.68   0.63   0.06\n",
            "18045  SAA                  PF00277.23         21     0.92    102   0.59   0.56   0.52   0.05\n",
            "18046  SAB                  PF04382.18         14     3.39     50   1.11   1.13   1.05   0.12\n",
            "18047  SabA_adhesion        PF18304.6           4     0.96    299   0.59   0.58   0.51   0.04\n",
            "18048  SAC3                 PF12209.13         25     3.95     77   0.73   0.74   0.70   0.07\n",
            "18049  SAC3_GANP            PF03399.21        114     2.86    289   0.59   0.56   0.52   0.04\n",
            "18050  Saccharop_dh_N       PF04455.17         70     1.45    100   0.59   0.61   0.54   0.02\n",
            "18051  Sacchrp_dh_C         PF16653.10        304     6.79    265   0.59   0.58   0.33   0.01\n",
            "18052  Sacchrp_dh_NADP      PF03435.23         79     6.21    129   0.59   0.61   0.50   0.02\n",
            "18053  Sad1_UNC             PF07738.18         23     2.80    132   0.59   0.55   0.50   0.02\n",
            "18054  SAD_SRA              PF02182.22        153     2.49    148   0.59   0.57   0.46   0.03\n",
            "18055  SAE2                 PF08573.15         64     2.22    110   0.59   0.51   0.48   0.06\n",
            "18056  SAF                  PF08666.17        114    46.08     63   0.89   0.96   0.82   0.04\n",
            "18057  Saf-Nte_pilin        PF09460.15          1     0.40    144   0.59   0.61   0.52   0.02\n",
            "18058  Saf4_Yju2            PF04502.18         74     2.58    332   0.59   0.56   0.46   0.06\n",
            "18059  SafA                 PF17073.10          2     0.64     66   0.85   0.88   0.78   0.07\n",
            "18060  Saf_2TM              PF18303.6           5     1.76    152   0.59   0.62   0.52   0.03\n",
            "18061  SAG                  PF04092.18         89     3.81    131   0.59   0.54   0.35   0.08\n",
            "18062  SAGA-Tad1            PF12767.12        113     4.42    238   0.59   0.60   0.44   0.03\n",
            "18063  SAICAR_synt          PF01259.23        516     2.22    233   0.59   0.59   0.50   0.02\n",
            "18064  Salp15               PF12115.13         44     3.31    111   0.59   0.49   0.44   0.08\n",
            "18065  Salt_tol_Pase        PF09506.15         13     0.76    388   0.59   0.60   0.52   0.01\n",
            "18066  Salyut               PF19227.5           7     5.29     51   1.09   1.02   1.05   0.11\n",
            "18067  SAM35                PF10806.13         20     1.40    125   0.59   0.58   0.51   0.03\n",
            "18068  Sam68-YY             PF16568.10         25     1.83     55   1.01   0.90   0.95   0.34\n",
            "18069  SAMD1_WH             PF21524.2          67     2.62     63   0.89   0.88   0.86   0.07\n",
            "18070  SAMP                 PF05924.16         15    15.00     22   2.09   2.10   2.08   0.27\n",
            "18071  SAMP2                PF21965.1         100     2.90     63   0.89   0.94   0.84   0.08\n",
            "18072  SAM_1                PF00536.36         50    11.17     64   0.88   0.85   0.84   0.04\n",
            "18073  SAM_2                PF07647.23         43    16.99     66   0.85   0.84   0.81   0.03\n",
            "18074  SAM_3                PF18016.7          17     3.76     65   0.86   0.87   0.83   0.03\n",
            "18075  SAM_4                PF18017.7           2     0.51     84   0.68   0.67   0.61   0.03\n",
            "18076  SAM_5                PF21872.2           1     0.68     55   1.01   1.02   0.95   0.04\n",
            "18077  SAM_decarbox         PF01536.21         89     1.57    343   0.59   0.56   0.49   0.03\n",
            "18078  SAM_DrpA             PF18255.7           8     1.40     62   0.90   0.91   0.86   0.06\n",
            "18079  SAM_Exu              PF18609.7          15     1.61     72   0.78   0.80   0.74   0.03\n",
            "18080  SAM_HAT_C            PF20257.3         252     6.42     84   0.68   0.68   0.59   0.02\n",
            "18081  SAM_HAT_N            PF01887.21        252     2.79    145   0.59   0.60   0.52   0.02\n",
            "18082  SAM_KSR1             PF13543.12          5     1.15     75   0.75   0.76   0.68   0.04\n",
            "18083  SAM_KSR1_N           PF20406.3           5     1.48     54   1.03   1.05   0.99   0.10\n",
            "18084  SAM_LFY              PF01698.22          5     0.70     80   0.71   0.73   0.65   0.03\n",
            "18085  SAM_MT               PF04445.18         29     1.06    230   0.59   0.60   0.53   0.02\n",
            "18086  SAM_PNT              PF02198.22         13     2.16     83   0.68   0.61   0.63   0.04\n",
            "18087  SAM_Ste50p           PF09235.16          3     0.88     75   0.75   0.75   0.69   0.06\n",
            "18088  SANBR_BTB            PF11822.13          7     0.77     97   0.59   0.55   0.51   0.04\n",
            "18089  SAND                 PF01342.26         53     2.47     76   0.74   0.68   0.70   0.04\n",
            "18090  SANTA                PF09133.15         26     1.84     92   0.62   0.58   0.54   0.03\n",
            "18091  SANT_DAMP1_like      PF16282.10         97     1.57     80   0.71   0.62   0.63   0.08\n",
            "18092  SAP                  PF02037.32        141   141.00     35   1.42   1.52   1.42   0.12\n",
            "18093  SAP130_C             PF16014.10          6     0.55    406   0.59   0.59   0.51   0.02\n",
            "18094  SAP18                PF06487.17        105     2.39    124   0.59   0.57   0.47   0.02\n",
            "18095  Sap1_N               PF21438.2           1     0.39    102   0.59   0.58   0.52   0.03\n",
            "18096  SAP25                PF15476.11          9     0.65    202   0.59   0.57   0.52   0.02\n",
            "18097  SAP30_Sin3_bdg       PF13867.11         45     5.18     53   1.05   1.02   0.99   0.08\n",
            "18098  SapA                 PF02199.20         76     6.14     33   1.64   1.31   1.63   0.35\n",
            "18099  SapB_1               PF05184.20        381   381.00     38   1.38   1.29   1.38   0.19\n",
            "18100  SapB_2               PF03489.22        427   427.00     34   1.53   1.39   1.53   0.20\n",
            "18101  SapC                 PF07277.16         93     3.64    220   0.59   0.60   0.53   0.02\n",
            "18102  SAPI                 PF16560.10          1     0.39    213   0.59   0.58   0.52   0.02\n",
            "18103  SAPIS-gp6            PF16722.10          3     0.98     72   0.78   0.79   0.72   0.10\n",
            "18104  Saposin              PF20825.2           7     0.54    131   0.59   0.55   0.52   0.04\n",
            "18105  SAPS                 PF04499.20         49     3.34    451   0.59   0.59   0.48   0.03\n",
            "18106  SAP_new25            PF18953.5         102     7.19     51   1.09   1.05   1.07   0.07\n",
            "18107  Sar8_2               PF03058.19          4     0.69     85   0.67   0.63   0.57   0.08\n",
            "18108  SARA                 PF11409.13          6     1.66     39   1.40   1.32   1.35   0.25\n",
            "18109  SARAF                PF06682.17         43     1.67    333   0.59   0.55   0.47   0.03\n",
            "18110  SARA_C               PF11979.13         24     1.17    353   0.59   0.58   0.53   0.02\n",
            "18111  Sarcoglycan_1        PF04790.18         45     3.16    259   0.59   0.61   0.49   0.02\n",
            "18112  Sarcoglycan_2        PF05510.18          5     0.67     95   0.60   0.56   0.54   0.02\n",
            "18113  Sarcoglycan_2_C      PF20989.2           5     0.59    125   0.59   0.57   0.50   0.02\n",
            "18114  Sarcolamban          PF21898.1           6     2.47     28   1.92   1.98   1.89   0.37\n",
            "18115  Sarcolipin           PF05366.16          3     1.61     31   1.74   1.70   1.71   0.26\n",
            "18116  SARG                 PF15385.11         19     1.16    572   0.59   0.61   0.51   0.05\n",
            "18117  SARS_3b              PF12383.13          1     0.42    153   0.59   0.58   0.52   0.06\n",
            "18118  SART-1               PF03343.18         97     3.05    585   0.59   0.60   0.43   0.09\n",
            "18119  Sas-5-like_implico   PF21518.2           3     1.20     44   1.25   1.31   1.21   0.08\n",
            "18120  Sas-6-like_oligomerization PF22331.1           2     0.69     66   0.85   0.85   0.79   0.11\n",
            "18121  Sas-6_C_CC           PF22216.1          12     1.63     57   0.98   0.98   0.94   0.10\n",
            "18122  SAS-6_C_CC_2         PF22218.1           4     1.52     52   1.07   1.07   0.97   0.12\n",
            "18123  Sas-6_helical        PF21503.2           3     1.51     37   1.47   1.46   1.44   0.13\n",
            "18124  SAS-6_N              PF16531.10         75     4.92    135   0.59   0.56   0.51   0.06\n",
            "18125  Sas10                PF09368.15         88     3.07     74   0.76   0.76   0.72   0.13\n",
            "18126  Sas10_Utp3           PF04000.20        295    12.37     87   0.65   0.69   0.47   0.05\n",
            "18127  SAS4                 PF15460.11         55     2.36     99   0.59   0.57   0.55   0.08\n",
            "18128  Sas6_CC              PF18594.6          17     5.50     30   1.80   1.84   1.78   0.30\n",
            "18129  SASA                 PF03629.23         68     1.84    226   0.59   0.57   0.48   0.02\n",
            "18130  SasG_E               PF17041.10          4     1.00     48   1.15   1.14   1.10   0.13\n",
            "18131  SASP                 PF00269.25        135     3.53     59   0.95   0.97   0.87   0.07\n",
            "18132  SASP_gamma           PF04259.19          3     0.84     84   0.68   0.69   0.60   0.13\n",
            "18133  SASRP1               PF15160.11          6     0.52    238   0.59   0.57   0.52   0.01\n",
            "18134  SAT                  PF16073.10         97     4.20    239   0.59   0.60   0.53   0.01\n",
            "18135  SATase_N             PF06426.19        112     2.07    105   0.59   0.60   0.55   0.03\n",
            "18136  SatD                 PF16264.10          4     0.60    211   0.59   0.59   0.53   0.02\n",
            "18137  SatRNA_48            PF17485.7           4     0.45    299   0.59   0.57   0.52   0.02\n",
            "18138  SAUGI                PF06106.16          2     0.50    115   0.59   0.57   0.52   0.03\n",
            "18139  SAV0927-like         PF11256.13         21     1.32     80   0.71   0.70   0.66   0.04\n",
            "18140  SAVED                PF18145.6          76     8.16    190   0.59   0.58   0.50   0.01\n",
            "18141  Saw1                 PF11561.13         16     1.69    247   0.59   0.59   0.50   0.04\n",
            "18142  SAWADEE              PF16719.10         31     1.39    125   0.59   0.52   0.51   0.05\n",
            "18143  SAXO1-2              PF05217.17          1     0.37    474   0.59   0.54   0.51   0.02\n",
            "18144  SAXO5-like           PF15373.11          8     0.86    438   0.59   0.55   0.53   0.02\n",
            "18145  Say1_Mug180          PF10340.14          5     1.05    374   0.59   0.57   0.51   0.02\n",
            "18146  SAYSvFN              PF10260.14         40     2.71     70   0.80   0.78   0.76   0.05\n",
            "18147  Sa_NUDIX             PF18167.6          14     1.52    199   0.59   0.58   0.53   0.03\n",
            "18148  SBBP                 PF06739.16         52    51.80     38   1.44   1.41   1.40   0.16\n",
            "18149  SBB_T7               PF21265.2           9     0.73    169   0.59   0.59   0.52   0.01\n",
            "18150  SbcC_Walker_B        PF13558.11         73     4.60     90   0.63   0.66   0.57   0.03\n",
            "18151  SbcD_C               PF12320.13        179    10.69     94   0.61   0.63   0.42   0.04\n",
            "18152  SBD                  PF17882.6          43     3.34     76   0.74   0.69   0.63   0.06\n",
            "18153  SBDS                 PF01172.23        229     3.78     88   0.65   0.65   0.57   0.05\n",
            "18154  SBDS_C               PF20268.3         129    14.22     71   0.79   0.83   0.70   0.04\n",
            "18155  SBDS_domain_II       PF09377.15        173     5.51     62   0.90   0.91   0.86   0.08\n",
            "18156  SBD_N                PF07005.16        172     4.13    230   0.59   0.62   0.47   0.02\n",
            "18157  SBE2_C               PF17076.11          3     0.47    317   0.59   0.56   0.52   0.04\n",
            "18158  SBE2_M               PF22874.1           3     0.48    124   0.59   0.61   0.52   0.03\n",
            "18159  SBE2_N               PF22876.1           3     0.80    302   0.59   0.59   0.50   0.04\n",
            "18160  SBF                  PF01758.21         17     2.91    191   0.59   0.61   0.53   0.08\n",
            "18161  SBF2                 PF12335.13         21     1.02    227   0.59   0.56   0.53   0.04\n",
            "18162  SBF_like             PF13593.11         32     2.53    313   0.59   0.61   0.53   0.06\n",
            "18163  Sbi-IV               PF11621.13          1     0.61     69   0.81   0.85   0.75   0.09\n",
            "18164  SbmA_BacA            PF05992.17          8     0.67    315   0.59   0.57   0.53   0.04\n",
            "18165  SBP                  PF03110.19         70     1.79     75   0.75   0.62   0.70   0.14\n",
            "18166  SBP56                PF05694.16         18     0.78    453   0.59   0.55   0.51   0.01\n",
            "18167  SBP_bac_1            PF01547.30        129    45.91    307   0.59   0.59   0.40   0.04\n",
            "18168  SBP_bac_10           PF07596.16         33     5.88    268   0.59   0.55   0.50   0.02\n",
            "18169  SBP_bac_11           PF13531.11        101    10.51    227   0.59   0.62   0.50   0.02\n",
            "18170  SBP_bac_3            PF00497.25        230     5.98    132   0.59   0.59   0.46   0.02\n",
            "18171  SBP_bac_5            PF00496.27        148    11.54    368   0.59   0.58   0.49   0.02\n",
            "18172  SBP_bac_6            PF13343.11         42     6.77    245   0.59   0.57   0.50   0.01\n",
            "18173  SBP_bac_8            PF13416.11         46    11.37    279   0.59   0.58   0.42   0.03\n",
            "18174  SbsC_C               PF18058.6          10     2.01    131   0.59   0.60   0.54   0.07\n",
            "18175  SbsC_spectrin-like   PF22360.1           4     1.82     62   0.90   0.97   0.86   0.08\n",
            "18176  SBSN_GxHH_rpt        PF21009.2           1     0.39    384   0.60   0.59   0.52   0.08\n",
            "18177  Sbt_1                PF05982.17         73     1.50    308   0.59   0.64   0.53   0.05\n",
            "18178  Sca2                 PF22203.1           1     0.40    329   0.59   0.59   0.52   0.03\n",
            "18179  SCA7                 PF08313.17         36     2.00     68   0.83   0.76   0.79   0.09\n",
            "18180  SCAB-ABD             PF16711.10         10     1.98     43   1.27   1.38   1.23   0.13\n",
            "18181  SCAB-Ig              PF16709.10         15     0.74     98   0.59   0.60   0.53   0.01\n",
            "18182  SCAB-PH              PF17684.6          15     0.60    108   0.59   0.58   0.53   0.02\n",
            "18183  Scabin-like          PF22596.1          24     1.61    130   0.59   0.56   0.49   0.02\n",
            "18184  SCAB_CC              PF16712.10          8     0.77    168   0.59   0.63   0.53   0.07\n",
            "18185  sCache_2             PF17200.9          19     3.93    155   0.59   0.56   0.54   0.03\n",
            "18186  sCache_3_2           PF17203.9          15     4.83    140   0.59   0.62   0.51   0.02\n",
            "18187  sCache_3_3           PF17202.9         227     4.35    107   0.59   0.59   0.51   0.03\n",
            "18188  sCache_4             PF09984.14         24     1.92    145   0.59   0.61   0.53   0.03\n",
            "18189  sCache_like          PF16736.10          6     1.71    117   0.59   0.60   0.52   0.04\n",
            "18190  Scaffold             PF20142.4         282     9.34    139   0.59   0.60   0.51   0.04\n",
            "18191  Scaffolding_pro      PF11418.13          2     0.62    100   0.59   0.61   0.52   0.04\n",
            "18192  SCAI                 PF12070.13         26     1.08    529   0.59   0.57   0.50   0.02\n",
            "18193  SCAMP                PF04144.18        117     3.54    172   0.59   0.53   0.52   0.08\n",
            "18194  SCAN                 PF02023.22        151     2.70     90   0.63   0.60   0.61   0.05\n",
            "18195  SCAPER_N             PF16501.10         10     0.72     98   0.59   0.55   0.53   0.04\n",
            "18196  SCC3_C               PF21767.2          12     1.19    321   0.59   0.60   0.53   0.05\n",
            "18197  SCD                  PF21581.2         307     2.80     75   0.75   0.70   0.72   0.04\n",
            "18198  ScdA_N               PF04405.19         42     3.89     56   0.99   0.98   0.97   0.05\n",
            "18199  SCF                  PF02404.20          6     0.59    275   0.59   0.57   0.52   0.02\n",
            "18200  SCFA_trans           PF02667.19          4     0.64    453   0.59   0.60   0.52   0.04\n",
            "18201  ScfRs                PF09856.14         84     1.30    157   0.59   0.55   0.54   0.02\n",
            "18202  SCGB3A               PF20490.3          30     1.99     68   0.83   0.90   0.74   0.05\n",
            "18203  SchA_CurD            PF04486.17         12     0.96    119   0.59   0.60   0.53   0.02\n",
            "18204  SCHIP-1              PF10148.14          5     0.74    244   0.59   0.60   0.51   0.04\n",
            "18205  SCIFF                PF13165.11         34     1.55     44   1.25   1.09   1.21   0.29\n",
            "18206  SCIMP                PF15050.11         10     0.67    132   0.59   0.55   0.52   0.02\n",
            "18207  Sclerostin           PF05463.16          2     0.55    214   0.59   0.55   0.51   0.02\n",
            "18208  Scm3                 PF10384.14         67     8.79     53   1.05   1.05   0.99   0.08\n",
            "18209  SCNM1_acidic         PF15805.10         12     2.25     47   1.17   1.10   1.15   0.11\n",
            "18210  SCO1-SenC            PF02630.19         50     2.23    134   0.59   0.56   0.54   0.02\n",
            "18211  SCO4226-like         PF14026.11         69     4.73     76   0.74   0.69   0.67   0.04\n",
            "18212  SCP-1                PF05483.17          2     0.47    787   0.59   0.60   0.52   0.06\n",
            "18213  SCP2                 PF02036.22         97     7.97    101   0.59   0.63   0.49   0.03\n",
            "18214  SCP2_2               PF13530.11        140     7.18    105   0.59   0.61   0.47   0.01\n",
            "18215  ScpA_C               PF22143.1           1     0.43     91   0.62   0.62   0.55   0.03\n",
            "18216  SCPU                 PF05229.20         47     3.67    138   0.59   0.57   0.49   0.04\n",
            "18217  SCP_3                PF17844.6          85     0.98     93   0.61   0.63   0.56   0.02\n",
            "18218  Scramblase           PF03803.20         10     1.40    221   0.59   0.56   0.51   0.02\n",
            "18219  SCRE                 PF15162.11         10     0.71    162   0.59   0.59   0.53   0.03\n",
            "18220  SCRG1                PF15224.11          6     0.53     76   0.75   0.65   0.68   0.05\n",
            "18221  SCRL                 PF06876.17         43     3.41     67   0.84   0.65   0.74   0.20\n",
            "18222  ScsC_N               PF18312.6          44    21.08     34   1.59   1.67   1.59   0.20\n",
            "18223  SCVP                 PF17619.7          47     6.71     96   0.60   0.55   0.53   0.04\n",
            "18224  Scytalone_dh         PF02982.19          3     0.45    160   0.59   0.54   0.52   0.02\n",
            "18225  Sda                  PF08970.15         15     3.59     45   1.22   1.25   1.20   0.12\n",
            "18226  SDA1_C               PF21638.2         114     8.90     46   1.20   1.19   1.16   0.31\n",
            "18227  SDA1_dom             PF05285.17        101     4.31    151   0.59   0.62   0.40   0.13\n",
            "18228  SDA1_HEAT            PF08158.17         55     1.34    364   0.59   0.59   0.53   0.04\n",
            "18229  SdbC_C               PF20856.2           3     0.88     70   0.80   0.81   0.74   0.03\n",
            "18230  Sde0182_C            PF21027.2          68     2.39     83   0.68   0.67   0.58   0.03\n",
            "18231  Sde182_NH-like       PF07632.16         90     1.78    268   0.59   0.56   0.50   0.02\n",
            "18232  SDE2_2C              PF13297.11         43     2.33     82   0.69   0.74   0.64   0.05\n",
            "18233  Sde2_C               PF22782.1         192     2.52    100   0.59   0.61   0.47   0.09\n",
            "18234  Sde2_N_Ubi_vert      PF22781.1          12     0.71     75   0.75   0.69   0.69   0.04\n",
            "18235  Sde2_N_Ubi_yeast     PF13019.11         16     3.41     86   0.66   0.74   0.58   0.07\n",
            "18236  SDF                  PF00375.23         75     4.05    387   0.59   0.63   0.54   0.08\n",
            "18237  SDG2_C               PF19633.4          14     0.64    420   0.59   0.56   0.53   0.01\n",
            "18238  Sdh5                 PF03937.21        210     4.68     76   0.74   0.72   0.71   0.04\n",
            "18239  SDH5_plant           PF14290.11          8     0.65    237   0.59   0.60   0.52   0.01\n",
            "18240  SDH_alpha            PF03313.20        441     3.37    260   0.59   0.61   0.46   0.04\n",
            "18241  SDH_beta             PF03315.20        255     1.40    155   0.59   0.60   0.53   0.02\n",
            "18242  SDH_C                PF18317.6         155   121.47     31   1.74   1.71   1.74   0.15\n",
            "18243  Sdh_cyt              PF01127.27         55     7.01    122   0.59   0.58   0.49   0.07\n",
            "18244  SDH_sah              PF01972.21          4     0.54    286   0.59   0.60   0.52   0.02\n",
            "18245  SdiA-regulated       PF06977.16         16     1.50    249   0.59   0.61   0.53   0.02\n",
            "18246  SdpA                 PF17418.7           4     0.93    141   0.59   0.54   0.52   0.02\n",
            "18247  SdpI                 PF13630.11        172     9.97     71   0.79   0.77   0.68   0.08\n",
            "18248  SDP_N                PF12278.13          8     0.97    138   0.59   0.58   0.52   0.07\n",
            "18249  SDR-like             PF21777.2          78     2.12     98   0.59   0.62   0.54   0.04\n",
            "18250  SdrD_B               PF17210.8          41     1.37    112   0.59   0.59   0.52   0.05\n",
            "18251  SdrG_C_C             PF10425.14         10     2.70    156   0.59   0.56   0.50   0.07\n",
            "18252  Sds3                 PF08598.16         55     4.58    217   0.59   0.59   0.40   0.07\n",
            "18253  SduA_C               PF14082.11         26     4.61    159   0.59   0.58   0.41   0.03\n",
            "18254  SduA_N               PF21407.2           3     0.71    102   0.59   0.60   0.53   0.03\n",
            "18255  SE                   PF08491.15         12     0.92    276   0.59   0.59   0.53   0.01\n",
            "18256  Se-cys_synth_N       PF12390.13        143    46.35     40   1.37   1.48   1.36   0.16\n",
            "18257  SEA                  PF01390.25        159     9.12    106   0.59   0.58   0.53   0.05\n",
            "18258  Seadorna_Vp10        PF07322.16          1     0.39    241   0.59   0.59   0.52   0.02\n",
            "18259  Seadorna_VP6         PF07407.16          1     0.38    420   0.59   0.58   0.51   0.01\n",
            "18260  Seadorna_VP7         PF07387.16          4     0.82    308   0.59   0.58   0.51   0.03\n",
            "18261  Sec-ASP3             PF15432.11          8     1.01    126   0.59   0.55   0.54   0.03\n",
            "18262  SEC-C                PF02810.20        142   142.00     19   2.75   2.33   2.73   0.77\n",
            "18263  Sec1                 PF00995.28        567     9.45    575   0.59   0.59   0.39   0.02\n",
            "18264  Sec10_HB             PF07393.16         26     3.73    637   0.59   0.59   0.51   0.04\n",
            "18265  Sec10_N              PF20667.2          24     4.05    118   0.59   0.60   0.54   0.05\n",
            "18266  Sec15_C              PF04091.17         95     3.30    348   0.59   0.58   0.52   0.04\n",
            "18267  Sec16                PF12932.12         87     2.80    119   0.59   0.59   0.48   0.01\n",
            "18268  Sec16_C              PF12931.12         91     2.85    298   0.59   0.59   0.49   0.02\n",
            "18269  Sec16_N              PF12935.12         12     2.34    239   0.59   0.59   0.48   0.09\n",
            "18270  Sec20                PF03908.18          6     1.39     92   0.62   0.64   0.58   0.05\n",
            "18271  Sec23_BS             PF08033.17        138     5.62     86   0.66   0.63   0.56   0.04\n",
            "18272  Sec23_helical        PF04815.20        309     6.78    103   0.59   0.60   0.53   0.03\n",
            "18273  Sec23_trunk          PF04811.20         21     1.96    241   0.59   0.59   0.51   0.02\n",
            "18274  Sec2p                PF06428.16         42     1.97     92   0.62   0.65   0.59   0.12\n",
            "18275  Sec3-PIP2_bind       PF15277.11        100     4.04     87   0.65   0.62   0.56   0.05\n",
            "18276  Sec31                PF11549.13         12     1.77     50   1.11   1.10   1.06   0.09\n",
            "18277  Sec39                PF08314.16         62     3.56    753   0.59   0.59   0.48   0.03\n",
            "18278  Sec3_C-term          PF20654.2          14     2.83    355   0.59   0.57   0.53   0.04\n",
            "18279  Sec3_CC              PF09763.14         18     2.76    133   0.59   0.63   0.56   0.07\n",
            "18280  Sec3_C_2             PF15278.11          2     0.74     86   0.66   0.67   0.60   0.04\n",
            "18281  Sec5                 PF15469.11         41     5.49    837   0.59   0.59   0.49   0.04\n",
            "18282  Sec6                 PF06046.18         32     4.59    537   0.59   0.57   0.50   0.03\n",
            "18283  Sec61_beta           PF03911.21        100    18.14     41   1.34   1.42   1.32   0.12\n",
            "18284  Sec62                PF03839.21         65     1.32    191   0.59   0.54   0.52   0.03\n",
            "18285  Sec63                PF02889.21         82     3.43    252   0.59   0.58   0.49   0.03\n",
            "18286  Sec66                PF09802.14         58     1.67    180   0.59   0.58   0.54   0.03\n",
            "18287  Sec7                 PF01369.25        354     3.64    183   0.59   0.57   0.51   0.04\n",
            "18288  Sec7-like_HDS        PF09324.15         92     3.42     84   0.68   0.64   0.64   0.05\n",
            "18289  Sec7-like_HUS        PF12783.12        207     4.94    161   0.59   0.59   0.50   0.05\n",
            "18290  Sec8_C               PF20652.2         149     3.48    260   0.59   0.60   0.45   0.03\n",
            "18291  Sec8_N               PF04048.19         16     1.67    142   0.59   0.60   0.55   0.05\n",
            "18292  Secapin              PF17521.7           1     0.78     45   1.22   1.19   1.17   0.11\n",
            "18293  SecA_DEAD            PF07517.19         79     1.61    290   0.59   0.60   0.51   0.03\n",
            "18294  SecA_PP_bind         PF01043.25        469     1.69    110   0.59   0.58   0.51   0.04\n",
            "18295  SecA_SW              PF07516.18        636     2.49    200   0.59   0.57   0.49   0.05\n",
            "18296  SecB                 PF02556.19        136     1.43    141   0.59   0.58   0.54   0.02\n",
            "18297  SecD-TM1             PF13721.11        101     2.31    103   0.59   0.60   0.54   0.03\n",
            "18298  SecDF_P1_head        PF22599.1         206     2.93    114   0.59   0.64   0.50   0.03\n",
            "18299  SecD_1st             PF21760.2         270     3.58     60   0.93   0.99   0.91   0.05\n",
            "18300  SecD_SecF            PF02355.21         17     1.73    189   0.59   0.63   0.54   0.06\n",
            "18301  SecE                 PF00584.25        626    18.98     55   1.01   1.03   1.00   0.13\n",
            "18302  SecG                 PF03840.19        552     9.39     70   0.80   0.89   0.69   0.15\n",
            "18303  SecIII_SopE_N        PF05364.19          1     0.54     74   0.76   0.75   0.70   0.04\n",
            "18304  SecM                 PF06558.17         10     1.05    147   0.59   0.59   0.51   0.02\n",
            "18305  SecM_small           PF10818.13          6     1.20     95   0.60   0.59   0.51   0.04\n",
            "18306  Secretin             PF00263.26         78     3.77    169   0.59   0.63   0.53   0.03\n",
            "18307  Secretin_N           PF03958.22        227    50.08     69   0.82   0.90   0.74   0.05\n",
            "18308  Secretin_N_2         PF07655.18         53     4.25     91   0.63   0.64   0.53   0.08\n",
            "18309  Secretogranin_V      PF05281.16          7     0.60    232   0.59   0.56   0.50   0.02\n",
            "18310  Securin              PF04856.18          7     2.36    219   0.59   0.61   0.42   0.05\n",
            "18311  SecY                 PF00344.25        660     3.30    306   0.59   0.61   0.42   0.06\n",
            "18312  Sec_GG               PF07549.19        308   308.00     28   1.64   1.72   1.64   0.14\n",
            "18313  Sedlin_N             PF04628.18         23     1.82    129   0.59   0.57   0.50   0.03\n",
            "18314  SEEEED               PF14797.11         15     1.13    113   0.59   0.64   0.52   0.28\n",
            "18315  SEEK1                PF15357.11          2     0.44    149   0.59   0.58   0.52   0.02\n",
            "18316  SEF14_adhesin        PF06443.16          2     0.57    165   0.59   0.60   0.52   0.03\n",
            "18317  SEFIR                PF08357.16         15     2.76    150   0.59   0.56   0.50   0.03\n",
            "18318  SegE_GIY-YIG         PF19835.4          27     1.28    123   0.59   0.53   0.52   0.05\n",
            "18319  SEHBP                PF21978.1           1     0.77     46   1.20   1.15   1.14   0.11\n",
            "18320  Seipin               PF06775.19        113     4.16    196   0.59   0.57   0.49   0.04\n",
            "18321  Sel1                 PF08238.17        164   164.00     36   1.17   1.15   1.08   0.11\n",
            "18322  SelA                 PF03841.18          9     0.78    367   0.59   0.63   0.53   0.02\n",
            "18323  SelB-wing_1          PF09105.15          2     0.93     59   0.94   1.00   0.89   0.04\n",
            "18324  SelB-wing_1_3        PF21458.2           4     1.46     48   1.15   1.15   1.11   0.11\n",
            "18325  SelB-wing_2          PF09106.16        112    15.33     57   0.98   1.03   0.94   0.05\n",
            "18326  SelB-wing_3          PF09107.16        147    11.77     47   1.17   1.22   1.15   0.07\n",
            "18327  SelB_WH1             PF22153.1           3     1.05     57   0.98   1.04   0.93   0.04\n",
            "18328  SelB_WH2             PF22154.1           2     0.96     62   0.90   0.95   0.85   0.08\n",
            "18329  SelB_WHD3            PF21580.2           2     0.94     51   1.08   1.14   1.04   0.07\n",
            "18330  SelB_WH_2nd_bact     PF21214.2          13     1.20     59   0.95   0.89   0.90   0.06\n",
            "18331  SeleniumBinding      PF11524.13          2     0.60     82   0.69   0.72   0.63   0.04\n",
            "18332  Selenoprotein_S      PF06936.16          8     0.83    192   0.59   0.58   0.52   0.03\n",
            "18333  Self-incomp_S1       PF05938.16        184     3.30    104   0.59   0.47   0.48   0.06\n",
            "18334  SelK_SelG            PF10961.13         32     2.60     86   0.66   0.65   0.52   0.04\n",
            "18335  SelO                 PF02696.19         45     1.10    466   0.59   0.57   0.52   0.02\n",
            "18336  SelP_C               PF04593.19          3     0.45    133   0.59   0.52   0.52   0.08\n",
            "18337  SelP_N               PF04592.19          6     0.76    234   0.59   0.53   0.51   0.03\n",
            "18338  SelR                 PF01641.24        706     1.51    121   0.59   0.53   0.54   0.03\n",
            "18339  Sel_put              PF04328.18         58     3.84     61   0.92   0.78   0.86   0.09\n",
            "18340  Sema                 PF01403.24        173     4.43    182   0.59   0.57   0.48   0.02\n",
            "18341  Sema4F_C             PF19428.4           5     0.65    197   0.59   0.59   0.52   0.01\n",
            "18342  Semenogelin          PF05474.16          1     0.41    582   0.59   0.57   0.52   0.04\n",
            "18343  Semialdhyde_dh       PF01118.29        141     4.15    121   0.59   0.62   0.51   0.01\n",
            "18344  Semialdhyde_dhC      PF02774.23         32     4.00    184   0.59   0.60   0.48   0.02\n",
            "18345  Semialdhyde_dhC_1    PF22698.1         575     2.56    165   0.59   0.57   0.54   0.02\n",
            "18346  Sen15                PF09631.15         39     2.45    101   0.59   0.58   0.47   0.02\n",
            "18347  SEN1_N               PF12726.12         89     6.03    747   0.59   0.57   0.50   0.03\n",
            "18348  SEN2-like_N          PF22502.1           1     0.52     69   0.82   0.82   0.75   0.04\n",
            "18349  Senescence           PF06911.17         96     5.75    184   0.59   0.63   0.52   0.06\n",
            "18350  Senescence_reg       PF04520.18         81     2.24    167   0.59   0.59   0.42   0.04\n",
            "18351  SENP3_5_N            PF19722.4          10     0.95    578   0.59   0.56   0.52   0.03\n",
            "18352  Sensor               PF13796.11         74     4.66    162   0.59   0.64   0.48   0.11\n",
            "18353  Sensor_TM1           PF13755.11         18     2.00     68   0.83   0.91   0.79   0.08\n",
            "18354  SEO_C                PF14577.11         26     1.13    232   0.59   0.53   0.53   0.03\n",
            "18355  SEO_N                PF14576.11         24     1.57    287   0.59   0.57   0.53   0.04\n",
            "18356  SEP                  PF08059.18        147     3.25     75   0.75   0.73   0.68   0.02\n",
            "18357  Sep15_SelM           PF08806.16         31     3.64     75   0.75   0.73   0.71   0.03\n",
            "18358  SepA                 PF17080.10          1     0.36    144   0.59   0.56   0.51   0.08\n",
            "18359  SepF                 PF04472.17        271     8.03     72   0.78   0.82   0.74   0.04\n",
            "18360  SepQ                 PF06622.16          2     0.40    305   0.59   0.57   0.52   0.02\n",
            "18361  SepRS_C              PF18006.6          22     6.92     31   1.74   1.75   1.68   0.14\n",
            "18362  SepSecS              PF05889.18          5     1.07    389   0.59   0.60   0.51   0.01\n",
            "18363  Septin               PF00735.23         13     1.00    280   0.59   0.58   0.53   0.02\n",
            "18364  Septum_form          PF13845.11         54     2.32    227   0.59   0.52   0.52   0.05\n",
            "18365  SepZ                 PF06066.16          4     0.54     99   0.59   0.66   0.52   0.05\n",
            "18366  SeqA                 PF03925.18         25     1.09    111   0.59   0.57   0.54   0.03\n",
            "18367  SeqA_N               PF17206.8           6     2.01     36   1.51   1.56   1.48   0.09\n",
            "18368  Serendipity_A        PF05482.17          7     0.65    541   0.59   0.60   0.52   0.03\n",
            "18369  SERF-like_N          PF04419.19         37    24.10     38   1.44   1.45   1.37   0.39\n",
            "18370  Serglycin            PF04360.17         15     0.88    149   0.59   0.58   0.53   0.02\n",
            "18371  SerH                 PF06873.16          9     0.94    419   0.59   0.52   0.51   0.11\n",
            "18372  Serinc               PF03348.20        135     2.28    434   0.59   0.54   0.50   0.05\n",
            "18373  Serine_rich          PF08824.15         46     1.95    157   0.59   0.60   0.55   0.04\n",
            "18374  Serpentine_r_xa      PF03383.20          7     0.89    153   0.59   0.55   0.52   0.07\n",
            "18375  Serpin               PF00079.25        104     4.08    370   0.59   0.58   0.51   0.03\n",
            "18376  Serpulina_VSP        PF05540.16          2     0.56    403   0.59   0.59   0.51   0.02\n",
            "18377  SERRATE_Ars2_N       PF12066.13         44     2.84    110   0.59   0.53   0.55   0.09\n",
            "18378  SERTA                PF06031.18         25     6.41     36   1.51   1.58   1.50   0.17\n",
            "18379  Serum_albumin        PF00273.25        151     3.13    176   0.59   0.50   0.54   0.09\n",
            "18380  Seryl_tRNA_N         PF02403.27         75     4.77    108   0.59   0.63   0.55   0.08\n",
            "18381  Ser_hydrolase        PF06821.18         17     1.56    172   0.59   0.56   0.53   0.03\n",
            "18382  SesA                 PF17107.10         63     6.23    122   0.59   0.63   0.54   0.04\n",
            "18383  Ses_B                PF17046.10         20    20.00     28   1.80   1.76   1.77   0.31\n",
            "18384  SET                  PF00856.33        251     9.52    105   0.59   0.54   0.31   0.02\n",
            "18385  SET1_RBD             PF21569.2          15     0.94    110   0.59   0.57   0.53   0.04\n",
            "18386  SET7_N               PF22648.1           5     0.91     74   0.76   0.74   0.70   0.02\n",
            "18387  SET_assoc            PF11767.13         58     4.46     66   0.85   0.76   0.81   0.05\n",
            "18388  Sex_peptide          PF08138.16          2     0.57     55   1.00   0.89   0.94   0.09\n",
            "18389  Sey1_3HB             PF20428.3           8     1.40    419   0.59   0.59   0.52   0.03\n",
            "18390  Se_S_carrier         PF11823.13        165    10.48     69   0.82   0.80   0.79   0.03\n",
            "18391  SF-assemblin         PF06705.16          3     1.03    247   0.59   0.62   0.54   0.08\n",
            "18392  SF1-HH               PF16275.10         77     2.35    114   0.59   0.56   0.52   0.05\n",
            "18393  SF3A2                PF16835.10         82     1.82     96   0.60   0.55   0.48   0.03\n",
            "18394  SF3A3                PF16837.10        134     8.19     49   1.13   1.06   0.96   0.11\n",
            "18395  SF3a60_bindingd      PF12108.13         51    19.94     27   1.98   1.85   1.97   0.23\n",
            "18396  SF3a60_Prp9_C        PF11931.13         66     0.85    120   0.59   0.54   0.53   0.02\n",
            "18397  SF3b1                PF08920.15         42     1.42    130   0.59   0.56   0.43   0.04\n",
            "18398  SF3b10               PF07189.16         67     2.39     78   0.73   0.67   0.68   0.04\n",
            "18399  Sf6_terminase        PF20901.2          71     3.39    106   0.59   0.57   0.53   0.04\n",
            "18400  Sfi1                 PF08457.15         30     3.24    570   0.59   0.54   0.57   0.10\n",
            "18401  Sfi1_C               PF10638.14          7     0.96    103   0.59   0.59   0.53   0.05\n",
            "18402  SfLAP                PF11139.13         28     4.65    213   0.59   0.65   0.54   0.08\n",
            "18403  SFM1-like            PF04252.18         87     1.61    201   0.59   0.59   0.51   0.02\n",
            "18404  SfsA                 PF03749.18        136     2.76    138   0.59   0.59   0.51   0.02\n",
            "18405  SfsA_N               PF17746.6          76     5.09     63   0.89   0.88   0.77   0.03\n",
            "18406  SFTA2                PF15210.11          4     1.02     59   0.95   0.96   0.90   0.05\n",
            "18407  SFXNs                PF03820.22        127     2.07    319   0.59   0.58   0.54   0.02\n",
            "18408  SGBP_BT4661-like     PF16319.10          3     0.65    731   0.59   0.58   0.51   0.02\n",
            "18409  SGBP_B_XBD           PF18329.6          34     3.61    186   0.59   0.52   0.43   0.04\n",
            "18410  Sgf11                PF08209.16         30     8.25     33   1.64   1.45   1.63   0.23\n",
            "18411  Sgf11_N              PF18519.6          15     3.21     39   1.40   1.41   1.38   0.11\n",
            "18412  SGF29_Tudor          PF07039.16         97     2.17    132   0.59   0.57   0.47   0.04\n",
            "18413  SGIII                PF15467.11         10     0.66    449   0.59   0.60   0.52   0.05\n",
            "18414  SGL                  PF08450.17         37     3.57    246   0.59   0.57   0.51   0.01\n",
            "18415  SGNH                 PF19040.5         198    14.98    248   0.59   0.56   0.35   0.01\n",
            "18416  Sgo0707_N1           PF18873.5           4     0.82    269   0.59   0.57   0.52   0.03\n",
            "18417  Sgo0707_N2           PF20623.3           9     0.87    141   0.59   0.60   0.53   0.04\n",
            "18418  SGP                  PF17228.7          23     1.06     93   0.61   0.54   0.42   0.10\n",
            "18419  SgrA_ig-like         PF22312.1           2     0.47    110   0.59   0.58   0.52   0.03\n",
            "18420  SgrR_N               PF12793.12         16     1.67    115   0.59   0.58   0.55   0.04\n",
            "18421  SgrT                 PF15894.10         11     2.12     46   1.20   1.06   1.12   0.13\n",
            "18422  SGS                  PF05002.20        121     1.27     82   0.69   0.64   0.62   0.06\n",
            "18423  SGSH_C               PF16347.10         25     0.78    154   0.59   0.54   0.53   0.03\n",
            "18424  SGT1                 PF07093.16         24     1.25    587   0.59   0.57   0.52   0.02\n",
            "18425  SGTA_dimer           PF16546.10         75     4.01     67   0.84   0.87   0.81   0.06\n",
            "18426  SH                   PF01445.22          1     0.65     57   0.98   0.96   0.92   0.10\n",
            "18427  SH2                  PF00017.29         52     3.69     77   0.73   0.70   0.66   0.03\n",
            "18428  SH2_1                PF21990.1          45     1.35    108   0.59   0.55   0.53   0.02\n",
            "18429  SH2_2                PF14633.11         76     1.85    211   0.59   0.56   0.53   0.03\n",
            "18430  SH3-RhoG_link        PF16609.10          3     0.50    261   0.59   0.60   0.52   0.03\n",
            "18431  SH3-WW_linker        PF16618.10         11     1.16    199   0.59   0.60   0.49   0.06\n",
            "18432  SH3BGR               PF04908.20          6     0.83     99   0.59   0.58   0.52   0.02\n",
            "18433  SH3BP5               PF05276.19         18     1.65    231   0.59   0.59   0.55   0.07\n",
            "18434  SH3DP                PF20287.3          28     3.26    122   0.59   0.57   0.54   0.03\n",
            "18435  SH3_1                PF00018.33         55    11.54     48   1.15   1.10   1.11   0.04\n",
            "18436  SH3_10               PF17902.6          16     2.43     65   0.86   0.83   0.83   0.03\n",
            "18437  SH3_11               PF18103.6           1     0.60     63   0.89   0.88   0.83   0.05\n",
            "18438  SH3_12               PF18129.6         101     5.90     69   0.81   0.83   0.69   0.04\n",
            "18439  SH3_13               PF18335.6         170     3.51     65   0.86   0.85   0.78   0.06\n",
            "18440  SH3_14               PF18343.6           2     0.61    134   0.59   0.57   0.51   0.02\n",
            "18441  SH3_15               PF18346.6          67     4.12     67   0.84   0.80   0.73   0.05\n",
            "18442  SH3_16               PF18348.6         108    13.54     49   1.13   1.12   1.08   0.05\n",
            "18443  SH3_17               PF18350.6           4     1.08     53   1.04   1.09   1.00   0.09\n",
            "18444  SH3_18               PF18354.6           2     0.51     85   0.67   0.70   0.60   0.02\n",
            "18445  SH3_19               PF18597.6          15     2.16     52   1.07   1.00   1.03   0.04\n",
            "18446  SH3_2                PF07653.22         28    15.35     55   1.01   0.98   0.87   0.03\n",
            "18447  SH3_3                PF08239.16        112    41.77     54   1.03   1.02   0.93   0.06\n",
            "18448  SH3_4                PF06347.18         18     6.11     56   0.99   0.94   0.96   0.07\n",
            "18449  SH3_5                PF08460.15         10     2.51     68   0.83   0.76   0.78   0.05\n",
            "18450  SH3_6                PF12913.12         33     4.09     53   1.05   0.96   1.02   0.05\n",
            "18451  SH3_7                PF12914.12         15    13.11     47   1.17   1.17   1.13   0.03\n",
            "18452  SH3_9                PF14604.11         20     8.71     49   1.13   1.11   1.03   0.02\n",
            "18453  Shadoo               PF14999.11          5     0.49    133   0.59   0.63   0.51   0.07\n",
            "18454  Shal-type            PF11601.13         15    11.55     27   1.98   1.94   1.89   0.28\n",
            "18455  Sharpin_PH           PF16764.10         10     0.93    121   0.59   0.60   0.52   0.02\n",
            "18456  SHD1                 PF03983.17         47     1.67     67   0.84   0.82   0.80   0.03\n",
            "18457  She2p                PF11435.13         16     0.90    201   0.59   0.58   0.53   0.05\n",
            "18458  SHE3                 PF17078.10          7     1.15    228   0.59   0.59   0.54   0.08\n",
            "18459  She9_MDM33           PF05546.16         51     1.03    198   0.59   0.57   0.54   0.05\n",
            "18460  Shigella_OspC        PF06128.16          2     0.44    292   0.59   0.57   0.51   0.03\n",
            "18461  Shikimate_DH         PF01488.25         35     1.97    138   0.59   0.62   0.53   0.03\n",
            "18462  Shikimate_dh_N       PF08501.16         36     2.44     83   0.68   0.69   0.64   0.02\n",
            "18463  SHIPPO-rpt           PF07004.17        318   318.00     33   1.01   0.97   0.93   0.16\n",
            "18464  SHIRT                PF18655.6          21     2.25     82   0.69   0.63   0.62   0.06\n",
            "18465  Shisa_N              PF13908.11        139     3.58     50   1.11   0.81   1.02   0.33\n",
            "18466  ShK                  PF01549.29        136    15.95     37   1.47   1.11   1.38   0.47\n",
            "18467  ShlB                 PF03865.18         55     3.85    311   0.59   0.58   0.52   0.03\n",
            "18468  SHLD1_C              PF15021.11         11     0.78    120   0.59   0.58   0.53   0.02\n",
            "18469  SHLD2_C              PF15793.10         19     0.85    166   0.59   0.58   0.52   0.03\n",
            "18470  SHLD2_OB1            PF21669.2          95     1.27    138   0.59   0.58   0.54   0.04\n",
            "18471  SHLD2_OB2            PF22779.1          78     1.65     82   0.69   0.67   0.64   0.05\n",
            "18472  SHMT                 PF00464.24         10     0.69    399   0.59   0.59   0.52   0.01\n",
            "18473  SHNi-TPR             PF10516.14         20    10.89     38   1.44   1.50   1.43   0.09\n",
            "18474  SHOCT                PF09851.14        485   485.00     28   1.67   1.81   1.66   0.29\n",
            "18475  SHOCT-like_DUF2089-C PF22746.1         139   139.00     32   1.36   1.51   1.36   0.21\n",
            "18476  SHOCT_2              PF20612.3          59     7.28     53   1.05   1.02   1.04   0.10\n",
            "18477  SHP                  PF03579.18          1     0.60     64   0.88   0.85   0.82   0.09\n",
            "18478  SHPRH_helical-1st    PF21325.2          14     1.06     95   0.60   0.62   0.55   0.04\n",
            "18479  SHPRH_helical-2nd    PF21324.2           1     0.39    222   0.59   0.58   0.52   0.02\n",
            "18480  SHQ1                 PF04925.20         57     1.58    186   0.59   0.57   0.53   0.04\n",
            "18481  SHQ1-like_CS         PF21413.2          76     1.28     93   0.61   0.60   0.57   0.03\n",
            "18482  Shr-like_HID         PF07550.16         18     5.76     60   0.93   0.94   0.87   0.07\n",
            "18483  SHR3_chaperone       PF08229.16         51     1.85    188   0.59   0.59   0.54   0.03\n",
            "18484  SHS2_FTSA            PF02491.25        243     5.30     79   0.72   0.73   0.67   0.03\n",
            "18485  SHS2_Rpb7-N          PF03876.22        114    36.00     57   0.98   1.00   0.90   0.04\n",
            "18486  Shufflon_N           PF04917.17          3     0.57    324   0.59   0.59   0.52   0.03\n",
            "18487  Shugoshin_C          PF07557.16         63    63.00     24   2.20   2.21   2.18   0.44\n",
            "18488  Shugoshin_N          PF07558.16         44    17.78     45   1.22   1.28   1.21   0.21\n",
            "18489  Sh_2                 PF17512.7           3     0.62    174   0.59   0.55   0.53   0.07\n",
            "18490  SiaC                 PF09345.15        122     2.60    120   0.59   0.56   0.54   0.03\n",
            "18491  Siah-Interact_N      PF09032.16         20     2.84     78   0.73   0.77   0.67   0.09\n",
            "18492  Sial-lect-inser      PF09264.16          3     0.83    198   0.59   0.59   0.52   0.03\n",
            "18493  Sialidase            PF02973.22          3     1.03    189   0.59   0.59   0.51   0.04\n",
            "18494  Sialidase-like_CBM   PF22585.2          22     1.06    156   0.59   0.57   0.53   0.02\n",
            "18495  SIC                  PF03482.18          2     1.65     30   1.79   1.75   1.74   0.28\n",
            "18496  SICA_alpha           PF12887.12         18     2.34    185   0.59   0.54   0.48   0.05\n",
            "18497  SICA_beta            PF12878.12         66     1.70    174   0.59   0.50   0.45   0.09\n",
            "18498  SICA_C               PF12879.12          2     0.45    145   0.59   0.59   0.51   0.02\n",
            "18499  SicP-binding         PF09119.15          2     0.76     84   0.67   0.69   0.61   0.03\n",
            "18500  SID                  PF11778.13         19     1.78    135   0.59   0.59   0.55   0.05\n",
            "18501  SID-1_RNA_chan       PF13965.11         28     1.73    594   0.59   0.55   0.52   0.04\n",
            "18502  Sid-5                PF17204.8           7     1.06     76   0.74   0.74   0.68   0.03\n",
            "18503  SidC_C               PF20875.2           2     0.49    123   0.59   0.59   0.53   0.03\n",
            "18504  SidC_lipid-bd        PF20892.2           2     0.52    127   0.59   0.59   0.52   0.04\n",
            "18505  SidC_N               PF18219.6           6     0.81    465   0.59   0.59   0.52   0.02\n",
            "18506  SidD_C               PF21513.2           1     0.40    161   0.59   0.61   0.52   0.02\n",
            "18507  SidD_cat             PF21511.2           1     0.39    302   0.59   0.59   0.52   0.02\n",
            "18508  SidE_DUB             PF19049.5           3     0.45    173   0.59   0.59   0.52   0.01\n",
            "18509  SidE_mART            PF19048.5           3     0.46    337   0.59   0.59   0.52   0.01\n",
            "18510  SidE_PDE             PF12252.13         29     3.59    226   0.59   0.56   0.48   0.04\n",
            "18511  SidF                 PF20988.2           2     0.50    839   0.59   0.59   0.52   0.02\n",
            "18512  SidM_N               PF20879.2           3     0.61    285   0.59   0.59   0.52   0.01\n",
            "18513  SidM_Rab1-act        PF20851.2           1     0.40    196   0.59   0.59   0.51   0.02\n",
            "18514  SieB                 PF14163.11         23     4.57    147   0.59   0.61   0.55   0.04\n",
            "18515  Sif                  PF06767.16          4     0.53    337   0.59   0.58   0.52   0.01\n",
            "18516  Sigma54_activat      PF00158.31        208     1.34    168   0.59   0.62   0.54   0.02\n",
            "18517  Sigma54_activ_2      PF14532.11          9     2.13    138   0.59   0.63   0.54   0.02\n",
            "18518  Sigma54_AID          PF00309.25        245    15.17     46   1.20   1.26   1.19   0.24\n",
            "18519  Sigma54_CBD          PF04963.18        349     3.13    185   0.59   0.61   0.50   0.02\n",
            "18520  Sigma54_DBD          PF04552.18         19     1.19    160   0.59   0.61   0.54   0.04\n",
            "18521  Sigma70_ECF          PF07638.16          7     1.40    185   0.59   0.61   0.53   0.03\n",
            "18522  Sigma70_ner          PF04546.18        139     4.55    210   0.59   0.60   0.49   0.05\n",
            "18523  Sigma70_r1_1         PF03979.19        107     4.88     81   0.70   0.74   0.63   0.09\n",
            "18524  Sigma70_r1_2         PF00140.25        352    58.30     34   1.59   1.69   1.58   0.12\n",
            "18525  Sigma70_r2           PF04542.19        197    18.06     71   0.79   0.77   0.75   0.05\n",
            "18526  Sigma70_r3           PF04539.21         61     8.84     78   0.73   0.78   0.70   0.06\n",
            "18527  Sigma70_r4           PF04545.21        102    15.55     50   1.11   1.18   1.07   0.09\n",
            "18528  Sigma70_r4_2         PF08281.17        141    49.73     54   1.03   1.09   1.03   0.07\n",
            "18529  Sigma_1s             PF02454.21          5     1.11    116   0.59   0.58   0.53   0.05\n",
            "18530  Sigma_1_2            PF03084.19          2     0.64    452   0.59   0.57   0.50   0.01\n",
            "18531  Sigma_M_inh          PF17453.7           2     0.39     96   0.60   0.62   0.52   0.06\n",
            "18532  Sigma_reg_C          PF13791.11         18     3.48    157   0.59   0.58   0.50   0.03\n",
            "18533  Sigma_reg_N          PF13800.11         19     4.69     91   0.63   0.62   0.58   0.05\n",
            "18534  SIKE                 PF05769.16         28     2.15    180   0.59   0.60   0.53   0.08\n",
            "18535  SIL1                 PF16782.10         23     2.00    292   0.59   0.58   0.50   0.04\n",
            "18536  Silic_transp         PF03842.18          4     0.59    513   0.59   0.60   0.52   0.02\n",
            "18537  SIMPL                PF04402.19        495    13.40    205   0.59   0.64   0.42   0.05\n",
            "18538  SIM_C                PF06621.17         13     1.39    309   0.59   0.55   0.50   0.04\n",
            "18539  SIN1                 PF05422.17         15     2.26    140   0.59   0.57   0.50   0.06\n",
            "18540  SIN1_PH              PF16979.10         53     2.71    105   0.59   0.56   0.48   0.05\n",
            "18541  Sin3a_C              PF16879.10        113     3.48    281   0.59   0.55   0.50   0.05\n",
            "18542  Sin3_corepress       PF08295.17         86     2.29     98   0.59   0.55   0.53   0.05\n",
            "18543  Sina_RING            PF21362.2         226     4.94     35   1.55   1.18   1.53   0.49\n",
            "18544  Sina_TRAF            PF03145.21         34     1.62     97   0.59   0.56   0.53   0.03\n",
            "18545  Sina_ZnF             PF21361.2          28     1.73     77   0.73   0.59   0.68   0.09\n",
            "18546  SinI                 PF08671.15         26    18.09     28   1.92   1.93   1.90   0.18\n",
            "18547  SIP                  PF04954.18         73     3.54    119   0.59   0.58   0.53   0.03\n",
            "18548  SIP1                 PF04938.17        147     9.31    215   0.59   0.56   0.45   0.03\n",
            "18549  SipA                 PF09052.15          3     0.81    214   0.59   0.61   0.54   0.02\n",
            "18550  SipA_2nd             PF22163.1           1     0.40    136   0.59   0.60   0.51   0.01\n",
            "18551  SipA_VBS             PF17985.6           3     3.00     22   1.63   1.74   1.61   0.21\n",
            "18552  Siphovirus-type_tail_C PF22768.1          86     4.30     97   0.59   0.57   0.50   0.02\n",
            "18553  Sipho_Gp157          PF05565.16         32     3.34    162   0.59   0.62   0.52   0.06\n",
            "18554  Sipho_Gp37           PF14594.11         27     3.88    345   0.59   0.59   0.50   0.02\n",
            "18555  Sipho_tail           PF05709.16         37    10.09    118   0.59   0.59   0.52   0.02\n",
            "18556  SipL                 PF12673.12        310    14.68     78   0.73   0.72   0.62   0.05\n",
            "18557  Sir1                 PF11603.13         34     4.31    120   0.59   0.56   0.54   0.04\n",
            "18558  SIR2                 PF02146.22         18     1.23    180   0.59   0.56   0.51   0.01\n",
            "18559  SIR2_2               PF13289.11         54     7.04    141   0.59   0.58   0.46   0.02\n",
            "18560  SIR3_C               PF22344.1           3     0.45    123   0.59   0.61   0.52   0.03\n",
            "18561  SIR4_SID             PF16991.10         10     2.24    163   0.59   0.56   0.51   0.06\n",
            "18562  SirA                 PF10747.14         12     1.37    140   0.59   0.56   0.54   0.04\n",
            "18563  SirB                 PF04247.17         72     2.59    120   0.59   0.59   0.53   0.10\n",
            "18564  SirC_C               PF22440.1          45     6.57     47   1.17   1.15   1.16   0.15\n",
            "18565  Sirohm_synth_C       PF14823.11         66     2.56     68   0.83   0.78   0.76   0.05\n",
            "18566  Sirohm_synth_M       PF14824.11         79    33.88     28   1.92   2.02   1.92   0.13\n",
            "18567  SIS                  PF01380.27         33     8.11    131   0.59   0.62   0.54   0.02\n",
            "18568  SIS_2                PF13580.11         31     3.14    138   0.59   0.63   0.55   0.02\n",
            "18569  SIT                  PF15330.11         28     2.20    114   0.59   0.58   0.50   0.03\n",
            "18570  Siva                 PF05458.17          4     0.63    173   0.59   0.54   0.52   0.05\n",
            "18571  SIX1_SD              PF16878.10         47     1.33    113   0.59   0.55   0.52   0.04\n",
            "18572  SKA1                 PF07160.17         32     2.63    237   0.59   0.59   0.52   0.05\n",
            "18573  SKA2                 PF16740.10         29     3.46    110   0.59   0.61   0.54   0.07\n",
            "18574  SKG6                 PF08693.15          9     1.20    668   0.59   0.57   0.51   0.05\n",
            "18575  SKI                  PF01202.27         74     3.61    159   0.59   0.60   0.53   0.02\n",
            "18576  Ski2_beta-barrel     PF21409.2          46     2.34     88   0.65   0.63   0.58   0.03\n",
            "18577  Ski2_N               PF17911.6          32     3.52    134   0.59   0.60   0.51   0.03\n",
            "18578  Ski7_2nd             PF21921.1           3     0.52    105   0.59   0.59   0.52   0.04\n",
            "18579  Ski7_3rd             PF21920.1           3     0.50     98   0.59   0.62   0.52   0.05\n",
            "18580  SKICH                PF17751.6          64     2.32    103   0.59   0.52   0.51   0.04\n",
            "18581  SKIP_SNW             PF02731.20         59     1.65    158   0.59   0.58   0.53   0.06\n",
            "18582  Ski_Sno              PF02437.22         16     1.22    108   0.59   0.58   0.53   0.04\n",
            "18583  SKN1_KRE6_Sbg1       PF03935.21         24     0.83    501   0.59   0.54   0.52   0.01\n",
            "18584  Skp1                 PF01466.24        179    13.75     48   1.15   1.15   1.13   0.09\n",
            "18585  Skp1_POZ             PF03931.20         13     2.80     62   0.90   0.90   0.85   0.06\n",
            "18586  SK_channel           PF03530.19         22     1.43    118   0.59   0.59   0.53   0.05\n",
            "18587  SL4P                 PF17618.7          48     4.29     88   0.65   0.63   0.62   0.04\n",
            "18588  SLAC1                PF03595.22        392     7.67    331   0.59   0.57   0.44   0.11\n",
            "18589  SLAIN                PF15301.11         28     1.27    429   0.59   0.59   0.49   0.05\n",
            "18590  SLAM                 PF06214.16         11     0.81    125   0.59   0.61   0.53   0.03\n",
            "18591  SLATT_1              PF18181.6          43     2.80    122   0.59   0.56   0.55   0.06\n",
            "18592  SLATT_2              PF18183.6          11     0.57    192   0.59   0.59   0.53   0.02\n",
            "18593  SLATT_3              PF18184.6          24     1.86    155   0.59   0.58   0.53   0.03\n",
            "18594  SLATT_4              PF18186.6          23     1.73    165   0.59   0.60   0.55   0.04\n",
            "18595  SLATT_5              PF18160.6          20     4.03    192   0.59   0.58   0.52   0.04\n",
            "18596  SLATT_6              PF18169.6          10     1.06    176   0.59   0.57   0.53   0.03\n",
            "18597  SLATT_fungal         PF18142.6         122     4.32    121   0.59   0.62   0.51   0.03\n",
            "18598  SLBB                 PF10531.14         55    18.65     55   1.01   1.06   0.91   0.06\n",
            "18599  SLBB_2               PF22461.1         175     4.38     79   0.72   0.75   0.63   0.03\n",
            "18600  SLBP_RNA_bind        PF15247.11         52     2.26     70   0.80   0.71   0.75   0.09\n",
            "18601  SLC12                PF03522.20         59     1.38    418   0.59   0.57   0.51   0.02\n",
            "18602  SLC25_like           PF18405.6           9     1.15    285   0.59   0.59   0.52   0.02\n",
            "18603  SLC35F               PF06027.17          5     0.68    299   0.59   0.59   0.53   0.06\n",
            "18604  SLC3A2_N             PF16028.10         18     1.36     77   0.73   0.69   0.68   0.04\n",
            "18605  SLC52_ribofla_tr     PF06237.17         55     1.52    429   0.59   0.58   0.53   0.04\n",
            "18606  Sld3_N               PF18523.6          18     2.76    115   0.59   0.55   0.54   0.04\n",
            "18607  Sld3_STD             PF08639.15         43     2.54    540   0.59   0.59   0.47   0.05\n",
            "18608  Sld5                 PF05916.16        151    21.27    107   0.59   0.59   0.46   0.04\n",
            "18609  SLD5_C               PF16922.10         70     9.04     56   0.99   1.00   0.91   0.04\n",
            "18610  Sld7_C               PF18596.6          52     2.81     77   0.73   0.73   0.65   0.05\n",
            "18611  Sld7_N               PF18636.6          12     1.01    122   0.59   0.56   0.49   0.03\n",
            "18612  SLED                 PF12140.13         28     2.07    112   0.59   0.56   0.53   0.03\n",
            "18613  SLFN-g3_helicase     PF09848.14         36     3.09    354   0.59   0.57   0.50   0.02\n",
            "18614  SLFN_AlbA_2          PF04326.19         63     6.05    118   0.59   0.59   0.45   0.03\n",
            "18615  SLFN_GTPase-like     PF21026.2          78     1.41    139   0.59   0.58   0.53   0.02\n",
            "18616  SLH                  PF00395.25         88    85.56     44   1.25   1.24   1.18   0.05\n",
            "18617  SLIDE                PF09111.15         98     2.62    116   0.59   0.57   0.54   0.05\n",
            "18618  SlipAM               PF04575.18         10     2.02    288   0.59   0.53   0.53   0.04\n",
            "18619  SLM4                 PF16818.10         15     1.45    160   0.59   0.60   0.50   0.04\n",
            "18620  SlmA-like_C          PF22276.1           4     0.55    117   0.59   0.60   0.52   0.03\n",
            "18621  Slo-like_RCK         PF22614.1         100     3.85    119   0.59   0.58   0.50   0.03\n",
            "18622  Slowpoke_C           PF21014.2          17     1.17    121   0.59   0.59   0.53   0.02\n",
            "18623  Slp                  PF03843.18         57     2.25    153   0.59   0.56   0.52   0.01\n",
            "18624  SlpA                 PF03217.19         29     5.48     63   0.89   0.87   0.82   0.13\n",
            "18625  SlpA_C               PF21620.2           2     0.50    878   0.59   0.60   0.52   0.02\n",
            "18626  SlpA_D2              PF22797.1           6     0.92    102   0.59   0.62   0.52   0.05\n",
            "18627  SlpA_N               PF22796.1           5     1.08    166   0.59   0.63   0.48   0.06\n",
            "18628  SLR1-BP              PF07333.17         27     2.77     56   0.99   0.74   0.89   0.32\n",
            "18629  Slr4                 PF19526.4          12     3.64    370   0.59   0.62   0.45   0.07\n",
            "18630  SLS                  PF14611.11         62    16.93     69   0.81   0.87   0.66   0.05\n",
            "18631  SLS1_C               PF20778.2          79     9.20    336   0.59   0.58   0.45   0.02\n",
            "18632  SLS1_KH2             PF20777.2          79    49.04     63   0.89   0.92   0.64   0.04\n",
            "18633  SLS1_N               PF20776.2          77     3.17     97   0.59   0.54   0.45   0.06\n",
            "18634  SLT                  PF01464.25         26     3.71    117   0.59   0.58   0.53   0.03\n",
            "18635  SLT_2                PF13406.11        157     2.79    292   0.59   0.58   0.52   0.02\n",
            "18636  SLT_3                PF18896.5          13     1.37     89   0.64   0.62   0.57   0.05\n",
            "18637  SLT_4                PF19489.4          47     1.47    184   0.59   0.52   0.53   0.03\n",
            "18638  SLT_beta             PF02258.21          1     0.54     70   0.81   0.78   0.74   0.03\n",
            "18639  SLT_L                PF14718.11         50     5.13     68   0.83   0.79   0.81   0.10\n",
            "18640  Slu7                 PF11708.13         96     1.92    253   0.59   0.56   0.48   0.07\n",
            "18641  SLX1_C               PF21202.2         162     1.74     65   0.86   0.68   0.80   0.16\n",
            "18642  Slx4                 PF09494.15         69     6.89     60   0.93   0.88   0.85   0.05\n",
            "18643  SLX9                 PF15341.11        127     5.95    127   0.59   0.61   0.46   0.12\n",
            "18644  SlyX                 PF04102.17        145     5.65     67   0.84   0.84   0.80   0.08\n",
            "18645  SM-ATX               PF14438.11         98     6.52     81   0.70   0.72   0.63   0.03\n",
            "18646  Smac_DIABLO          PF09057.15          9     0.78    237   0.59   0.61   0.51   0.03\n",
            "18647  SmaI                 PF17411.7           1     0.37    240   0.59   0.57   0.52   0.01\n",
            "18648  SmAKAP               PF15127.11         13     0.96     97   0.59   0.55   0.51   0.04\n",
            "18649  SMAP                 PF15477.11         69     3.95     75   0.75   0.74   0.59   0.09\n",
            "18650  SMBP                 PF16785.10         13     1.87    111   0.59   0.60   0.54   0.10\n",
            "18651  SMCHD1_S5            PF22899.1          83     1.83    168   0.59   0.53   0.50   0.03\n",
            "18652  SMC_hinge            PF06470.18         96     9.04    116   0.59   0.64   0.52   0.02\n",
            "18653  SMC_N                PF02463.24         28     1.85    220   0.59   0.61   0.49   0.02\n",
            "18654  SMC_Nse1             PF07574.18         76     2.85    194   0.59   0.59   0.45   0.04\n",
            "18655  SMC_ScpA             PF02616.19          5     1.46    232   0.59   0.61   0.49   0.04\n",
            "18656  SMC_ScpB             PF04079.21         88     2.85    160   0.59   0.63   0.54   0.03\n",
            "18657  SMEK_N               PF21941.1         135     3.87    140   0.59   0.59   0.54   0.06\n",
            "18658  SMG1                 PF15785.10          6     1.05    612   0.59   0.57   0.52   0.03\n",
            "18659  SMG1_N               PF17229.7           8     0.79    106   0.59   0.60   0.52   0.04\n",
            "18660  Smg4_UPF3            PF03467.20         81     3.46    172   0.59   0.58   0.49   0.04\n",
            "18661  Smg8_Smg9            PF10220.14         21     1.43    882   0.59   0.56   0.51   0.02\n",
            "18662  SMI1_KNR4            PF09346.15        151    16.75    125   0.59   0.56   0.41   0.02\n",
            "18663  Smim3                PF17307.7           6     0.84     60   0.93   0.93   0.87   0.11\n",
            "18664  SMIM43               PF21976.1           1     0.59     63   0.89   0.90   0.82   0.07\n",
            "18665  SMIM5_18_22          PF15831.10         10     2.25     56   0.99   0.93   0.95   0.18\n",
            "18666  SMK1_alpha_su        PF21415.2           1     0.57     62   0.90   0.91   0.84   0.08\n",
            "18667  SMK_beta_su          PF21414.2           1     0.41     90   0.63   0.63   0.56   0.03\n",
            "18668  SMMA_C               PF22427.1           1     0.46     78   0.73   0.73   0.66   0.04\n",
            "18669  SMMA_N               PF22426.1           1     0.37    100   0.59   0.59   0.51   0.03\n",
            "18670  SMN_G2-BD            PF20636.2          37    20.03     29   1.85   1.83   1.83   0.24\n",
            "18671  SMN_Tudor            PF06003.17          1     0.61     58   0.96   0.92   0.90   0.04\n",
            "18672  SMN_YG-box           PF20635.2           1     0.46     71   0.79   0.73   0.72   0.05\n",
            "18673  Smoa_sbd             PF17885.6          55     2.12    108   0.59   0.58   0.54   0.01\n",
            "18674  SMODS                PF18144.6          29     2.59    164   0.59   0.56   0.53   0.03\n",
            "18675  Smoothelin           PF12510.13         11     3.70     50   1.11   1.19   1.09   0.16\n",
            "18676  SMP                  PF04927.17         81     5.84     59   0.95   1.04   0.88   0.14\n",
            "18677  SmpA_OmlA            PF04355.18         73     6.24     71   0.79   0.76   0.74   0.03\n",
            "18678  SmpB                 PF01668.23        416     1.33    143   0.59   0.60   0.54   0.04\n",
            "18679  SMP_2                PF10144.14         14     1.37    159   0.59   0.62   0.54   0.04\n",
            "18680  SMP_C2CD2L           PF18696.6          32     1.50    152   0.59   0.57   0.54   0.04\n",
            "18681  SMP_LBD              PF17047.10          2     0.38    180   0.60   0.60   0.52   0.02\n",
            "18682  Smr                  PF01713.26        484     5.68     77   0.73   0.76   0.67   0.03\n",
            "18683  SMRP1                PF15181.11          3     0.45    262   0.59   0.54   0.51   0.02\n",
            "18684  SMUBP-2_HCS1_1B      PF21138.2          15     1.04    129   0.59   0.62   0.53   0.02\n",
            "18685  SMYLE_N              PF18615.6          12     1.15    256   0.59   0.57   0.51   0.03\n",
            "18686  Sm_like              PF16243.10         10     0.68     85   0.67   0.67   0.61   0.03\n",
            "18687  Sm_multidrug_ex      PF06695.16        131     3.09    121   0.59   0.62   0.51   0.05\n",
            "18688  SnAC                 PF14619.11         35     3.64     72   0.78   0.77   0.67   0.11\n",
            "18689  SNaCT1               PF22710.1           3     0.83    184   0.59   0.58   0.50   0.05\n",
            "18690  SNaCT10              PF22717.1           5     5.00     42   1.15   1.08   0.95   0.14\n",
            "18691  SNaCT11              PF22716.1           3     0.57    164   0.59   0.58   0.53   0.04\n",
            "18692  SNaCT12              PF22719.1           1     0.39    174   0.59   0.60   0.52   0.04\n",
            "18693  SNaCT13              PF22718.1           5     1.30    158   0.59   0.60   0.54   0.04\n",
            "18694  SNaCT14              PF22720.1           4     0.78    186   0.59   0.55   0.52   0.04\n",
            "18695  SNaCT5               PF22711.1          16     5.60    196   0.59   0.57   0.51   0.08\n",
            "18696  SNaCT6               PF22713.1           2     0.71    119   0.59   0.58   0.52   0.05\n",
            "18697  SNaCT7               PF22712.1           4     1.43    160   0.59   0.56   0.52   0.04\n",
            "18698  SNaCT8               PF22714.1           7     1.63    164   0.59   0.59   0.53   0.06\n",
            "18699  SNaCT9               PF22715.1           8     2.36    137   0.59   0.59   0.53   0.07\n",
            "18700  SNAD1                PF18744.6           6     1.20    213   0.59   0.53   0.50   0.04\n",
            "18701  SNAD2                PF18745.6           1     0.38    200   0.59   0.56   0.52   0.02\n",
            "18702  SNAD3                PF18749.6           3     0.77    379   0.59   0.56   0.52   0.03\n",
            "18703  SNAD4                PF18750.6          49     1.82    130   0.59   0.52   0.52   0.04\n",
            "18704  SNAP                 PF14938.11         49     2.05    282   0.59   0.59   0.55   0.05\n",
            "18705  SNAP-25              PF00835.24         27     2.66     56   0.99   0.91   0.89   0.08\n",
            "18706  SNAPC1               PF09808.14         59     3.49    191   0.59   0.55   0.53   0.05\n",
            "18707  SNAPC2               PF11035.13         23     1.65    334   0.59   0.60   0.46   0.03\n",
            "18708  SNAPC3               PF12251.13         57     2.08    199   0.59   0.50   0.51   0.05\n",
            "18709  SNAPC5               PF15497.11         52     4.94    101   0.59   0.63   0.49   0.13\n",
            "18710  Snapin_Pallidin      PF14712.11         66     5.49     89   0.64   0.68   0.60   0.08\n",
            "18711  SNARE                PF05739.24         64    25.17     53   1.05   1.07   1.05   0.11\n",
            "18712  SNase                PF00565.22         18     2.08    107   0.59   0.58   0.50   0.03\n",
            "18713  SNCAIP_SNCA_bd       PF16700.10          2     1.18     46   1.20   1.21   1.15   0.08\n",
            "18714  SNF                  PF00209.23         15     0.88    524   0.59   0.56   0.53   0.04\n",
            "18715  SNF2-rel_dom         PF00176.28         18     2.41    290   0.59   0.57   0.48   0.02\n",
            "18716  SNF2_assoc           PF08455.15         77     6.39    373   0.59   0.59   0.50   0.04\n",
            "18717  SNF5                 PF04855.17         91     1.72    225   0.59   0.57   0.46   0.03\n",
            "18718  Snf7                 PF03357.26         32     4.23    171   0.59   0.61   0.54   0.10\n",
            "18719  SNN_cytoplasm        PF09051.15          2     1.93     26   2.05   2.07   2.02   0.12\n",
            "18720  SNN_linker           PF09050.15          2     1.74     26   2.05   2.05   2.02   0.20\n",
            "18721  SNN_transmemb        PF09049.15          1     1.00     32   1.43   1.44   1.38   0.22\n",
            "18722  SNO                  PF01174.24          4     0.71    188   0.59   0.60   0.50   0.02\n",
            "18723  SnoaL                PF07366.17         29     4.62    126   0.59   0.57   0.55   0.02\n",
            "18724  SnoaL_2              PF12680.12        232    15.26    102   0.59   0.57   0.48   0.03\n",
            "18725  SnoaL_3              PF13474.11         32     5.48    121   0.59   0.54   0.52   0.03\n",
            "18726  SnoaL_4              PF13577.11         67     8.64    126   0.59   0.56   0.51   0.03\n",
            "18727  SnoaL_5              PF20409.3           5     0.66    118   0.59   0.57   0.53   0.02\n",
            "18728  SNRNP27              PF08648.17         75     2.33     56   0.99   0.94   0.95   0.08\n",
            "18729  Snu56_snRNP          PF19097.5          20     1.89    438   0.59   0.57   0.52   0.04\n",
            "18730  SNURF                PF07192.16          1     0.56     69   0.81   0.80   0.75   0.05\n",
            "18731  Snurportin1          PF11538.13         15     4.75     40   1.37   1.30   1.32   0.26\n",
            "18732  SNX17-27-31_F1_FERM  PF21273.2          87     1.99     90   0.63   0.62   0.58   0.04\n",
            "18733  SNX17-31_F2_FERM     PF21271.2          12     1.29     61   0.92   0.91   0.88   0.07\n",
            "18734  SNX17_FERM_C         PF18116.6          10     1.21    109   0.59   0.55   0.52   0.02\n",
            "18735  Snx8_BAR_dom         PF19566.4          41     1.08    258   0.59   0.59   0.54   0.04\n",
            "18736  SO2946-like_C        PF22237.2           6     1.97    163   0.59   0.61   0.50   0.05\n",
            "18737  SOAR                 PF16533.10         17     1.62    101   0.59   0.58   0.55   0.06\n",
            "18738  SOBP                 PF15279.11         11     1.26    325   0.59   0.55   0.49   0.04\n",
            "18739  Soc                  PF16855.10          1     0.50     74   0.76   0.73   0.69   0.03\n",
            "18740  SocA_Panacea         PF13274.11         66     5.98    113   0.59   0.55   0.47   0.03\n",
            "18741  SOCS                 PF12610.13          9     1.84     57   0.98   0.97   0.92   0.11\n",
            "18742  SOCS_box             PF07525.21        302    42.61     39   1.40   1.40   1.35   0.19\n",
            "18743  SoDot-IcmSS          PF16848.10          4     1.22    178   0.59   0.60   0.50   0.04\n",
            "18744  Sod_Cu               PF00080.25        360     3.76    133   0.59   0.58   0.42   0.04\n",
            "18745  Sod_Fe_C             PF02777.23        732     1.86    102   0.59   0.53   0.52   0.04\n",
            "18746  Sod_Fe_N             PF00081.27         19     1.31     82   0.69   0.62   0.64   0.06\n",
            "18747  Sod_Ni               PF09055.16         26     1.34    127   0.59   0.54   0.52   0.04\n",
            "18748  Sof1                 PF04158.19        117     2.70     87   0.65   0.65   0.61   0.12\n",
            "18749  SoFic-like_C         PF21248.2          47     1.55     76   0.74   0.75   0.71   0.04\n",
            "18750  SOG2                 PF10428.14         39     4.30    475   0.59   0.60   0.46   0.03\n",
            "18751  SOGA                 PF11365.13         20     1.31     94   0.61   0.65   0.55   0.07\n",
            "18752  SOGA1-2-like_CC      PF14818.11         80     4.39     54   1.03   0.98   1.00   0.21\n",
            "18753  SOGP_2nd             PF21250.2          29     0.85    249   0.59   0.57   0.53   0.03\n",
            "18754  SOGP_4th             PF21270.2           3     0.65     90   0.63   0.62   0.57   0.03\n",
            "18755  SOGP_N               PF21958.1         163     2.26    225   0.59   0.57   0.53   0.02\n",
            "18756  SOK                  PF06136.18         34     0.59     89   0.64   0.60   0.57   0.02\n",
            "18757  Solute_trans_a       PF03619.21        253     4.79    267   0.59   0.55   0.52   0.07\n",
            "18758  Sol_i_2              PF22750.1          13     1.25     90   0.63   0.59   0.58   0.06\n",
            "18759  Sol_Rieske_ferrdox   PF22543.1          18     1.63     40   1.37   1.31   1.33   0.11\n",
            "18760  Somatomedin_B        PF01033.22        128     3.46     40   1.37   0.99   1.30   0.56\n",
            "18761  Somatostatin         PF03002.20          5     4.92     18   2.91   2.53   2.90   0.64\n",
            "18762  SOP4                 PF17081.10         13     0.89    209   0.59   0.58   0.53   0.02\n",
            "18763  SopA                 PF13981.11          3     0.93    127   0.59   0.57   0.51   0.03\n",
            "18764  SopA_C               PF13979.11          2     0.61    177   0.59   0.58   0.52   0.01\n",
            "18765  SoPB_HTH             PF18090.6           2     0.86     75   0.76   0.79   0.70   0.03\n",
            "18766  SopD                 PF11047.13          2     0.53    319   0.59   0.57   0.52   0.02\n",
            "18767  SopE_GEF             PF07487.18          3     0.79    136   0.59   0.59   0.53   0.01\n",
            "18768  SOR                  PF07682.16          5     0.68    300   0.59   0.55   0.52   0.01\n",
            "18769  Sorb                 PF02208.21          3     1.32     42   1.30   1.21   1.24   0.08\n",
            "18770  Sororin              PF09666.15          9     2.01    155   0.59   0.59   0.48   0.06\n",
            "18771  Sortase              PF04203.18         60     3.64    125   0.59   0.57   0.45   0.02\n",
            "18772  Sortilin-Vps10       PF15902.10        127     3.12    446   0.59   0.56   0.50   0.03\n",
            "18773  Sortilin_C           PF15901.10        284     2.36    167   0.59   0.48   0.45   0.09\n",
            "18774  Sorting_nexin        PF03700.18          8     1.46     76   0.74   0.77   0.64   0.06\n",
            "18775  SOR_N                PF22496.1           1     1.00     37   1.40   1.46   1.35   0.10\n",
            "18776  SOR_SNZ              PF01680.22        109     0.80    207   0.59   0.62   0.53   0.03\n",
            "18777  SOS1_NGEF_PH         PF22697.1         162     3.51    106   0.59   0.55   0.49   0.04\n",
            "18778  Sos7                 PF20882.2         152     4.89     75   0.75   0.78   0.73   0.10\n",
            "18779  SOSSC                PF15925.10          8     1.23    101   0.59   0.59   0.48   0.02\n",
            "18780  SOTI                 PF17079.10          6     0.68    101   0.59   0.55   0.52   0.02\n",
            "18781  SOUL                 PF04832.17        192     4.04    173   0.59   0.56   0.45   0.03\n",
            "18782  Sox17_18_mid         PF12067.13         28     2.52     50   1.11   1.06   0.99   0.10\n",
            "18783  SoxA-TsdA_cyt-c      PF21342.2         127     2.47     88   0.65   0.60   0.60   0.03\n",
            "18784  SoxD                 PF04267.17         68     1.62     82   0.69   0.59   0.64   0.04\n",
            "18785  SoxE                 PF06525.16          6     0.67    149   0.59   0.58   0.52   0.03\n",
            "18786  SoxE_N               PF22852.1           4     1.66     48   1.15   1.21   1.11   0.22\n",
            "18787  SoxG                 PF04268.17          1     0.39    152   0.59   0.60   0.51   0.01\n",
            "18788  SOXp                 PF12336.13         12     0.98     88   0.65   0.61   0.52   0.04\n",
            "18789  SoxY                 PF13501.11        128     3.46    112   0.59   0.61   0.53   0.03\n",
            "18790  SoxZ                 PF08770.16        107     3.16     98   0.59   0.58   0.53   0.03\n",
            "18791  Sox_N                PF12444.13         18     1.61     76   0.74   0.74   0.66   0.05\n",
            "18792  Soyouz_module        PF14313.11         16     4.15     58   0.96   0.99   0.94   0.06\n",
            "18793  SO_alpha_A3          PF17806.6           8     1.32     87   0.65   0.67   0.60   0.02\n",
            "18794  Sp-CxC               PF20304.3           9     5.02     63   0.89   0.90   0.84   0.09\n",
            "18795  Sp-DndD              PF20306.3          34    34.00     51   1.05   1.09   1.01   0.12\n",
            "18796  SP2                  PF03014.19          2     0.64    205   0.59   0.59   0.51   0.02\n",
            "18797  SP24                 PF16504.10         41     6.63    147   0.59   0.59   0.54   0.09\n",
            "18798  SP2_C                PF20752.2           2     0.73    220   0.59   0.59   0.51   0.02\n",
            "18799  SP2_M                PF20751.2           2     0.70    141   0.59   0.62   0.52   0.01\n",
            "18800  Sp38                 PF07354.17          5     0.66    100   0.59   0.56   0.53   0.02\n",
            "18801  Sp38_C               PF20626.2           5     1.00     50   1.11   0.91   1.06   0.22\n",
            "18802  SPA                  PF08616.15         21     2.28    113   0.59   0.57   0.52   0.03\n",
            "18803  Spa1_C               PF18218.6          10     1.67     99   0.59   0.58   0.53   0.05\n",
            "18804  SpaA                 PF17802.6          88     4.49     91   0.63   0.65   0.52   0.05\n",
            "18805  SpaA_2               PF19403.4          12     1.06     96   0.60   0.59   0.51   0.04\n",
            "18806  SpaA_3               PF20674.2          44     2.40    121   0.59   0.59   0.50   0.05\n",
            "18807  SPACA7               PF15307.11          5     0.83    107   0.59   0.58   0.51   0.04\n",
            "18808  SPACA9               PF15120.11          3     0.58    164   0.59   0.56   0.52   0.03\n",
            "18809  Spaetzle             PF16077.10         41     1.41     96   0.60   0.48   0.52   0.09\n",
            "18810  SPAM                 PF02090.20         13     2.45    145   0.59   0.59   0.56   0.11\n",
            "18811  SPAN                 PF02510.19          1     0.41    336   0.59   0.61   0.52   0.02\n",
            "18812  SPAN-X               PF07458.17          6     0.83     95   0.60   0.59   0.53   0.06\n",
            "18813  SPAR                 PF22004.1           6     0.77     75   0.75   0.73   0.70   0.05\n",
            "18814  SPARC_Ca_bdg         PF10591.14         18     1.41    113   0.59   0.51   0.53   0.06\n",
            "18815  SPARK                PF19160.5         345     4.37    162   0.59   0.50   0.33   0.10\n",
            "18816  SPAR_C               PF11881.13         19     1.45    239   0.59   0.60   0.49   0.04\n",
            "18817  SPASM                PF13186.11        143     7.78     67   0.84   0.69   0.65   0.10\n",
            "18818  SPATA19              PF15212.11          6     0.52    130   0.59   0.57   0.52   0.05\n",
            "18819  SPATA1_C             PF15743.10         11     1.11    150   0.59   0.59   0.53   0.10\n",
            "18820  SPATA24              PF15175.11          4     0.94    182   0.59   0.59   0.53   0.07\n",
            "18821  SPATA25              PF15218.11          6     0.55    228   0.59   0.59   0.52   0.02\n",
            "18822  SPATA2_PUB-like      PF21388.2           6     0.63    193   0.59   0.58   0.52   0.02\n",
            "18823  SPATA3               PF15662.10          5     0.62    191   0.59   0.55   0.52   0.05\n",
            "18824  SPATA48              PF15073.11          7     0.79    128   0.59   0.56   0.54   0.03\n",
            "18825  SPATA6               PF14909.11         18     1.31    139   0.59   0.59   0.53   0.02\n",
            "18826  SPATA9               PF15824.10          4     0.57    253   0.59   0.60   0.52   0.02\n",
            "18827  Spatacsin_C          PF14649.11         29     2.04    296   0.59   0.58   0.51   0.04\n",
            "18828  SPATIAL              PF15256.11         22     1.39    201   0.59   0.57   0.50   0.03\n",
            "18829  SPATS2-like          PF07139.16         17     1.12    309   0.59   0.59   0.53   0.06\n",
            "18830  Spb1_C               PF07780.17         94     1.79    216   0.59   0.59   0.52   0.08\n",
            "18831  Spc110_C             PF18520.6          14     2.36     52   1.07   1.03   1.04   0.10\n",
            "18832  SPC12                PF06645.18        127     3.79     71   0.79   0.74   0.78   0.07\n",
            "18833  SPC22                PF04573.17         88     2.56    173   0.59   0.55   0.52   0.03\n",
            "18834  Spc24                PF08286.16         79     4.38    109   0.59   0.59   0.47   0.06\n",
            "18835  SPC25                PF06703.16        146     4.95    159   0.59   0.58   0.52   0.03\n",
            "18836  Spc29                PF17082.10          4     1.19    245   0.59   0.58   0.49   0.05\n",
            "18837  Spc42p               PF11544.13          5     1.13     75   0.76   0.78   0.69   0.08\n",
            "18838  Spc7                 PF08317.16         48     2.92    196   0.59   0.57   0.54   0.05\n",
            "18839  SpdB                 PF05122.18          3     1.11     50   1.10   1.04   1.05   0.10\n",
            "18840  SPDY                 PF03771.21         50     8.41     61   0.92   0.87   0.83   0.09\n",
            "18841  Spec3                PF15795.10         19     0.98     85   0.67   0.64   0.60   0.10\n",
            "18842  SpecificRecomb       PF10136.14         40     1.82    639   0.59   0.61   0.54   0.02\n",
            "18843  SPECT1               PF18680.6           1     0.40    180   0.59   0.62   0.52   0.03\n",
            "18844  Spectrin             PF00435.26         80     8.97    105   0.59   0.59   0.56   0.09\n",
            "18845  Spectrin_2           PF18373.6          31     1.57     78   0.72   0.68   0.69   0.06\n",
            "18846  Spectrin_3           PF21019.2         114     4.20     68   0.83   0.83   0.81   0.09\n",
            "18847  Spectrin_4           PF21020.2           5     0.59    100   0.59   0.59   0.52   0.04\n",
            "18848  SPEF2_D5             PF22946.1         106     5.03    134   0.59   0.56   0.57   0.09\n",
            "18849  SpeFL                PF10940.13          2     1.08     40   1.37   1.26   1.32   0.16\n",
            "18850  SPEG_u2              PF16650.10          2     0.77     57   0.98   0.97   0.93   0.12\n",
            "18851  Spem1                PF15670.10         10     0.69    253   0.59   0.57   0.52   0.01\n",
            "18852  Speriolin_C          PF15059.11          9     0.71    148   0.59   0.58   0.53   0.02\n",
            "18853  Speriolin_N          PF15058.11          4     0.88    228   0.59   0.64   0.50   0.02\n",
            "18854  Spermine_synth       PF01564.22         15     2.44    186   0.59   0.58   0.52   0.02\n",
            "18855  Spermine_synt_N      PF17284.7          13     3.68     53   1.05   1.01   1.00   0.05\n",
            "18856  Sperm_act_pep        PF08250.16         11    11.00     10   2.62   2.81   2.62   1.14\n",
            "18857  Sperm_Ag_HE2         PF05324.18          7     1.25     70   0.80   0.83   0.74   0.04\n",
            "18858  SPESP1               PF15754.10          8     0.81    322   0.59   0.60   0.52   0.03\n",
            "18859  Spexin               PF15171.11          5     0.60     90   0.63   0.63   0.56   0.03\n",
            "18860  SPG4                 PF17325.7          10     1.09    109   0.59   0.56   0.52   0.06\n",
            "18861  SPG48                PF14764.11         25     1.71    118   0.59   0.59   0.54   0.04\n",
            "18862  SpGH101_helical      PF20909.2           6     1.77     60   0.93   0.94   0.87   0.08\n",
            "18863  Spheroidin           PF05541.17          4     0.85    944   0.59   0.57   0.52   0.03\n",
            "18864  Spherulin4           PF12138.13         79     2.74    239   0.59   0.55   0.47   0.02\n",
            "18865  SPICE                PF15678.10         16     2.26    409   0.59   0.60   0.51   0.06\n",
            "18866  SPIDER               PF12485.13         23     1.69    156   0.59   0.59   0.51   0.08\n",
            "18867  Spidroin_MaSp        PF11260.13         26     1.88     85   0.67   0.76   0.62   0.10\n",
            "18868  Spidroin_N           PF16763.10         22     1.66    125   0.59   0.60   0.54   0.05\n",
            "18869  Spike_JR1            PF22464.1           1     0.49     78   0.73   0.75   0.66   0.03\n",
            "18870  Spike_JR2            PF22465.1           1     0.48     76   0.74   0.74   0.67   0.02\n",
            "18871  Spike_torovirin      PF17072.10         11     3.60   1272   0.59   0.55   0.49   0.03\n",
            "18872  Spin-Ssty            PF02513.22         21     3.43     50   1.11   1.05   1.08   0.06\n",
            "18873  SPIN90_LRD           PF09431.15         79     1.75    137   0.59   0.60   0.51   0.06\n",
            "18874  Spindle_Spc25        PF08234.17        124     7.16     73   0.77   0.76   0.74   0.05\n",
            "18875  Spiralin             PF05215.18        126    16.18     81   0.70   0.75   0.53   0.12\n",
            "18876  SPK                  PF04435.23        202     5.79    104   0.59   0.57   0.54   0.05\n",
            "18877  SPL                  PF20903.2          34     0.57    338   0.59   0.56   0.53   0.02\n",
            "18878  SplA                 PF11132.13          8     1.09     74   0.76   0.73   0.72   0.05\n",
            "18879  SPMAP1-like          PF15075.11         11     0.91    134   0.59   0.56   0.53   0.02\n",
            "18880  SPMIP1               PF22589.1         192     2.84     82   0.69   0.63   0.63   0.04\n",
            "18881  SPMIP10-like         PF14983.11          2     0.46    138   0.59   0.57   0.52   0.01\n",
            "18882  SPMIP11              PF22593.1          37     1.14    131   0.59   0.52   0.52   0.04\n",
            "18883  SPMIP4-like          PF15093.11          7     0.85    284   0.59   0.55   0.53   0.02\n",
            "18884  SPMIP5               PF22573.1          50     1.09     62   0.90   0.85   0.85   0.03\n",
            "18885  SPMIP8               PF22574.1          57     0.93    153   0.59   0.56   0.53   0.02\n",
            "18886  SpmSyn_N             PF17950.6          10     0.67     96   0.60   0.61   0.53   0.02\n",
            "18887  SPN1_m3Gcap_bd       PF21974.1          14     0.88    179   0.59   0.54   0.53   0.02\n",
            "18888  SpnB_Rossmann        PF22953.1           1     0.40    170   0.59   0.63   0.52   0.03\n",
            "18889  Spo0A_C              PF08769.16        102     2.84    104   0.59   0.58   0.55   0.03\n",
            "18890  Spo0M                PF07070.16         65     1.86    213   0.59   0.62   0.54   0.02\n",
            "18891  SPO11_like           PF03533.19          9     1.79     43   1.28   1.34   1.24   0.10\n",
            "18892  Spo12                PF05032.17         30     5.01     35   1.55   1.53   1.51   0.18\n",
            "18893  Spo16_C              PF21698.2          12     7.13     43   1.28   1.32   1.25   0.17\n",
            "18894  Spo16_N              PF19225.5          12     2.51    118   0.59   0.57   0.51   0.06\n",
            "18895  SPO22                PF08631.15         68     7.36    274   0.59   0.58   0.51   0.04\n",
            "18896  SPO24                PF22044.1           8     1.37     64   0.87   0.92   0.77   0.05\n",
            "18897  Spo7                 PF03907.18         57     1.53    211   0.59   0.57   0.50   0.03\n",
            "18898  Spo7_2_N             PF15407.11         35     4.24     64   0.88   0.85   0.85   0.04\n",
            "18899  SPOB_a               PF14689.11         23     5.17     62   0.90   0.88   0.87   0.09\n",
            "18900  SPOB_ab              PF14682.11         11     1.73    112   0.59   0.57   0.53   0.04\n",
            "18901  SPOC                 PF07744.18         51     6.55    153   0.59   0.57   0.45   0.01\n",
            "18902  SPOCS_spoVID-N       PF20918.2          21     1.66    132   0.59   0.61   0.54   0.03\n",
            "18903  SpoIIAA-like         PF11964.13        143     9.66    106   0.59   0.58   0.43   0.04\n",
            "18904  SpoIID               PF08486.15        393     3.59     97   0.59   0.58   0.44   0.06\n",
            "18905  SpoIIE               PF07228.17        152     7.05    192   0.59   0.62   0.52   0.01\n",
            "18906  SpoIIE_N             PF19732.4         168     1.20    605   0.59   0.60   0.54   0.03\n",
            "18907  SpoIIIAC             PF06686.16        348    18.36     56   0.99   1.09   0.98   0.18\n",
            "18908  SpoIIIAH             PF12685.12        106     4.53    191   0.59   0.62   0.49   0.07\n",
            "18909  SpoIIID              PF12116.13         13     0.84     82   0.69   0.70   0.64   0.06\n",
            "18910  SpoIIM               PF01944.22        302     8.50    163   0.59   0.63   0.40   0.09\n",
            "18911  SpoIIP               PF07454.16         50     2.88    269   0.59   0.58   0.49   0.04\n",
            "18912  SpoIISA_toxin        PF14171.11          8     1.63    243   0.59   0.57   0.51   0.05\n",
            "18913  SpoIISB_antitox      PF14185.11          2     0.93     57   0.98   0.96   0.92   0.08\n",
            "18914  SpoIVA_ATPase        PF09547.15        131     0.74    237   0.59   0.59   0.53   0.02\n",
            "18915  SpoIVA_C             PF20439.3         131     1.50     76   0.74   0.75   0.70   0.05\n",
            "18916  SpoIVA_middle        PF20438.3         131     1.42    179   0.59   0.60   0.55   0.02\n",
            "18917  Spond_N              PF06468.18         39     1.34    194   0.59   0.53   0.52   0.02\n",
            "18918  SpoOE-like           PF09388.15        119    29.48     42   1.31   1.35   1.27   0.17\n",
            "18919  SPOR                 PF05036.18        133    32.82     76   0.74   0.77   0.67   0.04\n",
            "18920  Spore-coat_CotD      PF11122.13          6     1.33     90   0.63   0.57   0.55   0.07\n",
            "18921  Spore-coat_CotZ      PF10612.14          8     1.05    159   0.59   0.53   0.49   0.06\n",
            "18922  Spore_coat_CotO      PF14153.11          6     1.39    182   0.59   0.58   0.51   0.05\n",
            "18923  Spore_Cse60          PF10957.13          4     0.90     60   0.93   0.90   0.88   0.07\n",
            "18924  Spore_GerAC          PF05504.16        239     9.28    167   0.59   0.58   0.47   0.04\n",
            "18925  Spore_GerQ           PF09671.15         20     0.87     76   0.74   0.72   0.69   0.04\n",
            "18926  Spore_III_AA         PF19568.4           4     0.54    306   0.59   0.60   0.52   0.02\n",
            "18927  Spore_III_AB         PF09548.15        148     6.53    170   0.59   0.62   0.55   0.04\n",
            "18928  Spore_III_AE         PF09546.15        120     3.57    321   0.59   0.63   0.56   0.08\n",
            "18929  Spore_III_AF         PF09581.15         71     5.17    185   0.59   0.62   0.51   0.06\n",
            "18930  Spore_II_R           PF09551.15        151     1.09    128   0.59   0.55   0.54   0.02\n",
            "18931  Spore_permease       PF03845.18         15     2.58    321   0.59   0.59   0.55   0.09\n",
            "18932  Spore_SspJ           PF09575.15          2     0.97     47   1.17   1.20   1.11   0.11\n",
            "18933  Spore_YabQ           PF09578.15        150     6.39     79   0.72   0.70   0.70   0.16\n",
            "18934  Spore_YhaL           PF14147.11         13     2.05     52   1.06   1.01   1.04   0.08\n",
            "18935  Spore_YhcN_YlaJ      PF09580.15         36     6.58    158   0.59   0.58   0.53   0.08\n",
            "18936  Spore_YpjB           PF09577.15         13     1.51    230   0.59   0.59   0.54   0.04\n",
            "18937  Spore_YtfJ           PF09579.15        160     3.21     81   0.70   0.75   0.59   0.04\n",
            "18938  Spore_YtrH           PF14034.11         31     1.16    101   0.59   0.64   0.54   0.06\n",
            "18939  Spore_YunB           PF09560.15        101     2.86     91   0.63   0.65   0.60   0.04\n",
            "18940  Sporozoite_P67       PF05642.16          3     0.59    727   0.59   0.62   0.50   0.02\n",
            "18941  SporV_AA             PF12164.13         68     3.15     90   0.63   0.64   0.58   0.06\n",
            "18942  Spot_14              PF07084.17         23     1.04    151   0.59   0.58   0.50   0.03\n",
            "18943  SPOUT_MTase          PF02590.22        482     2.22    154   0.59   0.58   0.51   0.03\n",
            "18944  SPOUT_MTase_2        PF14419.11         15     0.85    173   0.59   0.61   0.53   0.02\n",
            "18945  SpoU_methylase       PF00588.24        114     4.30    142   0.59   0.61   0.52   0.02\n",
            "18946  SpoU_methylas_C      PF12105.13         47     3.70     54   1.03   0.99   0.99   0.06\n",
            "18947  SpoU_sub_bind        PF08032.17         86    12.96     76   0.74   0.77   0.69   0.03\n",
            "18948  SpoU_sub_bind_like   PF22655.1           6     0.74     90   0.63   0.65   0.57   0.02\n",
            "18949  SpoV                 PF08183.16          5     4.33     25   2.13   2.14   2.12   0.28\n",
            "18950  SpoVAB               PF13782.11         74     1.73    109   0.59   0.62   0.54   0.07\n",
            "18951  SpoVAC_SpoVAEB       PF03862.18        276     1.71    117   0.59   0.62   0.54   0.07\n",
            "18952  SpoVAD               PF07451.16         32     0.89    329   0.59   0.60   0.54   0.02\n",
            "18953  SpoVAE               PF14097.11         28     0.93    180   0.59   0.63   0.54   0.04\n",
            "18954  SpoVG                PF04026.17        112     2.01     82   0.69   0.69   0.63   0.04\n",
            "18955  SpoVIF               PF14069.11         23     2.59     72   0.78   0.82   0.74   0.09\n",
            "18956  SpoVR                PF04293.18         87     1.20    419   0.59   0.55   0.52   0.03\n",
            "18957  SpoVS                PF04232.17         43     1.60     83   0.68   0.74   0.64   0.05\n",
            "18958  SpoVT_C              PF15714.10        160     2.08    128   0.59   0.62   0.53   0.04\n",
            "18959  SPP                  PF06550.16          3     0.70    283   0.59   0.62   0.52   0.06\n",
            "18960  Spp-24               PF07448.16         13     0.97     64   0.88   0.79   0.82   0.06\n",
            "18961  SPP1_GP23-1          PF20787.2           2     0.87     47   1.17   1.13   1.12   0.06\n",
            "18962  SPR1                 PF15356.11          3     0.41    114   0.59   0.54   0.52   0.13\n",
            "18963  SprA-related         PF12118.13          3     0.83    335   0.59   0.62   0.51   0.05\n",
            "18964  SprA_N               PF14349.11         22     1.73    510   0.59   0.57   0.50   0.03\n",
            "18965  SprB                 PF13573.11         42    18.66     37   1.47   1.43   1.45   0.23\n",
            "18966  SPRING1              PF10218.14         38     0.87    131   0.59   0.45   0.51   0.12\n",
            "18967  Sprouty              PF05210.18         34     1.19    116   0.59   0.45   0.48   0.10\n",
            "18968  SPRR2                PF14820.11          3     0.57     68   0.83   0.67   0.76   0.31\n",
            "18969  SprT-like            PF10263.14         51     3.63    108   0.59   0.53   0.48   0.04\n",
            "18970  SPRTN_ZBD            PF22934.1         132     1.66     48   1.15   0.91   1.11   0.15\n",
            "18971  SPRY                 PF00622.34         47     5.98    121   0.59   0.53   0.49   0.02\n",
            "18972  SPT16                PF08644.16         71     1.89    150   0.59   0.59   0.50   0.03\n",
            "18973  SPT16_C              PF21091.2         210     3.40     83   0.68   0.71   0.63   0.29\n",
            "18974  SPT2                 PF08243.16         38     3.21    109   0.59   0.59   0.50   0.18\n",
            "18975  Spt20_SEP            PF12090.13         24     3.54    177   0.59   0.57   0.46   0.04\n",
            "18976  SPT2_N               PF22878.1          16     1.82     91   0.63   0.63   0.56   0.13\n",
            "18977  Spt4                 PF06093.18         36     1.85     77   0.73   0.65   0.66   0.06\n",
            "18978  Spt46                PF17734.6           7     0.55    228   0.59   0.56   0.53   0.02\n",
            "18979  Spt5-NGN             PF03439.18         45     4.13     84   0.68   0.68   0.62   0.04\n",
            "18980  Spt5_N               PF11942.13         79     3.13     97   0.59   0.59   0.47   0.12\n",
            "18981  SPT6_acidic          PF14632.11        123     2.27     87   0.65   0.68   0.55   0.20\n",
            "18982  Spt6_S1              PF21710.2          34     2.00     91   0.63   0.63   0.59   0.04\n",
            "18983  SPT_ssu-like         PF11779.13         56     7.92     54   1.03   0.96   1.01   0.16\n",
            "18984  SpuA_C               PF18033.6           7     0.93     93   0.61   0.64   0.54   0.03\n",
            "18985  Spuma_A9PTase        PF03539.19          4     0.53    163   0.59   0.56   0.52   0.02\n",
            "18986  Sputnik_MCP_1st      PF21915.1           8     1.25    289   0.59   0.59   0.52   0.02\n",
            "18987  SpvB                 PF03534.18         37     1.22    287   0.59   0.54   0.50   0.02\n",
            "18988  SpvD                 PF05563.17          1     0.39    212   0.59   0.57   0.52   0.03\n",
            "18989  SPW                  PF03779.19         37     6.91     48   1.15   1.14   1.12   0.23\n",
            "18990  SPX                  PF03105.24        192     9.10    340   0.59   0.58   0.45   0.07\n",
            "18991  Spy1                 PF11357.13         27     1.04    131   0.59   0.54   0.53   0.04\n",
            "18992  SP_0191-like         PF21642.2           4     0.77    132   0.59   0.61   0.53   0.03\n",
            "18993  SP_C-Propep          PF08999.15          2     0.43     96   0.59   0.61   0.52   0.04\n",
            "18994  SQAPI                PF16845.10         12     1.13     83   0.68   0.69   0.63   0.04\n",
            "18995  SQHop_cyclase_C      PF13243.11         12     0.68    319   0.59   0.55   0.53   0.02\n",
            "18996  SQHop_cyclase_N      PF13249.11        103     2.24    291   0.59   0.57   0.54   0.01\n",
            "18997  SQS_PSY              PF00494.24         80     8.32    263   0.59   0.60   0.50   0.02\n",
            "18998  Squash               PF00299.23         12     2.06     29   1.85   1.52   1.81   0.47\n",
            "18999  SR-25                PF10500.14         10     0.75    226   0.59   0.60   0.51   0.10\n",
            "19000  SR1P                 PF13790.11          8     1.94     37   1.47   1.27   1.43   0.23\n",
            "19001  SRA                  PF18491.6          17     2.49    145   0.59   0.53   0.51   0.04\n",
            "19002  SRA1                 PF07304.16          7     0.68    147   0.59   0.58   0.52   0.03\n",
            "19003  SRAP                 PF02586.19        355     2.22    214   0.59   0.54   0.47   0.02\n",
            "19004  SRA_like             PF08136.16          2     1.13     46   1.20   1.20   1.14   0.08\n",
            "19005  SRC-1                PF08832.15         30     1.34     88   0.65   0.68   0.54   0.06\n",
            "19006  SRCR                 PF00530.23         65     2.43    100   0.59   0.50   0.50   0.07\n",
            "19007  SRCR_2               PF15494.11         32     1.62     99   0.59   0.51   0.53   0.05\n",
            "19008  Sre                  PF03125.23         19     2.87    365   0.59   0.55   0.53   0.07\n",
            "19009  SRF-TF               PF00319.23        214    10.40     48   1.15   1.16   1.14   0.06\n",
            "19010  SrfB                 PF07520.16         15     1.09    992   0.59   0.58   0.51   0.01\n",
            "19011  Srg                  PF02118.26         47     3.94    275   0.59   0.55   0.55   0.10\n",
            "19012  SRI                  PF08236.16        154     7.26     78   0.73   0.71   0.63   0.10\n",
            "19013  SRP-alpha_N          PF04086.18         95     4.02    301   0.59   0.59   0.44   0.06\n",
            "19014  SRP14                PF02290.20        119     3.36    106   0.59   0.59   0.49   0.05\n",
            "19015  SRP19                PF01922.22        188     2.56     95   0.60   0.59   0.54   0.03\n",
            "19016  SRP1_TIP1            PF00660.22         57     2.20     99   0.59   0.57   0.53   0.04\n",
            "19017  SRP40_C              PF05022.17         77     2.46     76   0.74   0.71   0.66   0.09\n",
            "19018  SRP54                PF00448.27         47     1.73    196   0.59   0.62   0.54   0.03\n",
            "19019  SRP54_N              PF02881.24       1086    11.43     75   0.75   0.84   0.67   0.08\n",
            "19020  SRP68                PF16969.10         64     3.11    565   0.59   0.60   0.50   0.04\n",
            "19021  SRP72                PF08492.17         21     2.80     58   0.96   0.92   0.91   0.17\n",
            "19022  SRP9-21              PF05486.17         90     3.32     89   0.64   0.64   0.55   0.05\n",
            "19023  SrpI-like            PF19307.4         114     0.78    254   0.59   0.60   0.53   0.01\n",
            "19024  SRPRB                PF09439.15          6     1.05    178   0.59   0.62   0.52   0.03\n",
            "19025  SRP_SPB              PF02978.24        668     2.25     91   0.63   0.62   0.49   0.05\n",
            "19026  SRP_TPR_like         PF17004.10          9     0.92    109   0.59   0.58   0.52   0.06\n",
            "19027  SRR                  PF07709.16         14    14.00     14   1.43   1.38   1.43   0.26\n",
            "19028  SRR1                 PF07985.17        125     6.08    172   0.59   0.57   0.42   0.02\n",
            "19029  SRRM_C               PF15230.11          9     1.09     67   0.84   0.81   0.78   0.18\n",
            "19030  SRTM1                PF15872.10          3     0.55    103   0.59   0.63   0.51   0.05\n",
            "19031  SRX                  PF09201.15         11     0.85    148   0.59   0.57   0.52   0.04\n",
            "19032  SR_plectin_7         PF21097.2          15     0.91    116   0.59   0.62   0.54   0.06\n",
            "19033  SSB                  PF00436.30         46     4.19    104   0.59   0.59   0.53   0.03\n",
            "19034  Ssb-like_OB          PF21473.2          47     1.13     97   0.59   0.61   0.52   0.03\n",
            "19035  SSB_1                PF22657.1         242     2.99     91   0.63   0.65   0.59   0.03\n",
            "19036  ssDBP                PF17878.6           1     0.50     72   0.78   0.76   0.72   0.02\n",
            "19037  ssDBP_DBD            PF18361.6           1     0.37    136   0.60   0.59   0.52   0.01\n",
            "19038  ssDNA-exonuc_C       PF10141.14         31     2.58    202   0.59   0.57   0.53   0.04\n",
            "19039  ssDNA_DBD            PF18333.6           5     1.09    106   0.59   0.61   0.54   0.07\n",
            "19040  ssDNA_TraI_N         PF18272.6           2     1.16     53   1.05   1.11   1.00   0.06\n",
            "19041  SSDP                 PF04503.18         14     0.99    291   0.59   0.57   0.42   0.07\n",
            "19042  SseB                 PF07179.17        113     9.52    124   0.59   0.62   0.45   0.02\n",
            "19043  SseB_C               PF14581.11         41     4.00    106   0.59   0.59   0.52   0.02\n",
            "19044  SseC                 PF04888.17         17     4.84    312   0.59   0.64   0.48   0.09\n",
            "19045  SSF                  PF00474.22         10     1.53    406   0.59   0.62   0.53   0.06\n",
            "19046  SSFA2_C              PF14723.11          4     0.69    176   0.59   0.59   0.51   0.03\n",
            "19047  SsfX3_N              PF21181.2           2     0.52    130   0.59   0.60   0.51   0.01\n",
            "19048  SsgA                 PF04686.17         83     1.97     98   0.59   0.60   0.53   0.01\n",
            "19049  SSI                  PF00720.22         21     1.16     92   0.62   0.58   0.56   0.03\n",
            "19050  Ssl1                 PF04056.19          6     0.82    193   0.59   0.60   0.53   0.02\n",
            "19051  SSL_N                PF20067.4          65     8.76     46   1.20   1.24   1.18   0.04\n",
            "19052  SSL_OB               PF09199.15         11     1.52     84   0.68   0.66   0.63   0.05\n",
            "19053  SSO1393-like_WHD     PF22165.1           2     0.71     59   0.95   0.94   0.89   0.04\n",
            "19054  Ssol_1539-like_2nd   PF22445.1           2     1.04     45   1.22   1.28   1.18   0.09\n",
            "19055  SSP160               PF06933.16          2     0.46    758   0.59   0.60   0.52   0.04\n",
            "19056  SspB                 PF04386.18         89     2.59    156   0.59   0.59   0.47   0.03\n",
            "19057  SspH                 PF08141.17         41     4.67     58   0.96   0.96   0.93   0.08\n",
            "19058  SSPI                 PF14098.11         22     1.62     65   0.86   0.88   0.82   0.04\n",
            "19059  SspK                 PF08176.17         12     1.94     47   1.17   1.13   1.12   0.13\n",
            "19060  SspN                 PF08177.16          9     2.02     46   1.20   1.13   1.16   0.15\n",
            "19061  SspO                 PF08175.17          5     1.37     48   1.15   1.11   1.08   0.18\n",
            "19062  SspP                 PF08179.17          6     2.17     44   1.25   1.19   1.20   0.24\n",
            "19063  SSrecog              PF03531.19         85     3.29     69   0.81   0.79   0.75   0.03\n",
            "19064  SSRP1_C              PF21092.2           7     1.64     49   1.13   1.19   1.06   0.30\n",
            "19065  SSSPR-51             PF18877.5           9     3.20     48   1.15   1.12   1.11   0.11\n",
            "19066  SSTK-IP              PF15836.10          5     0.57    124   0.59   0.59   0.52   0.03\n",
            "19067  Ssu72                PF04722.18        100     1.27    189   0.59   0.56   0.52   0.02\n",
            "19068  SSURE                PF11966.13          5     0.53    149   0.59   0.60   0.52   0.02\n",
            "19069  SSV1_D-63            PF20940.2           6     1.64     60   0.93   0.97   0.89   0.10\n",
            "19070  SSV1_ORF_D-335       PF07935.16          9     0.97     68   0.83   0.81   0.74   0.05\n",
            "19071  SSXRD                PF09514.15         12     5.25     31   1.74   1.71   1.71   0.24\n",
            "19072  SSXT                 PF05030.17         20     2.35     62   0.90   0.90   0.87   0.08\n",
            "19073  ST7                  PF04184.17          6     0.61    533   0.59   0.57   0.52   0.02\n",
            "19074  STAC2_u1             PF16664.10         15     1.12    126   0.59   0.60   0.50   0.04\n",
            "19075  STAG                 PF08514.16         97     3.12    114   0.59   0.58   0.54   0.05\n",
            "19076  STALD                PF18185.6          17     1.42    209   0.59   0.61   0.53   0.02\n",
            "19077  Stanniocalcin        PF03298.18          8     0.71    200   0.59   0.56   0.52   0.04\n",
            "19078  Staphopain_pro       PF14731.11          3     0.64    169   0.59   0.59   0.52   0.04\n",
            "19079  Staphostatin_A       PF09022.15          2     0.57    105   0.59   0.57   0.53   0.03\n",
            "19080  Staphostatin_B       PF09023.15          1     0.39    105   0.59   0.56   0.52   0.04\n",
            "19081  Staphylcoagulse      PF04022.17          2     1.86     27   1.98   1.91   1.95   0.28\n",
            "19082  Staphylokinase       PF02821.21          6     2.29    131   0.59   0.59   0.52   0.04\n",
            "19083  Staph_opine_DH       PF10100.14          8     0.90    424   0.59   0.59   0.54   0.02\n",
            "19084  Staph_reg_Sar_Rot    PF22381.1          20     3.14     85   0.67   0.68   0.61   0.07\n",
            "19085  Stap_Strp_toxin      PF01123.25          8     2.25     87   0.65   0.62   0.58   0.05\n",
            "19086  Stap_Strp_tox_C      PF02876.22         27     2.30     96   0.60   0.59   0.51   0.06\n",
            "19087  START                PF01852.24         16     5.35    207   0.59   0.55   0.51   0.02\n",
            "19088  START_2              PF19569.4           8     0.49    126   0.59   0.56   0.52   0.04\n",
            "19089  STAR_dimer           PF16544.10         10     2.36     53   1.05   1.05   1.01   0.07\n",
            "19090  STAS                 PF01740.26         67     9.32    106   0.59   0.62   0.53   0.02\n",
            "19091  STAS_2               PF13466.11         46    10.78     80   0.71   0.80   0.67   0.06\n",
            "19092  STAT1_TAZ2bind       PF12162.13          4     3.99     25   2.13   2.12   2.10   0.22\n",
            "19093  STAT2_C              PF12188.13          6     1.54     56   0.99   0.99   0.95   0.08\n",
            "19094  STAT6_C              PF14596.11          6     0.59    193   0.59   0.57   0.51   0.02\n",
            "19095  STATa_Ig             PF18214.6           6     1.20    122   0.59   0.60   0.52   0.04\n",
            "19096  Statherin            PF03875.18          2     1.02     41   1.33   1.23   1.26   0.23\n",
            "19097  Stathmin             PF00836.24         21     1.67    136   0.59   0.61   0.54   0.10\n",
            "19098  STAT_alpha           PF01017.25         51     3.16    167   0.59   0.57   0.53   0.07\n",
            "19099  STAT_bind            PF02864.20         44     2.26    133   0.59   0.59   0.49   0.04\n",
            "19100  STAT_int             PF02865.22         45     2.09    121   0.59   0.56   0.52   0.05\n",
            "19101  STAT_linker          PF21354.2          68     2.64     78   0.72   0.63   0.66   0.08\n",
            "19102  Staufen_C            PF16482.10          5     0.79    111   0.59   0.61   0.51   0.02\n",
            "19103  Staygreen            PF12638.12         30     1.38    147   0.59   0.57   0.52   0.02\n",
            "19104  Stb3                 PF10330.14         36     0.92     90   0.63   0.62   0.56   0.02\n",
            "19105  StbA_N               PF06406.16          6     0.72    158   0.59   0.58   0.52   0.02\n",
            "19106  STb_secrete          PF09075.15          2     0.78     48   1.15   1.11   1.09   0.10\n",
            "19107  Stc1                 PF12898.12         55     2.23     84   0.68   0.52   0.60   0.16\n",
            "19108  StcE_b-sandwich      PF20944.2          57     4.96     75   0.75   0.71   0.70   0.05\n",
            "19109  STD1                 PF17235.7          38     2.77    208   0.59   0.58   0.42   0.04\n",
            "19110  STE                  PF02200.21         18     0.61    109   0.59   0.56   0.53   0.03\n",
            "19111  STE2                 PF02116.20         84     3.07    280   0.59   0.59   0.54   0.08\n",
            "19112  STE3                 PF02076.20        171     3.70    292   0.59   0.53   0.53   0.06\n",
            "19113  Ste5                 PF11610.13          1     1.00     30   1.38   1.39   1.34   0.15\n",
            "19114  Ste5_C               PF12194.13         13     1.03    189   0.59   0.59   0.53   0.04\n",
            "19115  SteA-like_C          PF12555.13         80    11.96     52   1.07   1.14   1.04   0.14\n",
            "19116  Stealth_CR1          PF17101.10        104    54.51     29   1.85   1.74   1.86   0.15\n",
            "19117  Stealth_CR2          PF11380.13         90     1.15    108   0.59   0.54   0.53   0.03\n",
            "19118  Stealth_CR3          PF17102.10         95    13.90     49   1.13   1.03   1.11   0.11\n",
            "19119  Stealth_CR4          PF17103.10         42     5.65     55   1.01   0.95   0.92   0.07\n",
            "19120  STELLO               PF03385.22         13     0.56    387   0.59   0.54   0.52   0.02\n",
            "19121  Stereocilin          PF21058.2          17     1.19    397   0.59   0.61   0.54   0.03\n",
            "19122  Sterile              PF03015.24        283     4.43     93   0.61   0.55   0.58   0.04\n",
            "19123  Steroid_dh           PF02544.21         10     1.29    150   0.59   0.53   0.53   0.05\n",
            "19124  Sterol-sensing       PF12349.13         20     2.11    153   0.59   0.60   0.55   0.06\n",
            "19125  Sterol_MT_C          PF08498.15        115     3.16     66   0.85   0.86   0.82   0.04\n",
            "19126  Stevor               PF17410.7          19     0.90    275   0.59   0.57   0.52   0.03\n",
            "19127  STG                  PF15809.10          6     0.71    236   0.59   0.57   0.52   0.04\n",
            "19128  STI1-HOP_DP          PF17830.6          39     8.31     55   1.01   0.98   0.98   0.09\n",
            "19129  Stig1                PF04885.18         39     1.12    144   0.59   0.45   0.50   0.15\n",
            "19130  STIL_N               PF15253.11         11     0.80    404   0.59   0.59   0.53   0.01\n",
            "19131  STIMATE              PF12400.13        101     1.62    124   0.59   0.54   0.51   0.07\n",
            "19132  Stimulus_sens_1      PF13756.11         23     2.98    111   0.59   0.61   0.51   0.02\n",
            "19133  StiP                 PF11202.13         50     1.03    245   0.59   0.60   0.52   0.02\n",
            "19134  Stirrup              PF09061.11          2     0.53     79   0.71   0.71   0.65   0.03\n",
            "19135  STIV-like_capsid     PF22405.1           1     0.39    148   0.59   0.59   0.52   0.02\n",
            "19136  STIV_B116-like       PF08960.15          9     1.35    101   0.59   0.60   0.50   0.04\n",
            "19137  STIV_penton_2nd      PF21881.1           1     0.47     85   0.67   0.69   0.60   0.06\n",
            "19138  STIV_turret_1st      PF21878.1           1     0.39    134   0.59   0.59   0.52   0.03\n",
            "19139  STIV_turret_2nd      PF21876.1           1     0.39    106   0.59   0.59   0.52   0.03\n",
            "19140  STIV_turret_3rd      PF21879.1           1     0.42    111   0.59   0.60   0.52   0.04\n",
            "19141  Stk19                PF10494.14         46     2.35    278   0.59   0.60   0.46   0.01\n",
            "19142  STK_08120-like       PF11485.13         10     1.50    136   0.59   0.59   0.52   0.03\n",
            "19143  STL11_N              PF21369.2          10     0.60    117   0.59   0.53   0.51   0.05\n",
            "19144  Stm1_N               PF09598.15         64     5.60     70   0.80   0.84   0.69   0.10\n",
            "19145  STN                  PF07660.19        145   145.00     52   1.02   1.10   0.98   0.06\n",
            "19146  Stn1                 PF10451.14          5     1.57    252   0.59   0.56   0.51   0.02\n",
            "19147  STN1_2               PF09170.15         19     1.01    167   0.59   0.58   0.53   0.04\n",
            "19148  Stn1_C               PF12659.12         12     2.33    165   0.59   0.59   0.53   0.05\n",
            "19149  stn_TNFRSF12A        PF12191.13          2     0.39    129   0.59   0.59   0.51   0.03\n",
            "19150  Stomagen             PF16851.10          7     0.95     50   1.10   0.94   1.05   0.15\n",
            "19151  Stomoxyn             PF11585.13          1     1.00     40   1.30   1.37   1.25   0.17\n",
            "19152  Stonin2_N            PF12016.13          4     0.59    338   0.59   0.58   0.52   0.04\n",
            "19153  Stonustoxin_helical  PF21109.2          28     1.51     96   0.60   0.58   0.56   0.05\n",
            "19154  Stork_head           PF10264.14         14     1.36     79   0.72   0.70   0.68   0.03\n",
            "19155  STPPase_N            PF16891.10        189     9.33     48   1.15   1.22   1.10   0.09\n",
            "19156  STPRs                PF21107.2           3     0.99     74   0.76   0.77   0.71   0.15\n",
            "19157  Strabismus           PF06638.16         15     0.79    503   0.59   0.58   0.52   0.02\n",
            "19158  Strep_his_triad      PF04270.18         11     2.35     52   1.07   0.99   1.02   0.09\n",
            "19159  Strep_pep            PF14404.11          5     0.95     63   0.89   0.88   0.84   0.03\n",
            "19160  Strep_SA_rep         PF06696.16          8     8.00     25   1.97   2.01   1.94   0.49\n",
            "19161  Stress-antifung      PF01657.22        390     3.37     99   0.59   0.48   0.36   0.10\n",
            "19162  Striatin             PF08232.17         85     2.75    151   0.59   0.59   0.50   0.08\n",
            "19163  Strumpellin          PF10266.14         31     1.56   1084   0.59   0.58   0.53   0.03\n",
            "19164  Str_synth            PF03088.21          6     1.25     89   0.64   0.63   0.59   0.02\n",
            "19165  STT3                 PF02516.19         10     2.69    405   0.59   0.58   0.50   0.07\n",
            "19166  STT3-PglB_core       PF21436.2          20     1.09    131   0.59   0.54   0.52   0.03\n",
            "19167  STT3_PglB_C          PF18527.6           6     1.56     80   0.71   0.72   0.66   0.05\n",
            "19168  Stu2_CTS             PF21042.2          12     5.10     23   2.31   2.28   2.30   0.39\n",
            "19169  Stv                  PF21527.2          93     5.63    121   0.59   0.55   0.29   0.01\n",
            "19170  Styelin              PF17562.7           2     0.80     59   0.94   0.90   0.88   0.06\n",
            "19171  SUa-2TM              PF18179.6           3     0.96    279   0.59   0.57   0.51   0.04\n",
            "19172  Sua5_C               PF03481.18        478     4.41    136   0.59   0.61   0.40   0.03\n",
            "19173  Sua5_yciO_yrdC       PF01300.23         97     4.28    178   0.59   0.61   0.52   0.01\n",
            "19174  SUB1_ProdP9          PF18213.6           5     0.61     80   0.71   0.76   0.65   0.04\n",
            "19175  Sublancin            PF19151.5           2     0.77     57   0.98   0.92   0.92   0.11\n",
            "19176  Subtilosin_A         PF11420.13          2     1.14     35   1.55   1.51   1.49   0.20\n",
            "19177  Succ_CoA_lig         PF13607.11        122     3.93    138   0.59   0.63   0.56   0.03\n",
            "19178  Succ_DH_flav_C       PF02910.25        541     3.02    130   0.59   0.57   0.52   0.04\n",
            "19179  Sucrose_synth        PF00862.24         19     0.72    549   0.59   0.59   0.53   0.01\n",
            "19180  Suc_Fer-like         PF06999.17        238     2.81    191   0.59   0.55   0.38   0.02\n",
            "19181  Suc_Porlyase_C       PF09244.15          4     0.79     67   0.84   0.81   0.77   0.04\n",
            "19182  Suf                  PF05843.19         39     1.56    532   0.59   0.54   0.52   0.04\n",
            "19183  SUFBD                PF01458.22        696     5.79    218   0.59   0.60   0.52   0.03\n",
            "19184  SufBD_N              PF19295.4          79     1.55    172   0.59   0.57   0.55   0.03\n",
            "19185  SufE                 PF02657.20        280     3.02    121   0.59   0.59   0.54   0.02\n",
            "19186  SUFU                 PF05076.18         93     6.37    172   0.59   0.57   0.44   0.01\n",
            "19187  SUFU_C               PF12470.13         14     1.26    200   0.59   0.59   0.51   0.02\n",
            "19188  Sugar-bind           PF04198.18         20     3.11    257   0.59   0.62   0.55   0.02\n",
            "19189  Sugarporin_N         PF11471.13          7     7.00     31   1.64   1.79   1.63   0.40\n",
            "19190  Sugar_tr             PF00083.29         33     4.36    452   0.59   0.58   0.53   0.05\n",
            "19191  Sugar_transport      PF06800.17         13     1.01    280   0.59   0.61   0.53   0.06\n",
            "19192  SUI1                 PF01253.27        308     6.37     77   0.73   0.76   0.65   0.06\n",
            "19193  SUIM_assoc           PF16619.10         13     1.74     60   0.93   0.92   0.84   0.12\n",
            "19194  SUKH-3               PF14433.11         32     3.28    146   0.59   0.59   0.50   0.01\n",
            "19195  SUKH-4               PF14435.11         23     5.06    179   0.59   0.60   0.50   0.02\n",
            "19196  SUKH_5               PF14567.11         27     3.80    142   0.59   0.55   0.50   0.02\n",
            "19197  SUKH_6               PF14568.11         45     8.16    119   0.59   0.56   0.51   0.03\n",
            "19198  SulA                 PF03846.19          7     0.55     91   0.63   0.62   0.56   0.03\n",
            "19199  Sulfakinin           PF08257.16          5     5.00      9   3.34   3.04   3.32   1.00\n",
            "19200  Sulfatase            PF00884.28         57     9.29    301   0.59   0.55   0.47   0.02\n",
            "19201  Sulfatase_C          PF14707.11         75     3.03    127   0.59   0.51   0.48   0.04\n",
            "19202  Sulfate_transp       PF00916.25         22     2.85    380   0.59   0.62   0.53   0.05\n",
            "19203  Sulfolobus_pRN       PF05584.16          2     0.66     72   0.78   0.78   0.72   0.06\n",
            "19204  Sulfotransfer_1      PF00685.32         45     4.74    255   0.59   0.55   0.48   0.02\n",
            "19205  Sulfotransfer_2      PF03567.19         60     5.99    252   0.59   0.53   0.43   0.04\n",
            "19206  Sulfotransfer_3      PF13469.11        139     7.29    216   0.59   0.56   0.48   0.01\n",
            "19207  Sulfotransfer_4      PF17784.6         210     3.98    215   0.59   0.55   0.51   0.01\n",
            "19208  Sulfotransfer_5      PF19798.4           3     0.42    226   0.59   0.57   0.52   0.01\n",
            "19209  Sulf_coat_C          PF12193.13          8     0.70    134   0.59   0.58   0.51   0.03\n",
            "19210  Sulf_transp          PF04143.19         68     4.37    310   0.59   0.60   0.52   0.06\n",
            "19211  Sulphotransf         PF09037.15          6     0.85    244   0.59   0.57   0.52   0.02\n",
            "19212  SUN                  PF03856.18        161     1.10    245   0.59   0.51   0.49   0.04\n",
            "19213  SupH-like_C          PF22157.1          12     2.95     58   0.96   0.96   0.90   0.03\n",
            "19214  Suppressor_APC       PF11414.13         20     2.36     82   0.69   0.68   0.66   0.07\n",
            "19215  Suppressor_P21       PF11479.13          6     2.00     80   0.71   0.71   0.65   0.06\n",
            "19216  SUR7                 PF06687.17         93     6.50    201   0.59   0.57   0.46   0.07\n",
            "19217  SurA_N               PF09312.16         11     0.99    118   0.59   0.62   0.54   0.05\n",
            "19218  SurA_N_2             PF13623.11        137     4.80    145   0.59   0.60   0.51   0.04\n",
            "19219  SurA_N_3             PF13624.11        119     4.76    162   0.59   0.60   0.52   0.04\n",
            "19220  SurE                 PF01975.22        581     2.48    189   0.59   0.61   0.46   0.02\n",
            "19221  SURF1                PF02104.20        399     5.79    205   0.59   0.57   0.32   0.02\n",
            "19222  SURF2                PF05477.16         20     1.58    242   0.59   0.56   0.52   0.07\n",
            "19223  SURF4                PF02077.20          6     0.79    267   0.59   0.59   0.53   0.04\n",
            "19224  SURF6                PF04935.17        116     3.20    196   0.59   0.59   0.52   0.17\n",
            "19225  Surface-like_ins_dom PF22343.1           4     0.82    100   0.59   0.55   0.53   0.04\n",
            "19226  Surface_Ag_2         PF01617.22         10     1.18    247   0.59   0.57   0.51   0.03\n",
            "19227  Surface_antigen      PF11054.13        188     2.10    211   0.59   0.54   0.46   0.05\n",
            "19228  Surfac_D-trimer      PF09006.16          5     1.40     46   1.20   1.23   1.16   0.05\n",
            "19229  SURNod19             PF07712.17         22     0.78    377   0.59   0.54   0.52   0.02\n",
            "19230  Surp                 PF01805.25        293    17.27     53   1.05   0.99   1.00   0.07\n",
            "19231  SusD-like            PF12741.12         19     1.35    495   0.59   0.56   0.52   0.03\n",
            "19232  SusD-like_2          PF12771.12         14     1.65    415   0.59   0.55   0.46   0.03\n",
            "19233  SusD-like_3          PF14322.11         10     2.34    191   0.59   0.58   0.47   0.03\n",
            "19234  SusD_RagB            PF07980.16         33     6.65    297   0.59   0.54   0.47   0.04\n",
            "19235  SusE                 PF14292.11         63     8.41    106   0.59   0.61   0.49   0.05\n",
            "19236  SusF_SusE            PF16411.10         29     3.48    167   0.59   0.54   0.35   0.04\n",
            "19237  SusG_CBM58           PF20756.2           5     0.76    117   0.59   0.56   0.51   0.03\n",
            "19238  Sushi                PF00084.25         29     7.91     56   0.99   0.80   0.88   0.14\n",
            "19239  Sushi_2              PF09014.15         22     0.95     88   0.65   0.55   0.59   0.06\n",
            "19240  SutA-RBD             PF20661.2         203   117.14     35   1.55   1.64   1.55   0.13\n",
            "19241  SUV3_C               PF12513.13        146    16.44     48   1.15   1.13   1.13   0.10\n",
            "19242  Suv3_C_1             PF18147.6          30    13.00     43   1.28   1.29   1.26   0.08\n",
            "19243  Suv3_N               PF18114.6           8     0.89    116   0.59   0.58   0.53   0.03\n",
            "19244  SUZ                  PF12752.12        130    12.92     56   0.99   1.01   0.92   0.14\n",
            "19245  SUZ-C                PF12901.12         16    16.00     25   2.09   2.06   2.05   0.27\n",
            "19246  SVA                  PF05326.16         10     1.10    124   0.59   0.55   0.52   0.05\n",
            "19247  Svf1                 PF08622.15        106     2.17    163   0.59   0.55   0.47   0.03\n",
            "19248  Svf1_C               PF17187.9         106     1.70    163   0.59   0.55   0.50   0.03\n",
            "19249  SVIP                 PF15811.10         25     2.10     77   0.73   0.73   0.66   0.12\n",
            "19250  SVM_signal           PF12113.13          4     2.32     33   1.64   1.62   1.60   0.35\n",
            "19251  SVN                  PF20888.2           1     0.36     95   0.60   0.53   0.52   0.09\n",
            "19252  Svs_4_5_6            PF17381.7           6     1.41     91   0.63   0.64   0.54   0.11\n",
            "19253  SVS_QK               PF10578.14         11    11.00     12   2.35   2.28   2.34   0.75\n",
            "19254  SVWC                 PF15430.11         80     3.52     66   0.85   0.65   0.74   0.19\n",
            "19255  SwaI-like            PF22081.1           3     0.60    208   0.59   0.57   0.52   0.03\n",
            "19256  SWC7                 PF17330.7          12     0.91     98   0.59   0.60   0.53   0.04\n",
            "19257  SWI-SNF_Ssr4_C       PF20497.3          28     1.58    456   0.59   0.59   0.49   0.04\n",
            "19258  SWI-SNF_Ssr4_N       PF08549.15         28     0.83    217   0.59   0.54   0.52   0.02\n",
            "19259  SWI2_SNF2            PF18766.6         251     1.63    221   0.59   0.58   0.48   0.03\n",
            "19260  Swi3                 PF07962.17        109     3.88     83   0.68   0.66   0.63   0.03\n",
            "19261  Swi5                 PF07061.16         56     3.72     80   0.71   0.73   0.67   0.06\n",
            "19262  Swi6_N               PF18530.6           6     0.98    103   0.59   0.60   0.51   0.03\n",
            "19263  SWIB                 PF02201.23        317     6.22     74   0.76   0.75   0.73   0.04\n",
            "19264  SWIM                 PF04434.22         85    85.00     38   1.32   1.13   1.24   0.20\n",
            "19265  SWIRM                PF04433.22        211     6.57     84   0.68   0.63   0.55   0.03\n",
            "19266  SWIRM-assoc_1        PF16495.10         53     3.74     84   0.68   0.74   0.66   0.11\n",
            "19267  SWIRM-assoc_2        PF16496.10         22     0.73    134   0.59   0.57   0.53   0.02\n",
            "19268  SWIRM-assoc_3        PF16498.10         15     1.54     67   0.84   0.89   0.78   0.06\n",
            "19269  Swm2                 PF17083.10          9     1.33    130   0.59   0.58   0.52   0.05\n",
            "19270  SWM_repeat           PF13753.11         52     2.35     86   0.66   0.70   0.57   0.05\n",
            "19271  SwrA                 PF17423.7           1     0.39    116   0.59   0.59   0.52   0.04\n",
            "19272  SWT1_3rd             PF21693.2          20     1.32    117   0.59   0.54   0.54   0.05\n",
            "19273  SxtJ                 PF19588.4           2     0.40     99   0.59   0.58   0.52   0.05\n",
            "19274  Sybindin             PF04099.17         11     1.23    140   0.59   0.58   0.50   0.02\n",
            "19275  SYCE1                PF15233.11         13     1.44    152   0.59   0.61   0.54   0.10\n",
            "19276  SYCP2_ARLD           PF18581.6          18     1.27    175   0.59   0.59   0.54   0.05\n",
            "19277  SYCP2_SLD            PF18584.6          18     0.85    112   0.59   0.56   0.53   0.03\n",
            "19278  Syd                  PF07348.17         31     1.69    175   0.59   0.57   0.54   0.02\n",
            "19279  SYF2                 PF08231.17         87     2.49    154   0.59   0.55   0.49   0.11\n",
            "19280  Syja_N               PF02383.23        425     3.23    305   0.59   0.56   0.39   0.02\n",
            "19281  SymE_toxin           PF08845.15         22     4.95     54   1.03   1.01   0.94   0.03\n",
            "19282  SYMPK_PTA1_N         PF11935.13         89     4.39    229   0.59   0.59   0.49   0.04\n",
            "19283  Symplekin_C          PF12295.13         45     2.74    182   0.59   0.59   0.51   0.04\n",
            "19284  Synaphin             PF05835.17         21     1.74    142   0.59   0.60   0.51   0.09\n",
            "19285  Synapsin             PF02078.21         25     0.85    103   0.59   0.58   0.52   0.02\n",
            "19286  Synapsin_C           PF02750.19          2     0.41    203   0.60   0.57   0.52   0.02\n",
            "19287  Synapsin_N           PF10581.14          1     1.00     32   1.50   1.42   1.45   0.19\n",
            "19288  Synaptobrevin        PF00957.26         92     4.24     89   0.64   0.63   0.62   0.06\n",
            "19289  Synaptonemal_3       PF15191.11          2     0.68     92   0.62   0.61   0.55   0.05\n",
            "19290  Syncollin            PF15138.11          8     0.58    112   0.59   0.52   0.53   0.03\n",
            "19291  Syndecan             PF01034.25         12     1.46     64   0.88   0.90   0.80   0.04\n",
            "19292  Syndetin_C           PF10474.14         37     2.73    234   0.59   0.57   0.53   0.04\n",
            "19293  Syntaphilin          PF15290.11         18     0.86    305   0.59   0.59   0.51   0.04\n",
            "19294  Syntaxin             PF00804.30         61     2.16    199   0.59   0.60   0.54   0.07\n",
            "19295  Syntaxin-18_N        PF10496.14         64     6.14     87   0.65   0.65   0.54   0.07\n",
            "19296  Syntaxin-5_N         PF11416.13         27    27.00     23   2.12   2.10   2.12   0.23\n",
            "19297  Syntaxin-6_N         PF09177.16        129     4.63     99   0.59   0.60   0.52   0.06\n",
            "19298  Syntaxin_2           PF14523.11        118     7.39    101   0.59   0.60   0.54   0.10\n",
            "19299  Synthase_beta        PF11421.13         10     2.25     47   1.17   1.26   1.11   0.22\n",
            "19300  Syntrophin_4th       PF23012.1          66     1.51    100   0.59   0.56   0.52   0.02\n",
            "19301  Synuclein            PF01387.22         12     0.95    134   0.59   0.63   0.52   0.07\n",
            "19302  SyrA                 PF11089.13          3     2.15     38   1.43   1.47   1.41   0.30\n",
            "19303  SYS1                 PF09801.14         25     1.49    144   0.59   0.55   0.54   0.07\n",
            "19304  SYY_C-terminal       PF22421.1         133     1.73     83   0.68   0.71   0.64   0.04\n",
            "19305  S_100                PF01023.24        149    14.16     45   1.22   1.24   1.21   0.08\n",
            "19306  S_2TMBeta            PF18153.6          13     1.84    181   0.59   0.55   0.51   0.03\n",
            "19307  S_4TM                PF18159.6          21     3.20    291   0.59   0.57   0.55   0.03\n",
            "19308  S_layer_C            PF05124.17         11     1.12    177   0.59   0.64   0.46   0.04\n",
            "19309  S_layer_N            PF05123.17          6     1.23    359   0.59   0.60   0.48   0.04\n",
            "19310  S_locus_glycop       PF00954.25         27     1.17    111   0.59   0.47   0.51   0.05\n",
            "19311  S_tail_recep_bd      PF14928.11          1     0.39     93   0.62   0.58   0.54   0.02\n",
            "19312  T-box                PF00907.27         87     1.45    182   0.59   0.53   0.51   0.03\n",
            "19313  T-box_assoc          PF16176.10         13     0.75    226   0.59   0.57   0.47   0.03\n",
            "19314  T1RH-like_C          PF11867.13         83     2.77    329   0.59   0.61   0.53   0.05\n",
            "19315  T26-6p_Ig-like_dom   PF22092.1           2     0.58    137   0.59   0.56   0.51   0.03\n",
            "19316  T26-6p_Ig-like_dom_2 PF22265.1           4     0.73    130   0.59   0.57   0.52   0.03\n",
            "19317  T2SS-T3SS_pil_N      PF13629.11         69     8.04     72   0.78   0.85   0.76   0.05\n",
            "19318  T2SSB                PF16537.10         37     5.14     60   0.93   0.94   0.90   0.03\n",
            "19319  T2SSC                PF11356.13         23     3.09    141   0.59   0.63   0.50   0.02\n",
            "19320  T2SSE                PF00437.25         17     1.57    269   0.59   0.62   0.51   0.03\n",
            "19321  T2SSF                PF00482.28        231    22.05    125   0.59   0.65   0.55   0.04\n",
            "19322  T2SSG                PF08334.16         70     1.53    108   0.59   0.57   0.51   0.02\n",
            "19323  T2SSI                PF02501.22         80     6.86     80   0.71   0.71   0.66   0.05\n",
            "19324  T2SSJ                PF11612.13         65     4.51    140   0.59   0.59   0.49   0.03\n",
            "19325  T2SSK                PF03934.18         98    15.99     63   0.89   0.96   0.81   0.06\n",
            "19326  T2SSK_1st            PF21687.2          92     4.61    107   0.59   0.61   0.51   0.02\n",
            "19327  T2SSL                PF05134.18          8     1.71    231   0.59   0.59   0.53   0.02\n",
            "19328  T2SSM                PF04612.17         23     3.66    160   0.59   0.60   0.55   0.04\n",
            "19329  T2SSM_b              PF10741.14         16     3.43    111   0.59   0.64   0.54   0.03\n",
            "19330  T2SSN                PF01203.24         43     6.66    211   0.59   0.62   0.49   0.03\n",
            "19331  T2SSppdC             PF12528.13         12     2.88     79   0.72   0.61   0.66   0.08\n",
            "19332  T2SSS_2              PF16549.10          9     0.95    104   0.59   0.58   0.53   0.03\n",
            "19333  T2SS_PulS_OutS       PF09691.15          3     0.79    110   0.59   0.58   0.52   0.03\n",
            "19334  T3RM_EcoP15I_C       PF18273.6          23     1.55     97   0.59   0.58   0.54   0.04\n",
            "19335  T3SchapCesA          PF11439.13          2     0.46     95   0.60   0.63   0.53   0.04\n",
            "19336  T3SSipB              PF16535.10          4     1.16    155   0.59   0.63   0.53   0.07\n",
            "19337  T3SS_ATPase_C        PF18269.6         202     5.55     72   0.78   0.79   0.77   0.04\n",
            "19338  T3SS_basalb_I        PF17001.10         11     2.48     94   0.61   0.63   0.55   0.06\n",
            "19339  T3SS_CopN_3rd        PF22342.1           3     0.76     99   0.59   0.60   0.52   0.02\n",
            "19340  T3SS_ExsE            PF18286.6           2     1.49     46   1.20   1.26   1.14   0.07\n",
            "19341  T3SS_HrpK1           PF16937.10          4     0.82    251   0.59   0.62   0.52   0.03\n",
            "19342  T3SS_LEE_assoc       PF13327.11          5     1.01    162   0.59   0.58   0.53   0.03\n",
            "19343  T3SS_needle_E        PF08988.15          6     2.75     67   0.84   0.87   0.80   0.08\n",
            "19344  T3SS_needle_F        PF09392.15         17     3.13     71   0.79   0.80   0.76   0.07\n",
            "19345  T3SS_needle_reg      PF09025.15          3     0.82    145   0.59   0.61   0.51   0.02\n",
            "19346  T3SS_NleG            PF06416.17         12     0.89    113   0.59   0.55   0.53   0.03\n",
            "19347  T3SS_SCTL            PF06635.17          1     0.39    206   0.59   0.61   0.51   0.01\n",
            "19348  T3SS_TC              PF06511.16          3     0.71    355   0.59   0.60   0.51   0.02\n",
            "19349  T3S_SPI-1_N0         PF21304.2          12     0.94     69   0.81   0.81   0.77   0.02\n",
            "19350  T4-Gluco-transf      PF09198.15          2     0.49    167   0.59   0.58   0.52   0.03\n",
            "19351  T4-gp15_tss          PF16724.10          2     0.54    238   0.59   0.57   0.51   0.02\n",
            "19352  T4BSS_DotH_IcmK      PF12293.13          6     1.08    238   0.59   0.60   0.53   0.01\n",
            "19353  T4BSS_DotI_IcmL      PF11393.13          7     1.40    180   0.59   0.57   0.54   0.02\n",
            "19354  T4bSS_IcmS           PF12608.13          3     1.09     93   0.61   0.65   0.53   0.03\n",
            "19355  T4P_PilY1            PF05567.16          2     0.61    411   0.59   0.61   0.50   0.02\n",
            "19356  T4SS                 PF07996.16         23     3.63    197   0.59   0.61   0.51   0.07\n",
            "19357  T4SS-DNA_transf      PF02534.19          9     2.27    468   0.59   0.59   0.51   0.02\n",
            "19358  T4SS_CagC            PF16943.10          2     0.66    119   0.59   0.62   0.50   0.04\n",
            "19359  T4SS_pilin           PF18895.5          22    10.94     71   0.79   0.83   0.71   0.12\n",
            "19360  T4SS_TraI            PF16932.10          7     1.12    213   0.59   0.58   0.53   0.02\n",
            "19361  T4_baseplate         PF12322.13         18     2.77    205   0.59   0.57   0.51   0.04\n",
            "19362  T4_deiodinase        PF00837.22          3     0.69    237   0.59   0.57   0.52   0.01\n",
            "19363  T4_Gp59_C            PF08994.15          2     0.56    109   0.59   0.58   0.51   0.05\n",
            "19364  T4_Gp59_N            PF08993.15          2     0.55     93   0.61   0.59   0.55   0.04\n",
            "19365  T4_gp9_10            PF07880.16          3     0.89    285   0.59   0.59   0.50   0.02\n",
            "19366  T4_neck-protein      PF11649.13          2     0.55    278   0.59   0.57   0.52   0.02\n",
            "19367  T4_Rnl1_C            PF20819.2          28     1.42    122   0.59   0.57   0.54   0.05\n",
            "19368  T4_Rnl2_C            PF18043.6          14     3.48     82   0.69   0.70   0.64   0.06\n",
            "19369  T4_tail_cap          PF11091.13          4     0.89    339   0.59   0.59   0.51   0.02\n",
            "19370  T4_UVSX_C            PF21134.2           8     1.00     65   0.86   0.83   0.81   0.03\n",
            "19371  T5orf172             PF10544.14         91     7.61    101   0.59   0.54   0.43   0.03\n",
            "19372  T6PP_C               PF21141.2          21     1.16    267   0.59   0.58   0.53   0.01\n",
            "19373  T6PP_N               PF18572.6          28     2.25     99   0.59   0.60   0.53   0.05\n",
            "19374  T6SS-SciN            PF12790.12         60     3.91    120   0.59   0.61   0.54   0.02\n",
            "19375  T6SS_FHA_C           PF20232.3          52     2.82    180   0.59   0.58   0.55   0.03\n",
            "19376  T6SS_HCP             PF05638.17        122     3.94    129   0.59   0.57   0.48   0.03\n",
            "19377  T6SS_Tdi1_C          PF08906.16         19     4.09     75   0.75   0.76   0.72   0.03\n",
            "19378  T6SS_Tgi2PP          PF21576.2           1     0.39    102   0.59   0.60   0.52   0.02\n",
            "19379  T6SS_Tle1-like_C     PF22137.1           7     1.06    353   0.59   0.58   0.51   0.02\n",
            "19380  T6SS_Tle1-like_cat   PF09994.14        283     2.43    287   0.59   0.56   0.47   0.02\n",
            "19381  T6SS_Tse3_cat        PF22115.1           1     0.41    189   0.59   0.62   0.52   0.01\n",
            "19382  T6SS_Tse3_N          PF22120.1           4     0.96    104   0.59   0.62   0.51   0.03\n",
            "19383  T6SS_Tsi2-like       PF21643.2           2     0.69     71   0.79   0.81   0.74   0.04\n",
            "19384  T6SS_TssF            PF05947.17         77     3.49    608   0.59   0.59   0.52   0.01\n",
            "19385  T6SS_TssG            PF06996.16        105     4.76    304   0.59   0.60   0.50   0.01\n",
            "19386  T6SS_VasE            PF05936.17        101     4.19    429   0.59   0.60   0.55   0.02\n",
            "19387  T6SS_VasJ            PF16989.10         28     3.59    255   0.59   0.59   0.54   0.04\n",
            "19388  T6SS_Vgr             PF13296.11         55     2.08    108   0.59   0.63   0.54   0.05\n",
            "19389  T6SS_VgrG3-like_C    PF21277.2           8     0.99    183   0.59   0.58   0.51   0.03\n",
            "19390  T6SS_VipA            PF05591.17         84     1.66    155   0.59   0.62   0.54   0.04\n",
            "19391  T6_Ig_like           PF18002.6           3     0.59    145   0.59   0.60   0.52   0.04\n",
            "19392  T7-like_gp12         PF10922.13          1     0.41     85   0.67   0.61   0.60   0.04\n",
            "19393  T7-like_gp67         PF17570.7          16     1.98     61   0.92   0.91   0.84   0.07\n",
            "19394  T7-like_Y65          PF10911.13          4     0.84     77   0.74   0.74   0.68   0.02\n",
            "19395  T7SS_EccA1_N         PF21545.2          34     2.22    272   0.59   0.59   0.53   0.03\n",
            "19396  T7SS_ESX1_EccB       PF05108.18         44     2.18    474   0.59   0.61   0.51   0.01\n",
            "19397  T7SS_ESX_EspC        PF10824.13         17     4.14    100   0.59   0.62   0.56   0.06\n",
            "19398  T7SS_signal          PF21725.2          61     1.34    241   0.59   0.60   0.54   0.05\n",
            "19399  T9SS_plug_1st        PF17116.10         65     1.54    126   0.59   0.55   0.53   0.03\n",
            "19400  Ta0938               PF11494.13         10     0.63     92   0.62   0.54   0.56   0.04\n",
            "19401  TA0956               PF11513.13          3     0.57    110   0.59   0.58   0.52   0.03\n",
            "19402  Ta1207               PF19020.5          26     7.43    277   0.59   0.58   0.56   0.04\n",
            "19403  TAA-Trp-ring         PF15401.11         20     5.08     66   0.85   0.91   0.73   0.15\n",
            "19404  Tab2-like_C          PF20429.3          69     1.76    151   0.59   0.58   0.52   0.02\n",
            "19405  Tab2-like_N          PF06485.16         60     1.49    103   0.59   0.55   0.54   0.04\n",
            "19406  TACC_C               PF05010.19         21     2.02    201   0.59   0.59   0.54   0.10\n",
            "19407  Tachykinin           PF02202.21         14    14.00     11   2.82   2.62   2.82   0.60\n",
            "19408  Tachylectin          PF14517.11          5     0.96    230   0.59   0.53   0.49   0.03\n",
            "19409  Tachystatin_A        PF11406.13          2     0.71     44   1.24   1.09   1.19   0.19\n",
            "19410  Tachystatin_B        PF11478.13          2     0.73     42   1.30   1.13   1.25   0.25\n",
            "19411  TACI-CRD2            PF09305.15         12     1.53     39   1.40   1.15   1.36   0.27\n",
            "19412  TackOD1              PF18551.6          18     1.36    188   0.59   0.49   0.54   0.06\n",
            "19413  TACO1_YebC_N         PF20772.2         617     5.39     73   0.77   0.80   0.75   0.09\n",
            "19414  Tad                  PF13400.11        130    56.86     47   1.17   1.34   1.10   0.29\n",
            "19415  TAD2                 PF18521.6           1     1.00     25   1.52   1.46   1.47   0.19\n",
            "19416  TADA2A-like_3rd      PF22941.1         117     4.59     77   0.73   0.72   0.70   0.07\n",
            "19417  TadB_TadC_N          PF19360.4          82     0.67    143   0.59   0.61   0.53   0.04\n",
            "19418  TadE                 PF07811.17        292    58.41     43   1.28   1.39   1.27   0.21\n",
            "19419  TadF                 PF16964.10          7     1.27    178   0.59   0.60   0.51   0.03\n",
            "19420  TAdV_Fibre-like_head PF22439.1           1     0.39    135   0.59   0.59   0.52   0.03\n",
            "19421  TadZ-like_ARD        PF21194.2           8     0.68    108   0.59   0.58   0.53   0.04\n",
            "19422  TadZ_N               PF16968.10          1     0.41    133   0.59   0.64   0.51   0.02\n",
            "19423  Tad_C                PF09977.14         27     7.22     93   0.61   0.65   0.46   0.07\n",
            "19424  Tae4                 PF14113.11         17     2.24    125   0.59   0.55   0.41   0.03\n",
            "19425  Taeniidae_ag         PF05596.16         27     4.71     55   1.01   1.02   1.00   0.08\n",
            "19426  TAF                  PF02969.22          8     1.91     66   0.85   0.90   0.81   0.05\n",
            "19427  TAF1C_beta-prop      PF20641.2          33     0.89    138   0.59   0.55   0.53   0.02\n",
            "19428  TAF1C_C              PF20643.2          13     1.36     80   0.71   0.70   0.65   0.07\n",
            "19429  TAF1C_HB             PF20642.2          25     1.27    208   0.59   0.57   0.50   0.04\n",
            "19430  TAF1D                PF15333.11         10     0.73    215   0.59   0.58   0.51   0.03\n",
            "19431  TAF1_subA            PF14929.11         32     2.82    365   0.59   0.54   0.48   0.04\n",
            "19432  TAF4                 PF05236.19         61     3.41    262   0.59   0.61   0.49   0.06\n",
            "19433  TAF6_C               PF07571.18        103     2.59    184   0.59   0.60   0.53   0.03\n",
            "19434  TAF8_C               PF10406.14        106    10.50     49   1.13   1.08   1.12   0.10\n",
            "19435  TAFA                 PF12020.13          6     0.59     90   0.63   0.55   0.56   0.07\n",
            "19436  TAFH                 PF07531.19         26     1.92     90   0.63   0.65   0.59   0.05\n",
            "19437  Tafi-CsgC            PF10610.14          3     0.65     99   0.59   0.60   0.53   0.03\n",
            "19438  TAFII28              PF04719.19         74     2.45     86   0.66   0.66   0.61   0.04\n",
            "19439  TAFII55_N            PF04658.18        116     1.92    160   0.59   0.59   0.49   0.04\n",
            "19440  Tag1_C               PF22786.1          71     1.69    113   0.59   0.60   0.54   0.04\n",
            "19441  Tag1_M               PF22787.1          20     1.84    165   0.59   0.58   0.52   0.03\n",
            "19442  Tag1_N               PF20775.2          24     1.67    190   0.59   0.56   0.53   0.04\n",
            "19443  TagA                 PF12561.13         20     1.47     99   0.59   0.58   0.52   0.02\n",
            "19444  TagF_N               PF09867.14         38     2.27    135   0.59   0.57   0.54   0.01\n",
            "19445  TagH_C               PF22096.1          15     1.50    107   0.59   0.55   0.54   0.04\n",
            "19446  TagH_SH3-like        PF22269.1           6     1.43     62   0.90   0.90   0.85   0.05\n",
            "19447  TAGT                 PF20691.2          14     4.04    205   0.59   0.54   0.46   0.02\n",
            "19448  Tai4                 PF16695.10         12     1.77     92   0.62   0.59   0.57   0.04\n",
            "19449  Tailspike_C          PF21519.2           1     0.54     72   0.79   0.84   0.72   0.06\n",
            "19450  Tail_P2_I            PF09684.15         47     3.28    139   0.59   0.59   0.50   0.03\n",
            "19451  Tail_spike_N         PF18668.6           4     1.21     70   0.80   0.78   0.75   0.04\n",
            "19452  Tail_tube            PF10618.14         20     3.40    117   0.59   0.59   0.55   0.03\n",
            "19453  Takusan              PF04822.18         27     3.54     85   0.67   0.66   0.63   0.09\n",
            "19454  Tal-1A_2A            PF22856.1           1     1.00     11   1.52   1.44   1.47   0.26\n",
            "19455  Talin_IBS2B          PF21896.1          25     2.10    161   0.59   0.63   0.55   0.05\n",
            "19456  Talin_middle         PF09141.15         15     1.57    161   0.59   0.64   0.54   0.05\n",
            "19457  Talin_R4             PF21692.2          29     2.02    126   0.59   0.63   0.55   0.08\n",
            "19458  TALPID3              PF15324.11          7     0.93   1261   0.59   0.61   0.52   0.03\n",
            "19459  TAL_effector         PF03377.18          3     3.00     34   1.47   1.50   1.43   0.12\n",
            "19460  TAL_FSA              PF00923.24        121     4.26    285   0.59   0.61   0.47   0.03\n",
            "19461  Tal_Pri              PF21899.1           1     1.00     32   1.54   1.45   1.49   0.24\n",
            "19462  Tam41_Mmp37          PF09139.16        109     1.48    322   0.59   0.57   0.51   0.02\n",
            "19463  TamB                 PF04357.18         32     8.00    381   0.59   0.64   0.44   0.02\n",
            "19464  TAN                  PF11640.13         86     5.68    150   0.59   0.61   0.50   0.04\n",
            "19465  TANGO2               PF05742.17         98     2.74    257   0.59   0.57   0.45   0.02\n",
            "19466  Tankyrase_bdg_C      PF15327.11         14     1.26    167   0.59   0.60   0.49   0.06\n",
            "19467  Tannase              PF07519.16          6     1.37    490   0.59   0.56   0.50   0.01\n",
            "19468  Tantalus             PF15386.11          9     2.58     57   0.98   0.96   0.85   0.09\n",
            "19469  Tap-RNA_bind         PF09162.15         48     2.36     80   0.71   0.65   0.67   0.04\n",
            "19470  TAP35_44             PF07265.16          7     0.71    119   0.59   0.58   0.49   0.02\n",
            "19471  TAP42                PF04177.17        142     4.05    346   0.59   0.59   0.48   0.05\n",
            "19472  Tape_meas_lam_C      PF09718.15         31     5.53     76   0.74   0.75   0.72   0.07\n",
            "19473  TAPR1-like           PF15251.11          9     1.62    249   0.59   0.58   0.47   0.05\n",
            "19474  TAP_C                PF03943.18        131     8.59     49   1.13   1.04   1.11   0.08\n",
            "19475  Taq-exonuc           PF09281.15          7     1.00    129   0.59   0.61   0.51   0.02\n",
            "19476  TaqI_C               PF12950.12         16     2.54    118   0.59   0.57   0.50   0.03\n",
            "19477  TarH                 PF02203.20         63     6.35    177   0.59   0.62   0.54   0.05\n",
            "19478  TARP                 PF21951.1           1     0.62     58   0.97   0.92   0.91   0.08\n",
            "19479  TARSH_C              PF21731.2          49     1.35    127   0.59   0.51   0.51   0.03\n",
            "19480  TarS_C1              PF18674.6          11     2.70    147   0.59   0.57   0.53   0.03\n",
            "19481  TarS_C2              PF22377.1           1     0.50     78   0.72   0.75   0.66   0.04\n",
            "19482  TarS_linker          PF22181.1         233     7.10    100   0.59   0.61   0.55   0.04\n",
            "19483  TAS2R                PF05296.18         20     1.87    303   0.59   0.57   0.54   0.09\n",
            "19484  Tas3-like_N          PF22378.1           2     0.66     73   0.77   0.77   0.71   0.03\n",
            "19485  Tas3_C-hel           PF21487.2           1     0.84     49   1.13   1.19   1.08   0.04\n",
            "19486  Tash_PEST            PF07708.16         22    22.00     18   2.53   2.64   2.52   0.73\n",
            "19487  TASL                 PF15133.11         14     1.37    300   0.59   0.58   0.51   0.03\n",
            "19488  Tat                  PF00539.23          2     0.71     64   0.88   0.75   0.80   0.11\n",
            "19489  TatA_B_E             PF02416.21         11     2.89     53   1.05   1.14   1.00   0.05\n",
            "19490  TatC                 PF00902.23        542     3.91    208   0.59   0.60   0.50   0.08\n",
            "19491  TatD_DNase           PF01026.26         74     4.31    253   0.59   0.58   0.50   0.02\n",
            "19492  TATR                 PF03430.18          1     0.39    575   0.59   0.56   0.51   0.03\n",
            "19493  TAtT                 PF16811.10         16     1.92    263   0.59   0.60   0.55   0.02\n",
            "19494  TAT_signal           PF10518.14        326   326.00     21   1.81   2.01   1.82   0.40\n",
            "19495  TAT_ubiq             PF07706.17         21     4.00     40   1.37   1.35   1.31   0.10\n",
            "19496  Tau95                PF09734.14         55     2.76    144   0.59   0.54   0.48   0.03\n",
            "19497  Tau95_N              PF17682.6          65     3.51    121   0.59   0.59   0.47   0.02\n",
            "19498  TauD                 PF02668.21        116    12.68    267   0.59   0.56   0.47   0.02\n",
            "19499  TauE                 PF01925.24        483    11.48    237   0.59   0.66   0.49   0.13\n",
            "19500  Tautomerase          PF01361.26         18     7.47     60   0.93   0.96   0.90   0.04\n",
            "19501  Tautomerase-like     PF21420.2           1     0.84     45   1.23   1.24   1.17   0.05\n",
            "19502  Tautomerase_2        PF14552.11         54     3.44     82   0.69   0.67   0.66   0.03\n",
            "19503  Tautomerase_3        PF14832.11         16     1.36    138   0.59   0.56   0.52   0.02\n",
            "19504  Tax                  PF02959.21          2     0.39    222   0.59   0.56   0.52   0.02\n",
            "19505  Taxilin              PF09728.14         47     3.84    302   0.59   0.59   0.55   0.13\n",
            "19506  TAXi_C               PF14541.11         56     3.76    161   0.59   0.57   0.52   0.01\n",
            "19507  TAXi_N               PF14543.11        193     3.71    176   0.59   0.56   0.48   0.03\n",
            "19508  TAZ1_RBM             PF22332.1           1     1.00     34   1.31   1.36   1.26   0.10\n",
            "19509  TA_inhibitor         PF17574.7           1     0.46     89   0.65   0.64   0.58   0.05\n",
            "19510  TB                   PF00683.22        182     5.95     44   1.25   0.98   1.19   0.32\n",
            "19511  TB2_DP1_HVA22        PF03134.24        299     4.71     77   0.73   0.65   0.66   0.10\n",
            "19512  TBC1D23_C            PF19430.4           3     0.43    226   0.59   0.58   0.52   0.02\n",
            "19513  TBCA                 PF02970.21        141     3.62     99   0.59   0.62   0.54   0.09\n",
            "19514  TBCC                 PF07986.17         40     3.10    120   0.59   0.54   0.55   0.05\n",
            "19515  TBCC_N               PF16752.10         56     3.81    115   0.59   0.61   0.55   0.09\n",
            "19516  TBD                  PF12845.12         10     3.13     56   0.99   0.96   0.97   0.07\n",
            "19517  TBK1_CCD1            PF18394.6          10     1.57    256   0.59   0.58   0.54   0.05\n",
            "19518  TBK1_ULD             PF18396.6          22     1.52     88   0.65   0.62   0.60   0.05\n",
            "19519  TBP                  PF00352.26        251     5.11     85   0.67   0.67   0.62   0.03\n",
            "19520  TBP-binding          PF09247.16         13     1.80     60   0.93   0.99   0.86   0.09\n",
            "19521  TBP-TOTE             PF22721.1          62    11.76     78   0.73   0.70   0.71   0.03\n",
            "19522  TbpB_A               PF17484.7           5     1.45    136   0.59   0.57   0.51   0.05\n",
            "19523  TbpB_B_D             PF01298.23         42     2.22    126   0.59   0.59   0.47   0.05\n",
            "19524  TbpB_C               PF17483.7           6     1.01     98   0.59   0.56   0.49   0.03\n",
            "19525  TBPIP                PF07106.18          9     2.41     62   0.90   0.90   0.86   0.05\n",
            "19526  TBPIP_C              PF21664.2           4     0.46    115   0.59   0.55   0.52   0.04\n",
            "19527  TBPIP_N              PF15517.11          2     0.50    100   0.59   0.58   0.52   0.03\n",
            "19528  TBSV_P22             PF03558.18          3     0.71    187   0.59   0.59   0.51   0.01\n",
            "19529  TBX2-3_RD            PF20627.2           7     0.57    107   0.59   0.60   0.50   0.04\n",
            "19530  TBX2-3_TAD           PF12598.13         13     1.70     81   0.70   0.68   0.60   0.12\n",
            "19531  Tc-38                PF20054.4          19    11.59     70   0.80   0.77   0.72   0.02\n",
            "19532  Tc-R-P               PF22785.1           4     0.93    106   0.59   0.60   0.50   0.01\n",
            "19533  TC1                  PF15063.11          6     1.30     75   0.75   0.79   0.70   0.05\n",
            "19534  TcaA_2nd             PF22813.1          34     4.22    103   0.59   0.59   0.52   0.06\n",
            "19535  TcaA_3rd_4th         PF22820.1         224    16.42     73   0.77   0.78   0.69   0.07\n",
            "19536  TcaA_5th             PF22819.1          23     2.93    105   0.59   0.57   0.55   0.07\n",
            "19537  TCAD2                PF19971.4           1     0.46     90   0.64   0.66   0.57   0.03\n",
            "19538  TCAD3                PF19972.4           2     0.96     48   1.15   1.14   1.09   0.05\n",
            "19539  TCAD7                PF19973.4           9     2.57    109   0.59   0.63   0.49   0.03\n",
            "19540  TCAD9                PF19974.4          15     3.53    429   0.59   0.61   0.50   0.02\n",
            "19541  TcA_RBD              PF18518.6          10     1.33    132   0.59   0.55   0.51   0.03\n",
            "19542  TcA_TcB_BD           PF18276.6          34     1.35    287   0.59   0.57   0.53   0.02\n",
            "19543  TcdA1_RBD_2          PF21963.1           4     0.79     86   0.66   0.67   0.58   0.03\n",
            "19544  TcdA1_RBD_3          PF21968.1           1     0.39    106   0.59   0.58   0.52   0.02\n",
            "19545  TcdA_TcdB            PF12919.12          5     1.47    382   0.59   0.59   0.50   0.03\n",
            "19546  TcdA_TcdB_pore       PF12920.12          5     0.94    655   0.59   0.59   0.50   0.02\n",
            "19547  TcdB_N               PF12918.12         19     8.30     66   0.85   0.88   0.80   0.09\n",
            "19548  TcdB_toxin_midC      PF12255.13         14     1.46    147   0.59   0.57   0.52   0.03\n",
            "19549  TcdB_toxin_midN      PF12256.13          5     1.62    181   0.59   0.58   0.51   0.02\n",
            "19550  Tcell_CD4_C          PF12104.13        125     5.02     28   1.92   1.76   1.90   0.47\n",
            "19551  Tcf25                PF04910.19         74     2.72    337   0.59   0.57   0.48   0.03\n",
            "19552  TcfC                 PF16967.10          9     2.35     68   0.83   0.84   0.80   0.04\n",
            "19553  TCL1_MTCP1           PF01840.22         16     1.21    116   0.59   0.53   0.52   0.04\n",
            "19554  TCO89                PF10452.14          4     1.32    546   0.59   0.59   0.48   0.06\n",
            "19555  TCP                  PF03634.18         42     1.73    155   0.59   0.58   0.50   0.08\n",
            "19556  Tcp10_C              PF07202.18        134   134.00     35   1.53   1.52   1.53   0.11\n",
            "19557  Tcp11                PF05794.18        197     6.21    436   0.59   0.59   0.47   0.03\n",
            "19558  TcpA                 PF05946.17          2     0.58    130   0.59   0.62   0.52   0.03\n",
            "19559  TcpC-like_C          PF21497.2           1     0.39    109   0.59   0.56   0.52   0.03\n",
            "19560  TcpE                 PF12648.12         19     2.96    100   0.59   0.56   0.49   0.06\n",
            "19561  TcpF                 PF06340.16          1     0.39    317   0.59   0.57   0.52   0.01\n",
            "19562  TcpQ                 PF10671.14         24     4.37     80   0.71   0.67   0.65   0.05\n",
            "19563  TcpS                 PF17456.7           2     0.63    155   0.59   0.60   0.51   0.04\n",
            "19564  TCR                  PF03638.20        175     2.26     38   1.43   1.02   1.39   0.66\n",
            "19565  TCRP1                PF14944.11          7     0.58    235   0.59   0.54   0.50   0.05\n",
            "19566  TCR_zetazeta         PF11628.13          7     2.68     31   1.74   1.72   1.72   0.19\n",
            "19567  TctA                 PF01970.21        295     4.13    423   0.59   0.64   0.52   0.06\n",
            "19568  TctB                 PF07331.16        335    10.73    137   0.59   0.64   0.33   0.11\n",
            "19569  TctC                 PF03401.19          3     0.84    274   0.59   0.60   0.53   0.01\n",
            "19570  Tctex-1              PF03645.18        250     4.57     95   0.60   0.55   0.53   0.05\n",
            "19571  TCTN_DUF1619         PF07773.16         56     3.30    313   0.59   0.51   0.47   0.05\n",
            "19572  TCTP                 PF00838.23        125     2.16    164   0.59   0.57   0.51   0.04\n",
            "19573  TDA11                PF17084.10          1     0.42    465   0.59   0.60   0.52   0.03\n",
            "19574  TDBD                 PF16135.10         50     8.51     52   1.07   0.96   1.00   0.09\n",
            "19575  TDH                  PF03347.18          2     0.39    165   0.59   0.56   0.52   0.02\n",
            "19576  TdIF1_2nd            PF21229.2           9     0.62     97   0.59   0.58   0.52   0.02\n",
            "19577  TDP-43_C             PF20910.2          15     0.81    105   0.59   0.56   0.49   0.07\n",
            "19578  TDP43_N              PF18694.6          63     3.23     74   0.76   0.75   0.71   0.01\n",
            "19579  TDRP                 PF15683.10         11     0.86    147   0.59   0.58   0.51   0.05\n",
            "19580  TEA                  PF01285.23         33     2.64     68   0.83   0.79   0.75   0.06\n",
            "19581  TEADIR3              PF15238.11         42    42.00     16   2.86   2.69   2.86   0.64\n",
            "19582  Tea_C                PF22884.1          11     1.04    162   0.59   0.58   0.54   0.03\n",
            "19583  Tea_mid              PF22889.1          15     0.90    159   0.59   0.53   0.53   0.04\n",
            "19584  TEBP_beta            PF07404.16          3     0.70    140   0.59   0.57   0.53   0.04\n",
            "19585  TEBP_OB2-like        PF22236.1           8     1.30    100   0.59   0.59   0.52   0.04\n",
            "19586  TECR_N               PF21696.2          13     1.24     77   0.74   0.73   0.69   0.03\n",
            "19587  Tecti-min-caps       PF09300.15          1     0.44     84   0.67   0.66   0.60   0.03\n",
            "19588  Tectonin             PF19193.5          32     2.07    216   0.59   0.55   0.51   0.04\n",
            "19589  TED                  PF08341.16        101     8.48    102   0.59   0.54   0.41   0.05\n",
            "19590  TEDC1                PF14970.11         28     2.20    169   0.59   0.56   0.46   0.05\n",
            "19591  TED_2                PF20610.3          37     1.89    131   0.59   0.53   0.50   0.05\n",
            "19592  TED_complement       PF07678.19         17     1.51    312   0.59   0.57   0.51   0.03\n",
            "19593  Tegument_dsDNA       PF12818.12          5     1.20    277   0.59   0.59   0.53   0.01\n",
            "19594  TehB                 PF03848.19          4     0.53    193   0.59   0.57   0.52   0.02\n",
            "19595  Tektin               PF03148.19        120     4.26    383   0.59   0.59   0.57   0.08\n",
            "19596  TelA                 PF05816.16        122     3.33    332   0.59   0.62   0.55   0.06\n",
            "19597  Telethonin           PF09470.15          8     0.73    163   0.59   0.57   0.52   0.02\n",
            "19598  TelK_muzzle          PF20849.2           4     0.73    118   0.59   0.59   0.53   0.03\n",
            "19599  TelK_N               PF22853.1           2     0.61     75   0.75   0.77   0.69   0.05\n",
            "19600  TelK_stirrup         PF20818.2           3     0.63     92   0.62   0.64   0.56   0.02\n",
            "19601  Telomerase_RBD       PF12009.13        109     3.01    133   0.59   0.54   0.52   0.05\n",
            "19602  Telomere_reg-2       PF10193.14        135     4.03    112   0.59   0.62   0.54   0.04\n",
            "19603  Ten1                 PF12658.12         20     2.62    117   0.59   0.60   0.46   0.03\n",
            "19604  Ten1_2               PF15490.11         22     2.23    117   0.59   0.60   0.53   0.02\n",
            "19605  TENA_THI-4           PF03070.21         17     4.59    210   0.59   0.55   0.54   0.03\n",
            "19606  Tenui_N              PF05733.16         12     1.53    224   0.59   0.60   0.51   0.03\n",
            "19607  Tenui_NCP            PF04876.17          1     0.38    173   0.59   0.58   0.52   0.01\n",
            "19608  Tenui_NS3            PF05310.17          9     1.30    186   0.59   0.52   0.52   0.04\n",
            "19609  Tenui_NS4            PF03300.18          1     0.39    282   0.59   0.59   0.52   0.01\n",
            "19610  Tenui_PVC2           PF06656.16          1     0.38    784   0.59   0.58   0.52   0.02\n",
            "19611  Ten_N                PF06484.17          7     0.73    367   0.59   0.57   0.51   0.03\n",
            "19612  TEP1_CUB2            PF21412.2          19     2.52     60   0.93   0.92   0.88   0.04\n",
            "19613  TEP1_N               PF05386.16         20     5.04     29   1.85   1.85   1.84   0.13\n",
            "19614  Ter                  PF05472.16         13     1.22    296   0.59   0.58   0.53   0.03\n",
            "19615  TerB                 PF05099.18         62     4.28    118   0.59   0.63   0.55   0.05\n",
            "19616  TERB2                PF15101.11          7     1.25    207   0.59   0.55   0.52   0.03\n",
            "19617  TerB_C               PF15615.11         15     4.26    148   0.59   0.63   0.51   0.05\n",
            "19618  TerB_N               PF13208.11         25     4.71    207   0.59   0.56   0.50   0.02\n",
            "19619  TerC                 PF03741.21         55     2.50    178   0.59   0.64   0.52   0.08\n",
            "19620  TerD                 PF02342.23        186     2.40    187   0.59   0.59   0.53   0.02\n",
            "19621  TERF2_RBM            PF16772.10          6     2.05     41   1.33   1.38   1.31   0.08\n",
            "19622  TerL_ATPase          PF03354.20          3     1.10    178   0.59   0.58   0.51   0.02\n",
            "19623  TerL_nuclease        PF20441.3           3     1.03    286   0.59   0.57   0.52   0.02\n",
            "19624  Terminase-T7_RNaseH-like PF22530.1          15     1.34    108   0.59   0.60   0.52   0.02\n",
            "19625  Terminase_2          PF03592.21         36     3.62    141   0.59   0.61   0.48   0.06\n",
            "19626  Terminase_3          PF04466.18          7     1.54    201   0.59   0.57   0.51   0.03\n",
            "19627  Terminase_3C         PF17288.7           7     2.64    155   0.59   0.56   0.50   0.04\n",
            "19628  Terminase_4          PF05119.17         80     5.92     98   0.59   0.59   0.53   0.03\n",
            "19629  Terminase_5          PF06056.17          8     3.12     58   0.96   0.90   0.93   0.06\n",
            "19630  Terminase_6C         PF17289.7          22     7.26    155   0.59   0.57   0.48   0.02\n",
            "19631  Terminase_6N         PF03237.20         22     7.27    215   0.59   0.57   0.48   0.02\n",
            "19632  Terpene_synth        PF01397.26        155     3.14    188   0.59   0.58   0.45   0.04\n",
            "19633  Terpene_synth_C      PF03936.21        181     4.12    241   0.59   0.55   0.52   0.03\n",
            "19634  Terpene_syn_C_2      PF19086.5        1879    11.06    199   0.59   0.55   0.54   0.03\n",
            "19635  TerS_C               PF22412.1           1     0.50     79   0.72   0.74   0.65   0.03\n",
            "19636  TerS_N               PF21434.2           1     0.83     50   1.10   1.19   1.05   0.05\n",
            "19637  Tertiapin            PF21201.2           1     1.00     21   1.77   1.46   1.73   0.41\n",
            "19638  TERT_C               PF21399.2           8     0.64    130   0.59   0.58   0.53   0.04\n",
            "19639  TERT_thumb           PF17984.6           1     0.37    191   0.59   0.55   0.51   0.03\n",
            "19640  TerY_C               PF15616.11         10     0.73    129   0.59   0.52   0.51   0.04\n",
            "19641  TES                  PF08034.16          5     1.87     66   0.85   0.82   0.80   0.11\n",
            "19642  TetM_leader          PF08076.16          2     1.24     28   1.92   1.70   1.88   0.22\n",
            "19643  TetR                 PF13972.11         62     5.20    144   0.59   0.56   0.55   0.04\n",
            "19644  TetR-C_39            PF21626.2           3     0.65     96   0.60   0.54   0.53   0.05\n",
            "19645  Tetrabrachion        PF11401.13          1     0.84     49   1.13   1.18   1.07   0.13\n",
            "19646  Tetradecapep         PF08187.16          2     2.00     14   2.38   2.36   2.35   0.54\n",
            "19647  Tetraspanin          PF00335.25        110     7.05    225   0.59   0.55   0.44   0.06\n",
            "19648  TetR_C_1             PF02909.22          9     2.56    146   0.59   0.60   0.53   0.01\n",
            "19649  TetR_C_10            PF16295.10          4     0.83    132   0.59   0.57   0.54   0.03\n",
            "19650  TetR_C_11            PF16859.10        260     8.43    113   0.59   0.65   0.56   0.05\n",
            "19651  TetR_C_12            PF16914.10          4     0.95    105   0.59   0.61   0.53   0.03\n",
            "19652  TetR_C_13            PF16925.10        121     4.72    109   0.59   0.60   0.57   0.02\n",
            "19653  TetR_C_13_2          PF21993.1         322     4.51    103   0.59   0.62   0.57   0.03\n",
            "19654  TetR_C_14            PF17754.6          37     3.45    112   0.59   0.62   0.56   0.04\n",
            "19655  TetR_C_15            PF17918.6          62     9.84    108   0.59   0.63   0.56   0.05\n",
            "19656  TetR_C_16            PF17920.6          66     5.02    107   0.59   0.65   0.56   0.04\n",
            "19657  TetR_C_17            PF17922.6           9     1.29    100   0.59   0.58   0.54   0.03\n",
            "19658  TetR_C_18            PF17923.6           1     0.41    113   0.59   0.62   0.52   0.03\n",
            "19659  TetR_C_19            PF17924.6          48     5.35    117   0.59   0.56   0.55   0.06\n",
            "19660  TetR_C_2             PF08361.16          7     0.84    121   0.59   0.58   0.53   0.04\n",
            "19661  TetR_C_20            PF17925.6          24     2.19    107   0.59   0.62   0.55   0.03\n",
            "19662  TetR_C_21            PF17926.6          27     2.39    111   0.59   0.59   0.54   0.04\n",
            "19663  TetR_C_22            PF17928.6          12     3.57    113   0.59   0.61   0.57   0.03\n",
            "19664  TetR_C_23            PF17931.6          73     2.74    127   0.59   0.58   0.56   0.04\n",
            "19665  TetR_C_24            PF17932.6         141     5.77    114   0.59   0.59   0.57   0.02\n",
            "19666  TetR_C_25            PF17933.6          36     1.66    106   0.59   0.61   0.55   0.02\n",
            "19667  TetR_C_26            PF17934.6          25     0.97    109   0.59   0.57   0.54   0.04\n",
            "19668  TetR_C_27            PF17935.6          54     3.25    106   0.59   0.58   0.57   0.03\n",
            "19669  TetR_C_28            PF17937.6         168     9.19     96   0.60   0.63   0.53   0.04\n",
            "19670  TetR_C_29            PF17938.6         157     4.27    119   0.59   0.59   0.57   0.02\n",
            "19671  TetR_C_3             PF08362.16         14     1.10    143   0.59   0.58   0.54   0.02\n",
            "19672  TetR_C_30            PF17939.6         141     7.81    115   0.59   0.60   0.56   0.02\n",
            "19673  TetR_C_31            PF17940.6         214     9.04    107   0.59   0.67   0.51   0.07\n",
            "19674  TetR_C_32            PF19344.4          17     0.65    139   0.59   0.59   0.52   0.01\n",
            "19675  TetR_C_33            PF13305.11        322     8.37    103   0.59   0.60   0.47   0.04\n",
            "19676  TetR_C_34            PF17929.6          36     4.65    120   0.59   0.62   0.57   0.03\n",
            "19677  TetR_C_35            PF18556.6          55     2.19    105   0.59   0.64   0.55   0.04\n",
            "19678  TetR_C_36            PF18598.6          42     2.90    111   0.59   0.64   0.56   0.03\n",
            "19679  TetR_C_37            PF18665.6          14     1.36    117   0.59   0.57   0.54   0.05\n",
            "19680  TetR_C_38            PF19352.4          17     0.73    135   0.59   0.62   0.53   0.02\n",
            "19681  TetR_C_39            PF21303.2           2     0.51     92   0.62   0.61   0.55   0.03\n",
            "19682  TetR_C_4             PF08359.16          8     0.72    133   0.59   0.60   0.53   0.04\n",
            "19683  TetR_C_40            PF21306.2           5     0.91    117   0.59   0.64   0.53   0.03\n",
            "19684  TetR_C_41            PF21351.2          77     3.26    108   0.59   0.62   0.55   0.03\n",
            "19685  TetR_C_42            PF21542.2           1     0.38    101   0.59   0.60   0.52   0.05\n",
            "19686  TetR_C_43            PF21597.2          54     2.82    105   0.59   0.64   0.54   0.03\n",
            "19687  TetR_C_44            PF21776.2          12     0.69    131   0.59   0.63   0.53   0.03\n",
            "19688  TetR_C_45            PF21935.1          13     0.61    115   0.59   0.61   0.53   0.02\n",
            "19689  TetR_C_46            PF21943.1          34     2.96    109   0.59   0.62   0.53   0.03\n",
            "19690  TetR_C_5             PF08360.16          3     0.90    131   0.59   0.57   0.54   0.03\n",
            "19691  TetR_C_5-like        PF21256.2           3     1.29    127   0.59   0.60   0.51   0.03\n",
            "19692  TetR_C_6             PF13977.11         92     9.41    115   0.59   0.63   0.55   0.05\n",
            "19693  TetR_C_7             PF14246.11         57     4.26    119   0.59   0.61   0.57   0.03\n",
            "19694  TetR_C_8             PF14278.11        263     9.25    103   0.59   0.54   0.56   0.04\n",
            "19695  TetR_C_9             PF14514.11          2     0.54    130   0.59   0.59   0.52   0.01\n",
            "19696  TetR_HI_0893_C       PF22604.1          28     1.84    112   0.59   0.55   0.54   0.04\n",
            "19697  TetR_N               PF00440.28         73    17.41     47   1.17   1.25   1.17   0.07\n",
            "19698  Tet_C_40             PF22568.1           1     0.40    129   0.59   0.59   0.52   0.03\n",
            "19699  Tet_JBP              PF12851.12         12     2.25    164   0.59   0.55   0.48   0.01\n",
            "19700  Tet_res_leader       PF08050.17          4     3.46     20   2.64   2.54   2.62   0.42\n",
            "19701  TEX12                PF15219.11          2     0.57     85   0.67   0.70   0.60   0.04\n",
            "19702  TEX13                PF15186.11         30     1.21    150   0.59   0.58   0.54   0.04\n",
            "19703  TEX13B-like_middle   PF22835.1           1     0.39    110   0.59   0.61   0.52   0.04\n",
            "19704  TEX13B_C             PF22836.1           4     1.50     44   1.25   1.26   1.20   0.10\n",
            "19705  TEX13_ZnF            PF20864.2           2     1.74     20   2.63   2.11   2.60   0.77\n",
            "19706  TEX15                PF15326.11          7     0.57    236   0.59   0.57   0.52   0.02\n",
            "19707  TEX19                PF15553.11         11     0.67    159   0.59   0.53   0.52   0.02\n",
            "19708  TEX29                PF15839.10          7     0.82     76   0.74   0.67   0.68   0.08\n",
            "19709  TEX33                PF15400.11          8     0.96    138   0.59   0.57   0.51   0.03\n",
            "19710  Tex55                PF17819.6           7     0.85    146   0.59   0.55   0.52   0.04\n",
            "19711  Tex_central_region   PF22706.1         254     2.23    178   0.59   0.60   0.52   0.03\n",
            "19712  Tex_N                PF09371.15        393     2.57     75   0.75   0.78   0.73   0.05\n",
            "19713  Tex_YqgF             PF16921.10        177     1.03    126   0.59   0.61   0.53   0.02\n",
            "19714  TFA2_E-tether        PF22254.1          37     9.44     34   1.59   1.59   1.59   0.10\n",
            "19715  TFA2_Winged_2        PF18121.6          98     9.31     60   0.93   0.96   0.82   0.05\n",
            "19716  Tfb2                 PF03849.19        111     2.16    358   0.59   0.59   0.51   0.03\n",
            "19717  Tfb2_C               PF18307.6          36     3.48     68   0.83   0.76   0.80   0.06\n",
            "19718  Tfb4                 PF03850.19         21     1.17    273   0.59   0.56   0.51   0.03\n",
            "19719  Tfb5                 PF06331.17         79     4.33     67   0.84   0.84   0.79   0.06\n",
            "19720  TFB6                 PF17110.10         45     1.41    170   0.59   0.59   0.50   0.02\n",
            "19721  TFC3_eWH             PF21552.2          14     0.91     91   0.63   0.65   0.58   0.04\n",
            "19722  TFCD_C               PF12612.13         94     5.27    187   0.59   0.61   0.51   0.02\n",
            "19723  TFIIA                PF03153.18         51     3.00    415   0.59   0.57   0.43   0.05\n",
            "19724  TFIIA_gamma_C        PF02751.19         88     5.36     43   1.28   1.18   1.22   0.10\n",
            "19725  TFIIA_gamma_N        PF02268.21         62     3.96     47   1.17   1.19   1.15   0.06\n",
            "19726  TFIIB                PF00382.24         13     2.34     92   0.62   0.63   0.59   0.04\n",
            "19727  TFIIB_C_1            PF18542.6           2     0.57     99   0.59   0.62   0.53   0.03\n",
            "19728  TFIIB_C_2            PF22108.1           9     1.09    106   0.59   0.61   0.52   0.04\n",
            "19729  TFIIB_Zn-ribbon_Tryp PF22109.1           5     1.25     40   1.36   1.29   1.32   0.17\n",
            "19730  TFIID-18kDa          PF02269.21          6     1.04     93   0.61   0.62   0.56   0.04\n",
            "19731  TFIID-31kDa          PF02291.20         86     1.83    122   0.59   0.60   0.53   0.02\n",
            "19732  TFIID_20kDa          PF03847.18          8     1.43     68   0.83   0.84   0.78   0.07\n",
            "19733  TFIID_30kDa          PF03540.18         57     3.31     50   1.11   1.11   1.07   0.04\n",
            "19734  TFIID_NTD2           PF04494.20        101     1.94    133   0.59   0.56   0.54   0.04\n",
            "19735  TFIIE-A_C            PF11521.13         23     1.85     85   0.67   0.65   0.59   0.09\n",
            "19736  TFIIE_alpha          PF02002.22          6     1.61    105   0.59   0.61   0.54   0.05\n",
            "19737  TFIIE_beta           PF02186.20         62     5.53     66   0.85   0.83   0.75   0.06\n",
            "19738  TFIIF_alpha          PF05793.17          8     1.25    528   0.59   0.59   0.50   0.08\n",
            "19739  TFIIF_beta           PF02270.20         22     2.10     65   0.86   0.82   0.83   0.05\n",
            "19740  TFIIF_beta_N         PF17683.6          42     4.33    145   0.59   0.57   0.43   0.04\n",
            "19741  TFIIIA_zf-C2H2       PF22110.1          43     2.99     32   1.69   1.42   1.66   0.29\n",
            "19742  TFIIIC_delta         PF12657.12         42     5.65    444   0.59   0.56   0.40   0.02\n",
            "19743  TFIIIC_sub6          PF10419.14         76     4.36     98   0.59   0.59   0.44   0.02\n",
            "19744  TFIIS_C              PF01096.23         90     6.46     39   1.40   1.13   1.38   0.26\n",
            "19745  TFIIS_M              PF07500.19        262     6.15    114   0.59   0.60   0.49   0.06\n",
            "19746  TfoX_C               PF04994.18         29     2.76     81   0.70   0.73   0.67   0.04\n",
            "19747  TfoX_N               PF04993.18         86     5.28     91   0.63   0.61   0.55   0.02\n",
            "19748  TFR_dimer            PF04253.20        256     6.42    121   0.59   0.57   0.50   0.03\n",
            "19749  TfuA                 PF07812.17         57     1.14    119   0.59   0.61   0.54   0.02\n",
            "19750  TFX_C                PF14601.11         35     3.10     83   0.68   0.72   0.66   0.04\n",
            "19751  TF_AP-2              PF03299.19         25     1.99    197   0.59   0.60   0.54   0.03\n",
            "19752  TF_Otx               PF03529.18         15     0.93     91   0.63   0.59   0.52   0.08\n",
            "19753  TF_Zn_Ribbon         PF08271.17         55    17.61     43   1.28   1.11   1.24   0.18\n",
            "19754  TGase_elicitor       PF16683.10         15     0.70    361   0.59   0.55   0.53   0.02\n",
            "19755  TGBp3                PF02495.22         60     9.47     59   0.95   0.95   0.88   0.10\n",
            "19756  TGFb_propeptide      PF00688.23         56     3.24    242   0.59   0.57   0.46   0.03\n",
            "19757  TGF_beta             PF00019.25        228     2.59    104   0.59   0.48   0.48   0.08\n",
            "19758  TGF_beta_GS          PF08515.17         13     5.30     28   1.92   1.99   1.87   0.13\n",
            "19759  Tgi2PP               PF18250.6           1     0.89     43   1.27   1.31   1.22   0.06\n",
            "19760  TGL                  PF20085.4         185     1.89    245   0.59   0.55   0.52   0.02\n",
            "19761  TgMIC1               PF11476.14          1     0.42    137   0.59   0.64   0.52   0.03\n",
            "19762  TgpA_N               PF11992.13        126    13.69    333   0.59   0.63   0.47   0.05\n",
            "19763  TGS                  PF02824.26         52     8.38     60   0.93   0.95   0.86   0.03\n",
            "19764  TGT                  PF01702.23        449     3.81    357   0.59   0.57   0.51   0.01\n",
            "19765  TGT_C1               PF14809.11          2     0.59     70   0.81   0.80   0.75   0.04\n",
            "19766  TGT_C2               PF14810.11         70     4.72     72   0.78   0.81   0.73   0.02\n",
            "19767  TH1                  PF04858.18         11     0.88    579   0.59   0.59   0.52   0.02\n",
            "19768  ThaI                 PF15514.11          2     0.45    203   0.59   0.58   0.52   0.02\n",
            "19769  THAP                 PF05485.17        331     6.39     87   0.65   0.54   0.44   0.07\n",
            "19770  THAP4_heme-bd        PF08768.16        129     3.75    151   0.59   0.57   0.46   0.02\n",
            "19771  THAP9_C              PF22824.1           1     0.38    186   0.59   0.57   0.51   0.03\n",
            "19772  Thaumatin            PF00314.22        246     1.58    209   0.59   0.49   0.46   0.07\n",
            "19773  THB                  PF18362.6           9     2.60     34   1.59   1.59   1.57   0.08\n",
            "19774  Thc1_RRM             PF22877.1           4     1.08     75   0.75   0.73   0.69   0.04\n",
            "19775  ThcOx                PF22767.1          34     1.84    110   0.59   0.58   0.52   0.04\n",
            "19776  THDPS_M              PF14789.11         62     5.68     41   1.34   1.31   1.30   0.05\n",
            "19777  THDPS_N              PF14790.11         21     1.03    167   0.59   0.59   0.52   0.03\n",
            "19778  THDPS_N_2            PF14805.11        202     3.24     67   0.84   0.85   0.76   0.06\n",
            "19779  THEG                 PF14912.11         66     8.58     58   0.96   0.97   0.86   0.10\n",
            "19780  THEG4                PF15834.10          8     0.61    201   0.59   0.56   0.52   0.01\n",
            "19781  Thermo-DBP           PF20754.2           7     0.89     98   0.59   0.64   0.52   0.04\n",
            "19782  ThermoDBP-RP_arch    PF10015.14         25     1.60     78   0.73   0.76   0.67   0.06\n",
            "19783  Thermopsin           PF05317.16         26     1.85    256   0.59   0.56   0.50   0.04\n",
            "19784  THF_DHG_CYH          PF00763.28       1021     3.16    115   0.59   0.61   0.53   0.03\n",
            "19785  THF_DHG_CYH_C        PF02882.24         75     1.59    160   0.59   0.62   0.53   0.03\n",
            "19786  Thg1                 PF04446.17        157     1.82    130   0.59   0.56   0.51   0.03\n",
            "19787  Thg1C                PF14413.11         87     1.05    116   0.59   0.54   0.51   0.05\n",
            "19788  THH1_TOM1-3_dom      PF06454.16         15     0.71    273   0.59   0.58   0.53   0.05\n",
            "19789  Thi4                 PF01946.22          7     0.83    235   0.59   0.61   0.52   0.02\n",
            "19790  Thiaminase-1_dom     PF22141.1           4     0.79    146   0.59   0.59   0.52   0.02\n",
            "19791  Thiaminase_I-like_N  PF22156.1           2     0.49    136   0.59   0.61   0.52   0.03\n",
            "19792  Thiamine_BP          PF01910.22         80     3.20     92   0.62   0.64   0.57   0.04\n",
            "19793  Thia_YuaJ            PF09515.15         25     1.63    177   0.59   0.61   0.54   0.07\n",
            "19794  ThiC-associated      PF13667.11        167     1.67     81   0.70   0.68   0.63   0.03\n",
            "19795  ThiC_Rad_SAM         PF01964.23        193     1.15    420   0.59   0.59   0.54   0.02\n",
            "19796  ThiD2                PF17792.6          21     1.30    124   0.59   0.60   0.52   0.06\n",
            "19797  ThiF                 PF00899.26         44     4.38    238   0.59   0.59   0.50   0.01\n",
            "19798  ThiG                 PF05690.19         21     0.95    247   0.59   0.63   0.54   0.02\n",
            "19799  ThiI                 PF02568.19          6     1.05    197   0.59   0.61   0.53   0.02\n",
            "19800  ThiI_fer             PF22025.1         981     4.45     72   0.78   0.80   0.74   0.05\n",
            "19801  ThiJ_like            PF17124.10         10     1.27    186   0.59   0.56   0.50   0.01\n",
            "19802  Thio2_N              PF21352.2         173     9.71     27   1.98   1.76   1.98   0.45\n",
            "19803  Thioesterase         PF00975.25         16     4.72    226   0.59   0.58   0.49   0.01\n",
            "19804  Thiolase_C           PF02803.23         21     1.04    123   0.59   0.62   0.53   0.02\n",
            "19805  Thiolase_C_1         PF22691.1          64     2.28    144   0.59   0.61   0.55   0.03\n",
            "19806  Thiolase_N           PF00108.28         30     2.11    260   0.59   0.61   0.53   0.03\n",
            "19807  Thiol_cytolysin      PF01289.24          7     0.96    353   0.59   0.60   0.54   0.02\n",
            "19808  Thiol_cytolys_C      PF17440.7           8     0.68    102   0.59   0.54   0.52   0.03\n",
            "19809  Thionin              PF00321.22         14     2.87     46   1.20   0.94   1.16   0.32\n",
            "19810  Thiopep_pre          PF19409.4          32     9.29     36   1.51   1.49   1.46   0.33\n",
            "19811  Thioredoxin          PF00085.25         34     1.81    103   0.59   0.55   0.53   0.02\n",
            "19812  Thioredoxin_10       PF17991.6         136     2.42    142   0.59   0.59   0.51   0.02\n",
            "19813  Thioredoxin_11       PF18078.6          20     1.87    126   0.59   0.55   0.53   0.04\n",
            "19814  Thioredoxin_12       PF18400.6          90     3.38    190   0.59   0.56   0.48   0.03\n",
            "19815  Thioredoxin_13       PF18401.6         105     3.81    136   0.59   0.62   0.52   0.03\n",
            "19816  Thioredoxin_14       PF18402.6         132     5.49    255   0.59   0.58   0.46   0.02\n",
            "19817  Thioredoxin_15       PF18403.6         130     5.59    207   0.59   0.63   0.50   0.03\n",
            "19818  Thioredoxin_16       PF18569.6          20     1.80     93   0.61   0.59   0.54   0.04\n",
            "19819  Thioredoxin_17       PF21873.1           5     0.69     80   0.70   0.66   0.64   0.03\n",
            "19820  Thioredoxin_2        PF13098.11         77     8.39    104   0.59   0.56   0.45   0.02\n",
            "19821  Thioredoxin_3        PF13192.11         59     4.56     76   0.74   0.73   0.68   0.04\n",
            "19822  Thioredoxin_4        PF13462.11         19     3.43    165   0.59   0.57   0.50   0.02\n",
            "19823  Thioredoxin_5        PF13743.11         19     1.72    186   0.59   0.57   0.53   0.03\n",
            "19824  Thioredoxin_6        PF13848.11         39     8.56    184   0.59   0.57   0.51   0.04\n",
            "19825  Thioredoxin_7        PF13899.11         23     6.35     83   0.68   0.65   0.59   0.03\n",
            "19826  Thioredoxin_8        PF13905.11         44     7.58     94   0.61   0.58   0.51   0.03\n",
            "19827  Thioredoxin_9        PF14595.11         32     3.27    129   0.59   0.56   0.53   0.02\n",
            "19828  Thioredox_DsbH       PF03190.20         65     0.96    163   0.59   0.53   0.53   0.02\n",
            "19829  ThiP_synth           PF10120.14        103     4.30    165   0.59   0.61   0.53   0.02\n",
            "19830  ThiS                 PF02597.25        145    12.24     76   0.74   0.79   0.63   0.03\n",
            "19831  ThiS-like            PF14453.11         25     4.07     57   0.98   0.97   0.94   0.06\n",
            "19832  ThiW                 PF09512.15         18     1.26    150   0.59   0.64   0.53   0.08\n",
            "19833  Thi_PPkinase_C       PF21275.2          32     1.61     69   0.81   0.76   0.78   0.04\n",
            "19834  Tho1_MOS11_C         PF18592.6         143    16.35     37   1.47   1.57   1.43   0.24\n",
            "19835  Tho2                 PF11262.13        114     3.03    305   0.59   0.55   0.52   0.04\n",
            "19836  Thoc2                PF11732.13         84     2.75     76   0.74   0.72   0.71   0.04\n",
            "19837  THOC2_N              PF16134.10         74     2.85    632   0.59   0.59   0.50   0.02\n",
            "19838  THOC7                PF05615.18        122     7.46    136   0.59   0.61   0.51   0.09\n",
            "19839  THP2                 PF09432.15         13     1.02    129   0.59   0.58   0.53   0.05\n",
            "19840  THRAP3_BCLAF1        PF15440.11         24     2.32    647   0.59   0.57   0.47   0.10\n",
            "19841  ThrE                 PF06738.17         99     5.17    241   0.59   0.62   0.54   0.05\n",
            "19842  ThrE_2               PF12821.12         38     2.66    129   0.59   0.60   0.55   0.08\n",
            "19843  Thrombin_light       PF09396.15         19     1.60     48   1.15   1.13   1.12   0.09\n",
            "19844  Thr_dehydrat_C       PF00585.23         12     2.39     91   0.63   0.63   0.57   0.02\n",
            "19845  Thr_synth_N          PF14821.11        147     1.99     79   0.72   0.73   0.67   0.01\n",
            "19846  ThsA_Macro           PF20016.4          59     2.99    183   0.59   0.60   0.51   0.02\n",
            "19847  ThsB_TIR             PF08937.16         28     3.11    100   0.59   0.57   0.49   0.04\n",
            "19848  Tht1                 PF04163.17          2     0.64    595   0.59   0.57   0.50   0.03\n",
            "19849  ThuA                 PF06283.16        154     3.43    209   0.59   0.55   0.41   0.02\n",
            "19850  THUMP                PF02926.22       1131    12.78     84   0.68   0.68   0.61   0.04\n",
            "19851  Thump_like           PF18096.6          66     8.19     78   0.73   0.76   0.66   0.04\n",
            "19852  Thx                  PF17070.10          1     1.00     27   1.59   1.51   1.54   0.40\n",
            "19853  Thy1                 PF02511.20        254     5.30    191   0.59   0.56   0.45   0.03\n",
            "19854  ThylakoidFormat      PF11264.13         44     1.71    216   0.59   0.59   0.54   0.03\n",
            "19855  Thymidylate_kin      PF02223.22         22     3.88    185   0.59   0.60   0.52   0.02\n",
            "19856  Thymidylat_synt      PF00303.24        382     1.75    266   0.59   0.55   0.49   0.02\n",
            "19857  Thymopoietin         PF08198.16         14     2.25     48   1.15   1.18   1.12   0.07\n",
            "19858  Thymosin             PF01290.25         26     6.19     39   1.40   1.43   1.35   0.19\n",
            "19859  Thyroglobulin_1      PF00086.23        148     3.44     67   0.84   0.68   0.75   0.13\n",
            "19860  Thyroglob_assoc      PF16597.10          9     1.84     61   0.92   0.90   0.83   0.08\n",
            "19861  TH_ACT               PF21417.2           8     1.11     70   0.81   0.82   0.76   0.05\n",
            "19862  Tiam_CC_Ex           PF18385.6          21     0.96     98   0.59   0.58   0.54   0.03\n",
            "19863  TiaS-FLD             PF22641.1           9     1.31    156   0.59   0.60   0.52   0.02\n",
            "19864  TibC_1st             PF21129.2           6     0.99     63   0.89   0.85   0.84   0.03\n",
            "19865  Tic110               PF16940.10         13     0.94    574   0.59   0.62   0.53   0.02\n",
            "19866  TIC20                PF16166.10         13     1.15    181   0.59   0.55   0.52   0.04\n",
            "19867  Tic22                PF04278.17          7     0.92    244   0.59   0.61   0.50   0.02\n",
            "19868  tify                 PF06200.19         86    15.77     34   1.59   1.62   1.59   0.06\n",
            "19869  TIG                  PF01833.29         78    16.03     84   0.68   0.67   0.57   0.05\n",
            "19870  TIG_2                PF18020.6          21     2.65     94   0.61   0.57   0.54   0.03\n",
            "19871  TIG_plexin           PF17960.6          25     2.80     89   0.64   0.60   0.57   0.04\n",
            "19872  TIG_SUH              PF20144.4          42     2.18     94   0.61   0.59   0.53   0.01\n",
            "19873  TIL                  PF01826.22         94     2.75     55   1.01   0.73   0.92   0.38\n",
            "19874  TILa                 PF12714.12         31     2.21     55   1.01   0.75   0.95   0.29\n",
            "19875  TilS                 PF09179.16        183    18.83     67   0.84   0.85   0.75   0.09\n",
            "19876  TilS-like            PF22132.1           5     1.53     66   0.85   0.90   0.81   0.06\n",
            "19877  TilS_C               PF11734.13         50     5.73     74   0.76   0.75   0.72   0.03\n",
            "19878  TIL_2                PF22897.1           8     1.68     44   1.25   1.08   1.19   0.21\n",
            "19879  TIM                  PF00121.23        187     2.42    245   0.59   0.60   0.53   0.03\n",
            "19880  Tim17                PF02466.24        370     6.93    111   0.59   0.61   0.49   0.06\n",
            "19881  TIM21                PF08294.16         13     1.49    145   0.59   0.58   0.52   0.02\n",
            "19882  Tim29                PF10171.14         27     1.66    169   0.59   0.55   0.55   0.02\n",
            "19883  Tim44                PF04280.20         61     4.84    147   0.59   0.57   0.54   0.03\n",
            "19884  Tim54                PF11711.13         78     1.30    376   0.59   0.57   0.50   0.02\n",
            "19885  TIMELESS             PF04821.19        111     3.86    271   0.59   0.60   0.49   0.04\n",
            "19886  TIMELESS_C           PF05029.18         34     5.27     88   0.65   0.64   0.57   0.03\n",
            "19887  TIMP                 PF00965.22         11     1.09    183   0.59   0.49   0.51   0.06\n",
            "19888  TINF2_N              PF14973.11         19     1.49    149   0.59   0.59   0.53   0.03\n",
            "19889  TIP120               PF08623.15         92     2.24    166   0.59   0.60   0.53   0.03\n",
            "19890  TIP39                PF14980.11          4     1.11     51   1.08   1.09   1.04   0.08\n",
            "19891  TIP41                PF04176.18        126     1.41    174   0.59   0.57   0.49   0.03\n",
            "19892  TIP49                PF06068.18         15     0.81    347   0.59   0.62   0.52   0.02\n",
            "19893  TIP49_C              PF17856.6          15     2.29     66   0.85   0.90   0.82   0.05\n",
            "19894  Tipalpha             PF16753.10          2     0.56    151   0.59   0.59   0.52   0.03\n",
            "19895  TipAS                PF07739.18        197     5.98    117   0.59   0.54   0.53   0.06\n",
            "19896  TipE                 PF16972.10          3     0.55    486   0.59   0.57   0.51   0.01\n",
            "19897  TIP_N                PF12457.13         30     2.63    105   0.59   0.57   0.44   0.12\n",
            "19898  TIR                  PF01582.25         23     3.03    179   0.59   0.57   0.50   0.03\n",
            "19899  TIR_2                PF13676.11        345     9.11    122   0.59   0.58   0.43   0.02\n",
            "19900  TIR_3                PF18567.6          23     1.15    129   0.59   0.58   0.53   0.04\n",
            "19901  Tir_receptor_C       PF07489.16          6     0.72    221   0.59   0.62   0.51   0.04\n",
            "19902  Tir_receptor_M       PF03549.19          3     0.79     66   0.85   0.89   0.80   0.09\n",
            "19903  Tir_receptor_N       PF07490.16          4     0.53    269   0.59   0.63   0.52   0.02\n",
            "19904  Tis11B_N             PF04553.17         15     0.96    116   0.59   0.59   0.46   0.04\n",
            "19905  TisB_toxin           PF13939.11          1     1.00     28   1.32   1.41   1.27   0.28\n",
            "19906  Tissue_fac           PF01108.22         29     3.74    107   0.59   0.53   0.52   0.04\n",
            "19907  Titin_Z              PF09042.16         22     3.78     46   1.20   1.28   1.16   0.20\n",
            "19908  TK                   PF00265.23         11     1.21    176   0.59   0.56   0.52   0.03\n",
            "19909  Tk-SP_N-pro          PF18237.6           4     1.19     67   0.84   0.85   0.79   0.06\n",
            "19910  TKTI1                PF15041.11          6     0.50    183   0.59   0.52   0.52   0.03\n",
            "19911  TLC                  PF03219.19         11     0.94    491   0.59   0.58   0.53   0.05\n",
            "19912  TLD                  PF07534.21         96     5.33    139   0.59   0.54   0.46   0.03\n",
            "19913  Tlde1_dom            PF10908.13         18     2.01    120   0.59   0.54   0.51   0.02\n",
            "19914  TLE_N                PF03920.20         23     0.91    118   0.59   0.57   0.52   0.05\n",
            "19915  Tli4_C               PF18426.6          42     3.48    161   0.59   0.58   0.52   0.01\n",
            "19916  Tli4_N               PF18443.6          22     2.88    138   0.59   0.58   0.50   0.02\n",
            "19917  TLN1-like_RS         PF21865.1          50     2.05    165   0.59   0.62   0.55   0.05\n",
            "19918  Tlp                  PF19824.4          13     1.37     70   0.80   0.80   0.74   0.14\n",
            "19919  TLP-20               PF06088.16         21     2.07    164   0.59   0.59   0.50   0.04\n",
            "19920  TLP1_add_C           PF18313.6          22     1.88     83   0.68   0.71   0.61   0.03\n",
            "19921  Tlr3_TMD             PF17968.6          16     3.10     33   1.64   1.60   1.62   0.20\n",
            "19922  TLV_coat             PF00429.24         16     1.78    560   0.59   0.56   0.49   0.02\n",
            "19923  TM1266-like          PF21699.2         117     1.37     74   0.76   0.81   0.72   0.06\n",
            "19924  TM140                PF14985.11         13     0.59    180   0.59   0.61   0.53   0.04\n",
            "19925  TM1506               PF08973.15         34     1.86    130   0.59   0.60   0.55   0.03\n",
            "19926  TM1586_NiRdase       PF14512.11         22     2.47    215   0.59   0.57   0.52   0.02\n",
            "19927  TM2                  PF05154.21        110    10.97     51   1.09   1.07   1.02   0.20\n",
            "19928  TM231                PF10149.14         29     2.41    301   0.59   0.56   0.53   0.04\n",
            "19929  Tma16                PF11176.13        124     3.00    146   0.59   0.57   0.54   0.06\n",
            "19930  TMA7                 PF09072.15         54     2.65     62   0.90   0.96   0.86   0.20\n",
            "19931  TMC                  PF07810.18         72     1.36    111   0.59   0.55   0.54   0.05\n",
            "19932  TmcA_N               PF08351.16         30     3.04    178   0.59   0.59   0.50   0.02\n",
            "19933  TMCCDC2              PF15844.10          8     0.58    171   0.59   0.58   0.52   0.03\n",
            "19934  TMCO5                PF14992.11         16     0.95    280   0.59   0.60   0.53   0.05\n",
            "19935  Tme5_EGF_like        PF09064.15          3     1.75     34   1.59   1.33   1.52   0.33\n",
            "19936  TMEM100              PF16311.10          4     0.65    133   0.59   0.60   0.51   0.03\n",
            "19937  TMEM101              PF15111.11          5     0.77    252   0.59   0.58   0.52   0.04\n",
            "19938  TMEM106              PF07092.17         12     0.78    141   0.59   0.56   0.53   0.04\n",
            "19939  TMEM106_N            PF21002.2          12     0.95     91   0.63   0.61   0.56   0.03\n",
            "19940  TMEM107              PF14995.11         28     3.31    123   0.59   0.56   0.54   0.08\n",
            "19941  TMEM108              PF15759.10          6     0.71    514   0.59   0.59   0.51   0.03\n",
            "19942  TMEM117              PF15113.11          7     0.52    411   0.59   0.54   0.52   0.04\n",
            "19943  TMEM119              PF15724.10          8     0.60    252   0.59   0.61   0.52   0.02\n",
            "19944  TMEM125              PF15109.11          3     0.47    111   0.59   0.61   0.52   0.05\n",
            "19945  TMEM126              PF07114.16         30     1.42    181   0.59   0.59   0.54   0.03\n",
            "19946  TMEM127              PF20517.3          14     1.44    113   0.59   0.59   0.55   0.08\n",
            "19947  TMEM128              PF20479.3          73     4.76    124   0.59   0.55   0.52   0.11\n",
            "19948  TMEM131_like         PF19532.4           9     0.87   1428   0.59   0.56   0.52   0.02\n",
            "19949  TMEM131_like_N       PF12371.13         10     1.88     84   0.68   0.67   0.64   0.03\n",
            "19950  TMEM132              PF16070.10         41     2.26    345   0.59   0.60   0.52   0.03\n",
            "19951  TMEM132D_C           PF15706.10         34     1.66     87   0.65   0.61   0.57   0.02\n",
            "19952  TMEM132D_N           PF15705.10         40     1.27    129   0.59   0.56   0.53   0.02\n",
            "19953  TMEM135_C_rich       PF15982.10          3     0.54    134   0.59   0.57   0.52   0.03\n",
            "19954  TMEM138              PF14935.11         20     1.24    119   0.59   0.56   0.54   0.09\n",
            "19955  TMEM141              PF15110.11         15     1.07     91   0.63   0.59   0.58   0.03\n",
            "19956  TMEM144              PF07857.17          6     0.83    332   0.59   0.58   0.52   0.04\n",
            "19957  TMEM145_N            PF21892.1          38     1.01    131   0.59   0.49   0.51   0.05\n",
            "19958  TMEM151              PF14857.11         13     0.79    428   0.59   0.54   0.51   0.02\n",
            "19959  TMEM154              PF15102.11          5     1.28    153   0.59   0.60   0.51   0.04\n",
            "19960  TMEM156              PF15106.11         13     0.98    226   0.59   0.54   0.53   0.05\n",
            "19961  TMEM164              PF14808.11         86     6.12    217   0.59   0.55   0.48   0.10\n",
            "19962  TMEM169              PF15052.11         13     0.63    131   0.59   0.54   0.53   0.05\n",
            "19963  TMEM171              PF15471.11         15     0.76    320   0.59   0.59   0.53   0.01\n",
            "19964  TMEM173              PF15009.11         26     2.02    293   0.59   0.58   0.52   0.02\n",
            "19965  TMEM174              PF15029.11          7     0.65    233   0.59   0.58   0.52   0.01\n",
            "19966  TMEM175              PF06736.16        169     3.28     88   0.65   0.63   0.57   0.07\n",
            "19967  TMEM18               PF14770.11         48     1.84    118   0.59   0.56   0.55   0.06\n",
            "19968  TMEM181_GOLD         PF21885.1          60     1.16    111   0.59   0.53   0.51   0.04\n",
            "19969  TMEM187              PF15100.11         14     0.89    240   0.59   0.55   0.52   0.04\n",
            "19970  TMEM190              PF15431.11          1     0.35    128   0.60   0.54   0.52   0.02\n",
            "19971  TMEM191C             PF15194.11          5     0.61    121   0.59   0.65   0.53   0.03\n",
            "19972  TMEM192              PF14802.11         24     2.39    236   0.59   0.57   0.53   0.04\n",
            "19973  TMEM206              PF15122.11          7     0.50    296   0.59   0.56   0.52   0.02\n",
            "19974  TMEM208_SND2         PF05620.16        141     3.52    167   0.59   0.57   0.52   0.03\n",
            "19975  TMEM210              PF15195.11          7     0.55    114   0.59   0.61   0.52   0.02\n",
            "19976  TMEM213              PF15192.11          3     0.52     79   0.72   0.68   0.65   0.03\n",
            "19977  TMEM214              PF10151.14         13     1.22    660   0.59   0.57   0.52   0.02\n",
            "19978  TMEM215              PF15746.10          6     0.61    225   0.59   0.58   0.51   0.01\n",
            "19979  TMEM219              PF14940.11          7     1.44    240   0.59   0.56   0.51   0.02\n",
            "19980  TMEM220              PF15071.11         39     2.64    100   0.59   0.56   0.49   0.04\n",
            "19981  TMEM232              PF15877.10         13     0.77    452   0.59   0.57   0.53   0.03\n",
            "19982  TMEM234              PF10639.14         14     1.24    113   0.59   0.60   0.52   0.05\n",
            "19983  TMEM237              PF15383.11         16     1.45    247   0.59   0.58   0.50   0.02\n",
            "19984  TMEM238              PF15125.11         15     1.38     64   0.88   0.85   0.82   0.14\n",
            "19985  TMEM239              PF15841.10          2     0.45    155   0.59   0.56   0.51   0.05\n",
            "19986  TMEM240              PF15207.11          2     0.43    189   0.59   0.55   0.51   0.02\n",
            "19987  TMEM247              PF15444.11          7     0.54    206   0.59   0.57   0.51   0.01\n",
            "19988  TMEM249              PF15158.11          6     0.82    184   0.59   0.55   0.53   0.03\n",
            "19989  TMEM251              PF15190.11          3     0.47    131   0.59   0.56   0.51   0.04\n",
            "19990  TMEM252              PF15664.10         11     0.70    139   0.59   0.58   0.53   0.01\n",
            "19991  TMEM254              PF14934.11         35     3.28     91   0.63   0.60   0.58   0.06\n",
            "19992  Tmem26               PF09772.14          7     0.99    285   0.59   0.59   0.53   0.05\n",
            "19993  TMEM260-like         PF11028.13         81     2.19    179   0.59   0.58   0.53   0.04\n",
            "19994  TMEM33_Pom33         PF03661.18         19     1.73    240   0.59   0.58   0.53   0.05\n",
            "19995  TMEM37               PF15108.11          7     0.54    182   0.59   0.58   0.52   0.05\n",
            "19996  TMEM40               PF15817.10          3     0.42    120   0.59   0.58   0.51   0.02\n",
            "19997  TMEM43               PF07787.17         52     3.51    250   0.59   0.55   0.51   0.03\n",
            "19998  TMEM51               PF15345.11         11     0.91    237   0.59   0.60   0.50   0.02\n",
            "19999  TMEM52               PF14979.11         10     0.83    132   0.59   0.56   0.52   0.03\n",
            "20000  TMEM61               PF15105.11          5     0.60    189   0.59   0.59   0.53   0.01\n",
            "20001  TMEM65               PF10507.14         44     1.25    109   0.59   0.62   0.54   0.04\n",
            "20002  TMEM70               PF06979.17         44     3.77    157   0.59   0.57   0.51   0.03\n",
            "20003  TMEM71               PF15121.11         11     0.75    152   0.59   0.56   0.52   0.02\n",
            "20004  TMEM72               PF16054.10          7     1.21    135   0.59   0.58   0.54   0.06\n",
            "20005  TMEM82               PF15816.10         12     0.74    311   0.59   0.60   0.53   0.04\n",
            "20006  TMEM87A-B_GOLD       PF21901.1          24     1.29    158   0.59   0.52   0.51   0.03\n",
            "20007  TMEM89               PF15098.11          8     0.52    134   0.59   0.56   0.52   0.02\n",
            "20008  TMEM95               PF15203.11          9     0.64    151   0.59   0.54   0.52   0.04\n",
            "20009  Tmemb_14             PF03647.18        193     3.90     92   0.62   0.68   0.55   0.11\n",
            "20010  Tmemb_161AB          PF10268.14         21     1.19    485   0.59   0.58   0.53   0.04\n",
            "20011  Tmemb_170            PF10190.14         25     1.79    106   0.59   0.58   0.55   0.10\n",
            "20012  Tmemb_185A           PF10269.14         24     2.14    252   0.59   0.56   0.48   0.07\n",
            "20013  Tmemb_18A            PF09771.14         19     0.99    118   0.59   0.59   0.53   0.03\n",
            "20014  Tmemb_40             PF10160.14         15     1.19    255   0.59   0.56   0.52   0.06\n",
            "20015  Tmemb_55A            PF09788.14         16     0.82    248   0.59   0.53   0.50   0.04\n",
            "20016  Tmemb_9              PF05434.16         16     1.01    145   0.59   0.54   0.51   0.05\n",
            "20017  Tmemb_cc2            PF10267.14         25     1.58    397   0.59   0.60   0.50   0.05\n",
            "20018  TMEMspv1-c74-12      PF11044.13          1     0.79     49   1.13   1.13   1.08   0.07\n",
            "20019  TMEM_230_134         PF05915.17          9     1.01    119   0.59   0.58   0.50   0.02\n",
            "20020  TMF_DNA_bd           PF12329.13         45     8.92     74   0.76   0.85   0.75   0.20\n",
            "20021  TMF_TATA_bd          PF12325.13         76     2.78    115   0.59   0.63   0.57   0.10\n",
            "20022  TMIE                 PF16038.10         12     1.16     85   0.67   0.63   0.62   0.06\n",
            "20023  TmoB                 PF06234.17         10     1.95     78   0.73   0.75   0.66   0.04\n",
            "20024  TMP                  PF05017.19         95    95.00     11   1.56   1.42   1.56   0.40\n",
            "20025  TMP-TENI             PF02581.22         72     3.77    180   0.59   0.62   0.54   0.02\n",
            "20026  Tmp39                PF10271.14         10     1.08    422   0.59   0.55   0.51   0.04\n",
            "20027  TMPIT                PF07851.18         35     1.33    324   0.59   0.55   0.53   0.04\n",
            "20028  Tmpp129              PF10272.14         31     1.40    349   0.59   0.51   0.52   0.04\n",
            "20029  TMP_2                PF06791.18         23     2.56    208   0.59   0.63   0.54   0.05\n",
            "20030  TMP_3                PF20155.4         115     7.61    192   0.59   0.64   0.56   0.06\n",
            "20031  TMTC_DUF1736         PF08409.16         78     2.60     75   0.75   0.70   0.72   0.05\n",
            "20032  TMV_coat             PF00721.26          7     2.10    163   0.59   0.58   0.48   0.03\n",
            "20033  TM_ErbB1             PF21314.2          82    15.53     36   1.51   1.62   1.50   0.34\n",
            "20034  TM_PBP2_N            PF16296.10         18     1.60     83   0.68   0.70   0.62   0.03\n",
            "20035  Tn7_Tnp_TnsA_C       PF08721.16         39     6.21     83   0.68   0.68   0.63   0.04\n",
            "20036  Tn7_TnsA-like_N      PF08722.16         93     7.10     85   0.67   0.63   0.49   0.05\n",
            "20037  Tn7_TnsC_Int         PF11426.13          4     2.18     47   1.17   1.13   1.14   0.08\n",
            "20038  Tn916-Xis            PF09035.15          2     0.92     69   0.81   0.82   0.75   0.03\n",
            "20039  Tna_leader           PF08053.16          1     1.00     24   1.56   1.38   1.51   0.19\n",
            "20040  TNF                  PF00229.23         25     2.80    127   0.59   0.57   0.50   0.03\n",
            "20041  TNFR_16_TM           PF18422.6           4     1.46     38   1.43   1.39   1.39   0.10\n",
            "20042  TNFR_c6              PF00020.23        439    16.26     39   1.40   1.09   1.31   0.41\n",
            "20043  TniB                 PF05621.16         22     2.02    189   0.59   0.59   0.55   0.02\n",
            "20044  TniQ                 PF06527.16         67     7.19    142   0.59   0.53   0.47   0.03\n",
            "20045  TNK-like_SAM         PF22931.2         150     7.04     56   0.99   1.03   0.97   0.08\n",
            "20046  TNP-like_GBD         PF21788.2         157     2.71    119   0.59   0.56   0.53   0.04\n",
            "20047  TNP-like_RNaseH_C    PF21789.2          65    10.10     35   1.55   1.45   1.54   0.09\n",
            "20048  TNP-like_RNaseH_N    PF21787.2         120     3.55    137   0.59   0.58   0.54   0.02\n",
            "20049  TnpB_IS66            PF05717.18        120     2.07    101   0.59   0.57   0.54   0.02\n",
            "20050  TnpV                 PF14198.11         68     1.99    111   0.59   0.56   0.54   0.04\n",
            "20051  TnpW                 PF14202.11         16    10.72     35   1.55   1.56   1.54   0.16\n",
            "20052  Tnp_22_dsRBD         PF17490.7           1     0.58     65   0.86   0.86   0.80   0.02\n",
            "20053  Tnp_22_trimer        PF17489.7           1     0.88     43   1.27   1.21   1.22   0.15\n",
            "20054  Tnp_DNA_bind         PF14706.11         22     3.25     58   0.96   0.95   0.93   0.05\n",
            "20055  Tnp_P_element        PF12017.13         41     2.29     73   0.77   0.73   0.69   0.06\n",
            "20056  Tnp_P_element_C      PF12596.13         11     2.12    107   0.59   0.58   0.45   0.05\n",
            "20057  Tnp_zf-ribbon_2      PF13842.11          8     5.55     31   1.74   1.34   1.60   0.59\n",
            "20058  TNRC6-PABC_bdg       PF16608.10          7     0.75    291   0.59   0.56   0.49   0.04\n",
            "20059  TnsD                 PF15978.10         15     2.99    355   0.59   0.56   0.52   0.03\n",
            "20060  TnsE_C               PF18623.6          12     1.81    145   0.59   0.57   0.54   0.03\n",
            "20061  TNT                  PF14021.11         93     3.11     88   0.65   0.66   0.57   0.01\n",
            "20062  TNV_CP               PF03898.18          4     0.79    198   0.59   0.60   0.52   0.03\n",
            "20063  Toast-rack_PspC-Cterm PF22744.1          17     1.87    136   0.59   0.56   0.55   0.04\n",
            "20064  Toast_rack_N         PF17115.10          5     1.45     92   0.62   0.63   0.56   0.03\n",
            "20065  TOBE                 PF03459.22         39    13.61     64   0.88   0.95   0.82   0.04\n",
            "20066  TOBE_2               PF08402.15         91    10.36     76   0.74   0.77   0.65   0.03\n",
            "20067  TOBE_3               PF12857.12         49    18.28     59   0.95   1.03   0.89   0.06\n",
            "20068  Tobravirus_2B        PF05271.16          4     0.55    117   0.59   0.54   0.52   0.03\n",
            "20069  TOC159_MAD           PF11886.13         23     1.27    267   0.59   0.60   0.54   0.02\n",
            "20070  Tocopherol_cycl      PF14249.11         31     0.90    335   0.59   0.53   0.51   0.02\n",
            "20071  TOD1_MUCI70          PF04765.18         30     0.89    314   0.59   0.55   0.52   0.02\n",
            "20072  Toddler              PF22050.1          32     1.75     58   0.96   0.88   0.91   0.12\n",
            "20073  TOH_N                PF12549.13          7     6.48     25   2.13   2.20   2.13   0.25\n",
            "20074  TolA                 PF06519.16          9     1.14     96   0.60   0.60   0.54   0.03\n",
            "20075  TolA_bind_tri        PF16331.10         50     4.87     76   0.74   0.77   0.72   0.13\n",
            "20076  TolB_like            PF15869.10          5     1.43    296   0.59   0.56   0.50   0.02\n",
            "20077  TolB_N               PF04052.18         78     3.51    106   0.59   0.61   0.53   0.02\n",
            "20078  Toluene_X            PF03349.21          9     1.80    421   0.59   0.58   0.51   0.02\n",
            "20079  TOM13                PF08219.16         43     0.98     84   0.68   0.66   0.60   0.01\n",
            "20080  TOM20_plant          PF06552.17          5     0.68    187   0.59   0.58   0.53   0.02\n",
            "20081  Tom22                PF04281.18         80     1.85    141   0.59   0.61   0.52   0.04\n",
            "20082  Tom37                PF10568.14        130     5.26    124   0.59   0.55   0.49   0.03\n",
            "20083  Tom37_C              PF11801.13         19     2.08    145   0.59   0.59   0.53   0.04\n",
            "20084  Tom5                 PF10642.14         70    11.51     46   1.20   1.20   1.14   0.07\n",
            "20085  Tom6                 PF17112.10         16     2.04     45   1.22   1.24   1.19   0.08\n",
            "20086  TOM6p                PF15184.11          2     0.66     74   0.77   0.77   0.69   0.02\n",
            "20087  Tom7                 PF08038.17         53     5.08     41   1.34   1.32   1.26   0.08\n",
            "20088  Tombus_movement      PF05318.17          6     1.60     71   0.79   0.80   0.70   0.07\n",
            "20089  Tombus_P19           PF03220.18          2     0.49    171   0.59   0.58   0.52   0.01\n",
            "20090  Tombus_P33           PF08500.15         10     1.34    142   0.59   0.59   0.50   0.03\n",
            "20091  ToMV_Hel_N           PF20896.2           5     0.92    158   0.59   0.58   0.50   0.02\n",
            "20092  TonB_2               PF13103.11         43     6.22     85   0.67   0.66   0.57   0.02\n",
            "20093  TonB_C               PF03544.19         48     6.99     79   0.72   0.72   0.67   0.02\n",
            "20094  TonB_dep_Rec_b-barrel PF00593.29        657    73.58    436   0.59   0.56   0.34   0.07\n",
            "20095  TonB_N               PF16031.10          4     0.81    138   0.59   0.61   0.48   0.13\n",
            "20096  Top1B_N_bact         PF21338.2         186     2.72     49   1.13   1.09   1.10   0.03\n",
            "20097  TOP1_ZnF             PF21372.2          25     1.47     45   1.22   1.14   1.18   0.08\n",
            "20098  TOP6A-Spo11_Toprim   PF21180.2          79     1.78    170   0.59   0.59   0.53   0.02\n",
            "20099  Top6b_C              PF18000.6          13     1.09    111   0.59   0.59   0.52   0.04\n",
            "20100  TopBP1_BRCT0         PF21298.2           5     0.53     96   0.59   0.57   0.52   0.03\n",
            "20101  Topo-VIb_trans       PF09239.16         62     1.34    159   0.59   0.60   0.52   0.02\n",
            "20102  Topo-V_HhH_2_2nd     PF21616.2           1     0.69     58   0.96   1.02   0.90   0.06\n",
            "20103  Topoisom_bac         PF01131.25         56     3.18    411   0.59   0.59   0.49   0.03\n",
            "20104  Topoisom_I           PF01028.25         88     1.82    228   0.59   0.58   0.52   0.04\n",
            "20105  Topoisom_I_N         PF02919.20        114     1.12    213   0.59   0.53   0.53   0.03\n",
            "20106  TopoV_HHH            PF22409.1           1     1.00     29   1.35   1.43   1.30   0.13\n",
            "20107  Topo_C_assoc         PF14370.11         64     1.82     70   0.80   0.74   0.75   0.06\n",
            "20108  Topo_VI_alpha        PF20768.2          46     2.88     50   1.11   1.13   1.03   0.06\n",
            "20109  Topo_Zn_Ribbon       PF08272.16         40     5.96     41   1.33   1.27   1.27   0.09\n",
            "20110  Toprim               PF01751.27        132    10.84     96   0.60   0.63   0.46   0.02\n",
            "20111  Toprim_2             PF13155.11         76     3.90     88   0.65   0.66   0.54   0.03\n",
            "20112  Toprim_3             PF13362.11         14     3.46     95   0.60   0.64   0.49   0.03\n",
            "20113  Toprim_4             PF13662.11         74     8.32     85   0.67   0.71   0.55   0.03\n",
            "20114  TOPRIM_C             PF16898.10        136     1.50    128   0.59   0.53   0.50   0.06\n",
            "20115  Toprim_Crpt          PF13342.11        138     6.03     60   0.93   0.94   0.87   0.07\n",
            "20116  Toprim_C_rpt         PF13368.11        264     8.94     57   0.98   1.01   0.93   0.04\n",
            "20117  TOP_N                PF19310.4          43     0.84    123   0.59   0.60   0.54   0.03\n",
            "20118  TOR1A_C              PF21376.2          13     1.09     58   0.96   0.89   0.92   0.06\n",
            "20119  TORC_C               PF12886.12         15     1.32     75   0.75   0.79   0.67   0.04\n",
            "20120  TORC_M               PF12885.12         19     1.16    158   0.59   0.59   0.51   0.03\n",
            "20121  TORC_N               PF12884.12         10     1.79     63   0.89   0.88   0.80   0.07\n",
            "20122  Torsin               PF06309.16          8     0.77    127   0.59   0.55   0.53   0.02\n",
            "20123  TorS_sensor_domain   PF21689.2          13     0.85    131   0.59   0.62   0.53   0.05\n",
            "20124  TorS_sen_N           PF22519.1          39     2.61     87   0.65   0.71   0.62   0.06\n",
            "20125  Torus                PF16131.10         61     1.07    109   0.59   0.51   0.50   0.05\n",
            "20126  Tor_FN3_1nd          PF23006.1          12     1.42    115   0.59   0.54   0.51   0.04\n",
            "20127  Tor_FN3_2nd          PF21468.2           1     0.41     86   0.66   0.60   0.59   0.03\n",
            "20128  Tor_FN3_3nd          PF23008.1           6     0.81     92   0.62   0.59   0.56   0.03\n",
            "20129  Tospov_NS-S_N        PF03231.18          9     1.68    351   0.59   0.57   0.52   0.03\n",
            "20130  Tospo_nucleocap      PF01533.21          1     0.40    246   0.59   0.60   0.52   0.02\n",
            "20131  Totivirus_coat       PF05518.16          7     1.70    759   0.59   0.59   0.53   0.01\n",
            "20132  Tower                PF09121.15         14     2.90     42   1.31   1.31   1.28   0.17\n",
            "20133  Tox-ART-HYD1         PF15633.11          6     1.64     97   0.59   0.58   0.51   0.03\n",
            "20134  Tox-ART-HYE1         PF15634.11          1     0.39    282   0.59   0.58   0.52   0.02\n",
            "20135  Tox-GHH              PF15636.11         25     2.97     78   0.73   0.70   0.68   0.06\n",
            "20136  Tox-GHH2             PF15635.11          3     0.91    122   0.59   0.57   0.52   0.05\n",
            "20137  Tox-HDC              PF15656.11          3     0.79    130   0.59   0.58   0.50   0.02\n",
            "20138  Tox-HNH-EHHH         PF15657.11         28     4.23     69   0.81   0.71   0.71   0.11\n",
            "20139  Tox-HNH-HHH          PF15637.11          5     0.83    103   0.59   0.53   0.53   0.03\n",
            "20140  Tox-MPTase2          PF15638.11          5     1.45    196   0.59   0.56   0.52   0.04\n",
            "20141  Tox-MPTase3          PF15639.11          3     0.78    131   0.59   0.56   0.49   0.02\n",
            "20142  Tox-MPTase4          PF15640.11          6     0.81    132   0.59   0.57   0.53   0.04\n",
            "20143  Tox-MPTase5          PF15641.11          1     0.38    110   0.59   0.57   0.51   0.02\n",
            "20144  Tox-ODYAM1           PF15642.11          3     0.58    385   0.59   0.60   0.52   0.02\n",
            "20145  Tox-PAAR-like        PF13665.11         57     2.90    108   0.59   0.56   0.54   0.04\n",
            "20146  Tox-PL-2             PF15643.11          7     1.03    103   0.59   0.56   0.52   0.03\n",
            "20147  Tox-PLDMTX           PF15645.11          3     0.89    142   0.59   0.55   0.50   0.02\n",
            "20148  Tox-REase-2          PF15646.11          5     1.19    129   0.59   0.58   0.49   0.03\n",
            "20149  Tox-REase-3          PF15647.11          3     0.74    102   0.59   0.59   0.49   0.03\n",
            "20150  Tox-REase-5          PF15648.11         12     1.73     95   0.60   0.54   0.52   0.04\n",
            "20151  Tox-REase-7          PF15649.11          9     1.70     84   0.68   0.69   0.60   0.04\n",
            "20152  Tox-REase-9          PF15650.11          6     0.95     88   0.65   0.63   0.57   0.03\n",
            "20153  Tox-SGS              PF15651.11          2     0.43    109   0.59   0.52   0.51   0.02\n",
            "20154  Tox-SHH              PF15652.11          5     0.90     96   0.60   0.55   0.49   0.04\n",
            "20155  Tox-URI2             PF15653.11          5     1.59     86   0.66   0.64   0.59   0.05\n",
            "20156  Tox-WTIP             PF15654.11          2     0.64     79   0.71   0.68   0.64   0.04\n",
            "20157  ToxA_barrel          PF22771.1           1     0.43     89   0.64   0.66   0.57   0.02\n",
            "20158  ToxA_N               PF20178.3         102    15.46    249   0.59   0.60   0.49   0.03\n",
            "20159  ToxB_N               PF18224.6           7     1.97     61   0.92   0.87   0.87   0.08\n",
            "20160  Toxin-deaminase      PF14424.11          6     1.46    146   0.59   0.58   0.50   0.04\n",
            "20161  Toxin-JAB1           PF15659.11         12     2.49     87   0.65   0.61   0.56   0.06\n",
            "20162  Toxin_10             PF05431.16          2     0.67    188   0.59   0.56   0.51   0.02\n",
            "20163  Toxin_11             PF07473.16          2     2.00     28   1.74   1.40   1.69   0.51\n",
            "20164  Toxin_12             PF07740.17         31     8.32     30   1.80   1.34   1.72   0.61\n",
            "20165  Toxin_13             PF07822.16          2     0.65     55   1.01   0.85   0.95   0.18\n",
            "20166  Toxin_14             PF07829.16          2     1.33     26   2.06   1.70   2.01   0.62\n",
            "20167  Toxin_15             PF07906.18          5     1.26    278   0.59   0.57   0.51   0.03\n",
            "20168  Toxin_16             PF07945.16          2     0.91     36   1.51   1.30   1.45   0.39\n",
            "20169  Toxin_17             PF08086.16          4     0.91     41   1.33   1.09   1.29   0.34\n",
            "20170  Toxin_18             PF08087.16          7     1.81     31   1.74   1.53   1.70   0.37\n",
            "20171  Toxin_19             PF08088.17          8     1.64     40   1.37   1.04   1.28   0.42\n",
            "20172  Toxin_2              PF00451.24         47     7.32     32   1.69   1.33   1.66   0.58\n",
            "20173  Toxin_20             PF08089.16          3     0.96     39   1.40   1.22   1.31   0.28\n",
            "20174  Toxin_21             PF08091.16          2     1.13     40   1.37   1.11   1.30   0.36\n",
            "20175  Toxin_22             PF08092.16          3     1.23     38   1.44   1.12   1.36   0.30\n",
            "20176  Toxin_23             PF08093.16          5     2.54     30   1.79   1.49   1.73   0.48\n",
            "20177  Toxin_24             PF08094.16          2     1.19     33   1.64   1.43   1.59   0.37\n",
            "20178  Toxin_25             PF08095.16          1     1.00     22   1.79   1.49   1.75   0.45\n",
            "20179  Toxin_26             PF08097.16          2     2.00     11   3.38   2.53   3.35   1.61\n",
            "20180  Toxin_27             PF08099.16          2     1.08     33   1.64   1.44   1.59   0.40\n",
            "20181  Toxin_28             PF08115.16          2     1.43     35   1.55   1.32   1.51   0.18\n",
            "20182  Toxin_29             PF08116.16          4     1.49     31   1.74   1.45   1.70   0.41\n",
            "20183  Toxin_3              PF00537.23         33     2.40     55   1.01   0.79   0.94   0.24\n",
            "20184  Toxin_30             PF08117.16          2     1.29     36   1.51   1.31   1.45   0.28\n",
            "20185  Toxin_31             PF08119.16          2     0.89     37   1.47   1.31   1.42   0.27\n",
            "20186  Toxin_32             PF08120.16          2     0.93     35   1.55   1.37   1.51   0.27\n",
            "20187  Toxin_33             PF08121.16          2     1.46     22   2.40   2.18   2.37   0.70\n",
            "20188  Toxin_34             PF08396.15          7     0.89     75   0.76   0.57   0.66   0.21\n",
            "20189  Toxin_35             PF10530.14          9     1.73     61   0.91   0.72   0.86   0.25\n",
            "20190  Toxin_36             PF10550.14         13    13.00     17   2.00   2.08   1.95   0.44\n",
            "20191  Toxin_37             PF11415.13          8     1.61     35   1.55   1.21   1.52   0.36\n",
            "20192  Toxin_38             PF14866.11         13     1.74     55   1.01   0.84   0.92   0.19\n",
            "20193  Toxin_4              PF00706.22          7     1.53     43   1.28   1.05   1.21   0.24\n",
            "20194  Toxin_5              PF05294.18          6     1.54     32   1.69   1.32   1.63   0.57\n",
            "20195  Toxin_6              PF05453.17          5     3.75     28   1.92   1.58   1.88   0.63\n",
            "20196  Toxin_7              PF05980.17         30     3.53     34   1.60   1.28   1.57   0.43\n",
            "20197  Toxin_8              PF07365.17         22     3.61     50   1.11   1.01   1.04   0.15\n",
            "20198  Toxin_9              PF02819.20          6     1.72     43   1.28   0.98   1.20   0.38\n",
            "20199  Toxin_BN_H           PF22133.1           5     0.50    276   0.59   0.56   0.52   0.04\n",
            "20200  Toxin_cobra-type     PF21947.1           9     1.29     63   0.89   0.71   0.82   0.18\n",
            "20201  Toxin_GhoT_OrtT      PF10753.14          9     1.37     55   1.01   0.99   0.96   0.16\n",
            "20202  Toxin_R_bind_C       PF07951.17          2     0.61    222   0.59   0.56   0.51   0.03\n",
            "20203  Toxin_R_bind_N       PF07953.18         13     1.11    195   0.59   0.56   0.52   0.06\n",
            "20204  Toxin_TOLIP          PF00087.26          3     0.78     68   0.82   0.67   0.74   0.14\n",
            "20205  Toxin_ToxA           PF11584.13          2     0.50    118   0.59   0.58   0.52   0.03\n",
            "20206  Toxin_trans          PF07952.17          2     0.54    323   0.59   0.59   0.53   0.03\n",
            "20207  Toxin_YhaV           PF11663.13         26     1.11    138   0.59   0.56   0.53   0.03\n",
            "20208  ToxN_toxin           PF13958.11         15     1.65    159   0.59   0.56   0.51   0.04\n",
            "20209  Toxofilin-like_dom   PF21609.2           2     0.70     84   0.68   0.69   0.62   0.04\n",
            "20210  ToxS                 PF17323.7           6     0.68    147   0.59   0.58   0.53   0.03\n",
            "20211  ToxT_N               PF22404.1           1     0.38    158   0.59   0.57   0.51   0.04\n",
            "20212  TP0453               PF20740.2           6     1.29    252   0.59   0.60   0.52   0.02\n",
            "20213  TP1                  PF02079.21          1     0.72     51   1.08   1.06   1.02   0.19\n",
            "20214  TP2                  PF01254.23          7     0.69    133   0.59   0.52   0.51   0.12\n",
            "20215  TP53IP5              PF15331.11         14     0.81    217   0.59   0.57   0.52   0.04\n",
            "20216  TP6A_N               PF04406.19        163     8.51     62   0.90   0.88   0.86   0.06\n",
            "20217  TpcC                 PF12642.12         20     4.79    231   0.59   0.58   0.49   0.04\n",
            "20218  TPD                  PF14811.11         28     1.20    138   0.59   0.55   0.54   0.02\n",
            "20219  TPD52                PF04201.20         17     0.83    108   0.59   0.62   0.53   0.05\n",
            "20220  TPH                  PF13868.11        114     9.32    352   0.59   0.60   0.57   0.25\n",
            "20221  tPilZ                PF16823.10          3     0.69    136   0.59   0.62   0.50   0.03\n",
            "20222  TPIP1                PF15338.11          2     0.43    123   0.59   0.61   0.52   0.02\n",
            "20223  TPK_B1_binding       PF04265.19        200     9.68     67   0.84   0.82   0.74   0.03\n",
            "20224  TPK_catalytic        PF04263.21        106     3.29    113   0.59   0.58   0.51   0.03\n",
            "20225  TPMT                 PF05724.16         11     1.50    218   0.59   0.56   0.52   0.01\n",
            "20226  TPM_phosphatase      PF04536.19        460     8.57    125   0.59   0.62   0.51   0.03\n",
            "20227  TPP1                 PF10341.14         38     5.70    119   0.59   0.57   0.50   0.03\n",
            "20228  TPPII                PF12580.13         45     2.56    187   0.59   0.59   0.54   0.03\n",
            "20229  TPPII_C              PF12583.13         26     2.88     69   0.81   0.80   0.77   0.06\n",
            "20230  TPPII_GBD            PF21316.2          16     1.16     91   0.63   0.59   0.57   0.03\n",
            "20231  TPPII_Ig-like-1      PF21223.2          20     1.06    117   0.59   0.58   0.53   0.02\n",
            "20232  TPP_enzyme_C         PF02775.26        323     6.41    151   0.59   0.58   0.48   0.02\n",
            "20233  TPP_enzyme_M         PF00205.27        209     6.50    137   0.59   0.62   0.53   0.02\n",
            "20234  TPP_enzyme_M_2       PF16582.10         15     1.23    208   0.59   0.55   0.53   0.03\n",
            "20235  TPP_enzyme_N         PF02776.23         68     6.29    119   0.59   0.61   0.53   0.02\n",
            "20236  TPR-S                PF20308.3         102     6.58    102   0.59   0.60   0.41   0.04\n",
            "20237  TPR1-like_2nd        PF21889.1          11     0.78    125   0.59   0.59   0.53   0.05\n",
            "20238  TPR_1                PF00515.33        549   549.00     34   1.23   1.21   1.24   0.12\n",
            "20239  TPR_10               PF13374.11        144   144.00     42   1.09   1.11   1.08   0.07\n",
            "20240  TPR_11               PF13414.11         43    25.69     42   1.31   1.32   1.30   0.12\n",
            "20241  TPR_12               PF13424.11        111    71.73     77   0.73   0.77   0.70   0.08\n",
            "20242  TPR_14               PF13428.11        338   338.00     44   0.63   0.69   0.61   0.09\n",
            "20243  TPR_15               PF13429.11         10     2.26    280   0.59   0.59   0.53   0.05\n",
            "20244  TPR_16               PF13432.11         57    57.00     65   0.81   0.89   0.75   0.13\n",
            "20245  TPR_17               PF13431.11        185   185.00     34   0.94   0.93   0.91   0.07\n",
            "20246  TPR_18               PF13512.11          4     0.70    145   0.59   0.56   0.53   0.03\n",
            "20247  TPR_19               PF14559.11        131    52.49     68   0.83   0.91   0.80   0.10\n",
            "20248  TPR_2                PF07719.22        361   361.00     34   0.94   0.95   0.95   0.11\n",
            "20249  TPR_20               PF14561.11         92     4.03     90   0.63   0.68   0.61   0.05\n",
            "20250  TPR_21               PF09976.14         46     2.22    193   0.59   0.60   0.55   0.05\n",
            "20251  TPR_22               PF18833.6         105     3.28     92   0.62   0.61   0.57   0.05\n",
            "20252  TPR_3                PF07720.17         17    17.00     34   1.36   1.32   1.31   0.19\n",
            "20253  TPR_4                PF07721.19        142   142.00     26   1.11   1.21   1.11   0.28\n",
            "20254  TPR_5                PF12688.12          5     0.84    119   0.59   0.64   0.53   0.03\n",
            "20255  TPR_6                PF13174.11        207   207.00     33   1.06   1.08   1.01   0.09\n",
            "20256  TPR_7                PF13176.11         74    74.00     36   0.88   0.93   0.84   0.08\n",
            "20257  TPR_8                PF13181.11         91    91.00     34   0.93   0.93   0.93   0.09\n",
            "20258  TPR_9                PF13371.11         38     9.03     73   0.77   0.80   0.75   0.06\n",
            "20259  TPR_EMC2             PF22890.1           1     0.39    145   0.59   0.59   0.52   0.02\n",
            "20260  TPR_MalT             PF17874.6          14     3.80    336   0.59   0.61   0.54   0.05\n",
            "20261  TPR_MLP1_2           PF07926.17        106     5.52    129   0.59   0.62   0.58   0.15\n",
            "20262  TPS                  PF05860.18         34     4.61    252   0.59   0.62   0.49   0.05\n",
            "20263  TPT                  PF03151.21         14     1.25    291   0.59   0.59   0.52   0.06\n",
            "20264  TPX2                 PF06886.16         74     4.68     77   0.73   0.73   0.68   0.15\n",
            "20265  TPX2_importin        PF12214.13         25     1.64    135   0.59   0.59   0.51   0.06\n",
            "20266  TP_methylase         PF00590.25         71     9.42    212   0.59   0.64   0.48   0.01\n",
            "20267  TQ                   PF18202.6          51     4.62    127   0.59   0.60   0.48   0.07\n",
            "20268  Tr-sialidase_C       PF11052.13         43    14.67     24   2.22   2.40   2.08   0.50\n",
            "20269  TR61B_FKBP-like      PF21985.1          59     4.44     55   1.01   1.04   0.98   0.04\n",
            "20270  TRA-1_regulated      PF02343.21         66     2.43    129   0.59   0.51   0.49   0.04\n",
            "20271  Tra1_central         PF20175.3          26     1.68    585   0.59   0.60   0.52   0.03\n",
            "20272  Tra1_ring            PF20206.3          28     3.31   1682   0.59   0.59   0.51   0.03\n",
            "20273  TraA                 PF05513.16          1     0.41    118   0.59   0.63   0.52   0.05\n",
            "20274  TrAA12               PF22807.1           1     0.39    463   0.59   0.60   0.51   0.03\n",
            "20275  TraB_PrgY_gumN       PF01963.22        183     9.55    263   0.59   0.60   0.48   0.02\n",
            "20276  TraC                 PF07820.17          5     0.98     88   0.65   0.71   0.60   0.06\n",
            "20277  TraC_F_IV            PF11130.13         36     6.20    231   0.59   0.58   0.50   0.01\n",
            "20278  TraD                 PF06412.16         29     5.73     62   0.90   0.97   0.86   0.07\n",
            "20279  TRADD-like_N         PF20694.2          29    20.00     69   0.82   0.86   0.75   0.04\n",
            "20280  TRADD_N              PF09034.15         13     1.01    112   0.59   0.59   0.54   0.04\n",
            "20281  TraD_N               PF12615.13          4     1.55     92   0.62   0.59   0.54   0.08\n",
            "20282  TraE                 PF05309.16         21     3.93    182   0.59   0.60   0.56   0.03\n",
            "20283  TraF                 PF13728.11         33     3.83    224   0.59   0.57   0.54   0.02\n",
            "20284  TRAF-mep_MATH        PF21355.2         146     2.04    125   0.59   0.55   0.53   0.02\n",
            "20285  TRAF2_zf             PF21341.2          12     5.67     19   2.77   2.50   2.75   0.43\n",
            "20286  TRAF3_RING           PF21363.2           1     0.83     40   1.37   1.19   1.32   0.27\n",
            "20287  TRAF6_Z2             PF18048.6          13     3.27     27   1.98   1.68   1.96   0.37\n",
            "20288  TRAFD1-XIAF1_ZnF     PF21366.2           6     1.17     43   1.28   1.16   1.24   0.12\n",
            "20289  TraF_2               PF13729.11         26     3.94    264   0.59   0.62   0.47   0.02\n",
            "20290  TRAF_BIRC3_bd        PF16673.10          5     1.18     64   0.87   0.90   0.83   0.08\n",
            "20291  TraG-D_C             PF12696.12         71     5.28    126   0.59   0.59   0.52   0.02\n",
            "20292  TraG_N               PF07916.16         37     7.08    462   0.59   0.58   0.44   0.03\n",
            "20293  TraH                 PF06122.16         26     3.09    358   0.59   0.58   0.48   0.02\n",
            "20294  TraH_2               PF06871.16          2     0.49    208   0.59   0.61   0.52   0.01\n",
            "20295  TraH_VirB8-like      PF22351.1           1     0.42    123   0.59   0.58   0.52   0.05\n",
            "20296  TraI-like_C          PF22287.1          16     1.39     85   0.67   0.69   0.62   0.03\n",
            "20297  TraI_2               PF07514.16          4     1.05    324   0.59   0.59   0.53   0.01\n",
            "20298  TraI_2B              PF18340.6          14     2.20     79   0.72   0.70   0.66   0.04\n",
            "20299  TraI_2_C             PF07515.16         17     1.57    126   0.59   0.56   0.52   0.03\n",
            "20300  TraI_C               PF07057.16          6     1.32    154   0.59   0.61   0.52   0.02\n",
            "20301  TraI_hel_assoc_N     PF22232.1           8     1.61     99   0.59   0.66   0.55   0.04\n",
            "20302  TraI_middle          PF22863.1          27     1.32     89   0.64   0.64   0.59   0.04\n",
            "20303  TraK                 PF06586.16         24     5.28    228   0.59   0.62   0.50   0.02\n",
            "20304  TraK_2               PF20739.2           1     0.38    198   0.59   0.58   0.52   0.02\n",
            "20305  TraL                 PF07178.16         17     2.57     90   0.63   0.60   0.58   0.04\n",
            "20306  TraL_transposon      PF13150.11          5     1.55     86   0.66   0.66   0.61   0.03\n",
            "20307  TRAM                 PF01938.25         27    11.78     61   0.92   0.97   0.83   0.05\n",
            "20308  TRAM_2               PF18693.6          81     4.13     63   0.89   0.92   0.81   0.05\n",
            "20309  TRAM_LAG1_CLN8       PF03798.21        128     7.90    198   0.59   0.51   0.48   0.10\n",
            "20310  Transcript_VP30      PF11507.13          2     0.57    131   0.59   0.59   0.52   0.02\n",
            "20311  Transcrip_act        PF04949.18         20     0.74    154   0.59   0.62   0.53   0.07\n",
            "20312  Transcrip_reg        PF01709.25        620     4.03    160   0.59   0.60   0.49   0.03\n",
            "20313  Transferase          PF02458.20          7     1.98    434   0.59   0.58   0.51   0.01\n",
            "20314  Transferrin          PF00405.22          8     0.64    328   0.59   0.56   0.52   0.02\n",
            "20315  Transformer          PF06495.16          4     0.62    182   0.59   0.55   0.48   0.08\n",
            "20316  Transglut_C          PF00927.27         25     4.45    106   0.59   0.60   0.52   0.02\n",
            "20317  Transglut_core       PF01841.24         82     6.64    108   0.59   0.54   0.48   0.02\n",
            "20318  Transglut_core2      PF13369.11         79     4.14    155   0.59   0.60   0.52   0.01\n",
            "20319  Transglut_core3      PF13471.11         44     5.45    117   0.59   0.57   0.53   0.03\n",
            "20320  Transglut_i_TM       PF14400.11         41     2.24    161   0.59   0.58   0.54   0.03\n",
            "20321  Transglut_N          PF00868.25        175     3.06    114   0.59   0.56   0.52   0.03\n",
            "20322  Transglut_prok       PF09017.15          4     0.45    414   0.59   0.56   0.51   0.02\n",
            "20323  Transgly             PF00912.27         93     2.53    178   0.59   0.60   0.51   0.02\n",
            "20324  Transglycosylas      PF06737.19         15     1.31     77   0.73   0.68   0.68   0.08\n",
            "20325  Transgly_assoc       PF04226.18         77     8.14     49   1.13   1.25   1.08   0.28\n",
            "20326  Transketolase_C      PF02780.25         80     5.66    124   0.59   0.63   0.54   0.02\n",
            "20327  Transketolase_C_1    PF22613.1         600     2.59    113   0.59   0.61   0.53   0.02\n",
            "20328  Transketolase_N      PF00456.26         11     0.83    334   0.59   0.58   0.53   0.01\n",
            "20329  Transket_pyr         PF02779.29         79     5.18    174   0.59   0.59   0.52   0.01\n",
            "20330  Translat_reg         PF01818.22          2     0.46    122   0.59   0.57   0.51   0.03\n",
            "20331  Translin             PF01997.21        202     3.81    205   0.59   0.59   0.48   0.04\n",
            "20332  Transmemb_17         PF09799.14         84     5.44    107   0.59   0.58   0.55   0.12\n",
            "20333  Transpeptidase       PF00905.27         40     5.93    299   0.59   0.59   0.47   0.03\n",
            "20334  Transport_MerF       PF11431.13         10     1.59     45   1.22   1.30   1.19   0.16\n",
            "20335  Transposase_1        PF01359.23         21     3.09     80   0.71   0.63   0.67   0.06\n",
            "20336  Transposase_20       PF02371.21         63     6.03     87   0.65   0.67   0.60   0.02\n",
            "20337  Transposase_21       PF02992.19          7     0.68    213   0.59   0.55   0.53   0.01\n",
            "20338  Transposase_22       PF02994.19          1     0.41     98   0.59   0.61   0.52   0.03\n",
            "20339  Transposase_23       PF03017.19          3     1.10     66   0.85   0.89   0.79   0.04\n",
            "20340  Transposase_24       PF03004.19         40     5.46    138   0.59   0.56   0.49   0.05\n",
            "20341  Transposase_28       PF04195.17          9     1.86     69   0.81   0.77   0.77   0.09\n",
            "20342  Transposase_29       PF21804.2           2     0.39    562   0.59   0.59   0.52   0.01\n",
            "20343  Transposase_31       PF04754.17         19     1.40    203   0.59   0.58   0.53   0.03\n",
            "20344  Transposase_32       PF20167.3         119    11.92    177   0.59   0.54   0.46   0.02\n",
            "20345  Transposase_mut      PF00872.23          7     1.21    379   0.59   0.59   0.54   0.02\n",
            "20346  Transposon_TraM      PF12508.13         21     2.13    195   0.59   0.61   0.54   0.03\n",
            "20347  Transpos_assoc       PF13963.11         15     1.38     74   0.76   0.62   0.69   0.06\n",
            "20348  Transp_cyt_pur       PF02133.20         14     3.13    440   0.59   0.58   0.52   0.06\n",
            "20349  Transp_inhibit       PF18791.6          70     2.65     47   1.17   1.13   1.15   0.06\n",
            "20350  Transp_Tc5_C         PF04236.20          8     1.06     63   0.89   0.70   0.82   0.12\n",
            "20351  Transthyretin        PF00576.26        280     1.43    109   0.59   0.56   0.50   0.02\n",
            "20352  Trans_coact          PF16805.10         33     4.00     68   0.83   0.85   0.77   0.07\n",
            "20353  Trans_reg_C          PF00486.33        259     5.88     77   0.73   0.76   0.70   0.03\n",
            "20354  Trans_reg_dom        PF20109.4         130     5.92     60   0.93   0.86   0.89   0.11\n",
            "20355  TraO                 PF10626.14         17     1.37    168   0.59   0.58   0.53   0.02\n",
            "20356  TraP                 PF07296.16          3     0.80    202   0.59   0.58   0.52   0.02\n",
            "20357  TRAP-delta           PF05404.17         32     1.82    162   0.59   0.57   0.54   0.03\n",
            "20358  TRAP-gamma           PF07074.17         15     0.83    170   0.59   0.59   0.53   0.05\n",
            "20359  TRAPP                PF04051.21        290     4.17    154   0.59   0.58   0.45   0.02\n",
            "20360  TRAPPC-Trs85         PF12739.12         60     3.96    453   0.59   0.56   0.48   0.03\n",
            "20361  TRAPPC10             PF12584.13         13     2.20    150   0.59   0.57   0.50   0.02\n",
            "20362  TRAPPC9-Trs120       PF08626.16         55     2.70   1229   0.59   0.59   0.48   0.02\n",
            "20363  TRAP_alpha           PF03896.21         53     2.98    180   0.59   0.59   0.50   0.02\n",
            "20364  TRAP_beta            PF05753.19         15     2.66    178   0.59   0.59   0.53   0.02\n",
            "20365  TraQ                 PF09679.15          2     0.57     94   0.61   0.61   0.53   0.03\n",
            "20366  TraQ_transposon      PF12988.12         18     0.97    133   0.59   0.56   0.54   0.03\n",
            "20367  TraS                 PF10624.14          1     0.36    162   0.59   0.56   0.51   0.02\n",
            "20368  TraT                 PF05818.17         18     1.64    214   0.59   0.61   0.52   0.03\n",
            "20369  TraU                 PF06834.16         38     2.70    306   0.59   0.52   0.47   0.02\n",
            "20370  TRAUB                PF08164.17        106     2.33     83   0.68   0.68   0.57   0.06\n",
            "20371  TraV                 PF09676.15         19     2.70    128   0.59   0.53   0.50   0.04\n",
            "20372  TraW_N               PF12477.13          9     9.00     30   1.58   1.66   1.56   0.19\n",
            "20373  TraX                 PF05857.16          7     1.59    217   0.59   0.59   0.51   0.06\n",
            "20374  TraY                 PF05509.16         23     7.77     49   1.13   1.16   1.12   0.08\n",
            "20375  Tra_M                PF05261.16          1     0.41    126   0.59   0.60   0.52   0.02\n",
            "20376  TrbC                 PF04956.18        105     7.75     98   0.59   0.63   0.51   0.09\n",
            "20377  TrbC_Ftype           PF09673.15         26     2.03    111   0.59   0.60   0.50   0.01\n",
            "20378  TrbE                 PF11100.13          1     0.57     66   0.85   0.85   0.78   0.11\n",
            "20379  TrbH                 PF07283.16          2     0.51    134   0.59   0.60   0.52   0.01\n",
            "20380  TrbI                 PF03743.19        131     7.10    188   0.59   0.62   0.49   0.02\n",
            "20381  TrbI_Ftype           PF09677.15          7     1.50    106   0.59   0.63   0.54   0.04\n",
            "20382  TrbK                 PF20084.4          92     2.37    102   0.59   0.56   0.50   0.05\n",
            "20383  TrbL                 PF04610.19        111     8.99    215   0.59   0.62   0.52   0.11\n",
            "20384  TrbL_2               PF19478.4           1     0.39    174   0.59   0.60   0.52   0.04\n",
            "20385  TrbL_3               PF19590.4           2     0.45    379   0.59   0.61   0.52   0.02\n",
            "20386  TrbL_4               PF19597.4          33     2.35    276   0.59   0.60   0.53   0.06\n",
            "20387  TrbL_5               PF19511.4          17     0.63    290   0.59   0.60   0.52   0.04\n",
            "20388  TrbM                 PF07424.16          4     0.79    156   0.59   0.54   0.51   0.03\n",
            "20389  TRC8_N               PF13705.11         31     1.77    494   0.59   0.58   0.52   0.06\n",
            "20390  TRCF                 PF03461.20        431     3.32     95   0.60   0.61   0.58   0.03\n",
            "20391  TrcR                 PF06242.16         36     0.96    139   0.59   0.60   0.53   0.03\n",
            "20392  Treacle              PF03546.19          9     0.96    531   0.59   0.64   0.52   0.10\n",
            "20393  Trefoil              PF00088.23        173     6.98     43   1.28   0.98   1.19   0.31\n",
            "20394  Trehalase            PF01204.23         18     1.56    512   0.59   0.55   0.51   0.02\n",
            "20395  Trehalase_Ca-bi      PF07492.16         43     5.61     30   1.80   1.86   1.78   0.15\n",
            "20396  Trehalose_PPase      PF02358.21         19     3.46    235   0.59   0.58   0.53   0.02\n",
            "20397  Trehalose_recp       PF06151.18          8     1.16    410   0.59   0.56   0.53   0.04\n",
            "20398  TREH_N               PF19291.4          33     0.80    211   0.59   0.59   0.53   0.01\n",
            "20399  Trep_dent_lipo       PF09710.15          4     0.75    397   0.59   0.56   0.50   0.04\n",
            "20400  Trep_Strep           PF09605.15         92     4.03    185   0.59   0.59   0.54   0.10\n",
            "20401  Treslin_M            PF15292.11         15     1.31    143   0.59   0.60   0.53   0.03\n",
            "20402  Treslin_N            PF21854.1          14     0.79    195   0.59   0.59   0.52   0.01\n",
            "20403  Treslin_STD          PF21855.1          30     1.34    154   0.59   0.60   0.54   0.05\n",
            "20404  TreT_GT1             PF21269.2         103     1.43    144   0.59   0.55   0.52   0.02\n",
            "20405  TRF                  PF08558.15         25     4.32    239   0.59   0.60   0.49   0.04\n",
            "20406  TrfA                 PF07042.16          2     0.42    282   0.59   0.57   0.52   0.01\n",
            "20407  TRH                  PF05438.17         13     0.93    221   0.59   0.57   0.50   0.05\n",
            "20408  TRI-like_N           PF22664.1          65     1.39    157   0.59   0.58   0.53   0.02\n",
            "20409  TRI12                PF06609.18          2     0.42    599   0.59   0.60   0.52   0.02\n",
            "20410  Tri3                 PF07428.16          2     0.59    416   0.59   0.59   0.52   0.01\n",
            "20411  TRI5                 PF06330.16          3     0.56    362   0.59   0.55   0.52   0.02\n",
            "20412  TRI9                 PF08195.16          1     0.86     43   1.28   1.24   1.23   0.13\n",
            "20413  Triabin              PF03973.18          4     1.00    149   0.59   0.55   0.50   0.03\n",
            "20414  TRIC                 PF05197.18         30     1.27    193   0.59   0.58   0.54   0.04\n",
            "20415  Tricho_coat          PF05892.16          3     0.83    197   0.59   0.60   0.51   0.02\n",
            "20416  Tricorn_C1           PF14684.11         74    10.88     59   0.95   0.84   0.94   0.08\n",
            "20417  Tricorn_PDZ          PF14685.11         21     1.97     88   0.65   0.68   0.59   0.02\n",
            "20418  TRIF-NTD             PF17798.6           5     1.09    157   0.59   0.61   0.52   0.03\n",
            "20419  Trigger_C            PF05698.19         40     5.90    162   0.59   0.63   0.55   0.08\n",
            "20420  Trigger_N            PF05697.18        694     5.57    144   0.59   0.61   0.51   0.05\n",
            "20421  Trimer_CC            PF08954.16          7     1.52     52   1.06   1.13   1.03   0.11\n",
            "20422  TRIQK                PF15168.11          5     0.79     79   0.72   0.75   0.66   0.05\n",
            "20423  TrkA_C               PF02080.26        107    35.71     71   0.79   0.86   0.73   0.03\n",
            "20424  TrkA_N               PF02254.23        186     7.48    116   0.59   0.64   0.52   0.04\n",
            "20425  TrkA_TMD             PF18613.6           2     2.00     22   1.86   2.02   1.83   0.57\n",
            "20426  TrkH                 PF02386.21          9     2.67    502   0.59   0.59   0.52   0.03\n",
            "20427  TRL                  PF13146.11         25     3.92     77   0.73   0.74   0.68   0.04\n",
            "20428  TRM                  PF02005.21          9     1.16    376   0.59   0.58   0.50   0.02\n",
            "20429  Trm112p              PF03966.21         69    20.27     40   1.37   1.33   1.18   0.09\n",
            "20430  TRM13                PF05206.19         20     1.48    257   0.59   0.57   0.50   0.03\n",
            "20431  Trm56                PF01994.21         70     0.93    119   0.59   0.58   0.52   0.02\n",
            "20432  Trm5_N               PF18093.6          31    11.33     47   1.17   1.22   1.16   0.08\n",
            "20433  TrmB                 PF01978.24         45     5.56     68   0.83   0.88   0.78   0.04\n",
            "20434  TrmE_N               PF10396.14        730     2.08    117   0.59   0.61   0.49   0.02\n",
            "20435  TrmI-like_N          PF14801.11        192     4.14     50   1.11   1.14   1.09   0.07\n",
            "20436  TrmK                 PF04816.17         11     1.46    205   0.59   0.60   0.53   0.02\n",
            "20437  TrmO                 PF01980.21        320     1.58    116   0.59   0.59   0.50   0.01\n",
            "20438  TrmO_C               PF18389.6          48     3.59     65   0.86   0.83   0.81   0.04\n",
            "20439  tRNA-synt_1          PF00133.27         11     1.17    414   0.59   0.54   0.51   0.02\n",
            "20440  tRNA-synt_1b         PF00579.30         37     4.38    292   0.59   0.59   0.50   0.02\n",
            "20441  tRNA-synt_1c         PF00749.26         16     1.70    314   0.59   0.56   0.53   0.02\n",
            "20442  tRNA-synt_1c_C       PF03950.23        105     4.69     91   0.63   0.62   0.49   0.03\n",
            "20443  tRNA-synt_1c_C2      PF20974.2         106     5.20     69   0.82   0.77   0.66   0.03\n",
            "20444  tRNA-synt_1d         PF00750.24          4     0.77    349   0.59   0.58   0.51   0.02\n",
            "20445  tRNA-synt_1e         PF01406.24         10     0.83    301   0.59   0.55   0.53   0.02\n",
            "20446  tRNA-synt_1f         PF01921.23         26     1.65    360   0.59   0.56   0.52   0.02\n",
            "20447  tRNA-synt_1g         PF09334.16         23     1.46    391   0.59   0.54   0.53   0.02\n",
            "20448  tRNA-synt_1_2        PF13603.11         60     1.37    185   0.59   0.57   0.53   0.02\n",
            "20449  tRNA-synt_2          PF00152.25         83     2.59    308   0.59   0.57   0.48   0.02\n",
            "20450  tRNA-synt_2b         PF00587.30          8     2.30    179   0.59   0.56   0.46   0.02\n",
            "20451  tRNA-synt_2c         PF01411.24         22     1.60    552   0.59   0.57   0.52   0.02\n",
            "20452  tRNA-synt_2d         PF01409.25         42     1.80    246   0.59   0.57   0.51   0.02\n",
            "20453  tRNA-synt_2e         PF02091.20        140     0.82    278   0.59   0.55   0.53   0.02\n",
            "20454  tRNA-synt_2_TM       PF16995.10         17     1.52    215   0.59   0.63   0.53   0.06\n",
            "20455  tRNA-synt_His        PF13393.11         58     3.10    308   0.59   0.62   0.53   0.02\n",
            "20456  tRNA-Thr_ED          PF08915.16         71     1.40    137   0.59   0.58   0.53   0.02\n",
            "20457  Trnau1ap             PF17654.6          34     2.27    110   0.59   0.51   0.42   0.07\n",
            "20458  tRNA_anti-codon      PF01336.30        523    32.82     75   0.75   0.78   0.65   0.04\n",
            "20459  tRNA_anti-like       PF12869.12          7     2.42    146   0.59   0.60   0.48   0.04\n",
            "20460  tRNA_anti_2          PF13742.11        555     3.34     95   0.60   0.60   0.47   0.02\n",
            "20461  tRNA_bind            PF01588.25         37     3.09     96   0.60   0.63   0.53   0.03\n",
            "20462  tRNA_bind_2          PF13725.11        105     1.70    233   0.59   0.59   0.53   0.04\n",
            "20463  tRNA_bind_3          PF17176.9          12     1.81    119   0.59   0.61   0.53   0.03\n",
            "20464  tRNA_bind_4          PF18490.6          14     1.10    162   0.59   0.61   0.52   0.04\n",
            "20465  tRNA_deacylase       PF04414.17         73     2.16    205   0.59   0.57   0.52   0.02\n",
            "20466  tRNA_edit            PF04073.20         56     8.28    123   0.59   0.63   0.53   0.02\n",
            "20467  tRNA_int_endo        PF01974.22         60     5.56     85   0.67   0.65   0.61   0.04\n",
            "20468  tRNA_int_endo_N      PF02778.19         40     6.55     67   0.84   0.89   0.77   0.04\n",
            "20469  tRNA_int_end_N2      PF12928.12        124     6.38     69   0.81   0.79   0.71   0.03\n",
            "20470  tRNA_int_end_N3      PF22355.1           1     0.66     55   1.01   0.99   0.95   0.06\n",
            "20471  tRNA_lig_CPD         PF08302.16         73     1.93    250   0.59   0.57   0.48   0.03\n",
            "20472  tRNA_lig_kinase      PF08303.16          9     0.89    168   0.59   0.57   0.52   0.03\n",
            "20473  tRNA_m1G_MT          PF01746.26         15     3.18    195   0.59   0.58   0.51   0.02\n",
            "20474  tRNA_Me_trans        PF03054.21         18     1.10    202   0.59   0.57   0.53   0.02\n",
            "20475  tRNA_Me_trans_C      PF20258.3          18     2.62     77   0.73   0.72   0.68   0.02\n",
            "20476  tRNA_Me_trans_M      PF20259.3          17     2.44     66   0.85   0.84   0.79   0.04\n",
            "20477  tRNA_NucTran2_2      PF13735.11         19     2.90    149   0.59   0.60   0.54   0.04\n",
            "20478  tRNA_NucTransf2      PF09249.16         76     1.08    113   0.59   0.59   0.52   0.02\n",
            "20479  tRNA_SAD             PF07973.19        188    18.12     44   1.25   1.22   1.16   0.07\n",
            "20480  tRNA_synthFbeta      PF17759.6         191     5.29    214   0.59   0.60   0.53   0.01\n",
            "20481  tRNA_synt_1c_R1      PF04558.20        114     3.25    162   0.59   0.61   0.54   0.03\n",
            "20482  tRNA_synt_1c_R2      PF04557.20         64     2.27     94   0.61   0.62   0.49   0.07\n",
            "20483  tRNA_synt_2f         PF02092.22        358     2.06    536   0.59   0.60   0.52   0.02\n",
            "20484  tRNA_U5-meth_tr      PF05958.16          7     0.67    357   0.59   0.57   0.53   0.02\n",
            "20485  Trns_repr_metal      PF02583.22        504     5.06     78   0.73   0.75   0.64   0.08\n",
            "20486  Tropomodulin         PF03250.19         20     0.94    142   0.59   0.61   0.53   0.04\n",
            "20487  Tropomyosin          PF00261.25         33     2.56    242   0.59   0.63   0.55   0.13\n",
            "20488  Troponin             PF00992.25         77     3.41    134   0.59   0.62   0.54   0.11\n",
            "20489  Troponin-I_N         PF11636.13          4     1.73     32   1.69   1.70   1.61   0.38\n",
            "20490  TROVE                PF05731.16         22     1.84    442   0.59   0.58   0.48   0.03\n",
            "20491  TRP                  PF06011.17        191     3.60    426   0.59   0.57   0.50   0.06\n",
            "20492  TrpBP                PF02081.20         21     0.85     69   0.81   0.83   0.76   0.05\n",
            "20493  TRPM_tetra           PF16519.10         13     2.54     56   0.99   1.04   0.97   0.12\n",
            "20494  TrpP                 PF17099.10          3     0.59    169   0.59   0.64   0.52   0.07\n",
            "20495  TRP_2                PF08344.16         28     1.94     63   0.89   0.83   0.84   0.08\n",
            "20496  Trp_dioxygenase      PF03301.18          5     0.95    346   0.59   0.58   0.52   0.02\n",
            "20497  Trp_DMAT             PF11991.13         93     4.77    357   0.59   0.56   0.46   0.01\n",
            "20498  Trp_halogenase       PF04820.19         23     1.79    458   0.59   0.57   0.53   0.01\n",
            "20499  Trp_leader1          PF08055.16          2     2.00     18   2.61   2.13   2.58   0.81\n",
            "20500  Trp_leader2          PF08056.16          1     0.84     41   1.34   1.14   1.29   0.21\n",
            "20501  TRP_N                PF14558.11        205     2.60    137   0.59   0.55   0.50   0.04\n",
            "20502  Trp_oprn_chp         PF09534.15         71     3.36    180   0.59   0.64   0.51   0.06\n",
            "20503  Trp_repressor        PF01371.24         15     1.89     88   0.65   0.67   0.61   0.04\n",
            "20504  Trp_ring             PF18669.6          12     2.31     51   1.09   1.11   1.02   0.11\n",
            "20505  Trp_syntA            PF00290.25         12     1.23    259   0.59   0.62   0.54   0.01\n",
            "20506  Trp_Tyr_perm         PF03222.18         13     1.54    393   0.59   0.61   0.54   0.05\n",
            "20507  Trs65                PF12735.12         53     3.30    311   0.59   0.60   0.47   0.02\n",
            "20508  TRSP                 PF12500.13         29     1.45    128   0.59   0.59   0.51   0.02\n",
            "20509  TruB-C_2             PF09157.16         50     7.00     57   0.98   1.03   0.91   0.03\n",
            "20510  TruB_C               PF09142.16         19     5.41     56   0.99   1.08   0.96   0.06\n",
            "20511  TruB_C_2             PF16198.10        164     4.61     66   0.85   0.88   0.75   0.04\n",
            "20512  TruB_N               PF01509.23        158     2.32    149   0.59   0.60   0.53   0.02\n",
            "20513  TruD                 PF01142.23         58     2.17    412   0.59   0.59   0.52   0.02\n",
            "20514  TrwB_AAD_bind        PF10412.14         11     1.60    387   0.59   0.60   0.54   0.01\n",
            "20515  TrwC                 PF08751.16         89     3.23    286   0.59   0.58   0.46   0.04\n",
            "20516  Trypan_glycop        PF00913.24         24     6.58    367   0.59   0.61   0.52   0.06\n",
            "20517  Trypan_glycop_C      PF10659.14         44     2.04    104   0.59   0.49   0.47   0.14\n",
            "20518  Trypan_PARP          PF05887.16          6     0.51    134   0.59   0.63   0.50   0.05\n",
            "20519  Trypco1              PF19493.4         223     6.37     98   0.59   0.64   0.46   0.03\n",
            "20520  Trypco2              PF19631.4          62     6.52     78   0.73   0.81   0.68   0.05\n",
            "20521  Trypsin              PF00089.31         69     6.03    220   0.59   0.56   0.46   0.02\n",
            "20522  Trypsin_2            PF13365.11        135    12.01    144   0.59   0.62   0.43   0.02\n",
            "20523  Tryp_alpha_amyl      PF00234.27        182     7.05     76   0.74   0.60   0.56   0.17\n",
            "20524  Tryp_FSAP            PF08248.16          2     2.00     12   2.66   2.40   2.63   1.12\n",
            "20525  Tryp_inh             PF17983.6           2     1.01     33   1.64   1.45   1.59   0.38\n",
            "20526  TryThrA_C            PF12319.13         41     2.45    216   0.59   0.44   0.56   0.17\n",
            "20527  TSA                  PF03249.18          1     0.39    510   0.59   0.60   0.51   0.01\n",
            "20528  TsaD                 PF00814.30        111     5.11    264   0.59   0.61   0.45   0.01\n",
            "20529  TsaE                 PF02367.22        538     2.08    128   0.59   0.58   0.44   0.02\n",
            "20530  TSC21                PF15217.11         17     0.71    170   0.59   0.55   0.53   0.02\n",
            "20531  TSC22                PF01166.23         17     2.27     57   0.98   1.01   0.95   0.10\n",
            "20532  Tsc35                PF15079.11          8     0.65    205   0.59   0.57   0.52   0.05\n",
            "20533  TSCPD                PF12637.12        126     1.33     74   0.76   0.72   0.70   0.06\n",
            "20534  Tsg                  PF04668.17         27     0.96    134   0.59   0.49   0.51   0.07\n",
            "20535  TSGA13               PF14994.11         10     0.70    274   0.59   0.57   0.53   0.03\n",
            "20536  TSGP1                PF07771.16          4     0.71    120   0.59   0.52   0.51   0.07\n",
            "20537  Tsi1                 PF21565.2           3     0.61    146   0.59   0.57   0.52   0.01\n",
            "20538  Tsi3                 PF22211.1          10     2.24    111   0.59   0.59   0.52   0.01\n",
            "20539  Tsi6                 PF18660.6          12     2.03     83   0.68   0.70   0.64   0.03\n",
            "20540  TsiV                 PF11876.13         12     2.10    271   0.59   0.57   0.51   0.01\n",
            "20541  TsiV3                PF20889.2           3     0.74     90   0.63   0.61   0.56   0.03\n",
            "20542  TSKS                 PF15358.11          3     0.71    395   0.59   0.62   0.51   0.02\n",
            "20543  TSLP                 PF15216.11          3     0.63    125   0.59   0.53   0.52   0.04\n",
            "20544  TSLPR_D1             PF22012.1          12     1.25     82   0.69   0.62   0.61   0.04\n",
            "20545  TSNAXIP1_N           PF15739.10         36     3.53    111   0.59   0.59   0.52   0.07\n",
            "20546  TSNR_N               PF04705.17          4     0.83    111   0.59   0.62   0.53   0.03\n",
            "20547  TSP1_ADAMTS          PF19030.5          21     2.41     55   1.01   0.78   0.95   0.23\n",
            "20548  TSP1_CCN             PF19035.5          32     2.19     44   1.25   1.00   1.21   0.31\n",
            "20549  TSP1_spondin         PF19028.5          68     2.52     52   1.07   0.82   1.01   0.26\n",
            "20550  Tsp36_N              PF20894.2           4     0.60     78   0.73   0.70   0.67   0.04\n",
            "20551  TSP3_bac             PF18884.5          76    50.67     22   2.41   2.48   2.40   0.52\n",
            "20552  Tsp45I               PF06300.17          3     0.54    260   0.59   0.58   0.52   0.02\n",
            "20553  TSP9                 PF11493.13         14     1.17     81   0.70   0.75   0.60   0.04\n",
            "20554  TspO_MBR             PF03073.20        363     4.97    145   0.59   0.56   0.51   0.09\n",
            "20555  TSP_1                PF00090.24         23     4.88     49   1.13   0.91   1.06   0.25\n",
            "20556  TSP_3                PF02412.23         11     3.63     36   1.51   1.41   1.48   0.39\n",
            "20557  TSP_C                PF05735.18         18     0.81    198   0.59   0.54   0.53   0.02\n",
            "20558  TSP_Ig-like          PF20745.2           9     3.56     86   0.66   0.66   0.56   0.05\n",
            "20559  TSP_NTD              PF17804.6          75     3.72    188   0.59   0.56   0.53   0.06\n",
            "20560  TSR                  PF18487.6          24     1.71     50   1.11   0.92   1.06   0.20\n",
            "20561  Tsr0524-like         PF11061.13         34     1.76     61   0.92   0.92   0.87   0.05\n",
            "20562  Tsr1_G-like          PF22298.1          78     1.42    161   0.59   0.60   0.53   0.03\n",
            "20563  TssC                 PF17541.7          13     0.60    430   0.59   0.59   0.53   0.02\n",
            "20564  TSSC4                PF15264.11         12     1.16    124   0.59   0.57   0.50   0.04\n",
            "20565  TssD                 PF17642.6          66     3.29    128   0.59   0.56   0.54   0.03\n",
            "20566  TssN                 PF17555.7          16     1.80    274   0.59   0.56   0.54   0.06\n",
            "20567  TssO                 PF17561.7          18     3.34    148   0.59   0.57   0.55   0.07\n",
            "20568  TssR                 PF17643.6          13     1.44    102   0.59   0.55   0.54   0.04\n",
            "20569  TssR_C               PF20781.2          13     2.66    141   0.59   0.57   0.53   0.06\n",
            "20570  TssR_M               PF20780.2          13     1.84     86   0.66   0.65   0.62   0.04\n",
            "20571  TssR_VWA             PF20782.2          13     1.83    304   0.59   0.59   0.55   0.04\n",
            "20572  TSX                  PF17397.7           2     0.46    145   0.59   0.58   0.52   0.05\n",
            "20573  TS_C                 PF22925.2         137     2.17    202   0.59   0.58   0.52   0.02\n",
            "20574  Tt1218-like          PF22150.1          15     2.35     76   0.74   0.72   0.65   0.07\n",
            "20575  Tt1219-like          PF22149.1           3     0.52    191   0.59   0.61   0.51   0.01\n",
            "20576  TTC3_DZIP3_dom       PF19179.5          26     0.79    116   0.59   0.50   0.53   0.05\n",
            "20577  TTC5_OB              PF16669.10         29     2.41    115   0.59   0.59   0.54   0.03\n",
            "20578  TTC7_N               PF19440.4           6     0.91    386   0.59   0.61   0.53   0.02\n",
            "20579  TTc_toxin_rep        PF18807.6          12     2.61     45   1.22   1.24   1.20   0.06\n",
            "20580  TTD                  PF12148.13         29     1.07    155   0.59   0.56   0.52   0.03\n",
            "20581  TTHA0829-like_ACT    PF22190.1           8     1.00     70   0.80   0.87   0.74   0.05\n",
            "20582  TTHB210-like         PF18197.6          33    18.78     50   1.11   1.09   0.98   0.05\n",
            "20583  TTI1                 PF21547.2         386     5.45     90   0.63   0.66   0.61   0.05\n",
            "20584  Tti2                 PF10521.14         60     4.95    279   0.59   0.60   0.49   0.02\n",
            "20585  TTL                  PF03133.20         20     3.24    292   0.59   0.55   0.51   0.03\n",
            "20586  TTR-52               PF01060.28         95     2.91     79   0.72   0.66   0.64   0.03\n",
            "20587  TTRAP                PF14203.11         17     2.68     62   0.90   0.92   0.85   0.08\n",
            "20588  TTSSLRR              PF12468.13         13     5.16     54   1.03   0.93   0.94   0.09\n",
            "20589  TtuA_LIM_N           PF22082.1         323    82.22     26   2.06   1.79   2.05   0.37\n",
            "20590  TT_ORF1              PF02956.19         58     2.50    525   0.59   0.51   0.50   0.04\n",
            "20591  TT_ORF2-like         PF02957.20         17     2.52    103   0.59   0.53   0.50   0.06\n",
            "20592  TT_ORF2a             PF08197.16          1     0.75     49   1.13   1.14   1.07   0.08\n",
            "20593  Tub                  PF01167.23        112     2.05    256   0.59   0.55   0.47   0.02\n",
            "20594  TubC_N               PF18563.6          34     5.57     52   1.07   1.14   1.05   0.07\n",
            "20595  Tube                 PF17212.8           6     1.26    169   0.59   0.59   0.53   0.02\n",
            "20596  Tuberculin           PF12198.13          1     0.46     83   0.69   0.71   0.62   0.03\n",
            "20597  Tuberin              PF03542.21         13     1.22    347   0.59   0.58   0.52   0.04\n",
            "20598  Tubulin              PF00091.30         87     4.16    190   0.59   0.60   0.49   0.02\n",
            "20599  Tubulin-binding      PF00418.24         56     9.56     31   1.74   1.75   1.71   0.15\n",
            "20600  Tubulin_2            PF13809.11         31     4.33    356   0.59   0.59   0.46   0.01\n",
            "20601  Tubulin_3            PF14881.11         16     0.75    180   0.59   0.56   0.49   0.02\n",
            "20602  Tubulin_C            PF03953.22         34     1.78    126   0.59   0.55   0.53   0.03\n",
            "20603  TubZ-like_C          PF22453.1           6     1.11    144   0.59   0.60   0.52   0.02\n",
            "20604  TubZ_C               PF21493.2           1     0.41    133   0.59   0.61   0.52   0.01\n",
            "20605  TubZ_C_2             PF22334.1           7     0.98     88   0.65   0.64   0.59   0.03\n",
            "20606  TubZ_C_3             PF22340.1           2     0.71     88   0.65   0.68   0.58   0.04\n",
            "20607  Tub_N                PF16322.10         17     1.62    193   0.59   0.61   0.48   0.07\n",
            "20608  TUDOR                PF00567.29         44     7.16    121   0.59   0.55   0.53   0.02\n",
            "20609  Tudor-knot           PF11717.13         37     4.07     55   1.01   0.92   0.92   0.08\n",
            "20610  Tudor_1_RapA         PF18339.6          68     2.21     51   1.08   1.09   1.06   0.04\n",
            "20611  Tudor_2              PF18104.6          43    11.38     35   1.55   1.47   1.54   0.11\n",
            "20612  Tudor_3              PF18115.6          53     5.29     52   1.07   1.01   1.02   0.04\n",
            "20613  Tudor_4              PF18358.6           9     1.39     50   1.11   1.05   1.07   0.05\n",
            "20614  Tudor_5              PF18359.6          26     3.48     53   1.05   1.01   0.97   0.06\n",
            "20615  Tudor_FRX1           PF18336.6           6     1.21     49   1.13   1.11   1.09   0.04\n",
            "20616  Tudor_RapA           PF18337.6         113     4.93     62   0.90   0.91   0.86   0.03\n",
            "20617  TUG-UBL1             PF11470.13         90     7.92     65   0.86   0.89   0.83   0.04\n",
            "20618  Tugs                 PF17840.6          11     2.16     47   1.17   1.19   1.15   0.10\n",
            "20619  TUNAR                PF21954.1           1     0.84     48   1.15   1.20   1.09   0.09\n",
            "20620  Tup_N                PF08581.15         51     2.48     77   0.73   0.74   0.70   0.11\n",
            "20621  Turandot             PF07240.16         10     2.29     80   0.71   0.72   0.64   0.02\n",
            "20622  TusA                 PF01206.22        118    11.78     70   0.80   0.80   0.77   0.03\n",
            "20623  TUSC2                PF15000.11         12     1.19    111   0.59   0.56   0.52   0.02\n",
            "20624  TUTase               PF19088.5          12     0.59    138   0.59   0.59   0.53   0.02\n",
            "20625  TUTF7_u4             PF16631.10          8     0.74     88   0.65   0.63   0.59   0.14\n",
            "20626  Tweety               PF04906.18         11     1.02    431   0.59   0.58   0.53   0.03\n",
            "20627  TX13_rpt             PF20868.2          14     1.44    102   0.59   0.58   0.55   0.08\n",
            "20628  TXD17-like_Trx       PF06110.16          5     0.86    119   0.59   0.56   0.52   0.02\n",
            "20629  TxDE                 PF18711.6          15     3.00     55   1.01   1.06   0.98   0.04\n",
            "20630  TY-Chap1             PF22551.1          53     3.47    132   0.59   0.62   0.55   0.03\n",
            "20631  TY-Chap2             PF22553.1          15     3.15    223   0.59   0.56   0.51   0.02\n",
            "20632  TY-Chap3             PF22552.1         231     6.70    117   0.59   0.58   0.46   0.04\n",
            "20633  Ty3_capsid           PF19259.4           5     1.48    198   0.59   0.55   0.54   0.03\n",
            "20634  TYA                  PF01021.24          6     0.67    384   0.59   0.57   0.53   0.02\n",
            "20635  TyeA                 PF09059.15         11     1.88     81   0.70   0.72   0.66   0.04\n",
            "20636  TylF                 PF05711.16         10     0.90    264   0.59   0.58   0.52   0.01\n",
            "20637  Tymo_45kd_70kd       PF03251.18          1     0.39    468   0.59   0.58   0.52   0.04\n",
            "20638  Tymo_coat            PF00983.23          3     0.76    179   0.59   0.59   0.51   0.03\n",
            "20639  TypeIII_RM_meth      PF12564.13          8     1.89     56   0.99   0.99   0.95   0.08\n",
            "20640  Type_III_SycN        PF21665.2           5     0.74    119   0.59   0.59   0.53   0.03\n",
            "20641  Type_III_YscG        PF09477.15          4     0.76    116   0.59   0.61   0.52   0.02\n",
            "20642  Type_III_YscX        PF09474.15         27     4.16    119   0.59   0.63   0.54   0.04\n",
            "20643  type_II_gspD_N0      PF21305.2         216     3.81     71   0.79   0.83   0.76   0.05\n",
            "20644  Type_ISP_C           PF18135.6          74     4.29    342   0.59   0.55   0.49   0.02\n",
            "20645  Tyr-DNA_phospho      PF06087.17        104     2.98    445   0.59   0.55   0.46   0.02\n",
            "20646  Tyrosinase           PF00264.25        120     3.97    214   0.59   0.48   0.43   0.05\n",
            "20647  Tyrosinase_C         PF18132.6         226     4.27    121   0.59   0.58   0.48   0.02\n",
            "20648  TyrRSs_C             PF16714.10         43     1.38    120   0.59   0.60   0.54   0.02\n",
            "20649  Tyr_Deacylase        PF02580.21        712     2.33    144   0.59   0.60   0.52   0.02\n",
            "20650  tyr_de_CO2_C         PF21391.2          15     0.85    141   0.59   0.56   0.54   0.02\n",
            "20651  TYW3                 PF02676.19        130     2.65    211   0.59   0.59   0.47   0.02\n",
            "20652  T_Ag_DNA_bind        PF02217.21         11     1.18     94   0.61   0.57   0.55   0.05\n",
            "20653  T_cell_tran_alt      PF15128.11          7     0.58     92   0.62   0.58   0.55   0.02\n",
            "20654  T_hemolysin          PF12261.13         30     1.76    174   0.59   0.58   0.55   0.02\n",
            "20655  U-box                PF04564.20         16     2.17     73   0.77   0.74   0.73   0.04\n",
            "20656  U1snRNP70_N          PF12220.13        122     4.23     92   0.62   0.60   0.54   0.07\n",
            "20657  U3snoRNP10           PF12397.13        136     7.78    116   0.59   0.64   0.49   0.06\n",
            "20658  U3_assoc_6           PF08640.16         89     3.65     85   0.67   0.67   0.61   0.08\n",
            "20659  U3_snoRNA_assoc      PF08297.16         63     4.66     90   0.63   0.66   0.52   0.08\n",
            "20660  U5_2-snRNA_bdg       PF10597.14         17     0.64    134   0.59   0.59   0.53   0.02\n",
            "20661  U6-snRNA_bdg         PF10596.14          9     0.77    159   0.59   0.55   0.53   0.02\n",
            "20662  U62_UL91             PF17442.7           3     0.96     65   0.86   0.88   0.82   0.06\n",
            "20663  U71                  PF17474.7           2     0.73     52   1.07   1.03   1.02   0.08\n",
            "20664  U79_P34              PF03064.21         14     1.29    134   0.59   0.57   0.53   0.03\n",
            "20665  U83                  PF17465.7           2     0.43     97   0.59   0.60   0.52   0.03\n",
            "20666  U9-ORF               PF22040.1           1     0.43     81   0.70   0.67   0.63   0.04\n",
            "20667  UAA                  PF08449.16         16     2.17    311   0.59   0.58   0.52   0.05\n",
            "20668  UAE_UbL              PF14732.11         75     4.70     89   0.64   0.66   0.55   0.05\n",
            "20669  UAF_Rrn10            PF05234.16          2     0.64    122   0.59   0.59   0.50   0.02\n",
            "20670  Ub-Mut7C             PF14451.11          9     1.40     81   0.70   0.71   0.66   0.02\n",
            "20671  Ub-RnfH              PF03658.19        130     2.10     83   0.68   0.70   0.64   0.03\n",
            "20672  UB2H                 PF14814.11         71     4.29     85   0.67   0.69   0.64   0.03\n",
            "20673  UBA                  PF00627.36         61    61.00     37   1.37   1.39   1.35   0.08\n",
            "20674  UBA2_C               PF16195.10          8     1.39     95   0.60   0.62   0.52   0.12\n",
            "20675  UBact                PF20513.3          15     1.86     43   1.28   1.26   1.23   0.18\n",
            "20676  UBAP-1_UBA2          PF21267.2           3     1.30     39   1.40   1.41   1.36   0.12\n",
            "20677  UBAP2-Lig            PF12478.13         13     3.68     33   1.64   1.69   1.55   0.12\n",
            "20678  UBA_2                PF08587.16         41     4.41     45   1.22   1.24   1.16   0.11\n",
            "20679  UBA_3                PF09288.15          6     1.78     55   1.01   1.07   0.97   0.07\n",
            "20680  UBA_4                PF14555.11        144    43.35     43   1.28   1.27   1.25   0.09\n",
            "20681  UBA_5                PF16577.10          2     0.66     62   0.90   0.90   0.84   0.01\n",
            "20682  UBA_6                PF18039.6           6     2.99     42   1.30   1.38   1.27   0.06\n",
            "20683  UBA_7                PF22562.1          46     8.57     48   1.15   1.14   1.13   0.06\n",
            "20684  UBA_8                PF22566.1          14     7.71     46   1.20   1.18   1.17   0.09\n",
            "20685  UBA_9                PF22567.1           5     1.21     43   1.28   1.24   1.24   0.05\n",
            "20686  UBA_BRSK             PF21115.2          12     1.57     43   1.28   1.26   1.24   0.09\n",
            "20687  UBA_E1_SCCH          PF10585.14        199     2.44    254   0.59   0.55   0.50   0.04\n",
            "20688  UBC_like             PF23009.1          53     1.92     85   0.67   0.64   0.63   0.04\n",
            "20689  UBD                  PF16455.10         47     1.09    103   0.59   0.56   0.50   0.02\n",
            "20690  UbiA                 PF01040.23        770    12.08    250   0.59   0.60   0.50   0.08\n",
            "20691  UbiD                 PF01977.21        298     3.73    206   0.59   0.58   0.51   0.01\n",
            "20692  UbiD_C               PF20696.2         252     6.89    127   0.59   0.58   0.51   0.02\n",
            "20693  UbiD_N               PF20695.2         340    12.15     78   0.73   0.78   0.63   0.02\n",
            "20694  Ubie_methyltran      PF01209.23          5     0.79    233   0.59   0.58   0.52   0.01\n",
            "20695  Ubiq-assoc           PF09145.15          9     1.88     44   1.25   1.27   1.21   0.09\n",
            "20696  Ubiq-Cytc-red_N      PF09165.15         19     3.29     75   0.75   0.82   0.71   0.07\n",
            "20697  ubiquitin            PF00240.28         59     9.99     72   0.78   0.81   0.74   0.05\n",
            "20698  Ubiquitin_2          PF14560.11         62     3.86     86   0.66   0.67   0.61   0.03\n",
            "20699  Ubiquitin_3          PF14836.11          4     0.81     88   0.65   0.62   0.57   0.03\n",
            "20700  Ubiquitin_4          PF18036.6          13     1.37     89   0.64   0.60   0.59   0.05\n",
            "20701  Ubiquitin_5          PF18037.6          13     1.15     98   0.59   0.62   0.52   0.05\n",
            "20702  Ubiq_cyt_C_chap      PF03981.17         60     6.46    140   0.59   0.59   0.52   0.03\n",
            "20703  Ubiq_DUF_assoc       PF22932.1          33     2.04     81   0.70   0.70   0.64   0.01\n",
            "20704  UBM                  PF14377.11        169   169.00     35   1.29   1.35   1.27   0.17\n",
            "20705  UBN_AB               PF14075.11         21     2.58    211   0.59   0.57   0.53   0.04\n",
            "20706  UBR1-like_wing       PF22960.1         111     3.47    100   0.59   0.57   0.55   0.04\n",
            "20707  UBX                  PF00789.25         19     4.18     82   0.69   0.70   0.63   0.03\n",
            "20708  UBZ_FAAP20           PF15750.10          6     1.98     35   1.55   1.44   1.52   0.19\n",
            "20709  UCH                  PF00443.34         54     3.83    254   0.59   0.55   0.40   0.03\n",
            "20710  UCH_1                PF13423.11        193     3.75    317   0.59   0.55   0.45   0.03\n",
            "20711  UCH_C                PF18031.6          30     6.65     46   1.20   1.18   1.18   0.12\n",
            "20712  UCH_N                PF16674.10         29     1.60    105   0.59   0.60   0.54   0.04\n",
            "20713  UCH_UBL1             PF22900.1          72     3.08     87   0.65   0.64   0.61   0.05\n",
            "20714  UCMA                 PF17085.10          3     0.55    134   0.59   0.56   0.52   0.05\n",
            "20715  UcrQ                 PF02939.21         97     3.42     78   0.73   0.67   0.69   0.03\n",
            "20716  UCR_14kD             PF02271.21        102     1.98    100   0.59   0.58   0.55   0.04\n",
            "20717  UCR_6-4kD            PF08997.15         14     1.81     55   1.01   0.92   0.97   0.08\n",
            "20718  UCR_Fe-S_N           PF10399.14         27     4.50     41   1.34   1.36   1.32   0.08\n",
            "20719  UCR_hinge            PF02320.21        106     3.52     65   0.86   0.74   0.81   0.14\n",
            "20720  UCR_TM               PF02921.19         36     4.42     66   0.85   0.86   0.81   0.05\n",
            "20721  UCR_UQCRX_QCR9       PF05365.17         95     4.76     53   1.05   0.97   1.03   0.09\n",
            "20722  uDENN                PF03456.23        129     8.22     57   0.98   0.89   0.78   0.06\n",
            "20723  UDG                  PF03167.24        155     9.28    142   0.59   0.58   0.34   0.01\n",
            "20724  UDG-inhib_P56        PF20763.2           6     2.77     48   1.15   1.13   1.12   0.09\n",
            "20725  UDI                  PF18880.5           2     0.73     82   0.69   0.70   0.63   0.03\n",
            "20726  UDP-g_GGTase         PF06427.16        148     1.57    108   0.59   0.59   0.52   0.02\n",
            "20727  UDPGP                PF01704.23         73     1.57    413   0.59   0.59   0.52   0.02\n",
            "20728  UDPGT                PF00201.23         13     1.00    499   0.59   0.57   0.53   0.01\n",
            "20729  UDPG_MGDP_dh         PF00984.24        502     3.05     94   0.61   0.61   0.56   0.02\n",
            "20730  UDPG_MGDP_dh_C       PF03720.20        151     4.69    103   0.59   0.60   0.52   0.02\n",
            "20731  UDPG_MGDP_dh_N       PF03721.19         18     1.65    187   0.59   0.61   0.53   0.02\n",
            "20732  Uds1                 PF15456.11         81     3.47    123   0.59   0.62   0.52   0.08\n",
            "20733  UEV                  PF05743.18         20     1.53    121   0.59   0.55   0.53   0.02\n",
            "20734  UFC1                 PF08694.16         24     0.58    155   0.59   0.54   0.53   0.02\n",
            "20735  UFD1                 PF03152.19         96     1.64    173   0.59   0.59   0.51   0.02\n",
            "20736  Ufd2P_core           PF10408.14        148     4.23    628   0.59   0.57   0.49   0.03\n",
            "20737  Ufm1                 PF03671.19         12     1.22     75   0.75   0.77   0.70   0.03\n",
            "20738  UfSP2_N              PF20908.2          29     1.26    220   0.59   0.57   0.54   0.03\n",
            "20739  UfSP_MPN_N           PF22084.1           7     0.82     84   0.68   0.71   0.61   0.03\n",
            "20740  UIM                  PF02809.25        107   107.00     17   1.97   2.09   1.98   0.50\n",
            "20741  UK                   PF02512.19          1     0.39     96   0.60   0.56   0.53   0.03\n",
            "20742  UL11                 PF11094.13          3     1.81     39   1.40   1.34   1.29   0.10\n",
            "20743  UL141                PF16758.10          2     0.58    191   0.59   0.54   0.52   0.01\n",
            "20744  UL16                 PF17622.7           1     0.38    204   0.59   0.58   0.51   0.01\n",
            "20745  UL17                 PF17640.7           1     0.36    102   0.59   0.55   0.52   0.03\n",
            "20746  UL2                  PF08196.16          1     0.63     59   0.95   0.96   0.89   0.06\n",
            "20747  UL20                 PF17582.7           1     0.37    304   0.59   0.56   0.51   0.02\n",
            "20748  UL21a                PF17636.7           1     0.37    123   0.59   0.58   0.52   0.02\n",
            "20749  UL40                 PF10682.14          1     0.39    214   0.59   0.57   0.52   0.02\n",
            "20750  UL41A                PF17591.7           1     0.46     78   0.72   0.68   0.66   0.04\n",
            "20751  UL42                 PF17638.7           1     0.37    125   0.59   0.55   0.51   0.02\n",
            "20752  UL45                 PF05473.17         11     2.16    191   0.59   0.56   0.50   0.03\n",
            "20753  UL73_N               PF12522.13          5     4.35     27   1.98   2.17   1.96   0.59\n",
            "20754  UL97                 PF06734.17          2     0.61    187   0.59   0.57   0.52   0.02\n",
            "20755  ULD                  PF16534.10         14     1.05    109   0.59   0.60   0.53   0.03\n",
            "20756  ULD_2                PF22893.1         252     6.87     85   0.67   0.61   0.63   0.02\n",
            "20757  ULD_3                PF22947.1         112     3.02    115   0.59   0.59   0.54   0.02\n",
            "20758  Uma2                 PF05685.17        138    10.17    172   0.59   0.58   0.46   0.02\n",
            "20759  Umbravirus_LDM       PF04817.17          4     0.96    231   0.59   0.59   0.54   0.05\n",
            "20760  UME                  PF08064.18        109     4.62    102   0.59   0.59   0.55   0.04\n",
            "20761  UMP1                 PF05348.16         62     2.67    118   0.59   0.57   0.52   0.03\n",
            "20762  UMPH-1               PF05822.17          9     0.83    246   0.59   0.58   0.53   0.02\n",
            "20763  UnbV_ASPIC           PF07593.17        217     8.74     68   0.83   0.83   0.72   0.04\n",
            "20764  UNC-50               PF05216.18         86     1.72    221   0.59   0.55   0.53   0.07\n",
            "20765  UNC-79               PF14776.11          7     0.75    526   0.59   0.55   0.53   0.02\n",
            "20766  UNC-93               PF05978.21          9     1.35    379   0.59   0.59   0.54   0.05\n",
            "20767  UNC119_bdg           PF15435.11          7     0.56    198   0.59   0.59   0.52   0.02\n",
            "20768  UNC45-central        PF11701.13         92     5.98    154   0.59   0.62   0.52   0.04\n",
            "20769  UNC80                PF19424.4          14     0.82    706   0.59   0.57   0.51   0.02\n",
            "20770  UNC80_C              PF20262.3          14     0.78   1079   0.59   0.59   0.53   0.02\n",
            "20771  UNC80_N              PF15778.10         12     0.73    187   0.59   0.56   0.52   0.02\n",
            "20772  Unpaired             PF15972.10         10     1.01    273   0.59   0.55   0.50   0.05\n",
            "20773  Unstab_antitox       PF09720.15        120    20.97     54   1.03   1.09   0.99   0.11\n",
            "20774  UN_NPL4              PF11543.13          7     1.77     80   0.71   0.74   0.65   0.04\n",
            "20775  UPA                  PF17217.8          16     0.76    140   0.59   0.57   0.53   0.02\n",
            "20776  UPAR_LY6             PF00021.26         27     3.21     77   0.73   0.56   0.63   0.21\n",
            "20777  UPAR_LY6_2           PF16975.10         15     0.52    106   0.59   0.48   0.53   0.07\n",
            "20778  UPA_2                PF17809.6           4     0.76    131   0.59   0.58   0.52   0.02\n",
            "20779  UPF0004              PF00919.25         93     2.03     98   0.59   0.56   0.53   0.04\n",
            "20780  UPF0014              PF03649.18        129     2.51    242   0.59   0.62   0.56   0.07\n",
            "20781  UPF0016              PF01169.24        548     5.09     75   0.75   0.81   0.70   0.11\n",
            "20782  UPF0020              PF01170.23         13     2.41    197   0.59   0.59   0.51   0.01\n",
            "20783  UPF0029              PF01205.24        613     2.35    107   0.59   0.58   0.50   0.02\n",
            "20784  UPF0047              PF01894.22        548     2.38    117   0.59   0.58   0.49   0.03\n",
            "20785  UPF0058              PF01893.21         29     1.50     87   0.65   0.64   0.60   0.06\n",
            "20786  UPF0060              PF02694.20          4     0.62    107   0.59   0.59   0.52   0.07\n",
            "20787  UPF0093              PF03653.18         27     1.49    146   0.59   0.56   0.53   0.07\n",
            "20788  UPF0102              PF02021.22        669     3.36     80   0.71   0.72   0.51   0.05\n",
            "20789  UPF0113              PF03657.18         20     3.72     73   0.77   0.78   0.70   0.03\n",
            "20790  UPF0113_N            PF17833.6          26     3.43     82   0.69   0.71   0.59   0.04\n",
            "20791  UPF0114              PF03350.21        128     3.13    117   0.59   0.62   0.52   0.08\n",
            "20792  UPF0122              PF04297.19          9     1.08    101   0.59   0.59   0.54   0.07\n",
            "20793  UPF0128              PF03673.18          5     0.70    221   0.59   0.59   0.52   0.01\n",
            "20794  UPF0137              PF03677.18          5     0.85    237   0.59   0.59   0.52   0.03\n",
            "20795  UPF0146              PF03686.18          4     0.96    129   0.59   0.61   0.52   0.02\n",
            "20796  UPF0147              PF03685.18         49     1.62     81   0.70   0.72   0.66   0.05\n",
            "20797  UPF0149              PF03695.18        115     7.10    181   0.59   0.59   0.45   0.03\n",
            "20798  UPF0150              PF21748.2          84     2.97     64   0.88   0.86   0.84   0.03\n",
            "20799  UPF0154              PF03672.18         50     3.35     60   0.93   0.90   0.91   0.08\n",
            "20800  UPF0158              PF03682.18          5     0.72    157   0.59   0.57   0.52   0.04\n",
            "20801  UPF0160              PF03690.18        169     1.87    318   0.59   0.57   0.52   0.02\n",
            "20802  UPF0164              PF03687.18          5     0.99    325   0.59   0.60   0.51   0.02\n",
            "20803  UPF0167              PF03691.19         13     1.25    174   0.59   0.49   0.51   0.04\n",
            "20804  UPF0172              PF03665.18         76     2.45    192   0.59   0.56   0.50   0.03\n",
            "20805  UPF0175              PF03683.18         84     6.84     75   0.75   0.80   0.72   0.04\n",
            "20806  UPF0176_N            PF17773.6          71     2.78     92   0.62   0.62   0.57   0.02\n",
            "20807  UPF0179              PF03684.18         49     1.57    140   0.59   0.53   0.53   0.06\n",
            "20808  UPF0180              PF03698.18         43     3.10     77   0.73   0.76   0.68   0.05\n",
            "20809  UPF0181              PF03701.19         21     1.97     50   1.11   1.12   1.06   0.12\n",
            "20810  UPF0182              PF03699.18        180     2.02    770   0.59   0.56   0.52   0.02\n",
            "20811  UPF0184              PF03670.18          3     0.95     83   0.68   0.70   0.63   0.07\n",
            "20812  UPF0193              PF05250.16         22     1.61    217   0.59   0.59   0.52   0.05\n",
            "20813  UPF0203              PF05254.17         56     2.08     69   0.81   0.67   0.75   0.12\n",
            "20814  UPF0220              PF05255.16         65     1.81    166   0.59   0.58   0.51   0.04\n",
            "20815  UPF0223              PF05256.17         41     1.59     86   0.66   0.62   0.62   0.06\n",
            "20816  UPF0227              PF05728.17          8     1.39    187   0.59   0.57   0.53   0.02\n",
            "20817  UPF0228              PF05727.16          9     0.94    124   0.59   0.56   0.50   0.05\n",
            "20818  UPF0231              PF06062.16         30     1.67    121   0.59   0.56   0.53   0.04\n",
            "20819  UPF0236              PF06782.16          3     0.83    400   0.59   0.58   0.52   0.03\n",
            "20820  UPF0239              PF06783.16         14     1.10     88   0.65   0.66   0.58   0.03\n",
            "20821  UPF0240              PF06784.16         26     1.72    173   0.59   0.58   0.52   0.04\n",
            "20822  UPF0242              PF06785.16          6     1.49    194   0.59   0.60   0.51   0.06\n",
            "20823  UPF0253              PF06786.17          8     1.03     65   0.86   0.86   0.80   0.04\n",
            "20824  UPF0257              PF06788.18          4     0.68    236   0.59   0.58   0.53   0.02\n",
            "20825  UPF0259              PF06790.16          5     0.82    248   0.59   0.62   0.53   0.09\n",
            "20826  UPF0261              PF06792.16        104     1.95    402   0.59   0.61   0.53   0.02\n",
            "20827  UPF0262              PF06793.17         41     0.92    153   0.59   0.60   0.53   0.03\n",
            "20828  UPF0270              PF06794.17         53     2.90     69   0.82   0.84   0.77   0.04\n",
            "20829  UPF0300              PF08594.15         12     1.41    212   0.59   0.54   0.54   0.03\n",
            "20830  UPF0302              PF08864.15         25     2.00    105   0.59   0.57   0.55   0.05\n",
            "20831  UPF0370              PF13980.11          8     0.69     61   0.91   0.82   0.85   0.09\n",
            "20832  UPF0425_C            PF22583.1          23     1.85     70   0.80   0.84   0.71   0.03\n",
            "20833  UPF0444              PF15475.11          6     0.77     91   0.63   0.69   0.55   0.07\n",
            "20834  UPF0449              PF15136.11         25     2.03     98   0.59   0.60   0.53   0.05\n",
            "20835  UPF0489              PF12640.12         12     2.02    167   0.59   0.54   0.49   0.03\n",
            "20836  UPF0492              PF15744.10          5     0.83    359   0.59   0.59   0.52   0.03\n",
            "20837  UPF0506              PF11703.13          2     0.83     59   0.94   0.73   0.88   0.24\n",
            "20838  UPF0515              PF15135.11          9     0.88    271   0.59   0.53   0.51   0.03\n",
            "20839  UPF0524              PF15823.10          9     0.51    239   0.59   0.54   0.52   0.03\n",
            "20840  UPF0542              PF15086.11          5     1.26     68   0.83   0.82   0.76   0.05\n",
            "20841  UPF0547              PF10571.14         12     7.81     26   2.06   1.73   2.03   0.44\n",
            "20842  UPF0552              PF10574.14         10     0.82    224   0.59   0.58   0.53   0.02\n",
            "20843  UPF0556              PF10572.14         17     1.13    123   0.59   0.55   0.50   0.02\n",
            "20844  UPF0561              PF10573.14          3     0.51    120   0.59   0.57   0.52   0.03\n",
            "20845  UPF0640              PF15114.11         13     1.07     67   0.84   0.79   0.78   0.05\n",
            "20846  UPF0669              PF17065.10          5     0.52    185   0.59   0.58   0.52   0.01\n",
            "20847  UPF0688              PF15772.10         13     1.01    232   0.59   0.59   0.53   0.06\n",
            "20848  UPF0697              PF15117.11          5     0.47     98   0.59   0.57   0.52   0.03\n",
            "20849  UPF0715              PF17094.10          3     0.59    119   0.59   0.58   0.52   0.11\n",
            "20850  UPF0728              PF15092.11         19     1.26     88   0.65   0.63   0.59   0.02\n",
            "20851  UPF0730              PF15827.10          2     1.11     47   1.17   1.10   1.12   0.11\n",
            "20852  UPF0731              PF14982.11          2     0.52     78   0.72   0.71   0.65   0.03\n",
            "20853  UPF0738              PF19785.4         106     1.76    117   0.59   0.58   0.53   0.04\n",
            "20854  UPF0758_N            PF20582.3         712     2.93     77   0.73   0.80   0.66   0.03\n",
            "20855  UPF0767              PF15990.10         18     1.17     83   0.68   0.68   0.61   0.02\n",
            "20856  UPF1_1B_dom          PF18141.6          15     1.44     93   0.61   0.58   0.55   0.02\n",
            "20857  UPF1_Zn_bind         PF09416.15         43     1.07    152   0.59   0.51   0.53   0.05\n",
            "20858  Upf2                 PF04050.19        103     4.03    133   0.59   0.61   0.45   0.08\n",
            "20859  UPRTase              PF14681.11         80     2.75    206   0.59   0.61   0.54   0.02\n",
            "20860  UpxZ                 PF06603.16          6     0.79    105   0.59   0.60   0.53   0.03\n",
            "20861  UQCC2_CBP6           PF20180.3         166     5.09     90   0.63   0.61   0.51   0.05\n",
            "20862  UQCC3                PF15141.11         20     1.52     87   0.65   0.67   0.58   0.03\n",
            "20863  UQ_con               PF00179.31         64     2.36    140   0.59   0.54   0.52   0.02\n",
            "20864  Urate_ox_N           PF06181.16         40     0.96    295   0.59   0.55   0.53   0.04\n",
            "20865  Urb2                 PF10441.14        103     5.48    209   0.59   0.58   0.47   0.03\n",
            "20866  Urease_alpha         PF00449.25         62     0.82    121   0.59   0.61   0.52   0.03\n",
            "20867  Urease_beta          PF00699.25        267     1.15     98   0.59   0.61   0.53   0.02\n",
            "20868  Urease_gamma         PF00547.23        136     1.20     99   0.59   0.63   0.54   0.04\n",
            "20869  Urease_linker        PF18473.6          13     4.43     34   1.59   1.61   1.58   0.17\n",
            "20870  UreD                 PF01774.22        310     5.62    201   0.59   0.60   0.44   0.01\n",
            "20871  UreE_C               PF05194.17         83     2.47     87   0.65   0.60   0.54   0.06\n",
            "20872  UreE_N               PF02814.20         62    11.75     62   0.90   0.96   0.87   0.06\n",
            "20873  UreF                 PF01730.21        247     8.13    151   0.59   0.62   0.53   0.04\n",
            "20874  Ureide_permease      PF07168.16         12     0.60    359   0.59   0.60   0.52   0.02\n",
            "20875  Ureidogly_lyase      PF04115.17         38     1.52    162   0.59   0.56   0.48   0.02\n",
            "20876  Uricase              PF01014.23        236     4.98    140   0.59   0.56   0.44   0.03\n",
            "20877  Urm1                 PF09138.16         18     0.90     96   0.60   0.61   0.52   0.03\n",
            "20878  URO-D                PF01208.22        103     4.93    346   0.59   0.59   0.52   0.01\n",
            "20879  Urocanase            PF01175.23        114     1.29    209   0.59   0.59   0.53   0.02\n",
            "20880  Urocanase_C          PF17392.7          69     0.96    196   0.59   0.58   0.53   0.02\n",
            "20881  Urocanase_N          PF17391.7          57     0.90    127   0.59   0.57   0.54   0.01\n",
            "20882  Uroplakin_II         PF07353.17          2     0.50    179   0.59   0.62   0.53   0.02\n",
            "20883  Urotensin_II         PF02083.20          4     4.00     12   3.06   2.56   3.05   0.75\n",
            "20884  US10                 PF17617.7           1     0.37    161   0.59   0.57   0.52   0.01\n",
            "20885  US2                  PF02476.20         15     2.03    124   0.59   0.55   0.49   0.02\n",
            "20886  US22                 PF02393.21         36     7.69    124   0.59   0.56   0.50   0.02\n",
            "20887  US30                 PF17624.7           1     0.38    282   0.59   0.58   0.51   0.01\n",
            "20888  US6                  PF17616.7           1     0.37    161   0.60   0.56   0.52   0.02\n",
            "20889  Use1                 PF09753.14         29     2.50    243   0.59   0.59   0.51   0.06\n",
            "20890  Usg                  PF06233.17         27     0.90     80   0.71   0.66   0.66   0.03\n",
            "20891  USH1C_N              PF21219.2           2     0.71     74   0.76   0.77   0.71   0.05\n",
            "20892  USHBP1_PDZ-bd        PF10506.14         20     4.49     65   0.86   0.92   0.82   0.11\n",
            "20893  Usher                PF00577.25         17     1.96    549   0.59   0.58   0.53   0.03\n",
            "20894  Usher_TcfC           PF17271.7           2     0.67    429   0.59   0.59   0.52   0.02\n",
            "20895  Uso1_p115_C          PF04871.18         15     2.87    121   0.59   0.65   0.49   0.15\n",
            "20896  Uso1_p115_head       PF04869.19         72     2.76    313   0.59   0.59   0.51   0.03\n",
            "20897  Usp                  PF00582.31        151    17.40    141   0.59   0.64   0.54   0.03\n",
            "20898  USP19_linker         PF16602.10          6     0.58    121   0.59   0.59   0.51   0.04\n",
            "20899  Usp38-like_N         PF21246.2           6     0.86    337   0.59   0.58   0.53   0.03\n",
            "20900  USP47_C              PF19718.4           9     0.94    240   0.59   0.57   0.52   0.04\n",
            "20901  USP7_C2              PF14533.11        128     3.44    204   0.59   0.58   0.50   0.04\n",
            "20902  USP7_ICP0_bdg        PF12436.13         99     2.40    241   0.59   0.57   0.47   0.03\n",
            "20903  USP8_dimer           PF08969.16         17     2.34    114   0.59   0.58   0.53   0.06\n",
            "20904  USP8_interact        PF08941.15         18     0.83    179   0.59   0.56   0.53   0.02\n",
            "20905  UspA1_rep            PF18792.6           4     4.00     13   2.42   2.55   2.40   0.57\n",
            "20906  UspB                 PF10625.14          8     0.66    107   0.59   0.57   0.53   0.05\n",
            "20907  USP_UIM_N            PF21909.1          14     2.64     53   1.05   1.11   1.00   0.08\n",
            "20908  Ustilago_mating      PF05722.17          1     0.40    226   0.59   0.59   0.52   0.02\n",
            "20909  UstYa                PF11807.13         45     6.22    220   0.59   0.50   0.49   0.03\n",
            "20910  UT                   PF03253.19         58     3.30    292   0.59   0.62   0.54   0.07\n",
            "20911  Uteroglobin          PF01099.22         69     3.58     90   0.63   0.62   0.58   0.06\n",
            "20912  Utp11                PF03998.18        153     2.42    242   0.59   0.58   0.48   0.10\n",
            "20913  Utp12                PF04003.17        353     8.27    107   0.59   0.61   0.54   0.05\n",
            "20914  Utp13                PF08625.16        136     2.83    140   0.59   0.59   0.53   0.05\n",
            "20915  Utp14                PF04615.18        129     3.55    731   0.59   0.59   0.46   0.09\n",
            "20916  UTP15_C              PF09384.15        135     4.00    147   0.59   0.61   0.55   0.04\n",
            "20917  UTP20_C              PF20416.3          60     2.66    221   0.59   0.62   0.55   0.04\n",
            "20918  UTP20_N              PF07539.17        133     4.52    615   0.59   0.61   0.51   0.03\n",
            "20919  Utp21                PF04192.17        107     2.24    209   0.59   0.60   0.52   0.03\n",
            "20920  Utp25_C              PF06862.17         26     1.36    190   0.59   0.56   0.53   0.04\n",
            "20921  UTP25_NTPase-like    PF22916.1         271     1.65    266   0.59   0.57   0.51   0.03\n",
            "20922  Utp8_b_propeller     PF10395.14         24     2.62    382   0.59   0.60   0.49   0.03\n",
            "20923  Utp8_C               PF22542.1          24     2.47    319   0.59   0.61   0.48   0.04\n",
            "20924  UTRA                 PF07702.18        136    10.09    141   0.59   0.61   0.56   0.02\n",
            "20925  UVB_sens_prot        PF04884.19         57     2.30    241   0.59   0.60   0.54   0.03\n",
            "20926  UvdE                 PF03851.19          6     1.08    275   0.59   0.56   0.52   0.02\n",
            "20927  UVR                  PF02151.24         69    69.00     36   1.28   1.35   1.28   0.26\n",
            "20928  UvrA_DNA-bind        PF17755.6         336     4.22    110   0.59   0.58   0.55   0.02\n",
            "20929  UvrA_inter           PF17760.6          61     2.59    109   0.59   0.62   0.56   0.04\n",
            "20930  UvrB                 PF12344.13        546     4.21     43   1.28   1.22   1.26   0.12\n",
            "20931  UvrB_D3-like         PF22679.1          98     1.30    158   0.59   0.57   0.48   0.04\n",
            "20932  UvrB_inter           PF17757.6         224     2.72     91   0.63   0.63   0.60   0.02\n",
            "20933  UvrC_RNaseH          PF22920.1         364     4.77    116   0.59   0.60   0.54   0.02\n",
            "20934  UvrC_RNaseH_dom      PF08459.16        580     2.56    159   0.59   0.59   0.49   0.02\n",
            "20935  UvrD-helicase        PF00580.26         36     2.82    271   0.59   0.59   0.49   0.03\n",
            "20936  UvrD_C               PF13361.11         17     2.59    350   0.59   0.59   0.48   0.04\n",
            "20937  UvrD_C_2             PF13538.11         56    11.13     52   1.07   1.05   1.04   0.06\n",
            "20938  UVSSA_N              PF20867.2          21     1.27    142   0.59   0.59   0.54   0.03\n",
            "20939  UvsW-1               PF11637.13          2     1.18     56   0.99   1.02   0.94   0.07\n",
            "20940  UvsW_N               PF21241.2           8     0.68     81   0.70   0.65   0.64   0.04\n",
            "20941  UvsY                 PF11056.13          7     1.19    128   0.59   0.56   0.53   0.05\n",
            "20942  UxaC                 PF02614.19         14     0.89    464   0.59   0.55   0.53   0.02\n",
            "20943  UxaE                 PF16257.10         15     1.01    475   0.59   0.58   0.53   0.03\n",
            "20944  UXS1_N               PF11803.13          8     1.04     75   0.75   0.76   0.67   0.02\n",
            "20945  UxuA                 PF03786.18          8     0.86    352   0.59   0.56   0.52   0.01\n",
            "20946  V-ATPase_C           PF03223.20        106     2.19    371   0.59   0.58   0.53   0.03\n",
            "20947  V-ATPase_G           PF03179.20         25     1.49    105   0.59   0.62   0.55   0.10\n",
            "20948  V-ATPase_G_2         PF16999.10         10     1.06    104   0.59   0.65   0.54   0.10\n",
            "20949  V-ATPase_H_C         PF11698.13        111     1.99    117   0.59   0.56   0.52   0.04\n",
            "20950  V-ATPase_H_N         PF03224.19         31     4.00    314   0.59   0.60   0.50   0.04\n",
            "20951  V-cyclin_C           PF21608.2           1     0.36     97   0.59   0.54   0.52   0.04\n",
            "20952  V-set                PF07686.22         54     8.39    109   0.59   0.54   0.50   0.02\n",
            "20953  V-set_2              PF15910.10          6     0.53    112   0.59   0.55   0.52   0.03\n",
            "20954  V-set_CD47           PF08204.16         12     1.44    111   0.59   0.52   0.50   0.06\n",
            "20955  V-SNARE              PF05008.20         71    10.40     79   0.72   0.75   0.67   0.10\n",
            "20956  V-SNARE_C            PF12352.13         20     6.17     66   0.85   0.89   0.84   0.10\n",
            "20957  v110                 PF01639.22          4     0.65    104   0.59   0.46   0.51   0.06\n",
            "20958  V18-19               PF21507.2           1     0.39    380   0.59   0.57   0.51   0.02\n",
            "20959  V1R                  PF03402.19         15     1.36    292   0.59   0.57   0.54   0.06\n",
            "20960  V1R_C                PF08983.15          8     3.04     48   1.15   1.16   1.06   0.15\n",
            "20961  V4R                  PF02830.23         25     7.46     62   0.90   0.83   0.89   0.08\n",
            "20962  Vac14_Fab1_bd        PF12755.12         27     1.12     97   0.59   0.60   0.54   0.03\n",
            "20963  Vac14_Fig4_bd        PF11916.13         71     1.30    179   0.59   0.61   0.53   0.05\n",
            "20964  Vac17                PF17321.7          19     3.10    441   0.59   0.59   0.45   0.07\n",
            "20965  Vac7                 PF12751.12         31     1.14    383   0.59   0.59   0.49   0.03\n",
            "20966  VacA                 PF02691.20          2     0.46   1002   0.59   0.60   0.52   0.02\n",
            "20967  VacA2                PF03077.19         13     2.16     60   0.93   0.95   0.89   0.10\n",
            "20968  Vac_Fusion           PF02346.21          7     1.92     56   0.99   1.01   0.96   0.09\n",
            "20969  Vac_ImportDeg        PF09783.14         75     1.32    162   0.59   0.53   0.48   0.04\n",
            "20970  VAD1-2               PF15310.11         24     1.85    249   0.59   0.59   0.52   0.06\n",
            "20971  Val_tRNA-synt_C      PF10458.14         18     3.07     66   0.85   0.92   0.83   0.11\n",
            "20972  Vanabin-2            PF11437.13          8     0.75     93   0.61   0.42   0.53   0.23\n",
            "20973  VanA_C               PF19112.5          31     5.73    193   0.59   0.56   0.50   0.02\n",
            "20974  Vanin_C              PF19018.5          69     4.14    165   0.59   0.52   0.48   0.04\n",
            "20975  VanW                 PF04294.18        314     2.52    130   0.59   0.58   0.54   0.02\n",
            "20976  VanY                 PF02557.22         41     1.84    132   0.59   0.54   0.52   0.04\n",
            "20977  VanZ                 PF04892.17        257     6.94    126   0.59   0.59   0.33   0.06\n",
            "20978  VAPB_antitox         PF02697.19         33     4.35     69   0.82   0.86   0.74   0.09\n",
            "20979  VapB_antitoxin       PF09957.14         24     7.75     47   1.17   1.27   1.13   0.12\n",
            "20980  VapE-like_dom        PF05272.16         46     2.40    217   0.59   0.56   0.52   0.03\n",
            "20981  VAR1                 PF05316.17         13     2.18    195   0.59   0.57   0.48   0.06\n",
            "20982  VARLMGL              PF14383.11         19     7.77     32   1.69   1.75   1.68   0.13\n",
            "20983  VAS1_LD              PF05827.17         23     2.96    140   0.59   0.53   0.47   0.04\n",
            "20984  Vasculin             PF15337.11          6     1.08     97   0.59   0.60   0.52   0.07\n",
            "20985  VasI                 PF11319.13         12     1.49    184   0.59   0.56   0.52   0.03\n",
            "20986  VasL                 PF12486.13          6     1.29    147   0.59   0.58   0.54   0.05\n",
            "20987  Vasohibin            PF14822.11         10     0.86    245   0.59   0.56   0.53   0.02\n",
            "20988  VASP_tetra           PF08776.16         17     4.05     36   1.51   1.59   1.50   0.27\n",
            "20989  VASt                 PF16016.10        301     7.26    152   0.59   0.54   0.47   0.03\n",
            "20990  VasX_N               PF20249.3          33     4.18    156   0.59   0.54   0.44   0.02\n",
            "20991  VATC                 PF18716.6          12     2.88     43   1.28   1.04   1.23   0.21\n",
            "20992  vATP-synt_AC39       PF01992.21        269    10.83    339   0.59   0.61   0.50   0.03\n",
            "20993  vATP-synt_E          PF01991.23         12     2.06    199   0.59   0.63   0.54   0.06\n",
            "20994  Vault                PF01505.23        116    17.24     42   1.31   1.30   1.23   0.03\n",
            "20995  Vault_2              PF17794.6           6     1.63     60   0.93   0.91   0.86   0.02\n",
            "20996  Vault_3              PF17795.6           5     1.18     62   0.90   0.93   0.86   0.03\n",
            "20997  Vault_4              PF17796.6           1     0.61     63   0.89   0.92   0.83   0.03\n",
            "20998  VbhA                 PF18495.6          33    16.77     47   1.17   1.25   1.17   0.13\n",
            "20999  VBS                  PF08913.15          2     0.50    125   0.59   0.61   0.53   0.03\n",
            "21000  VC2046               PF11993.13         12     1.74    163   0.59   0.60   0.52   0.04\n",
            "21001  VCA0040-like         PF04018.18         50     1.47    246   0.59   0.62   0.53   0.06\n",
            "21002  VCD                  PF20792.2           3     0.69    240   0.59   0.60   0.53   0.02\n",
            "21003  VCH_CASS14           PF18315.6           1     0.39     97   0.60   0.60   0.52   0.01\n",
            "21004  VCIP135_N            PF19437.4          15     0.79    169   0.59   0.57   0.52   0.03\n",
            "21005  VCPO_2nd             PF22778.1           1     0.38    177   0.59   0.57   0.52   0.02\n",
            "21006  VCPO_N               PF17897.6           7     0.71    216   0.59   0.58   0.52   0.01\n",
            "21007  VCX_VCY              PF15231.11          2     0.53    122   0.59   0.62   0.51   0.09\n",
            "21008  VD10_N               PF08476.15          7     3.11     43   1.28   1.23   1.26   0.09\n",
            "21009  VDE                  PF07137.16         23     1.30    242   0.59   0.51   0.50   0.04\n",
            "21010  VEFS-Box             PF09733.14         24     1.53    137   0.59   0.54   0.54   0.04\n",
            "21011  VEG                  PF06257.16         86     2.26     73   0.77   0.78   0.74   0.05\n",
            "21012  VEGFR-1-like_5th     PF22971.1           1     0.37    130   0.59   0.55   0.51   0.01\n",
            "21013  VEGFR-1-like_Ig-like PF21339.2          12     0.95     83   0.68   0.64   0.62   0.02\n",
            "21014  VEGFR-2_TMD          PF17988.6          13     2.87     35   1.55   1.58   1.53   0.19\n",
            "21015  VEGFR1-3_N_Ig-like   PF22854.1           1     0.37    110   0.59   0.57   0.51   0.01\n",
            "21016  VEGF_C               PF14554.11          3     0.81     54   1.03   0.88   0.96   0.21\n",
            "21017  VEK-30               PF12107.13          8     8.00     17   2.61   2.65   2.60   0.77\n",
            "21018  Vel1p                PF10339.14          4     0.57    202   0.59   0.57   0.52   0.02\n",
            "21019  Velvet               PF11754.13         98     2.99    237   0.59   0.58   0.46   0.02\n",
            "21020  VERL                 PF11386.13          5     0.71     78   0.73   0.68   0.66   0.03\n",
            "21021  Vert_HS_TF           PF06546.16         14     1.36    273   0.59   0.62   0.51   0.04\n",
            "21022  Vert_IL3-reg_TF      PF06529.16         14     0.76    332   0.59   0.58   0.53   0.03\n",
            "21023  VESA1_N              PF12785.12          4     0.53    448   0.59   0.54   0.50   0.03\n",
            "21024  Vesiculo_matrix      PF06326.17          2     0.78    240   0.59   0.57   0.52   0.01\n",
            "21025  Vesi_VP2             PF05332.16          2     0.63    113   0.59   0.60   0.51   0.03\n",
            "21026  Vexin                PF15505.11          6     0.95     76   0.75   0.75   0.68   0.03\n",
            "21027  Vezatin              PF12632.12         28     4.84    282   0.59   0.60   0.50   0.04\n",
            "21028  VF530                PF09905.14         93     1.53     63   0.89   0.86   0.85   0.03\n",
            "21029  Vfa1                 PF08432.15         83     1.99    179   0.59   0.54   0.48   0.08\n",
            "21030  VGCC_alpha2          PF08473.16         12     0.69    431   0.59   0.55   0.52   0.02\n",
            "21031  VGCC_beta4Aa_N       PF12052.13          8     2.38     41   1.34   1.42   1.28   0.18\n",
            "21032  VGLL4                PF15245.11         12     1.08    217   0.59   0.56   0.49   0.04\n",
            "21033  VGPC1_C              PF16799.10          5     1.58     48   1.15   1.18   1.10   0.13\n",
            "21034  Vg_Tdu               PF07545.19         11     3.66     31   1.74   1.64   1.72   0.13\n",
            "21035  VHL                  PF01847.21          5     0.71     82   0.69   0.65   0.63   0.01\n",
            "21036  VHL_C                PF17211.8           4     1.23     49   1.13   1.15   1.08   0.10\n",
            "21037  VHP                  PF02209.24        172    12.75     36   1.51   1.44   1.51   0.18\n",
            "21038  Vhr1                 PF04001.18         20     0.76     91   0.63   0.62   0.57   0.06\n",
            "21039  VHS                  PF00790.24         15     2.35    141   0.59   0.59   0.54   0.03\n",
            "21040  Vicilin_N            PF04702.17          4     0.75    147   0.59   0.48   0.50   0.22\n",
            "21041  VID                  PF22364.1           1     0.61     64   0.88   0.89   0.82   0.05\n",
            "21042  VID27                PF08553.15         50     0.80    356   0.59   0.59   0.53   0.02\n",
            "21043  VID27_N              PF17748.6          50     1.32    173   0.59   0.57   0.51   0.04\n",
            "21044  VID27_PH             PF17747.6          50     1.79    109   0.59   0.53   0.53   0.05\n",
            "21045  Vif                  PF00559.22          2     0.57    200   0.59   0.52   0.51   0.03\n",
            "21046  VIGSSK               PF14773.11         33     1.49     62   0.90   0.95   0.85   0.07\n",
            "21047  Vinculin             PF01044.24         10     1.32    852   0.59   0.61   0.52   0.04\n",
            "21048  VinK_C               PF21124.2          11     1.21     62   0.90   0.88   0.86   0.03\n",
            "21049  Vint                 PF14623.11         33     1.28    166   0.59   0.56   0.50   0.02\n",
            "21050  VioE                 PF18234.6           4     0.54    182   0.59   0.55   0.52   0.01\n",
            "21051  Vip3A_N              PF12495.13         12     0.44    177   0.59   0.61   0.52   0.04\n",
            "21052  VipB                 PF05943.17         83     1.56    301   0.59   0.57   0.54   0.01\n",
            "21053  VipB_2               PF18945.5          83     1.27    112   0.59   0.57   0.55   0.02\n",
            "21054  VipD-like_C          PF22140.1           3     0.60    137   0.59   0.59   0.52   0.04\n",
            "21055  VipE                 PF20876.2           5     0.82    125   0.59   0.58   0.52   0.03\n",
            "21056  Vir1p                PF22575.1           2     0.46    922   0.59   0.58   0.51   0.02\n",
            "21057  Viral_alk_exo        PF01771.22          9     1.58    461   0.59   0.57   0.51   0.02\n",
            "21058  Viral_Beta_CD        PF04530.17          3     0.72    123   0.59   0.55   0.51   0.02\n",
            "21059  Viral_coat           PF00729.23         10     2.34    207   0.59   0.58   0.51   0.02\n",
            "21060  Viral_cys_rich       PF08008.17          3     0.83     83   0.69   0.65   0.62   0.02\n",
            "21061  Viral_DNA_bi         PF02236.22          3     1.19     79   0.72   0.71   0.66   0.02\n",
            "21062  Viral_DNA_bp         PF00747.22          8     1.50   1126   0.59   0.57   0.52   0.01\n",
            "21063  Viral_DNA_Zn_bi      PF03728.18          8     1.68     98   0.59   0.51   0.49   0.05\n",
            "21064  Viral_env_E26        PF11050.13          1     0.40    225   0.59   0.58   0.52   0.03\n",
            "21065  Viral_helicase1      PF01443.23         72     6.97    228   0.59   0.59   0.48   0.02\n",
            "21066  Viral_Hsp90          PF03225.19         17     3.66    512   0.59   0.57   0.52   0.03\n",
            "21067  Viral_NABP           PF05515.16          2     0.64    227   0.59   0.56   0.51   0.02\n",
            "21068  Viral_P18            PF04521.18          2     0.65    137   0.59   0.59   0.51   0.02\n",
            "21069  Viral_Rep            PF02407.21          4     0.93     82   0.69   0.64   0.61   0.03\n",
            "21070  VirArc_Nuclease      PF12187.13          3     0.67    190   0.59   0.56   0.51   0.03\n",
            "21071  VirB3                PF05101.18         41     5.00     84   0.68   0.64   0.61   0.07\n",
            "21072  VirB7                PF17413.7           8     2.86     35   1.55   1.49   1.53   0.09\n",
            "21073  VirB8                PF04335.18         33     5.92    213   0.59   0.59   0.52   0.03\n",
            "21074  VirC1                PF07015.16          4     0.64    231   0.59   0.60   0.53   0.02\n",
            "21075  VirC2                PF07181.16          2     0.49    202   0.59   0.61   0.52   0.02\n",
            "21076  VirD1                PF07328.16          2     0.46    142   0.59   0.63   0.52   0.02\n",
            "21077  VirDNA-topo-I_N      PF09266.15         10     1.67     58   0.96   0.96   0.92   0.04\n",
            "21078  VirE1                PF12189.13          2     0.71     63   0.89   0.89   0.84   0.04\n",
            "21079  VirE2                PF07229.17          1     0.39    556   0.59   0.58   0.52   0.01\n",
            "21080  VirE3                PF06661.16          1     0.40    312   0.59   0.60   0.52   0.03\n",
            "21081  VirionAssem_T7       PF11653.13          1     0.43     99   0.59   0.65   0.52   0.06\n",
            "21082  VirJ                 PF06057.16          6     0.69    191   0.59   0.57   0.52   0.01\n",
            "21083  VirK                 PF06903.17          8     1.15     98   0.59   0.58   0.54   0.02\n",
            "21084  Virulence_fact       PF13769.11         28     2.18     81   0.70   0.70   0.65   0.04\n",
            "21085  Virulence_RhuM       PF13310.11         98     0.94    252   0.59   0.57   0.53   0.04\n",
            "21086  Virul_Fac            PF10139.14         12     1.25    878   0.59   0.59   0.53   0.02\n",
            "21087  Virul_fac_BrkB       PF03631.20         79     7.28    259   0.59   0.62   0.50   0.08\n",
            "21088  Vir_act_alpha_C      PF10400.14        170    10.70     78   0.73   0.73   0.61   0.06\n",
            "21089  VIR_N                PF15912.10          6     0.82    258   0.59   0.58   0.51   0.02\n",
            "21090  VIT                  PF08487.15        257     5.22    111   0.59   0.59   0.54   0.04\n",
            "21091  VIT1                 PF01988.24        433     4.30    214   0.59   0.64   0.51   0.04\n",
            "21092  VitD-bind_III        PF09164.15         10     1.27     65   0.86   0.78   0.82   0.08\n",
            "21093  Vitelline_membr      PF10542.14          7     1.75     38   1.44   1.31   1.37   0.28\n",
            "21094  Vitellogenin_N       PF01347.27         42     4.86    590   0.59   0.58   0.51   0.03\n",
            "21095  VitK2_biosynth       PF02621.19         77     4.18    252   0.59   0.60   0.50   0.01\n",
            "21096  VIT_2                PF13757.11          4     1.44     78   0.72   0.73   0.68   0.02\n",
            "21097  Vit_b-sht_shell      PF09175.15         25     1.75    167   0.59   0.55   0.55   0.03\n",
            "21098  Vit_open_b-sht       PF09172.16         88     9.04    303   0.59   0.59   0.49   0.02\n",
            "21099  VKGC_lumenal_dom     PF22777.1          39     1.51    152   0.59   0.52   0.53   0.03\n",
            "21100  VKOR                 PF07884.19        212     4.98    134   0.59   0.57   0.49   0.10\n",
            "21101  VLPT                 PF07122.16          2     1.88     30   1.79   1.82   1.76   0.22\n",
            "21102  Vma12                PF11712.13         86     2.82    137   0.59   0.60   0.46   0.03\n",
            "21103  VMA21                PF09446.15         45     5.30     64   0.88   0.88   0.83   0.16\n",
            "21104  Vma22_CCDC115        PF21730.2          61     3.30    143   0.59   0.58   0.49   0.06\n",
            "21105  VMAP-C               PF20028.4         122     9.27    226   0.59   0.58   0.34   0.05\n",
            "21106  VMAP-M0              PF19916.4          53     5.61    104   0.59   0.62   0.53   0.05\n",
            "21107  VMAP-M1              PF19963.4          24     4.85    113   0.59   0.58   0.47   0.05\n",
            "21108  VMAP-M12             PF20015.4           1     0.38    136   0.59   0.58   0.52   0.03\n",
            "21109  VMAP-M14             PF20018.4           2     0.50    132   0.59   0.58   0.52   0.04\n",
            "21110  VMAP-M15             PF20017.4           1     0.44     81   0.70   0.69   0.63   0.03\n",
            "21111  VMAP-M18             PF20021.4           1     0.39    109   0.59   0.59   0.52   0.02\n",
            "21112  VMAP-M19             PF20022.4           1     0.37    115   0.59   0.58   0.52   0.01\n",
            "21113  VMAP-M2              PF19965.4          24     3.29    214   0.59   0.64   0.51   0.04\n",
            "21114  VMAP-M20             PF20019.4           1     0.38    129   0.58   0.61   0.51   0.02\n",
            "21115  VMAP-M4              PF19966.4           3     0.79    107   0.59   0.62   0.53   0.02\n",
            "21116  VMAP-M5              PF19967.4           1     0.38    136   0.59   0.56   0.51   0.04\n",
            "21117  VMAP-M7              PF19968.4           2     0.61    160   0.59   0.61   0.53   0.02\n",
            "21118  VMAP-M8              PF19969.4           5     0.99     97   0.59   0.60   0.51   0.02\n",
            "21119  VMAP-M9              PF19970.4           1     0.39    135   0.59   0.59   0.51   0.04\n",
            "21120  Vmethyltransf        PF01660.22        122     7.81    310   0.59   0.55   0.44   0.02\n",
            "21121  Vmethyltransf_C      PF08456.15          3     0.88    230   0.59   0.59   0.51   0.02\n",
            "21122  VMR2                 PF17055.10          1     0.38    235   0.59   0.59   0.51   0.00\n",
            "21123  vMSA                 PF00695.24          2     0.62    447   0.59   0.57   0.50   0.01\n",
            "21124  Voldacs              PF03517.18         34     5.50    138   0.59   0.57   0.46   0.04\n",
            "21125  Voltage_CLC          PF00654.25        101     5.11    346   0.59   0.63   0.51   0.07\n",
            "21126  VOMI                 PF03762.22         33     1.45    167   0.59   0.51   0.50   0.05\n",
            "21127  VopS_N               PF20793.2           1     0.43    141   0.59   0.62   0.52   0.03\n",
            "21128  VP1-like_C           PF21416.2           7     1.63    135   0.59   0.57   0.53   0.02\n",
            "21129  VP16-like            PF22024.1           1     0.39    133   0.59   0.61   0.52   0.01\n",
            "21130  VP17_central_barrel  PF22210.1           3     0.64     82   0.69   0.67   0.63   0.02\n",
            "21131  VP1_protrusion       PF20855.2           1     0.40    132   0.59   0.58   0.52   0.03\n",
            "21132  VP1_VP3              PF06281.17          8     1.08     73   0.77   0.80   0.68   0.08\n",
            "21133  VP3-like             PF18965.5           3     0.89   1062   0.59   0.57   0.47   0.01\n",
            "21134  VP40                 PF07447.17          2     0.60    292   0.59   0.59   0.52   0.01\n",
            "21135  VP4_2                PF08935.15          2     0.79     84   0.68   0.68   0.61   0.07\n",
            "21136  VP4_haemagglut       PF00426.24          4     0.56    160   0.59   0.56   0.52   0.01\n",
            "21137  VP4_helical          PF17478.7           2     0.61    292   0.59   0.61   0.52   0.03\n",
            "21138  VP5                  PF20807.2           5     1.23    195   0.59   0.58   0.51   0.02\n",
            "21139  VP7                  PF00434.23          2     0.54    336   0.59   0.56   0.51   0.03\n",
            "21140  VP9                  PF09625.15          5     1.03     79   0.72   0.74   0.66   0.03\n",
            "21141  VPDSG-CTERM          PF18205.6          20    20.00     26   1.86   2.06   1.86   0.29\n",
            "21142  VPg                  PF20915.2           4     1.11     61   0.91   0.86   0.85   0.07\n",
            "21143  VPR                  PF00522.23         12     1.30     83   0.68   0.62   0.59   0.03\n",
            "21144  VPS11_C              PF12451.13         80    14.57     45   1.22   1.24   1.19   0.09\n",
            "21145  VPS13_C              PF21679.2         128     2.57    114   0.59   0.56   0.50   0.02\n",
            "21146  VPS13_DH-like        PF16909.10         76     2.41    173   0.59   0.60   0.54   0.02\n",
            "21147  VPS13_ext_chorein    PF16908.10        180     4.46    234   0.59   0.55   0.49   0.03\n",
            "21148  VPS13_mid_rpt        PF16910.10        116     3.27    237   0.59   0.60   0.54   0.03\n",
            "21149  VPS13_VAB            PF06650.17         24     2.83    565   0.59   0.56   0.50   0.03\n",
            "21150  VPS15-like_hel       PF22956.1           1     0.39    610   0.59   0.58   0.52   0.02\n",
            "21151  Vps16_C              PF04840.17          4     1.05    320   0.59   0.60   0.53   0.03\n",
            "21152  Vps16_N              PF04841.18          5     1.30    410   0.59   0.58   0.50   0.02\n",
            "21153  Vps23_core           PF09454.15         97     6.96     64   0.88   0.91   0.85   0.06\n",
            "21154  Vps26                PF03643.20          8     0.82    275   0.59   0.58   0.51   0.03\n",
            "21155  Vps27_GAT-like       PF21356.2          16     1.04     71   0.79   0.80   0.75   0.06\n",
            "21156  VPS28                PF03997.17         95     1.81    188   0.59   0.59   0.52   0.04\n",
            "21157  Vps35                PF03635.22        111     2.10    726   0.59   0.58   0.50   0.04\n",
            "21158  Vps36-NZF-N          PF16988.10         28     2.75     65   0.86   0.73   0.81   0.11\n",
            "21159  Vps36_ESCRT-II       PF11605.13         19     2.38     92   0.62   0.63   0.56   0.03\n",
            "21160  VPS38                PF17649.6          10     1.36    424   0.59   0.58   0.53   0.04\n",
            "21161  Vps39_1              PF10366.14         20     1.74    108   0.59   0.59   0.53   0.04\n",
            "21162  Vps39_2              PF10367.14         18     1.96    109   0.59   0.55   0.55   0.05\n",
            "21163  Vps4_C               PF09336.15        115     4.07     61   0.92   0.89   0.89   0.03\n",
            "21164  Vps5                 PF09325.15         19     2.18    236   0.59   0.59   0.55   0.06\n",
            "21165  VPS51_Exo84_N        PF08700.16         44     8.29     87   0.65   0.67   0.63   0.07\n",
            "21166  Vps52_C              PF20655.2           5     1.11    310   0.59   0.57   0.52   0.03\n",
            "21167  Vps52_CC             PF04129.17          5     1.26    173   0.59   0.61   0.53   0.05\n",
            "21168  VPS53_C              PF16854.10         56     2.59     89   0.64   0.66   0.59   0.05\n",
            "21169  Vps53_N              PF04100.17          6     1.13    372   0.59   0.58   0.52   0.04\n",
            "21170  Vps54                PF07928.17        103     3.25    133   0.59   0.60   0.54   0.05\n",
            "21171  Vps54_N              PF10475.14          9     1.59    291   0.59   0.57   0.54   0.05\n",
            "21172  Vps55                PF04133.19        108     2.69    120   0.59   0.60   0.51   0.07\n",
            "21173  Vps62                PF06101.16         36     0.77    541   0.59   0.56   0.52   0.01\n",
            "21174  Vps8                 PF12816.12        133     2.67    194   0.59   0.58   0.52   0.03\n",
            "21175  VPS9                 PF02204.23         43     4.69    104   0.59   0.58   0.53   0.03\n",
            "21176  VpsR                 PF20161.4          25     2.01    121   0.59   0.56   0.54   0.02\n",
            "21177  VpsT-like_REC        PF21155.2           3     0.67    124   0.59   0.59   0.53   0.03\n",
            "21178  Vpu                  PF00558.24         16     1.77     81   0.70   0.70   0.62   0.05\n",
            "21179  VP_N-CPKC            PF11475.13          1     1.00     32   1.51   1.36   1.47   0.16\n",
            "21180  VQ                   PF05678.19         88    53.12     28   1.92   1.92   1.92   0.13\n",
            "21181  VraX                 PF17412.7           2     0.87     55   1.01   0.97   0.96   0.05\n",
            "21182  vRNAP_dom            PF21769.2          18     1.32    145   0.59   0.58   0.53   0.03\n",
            "21183  vRNAP_dom_2          PF21867.1           4     0.68    117   0.59   0.59   0.52   0.03\n",
            "21184  vRNAP_plug           PF21624.2           1     0.45     85   0.67   0.69   0.60   0.02\n",
            "21185  VRP1                 PF03538.19          3     0.84    328   0.59   0.58   0.51   0.02\n",
            "21186  VRP3                 PF03536.20          4     0.59    240   0.59   0.57   0.53   0.01\n",
            "21187  VRR_NUC              PF08774.16         25     4.98    116   0.59   0.58   0.50   0.02\n",
            "21188  VSG_B                PF13206.11         55     6.16    352   0.59   0.56   0.50   0.06\n",
            "21189  VSP                  PF03302.18          6     0.57    397   0.59   0.52   0.51   0.09\n",
            "21190  Vsr                  PF03852.20          4     1.00     74   0.76   0.70   0.72   0.02\n",
            "21191  Vta1                 PF04652.21        127     4.10    145   0.59   0.57   0.52   0.05\n",
            "21192  Vta1_C               PF18097.6          13     6.33     38   1.44   1.50   1.43   0.14\n",
            "21193  VTC                  PF09359.15         41     3.66    235   0.59   0.56   0.47   0.04\n",
            "21194  VtrA_C               PF22100.1           2     0.58     87   0.65   0.66   0.59   0.04\n",
            "21195  VtrC                 PF22280.1           1     0.39    126   0.59   0.56   0.52   0.04\n",
            "21196  VTT_dom              PF09335.16         47     7.18    123   0.59   0.60   0.52   0.07\n",
            "21197  VUPS                 PF20973.2          82     2.06    206   0.59   0.61   0.55   0.08\n",
            "21198  Vut_1                PF02592.20        275     4.24    152   0.59   0.59   0.49   0.08\n",
            "21199  VWA                  PF00092.33        126     7.93    174   0.59   0.61   0.49   0.03\n",
            "21200  vWA-TerF-like        PF10138.14         16     1.11    200   0.59   0.58   0.53   0.02\n",
            "21201  Vwaint               PF14624.11         49     3.65     70   0.80   0.75   0.72   0.04\n",
            "21202  VWA_2                PF13519.11        158     9.31    107   0.59   0.62   0.47   0.03\n",
            "21203  VWA_3                PF13768.11          9     3.63    155   0.59   0.61   0.51   0.02\n",
            "21204  VWA_3_C              PF18571.6          11     2.71     47   1.17   1.27   1.15   0.10\n",
            "21205  VWA_CoxE             PF05762.19         11     2.44    222   0.59   0.61   0.51   0.02\n",
            "21206  VWA_N                PF08399.16         35     1.59    128   0.59   0.53   0.52   0.04\n",
            "21207  VWA_N2               PF16164.10          4     0.69     79   0.72   0.64   0.65   0.06\n",
            "21208  VWC                  PF00093.23         19     1.85     57   0.98   0.69   0.85   0.36\n",
            "21209  VWD                  PF00094.30         40     5.06    156   0.59   0.54   0.50   0.03\n",
            "21210  vWF_A                PF12450.13         74     1.24     94   0.61   0.58   0.55   0.04\n",
            "21211  V_ATPase_I           PF01496.24        218     1.70    815   0.59   0.56   0.52   0.03\n",
            "21212  V_ATPase_I_N         PF18670.6          12     1.23     90   0.63   0.71   0.59   0.04\n",
            "21213  V_ATPase_prox        PF18274.6           1     0.75     52   1.07   1.12   1.01   0.06\n",
            "21214  V_cholerae_RfbT      PF05575.16          2     0.38    286   0.59   0.56   0.51   0.02\n",
            "21215  W2                   PF02020.23        325     6.70     79   0.72   0.70   0.63   0.07\n",
            "21216  WaaY                 PF06176.16          3     0.48    229   0.59   0.58   0.52   0.03\n",
            "21217  WAC_Acf1_DNA_bd      PF10537.14         85     2.24    101   0.59   0.54   0.55   0.05\n",
            "21218  Waikav_capsid_1      PF12264.13          3     0.57    197   0.59   0.57   0.53   0.02\n",
            "21219  WAK                  PF08488.16         13     1.13     81   0.70   0.62   0.63   0.04\n",
            "21220  WAK_assoc            PF14380.11         46     2.19    104   0.59   0.49   0.48   0.10\n",
            "21221  WAP                  PF00095.26        340     7.31     41   1.34   1.02   1.19   0.48\n",
            "21222  Wap1                 PF16997.10         12     1.91    379   0.59   0.59   0.51   0.05\n",
            "21223  WAPL                 PF07814.18         56     5.94    344   0.59   0.62   0.51   0.04\n",
            "21224  WASH-4_N             PF14745.11         15     2.35    572   0.59   0.58   0.53   0.04\n",
            "21225  WASH-7_C             PF14746.11         29     1.58    175   0.59   0.57   0.53   0.04\n",
            "21226  WASH-7_mid           PF14744.11         29     1.75    346   0.59   0.57   0.53   0.03\n",
            "21227  WASH_WAHD            PF11945.13          8     1.36    292   0.59   0.59   0.52   0.03\n",
            "21228  WavE                 PF07507.16          5     1.05    305   0.59   0.56   0.52   0.01\n",
            "21229  WbmS-like            PF22537.1          12     1.29    186   0.59   0.54   0.52   0.03\n",
            "21230  WBP-1                PF11669.13         10     1.15    102   0.59   0.48   0.46   0.06\n",
            "21231  Wbp11                PF09429.15         76     4.89     78   0.73   0.76   0.68   0.17\n",
            "21232  WbqC                 PF08889.16         67     2.85    218   0.59   0.55   0.53   0.02\n",
            "21233  WBS28                PF15164.11         10     0.70    267   0.59   0.60   0.53   0.04\n",
            "21234  WBS_methylT          PF12589.13         44     2.31     82   0.69   0.68   0.58   0.11\n",
            "21235  WC-rich              PF20053.4          46     1.47    106   0.59   0.49   0.51   0.06\n",
            "21236  WcbI                 PF18588.6          52     4.17    205   0.59   0.58   0.52   0.02\n",
            "21237  WCCH                 PF03716.19         30    30.00     25   2.02   1.62   2.00   0.46\n",
            "21238  WCOB                 PF19913.4          39     3.09    121   0.59   0.57   0.51   0.03\n",
            "21239  WCOR413              PF05562.16         18     0.85    183   0.59   0.61   0.52   0.04\n",
            "21240  WD-like_fungi        PF20493.3          21     1.70    199   0.59   0.52   0.53   0.06\n",
            "21241  WD40                 PF00400.37       1465  1465.00     39   1.03   0.95   0.91   0.08\n",
            "21242  WD40_2               PF19056.5          20     1.06    488   0.59   0.59   0.51   0.02\n",
            "21243  WD40_3               PF15911.10         37     5.14     57   0.98   0.91   0.95   0.03\n",
            "21244  WD40_4               PF16300.10        156     9.71     44   1.25   1.19   1.22   0.05\n",
            "21245  WD40_alt             PF14077.11          6     1.49     48   1.15   1.18   1.11   0.06\n",
            "21246  WD40_like            PF17005.10          6     1.16    301   0.59   0.58   0.53   0.04\n",
            "21247  WDCP                 PF15390.11         24     0.88    694   0.59   0.59   0.53   0.02\n",
            "21248  WDR54                PF21031.2           1     0.38    334   0.59   0.58   0.52   0.01\n",
            "21249  WDR93                PF21030.2          21     1.33    321   0.59   0.57   0.52   0.01\n",
            "21250  WEF-hand             PF19914.4          39     3.43     81   0.70   0.72   0.60   0.07\n",
            "21251  WelO5                PF22814.1          10     1.37    277   0.59   0.57   0.52   0.01\n",
            "21252  WEMBL                PF05701.16         24     1.30    562   0.59   0.64   0.54   0.08\n",
            "21253  WGG                  PF10273.14        140     4.00     80   0.71   0.69   0.65   0.04\n",
            "21254  WGR                  PF05406.20        105     5.30     79   0.72   0.66   0.66   0.06\n",
            "21255  WG_beta_rep          PF14903.11        163   163.00     35   1.53   1.42   1.51   0.15\n",
            "21256  WH1                  PF00568.28          8     1.54    111   0.59   0.54   0.51   0.03\n",
            "21257  WH2                  PF02205.25         39    39.00     30   1.51   1.64   1.50   0.15\n",
            "21258  WHAMM-JMY_N          PF15920.10          5     0.57    162   0.59   0.56   0.51   0.02\n",
            "21259  WHD                  PF22977.1          75     6.51    221   0.59   0.61   0.52   0.03\n",
            "21260  Wheel                PF18972.5          24     1.33    114   0.59   0.57   0.51   0.03\n",
            "21261  WHEP-TRS             PF00458.25        139     7.56     53   1.05   1.15   1.02   0.13\n",
            "21262  WHH                  PF14414.11         20     8.10     43   1.28   1.13   1.19   0.15\n",
            "21263  Whi5                 PF08528.16         67    67.00     25   2.05   2.08   2.05   0.20\n",
            "21264  WhiA_N               PF10298.14         58     4.80     86   0.66   0.70   0.63   0.05\n",
            "21265  Whib                 PF02467.21         62     2.07     65   0.86   0.75   0.76   0.09\n",
            "21266  WHIM1                PF15612.11         94    94.00     46   1.11   1.12   1.08   0.05\n",
            "21267  Whirly               PF08536.16         21     1.22    136   0.59   0.60   0.52   0.02\n",
            "21268  WI12                 PF07107.16          4     0.67    104   0.59   0.58   0.51   0.03\n",
            "21269  WIF                  PF02019.23         23     1.60    131   0.59   0.56   0.52   0.02\n",
            "21270  WipA_N               PF21662.2           5     0.91    105   0.59   0.58   0.54   0.07\n",
            "21271  WipA_Phos            PF21663.2          31     1.49    238   0.59   0.57   0.52   0.03\n",
            "21272  WIV_2                PF23017.1           4     0.68     89   0.64   0.63   0.58   0.03\n",
            "21273  WIV_dom              PF22532.1           7     2.07     66   0.85   0.86   0.80   0.06\n",
            "21274  WIV_dom_3            PF22533.1           5     1.32     85   0.67   0.66   0.59   0.04\n",
            "21275  WIYLD                PF10440.14         27     3.36     61   0.92   0.91   0.87   0.04\n",
            "21276  WKF                  PF10180.14        117     4.13     63   0.89   0.82   0.84   0.07\n",
            "21277  WKTW_rpt             PF20853.2           6     2.72     21   2.52   2.04   2.46   1.03\n",
            "21278  WLM                  PF08325.15         18     1.75    190   0.59   0.58   0.51   0.03\n",
            "21279  WLS-like_TM          PF06664.17         51     4.39    247   0.59   0.55   0.50   0.10\n",
            "21280  WLS_GOLD             PF21883.1          68     1.48    183   0.59   0.53   0.51   0.03\n",
            "21281  WND                  PF07861.16          3     0.42    239   0.59   0.58   0.52   0.04\n",
            "21282  wnt                  PF00110.24        147     1.36    305   0.59   0.49   0.51   0.08\n",
            "21283  Wound_ind            PF08186.16          4     1.52     44   1.25   1.24   1.12   0.08\n",
            "21284  WPP                  PF13943.11         23     0.96    102   0.59   0.61   0.52   0.03\n",
            "21285  WRC                  PF08879.15         26     3.56     43   1.28   1.06   1.26   0.25\n",
            "21286  WRKY                 PF03106.20        197     2.69     59   0.95   0.77   0.87   0.14\n",
            "21287  WRNPLPNID            PF15017.11         30     3.99     70   0.80   0.78   0.67   0.10\n",
            "21288  WRW                  PF10206.14         19     1.03    102   0.59   0.50   0.54   0.04\n",
            "21289  WsaF_C               PF22772.1          99     2.36    133   0.59   0.59   0.54   0.01\n",
            "21290  WsaF_N               PF21374.2          10     0.92    156   0.59   0.55   0.52   0.03\n",
            "21291  WSC                  PF01822.24        102     3.43     82   0.69   0.55   0.61   0.15\n",
            "21292  WSD                  PF15613.11         61     3.44     69   0.82   0.79   0.71   0.05\n",
            "21293  WSK                  PF03832.18         13    10.27     29   1.85   1.87   1.85   0.27\n",
            "21294  WSLR                 PF20023.4          58    58.00     39   1.39   1.38   1.35   0.10\n",
            "21295  WSN                  PF02206.23        165    10.05     66   0.85   0.92   0.84   0.07\n",
            "21296  WSS_VP               PF12175.13          3     0.88    201   0.59   0.61   0.49   0.03\n",
            "21297  WS_DGAT_C            PF06974.18        113     4.31    145   0.59   0.60   0.54   0.02\n",
            "21298  WS_DGAT_cat          PF03007.21         15     2.57    263   0.59   0.59   0.54   0.02\n",
            "21299  WT1                  PF02165.20          9     0.61    290   0.59   0.56   0.51   0.02\n",
            "21300  Wtap                 PF17098.10         31     3.41    155   0.59   0.62   0.54   0.07\n",
            "21301  WTF                  PF03303.18          7     0.91    224   0.59   0.57   0.50   0.03\n",
            "21302  WVELL                PF14043.11         12     1.07     74   0.76   0.73   0.71   0.04\n",
            "21303  WW                   PF00397.31        422   181.76     31   1.74   1.50   1.73   0.27\n",
            "21304  WWamide              PF08258.16          2     2.00      7   2.66   2.05   2.63   1.14\n",
            "21305  WWE                  PF02825.25        175     8.32     67   0.84   0.71   0.74   0.09\n",
            "21306  WWE_1                PF22005.1           5     0.90     72   0.78   0.75   0.73   0.05\n",
            "21307  WW_1                 PF18507.6          10     2.47     27   1.98   1.85   1.96   0.19\n",
            "21308  WW_FCH_linker        PF16623.10          8     0.65     92   0.62   0.58   0.55   0.04\n",
            "21309  WW_like              PF17890.6           1     0.67     53   1.05   0.99   0.99   0.07\n",
            "21310  WW_USP8              PF20625.2           6     1.22     33   1.64   1.49   1.60   0.20\n",
            "21311  Wx5_PLAF3D7          PF09688.15         11     1.49    144   0.59   0.51   0.54   0.08\n",
            "21312  WXG100               PF06013.17         69    10.48     86   0.66   0.66   0.64   0.08\n",
            "21313  WxL                  PF13731.11         53     2.83    214   0.59   0.59   0.44   0.04\n",
            "21314  WxLIP_HBD            PF11797.13         54     2.74    138   0.59   0.57   0.55   0.05\n",
            "21315  WxLIP_PGBD           PF06030.17         59     2.73    121   0.59   0.59   0.55   0.04\n",
            "21316  WXXGXW               PF12779.12         49    44.79     26   2.06   1.64   2.04   0.72\n",
            "21317  WYL                  PF13280.11        310    10.78    172   0.59   0.58   0.52   0.03\n",
            "21318  WYL_2                PF10902.13         35     4.29     74   0.76   0.71   0.72   0.04\n",
            "21319  WYL_3                PF18488.6          14     2.81     63   0.89   0.86   0.85   0.09\n",
            "21320  Wyosine_form         PF08608.17         70     6.85     63   0.89   0.86   0.82   0.02\n",
            "21321  Wza_C                PF18412.6          28     8.34     30   1.80   1.78   1.79   0.18\n",
            "21322  Wzt_C                PF14524.11        129     9.48    143   0.59   0.58   0.50   0.02\n",
            "21323  WzyE                 PF06899.16          4     0.46    447   0.59   0.59   0.52   0.05\n",
            "21324  Wzy_C                PF04932.20         75    13.22    149   0.59   0.61   0.50   0.06\n",
            "21325  Wzy_C_2              PF11846.13         19     4.08    191   0.59   0.58   0.55   0.03\n",
            "21326  Wzz                  PF02706.20         37     5.98     91   0.63   0.67   0.54   0.05\n",
            "21327  W_rich_C             PF07483.16         13     1.13    105   0.59   0.54   0.53   0.06\n",
            "21328  X                    PF00739.24          5     0.60    142   0.59   0.58   0.52   0.02\n",
            "21329  X-Tfes_XVIPCD        PF20410.3          22     3.10    101   0.59   0.60   0.52   0.05\n",
            "21330  X25_BaPul_like       PF22058.1          35     1.99     87   0.65   0.60   0.58   0.04\n",
            "21331  X8                   PF07983.18        186     3.01     76   0.74   0.63   0.67   0.09\n",
            "21332  XAF1_C               PF18608.6           8     1.65     54   1.03   0.94   0.99   0.09\n",
            "21333  Xan_ur_permease      PF00860.25         20     3.77    389   0.59   0.62   0.53   0.07\n",
            "21334  XAP5                 PF04921.19         55     1.01    155   0.59   0.55   0.49   0.03\n",
            "21335  XdhC_C               PF13478.11        463     3.66    125   0.59   0.63   0.47   0.02\n",
            "21336  XdhC_CoxI            PF02625.21        374    11.06     68   0.83   0.88   0.77   0.05\n",
            "21337  XendoU               PF09412.15         72     2.13    262   0.59   0.55   0.52   0.03\n",
            "21338  XET_C                PF06955.18        172     6.52     47   1.17   0.97   1.10   0.15\n",
            "21339  XFP                  PF03894.20        119     1.31    177   0.59   0.56   0.53   0.02\n",
            "21340  XFP_C                PF09363.15        213     1.39    202   0.59   0.56   0.54   0.02\n",
            "21341  XFP_N                PF09364.15          5     0.63    364   0.59   0.57   0.52   0.00\n",
            "21342  XG_FTase             PF03254.18          3     0.61    475   0.59   0.56   0.52   0.01\n",
            "21343  XH                   PF03469.19         58     1.50    131   0.59   0.56   0.54   0.04\n",
            "21344  XhlA                 PF10779.14         29     7.13     67   0.84   0.86   0.81   0.08\n",
            "21345  XhoI                 PF04555.18          4     0.53    191   0.59   0.58   0.52   0.01\n",
            "21346  Xin                  PF08043.17         52    52.00     16   2.69   2.54   2.69   0.23\n",
            "21347  XisH                 PF08814.15         62     1.20    134   0.59   0.59   0.54   0.04\n",
            "21348  XisI                 PF08869.16         93     1.44    110   0.59   0.56   0.53   0.04\n",
            "21349  XK-related           PF09815.14        103     4.61    337   0.59   0.55   0.51   0.08\n",
            "21350  XkdW                 PF09636.15          3     0.60    108   0.59   0.59   0.52   0.02\n",
            "21351  XLF                  PF09302.16        171     8.05    115   0.59   0.55   0.45   0.04\n",
            "21352  XLF_CC               PF21928.1          60    10.43     53   1.05   1.13   1.02   0.06\n",
            "21353  Xlink                PF00193.22        149     1.90     92   0.62   0.54   0.55   0.05\n",
            "21354  XMAP215_CLASP_TOG    PF21041.2         293     3.41    230   0.59   0.60   0.52   0.03\n",
            "21355  Xol-1_GHMP-like      PF09109.15          2     0.65    196   0.59   0.59   0.52   0.02\n",
            "21356  Xol-1_N              PF09108.15          2     0.56    160   0.59   0.59   0.52   0.02\n",
            "21357  XOO_2897-deam        PF14440.11          7     1.79    101   0.59   0.57   0.50   0.03\n",
            "21358  XopL_C               PF20817.2           2     0.59    173   0.59   0.58   0.51   0.01\n",
            "21359  Xp10_P7              PF20800.2           1     0.52     73   0.77   0.78   0.71   0.02\n",
            "21360  XPA_C                PF05181.17         49     2.99     51   1.09   1.04   1.04   0.08\n",
            "21361  XPA_N                PF01286.23         27     7.17     32   1.69   1.42   1.69   0.37\n",
            "21362  XPB_DRD              PF18458.6          72     6.24     57   0.98   0.98   0.93   0.05\n",
            "21363  XPC-binding          PF09280.16         82     5.54     57   0.98   0.99   0.97   0.13\n",
            "21364  XPD_arch             PF22139.1           2     0.55    110   0.59   0.60   0.53   0.02\n",
            "21365  XPG_I                PF00867.23        103     3.38     89   0.64   0.64   0.56   0.03\n",
            "21366  XPG_I_2              PF12813.12         16     2.21    240   0.59   0.58   0.50   0.02\n",
            "21367  XPG_N                PF00752.22         10     1.65    101   0.59   0.58   0.52   0.02\n",
            "21368  Xpo1                 PF08389.17         52     5.28    147   0.59   0.58   0.51   0.04\n",
            "21369  XRCC1_N              PF01834.21         29     1.44    148   0.59   0.56   0.53   0.03\n",
            "21370  XRCC4                PF06632.17         69     4.75    102   0.59   0.54   0.52   0.04\n",
            "21371  XRCC4_C              PF21925.1          10     1.53    112   0.59   0.60   0.52   0.10\n",
            "21372  XRCC4_CC             PF21924.1          10     1.00     78   0.72   0.74   0.68   0.07\n",
            "21373  Xre-like-HTH         PF20432.3         126    73.30     63   0.89   0.99   0.83   0.07\n",
            "21374  Xre_MbcA_ParS_C      PF09722.15        141    22.54     51   1.09   1.12   1.02   0.04\n",
            "21375  XRN1_D1              PF18332.6          83     1.36    191   0.59   0.58   0.53   0.02\n",
            "21376  XRN1_D2_D3           PF18334.6          90     1.35    228   0.59   0.59   0.53   0.03\n",
            "21377  XRN1_DBM             PF18245.6           1     1.00     26   1.60   1.46   1.55   0.30\n",
            "21378  XRN_M                PF17846.6          10     1.10    443   0.59   0.56   0.52   0.02\n",
            "21379  XRN_N                PF03159.23        125     1.47    241   0.59   0.57   0.49   0.02\n",
            "21380  xRRM                 PF19977.4          39     1.29    120   0.59   0.56   0.52   0.03\n",
            "21381  XS                   PF03468.19         66     1.91    113   0.59   0.53   0.51   0.03\n",
            "21382  XTBD                 PF11952.13         33     2.51     85   0.67   0.62   0.61   0.04\n",
            "21383  Xylo_C               PF12529.13         28     2.07    182   0.59   0.55   0.52   0.02\n",
            "21384  XylR_N               PF06505.16         50     2.05    101   0.59   0.61   0.54   0.02\n",
            "21385  XYPPX                PF02162.22          7     7.00     15   3.37   3.21   3.28   1.78\n",
            "21386  Y1_Tnp               PF01797.21         77     3.22    121   0.59   0.53   0.54   0.05\n",
            "21387  Y2_Tnp               PF04986.18         20     1.99    183   0.59   0.55   0.52   0.03\n",
            "21388  YaaC                 PF14175.11        311     6.51    307   0.59   0.55   0.46   0.03\n",
            "21389  YabA                 PF06156.18         44     1.21    103   0.59   0.54   0.51   0.06\n",
            "21390  YABBY                PF04690.18         52     0.87    164   0.59   0.54   0.49   0.03\n",
            "21391  YabP                 PF07873.16        316     7.43     64   0.88   0.90   0.84   0.07\n",
            "21392  YAcAr                PF10686.14         45     5.51     66   0.85   0.83   0.79   0.04\n",
            "21393  YacG                 PF03884.19         35     2.01     52   1.07   0.89   1.03   0.13\n",
            "21394  YadA_anchor          PF03895.20         38     8.70     60   0.93   0.94   0.89   0.12\n",
            "21395  YadA_head            PF05658.19        254   254.00     27   1.80   1.97   1.79   0.47\n",
            "21396  YadA_stalk           PF05662.19         73    22.95     43   1.28   1.35   1.25   0.20\n",
            "21397  Yae1_N               PF09811.14         56    31.37     39   1.40   1.46   1.40   0.24\n",
            "21398  YaeQ                 PF07152.17         95     1.56    173   0.59   0.58   0.54   0.03\n",
            "21399  YAF2_RYBP            PF17219.8          13     4.18     33   1.64   1.67   1.61   0.15\n",
            "21400  YafO_toxin           PF13957.11          9     1.34    101   0.59   0.54   0.51   0.03\n",
            "21401  YafQ_toxin           PF15738.10         74     1.96     88   0.65   0.62   0.58   0.04\n",
            "21402  YaiA                 PF16362.10          2     0.75     63   0.89   0.86   0.83   0.05\n",
            "21403  YaiO                 PF19413.4          80     4.91    173   0.59   0.56   0.54   0.04\n",
            "21404  YajC                 PF02699.20        602     5.09     78   0.73   0.75   0.63   0.06\n",
            "21405  YajR_YAM             PF21987.1           6     1.39     56   0.99   1.06   0.95   0.06\n",
            "21406  YARHG                PF13308.11         21     4.50     85   0.67   0.64   0.61   0.04\n",
            "21407  YbaB_DNA_bd          PF02575.21        579     5.89     91   0.63   0.67   0.57   0.08\n",
            "21408  YbaJ                 PF10757.14          4     0.57    118   0.59   0.56   0.52   0.03\n",
            "21409  YbbR                 PF07949.17        177    26.27     81   0.70   0.78   0.60   0.09\n",
            "21410  YbcO                 PF07102.17          8     1.09     92   0.62   0.58   0.56   0.04\n",
            "21411  YBD                  PF17725.6           5     0.77    201   0.59   0.57   0.50   0.03\n",
            "21412  YbeY                 PF02130.22        180     2.14    129   0.59   0.57   0.50   0.04\n",
            "21413  Ybfg_C               PF21015.2           3     0.96    148   0.59   0.63   0.52   0.04\n",
            "21414  YbfN                 PF13982.11          2     0.53     88   0.64   0.61   0.58   0.05\n",
            "21415  YbgS                 PF13985.11          5     0.70    124   0.59   0.60   0.52   0.07\n",
            "21416  YbgT_YccB            PF08173.16         26     7.08     27   1.98   1.83   1.98   0.42\n",
            "21417  YbhQ                 PF11076.13          4     0.57    132   0.59   0.61   0.52   0.05\n",
            "21418  YbjM                 PF11045.13          7     0.89    117   0.59   0.59   0.53   0.06\n",
            "21419  YbjN                 PF10722.14         89    15.74    126   0.59   0.58   0.53   0.02\n",
            "21420  YbjO_DH-like         PF10767.14          7     0.70    141   0.59   0.59   0.53   0.05\n",
            "21421  YbjQ_1               PF01906.22         95     1.97    103   0.59   0.62   0.54   0.05\n",
            "21422  YBL028C_N            PF10338.14         70     7.62     44   1.25   1.31   1.23   0.23\n",
            "21423  YcaO                 PF02624.21        191     7.37    325   0.59   0.60   0.46   0.01\n",
            "21424  YcaO_C               PF18381.6          53     1.43    172   0.59   0.59   0.54   0.03\n",
            "21425  YcbB                 PF08664.15         25     1.59    136   0.59   0.59   0.52   0.04\n",
            "21426  YccF                 PF03733.18        346     7.96     51   1.09   1.03   1.05   0.24\n",
            "21427  YccJ                 PF13993.11          3     0.79     67   0.84   0.81   0.78   0.07\n",
            "21428  YccV-like            PF08755.16        121     2.80    101   0.59   0.53   0.49   0.03\n",
            "21429  YceD                 PF02620.22         68     6.51    119   0.59   0.58   0.49   0.04\n",
            "21430  YceG                 PF02618.21        662     3.41    275   0.59   0.59   0.45   0.02\n",
            "21431  YceG_bac             PF14266.11         44     4.34    250   0.59   0.57   0.49   0.03\n",
            "21432  YceI                 PF04264.18        626     6.55    150   0.59   0.60   0.37   0.03\n",
            "21433  YceM-like_C          PF21378.2          19     0.95    120   0.59   0.56   0.53   0.03\n",
            "21434  Ycf1                 PF05758.17          5     0.54    945   0.59   0.57   0.51   0.02\n",
            "21435  Ycf15                PF10705.14          3     0.44     86   0.66   0.61   0.59   0.03\n",
            "21436  Ycf34                PF10718.14         21     0.92     78   0.73   0.59   0.65   0.07\n",
            "21437  Ycf4                 PF02392.21         45     0.98    170   0.59   0.60   0.54   0.02\n",
            "21438  Ycf54                PF10674.14         32     1.10     92   0.62   0.59   0.56   0.04\n",
            "21439  Ycf66_N              PF07444.16         33     2.40     76   0.74   0.78   0.71   0.08\n",
            "21440  Ycf70                PF17382.7           2     0.41     89   0.64   0.62   0.57   0.03\n",
            "21441  Ycf9                 PF01737.22         30     2.32     58   0.96   1.01   0.93   0.16\n",
            "21442  YCF90                PF17088.10          2     0.39    332   0.59   0.56   0.52   0.04\n",
            "21443  YcgL                 PF05166.18         56     1.70     73   0.77   0.75   0.73   0.03\n",
            "21444  YchF-GTPase_C        PF06071.18        317     1.53     84   0.68   0.64   0.63   0.04\n",
            "21445  YchJ_M-like          PF17775.6         118     2.02     99   0.59   0.54   0.53   0.05\n",
            "21446  YchJ_N               PF21612.2           1     0.93     36   1.52   1.36   1.47   0.19\n",
            "21447  YCII                 PF03795.19         26     4.43     95   0.60   0.61   0.52   0.01\n",
            "21448  YcxB                 PF14317.11        106    86.92     61   0.92   0.88   0.87   0.05\n",
            "21449  YdaS_toxin           PF15943.10         13     2.78     65   0.86   0.88   0.82   0.03\n",
            "21450  YdaT_toxin           PF06254.16          2     0.70     92   0.62   0.64   0.55   0.02\n",
            "21451  Ydc2-catalyt         PF09159.15         47     3.56    258   0.59   0.58   0.48   0.03\n",
            "21452  YdfZ                 PF14001.11          7     1.47     64   0.88   0.89   0.84   0.03\n",
            "21453  YDG                  PF18657.6          42     6.63     85   0.67   0.70   0.56   0.08\n",
            "21454  YdgH_BhsA-like       PF07338.18         61     7.93     55   1.01   1.04   0.94   0.07\n",
            "21455  ydhR                 PF08803.16         27     1.64     96   0.60   0.58   0.55   0.02\n",
            "21456  YdiH                 PF15930.10          2     0.72     62   0.90   0.93   0.84   0.04\n",
            "21457  YdjC                 PF04794.17        138     6.06    266   0.59   0.58   0.39   0.02\n",
            "21458  YdjM                 PF04307.19         84     5.91    176   0.59   0.58   0.46   0.05\n",
            "21459  YdjO                 PF14169.11         16     1.79     59   0.95   0.77   0.91   0.11\n",
            "21460  YDR124W_N            PF21652.2           3     1.05     67   0.84   0.86   0.78   0.04\n",
            "21461  Ydr279_N             PF17745.6         110     6.62     79   0.72   0.71   0.56   0.02\n",
            "21462  Yeast-kill-tox       PF09207.16          2     0.39     87   0.65   0.55   0.58   0.06\n",
            "21463  Yeast_MT             PF11403.13          2     0.79     39   1.40   1.13   1.35   0.44\n",
            "21464  YEATS                PF03366.21        207     2.87     80   0.71   0.63   0.61   0.04\n",
            "21465  YebF                 PF13995.11          6     0.86     89   0.64   0.60   0.58   0.04\n",
            "21466  YebG                 PF07130.17         32     1.99     74   0.76   0.79   0.72   0.06\n",
            "21467  YebO                 PF13974.11         10     1.13     80   0.71   0.74   0.64   0.06\n",
            "21468  YebU_pre-PUA_dom     PF21150.2          15     0.84     77   0.73   0.69   0.68   0.04\n",
            "21469  YecM                 PF06185.17         28     1.44    180   0.59   0.58   0.53   0.03\n",
            "21470  YecR                 PF13992.11          8     1.54     73   0.77   0.68   0.72   0.08\n",
            "21471  YedD                 PF13987.11          5     0.57    106   0.59   0.58   0.52   0.02\n",
            "21472  YegS_C               PF19279.4          35     1.36    158   0.59   0.62   0.53   0.02\n",
            "21473  YejG                 PF13989.11          9     0.73    106   0.59   0.60   0.53   0.02\n",
            "21474  YesK                 PF14150.11          8     2.10     81   0.70   0.73   0.65   0.13\n",
            "21475  YetF_N               PF20730.2          23     2.28     76   0.74   0.79   0.72   0.12\n",
            "21476  YezG-like            PF04634.17         12     1.09    146   0.59   0.53   0.54   0.05\n",
            "21477  YfaZ                 PF07437.16          3     0.76    180   0.59   0.61   0.53   0.02\n",
            "21478  YfbK_C               PF12034.13         66     1.24    184   0.59   0.60   0.52   0.03\n",
            "21479  YfbR-like            PF12917.12         76     1.68    182   0.59   0.57   0.54   0.04\n",
            "21480  YfbU                 PF03887.19         16     1.42    166   0.59   0.55   0.51   0.04\n",
            "21481  YfcL                 PF08891.16         23     1.41     83   0.68   0.70   0.63   0.04\n",
            "21482  YfdX                 PF10938.13         28     4.21    149   0.59   0.65   0.53   0.06\n",
            "21483  YfeC-like            PF07037.16          5     0.72    122   0.59   0.60   0.53   0.02\n",
            "21484  YfhD                 PF14151.11         11     2.44     59   0.95   0.93   0.89   0.14\n",
            "21485  YfhE                 PF14152.11          9     4.06     36   1.51   1.51   1.49   0.24\n",
            "21486  YfhO                 PF09586.15         28     3.67    835   0.59   0.57   0.52   0.04\n",
            "21487  YfiO                 PF13525.11         11     1.72    203   0.59   0.57   0.54   0.05\n",
            "21488  YfjS_YafY            PF20494.3           2     0.48    113   0.59   0.56   0.52   0.02\n",
            "21489  YfkB                 PF08756.15         21     0.63    151   0.59   0.55   0.53   0.02\n",
            "21490  YfkD                 PF14167.11         12     0.74    232   0.59   0.58   0.53   0.02\n",
            "21491  YflT                 PF11181.13         62     6.87     99   0.59   0.63   0.53   0.03\n",
            "21492  YfmQ                 PF10787.14          4     0.53    133   0.59   0.58   0.51   0.03\n",
            "21493  YfzA                 PF14118.11          3     0.62     90   0.63   0.60   0.56   0.05\n",
            "21494  YgaB                 PF14182.11          5     0.97     77   0.74   0.73   0.69   0.09\n",
            "21495  YgaP-like_TM         PF11127.13         71     6.95     66   0.85   0.87   0.76   0.08\n",
            "21496  YgbA_NO              PF11756.13         40     0.81    106   0.59   0.50   0.52   0.06\n",
            "21497  YgbB                 PF02542.21        542     2.05    155   0.59   0.61   0.54   0.02\n",
            "21498  YgfZ_barrel          PF21130.2          36     1.65     68   0.83   0.83   0.79   0.03\n",
            "21499  YggL_50S_bp          PF04320.19         23     1.40    102   0.59   0.58   0.52   0.03\n",
            "21500  YGGT                 PF02325.22        593    10.57     70   0.80   0.82   0.65   0.13\n",
            "21501  YgjK_N               PF21152.2           7     0.79    254   0.59   0.57   0.51   0.02\n",
            "21502  YgjP-like            PF01863.22         56     3.95    207   0.59   0.55   0.54   0.04\n",
            "21503  YGR210-like_G4       PF08438.15        123     2.43    108   0.59   0.61   0.50   0.03\n",
            "21504  YgxA-like_sub_bind   PF22339.1          30     0.79    100   0.59   0.58   0.54   0.04\n",
            "21505  YhcG_C               PF06250.16         24     1.05    157   0.59   0.59   0.53   0.03\n",
            "21506  YhdB                 PF14148.11          9     0.91     71   0.79   0.74   0.74   0.08\n",
            "21507  YhdX                 PF17444.7           2     1.45     33   1.64   1.68   1.60   0.17\n",
            "21508  YhfC                 PF10086.14          8     1.46    222   0.59   0.61   0.53   0.05\n",
            "21509  YhfG                 PF10832.13         20     5.18     54   1.03   1.05   1.00   0.09\n",
            "21510  YhfH                 PF14149.11         20     3.24     37   1.47   1.24   1.45   0.27\n",
            "21511  YhfS-like_C          PF22475.1          15     0.75    101   0.59   0.62   0.54   0.02\n",
            "21512  YhfT                 PF10797.14         11     0.88    430   0.59   0.64   0.53   0.05\n",
            "21513  YhfX-like_C          PF21279.2          55     1.11     97   0.59   0.57   0.52   0.03\n",
            "21514  YhfZ_C               PF14503.11         13     1.59    236   0.59   0.60   0.54   0.02\n",
            "21515  YhhN                 PF07947.19         57     3.91    182   0.59   0.60   0.51   0.09\n",
            "21516  YHS                  PF04945.18         10     5.28     47   1.17   1.04   1.13   0.11\n",
            "21517  YHYH                 PF14240.11         13     1.46    193   0.59   0.55   0.46   0.03\n",
            "21518  YhzD                 PF14120.11         12     1.55     61   0.92   0.92   0.87   0.05\n",
            "21519  YiaAB                PF05360.19         99     7.90     53   1.05   1.05   1.01   0.15\n",
            "21520  YibE_F               PF07907.16        236     3.96    241   0.59   0.64   0.56   0.09\n",
            "21521  YibX                 PF22870.1           1     0.48     80   0.71   0.72   0.64   0.03\n",
            "21522  YicC-like_C          PF08340.16        313     1.50    120   0.59   0.60   0.54   0.07\n",
            "21523  YicC-like_N          PF03755.18        506     3.39    156   0.59   0.61   0.49   0.04\n",
            "21524  YidB                 PF20159.4           8     0.85    132   0.59   0.64   0.47   0.04\n",
            "21525  YidC_periplas        PF14849.11        125     4.33    283   0.59   0.58   0.48   0.02\n",
            "21526  YidD                 PF01809.23        559     2.18     67   0.84   0.74   0.78   0.06\n",
            "21527  YIEGIA               PF14045.11         33     1.20    282   0.59   0.61   0.54   0.02\n",
            "21528  YIEGIA_cap           PF21835.1          84     5.98     58   0.96   1.01   0.93   0.06\n",
            "21529  YIF1                 PF03878.20         99     1.80    245   0.59   0.56   0.51   0.04\n",
            "21530  YihI                 PF04220.17         24     1.34    155   0.59   0.62   0.51   0.08\n",
            "21531  YiiD_C               PF09500.15         19     1.47    144   0.59   0.58   0.54   0.01\n",
            "21532  YiiM_3-alpha         PF03475.19         73    28.40     44   1.25   1.27   1.24   0.08\n",
            "21533  Yip1                 PF04893.22        208    22.73    173   0.59   0.62   0.44   0.13\n",
            "21534  Yippee-Mis18         PF03226.20         43     3.16    108   0.59   0.52   0.52   0.06\n",
            "21535  YitT_membrane        PF02588.20         49     3.09    205   0.59   0.62   0.55   0.08\n",
            "21536  YjbE                 PF11106.13          3     0.93     79   0.72   0.82   0.65   0.15\n",
            "21537  YjbF                 PF11102.13         18     2.94    189   0.59   0.58   0.52   0.02\n",
            "21538  YjbH                 PF06082.16         33     1.98    658   0.59   0.58   0.52   0.01\n",
            "21539  YjbR                 PF04237.18        424     8.21     88   0.65   0.61   0.41   0.02\n",
            "21540  YjbT                 PF17089.10          1     0.41     92   0.62   0.62   0.55   0.01\n",
            "21541  YjcB                 PF15940.10          4     0.64     90   0.63   0.62   0.56   0.05\n",
            "21542  YjcQ                 PF09639.15          9     1.46     95   0.60   0.60   0.54   0.04\n",
            "21543  YjcZ                 PF13990.11          3     0.79    272   0.59   0.59   0.53   0.02\n",
            "21544  YjcZ_2               PF09680.15         23     7.36     24   2.22   2.31   2.21   0.60\n",
            "21545  YjdM                 PF03831.19        150     2.01     69   0.81   0.86   0.75   0.07\n",
            "21546  YjdM_Zn_Ribbon       PF08274.17         27     8.11     30   1.80   1.51   1.70   0.36\n",
            "21547  YjeF_N               PF03853.20         46     4.20    169   0.59   0.62   0.50   0.02\n",
            "21548  YjeJ                 PF15922.10          3     0.51    283   0.59   0.60   0.53   0.02\n",
            "21549  YjfB_motility        PF14070.11         73    10.27     57   0.98   1.01   0.90   0.13\n",
            "21550  YjgF_endoribonc      PF14588.11          1     0.39    148   0.59   0.60   0.51   0.01\n",
            "21551  YjhX_toxin           PF09857.14         17     0.98     85   0.67   0.67   0.62   0.03\n",
            "21552  YjjI-like            PF11230.13         25     1.01    485   0.59   0.59   0.54   0.01\n",
            "21553  YJL171C_Tos1_C       PF10287.15         97     1.45    228   0.59   0.54   0.50   0.03\n",
            "21554  YJL171C_Tos1_N       PF10290.14         92     2.04     63   0.89   0.85   0.83   0.02\n",
            "21555  YjzC                 PF14168.11         21     2.49     55   1.01   0.94   0.94   0.08\n",
            "21556  Ykof                 PF07615.16          5     1.21     81   0.70   0.67   0.65   0.02\n",
            "21557  YkoP                 PF22790.1         271     2.94    183   0.59   0.57   0.55   0.02\n",
            "21558  YkpC                 PF17447.7           4     1.61     42   1.31   1.38   1.27   0.10\n",
            "21559  YkuD                 PF03734.19        162    16.77    146   0.59   0.54   0.40   0.02\n",
            "21560  YkuD_2               PF13645.11         59     1.47    169   0.59   0.58   0.53   0.02\n",
            "21561  YkuI_C               PF10388.14          7     0.63    166   0.59   0.55   0.53   0.04\n",
            "21562  YkyA                 PF10368.14         94     4.34    186   0.59   0.59   0.56   0.12\n",
            "21563  YkyB                 PF14177.11        141     1.17    116   0.59   0.59   0.54   0.04\n",
            "21564  YL1                  PF05764.18        101     4.34    239   0.59   0.61   0.41   0.14\n",
            "21565  YL1_C                PF08265.16        173    27.06     29   1.85   1.76   1.85   0.15\n",
            "21566  YlaC                 PF10777.14          9     0.67    154   0.59   0.56   0.53   0.04\n",
            "21567  YlaH                 PF14036.11         17     1.41     76   0.74   0.77   0.69   0.15\n",
            "21568  YLATT                PF20303.3          32     1.61    107   0.59   0.62   0.53   0.04\n",
            "21569  YlbD_coat            PF14071.11         13     1.08    126   0.59   0.54   0.52   0.06\n",
            "21570  YlbE                 PF14003.11         14     1.36     61   0.92   0.82   0.88   0.08\n",
            "21571  YLDV-IL18BP-like     PF22009.1           7     1.02    104   0.59   0.55   0.52   0.04\n",
            "21572  YliH                 PF10799.13          2     0.45    126   0.59   0.60   0.52   0.03\n",
            "21573  YlmH_1st             PF21278.2           5     1.06     65   0.86   0.85   0.81   0.03\n",
            "21574  YlmH_RBD             PF17774.6          56     1.23     84   0.68   0.67   0.63   0.02\n",
            "21575  YLP                  PF02757.22         26    26.00      9   2.79   2.82   2.79   0.70\n",
            "21576  YlqD                 PF11068.13         57     2.33    131   0.59   0.61   0.54   0.08\n",
            "21577  YlxR                 PF04296.18        477     4.44     75   0.75   0.75   0.69   0.04\n",
            "21578  YlzJ                 PF14035.11         45     4.45     65   0.86   0.85   0.82   0.03\n",
            "21579  YmaF                 PF12788.12         16     1.01     97   0.59   0.50   0.52   0.11\n",
            "21580  YmcE_antitoxin       PF15939.10          1     0.47     76   0.75   0.71   0.68   0.04\n",
            "21581  YmdB                 PF13277.11        261     1.79    253   0.59   0.60   0.53   0.02\n",
            "21582  Yme1-like_N          PF21232.2          15     1.26     61   0.92   0.91   0.87   0.03\n",
            "21583  YMF19                PF02326.20         23     3.03     84   0.68   0.63   0.59   0.07\n",
            "21584  YMF19_C              PF06449.16          2     0.89     51   1.08   1.07   1.02   0.08\n",
            "21585  YmgB                 PF10798.13          8     1.92     59   0.95   1.00   0.92   0.10\n",
            "21586  YmgD                 PF16456.10          2     0.62     82   0.69   0.71   0.63   0.03\n",
            "21587  YmiA-like            PF22868.1           1     0.75     46   1.20   1.15   1.15   0.10\n",
            "21588  YmzC                 PF14157.11          2     1.14     62   0.90   0.85   0.84   0.08\n",
            "21589  YNCE                 PF21783.2           1     0.39    339   0.58   0.59   0.51   0.01\n",
            "21590  YndJ                 PF14158.11         16     2.06    258   0.59   0.62   0.53   0.07\n",
            "21591  YnfE                 PF17452.7           4     0.90     78   0.72   0.71   0.67   0.06\n",
            "21592  YniB                 PF14002.11         10     0.81    166   0.59   0.59   0.53   0.04\n",
            "21593  YNR034W-A-like       PF11503.13         13     1.95     72   0.78   0.81   0.73   0.04\n",
            "21594  YoaL                 PF22867.1           1     0.70     52   1.07   1.00   1.01   0.09\n",
            "21595  YoaP                 PF14268.11         33     6.42     44   1.25   1.21   1.22   0.05\n",
            "21596  YobH                 PF13996.11          4     0.61     70   0.80   0.82   0.74   0.06\n",
            "21597  YobI-ATPase          PF20693.2          47     1.60    363   0.59   0.59   0.51   0.04\n",
            "21598  YodL                 PF14191.11         25     1.25     98   0.59   0.55   0.51   0.04\n",
            "21599  YoeB_toxin           PF06769.19         13     0.90     80   0.71   0.65   0.66   0.03\n",
            "21600  YokU                 PF14122.11          8     0.81     87   0.65   0.59   0.60   0.05\n",
            "21601  YolD                 PF08863.15         35     3.88     94   0.61   0.58   0.58   0.06\n",
            "21602  YonK                 PF09642.15          4     1.64     60   0.93   0.94   0.87   0.07\n",
            "21603  Yop-YscD_cpl         PF16697.10         17     2.45     94   0.61   0.64   0.56   0.02\n",
            "21604  Yop-YscD_ppl_1st     PF16693.10         18     8.69     62   0.90   0.94   0.88   0.07\n",
            "21605  Yop-YscD_ppl_2nd     PF21937.1          19     4.06     67   0.84   0.87   0.81   0.04\n",
            "21606  Yop-YscD_ppl_3rd     PF21934.1          75    13.22     64   0.88   0.92   0.87   0.06\n",
            "21607  YopD                 PF05844.17          3     0.72    297   0.59   0.62   0.53   0.04\n",
            "21608  YopE                 PF03545.18          4     1.18     70   0.81   0.83   0.76   0.04\n",
            "21609  YopE_N               PF09020.15          1     0.43    126   0.59   0.61   0.52   0.03\n",
            "21610  YopH_N               PF09013.15          2     0.64    123   0.59   0.63   0.51   0.04\n",
            "21611  Yopt                 PF09467.15          1     0.57     71   0.79   0.83   0.73   0.04\n",
            "21612  YopX                 PF09643.15         14     2.57    128   0.59   0.56   0.50   0.03\n",
            "21613  YoqO                 PF14037.11          5     1.13    116   0.59   0.60   0.53   0.10\n",
            "21614  YorP                 PF09629.15          1     0.50     71   0.79   0.75   0.73   0.03\n",
            "21615  Yos1                 PF08571.15         78     1.71     81   0.70   0.75   0.62   0.06\n",
            "21616  Yos9_DD              PF17880.6           9     1.19    128   0.59   0.57   0.53   0.03\n",
            "21617  YozD                 PF14162.11         18     1.14     56   1.00   1.00   0.95   0.08\n",
            "21618  YozE_SAM_like        PF06855.17         46     3.89     66   0.85   0.76   0.80   0.08\n",
            "21619  YPEB_N               PF20769.2          38     2.71    136   0.59   0.60   0.57   0.05\n",
            "21620  YPEB_PepSY1-2        PF14620.11         39     2.40    195   0.59   0.59   0.54   0.03\n",
            "21621  YPEH2ZP              PF14976.12          7     0.56    145   0.59   0.53   0.52   0.03\n",
            "21622  YpjP                 PF14005.11         15     1.04    134   0.59   0.54   0.54   0.03\n",
            "21623  YpM                  PF09144.15          2     0.38    117   0.59   0.58   0.52   0.02\n",
            "21624  YpmT                 PF17431.7          10     1.34     60   0.93   0.88   0.87   0.11\n",
            "21625  YpoC                 PF21747.2         139     3.00    111   0.59   0.56   0.55   0.05\n",
            "21626  YppF                 PF14178.11          8     1.89     59   0.95   0.92   0.91   0.05\n",
            "21627  YppG                 PF14179.11         10     1.39    105   0.59   0.53   0.46   0.06\n",
            "21628  YpsA                 PF06908.16         51     1.79    168   0.59   0.56   0.54   0.03\n",
            "21629  YpzG                 PF14139.11          9     2.77     49   1.13   1.05   1.07   0.10\n",
            "21630  YpzI                 PF14140.11          7     1.64     42   1.31   1.33   1.27   0.17\n",
            "21631  YqaH                 PF17448.7           3     0.77     88   0.65   0.67   0.58   0.03\n",
            "21632  Yqai                 PF09466.15          3     1.38     66   0.85   0.82   0.76   0.05\n",
            "21633  YqaJ                 PF09588.15         16     3.28    150   0.59   0.56   0.48   0.02\n",
            "21634  YqbF                 PF14553.11          7     5.15     43   1.28   1.26   1.26   0.09\n",
            "21635  YqbF_HeH             PF21488.2           8     3.16     39   1.40   1.48   1.37   0.15\n",
            "21636  YqcI_YcgG            PF08892.16         49     2.18    211   0.59   0.53   0.53   0.02\n",
            "21637  YqeC                 PF19842.4         131     2.89    145   0.59   0.61   0.50   0.02\n",
            "21638  YqeH-like_C          PF21516.2          23     0.82     95   0.60   0.59   0.54   0.01\n",
            "21639  YqeY                 PF09424.15         36     1.44    143   0.59   0.62   0.54   0.05\n",
            "21640  YqfD                 PF06898.16        100     4.64    379   0.59   0.60   0.55   0.03\n",
            "21641  YqfQ                 PF14181.11          5     1.09    166   0.59   0.62   0.49   0.03\n",
            "21642  YqgB                 PF11036.13          2     1.16     43   1.28   1.27   1.23   0.07\n",
            "21643  YqgC                 PF17430.7           2     0.82     51   1.09   1.04   1.04   0.07\n",
            "21644  YqgF                 PF14639.11          4     0.84    150   0.59   0.61   0.52   0.02\n",
            "21645  YqgQ-like            PF06014.16         32     2.47     61   0.92   0.90   0.89   0.09\n",
            "21646  YqgU                 PF21101.2          12     1.43    264   0.59   0.55   0.53   0.03\n",
            "21647  YqhG                 PF11079.13         15     0.78    258   0.59   0.56   0.53   0.02\n",
            "21648  YqhR                 PF11085.13          9     0.65    165   0.59   0.57   0.53   0.04\n",
            "21649  YqiJ_N               PF21001.2          25     1.57    116   0.59   0.64   0.52   0.07\n",
            "21650  YqiJ_OB              PF07290.16         25     1.79     64   0.87   0.90   0.84   0.03\n",
            "21651  YqjK                 PF13997.11         20     4.32     73   0.77   0.77   0.75   0.09\n",
            "21652  YqzE                 PF14038.11         17     2.39     54   1.03   0.97   0.99   0.07\n",
            "21653  YqzH                 PF14164.11          5     1.76     64   0.88   0.83   0.83   0.05\n",
            "21654  YqzL                 PF14006.11         13     3.89     44   1.25   1.20   1.21   0.09\n",
            "21655  YqzM                 PF14141.11         12     2.26     42   1.30   1.30   1.27   0.19\n",
            "21656  YrbL-PhoP_reg        PF10707.14          9     1.45    185   0.59   0.59   0.53   0.02\n",
            "21657  YrhC                 PF14143.11         12     2.28     72   0.78   0.80   0.75   0.06\n",
            "21658  YrhK                 PF14145.11         37     5.82     58   0.96   0.95   0.87   0.13\n",
            "21659  YrpD                 PF15493.12          4     0.76    214   0.59   0.58   0.50   0.03\n",
            "21660  YrvL                 PF14184.11         12     2.36    127   0.59   0.61   0.55   0.12\n",
            "21661  YrzK                 PF17449.7           3     1.30     54   1.03   0.99   0.99   0.08\n",
            "21662  YrzO                 PF14142.11          2     1.06     46   1.19   1.23   1.15   0.09\n",
            "21663  YsaB                 PF13983.11          2     0.66     75   0.75   0.71   0.69   0.04\n",
            "21664  Ysc84                PF04366.17        236     4.25    127   0.59   0.62   0.51   0.02\n",
            "21665  YscJ_FliF            PF01514.22        285     3.69    179   0.59   0.63   0.54   0.04\n",
            "21666  YscJ_FliF_C          PF08345.16        309     4.41    147   0.59   0.62   0.44   0.06\n",
            "21667  YscK                 PF06578.17          5     1.03    209   0.59   0.57   0.51   0.03\n",
            "21668  YscO                 PF07321.17         11     3.03    149   0.59   0.61   0.56   0.12\n",
            "21669  YscO-like            PF16789.10          5     1.06    161   0.59   0.60   0.54   0.10\n",
            "21670  YscW                 PF09619.15         32     2.79    107   0.59   0.62   0.53   0.03\n",
            "21671  YSIRK_signal         PF04650.22         34    27.40     26   2.06   2.14   2.05   0.24\n",
            "21672  Ytca                 PF17090.10          6     1.21     62   0.90   0.86   0.84   0.15\n",
            "21673  YtfJ_HI0045          PF09695.15         10     0.71    160   0.59   0.60   0.53   0.03\n",
            "21674  YTH                  PF04146.20        195     2.82    135   0.59   0.54   0.50   0.03\n",
            "21675  YtkA                 PF13115.11         42     7.48     85   0.67   0.64   0.58   0.04\n",
            "21676  Ytp1                 PF10355.14        108     1.41    276   0.59   0.55   0.50   0.04\n",
            "21677  YtpI                 PF14007.11         20     2.00     87   0.65   0.66   0.62   0.08\n",
            "21678  YTV                  PF07639.16         26     7.10     43   1.28   1.15   1.24   0.35\n",
            "21679  YtxC                 PF08812.16         62     2.98    218   0.59   0.58   0.54   0.05\n",
            "21680  YtxH                 PF12732.12        134    10.88     71   0.79   0.89   0.71   0.06\n",
            "21681  YtxK_like            PF21106.2          34     1.45     81   0.70   0.73   0.65   0.05\n",
            "21682  YtzH                 PF14165.11         14     1.36     86   0.66   0.64   0.61   0.06\n",
            "21683  YuA_Gp49             PF20788.2           5     1.18     72   0.78   0.73   0.71   0.03\n",
            "21684  YueH                 PF14166.11          7     1.71     79   0.72   0.71   0.66   0.05\n",
            "21685  YugN                 PF08868.15         16     0.92    130   0.59   0.57   0.53   0.02\n",
            "21686  YuiB                 PF14068.11         17     0.72    101   0.59   0.60   0.53   0.08\n",
            "21687  YukC                 PF10140.14         15     2.00    357   0.59   0.58   0.55   0.04\n",
            "21688  YukD                 PF08817.15        103     8.45     76   0.74   0.81   0.67   0.03\n",
            "21689  Yuri_gagarin         PF15934.10          3     0.58    234   0.59   0.58   0.52   0.05\n",
            "21690  YusW                 PF14039.11         16     2.32     92   0.62   0.63   0.56   0.07\n",
            "21691  YutD-like            PF06265.16         40     0.98     85   0.67   0.62   0.61   0.05\n",
            "21692  YuzL                 PF14115.11          8     2.91     41   1.34   1.39   1.29   0.23\n",
            "21693  YvbH_ext             PF11724.13         11     2.39     61   0.92   0.86   0.88   0.09\n",
            "21694  YvfG                 PF09628.15          1     0.59     67   0.84   0.82   0.78   0.05\n",
            "21695  YvrJ                 PF12841.12         41     8.27     36   1.51   1.61   1.50   0.09\n",
            "21696  YwcE                 PF17368.7           4     0.66     85   0.67   0.65   0.61   0.09\n",
            "21697  YWFCY                PF14293.11         33     1.79    144   0.59   0.60   0.55   0.06\n",
            "21698  YwhD                 PF08741.15         18     0.73    162   0.59   0.57   0.53   0.03\n",
            "21699  YwiC                 PF14256.11         55     3.56    126   0.59   0.59   0.54   0.09\n",
            "21700  YwpF                 PF14183.11         16     1.11    133   0.59   0.60   0.53   0.04\n",
            "21701  YwqJ-deaminase       PF14431.11          7     2.72    138   0.59   0.58   0.49   0.03\n",
            "21702  YxiJ                 PF14176.11          4     0.76    109   0.59   0.54   0.52   0.04\n",
            "21703  YycC                 PF14174.11          3     1.39     51   1.09   1.08   1.04   0.10\n",
            "21704  YycE-like_C          PF22659.1          44     1.67     55   1.01   0.96   0.97   0.04\n",
            "21705  YycE-like_N          PF22658.1          65     1.92     52   1.07   1.05   1.04   0.04\n",
            "21706  YycH                 PF07435.16          5     1.39    438   0.59   0.57   0.52   0.03\n",
            "21707  YycI                 PF09648.15         21     3.84    232   0.59   0.58   0.50   0.05\n",
            "21708  YyzF                 PF14116.11         19     2.93     48   1.15   1.04   1.09   0.15\n",
            "21709  Y_phosphatase        PF00102.32         89     3.37    234   0.59   0.52   0.50   0.03\n",
            "21710  Y_phosphatase2       PF03162.18          8     1.57    165   0.59   0.56   0.53   0.02\n",
            "21711  Y_phosphatase3       PF13350.11        176     3.75    240   0.59   0.60   0.47   0.02\n",
            "21712  Y_Y_Y                PF07495.18         74    17.05     65   0.86   0.81   0.81   0.07\n",
            "21713  z-alpha              PF02295.22          5     2.08     67   0.84   0.83   0.79   0.03\n",
            "21714  Z1                   PF10593.14         48     1.94    228   0.59   0.57   0.50   0.03\n",
            "21715  Zap1_zf1             PF21816.2          33     9.32     27   1.98   1.71   1.98   0.32\n",
            "21716  Zap1_zf2             PF18217.6          14     3.97     24   2.22   1.85   2.20   0.43\n",
            "21717  ZapA                 PF05164.18        320    11.04     88   0.65   0.69   0.57   0.06\n",
            "21718  ZapB                 PF06005.17         28     4.38     71   0.79   0.85   0.74   0.16\n",
            "21719  ZapC_C               PF07126.17         25     2.13     81   0.70   0.67   0.65   0.03\n",
            "21720  ZapC_N               PF21083.2          25     1.82     88   0.65   0.60   0.60   0.04\n",
            "21721  ZapD                 PF07072.16         82     2.08    210   0.59   0.59   0.55   0.03\n",
            "21722  ZapG-like            PF06295.17         20     2.28    123   0.59   0.60   0.52   0.04\n",
            "21723  Zds_C                PF08632.15         11     1.32     52   1.07   1.03   1.03   0.08\n",
            "21724  Zea_mays_MuDR        PF05928.16          4     0.40    207   0.59   0.58   0.52   0.02\n",
            "21725  Zein                 PF01559.21          4     0.68    246   0.59   0.60   0.50   0.07\n",
            "21726  Zein-binding         PF04576.20         57     1.73     92   0.62   0.64   0.59   0.10\n",
            "21727  Zemlya               PF17646.6          11     1.85    106   0.59   0.62   0.54   0.05\n",
            "21728  ZER1-like_2nd        PF22964.1          32     1.43    356   0.59   0.57   0.54   0.03\n",
            "21729  Zeta_toxin           PF06414.17         13     3.05    199   0.59   0.60   0.51   0.02\n",
            "21730  zf-3CxxC             PF13695.11        111     3.00     97   0.59   0.46   0.44   0.09\n",
            "21731  zf-3CxxC_2           PF17180.9           3     0.81     74   0.76   0.62   0.70   0.09\n",
            "21732  zf-4CXXC_R1          PF10497.14         44     0.86     99   0.59   0.45   0.51   0.10\n",
            "21733  zf-A20               PF01754.21        106    56.05     24   2.22   1.86   2.18   0.54\n",
            "21734  zf-ACC               PF17848.6          26    19.29     26   2.06   1.70   2.06   0.43\n",
            "21735  zf-AD                PF07776.20         42     7.96     75   0.75   0.64   0.66   0.11\n",
            "21736  zf-AN1               PF01428.21        380     6.80     40   1.37   1.03   1.31   0.42\n",
            "21737  zf-ANAPC11           PF12861.12          5     0.69     65   0.86   0.69   0.80   0.10\n",
            "21738  zf-BED               PF02892.20         17     7.65     44   1.25   1.04   1.17   0.18\n",
            "21739  zf-B_box             PF00643.29         85    13.61     42   1.31   1.00   1.22   0.35\n",
            "21740  zf-C2H2              PF00096.31        159   159.00     23   1.62   1.29   1.56   0.39\n",
            "21741  zf-C2H2_10           PF16588.10         22     5.17     23   2.31   2.05   2.29   0.45\n",
            "21742  zf-C2H2_11           PF16622.10         10     3.25     29   1.85   1.65   1.83   0.30\n",
            "21743  zf-C2H2_12           PF18658.6          23     2.73     64   0.88   0.74   0.84   0.07\n",
            "21744  zf-C2H2_2            PF12756.12         86     2.12    100   0.59   0.48   0.51   0.08\n",
            "21745  zf-C2H2_3            PF13878.11        116    11.55     40   1.37   1.18   1.32   0.17\n",
            "21746  zf-C2H2_3rep         PF18868.6          10     0.57    126   0.59   0.52   0.52   0.04\n",
            "21747  zf-C2H2_4            PF13894.11        304   304.00     24   1.26   0.95   1.22   0.33\n",
            "21748  zf-C2H2_5            PF21225.2          18    11.83     22   2.41   2.20   2.41   0.43\n",
            "21749  zf-C2H2_6            PF13912.11         25    25.00     27   1.80   1.56   1.79   0.28\n",
            "21750  zf-C2H2_7            PF15269.11          1     0.61     54   1.02   0.89   0.96   0.10\n",
            "21751  zf-C2H2_8            PF15909.10          1     0.37     98   0.59   0.53   0.51   0.04\n",
            "21752  zf-C2H2_9            PF16293.10          8     1.22     59   0.94   0.84   0.88   0.06\n",
            "21753  zf-C2H2_aberr        PF17017.10          5     0.84    172   0.59   0.51   0.51   0.06\n",
            "21754  zf-C2H2_assoc        PF16606.10          5     0.64     81   0.70   0.74   0.64   0.09\n",
            "21755  zf-C2H2_assoc2       PF16624.10          5     0.97     95   0.60   0.62   0.51   0.09\n",
            "21756  zf-C2H2_assoc3       PF16637.10          6     0.91     85   0.67   0.64   0.59   0.07\n",
            "21757  zf-C2H2_jaz          PF12171.13        108   108.00     27   1.82   1.53   1.82   0.36\n",
            "21758  zf-C2HC              PF01530.23         48     7.45     29   1.85   1.62   1.82   0.36\n",
            "21759  zf-C2HC5             PF06221.18         59     1.97     54   1.03   0.84   0.98   0.23\n",
            "21760  zf-C2HCIx2C          PF10782.14         11     1.39     58   0.96   0.86   0.92   0.10\n",
            "21761  zf-C2HC_2            PF13913.11         88    88.00     25   1.92   1.67   1.91   0.33\n",
            "21762  zf-C2HE              PF16278.10         91     6.97     63   0.89   0.81   0.82   0.08\n",
            "21763  zf-C3H1              PF10650.14         37    17.63     22   2.41   2.09   2.38   0.42\n",
            "21764  zf-C3H2C3            PF17122.10          3     1.75     35   1.55   1.25   1.51   0.35\n",
            "21765  zf-C3HC              PF07967.18         21     2.32    132   0.59   0.51   0.50   0.05\n",
            "21766  zf-C3Hc3H            PF13891.11         71     2.42     62   0.90   0.70   0.80   0.18\n",
            "21767  zf-C3HC4             PF00097.30         26     4.66     40   1.37   1.01   1.28   0.46\n",
            "21768  zf-C3HC4_2           PF13923.11          9     2.53     40   1.37   1.05   1.31   0.37\n",
            "21769  zf-C3HC4_3           PF13920.11        311     5.79     49   1.13   0.84   1.06   0.35\n",
            "21770  zf-C3HC4_4           PF15227.11          7     1.84     42   1.31   1.02   1.26   0.34\n",
            "21771  zf-C3HC4_5           PF17121.10          6     0.88     51   1.09   0.90   1.04   0.15\n",
            "21772  zf-C4                PF00105.23         26     0.89     70   0.80   0.65   0.75   0.16\n",
            "21773  zf-C4H2              PF10146.14         23     1.38    240   0.59   0.56   0.50   0.07\n",
            "21774  zf-C4pol             PF14260.11        204     3.19     70   0.80   0.62   0.66   0.21\n",
            "21775  zf-C4_ClpX           PF06689.18        264     9.39     39   1.40   1.22   1.38   0.25\n",
            "21776  zf-C4_Topoisom       PF01396.24         22     5.13     39   1.40   1.19   1.35   0.22\n",
            "21777  zf-C5HC2             PF02928.21        176     6.41     54   1.03   0.83   0.95   0.22\n",
            "21778  zf-C6H2              PF15801.10         98     1.93     47   1.17   0.90   1.11   0.26\n",
            "21779  zf-CCCH              PF00642.29         69    69.00     27   1.82   1.50   1.82   0.38\n",
            "21780  zf-CCCH_10           PF22628.1          19     2.88     39   1.40   1.20   1.37   0.18\n",
            "21781  zf-CCCH_2            PF14608.11        144   144.00     19   2.25   1.80   2.16   0.67\n",
            "21782  zf-CCCH_2-like_3     PF21457.2           4     1.54     29   1.85   1.63   1.82   0.27\n",
            "21783  zf-CCCH_3            PF15663.10          9     0.57    110   0.59   0.48   0.52   0.05\n",
            "21784  zf-CCCH_4            PF18044.6          15    15.00     22   2.28   1.87   2.23   0.50\n",
            "21785  zf-CCCH_6            PF18585.6          39     2.16     53   1.05   0.97   1.02   0.08\n",
            "21786  zf-CCCH_7            PF18586.6           3     0.92     90   0.63   0.60   0.57   0.03\n",
            "21787  zf-CCCH_8            PF18633.6           5     2.00     28   1.91   1.70   1.89   0.29\n",
            "21788  zf-CCCH_9            PF22623.1           2     1.05     38   1.44   1.32   1.39   0.13\n",
            "21789  zf-CCHC              PF00098.28         79    79.00     18   2.27   1.85   2.27   0.58\n",
            "21790  zf-CCHC_2            PF13696.11         44    27.01     21   2.52   2.07   2.50   0.55\n",
            "21791  zf-CCHC_3            PF13917.11         19     4.78     41   1.33   1.15   1.25   0.21\n",
            "21792  zf-CCHC_4            PF14392.11          9     2.87     49   1.13   1.00   1.10   0.09\n",
            "21793  zf-CCHC_5            PF14787.11          5     2.47     36   1.51   1.35   1.48   0.22\n",
            "21794  zf-CCHC_6            PF15288.11         15     2.88     42   1.31   1.15   1.28   0.17\n",
            "21795  zf-CCHH              PF10283.14         55    14.13     26   2.06   1.72   2.04   0.43\n",
            "21796  zf-CDGSH             PF09360.15        155     5.89     48   1.15   1.04   1.04   0.10\n",
            "21797  zf-CGNR              PF11706.13        196     5.15     44   1.25   1.06   1.21   0.27\n",
            "21798  zf-CHC2              PF01807.25         16     0.93     98   0.59   0.54   0.54   0.03\n",
            "21799  zf-CHCC              PF10276.14         91     5.82     37   1.47   1.27   1.41   0.18\n",
            "21800  zf-CHY               PF05495.17        227     2.05     68   0.83   0.58   0.70   0.23\n",
            "21801  zf-CRD               PF17979.6          39     2.02    158   0.59   0.47   0.48   0.08\n",
            "21802  zf-CSL               PF05207.18         55     4.47     55   1.01   0.88   0.92   0.13\n",
            "21803  zf-CW                PF07496.20        134     5.67     47   1.17   0.92   1.04   0.24\n",
            "21804  zf-CXXC              PF02008.25         30     2.49     48   1.15   0.87   1.10   0.44\n",
            "21805  zf-DBF               PF07535.17         91     3.99     45   1.22   1.06   1.20   0.15\n",
            "21806  zf-Di19              PF05605.17         42     2.72     54   1.03   0.83   0.96   0.17\n",
            "21807  zf-DNA_Pol           PF08996.15        119     2.49    186   0.59   0.51   0.51   0.06\n",
            "21808  zf-DNL               PF05180.17         95     1.70     64   0.88   0.76   0.82   0.09\n",
            "21809  zf-Dof               PF02701.20         43     1.04     59   0.95   0.78   0.89   0.12\n",
            "21810  zf-DPOE              PF22912.1         256     2.67     65   0.86   0.75   0.83   0.10\n",
            "21811  zf-dskA_traR         PF01258.22        112    76.35     36   1.51   1.36   1.47   0.26\n",
            "21812  zf-FCS               PF06467.19         36    27.86     40   1.37   1.10   1.31   0.29\n",
            "21813  zf-FCS_1             PF21319.2           5     2.03     31   1.74   1.51   1.70   0.36\n",
            "21814  zf-FLZ               PF04570.19         48     2.43     54   1.03   0.88   1.00   0.14\n",
            "21815  zf-FPG_IleRS         PF06827.19         69    69.00     30   1.72   1.43   1.71   0.35\n",
            "21816  zf-GRF               PF06839.17         12     2.47     45   1.22   0.99   1.15   0.19\n",
            "21817  zf-H2C2              PF09337.15         22     7.98     39   1.40   1.27   1.36   0.17\n",
            "21818  zf-H2C2_2            PF13465.11         80    80.00     26   1.43   1.16   1.39   0.28\n",
            "21819  zf-H2C2_5            PF13909.11          6     5.52     26   2.06   1.76   2.03   0.36\n",
            "21820  zf-H3C2              PF16721.10          1     0.39    102   0.59   0.55   0.51   0.02\n",
            "21821  zf-HC2               PF13490.11        498   498.00     35   1.51   1.41   1.49   0.24\n",
            "21822  zf-HC3               PF16827.10         10     1.54     67   0.84   0.70   0.76   0.08\n",
            "21823  zf-HC5HC2H           PF13771.11         78     2.02     91   0.63   0.46   0.54   0.14\n",
            "21824  zf-HC5HC2H_2         PF13832.11         24     1.25    109   0.59   0.45   0.50   0.10\n",
            "21825  ZF-HD_dimer          PF04770.17         50     1.45     55   1.01   0.85   0.96   0.14\n",
            "21826  zf-His_Me_endon      PF05551.16         11     1.96    131   0.59   0.49   0.48   0.07\n",
            "21827  zf-HIT               PF04438.21         84    14.82     30   1.80   1.42   1.75   0.57\n",
            "21828  zf-HYPF              PF07503.17        425     9.22     34   1.59   1.33   1.58   0.27\n",
            "21829  zf-IS66              PF13005.12        125    15.96     46   1.20   1.02   1.14   0.22\n",
            "21830  zf-ISL3              PF14690.11         50    14.93     47   1.17   0.97   1.14   0.20\n",
            "21831  zf-like              PF04071.17         22     0.98     82   0.69   0.52   0.61   0.10\n",
            "21832  zf-LITAF-like        PF10601.14        280     5.29     70   0.80   0.62   0.74   0.16\n",
            "21833  zf-LSD1              PF06943.17         71    14.91     25   2.13   1.85   2.11   0.51\n",
            "21834  zf-LYAR              PF08790.16         74     6.74     28   1.92   1.64   1.90   0.36\n",
            "21835  zf-met               PF12874.12        254   254.00     25   1.82   1.53   1.82   0.33\n",
            "21836  zf-met2              PF12907.12          8     2.91     38   1.43   1.24   1.40   0.20\n",
            "21837  zf-MIZ               PF02891.25         28     2.09     50   1.11   0.87   1.07   0.18\n",
            "21838  zf-Mss51             PF13824.11         73     2.86     55   1.01   0.84   0.92   0.13\n",
            "21839  zf-MYND              PF01753.23        120     7.12     39   1.40   1.02   1.33   0.48\n",
            "21840  zf-MYST              PF17772.6          30     2.94     55   1.01   0.84   0.98   0.10\n",
            "21841  zf-NADH-PPase        PF09297.16         43    36.06     32   1.69   1.43   1.68   0.30\n",
            "21842  zf-nanos             PF05741.18         57     2.74     54   1.03   0.82   0.97   0.18\n",
            "21843  zf-NF-X1             PF01422.22         96    12.81     19   2.77   2.18   2.74   1.13\n",
            "21844  zf-NOSIP             PF15906.10          1     0.48     75   0.75   0.72   0.69   0.03\n",
            "21845  zf-NPL4              PF05020.20         69     0.86    145   0.59   0.52   0.53   0.03\n",
            "21846  zf-Nse               PF11789.13         19     1.97     57   0.98   0.86   0.90   0.09\n",
            "21847  Zf-Nsp1alpha         PF21905.1           2     1.07     42   1.31   1.26   1.26   0.10\n",
            "21848  zf-P11               PF03854.19          3     0.94     50   1.11   0.93   1.06   0.17\n",
            "21849  zf-Paramyx-P         PF13008.12          6     1.81     45   1.22   0.96   1.19   0.27\n",
            "21850  zf-PARP              PF00645.23        172     3.66     87   0.65   0.58   0.56   0.07\n",
            "21851  zf-PHD-like          PF15446.11         45     0.81    170   0.59   0.49   0.52   0.07\n",
            "21852  zf-piccolo           PF05715.18         12     0.89     60   0.93   0.76   0.87   0.14\n",
            "21853  zf-primase           PF09329.16        114     5.68     46   1.20   1.04   1.16   0.16\n",
            "21854  zf-RAG1              PF10426.14          5     2.71     30   1.80   1.63   1.76   0.24\n",
            "21855  zf-RanBP             PF00641.23         24    22.38     30   1.80   1.47   1.78   0.41\n",
            "21856  zf-rbx1              PF12678.12         63     2.36     55   1.01   0.76   0.93   0.20\n",
            "21857  zf-ribbon_3          PF13248.11         20    20.00     26   1.71   1.44   1.67   0.39\n",
            "21858  zf-RING-like         PF08746.16         99     5.07     43   1.28   0.96   1.23   0.37\n",
            "21859  zf-RING_10           PF16685.10         15     0.93     70   0.81   0.66   0.75   0.14\n",
            "21860  zf-RING_11           PF17123.10        109    10.42     29   1.85   1.51   1.83   0.43\n",
            "21861  zf-RING_12           PF17976.6           5     0.68     73   0.78   0.65   0.72   0.07\n",
            "21862  zf-RING_13           PF17977.6           1     0.48     68   0.82   0.71   0.76   0.09\n",
            "21863  zf-RING_14           PF17978.6          15     0.64     91   0.63   0.53   0.57   0.06\n",
            "21864  zf-RING_15           PF16744.10          3     0.54     73   0.77   0.68   0.71   0.06\n",
            "21865  zf-RING_16           PF17120.10          9     1.52     57   0.98   0.74   0.92   0.22\n",
            "21866  zf-RING_2            PF13639.11        160     8.95     44   1.25   0.93   1.13   0.37\n",
            "21867  zf-RING_4            PF14570.11         44     2.30     48   1.15   0.89   1.09   0.26\n",
            "21868  zf-RING_5            PF14634.11         71     7.93     43   1.28   0.94   1.17   0.43\n",
            "21869  zf-RING_6            PF14835.11          1     0.50     65   0.87   0.78   0.80   0.07\n",
            "21870  zf-RING_7            PF02591.20        161    44.88     33   1.64   1.46   1.60   0.24\n",
            "21871  zf-RING_UBOX         PF13445.11        411    28.43     39   1.40   1.12   1.21   0.32\n",
            "21872  zf-RNPHF             PF08080.17          3     1.30     36   1.51   1.39   1.47   0.21\n",
            "21873  zf-RRN7              PF11781.13         67    67.00     32   1.66   1.39   1.61   0.31\n",
            "21874  zf-RRPl_C4           PF17026.10         10     1.08    108   0.59   0.46   0.52   0.12\n",
            "21875  zf-RVT               PF13966.11         83     3.32     85   0.67   0.52   0.60   0.08\n",
            "21876  zf-SAP30             PF13866.11         10     0.83     72   0.78   0.72   0.72   0.09\n",
            "21877  zf-SCNM1             PF15803.10         15     4.42     27   1.98   1.81   1.97   0.26\n",
            "21878  zf-Sec23_Sec24       PF04810.20        288    10.08     39   1.40   1.11   1.35   0.26\n",
            "21879  zf-TAZ               PF02135.21        136     1.95     72   0.78   0.57   0.62   0.24\n",
            "21880  zf-tcix              PF14952.11          9     2.75     42   1.31   1.15   1.28   0.24\n",
            "21881  zf-TFIIB             PF13453.11         48     9.02     41   1.34   1.17   1.30   0.17\n",
            "21882  zf-TFIIIC            PF12660.12         19     1.62     98   0.59   0.51   0.48   0.06\n",
            "21883  zf-Tim10_DDP         PF02953.20        399     7.63     64   0.88   0.76   0.84   0.13\n",
            "21884  zf-TRAF              PF02176.23          9     1.87     60   0.93   0.73   0.87   0.19\n",
            "21885  zf-TRAF_2            PF15965.10         20     0.68     93   0.62   0.47   0.55   0.12\n",
            "21886  zf-trcl              PF13451.11         55     3.04     48   1.15   0.96   1.11   0.22\n",
            "21887  zf-TRM13_CCCH        PF11722.13         74    17.15     29   1.85   1.56   1.84   0.35\n",
            "21888  zf-U1                PF06220.17         11     3.18     38   1.43   1.19   1.40   0.20\n",
            "21889  zf-U11-48K           PF05253.17        182   182.00     25   2.00   1.72   2.00   0.33\n",
            "21890  zf-UBP               PF02148.24        142     4.83     63   0.89   0.67   0.79   0.16\n",
            "21891  zf-UBP_var           PF17807.6          20     3.22     64   0.88   0.79   0.84   0.05\n",
            "21892  zf-UBR               PF02207.25        259     2.60     69   0.81   0.59   0.69   0.22\n",
            "21893  zf-UDP               PF14569.11         22     1.00     78   0.73   0.64   0.67   0.08\n",
            "21894  zf-WIZ               PF23015.1          17     2.09     35   1.55   1.39   1.52   0.13\n",
            "21895  zf-WRNIP1_ubi        PF18279.6           1     1.00     21   1.58   1.33   1.53   0.34\n",
            "21896  zf-XS                PF03470.19         12     3.07     43   1.28   1.19   1.22   0.12\n",
            "21897  zf-ZPR1              PF03367.18        350     2.90     56   0.99   0.87   0.96   0.12\n",
            "21898  Zfx_Zfy_act          PF04704.18          9     0.88    328   0.59   0.60   0.50   0.04\n",
            "21899  ZFYVE21_C            PF16696.10         11     1.13    128   0.59   0.59   0.51   0.02\n",
            "21900  zf_C2H2_10           PF18414.6           3     1.81     26   2.06   1.81   2.03   0.37\n",
            "21901  zf_C2H2_13           PF18508.6           7     1.93     42   1.31   1.20   1.25   0.11\n",
            "21902  zf_C2H2_6            PF18450.6           5     2.15     28   1.92   1.71   1.89   0.29\n",
            "21903  zf_C2H2_ZHX          PF18387.6          18     2.03     53   1.05   0.96   1.01   0.07\n",
            "21904  zf_C2HC_14           PF18574.6          16     5.67     33   1.64   1.40   1.63   0.28\n",
            "21905  zf_CCCH_4            PF18345.6         255   255.00     19   2.41   1.94   2.37   0.63\n",
            "21906  zf_CCCH_5            PF18384.6           3     1.31     40   1.36   1.09   1.33   0.26\n",
            "21907  zf_CopZ              PF18423.6          56     3.56     62   0.90   0.79   0.85   0.10\n",
            "21908  zf_Hakai             PF18408.6           6     1.83     32   1.69   1.54   1.63   0.19\n",
            "21909  zf_PR_Knuckle        PF18445.6           2     1.02     38   1.43   1.26   1.39   0.13\n",
            "21910  zf_Rg                PF17915.6           6     1.60     49   1.13   1.02   1.08   0.13\n",
            "21911  zf_topless           PF21359.2          18     2.14     41   1.34   1.19   1.30   0.12\n",
            "21912  zf_UBZ               PF18439.6          69    15.01     32   1.69   1.47   1.68   0.23\n",
            "21913  zf_ZIC               PF18366.6          12     1.51     46   1.20   1.06   1.16   0.08\n",
            "21914  ZGRF1-like_N         PF10382.14         74     2.54     80   0.71   0.66   0.62   0.06\n",
            "21915  zinc-ribbons_6       PF07191.17         19     1.48     67   0.84   0.65   0.77   0.20\n",
            "21916  zinc-ribbon_6        PF10005.14         17     1.00     93   0.61   0.48   0.54   0.09\n",
            "21917  Zincin_1             PF06262.16         87     2.75     97   0.59   0.59   0.52   0.02\n",
            "21918  Zincin_2             PF10103.14        120     2.49    344   0.59   0.60   0.51   0.02\n",
            "21919  zinc_ribbon_10       PF10058.14         24     3.40     54   1.03   0.88   0.98   0.11\n",
            "21920  zinc_ribbon_11       PF11682.13          2     0.50    129   0.59   0.52   0.51   0.04\n",
            "21921  zinc_ribbon_12       PF11331.13         28     2.49     45   1.22   1.13   1.18   0.16\n",
            "21922  zinc_ribbon_13       PF09855.14         29     1.66     63   0.89   0.78   0.84   0.09\n",
            "21923  zinc_ribbon_15       PF17032.10         50     2.31     73   0.77   0.57   0.70   0.19\n",
            "21924  zinc_ribbon_16       PF17034.10          3     0.57    125   0.59   0.48   0.50   0.06\n",
            "21925  zinc_ribbon_2        PF13240.11        153   153.00     23   2.00   1.67   1.99   0.52\n",
            "21926  zinc_ribbon_4        PF13717.11         47    10.65     36   1.51   1.27   1.50   0.25\n",
            "21927  zinc_ribbon_5        PF13719.11          3     2.14     37   1.47   1.32   1.43   0.21\n",
            "21928  zinc_ribbon_6        PF14599.11         97     2.23     59   0.95   0.77   0.91   0.12\n",
            "21929  zinc_ribbon_9        PF14369.11         55    10.80     35   1.55   1.26   1.51   0.26\n",
            "21930  ZinT                 PF09223.16         28     0.77    181   0.59   0.53   0.53   0.04\n",
            "21931  Zip                  PF02535.27         37     5.88    323   0.59   0.61   0.50   0.03\n",
            "21932  ZIP4_domain          PF18292.6          25     2.08    160   0.59   0.57   0.50   0.05\n",
            "21933  ZipA_C               PF04354.18         75     2.91    128   0.59   0.59   0.55   0.02\n",
            "21934  ZirS_C               PF16583.10          2     0.46    145   0.59   0.59   0.52   0.02\n",
            "21935  Zmiz1_N              PF18028.6           3     0.64     99   0.59   0.56   0.53   0.03\n",
            "21936  Zn-C2H2_12           PF18112.6           6     5.03     27   1.98   1.72   1.97   0.28\n",
            "21937  zn-ribbon_14         PF16503.10         61     5.80     32   1.69   1.54   1.68   0.31\n",
            "21938  Zn-ribbon_8          PF09723.15        445    11.24     41   1.34   1.12   1.26   0.20\n",
            "21939  ZNF512_C2HC          PF21276.2          10     2.05     38   1.44   1.27   1.39   0.16\n",
            "21940  ZNF512_zf-C2H2       PF21367.2           2     2.00     20   2.45   2.19   2.42   0.50\n",
            "21941  ZnF_RZ-type          PF20173.3          52     3.66     54   1.03   0.86   0.97   0.15\n",
            "21942  ZNHIT3_C             PF21373.2           6     0.96     58   0.96   0.98   0.91   0.05\n",
            "21943  ZNRF_3_ecto          PF18212.6           5     0.73    105   0.59   0.61   0.53   0.03\n",
            "21944  ZnuA                 PF01297.22        717     6.16    263   0.59   0.59   0.47   0.03\n",
            "21945  zn_beta_ribbon-like  PF21957.1         145     2.39     65   0.86   0.70   0.80   0.13\n",
            "21946  Zn_clus              PF00172.23         28     6.24     39   1.40   1.11   1.33   0.46\n",
            "21947  Zn_dep_PLPC          PF00882.23        123    17.38    167   0.59   0.53   0.40   0.03\n",
            "21948  Zn_finger_prot       PF19242.4           3     0.63    268   0.59   0.55   0.50   0.04\n",
            "21949  Zn_peptidase         PF04228.18          5     0.69    290   0.59   0.57   0.52   0.02\n",
            "21950  Zn_peptidase_2       PF04298.17        249     1.74    218   0.59   0.61   0.54   0.04\n",
            "21951  Zn_protease          PF05618.16         10     2.10    138   0.59   0.60   0.51   0.03\n",
            "21952  Zn_ribbon_2          PF12674.12         74     1.93     81   0.70   0.57   0.62   0.08\n",
            "21953  Zn_ribbon_recom      PF13408.11        224    20.78     58   0.96   0.84   0.84   0.11\n",
            "21954  Zn_ribbon_SprT       PF17283.7          30    19.78     38   1.44   1.15   1.34   0.31\n",
            "21955  Zn_Tnp_IS1           PF03811.18          3     1.58     35   1.55   1.32   1.50   0.27\n",
            "21956  Zn_Tnp_IS1595        PF12760.12         25     6.31     46   1.20   0.95   1.10   0.22\n",
            "21957  Zn_Tnp_IS91          PF14319.11        215     2.86     90   0.63   0.50   0.58   0.09\n",
            "21958  Zona_CL1             PF22320.1           4     0.60    131   0.59   0.58   0.52   0.03\n",
            "21959  Zona_CL2             PF22321.1           3     0.61    154   0.59   0.59   0.50   0.02\n",
            "21960  Zona_pellucida       PF00100.28        278     6.80    263   0.59   0.52   0.40   0.05\n",
            "21961  ZoocinA_TRD          PF16775.10          4     0.67    106   0.59   0.55   0.51   0.03\n",
            "21962  Zot                  PF05707.17          6     1.78    197   0.59   0.57   0.46   0.02\n",
            "21963  ZP1_ZP4_Ig-like      PF22821.1          94     2.22     94   0.61   0.54   0.56   0.04\n",
            "21964  ZSWIM1-3_C           PF19286.4          16     3.39     45   1.22   1.24   1.20   0.09\n",
            "21965  ZSWIM1-3_helical     PF21600.2          18     1.61    126   0.59   0.55   0.55   0.05\n",
            "21966  ZSWIM1-3_RNaseH-like PF21056.2          85     3.18    128   0.59   0.55   0.54   0.04\n",
            "21967  ZSWIM3_N             PF21599.2           9     1.14    102   0.59   0.52   0.51   0.06\n",
            "21968  ZSWIM4-8_C           PF21055.2          32     1.23    173   0.59   0.58   0.51   0.03\n",
            "21969  ZT_dimer             PF16916.10        310     6.52     79   0.72   0.71   0.67   0.06\n",
            "21970  ZU5                  PF00791.25         52     2.80     97   0.59   0.59   0.55   0.01\n",
            "21971  ZUO1-like_ZHD        PF21884.1         104     1.28    104   0.59   0.54   0.54   0.07\n",
            "21972  ZW10_C               PF20666.2           6     1.17    131   0.59   0.57   0.54   0.03\n",
            "21973  ZW10_C2              PF22766.1           6     1.07    152   0.59   0.60   0.53   0.04\n",
            "21974  Zw10_middle          PF20665.2           6     1.46    230   0.59   0.60   0.50   0.03\n",
            "21975  Zw10_N               PF06248.18          6     2.19     93   0.61   0.65   0.57   0.06\n",
            "21976  Zwilch               PF09817.14         14     1.99    568   0.59   0.58   0.50   0.02\n",
            "21977  Zwint                PF15556.11          7     0.63    252   0.59   0.60   0.52   0.06\n",
            "21978  ZYG-11_interact      PF05884.17          8     1.10    295   0.59   0.62   0.51   0.03\n",
            "21979  ZZ                   PF00569.22          7     3.43     45   1.22   0.95   1.18   0.26\n"
          ]
        }
      ],
      "source": [
        "!hmmstat Pfam-A.hmm"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MjOtG1XIHQ0M"
      },
      "source": [
        "Поиск по цепям Маркова"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WHuc6bktHQfC",
        "outputId": "7a0aff6d-8b0a-4cb2-b07a-fd976027599e"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[1;30;43mВыходные данные были обрезаны до нескольких последних строк (5000).\u001b[0m\n",
            "   1 ?    9.5   0.1   3.5e-05      0.53      87     126 ..     181     220 ..     108     225 .. 0.65\n",
            "   2 ?   -3.5   0.0       0.3   4.7e+03     170     207 ..     260     298 ..     259     316 .. 0.65\n",
            "   3 ?    1.7   0.8    0.0083   1.3e+02      85     109 ..     344     374 ..     321     379 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.5 bits;  conditional E-value: 3.5e-05\n",
            "                   CSTTCCETCCSESTGGGSHHHHHHHHHHHHHHHHHHTGGG CS\n",
            "                   xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx RF\n",
            "          ZnuA  87 eeeehdheheeDpHvWldpknakkvaeniadaLseldpen 126\n",
            "                   + ++h+h+ + Dp + +dp++ +++ e+i+ +L +ld  n\n",
            "  CAH8610073.1 181 QLDDHNHDPDIDPDLDHDPDRFIQINEQIKINLNKLDRIN 220\n",
            "                   33444445555*************************9765 PP\n",
            "\n",
            "  == domain 2  score: -3.5 bits;  conditional E-value: 0.3\n",
            "                   HHHHHHCT-EEEEE.ESSSSSSSSSHHHHHHHHHHHHHT CS\n",
            "                   xxxxxxxxxxxxxx.xxxxxxxxxxxxxxxxxxxxxxxx RF\n",
            "          ZnuA 170 gYlaraygleqvai.egvspesepsaadlaelidkikek 207\n",
            "                   +Y+ ++yg + v i  g +  +e   ++l  + ++i+  \n",
            "  CAH8610073.1 260 NYWLHHYGYRLVPIeIGQKYTQENWGQKLMTIHEFINHY 298\n",
            "                   689999999888883344446666777777777777655 PP\n",
            "\n",
            "  == domain 3  score: 1.7 bits;  conditional E-value: 0.0083\n",
            "                   CE..CSTTCCETCCS....ESTGGGSHHHHH CS\n",
            "                   xx..xxxxxxxxxxx....xxxxxxxxxxxx RF\n",
            "          ZnuA  85 pl..eeeehdhehee....DpHvWldpknak 109\n",
            "                   p+   +++++h+h++       +W+ pkn +\n",
            "  CAH8610073.1 344 PInnSDDNDNHHHHHidtiETNIWFGPKNTI 374\n",
            "                   433333444444445778999*******976 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (263 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1073  (0.0699023); expected 307.0 (0.02)\n",
            "Passed bias filter:                      403  (0.0262541); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        58  (0.0037785); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000456026); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.51u 0.01s 00:00:00.52 Elapsed: 00:00:00.27\n",
            "# Mc/sec: 10041.72\n",
            "//\n",
            "Query:       zn_beta_ribbon-like  [M=65]\n",
            "Accession:   PF21957.1\n",
            "Description: Zinc beta-ribbon finger, putative\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.012   15.3   2.8      0.094   12.4   0.5    2.2  2  CAH8645473.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.015   15.0   2.8       0.11   12.2   0.5    2.2  2  CAH8645487.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8645473.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    3.1   0.1      0.01        80      15      27 ..      36      48 ..      28      50 .. 0.79\n",
            "   2 ?   12.4   0.5   1.2e-05     0.094       3      23 ..      78      96 ..      76     100 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 3.1 bits;  conditional E-value: 0.01\n",
            "  zn_beta_ribbon-like 15 ftCPkCkkktfvr 27\n",
            "                         +tCP C+++  +r\n",
            "         CAH8645473.1 36 ITCPFCHNSGVTR 48\n",
            "                         69****9876665 PP\n",
            "\n",
            "  == domain 2  score: 12.4 bits;  conditional E-value: 1.2e-05\n",
            "  zn_beta_ribbon-like  3 ysLekykgkskkftCPkCkkk 23\n",
            "                         + L+  ++ks ++ CP+C+++\n",
            "         CAH8645473.1 78 FCLD--STKSARHFCPSCNRQ 96\n",
            "                         8899..6668********875 PP\n",
            "\n",
            ">> CAH8645487.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.9   0.1     0.012        92      15      27 ..      46      58 ..      38      60 .. 0.79\n",
            "   2 ?   12.2   0.5   1.4e-05      0.11       3      23 ..      88     106 ..      86     110 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.9 bits;  conditional E-value: 0.012\n",
            "  zn_beta_ribbon-like 15 ftCPkCkkktfvr 27\n",
            "                         +tCP C+++  +r\n",
            "         CAH8645487.1 46 ITCPFCHNSGVTR 58\n",
            "                         69****9876665 PP\n",
            "\n",
            "  == domain 2  score: 12.2 bits;  conditional E-value: 1.4e-05\n",
            "  zn_beta_ribbon-like   3 ysLekykgkskkftCPkCkkk 23 \n",
            "                          + L+  ++ks ++ CP+C+++\n",
            "         CAH8645487.1  88 FCLD--STKSARHFCPSCNRQ 106\n",
            "                          8899..6668********875 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (65 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       987  (0.0642997); expected 307.0 (0.02)\n",
            "Passed bias filter:                      349  (0.0227362); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        29  (0.00188925); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000130293); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.20u 0.00s 00:00:00.20 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 6384.29\n",
            "//\n",
            "Query:       Zn_clus  [M=39]\n",
            "Accession:   PF00172.23\n",
            "Description: Fungal Zn(2)-Cys(6) binuclear cluster domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (39 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1312  (0.0854723); expected 307.0 (0.02)\n",
            "Passed bias filter:                      476  (0.0310098); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        31  (0.00201954); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.19u 0.01s 00:00:00.20 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 3727.62\n",
            "//\n",
            "Query:       Zn_dep_PLPC  [M=167]\n",
            "Accession:   PF00882.23\n",
            "Description: Zinc dependent phospholipase C\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (167 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       705  (0.0459283); expected 307.0 (0.02)\n",
            "Passed bias filter:                      313  (0.0203909); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        17  (0.00110749); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.24u 0.00s 00:00:00.24 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 13171.16\n",
            "//\n",
            "Query:       Zn_finger_prot  [M=268]\n",
            "Accession:   PF19242.4\n",
            "Description: Putative zinc finger protein\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (268 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1606  (0.104625); expected 307.0 (0.02)\n",
            "Passed bias filter:                      434  (0.0282736); expected 307.0 (0.02)\n",
            "Passed Vit filter:                       171  (0.0111401); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                       135  (0.00879479); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 6.23u 0.04s 00:00:06.27 Elapsed: 00:00:03.82\n",
            "# Mc/sec: 733.35\n",
            "//\n",
            "Query:       Zn_peptidase  [M=290]\n",
            "Accession:   PF04228.18\n",
            "Description: Putative neutral zinc metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (290 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       417  (0.0271661); expected 307.0 (0.02)\n",
            "Passed bias filter:                      277  (0.0180456); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        20  (0.00130293); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.35u 0.00s 00:00:00.35 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 16949.24\n",
            "//\n",
            "Query:       Zn_peptidase_2  [M=218]\n",
            "Accession:   PF04298.17\n",
            "Description: Putative neutral zinc metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (218 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       690  (0.0449511); expected 307.0 (0.02)\n",
            "Passed bias filter:                      480  (0.0312704); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        59  (0.00384365); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         4  (0.000260586); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.32u 0.01s 00:00:00.33 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 12812.01\n",
            "//\n",
            "Query:       Zn_protease  [M=138]\n",
            "Accession:   PF05618.16\n",
            "Description: Putative ATP-dependant zinc protease\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (138 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       365  (0.0237785); expected 307.0 (0.02)\n",
            "Passed bias filter:                      256  (0.0166775); expected 307.0 (0.02)\n",
            "Passed Vit filter:                         8  (0.000521173); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.20u 0.00s 00:00:00.20 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 13467.63\n",
            "//\n",
            "Query:       Zn_ribbon_2  [M=81]\n",
            "Accession:   PF12674.12\n",
            "Description: Putative zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.041   14.4   0.3       0.16   12.5   0.3    2.1  1  CAH8494746.1  unnamed protein product [Schistosoma intercalat\n",
            "          6    7.4   8.1        6.2    7.4   0.4    3.5  3  CAH8602055.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8494746.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.5   0.3     2e-05      0.16       1      50 [.      98     144 ..      98     166 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.5 bits;  conditional E-value: 2e-05\n",
            "   Zn_ribbon_2   1 iCqsCGmPlkkeedlgteadgsknedYCkyCykdGeftedltleemiefc 50 \n",
            "                   +CqsCG P +   ++  ++ g +  + Ck C ++G+   +  l + i + \n",
            "  CAH8494746.1  98 LCQSCGNPET---TMHVNRRGGTVATTCKACGSQGQLDVTHRLTQYIVKN 144\n",
            "                   5*******75...3446999****************99876666666554 PP\n",
            "\n",
            ">> CAH8602055.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.8   0.0     0.043   3.3e+02      25      37 ..     131     141 ..     122     171 .. 0.65\n",
            "   2 ?    0.4   1.0      0.12   9.1e+02      18      39 ..     184     203 ..     175     229 .. 0.57\n",
            "   3 ?    7.4   0.4   0.00081       6.2      11      50 ..     225     268 ..     222     293 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.8 bits;  conditional E-value: 0.043\n",
            "   Zn_ribbon_2  25 edYCkyCykdGef 37 \n",
            "                   + YCkyC  d +f\n",
            "  CAH8602055.1 131 RIYCKYC--DVAF 141\n",
            "                   5699999..3333 PP\n",
            "\n",
            "  == domain 2  score: 0.4 bits;  conditional E-value: 0.12\n",
            "   Zn_ribbon_2  18 eadgsknedYCkyCykdGefte 39 \n",
            "                   ++ + k + YC+ C +   f++\n",
            "  CAH8602055.1 184 NNRNRKPNFYCEDCKRS--FQR 203\n",
            "                   455556667****9774..333 PP\n",
            "\n",
            "  == domain 3  score: 7.4 bits;  conditional E-value: 0.00081\n",
            "   Zn_ribbon_2  11 keedlgteadgsknedYCkyC...ykdGefte.dltleemiefc 50 \n",
            "                   +e+d g+ ++  + +dYC +C   +++G+  + +l+ ++    c\n",
            "  CAH8602055.1 225 NESDEGESKKAPRRTDYCDVCeqtFQNGRHFYaHLKWKKHKRLC 268\n",
            "                   57888999999**********55455898777677766665555 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (81 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1003  (0.065342); expected 307.0 (0.02)\n",
            "Passed bias filter:                      553  (0.0360261); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        49  (0.00319218); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000130293); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.23u 0.00s 00:00:00.23 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 6118.54\n",
            "//\n",
            "Query:       Zn_ribbon_recom  [M=58]\n",
            "Accession:   PF13408.11\n",
            "Description: Recombinase zinc beta ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    1.7e-07   31.3   4.8    1.2e-06   28.6   0.4    2.2  2  CAH8429076.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0087   16.2  14.7        5.3    7.3   0.0    5.8  6  CAH8499498.1  unnamed protein product [Schistosoma intercalat\n",
            "  ------ inclusion threshold ------\n",
            "        1.9    8.7   7.9        1.8    8.8   2.9    2.5  2  CAH8656188.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8429076.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   28.6   0.4   2.3e-10   1.2e-06       4      53 ..      22      71 ..      21      75 .. 0.90\n",
            "   2 !    4.4   0.4    0.0079        40       8      31 ..      95     125 ..      94     130 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 28.6 bits;  conditional E-value: 2.3e-10\n",
            "  Zn_ribbon_recom  4 GllrCgeCGsamtgrtsk.ggkryYrCstrrrkgkkCsnksireeeleeav 53\n",
            "                     G+l+C+eC + ++++  k  ++ yY C+++  +  + +n+ ++ ++le+ v\n",
            "     CAH8429076.1 22 GILFCSECNNMLYPKEDKrTKTLYYACRNCDYSQ-EADNPCVYINKLEQEV 71\n",
            "                     9*************99885666********8888.9************998 PP\n",
            "\n",
            "  == domain 2  score: 4.4 bits;  conditional E-value: 0.0079\n",
            "  Zn_ribbon_recom   8 CgeCGsamtg....rtsk...ggkryYrCst 31 \n",
            "                      C +CG+   +    +t k   + + yY+C++\n",
            "     CAH8429076.1  95 CRRCGKQEAVffqsQTLKaeeNMRLYYVCTN 125\n",
            "                      8999988444442244446667779****97 PP\n",
            "\n",
            ">> CAH8499498.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !    4.5   0.4    0.0074        38       5      20 ..      72      87 ..      70     119 .. 0.73\n",
            "   2 !    5.4   0.6     0.004        20       5      35 ..     128     165 ..     126     171 .. 0.74\n",
            "   3 ?    0.5   0.1      0.14     7e+02       8      19 ..     159     170 ..     154     177 .. 0.85\n",
            "   4 ?    2.1   0.1     0.042   2.2e+02       5      14 ..     212     221 ..     210     241 .. 0.81\n",
            "   5 ?    1.6   0.0     0.059     3e+02       5      19 ..     265     279 ..     262     289 .. 0.81\n",
            "   6 !    7.3   0.0     0.001       5.3       4      17 ..     292     305 ..     291     314 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.5 bits;  conditional E-value: 0.0074\n",
            "  Zn_ribbon_recom  5 llrCgeCGsamtgrts 20\n",
            "                     + +C+ C +++t r++\n",
            "     CAH8499498.1 72 IFKCSSCDRTFTSRSG 87\n",
            "                     689*********9986 PP\n",
            "\n",
            "  == domain 2  score: 5.4 bits;  conditional E-value: 0.004\n",
            "  Zn_ribbon_recom   5 llrCgeCGsamtg.....rtsk..ggkryYrCstrrrk 35 \n",
            "                      l +C  CG ++       ++ k  +g r + C ++ r+\n",
            "     CAH8499498.1 128 LAKCEVCGTTFASlgnlkKHEKsiHGGRLFDCPSCDRS 165\n",
            "                      789********998866533334666688999887765 PP\n",
            "\n",
            "  == domain 3  score: 0.5 bits;  conditional E-value: 0.14\n",
            "  Zn_ribbon_recom   8 CgeCGsamtgrt 19 \n",
            "                      C+ C +++t r+\n",
            "     CAH8499498.1 159 CPSCDRSFTSRS 170\n",
            "                      *********987 PP\n",
            "\n",
            "  == domain 4  score: 2.1 bits;  conditional E-value: 0.042\n",
            "  Zn_ribbon_recom   5 llrCgeCGsa 14 \n",
            "                      +++C +CG  \n",
            "     CAH8499498.1 212 MVKCEQCGDF 221\n",
            "                      79******75 PP\n",
            "\n",
            "  == domain 5  score: 1.6 bits;  conditional E-value: 0.059\n",
            "  Zn_ribbon_recom   5 llrCgeCGsamtgrt 19 \n",
            "                      + +C+ C +++  r+\n",
            "     CAH8499498.1 265 IFKCASCNKSFASRS 279\n",
            "                      579*****9998765 PP\n",
            "\n",
            "  == domain 6  score: 7.3 bits;  conditional E-value: 0.001\n",
            "  Zn_ribbon_recom   4 GllrCgeCGsamtg 17 \n",
            "                       l++C+eC +++t \n",
            "     CAH8499498.1 292 KLIKCAECDRTFTS 305\n",
            "                      59**********97 PP\n",
            "\n",
            ">> CAH8656188.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.9   0.2     0.051   2.6e+02       8      19 ..     202     213 ..     200     218 .. 0.86\n",
            "   2 ?    8.8   2.9   0.00035       1.8       7      36 ..     227     266 ..     226     279 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.9 bits;  conditional E-value: 0.051\n",
            "  Zn_ribbon_recom   8 CgeCGsamtgrt 19 \n",
            "                      C +CG++++ ++\n",
            "     CAH8656188.1 202 CRYCGKSYSLKS 213\n",
            "                      9******98765 PP\n",
            "\n",
            "  == domain 2  score: 8.8 bits;  conditional E-value: 0.00035\n",
            "  Zn_ribbon_recom   7 rCgeCGsamtg......rtsk..ggkry..YrCstrrrkg 36 \n",
            "                       C +CG+++++      ++++  g+k+y    C +r  ++\n",
            "     CAH8656188.1 227 HCQDCGKRFSRpwlrdiHYRThtGEKPYecIICGRRFADR 266\n",
            "                      6*************99955555577766669999987666 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (58 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1068  (0.0695765); expected 307.0 (0.02)\n",
            "Passed bias filter:                      619  (0.0403257); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        80  (0.00521173); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000456026); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.23u 0.01s 00:00:00.24 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 4540.96\n",
            "//\n",
            "Query:       Zn_ribbon_SprT  [M=38]\n",
            "Accession:   PF17283.7\n",
            "Description: SprT-like zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    3.2e-07   30.1   4.6    8.2e-07   28.7   4.6    1.7  1  CAH8531114.1  unnamed protein product [Schistosoma intercalat\n",
            "  ------ inclusion threshold ------\n",
            "      0.017   14.9  16.7      0.018   14.9   7.8    3.0  2  CAH8435625.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.037   13.9   0.3      0.037   13.9   0.3    7.9  8  CAH8584746.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.089   12.6   0.1      0.089   12.6   0.1    3.3  4  CAH8429076.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.13   12.1   3.1          2    8.3   0.0    2.9  3  CAH8456994.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.15   11.9  14.8        6.6    6.6   3.5    3.9  3  CAH8437590.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.33   10.8   5.1       0.74    9.7   5.1    1.5  1  CAH8480268.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.53   10.2   4.6          2    8.3   1.8    2.5  2  CAH8508619.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.61   10.0  12.7      0.082   12.7   2.3    2.8  3  CAH8518337.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.99    9.3  25.1        6.3    6.7   8.2    4.5  3  CAH8620742.1  unnamed protein product, partial [Schistosoma i\n",
            "        1.4    8.9  13.2        0.1   12.5   2.3    2.8  3  CAH8518323.1  unnamed protein product [Schistosoma intercalat\n",
            "        3.1    7.7   5.8        8.6    6.3   1.0    3.6  3  CAH8608243.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8531114.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   28.7   4.6   6.4e-10   8.2e-07       2      34 ..     499     532 ..     498     535 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 28.7 bits;  conditional E-value: 6.4e-10\n",
            "  Zn_ribbon_SprT   2 kyvYvCpgCgreyprhrrs.drhnyrCrrCkgtL 34 \n",
            "                     +++Y+C+gCg  ++rh++s d+ + +C++C+ ++\n",
            "    CAH8531114.1 499 RFTYRCTGCGVCINRHSKSlDITKKICGHCRSKF 532\n",
            "                     8*****************************9876 PP\n",
            "\n",
            ">> CAH8435625.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   14.9   7.8   1.4e-05     0.018       3      34 ..     577     616 ..     577     620 .. 0.82\n",
            "   2 ?    4.0   1.5     0.035        45      15      37 ..     624     648 ..     621     649 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.9 bits;  conditional E-value: 1.4e-05\n",
            "  Zn_ribbon_SprT   3 yvYvCpgCgreyprhrrsdrhn........yrCrrCkgtL 34 \n",
            "                     y Y+Cp+C +e+pr    +rh         y+C +C+  +\n",
            "    CAH8435625.1 577 YRYQCPHCNKEFPRSANLNRHLrthtgekpYHCEYCQRGF 616\n",
            "                     99***********976666665666699*********766 PP\n",
            "\n",
            "  == domain 2  score: 4.0 bits;  conditional E-value: 0.035\n",
            "  Zn_ribbon_SprT  15 prhrrs.drhn.yrCrrCkgtLvqk 37 \n",
            "                      ++r + +r++ ++C  Ck  + q \n",
            "    CAH8435625.1 624 RHIRNIhQRERpFICTICKRAFAQR 648\n",
            "                     68999999999********998875 PP\n",
            "\n",
            ">> CAH8584746.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    3.2   0.1      0.06        77      13      31 ..      60      88 ..      59      89 .. 0.91\n",
            "   2 ?   -2.4   0.1       3.5   4.5e+03      26      31 ..     115     120 ..     115     123 .. 0.83\n",
            "   3 ?    0.4   0.1      0.46   5.9e+02       6      10 ..     153     157 ..     146     158 .. 0.76\n",
            "   4 ?    0.5   9.1      0.44   5.6e+02       4      34 ..     181     219 ..     181     233 .. 0.87\n",
            "   5 ?   -0.5   2.0      0.85   1.1e+03       6      34 ..     241     278 ..     239     282 .. 0.74\n",
            "   6 ?   13.9   0.3   2.9e-05     0.037       4      31 ..     323     361 ..     323     368 .. 0.87\n",
            "   7 ?   -2.7   0.2       4.3   5.5e+03      25      34 ..     379     388 ..     370     390 .. 0.74\n",
            "   8 ?    0.4   1.0      0.45   5.8e+02      15      29 ..     395     414 ..     393     419 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 3.2 bits;  conditional E-value: 0.06\n",
            "  Zn_ribbon_SprT 13 eyprhrrsdrhn..........yrCrrCk 31\n",
            "                    +++rh+r ++++          + C+ C+\n",
            "    CAH8584746.1 60 TQTRHLRQRVDKedlsslgirkMKCKLCG 88\n",
            "                    5799999999999999999999******6 PP\n",
            "\n",
            "  == domain 2  score: -2.4 bits;  conditional E-value: 3.5\n",
            "  Zn_ribbon_SprT  26 rCrrCk 31 \n",
            "                     +Cr C+\n",
            "    CAH8584746.1 115 ICRICA 120\n",
            "                     6****7 PP\n",
            "\n",
            "  == domain 3  score: 0.4 bits;  conditional E-value: 0.46\n",
            "  Zn_ribbon_SprT   6 vCpgC 10 \n",
            "                     +C++C\n",
            "    CAH8584746.1 153 RCTHC 157\n",
            "                     67777 PP\n",
            "\n",
            "  == domain 4  score: 0.5 bits;  conditional E-value: 0.44\n",
            "  Zn_ribbon_SprT   4 vYvCpgCgreyprhrrsdrhn........yrCrrCkgtL 34 \n",
            "                     +Y C +C+++++  r  + h         +rC  C+  +\n",
            "    CAH8584746.1 181 IYECIHCYERFTDKRVWRAHMdkhnegysHRCMECGRAF 219\n",
            "                     699999999999999999999999999999999998765 PP\n",
            "\n",
            "  == domain 5  score: -0.5 bits;  conditional E-value: 0.85\n",
            "  Zn_ribbon_SprT   6 vCpgCgreyprhrrsdrhn.........yrCrrCkgtL 34 \n",
            "                     +C  Cg+ yp +++ + h          + C +C+ ++\n",
            "    CAH8584746.1 241 TCEYCGLIYPKRSSLRYHIfrmhnqelaHECFMCQRRF 278\n",
            "                     67679999999999888876666677777777777665 PP\n",
            "\n",
            "  == domain 6  score: 13.9 bits;  conditional E-value: 2.9e-05\n",
            "  Zn_ribbon_SprT   4 vYvCpgCgreyprhrrsdrhn...........yrCrrCk 31 \n",
            "                     +Y+C++C+++++  +  +rh            + C+ C+\n",
            "    CAH8584746.1 323 IYQCTDCQTTFKSKLAMKRHIrvehlqlgpekFECQICG 361\n",
            "                     6******************98888888888888888886 PP\n",
            "\n",
            "  == domain 7  score: -2.7 bits;  conditional E-value: 4.3\n",
            "  Zn_ribbon_SprT  25 yrCrrCkgtL 34 \n",
            "                     + C +C+ ++\n",
            "    CAH8584746.1 379 FECEYCNKRF 388\n",
            "                     6699998776 PP\n",
            "\n",
            "  == domain 8  score: 0.4 bits;  conditional E-value: 0.45\n",
            "  Zn_ribbon_SprT  15 prhrrsdrhn.....yrCrr 29 \n",
            "                     + h+r++r       ++Cr+\n",
            "    CAH8584746.1 395 NEHLRIHRGEqpfecHICRK 414\n",
            "                     56999955556*66666664 PP\n",
            "\n",
            ">> CAH8429076.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.0   0.0      0.61   7.8e+02      27      37 ..      26      36 ..      20      37 .. 0.76\n",
            "   2 ?   -1.0   4.8       1.2   1.6e+03      24      30 ..      42      51 ..      25      54 .. 0.70\n",
            "   3 ?   12.6   0.1     7e-05     0.089      20      34 ..      88     103 ..      83     107 .. 0.77\n",
            "   4 ?   -2.6   0.5         4   5.1e+03       5       8 ..     121     124 ..     120     125 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.0 bits;  conditional E-value: 0.61\n",
            "  Zn_ribbon_SprT 27 CrrCkgtLvqk 37\n",
            "                    C  C+++L +k\n",
            "    CAH8429076.1 26 CSECNNMLYPK 36\n",
            "                    77788888776 PP\n",
            "\n",
            "  == domain 2  score: -1.0 bits;  conditional E-value: 1.2\n",
            "  Zn_ribbon_SprT 24 n...yrCrrC 30\n",
            "                    +   y Cr C\n",
            "    CAH8429076.1 42 KtlyYACRNC 51\n",
            "                    2455999999 PP\n",
            "\n",
            "  == domain 3  score: 12.6 bits;  conditional E-value: 7e-05\n",
            "  Zn_ribbon_SprT  20 sdrhnyrCrrCk.gtL 34 \n",
            "                     ++++ ++CrrC+ ++ \n",
            "    CAH8429076.1  88 PRTEDHICRRCGkQEA 103\n",
            "                     39*********97765 PP\n",
            "\n",
            "  == domain 4  score: -2.6 bits;  conditional E-value: 4\n",
            "  Zn_ribbon_SprT   5 YvCp 8  \n",
            "                     YvC+\n",
            "    CAH8429076.1 121 YVCT 124\n",
            "                     8997 PP\n",
            "\n",
            ">> CAH8456994.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.3   0.0    0.0016         2      12      32 ..     237     269 ..     236     272 .. 0.92\n",
            "   2 ?   -3.3   0.0       6.5   8.3e+03      23      28 ..     278     283 ..     276     285 .. 0.75\n",
            "   3 ?    3.0   0.4     0.073        93       5      11 ..     290     296 ..     289     309 .. 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.3 bits;  conditional E-value: 0.0016\n",
            "  Zn_ribbon_SprT  12 reyprhrrs.drhn...........yrCrrCkg 32 \n",
            "                     +++p h+++ ++ +           y+Cr C++\n",
            "    CAH8456994.1 237 TNIPKHLTVqKILKrlaklpnnlgpYICRLCNQ 269\n",
            "                     78999****999999****************97 PP\n",
            "\n",
            "  == domain 2  score: -3.3 bits;  conditional E-value: 6.5\n",
            "  Zn_ribbon_SprT  23 hnyrCr 28 \n",
            "                     +n+rC+\n",
            "    CAH8456994.1 278 ANHRCP 283\n",
            "                     689996 PP\n",
            "\n",
            "  == domain 3  score: 3.0 bits;  conditional E-value: 0.073\n",
            "  Zn_ribbon_SprT   5 YvCpgCg 11 \n",
            "                     Y+Cp+C \n",
            "    CAH8456994.1 290 YRCPECD 296\n",
            "                     99***98 PP\n",
            "\n",
            ">> CAH8437590.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.5   0.0     0.024        31      14      33 ..     515     535 ..     512     539 .. 0.82\n",
            "   2 ?    6.6   3.5    0.0052       6.6       6      30 ..     554     586 ..     552     593 .. 0.93\n",
            "   3 ?    5.8   0.9    0.0094        12       5      16 ..     609     620 ..     608     621 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.5 bits;  conditional E-value: 0.024\n",
            "  Zn_ribbon_SprT  14 yprhrrsdrhn.yrCrrCkgt 33 \n",
            "                      +r + +++ n y C+ C   \n",
            "    CAH8437590.1 515 TDRTLIINGPNsYACKLCSKV 535\n",
            "                     589999999999******765 PP\n",
            "\n",
            "  == domain 2  score: 6.6 bits;  conditional E-value: 0.0052\n",
            "  Zn_ribbon_SprT   6 vCpgCgreyprhrrsdrhn........yrCrrC 30 \n",
            "                     +C++Cg+ ++r    ++h         y C  C\n",
            "    CAH8437590.1 554 RCTHCGKSFSRKWLLKGHErthtgerpYSCSVC 586\n",
            "                     7**************999999999999999999 PP\n",
            "\n",
            "  == domain 3  score: 5.8 bits;  conditional E-value: 0.0094\n",
            "  Zn_ribbon_SprT   5 YvCpgCgreypr 16 \n",
            "                     Y+Cp+C r ++r\n",
            "    CAH8437590.1 609 YSCPHCPRSFSR 620\n",
            "                     9******98876 PP\n",
            "\n",
            ">> CAH8480268.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   5.1   0.00058      0.74      12      27 ..     163     178 ..     163     178 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.00058\n",
            "  Zn_ribbon_SprT  12 reyprhrrsdrhnyrC 27 \n",
            "                     ++y++h+r +rhn+rC\n",
            "    CAH8480268.1 163 TNYHAHSRLRRHNHRC 178\n",
            "                     589************* PP\n",
            "\n",
            ">> CAH8508619.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.6   0.0       0.2   2.6e+02      23      35 ..     275     293 ..     266     296 .. 0.74\n",
            "   2 ?    8.3   1.8    0.0015         2       6      30 ..     310     342 ..     308     347 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.6 bits;  conditional E-value: 0.2\n",
            "  Zn_ribbon_SprT  23 hn......yrCrrCkgtLv 35 \n",
            "                     hn      ++C +C+ +++\n",
            "    CAH8508619.1 275 HNrryaksFICNQCRKPFT 293\n",
            "                     4444556699****98775 PP\n",
            "\n",
            "  == domain 2  score: 8.3 bits;  conditional E-value: 0.0015\n",
            "  Zn_ribbon_SprT   6 vCpgCgreyprhrrsdrhn........yrCrrC 30 \n",
            "                      C+ Cg +++ ++  +rh         ++C+ C\n",
            "    CAH8508619.1 310 KCTICGAQFTQRSNLQRHSlrhvgfkpFVCKIC 342\n",
            "                     6******************************** PP\n",
            "\n",
            ">> CAH8518337.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.7   2.3   6.4e-05     0.082       5      32 ..      18      44 ..      16      46 .. 0.92\n",
            "   2 ?   -1.9   0.2       2.3     3e+03      27      31 ..     125     129 ..     119     131 .. 0.76\n",
            "   3 ?    4.0   0.3     0.035        45       4      11 ..     150     157 ..     147     160 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.7 bits;  conditional E-value: 6.4e-05\n",
            "  Zn_ribbon_SprT  5 YvCpgCgreyprhrrs.drhnyrCrrCkg 32\n",
            "                    Y C +Cg+  p+ + + d    +Cr+Ck+\n",
            "    CAH8518337.1 18 YFCYVCGTLLPHKLEVhDL--LVCRKCKT 44\n",
            "                    9******************..88****95 PP\n",
            "\n",
            "  == domain 2  score: -1.9 bits;  conditional E-value: 2.3\n",
            "  Zn_ribbon_SprT  27 CrrCk 31 \n",
            "                     C +C+\n",
            "    CAH8518337.1 125 CAYCG 129\n",
            "                     77776 PP\n",
            "\n",
            "  == domain 3  score: 4.0 bits;  conditional E-value: 0.035\n",
            "  Zn_ribbon_SprT   4 vYvCpgCg 11 \n",
            "                     +Y+C+ C \n",
            "    CAH8518337.1 150 IYTCTQCS 157\n",
            "                     79*****8 PP\n",
            "\n",
            ">> CAH8620742.1  unnamed protein product, partial [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.7   8.2    0.0049       6.3       5      33 ..     216     280 ..     215     283 .. 0.82\n",
            "   2 ?   -1.3   0.3       1.6     2e+03      24      30 ..     299     305 ..     293     310 .. 0.66\n",
            "   3 ?    4.8   3.3      0.02        25       2      34 ..     325     365 ..     324     369 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.7 bits;  conditional E-value: 0.0049\n",
            "  Zn_ribbon_SprT   5 YvCpgCgreyprhrrsdrhn....................................yrCrrCkgt 33 \n",
            "                     ++C +C ++++r +  +rh                                     y+Cr C+ +\n",
            "    CAH8620742.1 216 HTCEHCDKQFDRPSLLKRHTlthtgerpfeckycskgfstrsgvntherthtgqrpYVCRICGRR 280\n",
            "                     6899************************7777777777777777777777777777777777765 PP\n",
            "\n",
            "  == domain 2  score: -1.3 bits;  conditional E-value: 1.6\n",
            "  Zn_ribbon_SprT  24 nyrCrrC 30 \n",
            "                     +++C +C\n",
            "    CAH8620742.1 299 RHVCSQC 305\n",
            "                     4677777 PP\n",
            "\n",
            "  == domain 3  score: 4.8 bits;  conditional E-value: 0.02\n",
            "  Zn_ribbon_SprT   2 kyvYvCpgCgreyprhrrsdrhn........yrCrrCkgtL 34 \n",
            "                     k+ Y+C  C r +  +r  + h         ++C  C+  +\n",
            "    CAH8620742.1 325 KWPYRCHLCDRGFATERNLKSHEtthsgrkpHICTICGKGY 365\n",
            "                     8*****99***999999999999999***********9765 PP\n",
            "\n",
            ">> CAH8518323.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.5   2.3   7.9e-05       0.1       5      32 ..      42      68 ..      40      70 .. 0.92\n",
            "   2 ?   -2.2   0.2         3   3.9e+03      27      31 ..     149     153 ..     144     155 .. 0.76\n",
            "   3 ?    3.8   0.3     0.041        53       4      11 ..     174     181 ..     171     184 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.5 bits;  conditional E-value: 7.9e-05\n",
            "  Zn_ribbon_SprT  5 YvCpgCgreyprhrrs.drhnyrCrrCkg 32\n",
            "                    Y C +Cg+  p+ + + d    +Cr+Ck+\n",
            "    CAH8518323.1 42 YFCYVCGTLLPHKLEVhDL--LVCRKCKT 68\n",
            "                    9******************..88****95 PP\n",
            "\n",
            "  == domain 2  score: -2.2 bits;  conditional E-value: 3\n",
            "  Zn_ribbon_SprT  27 CrrCk 31 \n",
            "                     C +C+\n",
            "    CAH8518323.1 149 CAYCG 153\n",
            "                     77886 PP\n",
            "\n",
            "  == domain 3  score: 3.8 bits;  conditional E-value: 0.041\n",
            "  Zn_ribbon_SprT   4 vYvCpgCg 11 \n",
            "                     +Y+C+ C \n",
            "    CAH8518323.1 174 IYTCTQCS 181\n",
            "                     69*****8 PP\n",
            "\n",
            ">> CAH8608243.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.0   0.1       0.3   3.8e+02       7      32 ..     623     628 ..     618     633 .. 0.71\n",
            "   2 ?   -2.2   0.0       2.9   3.8e+03      24      34 ..     647     655 ..     640     656 .. 0.71\n",
            "   3 ?    6.3   1.0    0.0067       8.6       4      31 ..     704     727 ..     704     735 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.0 bits;  conditional E-value: 0.3\n",
            "  Zn_ribbon_SprT   7 CpgCgreyprhrrsdrhnyrCrrCkg 32 \n",
            "                     Cp+C+                    +\n",
            "    CAH8608243.1 623 CPHCK--------------------Q 628\n",
            "                     66665....................3 PP\n",
            "\n",
            "  == domain 2  score: -2.2 bits;  conditional E-value: 2.9\n",
            "  Zn_ribbon_SprT  24 nyrCrrCkgtL 34 \n",
            "                        C+ Ck+ L\n",
            "    CAH8608243.1 647 T--CPICKTSL 655\n",
            "                     4..88998766 PP\n",
            "\n",
            "  == domain 3  score: 6.3 bits;  conditional E-value: 0.0067\n",
            "  Zn_ribbon_SprT   4 vYvCpgCgreyprhrrsdrhnyrCrrCk 31 \n",
            "                     +YvCp+C+     h+ ++r +    +C+\n",
            "    CAH8608243.1 704 KYVCPVCHFATENHPLIRRLK----QCG 727\n",
            "                     5*******9999999999998....444 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (38 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1238  (0.0806515); expected 307.0 (0.02)\n",
            "Passed bias filter:                      541  (0.0352443); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        90  (0.00586319); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                        38  (0.00247557); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):              12  [number of targets reported over threshold]\n",
            "# CPU time: 0.30u 0.01s 00:00:00.31 Elapsed: 00:00:00.16\n",
            "# Mc/sec: 2401.25\n",
            "//\n",
            "Query:       Zn_Tnp_IS1  [M=35]\n",
            "Accession:   PF03811.18\n",
            "Description: InsA N-terminal domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.15   11.6   0.2       0.15   11.6   0.2    1.9  2  CAH8595560.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.18   11.3   0.2       0.18   11.3   0.2    1.9  2  CAH8595542.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.69    9.5   6.9        1.5    8.4   0.5    3.1  3  CAH8645487.1  unnamed protein product [Schistosoma intercalat\n",
            "        9.3    5.9   7.9         25    4.5   0.1    3.2  3  CAH8513280.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8595560.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.6   0.3       1.1   4.1e+03       8      12 ..     107     113 ..     106     115 .. 0.64\n",
            "   2 ?   11.6   0.2   3.9e-05      0.15       2      14 ..     129     141 ..     128     151 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.6 bits;  conditional E-value: 1.1\n",
            "    Zn_Tnp_IS1   8 CPr..Cs 12 \n",
            "                   CPr  C+\n",
            "  CAH8595560.1 107 CPRvyCE 113\n",
            "                   9985555 PP\n",
            "\n",
            "  == domain 2  score: 11.6 bits;  conditional E-value: 3.9e-05\n",
            "    Zn_Tnp_IS1   2 AsVsihCPrCssa 14 \n",
            "                   A V i CPrC+ +\n",
            "  CAH8595560.1 129 AMVKIYCPRCQDT 141\n",
            "                   78********965 PP\n",
            "\n",
            ">> CAH8595542.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.9   0.3       1.3   4.9e+03       8      12 ..     107     113 ..     106     115 .. 0.64\n",
            "   2 ?   11.3   0.2   4.8e-05      0.18       2      14 ..     129     141 ..     128     151 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.9 bits;  conditional E-value: 1.3\n",
            "    Zn_Tnp_IS1   8 CPr..Cs 12 \n",
            "                   CPr  C+\n",
            "  CAH8595542.1 107 CPRvyCE 113\n",
            "                   9985555 PP\n",
            "\n",
            "  == domain 2  score: 11.3 bits;  conditional E-value: 4.8e-05\n",
            "    Zn_Tnp_IS1   2 AsVsihCPrCssa 14 \n",
            "                   A V i CPrC+ +\n",
            "  CAH8595542.1 129 AMVKIYCPRCQDT 141\n",
            "                   78********965 PP\n",
            "\n",
            ">> CAH8645487.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.5   0.0      0.11   4.3e+02       2      11 ..      25      34 ..      24      41 .. 0.76\n",
            "   2 ?    8.4   0.5    0.0004       1.5       4      18 ..      44      58 ..      43      60 .. 0.90\n",
            "   3 ?    2.7   0.2     0.024        92       8      12 ..     100     104 ..      94     109 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.5 bits;  conditional E-value: 0.11\n",
            "    Zn_Tnp_IS1  2 AsVsihCPrC 11\n",
            "                  A+V +h P  \n",
            "  CAH8645487.1 25 ATVIVHQPTV 34\n",
            "                  9*****9975 PP\n",
            "\n",
            "  == domain 2  score: 8.4 bits;  conditional E-value: 0.0004\n",
            "    Zn_Tnp_IS1  4 VsihCPrCssadVyR 18\n",
            "                  V i CP C+   V R\n",
            "  CAH8645487.1 44 VGITCPFCHNSGVTR 58\n",
            "                  88*******999987 PP\n",
            "\n",
            "  == domain 3  score: 2.7 bits;  conditional E-value: 0.024\n",
            "    Zn_Tnp_IS1   8 CPrCs 12 \n",
            "                   CP+C+\n",
            "  CAH8645487.1 100 CPSCN 104\n",
            "                   99996 PP\n",
            "\n",
            ">> CAH8513280.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.3   0.3      0.13     5e+02       6      11 ..      48      53 ..      45      57 .. 0.80\n",
            "   2 ?    4.5   0.1    0.0066        25       5      12 ..     240     247 ..     238     250 .. 0.88\n",
            "   3 ?    3.6   0.4     0.012        48       5      20 ..     269     285 ..     268     297 .. 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.3 bits;  conditional E-value: 0.13\n",
            "    Zn_Tnp_IS1  6 ihCPrC 11\n",
            "                   +CP C\n",
            "  CAH8513280.1 48 FSCPHC 53\n",
            "                  689999 PP\n",
            "\n",
            "  == domain 2  score: 4.5 bits;  conditional E-value: 0.0066\n",
            "    Zn_Tnp_IS1   5 sihCPrCs 12 \n",
            "                   ++ CP+C+\n",
            "  CAH8513280.1 240 NVNCPSCN 247\n",
            "                   89*****7 PP\n",
            "\n",
            "  == domain 3  score: 3.6 bits;  conditional E-value: 0.012\n",
            "    Zn_Tnp_IS1   5 sihCPrCssad.VyRhG 20 \n",
            "                    i CP C   d  +R G\n",
            "  CAH8513280.1 269 AINCPACGYRDcEVRSG 285\n",
            "                   599****9998778876 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (35 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       773  (0.0503583); expected 307.0 (0.02)\n",
            "Passed bias filter:                      368  (0.0239739); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        45  (0.0029316); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         5  (0.000325733); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               4  [number of targets reported over threshold]\n",
            "# CPU time: 0.18u 0.00s 00:00:00.18 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 3547.81\n",
            "//\n",
            "Query:       Zn_Tnp_IS1595  [M=46]\n",
            "Accession:   PF12760.12\n",
            "Description: Transposase zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    0.00088   19.0   2.5    0.00088   19.0   2.5    2.6  2  CAH8451923.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0011   18.6   2.3      0.039   13.7   0.1    2.3  2  CAH8645473.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0015   18.2   2.3      0.046   13.5   0.1    2.3  2  CAH8645487.1  unnamed protein product [Schistosoma intercalat\n",
            "  ------ inclusion threshold ------\n",
            "      0.028   14.1   0.1      0.054   13.2   0.1    1.4  1  CAH8645501.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.16   11.8   3.3          1    9.2   0.0    2.6  2  CAH8582886.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.23   11.2   1.7       0.63    9.8   0.3    2.4  2  CAH8531114.1  unnamed protein product [Schistosoma intercalat\n",
            "        1.2    9.0   5.7          8    6.3   0.1    3.2  3  CAH8513280.1  unnamed protein product [Schistosoma intercalat\n",
            "        2.1    8.1   7.8        1.1    9.1   0.0    3.1  3  CAH8582860.1  unnamed protein product [Schistosoma intercalat\n",
            "        3.3    7.5  10.8       0.57   10.0   0.7    2.7  3  CAH8429076.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8451923.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.3   2.9      0.18     3e+02      21      44 ..       6      32 ..       2      34 .. 0.63\n",
            "   2 !   19.0   2.5   5.1e-07   0.00088      20      45 ..      76     110 ..      71     111 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.3 bits;  conditional E-value: 0.18\n",
            "  Zn_Tnp_IS1595 21 CPhCgsr.eayriktrklfqCka..Cr 44\n",
            "                   CP C+s     +  +   + C+a  C+\n",
            "   CAH8451923.1  6 CPLCHSLlMIEESAHCYSLNCSAplCS 32\n",
            "                   *****8744444444666666666665 PP\n",
            "\n",
            "  == domain 2  score: 19.0 bits;  conditional E-value: 5.1e-07\n",
            "  Zn_Tnp_IS1595  20 vCPhCgsreayrikt.........rklfqCkaCrk 45 \n",
            "                    +CP+Cg+++ay+++            ++ C +C++\n",
            "   CAH8451923.1  76 KCPKCGHTRAYFVQMqtrsadepsTTKYSCMKCHH 110\n",
            "                    6************9999****************98 PP\n",
            "\n",
            ">> CAH8645473.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   13.7   0.1   2.3e-05     0.039      13      32 ..      30      49 ..      27      54 .. 0.85\n",
            "   2 ?    4.4   0.3     0.018        30      19      30 ..      88     100 ..      83     101 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.7 bits;  conditional E-value: 2.3e-05\n",
            "  Zn_Tnp_IS1595 13 arWpngfvCPhCgsreayri 32\n",
            "                   +r+p g +CP+C+++ ++r+\n",
            "   CAH8645473.1 30 HRYPVGITCPFCHNSGVTRV 49\n",
            "                   699**********9977665 PP\n",
            "\n",
            "  == domain 2  score: 4.4 bits;  conditional E-value: 0.018\n",
            "  Zn_Tnp_IS1595  19 fvCPhCgsr.eay 30 \n",
            "                    + CP C  +  +y\n",
            "   CAH8645473.1  88 HFCPSCNRQvGYY 100\n",
            "                    45****9775555 PP\n",
            "\n",
            ">> CAH8645487.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   13.5   0.1   2.7e-05     0.046      13      32 ..      40      59 ..      37      64 .. 0.85\n",
            "   2 ?    4.3   0.3      0.02        35      19      30 ..      98     110 ..      93     111 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.5 bits;  conditional E-value: 2.7e-05\n",
            "  Zn_Tnp_IS1595 13 arWpngfvCPhCgsreayri 32\n",
            "                   +r+p g +CP+C+++ ++r+\n",
            "   CAH8645487.1 40 HRYPVGITCPFCHNSGVTRV 59\n",
            "                   699**********9977665 PP\n",
            "\n",
            "  == domain 2  score: 4.3 bits;  conditional E-value: 0.02\n",
            "  Zn_Tnp_IS1595  19 fvCPhCgsr.eay 30 \n",
            "                    + CP C  +  +y\n",
            "   CAH8645487.1  98 HFCPSCNRQvGYY 110\n",
            "                    45****9775555 PP\n",
            "\n",
            ">> CAH8645501.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.2   0.1   3.2e-05     0.054      13      32 ..      40      59 ..      37      64 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.2 bits;  conditional E-value: 3.2e-05\n",
            "  Zn_Tnp_IS1595 13 arWpngfvCPhCgsreayri 32\n",
            "                   +r+p g +CP+C+++ ++r+\n",
            "   CAH8645501.1 40 HRYPVGITCPFCHNSGVTRV 59\n",
            "                   699**********9977665 PP\n",
            "\n",
            ">> CAH8582886.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.4   1.5      0.16   2.8e+02      20      28 ..      77      86 ..      73     105 .. 0.75\n",
            "   2 ?    9.2   0.0    0.0006         1      11      42 ..     221     250 ..     217     251 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.4 bits;  conditional E-value: 0.16\n",
            "  Zn_Tnp_IS1595 20 vCPhCgsr.e 28\n",
            "                    CPhC++  +\n",
            "   CAH8582886.1 77 SCPHCQKPlK 86\n",
            "                   5****96544 PP\n",
            "\n",
            "  == domain 2  score: 9.2 bits;  conditional E-value: 0.0006\n",
            "  Zn_Tnp_IS1595  11 eqarWpngfvCPhCgsreayriktrklfqCka 42 \n",
            "                     ++++++ + CP C    +  ++  ++f+C+a\n",
            "   CAH8582886.1 221 DAMIFGALPYCPSCK-NGPLHYSN-GQFKCSA 250\n",
            "                    57999999*******.67777777.99***85 PP\n",
            "\n",
            ">> CAH8531114.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.8   0.3   0.00037      0.63      15      44 ..     482     508 ..     478     509 .. 0.87\n",
            "   2 ?   -1.6   0.0       1.4   2.4e+03      20      27 ..     524     531 ..     521     538 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.8 bits;  conditional E-value: 0.00037\n",
            "  Zn_Tnp_IS1595  15 WpngfvCPhCgsreayriktrklfqCkaCr 44 \n",
            "                    Wp  +v + C+   ay+i tr +++C  C+\n",
            "   CAH8531114.1 482 WPKLPVVSVCH---AYTIDTRFTYRCTGCG 508\n",
            "                    77777777888...***************7 PP\n",
            "\n",
            "  == domain 2  score: -1.6 bits;  conditional E-value: 1.4\n",
            "  Zn_Tnp_IS1595  20 vCPhCgsr 27 \n",
            "                    +C hC+s \n",
            "   CAH8531114.1 524 ICGHCRSK 531\n",
            "                    57888765 PP\n",
            "\n",
            ">> CAH8513280.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.3   0.1    0.0047         8      19      27 ..      48      56 ..      41      67 .. 0.82\n",
            "   2 ?    1.3   0.0      0.17     3e+02      19      27 ..     241     249 ..     239     256 .. 0.79\n",
            "   3 ?    1.9   0.6      0.11   1.9e+02      20      27 ..     271     278 ..     268     284 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.3 bits;  conditional E-value: 0.0047\n",
            "  Zn_Tnp_IS1595 19 fvCPhCgsr 27\n",
            "                   f CPhCg +\n",
            "   CAH8513280.1 48 FSCPHCGYE 56\n",
            "                   88****976 PP\n",
            "\n",
            "  == domain 2  score: 1.3 bits;  conditional E-value: 0.17\n",
            "  Zn_Tnp_IS1595  19 fvCPhCgsr 27 \n",
            "                    ++CP C + \n",
            "   CAH8513280.1 241 VNCPSCNGL 249\n",
            "                    57****875 PP\n",
            "\n",
            "  == domain 3  score: 1.9 bits;  conditional E-value: 0.11\n",
            "  Zn_Tnp_IS1595  20 vCPhCgsr 27 \n",
            "                    +CP Cg r\n",
            "   CAH8513280.1 271 NCPACGYR 278\n",
            "                    7****965 PP\n",
            "\n",
            ">> CAH8582860.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.1   0.1         2   3.4e+03      37      44 ..      16      23 ..      11      27 .. 0.71\n",
            "   2 ?    1.3   1.5      0.17   2.9e+02      20      28 ..     111     120 ..     107     139 .. 0.75\n",
            "   3 ?    9.1   0.0   0.00062       1.1      11      42 ..     255     284 ..     251     285 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.1 bits;  conditional E-value: 2\n",
            "  Zn_Tnp_IS1595 37 lfqCkaCr 44\n",
            "                   +  C++Cr\n",
            "   CAH8582860.1 16 RSNCNKCR 23\n",
            "                   45677776 PP\n",
            "\n",
            "  == domain 2  score: 1.3 bits;  conditional E-value: 0.17\n",
            "  Zn_Tnp_IS1595  20 vCPhCgsr.e 28 \n",
            "                     CPhC++  +\n",
            "   CAH8582860.1 111 SCPHCQKPlK 120\n",
            "                    5****96544 PP\n",
            "\n",
            "  == domain 3  score: 9.1 bits;  conditional E-value: 0.00062\n",
            "  Zn_Tnp_IS1595  11 eqarWpngfvCPhCgsreayriktrklfqCka 42 \n",
            "                     ++++++ + CP C    +  ++  ++f+C+a\n",
            "   CAH8582860.1 255 DAMIFGALPYCPSCK-NGPLHYSN-GQFKCSA 284\n",
            "                    57999999*******.67777777.99***85 PP\n",
            "\n",
            ">> CAH8429076.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.5   0.2      0.29     5e+02      35      44 ..      21      30 ..      11      32 .. 0.69\n",
            "   2 ?    2.6   2.1     0.066   1.1e+02      21      45 ..      26      53 ..      22      54 .. 0.69\n",
            "   3 ?   10.0   0.7   0.00033      0.57      19      40 ..      93     123 ..      88     131 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.5 bits;  conditional E-value: 0.29\n",
            "  Zn_Tnp_IS1595 35 rklfqCkaCr 44\n",
            "                   ++ + C++C+\n",
            "   CAH8429076.1 21 VGILFCSECN 30\n",
            "                   5667788886 PP\n",
            "\n",
            "  == domain 2  score: 2.6 bits;  conditional E-value: 0.066\n",
            "  Zn_Tnp_IS1595 21 CPhCgsr.eayrikt..rklfqCkaCrk 45\n",
            "                   C+ C ++  + + k+    ++ C+ C +\n",
            "   CAH8429076.1 26 CSECNNMlYPKEDKRtkTLYYACRNCDY 53\n",
            "                   8888877644444445567899999986 PP\n",
            "\n",
            "  == domain 3  score: 10.0 bits;  conditional E-value: 0.00033\n",
            "  Zn_Tnp_IS1595  19 fvCPhCgsreayrikt.........rklfqC 40 \n",
            "                    ++C++Cg++ea +++          r ++ C\n",
            "   CAH8429076.1  93 HICRRCGKQEAVFFQSqtlkaeenmRLYYVC 123\n",
            "                    78*******9998887566666666666666 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (46 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1159  (0.0755049); expected 307.0 (0.02)\n",
            "Passed bias filter:                      564  (0.0367427); expected 307.0 (0.02)\n",
            "Passed Vit filter:                       107  (0.00697068); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                        34  (0.00221498); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               9  [number of targets reported over threshold]\n",
            "# CPU time: 0.29u 0.00s 00:00:00.28 Elapsed: 00:00:00.16\n",
            "# Mc/sec: 2928.74\n",
            "//\n",
            "Query:       Zn_Tnp_IS91  [M=90]\n",
            "Accession:   PF14319.11\n",
            "Description: Transposase zinc-binding domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.065   13.0   1.0       0.17   11.7   1.0    1.7  1  CAH8464169.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.069   12.9   1.0       0.18   11.6   1.0    1.6  1  CAH8464186.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8464169.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.7   1.0   2.2e-05      0.17      18      61 ..     198     241 ..     186     249 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.7 bits;  conditional E-value: 2.2e-05\n",
            "   Zn_Tnp_IS91  18 eqrkaleailkCrtaalGghvyaCeeCghekivynsCknrhCps 61 \n",
            "                   +++k+++ + + +t  l +h+  C++C+ ++ v n+ + r C s\n",
            "  CAH8464169.1 198 SVTKEASPYAESKTVPLNFHEKVCDSCQATSSVSNTPECRKCIS 241\n",
            "                   6778888899999999********************99999976 PP\n",
            "\n",
            ">> CAH8464186.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.6   1.0   2.3e-05      0.18      18      61 ..     208     251 ..     196     259 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.6 bits;  conditional E-value: 2.3e-05\n",
            "   Zn_Tnp_IS91  18 eqrkaleailkCrtaalGghvyaCeeCghekivynsCknrhCps 61 \n",
            "                   +++k+++ + + +t  l +h+  C++C+ ++ v n+ + r C s\n",
            "  CAH8464186.1 208 SVTKEASPYAESKTVPLNFHEKVCDSCQATSSVSNTPECRKCIS 251\n",
            "                   6778888899999999********************99999976 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (90 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1463  (0.0953094); expected 307.0 (0.02)\n",
            "Passed bias filter:                      447  (0.0291205); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        45  (0.0029316); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000130293); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.23u 0.01s 00:00:00.24 Elapsed: 00:00:00.12\n",
            "# Mc/sec: 7541.26\n",
            "//\n",
            "Query:       Zona_CL1  [M=131]\n",
            "Accession:   PF22320.1\n",
            "Description: Zonadhesin, first cohesin-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (131 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       387  (0.0252117); expected 307.0 (0.02)\n",
            "Passed bias filter:                      216  (0.0140717); expected 307.0 (0.02)\n",
            "Passed Vit filter:                         8  (0.000521173); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.19u 0.00s 00:00:00.19 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 13291.20\n",
            "//\n",
            "Query:       Zona_CL2  [M=154]\n",
            "Accession:   PF22321.1\n",
            "Description: Zonadhesin, second cohesin-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (154 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       642  (0.0418241); expected 307.0 (0.02)\n",
            "Passed bias filter:                      289  (0.0188274); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        12  (0.000781759); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.22u 0.00s 00:00:00.22 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 12224.26\n",
            "//\n",
            "Query:       Zona_pellucida  [M=263]\n",
            "Accession:   PF00100.28\n",
            "Description: Zona pellucida-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    5.5e-31  107.8   5.4    8.2e-31  107.3   5.4    1.3  1  CAH8579277.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8579277.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  107.3   5.4   5.3e-35   8.2e-31       7     263 .]     306     554 ..     301     554 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 107.3 bits;  conditional E-value: 5.3e-35\n",
            "                     EEEEESSTTSS---GGGEEETT....TTB--SEEETTTTEEEEEEETTSTT-EEE..E-SSEEEEEEEEEE-----SSTT..EESS--EEEEEEE CS\n",
            "  Zona_pellucida   7 tvsvskdlltpaldpssltlgdr..dpsCkavpsteasaevvfsfpltsCGttlqveengdhliyenelvyspevsseirssisrssitelpfsC 99 \n",
            "                      + +  ++++p l+ s ++l ++    +C+   +++++ +++  +pl +CG +l+  +n++++  + +l  ++ +    ++ i  + + +++++C\n",
            "    CAH8579277.1 306 AIIIYDKYYNPDLNSSMIRLDQDllPIECQYHVDDNTT-HIIAYVPLVGCGGSLT-LSNSTSIAIKLNLLREYLTP---ENGIISNLPMKFTVTC 395\n",
            "                     455556666778999999998879999***99977776.599**********998.55777777777777444433...23344555569***** PP\n",
            "\n",
            "                     EEE-----................------EEEEEE-TEEB.............-TT-EEEEEEEE--TSC--EEEEEEEEEEESSS-.TTSS.S CS\n",
            "  Zona_pellucida 100 rYprksnvslsaaeksssplavv.gsgsltvslrlytdssysspy.eedypvvvllgdplyvevsllsrtdpnlvlvldsCwAtpspdpnsspky 192\n",
            "                       pr+++++++++++++ + +   +s+s+  +l+ly+d+ +   + + + p+++  gd++y  vsl + + +  +l l++CwAt++p++ns+ + \n",
            "    CAH8579277.1 396 LIPRSKQIQSEVVASPNITTNLLyQSESVLSTLKLYRDQLF---TvQLQQPITIPPGDKVYALVSLVNPQ-KTSKLILENCWATATPNRNSTLRQ 486\n",
            "                     ********98888884444444557777779**********...447899***************88885.89********************** PP\n",
            "\n",
            "                     EEEBTTEEGGGG.TT-EE-SS-SST.TEEEEEEE--EET....T....SS-EEEEEEEEEEEE.TTS--BTB--EE CS\n",
            "  Zona_pellucida 193 dliengCpnded.tyskvsvsengpshkkrfsvkaFrFvnsskk....lssevYlhCsvkvCdkkdsssrcnkaCs 263\n",
            "                     dli+n C+  ed t     v++    +++ f+++aF   +   +     +s++YlhC+++vCd ++++++c++ C+\n",
            "    CAH8579277.1 487 DLIVNRCAAYEDlT----VVPS--SATQVGFTFNAFYLNTA--GsivpIYSNIYLHCDIRVCDIRENNENCQQYCQ 554\n",
            "                     ************55....3233..3699********99843..257999**************9999999999996 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (263 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1683  (0.109642); expected 307.0 (0.02)\n",
            "Passed bias filter:                      224  (0.0145928); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        74  (0.00482085); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         1  (6.51466e-05); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.39u 0.01s 00:00:00.40 Elapsed: 00:00:00.22\n",
            "# Mc/sec: 12194.54\n",
            "//\n",
            "Query:       ZoocinA_TRD  [M=106]\n",
            "Accession:   PF16775.10\n",
            "Description: Target recognition domain of lytic exoenzyme\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (106 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       400  (0.0260586); expected 307.0 (0.02)\n",
            "Passed bias filter:                      275  (0.0179153); expected 307.0 (0.02)\n",
            "Passed Vit filter:                         9  (0.000586319); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.18u 0.00s 00:00:00.18 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 10828.69\n",
            "//\n",
            "Query:       Zot  [M=197]\n",
            "Accession:   PF05707.17\n",
            "Description: Zonular occludens toxin (Zot)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    0.00037   19.8   0.0    0.00061   19.1   0.0    1.3  1  CAH8509083.1  unnamed protein product [Schistosoma intercalat\n",
            "  ------ inclusion threshold ------\n",
            "      0.038   13.2   0.0       0.25   10.5   0.0    2.2  2  CAH8640559.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8509083.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.1   0.0     8e-08   0.00061       4      97 ..     150     244 ..     147     275 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.1 bits;  conditional E-value: 8e-08\n",
            "           Zot   4 fhtGklgsgKtsgavkkdllpalkdgrkiitnvdGLrleriaqvlkekkstarliridspekeetldelalwav..idnkgAllviDEcqeifpvr 97 \n",
            "                    + G++g+gKt+++++  +++++  gr   + ++ ++l     +++   +  +  + +++++ ++ d+l +     +++k  +lv+DE+ ++   +\n",
            "  CAH8509083.1 150 YVSGAPGTGKTAVVLNV-VQNFKTFGRCNTAVINCMQLTSCTDIFGRISTVLEAHNGKENNSISDADSLECFLNqqPQKKTIILVLDEVDQLSTRS 244\n",
            "                   578*********99988.*************************99998884333343355666666666555443256889999*****9985544 PP\n",
            "\n",
            ">> CAH8640559.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.5   0.0   3.3e-05      0.25       4      65 ..     239     305 ..     238     363 .. 0.84\n",
            "   2 ?   -0.6   0.0     0.086   6.6e+02       4      17 ..     512     525 ..     511     576 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.5 bits;  conditional E-value: 3.3e-05\n",
            "           Zot   4 fhtGklgsgKtsga..vkkdllp..alkdgrkiitnvdGLrleriaqvlkekks.tarliridspek 65 \n",
            "                   + +G++g+gKt++a  v ++     +l +g +i+ ++ G    ++ + ++e+ + ++ +i+id+ + \n",
            "  CAH8640559.1 239 LLYGPPGTGKTLVAraVANESGSffFLINGPEIMSKLAGESESNLRKAFEEAEKnAPAIIFIDELDA 305\n",
            "                   569********86511555544487888*********************999888999****94443 PP\n",
            "\n",
            "  == domain 2  score: -0.6 bits;  conditional E-value: 0.086\n",
            "           Zot   4 fhtGklgsgKtsga 17 \n",
            "                   +++G++g gKt+ a\n",
            "  CAH8640559.1 512 LFYGPPGCGKTLLA 525\n",
            "                   789********855 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (197 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       475  (0.0309446); expected 307.0 (0.02)\n",
            "Passed bias filter:                      352  (0.0229316); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        33  (0.00214984); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000130293); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.27u 0.00s 00:00:00.27 Elapsed: 00:00:00.14\n",
            "# Mc/sec: 14487.55\n",
            "//\n",
            "Query:       ZP1_ZP4_Ig-like  [M=94]\n",
            "Accession:   PF22821.1\n",
            "Description: Zona pellucida sperm-binding proteins 1/4, Ig-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (94 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       454  (0.0295765); expected 307.0 (0.02)\n",
            "Passed bias filter:                      291  (0.0189577); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        19  (0.00123779); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.18u 0.01s 00:00:00.19 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 9045.21\n",
            "//\n",
            "Query:       ZSWIM1-3_C  [M=45]\n",
            "Accession:   PF19286.4\n",
            "Description: Zinc finger SWIM domain-containing protein 1/3, C-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.062   12.8   0.2          1    8.9   0.1    2.7  2  CAH8600613.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8600613.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.5   0.0     0.029   4.5e+02       4      15 ..     301     312 ..     300     314 .. 0.91\n",
            "   2 ?    8.9   0.1   6.8e-05         1      17      35 ..     480     498 ..     479     499 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.5 bits;  conditional E-value: 0.029\n",
            "    ZSWIM1-3_C   4 rslkIqsLskEl 15 \n",
            "                   r++k+++L+k +\n",
            "  CAH8600613.1 301 RHEKVKQLTKNI 312\n",
            "                   99*******986 PP\n",
            "\n",
            "  == domain 2  score: 8.9 bits;  conditional E-value: 6.8e-05\n",
            "    ZSWIM1-3_C  17 nLLlqcegeELeeRysTLr 35 \n",
            "                   nLL+ ce ++  +R++T r\n",
            "  CAH8600613.1 480 NLLTACETKDYTKRSNTFR 498\n",
            "                   9****************87 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (45 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       565  (0.0368078); expected 307.0 (0.02)\n",
            "Passed bias filter:                      392  (0.0255375); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        49  (0.00319218); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         1  (6.51466e-05); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.18u 0.01s 00:00:00.19 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 4345.98\n",
            "//\n",
            "Query:       ZSWIM1-3_helical  [M=126]\n",
            "Accession:   PF21600.2\n",
            "Description: ZSWIM1/3, helical domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (126 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       745  (0.0485342); expected 307.0 (0.02)\n",
            "Passed bias filter:                      347  (0.0226059); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        23  (0.00149837); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.22u 0.00s 00:00:00.22 Elapsed: 00:00:00.14\n",
            "# Mc/sec: 9420.00\n",
            "//\n",
            "Query:       ZSWIM1-3_RNaseH-like  [M=128]\n",
            "Accession:   PF21056.2\n",
            "Description: Zinc finger SWIM domain-containing protein 1/3, RNaseH-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    1.1e-26   93.1   0.3    2.5e-26   91.9   0.3    1.6  1  CAH8610639.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-25   89.0   0.1    3.3e-25   88.3   0.1    1.4  1  CAH8521522.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.3e-25   88.3   0.1    7.4e-25   87.1   0.1    1.6  1  CAH8628311.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.5e-25   88.2   0.1      7e-25   87.2   0.1    1.5  1  CAH8521489.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.7e-25   88.1   0.1    7.6e-25   87.1   0.1    1.5  1  CAH8566054.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.8e-25   88.1   0.1    7.5e-25   87.1   0.1    1.5  1  CAH8521511.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.8e-25   88.1   0.1    7.5e-25   87.1   0.1    1.5  1  CAH8628337.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.8e-25   88.0   0.1    7.9e-25   87.0   0.1    1.5  1  CAH8566075.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.8e-25   88.0   0.1    7.7e-25   87.1   0.1    1.5  1  CAH8628285.1  unnamed protein product [Schistosoma intercalat\n",
            "      4e-25   88.0   0.1      8e-25   87.0   0.1    1.5  1  CAH8521472.1  unnamed protein product [Schistosoma intercalat\n",
            "      4e-25   88.0   0.1      8e-25   87.0   0.1    1.5  1  CAH8628294.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.2e-24   86.4   0.1    2.4e-24   85.5   0.1    1.5  1  CAH8496708.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.7e-24   86.0   0.0    3.7e-24   84.9   0.0    1.5  1  CAH8496729.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.1e-24   85.6   0.0    4.8e-24   84.5   0.0    1.6  1  CAH8496714.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.1e-24   85.6   0.0    4.7e-24   84.5   0.0    1.6  1  CAH8496721.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-24   85.2   0.0    6.8e-24   84.0   0.0    1.6  1  CAH8496687.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-24   85.2   0.0    6.7e-24   84.0   0.0    1.6  1  CAH8496694.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-24   85.2   0.0      7e-24   84.0   0.0    1.6  1  CAH8496701.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.3e-24   85.0   0.3    7.1e-24   83.9   0.3    1.6  1  CAH8568710.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.5e-24   84.9   0.3    7.8e-24   83.8   0.3    1.6  1  CAH8568720.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.1e-23   82.4   0.2    7.6e-23   80.6   0.0    2.0  1  CAH8439424.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.7e-23   82.1   0.2    9.1e-23   80.4   0.0    2.0  1  CAH8439409.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.1e-20   73.6   0.2    1.6e-20   73.1   0.2    1.2  1  CAH8439799.1  unnamed protein product [Schistosoma intercalat\n",
            "    8.4e-12   44.9   0.0    1.2e-11   44.4   0.0    1.2  1  CAH8288197.1  unnamed protein product, partial [Schistosoma i\n",
            "    3.2e-10   39.8   0.0    4.8e-10   39.2   0.0    1.3  1  CAH8293817.1  unnamed protein product, partial [Schistosoma i\n",
            "    3.9e-08   33.1   0.0    8.6e-08   32.0   0.0    1.6  1  CAH8628351.1  unnamed protein product [Schistosoma intercalat\n",
            "    4.5e-08   32.9   0.0      1e-07   31.7   0.0    1.6  1  CAH8628322.1  unnamed protein product [Schistosoma intercalat\n",
            "    7.1e-08   32.2   0.1    1.5e-07   31.1   0.0    1.6  2  CAH8535678.1  unnamed protein product [Schistosoma intercalat\n",
            "    4.5e-05   23.2   0.0    7.9e-05   22.4   0.0    1.5  1  CAH8292344.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8610639.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   91.9   0.3   4.6e-29   2.5e-26      18     127 ..     186     295 ..     169     296 .] 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 91.9 bits;  conditional E-value: 4.6e-29\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+ ++++++f++  + +l++++Pev+++d+T+ktn++++ lf+++v+d+ g+g+ v++a  ++e++  l+++l++Fke + + +k++++\n",
            "          CAH8610639.1 186 ENGSITAVCFSSYDQIRLYNQYPEVVCIDSTYKTNNKKHSLFQLVVTDNWGRGRTVMFAWTKKEKRADLTWILDKFKEIMGNTTKTETF 274\n",
            "                           66679************************************************************************************ PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++ +e a+++    +a++\n",
            "          CAH8610639.1 275 VMDCARSESAAVRITHGHANI 295\n",
            "                           ********9999877777665 PP\n",
            "\n",
            ">> CAH8521522.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   88.3   0.1   6.2e-28   3.3e-25      18     127 ..      91     200 ..      77     201 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 88.3 bits;  conditional E-value: 6.2e-28\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8521522.1  91 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 179\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8521522.1 180 VMDCARCESAAVRMTHGHANI 200\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8628311.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.1   0.1   1.4e-27   7.4e-25      18     127 ..     205     314 ..     191     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.1 bits;  conditional E-value: 1.4e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8628311.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8628311.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8521489.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.2   0.1   1.3e-27     7e-25      18     127 ..     183     292 ..     169     293 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.2 bits;  conditional E-value: 1.3e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8521489.1 183 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 271\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8521489.1 272 VMDCARCESAAVRMTHGHANI 292\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8566054.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.1   0.1   1.4e-27   7.6e-25      18     127 ..     203     312 ..     189     313 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.1 bits;  conditional E-value: 1.4e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8566054.1 203 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 291\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8566054.1 292 VMDCARCESAAVRMTHGHANI 312\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8521511.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.1   0.1   1.4e-27   7.5e-25      18     127 ..     193     302 ..     179     303 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.1 bits;  conditional E-value: 1.4e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8521511.1 193 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 281\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8521511.1 282 VMDCARCESAAVRMTHGHANI 302\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8628337.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.1   0.1   1.4e-27   7.5e-25      18     127 ..     193     302 ..     179     303 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.1 bits;  conditional E-value: 1.4e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8628337.1 193 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 281\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8628337.1 282 VMDCARCESAAVRMTHGHANI 302\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8566075.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.0   0.1   1.5e-27   7.9e-25      18     127 ..     203     312 ..     189     313 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.0 bits;  conditional E-value: 1.5e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8566075.1 203 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 291\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8566075.1 292 VMDCARCESAAVRMTHGHANI 312\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8628285.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.1   0.1   1.5e-27   7.7e-25      18     127 ..     205     314 ..     191     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.1 bits;  conditional E-value: 1.5e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8628285.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8628285.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8521472.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.0   0.1   1.5e-27     8e-25      18     127 ..     205     314 ..     191     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.0 bits;  conditional E-value: 1.5e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8521472.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8521472.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8628294.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   87.0   0.1   1.5e-27     8e-25      18     127 ..     205     314 ..     191     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 87.0 bits;  conditional E-value: 1.5e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++\n",
            "          CAH8628294.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8628294.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496708.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   85.5   0.1   4.6e-27   2.4e-24      18     127 ..     205     314 ..     191     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 85.5 bits;  conditional E-value: 4.6e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496708.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           555699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496708.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496729.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   84.9   0.0   6.9e-27   3.7e-24      18     127 ..      15     124 ..       3     125 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 84.9 bits;  conditional E-value: 6.9e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496729.1  15 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 103\n",
            "                           455699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496729.1 104 VMDCARCESAAVRMTHGHANI 124\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496714.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   84.5   0.0   9.1e-27   4.8e-24      18     127 ..      91     200 ..      78     201 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 84.5 bits;  conditional E-value: 9.1e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496714.1  91 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 179\n",
            "                           455699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496714.1 180 VMDCARCESAAVRMTHGHANI 200\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496721.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   84.5   0.0   8.9e-27   4.7e-24      18     127 ..      91     200 ..      78     201 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 84.5 bits;  conditional E-value: 8.9e-27\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496721.1  91 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 179\n",
            "                           455699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496721.1 180 VMDCARCESAAVRMTHGHANI 200\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496687.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   84.0   0.0   1.3e-26   6.8e-24      18     127 ..     205     314 ..     192     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 84.0 bits;  conditional E-value: 1.3e-26\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496687.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           455699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496687.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496694.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   84.0   0.0   1.3e-26   6.7e-24      18     127 ..     205     314 ..     192     315 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 84.0 bits;  conditional E-value: 1.3e-26\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496694.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 293\n",
            "                           455699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496694.1 294 VMDCARCESAAVRMTHGHANI 314\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8496701.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   84.0   0.0   1.3e-26     7e-24      18     127 ..     215     324 ..     202     325 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 84.0 bits;  conditional E-value: 1.3e-26\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvv 106\n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn+++y lf+++v+d++g+g+ v++a   +e++  + ++l++Fke + +   ++++\n",
            "          CAH8496701.1 215 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNKKYSLFQLVVTDNCGRGRTVMFAWTLREKRADVIWILDQFKEIMGDTMLTETF 303\n",
            "                           455699*********************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 107 vvDkdftelavLkeeFPearv 127\n",
            "                           v+D++  e a++++   +a++\n",
            "          CAH8496701.1 304 VMDCARCESAAVRMTHGHANI 324\n",
            "                           ************999988876 PP\n",
            "\n",
            ">> CAH8568710.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   83.9   0.3   1.3e-26   7.1e-24       5     127 ..     145     266 ..     141     267 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 83.9 bits;  conditional E-value: 1.3e-26\n",
            "  ZSWIM1-3_RNaseH-like   5 kedsgnvaeifvdeekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletF 93 \n",
            "                           +  + n+   + +e+ ++++i+f+ +++  ++ aFPe +++d+T++tnk g++lf+++v+d++g+g+ v++al+++e+++ ++k+le F\n",
            "          CAH8568710.1 145 QRVQANTEVKVFNENGNMSMICFSRREQIAVYHAFPEAICIDSTYQTNKVGFPLFQLVVTDSLGQGHTVMYALCSQERRD-VEKLLECF 232\n",
            "                           45555555556789999***********************************************************9976.899***** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like  94 kennpswekvkvvvvDkdftelavLkeeFPearv 127\n",
            "                           ke +     ++++++D + te+++++++  ++++\n",
            "          CAH8568710.1 233 KEIMCGTYATRTFIMDSAATEISAVQSAHSHSNI 266\n",
            "                           **************************99887765 PP\n",
            "\n",
            ">> CAH8568720.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   83.8   0.3   1.5e-26   7.8e-24       6     127 ..     163     283 ..     158     284 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 83.8 bits;  conditional E-value: 1.5e-26\n",
            "  ZSWIM1-3_RNaseH-like   6 edsgnvaeifvdeekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFk 94 \n",
            "                             + n+   + +e+ ++++i+f+ +++  ++ aFPe +++d+T++tnk g++lf+++v+d++g+g+ v++al+++e+++ ++k+le Fk\n",
            "          CAH8568720.1 163 RVQANTEVKVFNENGNMSMICFSRREQIAVYHAFPEAICIDSTYQTNKVGFPLFQLVVTDSLGQGHTVMYALCSQERRD-VEKLLECFK 250\n",
            "                           5555555556789999***********************************************************9976.899****** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like  95 ennpswekvkvvvvDkdftelavLkeeFPearv 127\n",
            "                           e +     ++++++D + te+++++++  ++++\n",
            "          CAH8568720.1 251 EIMCGTYATRTFIMDSAATEISAVQSAHSHSNI 283\n",
            "                           *************************99887765 PP\n",
            "\n",
            ">> CAH8439424.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   80.6   0.0   1.4e-25   7.6e-23      10     128 .]     111     229 ..     104     229 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 80.6 bits;  conditional E-value: 1.4e-25\n",
            "  ZSWIM1-3_RNaseH-like  10 nvaeifvdeekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennp 98 \n",
            "                           + + i  ++++++++it+   ++ ++++++Pev+l+ a +++n+ g+ lf+f+v+d+fg  +++++++ + ++ke ++ +l++F     \n",
            "          CAH8439424.1 111 SYVGIERNSSDQVTHITLARYEQVDVYRQYPEVVLLNAAYDVNSTGFSLFQFVVTDGFGCPRPIMYSISTIQQKEEIKILLNHFCGIVG 199\n",
            "                           566677789999***************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like  99 swekvkvvvvDkdftelavLkeeFPearvl 128\n",
            "                           s+++++++++D+++ +l+v++++FP+ar+l\n",
            "          CAH8439424.1 200 SVDDTRTFIMDCAIVQLDVVQKMFPKARIL 229\n",
            "                           ****************************97 PP\n",
            "\n",
            ">> CAH8439409.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   80.4   0.0   1.7e-25   9.1e-23      10     128 .]     173     291 ..     166     291 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 80.4 bits;  conditional E-value: 1.7e-25\n",
            "  ZSWIM1-3_RNaseH-like  10 nvaeifvdeekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennp 98 \n",
            "                           + + i  ++++++++it+   ++ ++++++Pev+l+ a +++n+ g+ lf+f+v+d+fg  +++++++ + ++ke ++ +l++F     \n",
            "          CAH8439409.1 173 SYVGIERNSSDQVTHITLARYEQVDVYRQYPEVVLLNAAYDVNSTGFSLFQFVVTDGFGCPRPIMYSISTIQQKEEIKILLNHFCGIVG 261\n",
            "                           566677789999***************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like  99 swekvkvvvvDkdftelavLkeeFPearvl 128\n",
            "                           s+++++++++D+++ +l+v++++FP+ar+l\n",
            "          CAH8439409.1 262 SVDDTRTFIMDCAIVQLDVVQKMFPKARIL 291\n",
            "                           ****************************97 PP\n",
            "\n",
            ">> CAH8439799.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   73.1   0.2     3e-23   1.6e-20      11     127 ..      74     190 ..      64     191 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 73.1 bits;  conditional E-value: 3e-23\n",
            "  ZSWIM1-3_RNaseH-like  11 vaeifvd.eekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennp 98 \n",
            "                           ++ + v+  + +++++++++ ++ +l+++FPevl + +T++    g+++f++ v+d++  +++++++l+++e++e l  +l++F+e  +\n",
            "          CAH8439799.1  74 SIVF-VEqCDGEITQVSLSHYEQIELYKRFPEVLSFSVTYNACLVGFTIFQVAVTDGLLCTRPIMFSLHSTEQTEDLCVLLKHFREIFK 161\n",
            "                           4444.4448888***************************************************************************** PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like  99 swekvkvvvvDkdftelavLkeeFPearv 127\n",
            "                           +++++ +v vD+ +++ ++++e FP++r+\n",
            "          CAH8439799.1 162 DVSSTLTVAVDCPVSKPELVQEFFPTSRI 190\n",
            "                           **************************998 PP\n",
            "\n",
            ">> CAH8288197.1  unnamed protein product, partial [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   44.4   0.0   2.3e-14   1.2e-11      11     126 ..      49     163 ..      42     165 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 44.4 bits;  conditional E-value: 2.3e-14\n",
            "  ZSWIM1-3_RNaseH-like  11 vaeifvdeekvlevitfqtakmkslfeaFPevllvdaThktnksgyklfsfmvedafgkgqvvqhallenetkenlrkaletFkennps 99 \n",
            "                           + e+ +d+e+ l+  +f ta++ +l  +  +v+ +d T+ktn+++++l+++++ d + k  ++++ +l +et++ ++ +l+ F+++  +\n",
            "          CAH8288197.1  49 LCEYELDDENCLSYFFFMTAEQVNLAGHVCDVIGFDGTYKTNNENVHLYQVVALDMNFKAIPMCIGFLCRETSTFISAFLRFFQRCT-N 136\n",
            "                           567789999***************************************************************************999.5 PP\n",
            "\n",
            "  ZSWIM1-3_RNaseH-like 100 wekvkvvvvDkdftelavLkeeFPear 126\n",
            "                            +++  +v+D +    a++ +++P+a \n",
            "          CAH8288197.1 137 SRQLVGIVTDDSPAIAAAITQVYPDAH 163\n",
            "                           677778899***************996 PP\n",
            "\n",
            ">> CAH8293817.1  unnamed protein product, partial [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   39.2   0.0   9.1e-13   4.8e-10      56     127 ..       1      72 [.       1      73 [. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 39.2 bits;  conditional E-value: 9.1e-13\n",
            "  ZSWIM1-3_RNaseH-like  56 yklfsfmvedafgkgqvvqhallenetkenlrkaletFkennpswekvkvvvvDkdftelavLkeeFPearv 127\n",
            "                           y lf+++v+d++g+g+ v++a  ++e++  + ++l++Fke + +   ++++v+D++  e a++++   +a++\n",
            "          CAH8293817.1   1 YSLFQLVVTDNCGRGRTVMFAWTRREKRADVIWILDQFKEIMGDTMLTETFVMDCARCESAAVRMTHGHANI 72 \n",
            "                           89*************************************************************999998887 PP\n",
            "\n",
            ">> CAH8628351.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   32.0   0.0   1.6e-10   8.6e-08      18      54 ..     183     219 ..     169     220 .] 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 32.0 bits;  conditional E-value: 1.6e-10\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnks 54 \n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn++\n",
            "          CAH8628351.1 183 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNK 219\n",
            "                           555699*****************************97 PP\n",
            "\n",
            ">> CAH8628322.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   31.7   0.0   1.9e-10     1e-07      18      54 ..     205     241 ..     191     242 .] 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 31.7 bits;  conditional E-value: 1.9e-10\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnks 54 \n",
            "                           e+  +++i+f+++++ +l++++Pev+++d+T++tn++\n",
            "          CAH8628322.1 205 ENGYATRICFSSSNQIQLYRKYPEVVCIDSTYNTNNK 241\n",
            "                           555699*****************************97 PP\n",
            "\n",
            ">> CAH8535678.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.2   0.0       5.9   3.1e+03      81      95 ..      41      55 ..      36      74 .. 0.50\n",
            "   2 !   31.1   0.0   2.9e-10   1.5e-07      18      54 ..     107     143 ..      89     144 .] 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.2 bits;  conditional E-value: 5.9\n",
            "  ZSWIM1-3_RNaseH-like 81 etkenlrkaletFke 95\n",
            "                          e+kenl+ ++ + ++\n",
            "          CAH8535678.1 41 EEKENLKPVILHCQS 55\n",
            "                          444444444333322 PP\n",
            "\n",
            "  == domain 2  score: 31.1 bits;  conditional E-value: 2.9e-10\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnks 54 \n",
            "                           e+  +++i+f++    +l++++Pev+++d+T+ktn++\n",
            "          CAH8535678.1 107 ENGCITAICFSSYDRIRLYNQYPEVVCIDSTYKTNNK 143\n",
            "                           556699*****************************97 PP\n",
            "\n",
            ">> CAH8292344.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   22.4   0.0   1.5e-07   7.9e-05      18      55 ..      91     127 ..      73     128 .] 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 22.4 bits;  conditional E-value: 1.5e-07\n",
            "  ZSWIM1-3_RNaseH-like  18 eekvlevitfqtakmkslfeaFPevllvdaThktnksg 55 \n",
            "                           e+  +++i+f+++++ +l++++P v+++d+T++tn+++\n",
            "          CAH8292344.1  91 ENGYATRICFSSSNQIQLYRKYP-VVCIDSTYNTNNKK 127\n",
            "                           556699*****************.899********986 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (128 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       612  (0.0398697); expected 307.0 (0.02)\n",
            "Passed bias filter:                      372  (0.0242345); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        59  (0.00384365); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                        29  (0.00188925); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):              29  [number of targets reported over threshold]\n",
            "# CPU time: 0.22u 0.01s 00:00:00.23 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 7633.24\n",
            "//\n",
            "Query:       ZSWIM3_N  [M=102]\n",
            "Accession:   PF21599.2\n",
            "Description: Zinc finger SWIM domain-containing protein 3, N-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    7.4e-06   26.2   4.6    1.4e-05   25.4   4.6    1.4  1  CAH8608449.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.6e-05   25.1   5.3    2.7e-05   24.4   5.3    1.3  1  CAH8628366.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.2e-05   24.7   5.3    3.7e-05   24.0   5.3    1.4  1  CAH8521553.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.6e-05   24.0   6.1      0.001   19.4   6.1    2.4  1  CAH8608433.1  unnamed protein product [Schistosoma intercalat\n",
            "    5.2e-05   23.5   6.1     0.0014   19.0   6.1    2.4  1  CAH8608418.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00013   22.2   5.3    0.00013   22.2   5.3    1.9  1  CAH8521511.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00013   22.2   5.3    0.00013   22.2   5.3    1.9  1  CAH8628337.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00014   22.1   4.6    0.00014   22.1   4.6    3.0  1  CAH8496701.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00016   22.0   7.3     0.0041   17.4   7.3    2.3  1  CAH8521570.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00016   21.9   7.3     0.0041   17.4   7.3    2.3  1  CAH8521538.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0002   21.6   6.3     0.0057   17.0   6.3    2.3  1  CAH8566095.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00023   21.4   7.3     0.0062   16.8   7.3    2.4  1  CAH8628351.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00033   21.0   7.3      0.008   16.5   7.3    2.4  1  CAH8628322.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00064   20.0   4.4    0.00064   20.0   4.4    2.3  2  CAH8483955.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00069   19.9   7.4      0.017   15.4   7.4    2.7  1  CAH8628285.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00077   19.8   7.4      0.016   15.5   7.4    2.5  1  CAH8628311.1  unnamed protein product [Schistosoma intercalat\n",
            "    0.00081   19.7   6.3      0.022   15.1   6.3    2.7  1  CAH8566054.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.001   19.4   9.1     0.0046   17.3   6.2    2.9  1  CAH8496708.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0017   18.7   3.7     0.0083   16.4   3.6    1.9  1  CAH8655637.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0032   17.8   9.6      0.018   15.3   7.4    2.7  1  CAH8521472.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0032   17.8   9.6      0.018   15.3   7.4    2.7  1  CAH8628294.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0036   17.6   8.4      0.023   15.0   6.3    2.7  1  CAH8566075.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0042   17.4  10.3      0.015   15.6   7.3    2.8  1  CAH8521489.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0078   16.5   3.1     0.0078   16.5   3.1    1.9  2  CAH8650437.1  unnamed protein product [Schistosoma intercalat\n",
            "  ------ inclusion threshold ------\n",
            "      0.012   15.9   6.3      0.012   15.9   6.3    3.8  1  CAH8496694.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.012   15.9   6.3      0.012   15.9   6.3    3.8  1  CAH8496687.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.013   15.8   3.1      0.013   15.8   3.1    2.3  2  CAH8650373.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.018   15.3   3.7       0.11   12.9   1.9    2.5  2  CAH8506104.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.019   15.3   5.7      0.019   15.3   5.7    3.2  2  CAH8483972.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.023   15.0   5.7      0.023   15.0   5.7    3.2  2  CAH8483938.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.023   15.0   4.1      0.091   13.1   1.8    2.5  1  CAH8506120.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.026   14.9   4.1       0.13   12.6   2.0    2.5  1  CAH8506073.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.029   14.7   4.3       0.12   12.7   2.1    2.5  1  CAH8506087.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.11   12.8   2.4       0.13   12.6   1.0    1.7  2  CAH8506135.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.21   12.0   8.9       0.13   12.6   3.2    2.8  2  CAH8650406.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.29   11.5   9.7       0.12   12.7   3.2    2.8  2  CAH8650421.1  unnamed protein product [Schistosoma intercalat\n",
            "       0.99    9.8   8.6       0.23   11.8   1.8    2.8  2  CAH8460993.1  unnamed protein product [Schistosoma intercalat\n",
            "        1.6    9.1  12.6       0.15   12.4   3.1    3.3  2  CAH8650391.1  unnamed protein product [Schistosoma intercalat\n",
            "        1.9    8.9  12.5       0.17   12.2   3.1    3.2  2  CAH8650356.1  unnamed protein product [Schistosoma intercalat\n",
            "        2.4    8.5   9.8       0.26   11.6   1.9    2.8  2  CAH8461025.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8608449.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   25.4   4.6   3.5e-08   1.4e-05       1     102 []      23     129 ..      23     129 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 25.4 bits;  conditional E-value: 3.5e-08\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + ++         ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8608449.1  23 MILNKRFVLWDDFESALKEFQKVWNYSFIASQTTYTRYIHTESRLLKDVRFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 117\n",
            "                   67899*****************999999988887777776665555555667999*****9886234444444445566777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8608449.1 118 YVIKSYNMLHNH 129\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8628366.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   24.4   5.3   7.1e-08   2.7e-05       1     102 []       1     107 [.       1     107 [. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 24.4 bits;  conditional E-value: 7.1e-08\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + ++         ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628366.1   1 MILNKRFVFWDDFESALKEFQKVWNYNFIASQTTYTRYIHTESRLLKDVRFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 95 \n",
            "                   67899*******************9999998888888777666555555667999*****9886234444444445566777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628366.1  96 YVIKSYNMLHNH 107\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8521553.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   24.0   5.3   9.7e-08   3.7e-05       1     102 []      23     129 ..      23     129 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 24.0 bits;  conditional E-value: 9.7e-08\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + ++         ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8521553.1  23 MILNKRFVFWDDFESALKEFQKVWNYNFIASQTTYTRYIHTESRLLKDVRFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 117\n",
            "                   67899*******************9999998888888777666555555667999*****9886234444444445566777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8521553.1 118 YVIKSYNMLHNH 129\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8608433.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.4   6.1   2.6e-06     0.001       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.4 bits;  conditional E-value: 2.6e-06\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8608433.1  23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........468999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8608433.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8608418.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.0   6.1   3.6e-06    0.0014       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.0 bits;  conditional E-value: 3.6e-06\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8608418.1  23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8608418.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8521511.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   22.2   5.3   3.4e-07   0.00013       1     102 []       1     107 [.       1     107 [. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 22.2 bits;  conditional E-value: 3.4e-07\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + ++         ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8521511.1   1 MILNKRFVFWDDFESALKEFQKVWNYNFIASQTTYTRYIHTESRLLKDVRFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 95 \n",
            "                   67899*******************9999998888888777666555555667999*****9886234444444445566777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8521511.1  96 YVIKSYNMLHNH 107\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8628337.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   22.2   5.3   3.4e-07   0.00013       1     102 []       1     107 [.       1     107 [. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 22.2 bits;  conditional E-value: 3.4e-07\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + ++         ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628337.1   1 MILNKRFVFWDDFESALKEFQKVWNYNFIASQTTYTRYIHTESRLLKDVRFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 95 \n",
            "                   67899*******************9999998888888777666555555667999*****9886234444444445566777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628337.1  96 YVIKSYNMLHNH 107\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8496701.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   22.1   4.6   3.6e-07   0.00014       1     102 []      23     129 ..      23     129 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 22.1 bits;  conditional E-value: 3.6e-07\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + ++         ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8496701.1  23 MILNKRFVLWDDFESALKEFQKVWNYSFIASQTTYTRYIHTESRLLKDVRFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 117\n",
            "                   67899*****************999999988887777776665555555667999*****9886234444444445566777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8496701.1 118 YVIKSYNMLHNH 129\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8521570.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.4   7.3   1.1e-05    0.0041       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.4 bits;  conditional E-value: 1.1e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8521570.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8521570.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8521538.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.4   7.3   1.1e-05    0.0041       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.4 bits;  conditional E-value: 1.1e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8521538.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8521538.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8566095.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.0   6.3   1.5e-05    0.0057       3     102 .]      23     117 ..      22     117 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.0 bits;  conditional E-value: 1.5e-05\n",
            "      ZSWIM3_N   3 lgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldrLv 92 \n",
            "                   +++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++   v\n",
            "  CAH8566095.1  23 MNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG--YV 107\n",
            "                   57899**********************9999887643..........467999999999875234444444555567777**********998..68 PP\n",
            "\n",
            "      ZSWIM3_N  93 ikelnekHnh 102\n",
            "                   ik+ n  Hnh\n",
            "  CAH8566095.1 108 IKSYNMLHNH 117\n",
            "                   9999999998 PP\n",
            "\n",
            ">> CAH8628351.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.8   7.3   1.6e-05    0.0062       1     102 []       1      97 [.       1      97 [. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.8 bits;  conditional E-value: 1.6e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628351.1   1 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 85 \n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628351.1  86 YVIKSYNMLHNH 97 \n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8628322.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.5   7.3   2.1e-05     0.008       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.5 bits;  conditional E-value: 2.1e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628322.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628322.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8483955.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   20.0   4.4   1.7e-06   0.00064       1     102 []      23     129 ..      23     129 .. 0.76\n",
            "   2 ?   -1.0   1.1       5.8   2.2e+03      52      72 ..     242     264 ..     227     310 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 20.0 bits;  conditional E-value: 1.7e-06\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyq..vkfvCvrgq.......evrkkkkesklCeAylvLqasekl 88 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + +  +  + +dv + +  v f C+ g+       +vr+k+ + + C  ++  + +e++\n",
            "  CAH8483955.1  23 MILNKRFVLWDDFESALKEFQKVWNYNFIASQTTYTRYIHT--ESRLLKDVRFKNlfVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG 117\n",
            "                   67899*****************9999998877766666544..4455588888663379***9886234444444445566777**********998 PP\n",
            "\n",
            "      ZSWIM3_N  89 drLvikelnekHnh 102\n",
            "                      vik+ n  Hnh\n",
            "  CAH8483955.1 118 --YVIKSYNMLHNH 129\n",
            "                   ..689999999998 PP\n",
            "\n",
            "  == domain 2  score: -1.0 bits;  conditional E-value: 5.8\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkkke 72 \n",
            "                   +++  ++ C +     v++++ +\n",
            "  CAH8483955.1 242 TFMMCRYPCRHMllAYVKRRNLT 264\n",
            "                   58899999998853345555432 PP\n",
            "\n",
            ">> CAH8628285.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.4   7.4   4.4e-05     0.017       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.4 bits;  conditional E-value: 4.4e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628285.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628285.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8628311.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.5   7.4   4.2e-05     0.016       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.5 bits;  conditional E-value: 4.2e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628311.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628311.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8566054.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.1   6.3   5.7e-05     0.022       3     102 .]      23     117 ..      22     117 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.1 bits;  conditional E-value: 5.7e-05\n",
            "      ZSWIM3_N   3 lgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldrLv 92 \n",
            "                   +++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++   v\n",
            "  CAH8566054.1  23 MNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG--YV 107\n",
            "                   57899**********************9999887643..........467999999999875234444444555567777**********998..68 PP\n",
            "\n",
            "      ZSWIM3_N  93 ikelnekHnh 102\n",
            "                   ik+ n  Hnh\n",
            "  CAH8566054.1 108 IKSYNMLHNH 117\n",
            "                   9999999998 PP\n",
            "\n",
            ">> CAH8496708.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.3   6.2   1.2e-05    0.0046       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.3 bits;  conditional E-value: 1.2e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8496708.1  23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8496708.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8655637.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.4   3.6   2.2e-05    0.0083       3      67 ..      17      72 ..      15      98 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.4 bits;  conditional E-value: 2.2e-05\n",
            "      ZSWIM3_N  3 lgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgqevr 67\n",
            "                  l   F+++++F++++ke +k +++k+++  ck    +n+        ++ky+ v++vC++g++  \n",
            "  CAH8655637.1 17 LSRRFTSWDNFETCLKEVHKYTHTKYVMSVCK----KNS-----DDPTLKYFFVRYVCTYGRKRS 72\n",
            "                  6789***********************98885....222.....245789*********998653 PP\n",
            "\n",
            ">> CAH8521472.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.3   7.4   4.7e-05     0.018       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.3 bits;  conditional E-value: 4.7e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8521472.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8521472.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8628294.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.3   7.4   4.7e-05     0.018       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.3 bits;  conditional E-value: 4.7e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8628294.1  23 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8628294.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8566075.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.0   6.3   5.9e-05     0.023       3     102 .]      23     117 ..      22     117 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.0 bits;  conditional E-value: 5.9e-05\n",
            "      ZSWIM3_N   3 lgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldrLv 92 \n",
            "                   +++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++   v\n",
            "  CAH8566075.1  23 MNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG--YV 107\n",
            "                   57899**********************9999887643..........467999999999875234444444555567777**********998..68 PP\n",
            "\n",
            "      ZSWIM3_N  93 ikelnekHnh 102\n",
            "                   ik+ n  Hnh\n",
            "  CAH8566075.1 108 IKSYNMLHNH 117\n",
            "                   9999999998 PP\n",
            "\n",
            ">> CAH8521489.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.6   7.3   3.9e-05     0.015       1     102 []       1      97 [.       1      97 [. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.6 bits;  conditional E-value: 3.9e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8521489.1   1 MILNKRFVFWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 85 \n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8521489.1  86 YVIKSYNMLHNH 97 \n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8650437.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   16.5   3.1     2e-05    0.0078       1     102 []      23     122 ..      23     122 .. 0.75\n",
            "   2 ?   -1.5   0.5       8.4   3.2e+03      52      70 ..     235     255 ..     220     260 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 16.5 bits;  conditional E-value: 2e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyq..vkfvCvrgqevrkkkkesklCeAylvLqasekldrLvike 95 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + +  +  + +dv + +  v f C+ g++ +    ++++   ++  + +e++   vik+\n",
            "  CAH8650437.1  23 MILNKRFVLWDDFESALKEFQKVWNYNFIASQTTYTRYIHT--ESRLLKDVRFKNlfVSFNCTFGHKRKSEGLKMRQKSSKFRVRLEEQG--YVIKS 115\n",
            "                   67899*****************9999998877766666544..445558888876337999*9987665555555556667777777766..79999 PP\n",
            "\n",
            "      ZSWIM3_N  96 lnekHnh 102\n",
            "                    n  Hnh\n",
            "  CAH8650437.1 116 YNMLHNH 122\n",
            "                   9999998 PP\n",
            "\n",
            "  == domain 2  score: -1.5 bits;  conditional E-value: 8.4\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkk 70 \n",
            "                   +++  ++ C +     v++++\n",
            "  CAH8650437.1 235 TFMMCRYPCRHMllAYVKRRN 255\n",
            "                   699999999998533455554 PP\n",
            "\n",
            ">> CAH8496694.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   15.9   6.3   3.1e-05     0.012       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.9 bits;  conditional E-value: 3.1e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8496694.1  23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8496694.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8496687.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   15.9   6.3   3.2e-05     0.012       1     102 []      23     119 ..      23     119 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.9 bits;  conditional E-value: 3.2e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k              ky+ v f C+ g+       +vr+k+ + + C+ ++  + +e++  \n",
            "  CAH8496687.1  23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV----------RFKYLFVSFNCTFGHkrkseglKVRQKSSKFRNCQSKFRVRLEEQG-- 107\n",
            "                   67899************************9999887643..........467999999999875234444444555567777**********998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8496687.1 108 YVIKSYNMLHNH 119\n",
            "                   689999999998 PP\n",
            "\n",
            ">> CAH8650373.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   15.8   3.1   3.3e-05     0.013       1     102 []      23     122 ..      23     122 .. 0.75\n",
            "   2 ?   -1.0   1.1       5.9   2.2e+03      52      72 ..     235     257 ..     221     287 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.8 bits;  conditional E-value: 3.3e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyq..vkfvCvrgqevrkkkkesklCeAylvLqasekldrLvike 95 \n",
            "                   m l++ F  +++F++ +ke++k  + +f+  +++  + + +  +  + +dv + +  v f C+ g++ +    ++++   ++  + +e++   vik+\n",
            "  CAH8650373.1  23 MILNKRFVLWDDFESALKEFQKVWNYNFIASQTTYTRYIHT--ESRLLKDVRFKNlfVSFNCTFGHKRKSEGLKMRQKSSKFRVRLEEQG--YVIKS 115\n",
            "                   67899*****************9999998877766666544..445558888876337999*9987665555555556667777777766..79999 PP\n",
            "\n",
            "      ZSWIM3_N  96 lnekHnh 102\n",
            "                    n  Hnh\n",
            "  CAH8650373.1 116 YNMLHNH 122\n",
            "                   9999998 PP\n",
            "\n",
            "  == domain 2  score: -1.0 bits;  conditional E-value: 5.9\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkkke 72 \n",
            "                   +++  ++ C +     v++++ +\n",
            "  CAH8650373.1 235 TFMMCRYPCRHMllAYVKRRNLT 257\n",
            "                   68999999999853345666433 PP\n",
            "\n",
            ">> CAH8506104.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.9   1.9   0.00028      0.11      15     102 .]      14      96 ..       1      96 [. 0.64\n",
            "   2 ?   -1.6   0.0         9   3.5e+03      27      47 ..     121     145 ..     109     160 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.9 bits;  conditional E-value: 0.00028\n",
            "      ZSWIM3_N  15 klfkeykkekklkfsikdcksvkfankklgtairedv..kyyqvkfvCvrgqevrkk.....kkeskl..CeAylvLqasekldrLvikelnekHnh 102\n",
            "                   + +k ++k ++++++  +++ +            +dv  ky+ v f C+ g++ +       kk sk   C+ ++  ++  ++++ vik+ n  Hnh\n",
            "  CAH8506104.1  14 NALKDFHKMTHTHYVHAESRLL------------KDVryKYLFVVFRCTFGRKRKSEglnvnKKPSKFlnCQSMF--RVRLEGQQYVIKSYNMSHNH 96 \n",
            "                   5555555555555554444444............34411577799***999987776887754555554577766..55668999***********9 PP\n",
            "\n",
            "  == domain 2  score: -1.6 bits;  conditional E-value: 9\n",
            "      ZSWIM3_N  27 kfsikdcksv....kfankklgtai 47 \n",
            "                   k +i  c+s     +++++++g+++\n",
            "  CAH8506104.1 121 KPVILHCQSTdeviESIKERTGKQV 145\n",
            "                   4455555554222244445554444 PP\n",
            "\n",
            ">> CAH8483972.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   15.3   5.7   5.1e-05     0.019       1     102 []       1      97 [.       1      97 [. 0.75\n",
            "   2 ?   -0.9   1.0       5.6   2.1e+03      52      72 ..     210     232 ..     200     264 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.3 bits;  conditional E-value: 5.1e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k    k+            v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8483972.1   1 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDVRFKN-LF---------VSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 85 \n",
            "                   67899*************************9999887654432.22.........445554443112233356666777788*********9998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8483972.1  86 YVIKSYNMLHNH 97 \n",
            "                   589999999998 PP\n",
            "\n",
            "  == domain 2  score: -0.9 bits;  conditional E-value: 5.6\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkkke 72 \n",
            "                   +++  ++ C +     v++++ +\n",
            "  CAH8483972.1 210 TFMMCRYPCRHMllAYVKRRNLT 232\n",
            "                   68999999999853345565433 PP\n",
            "\n",
            ">> CAH8483938.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   15.0   5.7   5.9e-05     0.023       1     102 []      23     119 ..      23     119 .. 0.75\n",
            "   2 ?   -0.9   0.9       5.3     2e+03      52      72 ..     232     254 ..     220     286 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.0 bits;  conditional E-value: 5.9e-05\n",
            "      ZSWIM3_N   1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgq.......evrkkkkesklCeAylvLqasekldr 90 \n",
            "                   m l++ F  +++F++ +ke++k+++++++  +++ +k    k+            v f C+ g+       +vr+k+ + + C  ++  + +e++  \n",
            "  CAH8483938.1  23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDVRFKN-LF---------VSFNCTFGHkrkseglKVRQKSSKFRNCRSKFRVRLEEQG-- 107\n",
            "                   67899*************************9999887654432.22.........445554443112233356666777788*********9998.. PP\n",
            "\n",
            "      ZSWIM3_N  91 LvikelnekHnh 102\n",
            "                    vik+ n  Hnh\n",
            "  CAH8483938.1 108 YVIKSYNMLHNH 119\n",
            "                   589999999998 PP\n",
            "\n",
            "  == domain 2  score: -0.9 bits;  conditional E-value: 5.3\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkkke 72 \n",
            "                   +++  ++ C +     v++++ +\n",
            "  CAH8483938.1 232 TFMMCRYPCRHMllAYVKRRNLT 254\n",
            "                   68999999999853345565433 PP\n",
            "\n",
            ">> CAH8506120.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.1   1.8   0.00024     0.091      14     102 .]      13      96 ..       1      96 [. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.1 bits;  conditional E-value: 0.00024\n",
            "      ZSWIM3_N  14 kklfkeykkekklkfsikdcksvkfankklgtairedv..kyyqvkfvCvrgqevrkk.....kkeskl..CeAylvLqasekldrLvikelnekHn 101\n",
            "                   ++ +k ++k ++++++  +++ +            +dv  ky+ v f C+ g++ +       kk sk   C+ ++  ++  ++++ vik+ n  Hn\n",
            "  CAH8506120.1  13 DNALKDFHKMTHTHYVHAESRLL------------KDVryKYLFVVFRCTFGRKRKSEglnvnKKPSKFlnCQSMF--RVRLEGQQYVIKSYNMSHN 95 \n",
            "                   55555555555555554444444............34411577799***999987776887754555554577766..55668999*********** PP\n",
            "\n",
            "      ZSWIM3_N 102 h 102\n",
            "                   h\n",
            "  CAH8506120.1  96 H 96 \n",
            "                   9 PP\n",
            "\n",
            ">> CAH8506073.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   2.0   0.00033      0.13       5     102 .]      27     119 ..      23     119 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 0.00033\n",
            "      ZSWIM3_N   5 stFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgqevrkk.....kkeskl..CeAylvLqasekldrLvik 94 \n",
            "                   + F ++ e ++ +k ++k ++++++  +++ +k              ky+ v f C+ g++ +       kk sk   C+ ++  ++  ++++ vik\n",
            "  CAH8506073.1  27 KRFGSWIELDNALKDFHKMTHTHYVHAESRLLKDV----------RYKYLFVVFRCTFGRKRKSEglnvnKKPSKFlnCQSMF--RVRLEGQQYVIK 111\n",
            "                   55666666666666666666666665555544422..........23577899***999987776787754555554577766..55668999**** PP\n",
            "\n",
            "      ZSWIM3_N  95 elnekHnh 102\n",
            "                   + n  Hnh\n",
            "  CAH8506073.1 112 SYNMSHNH 119\n",
            "                   *******9 PP\n",
            "\n",
            ">> CAH8506087.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.7   2.1   0.00031      0.12       5     102 .]      27     119 ..      23     119 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.7 bits;  conditional E-value: 0.00031\n",
            "      ZSWIM3_N   5 stFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgqevrkk.....kkeskl..CeAylvLqasekldrLvik 94 \n",
            "                   + F ++ e ++ +k ++k ++++++  +++ +k              ky+ v f C+ g++ +       kk sk   C+ ++  ++  ++++ vik\n",
            "  CAH8506087.1  27 KRFGSWIELDNALKDFHKMTHTHYVHAESRLLKDV----------RYKYLFVVFRCTFGRKRKSEglnvnKKPSKFlnCQSMF--RVRLEGQQYVIK 111\n",
            "                   55666666666666666666666665555544422..........23577899***999987776787754555554577766..55668999**** PP\n",
            "\n",
            "      ZSWIM3_N  95 elnekHnh 102\n",
            "                   + n  Hnh\n",
            "  CAH8506087.1 112 SYNMSHNH 119\n",
            "                   *******9 PP\n",
            "\n",
            ">> CAH8506135.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   1.0   0.00033      0.13      48     102 .]      18      79 ..       4      79 .. 0.73\n",
            "   2 ?   -1.3   0.0       7.3   2.8e+03      27      47 ..     104     128 ..      92     144 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 0.00033\n",
            "      ZSWIM3_N  48 redvk..yyqvkfvCvrgqevrkk.....kkeskl..CeAylvLqasekldrLvikelnekHnh 102\n",
            "                    +dv   y+ v f C+ g++ +       kk sk   C+ ++  ++  ++++ vik+ n  Hnh\n",
            "  CAH8506135.1  18 LKDVRykYLFVVFRCTFGRKRKSEglnvnKKPSKFlnCQSMF--RVRLEGQQYVIKSYNMSHNH 79 \n",
            "                   466652267789****99987777888854555554577766..55668999***********9 PP\n",
            "\n",
            "  == domain 2  score: -1.3 bits;  conditional E-value: 7.3\n",
            "      ZSWIM3_N  27 kfsikdcksv....kfankklgtai 47 \n",
            "                   k +i  c+s     +++++++g+++\n",
            "  CAH8506135.1 104 KPVILHCQSTdeviESIKERTGKQV 128\n",
            "                   4455555554222244445554444 PP\n",
            "\n",
            ">> CAH8650406.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   3.2   0.00034      0.13       1      39 [.      23      61 ..      23     112 .. 0.67\n",
            "   2 ?   -1.5   0.3       8.5   3.3e+03      52      69 ..     225     244 ..     210     252 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 0.00034\n",
            "      ZSWIM3_N  1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfa 39\n",
            "                  m l++ F  +++F++ +ke++k+++++++  +++ +k  \n",
            "  CAH8650406.1 23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV 61\n",
            "                  67899************************9999876543 PP\n",
            "\n",
            "  == domain 2  score: -1.5 bits;  conditional E-value: 8.5\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkk 69 \n",
            "                   +++  ++ C +     v+++\n",
            "  CAH8650406.1 225 TFMMCRYPCRHMllAYVKRR 244\n",
            "                   69999999999842334444 PP\n",
            "\n",
            ">> CAH8650421.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.7   3.2   0.00031      0.12       1      39 [.      23      61 ..      23     112 .. 0.67\n",
            "   2 ?   -1.4   0.5       7.9     3e+03      52      70 ..     225     245 ..     210     250 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.7 bits;  conditional E-value: 0.00031\n",
            "      ZSWIM3_N  1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfa 39\n",
            "                  m l++ F  +++F++ +ke++k+++++++  +++ +k  \n",
            "  CAH8650421.1 23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV 61\n",
            "                  67899************************9999876543 PP\n",
            "\n",
            "  == domain 2  score: -1.4 bits;  conditional E-value: 7.9\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkk 70 \n",
            "                   +++  ++ C +     v++++\n",
            "  CAH8650421.1 225 TFMMCRYPCRHMllAYVKRRN 245\n",
            "                   699999999998533455554 PP\n",
            "\n",
            ">> CAH8460993.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.8   1.8   0.00061      0.23       5     102 .]      27     118 ..      23     118 .. 0.63\n",
            "   2 ?   -1.3   0.7       7.5   2.9e+03      70      80 ..     201     211 ..     143     232 .. 0.58\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.8 bits;  conditional E-value: 0.00061\n",
            "      ZSWIM3_N   5 stFktfeeFkklfkeykkekklkfsikdcksvkfankklgtairedvkyyqvkfvCvrgqevrkk....kkeskl..CeAylvLqasekldrLvike 95 \n",
            "                   + F ++ e ++ +k ++k ++++++  +++ +k              ky+ v f C+ g++ +      kk sk   C+ ++   +  ++++ vik+\n",
            "  CAH8460993.1  27 KRFGSWIELDNALKDFHKMTHTHYVHAESRLLKDV----------RYKYLFVVFRCTFGRKRKSGlnvnKKPSKFlnCQSMFR--VRLEGQQYVIKS 111\n",
            "                   55666666666666666666666665555544422..........23577789999999866544666444555444777765..5567999***** PP\n",
            "\n",
            "      ZSWIM3_N  96 lnekHnh 102\n",
            "                    n  Hnh\n",
            "  CAH8460993.1 112 YNMSHNH 118\n",
            "                   ******9 PP\n",
            "\n",
            "  == domain 2  score: -1.3 bits;  conditional E-value: 7.5\n",
            "      ZSWIM3_N  70 kkesklCeAyl 80 \n",
            "                   ++ +++C+ ++\n",
            "  CAH8460993.1 201 QRLQRMCKQQI 211\n",
            "                   23333444444 PP\n",
            "\n",
            ">> CAH8650391.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.4   3.1    0.0004      0.15       1      39 [.       1      39 [.       1      90 [. 0.68\n",
            "   2 ?   -1.0   1.4       5.7   2.2e+03      52      72 ..     203     225 ..     189     269 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.4 bits;  conditional E-value: 0.0004\n",
            "      ZSWIM3_N  1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfa 39\n",
            "                  m l++ F  +++F++ +ke++k+++++++  +++ +k  \n",
            "  CAH8650391.1  1 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV 39\n",
            "                  67899************************9999876543 PP\n",
            "\n",
            "  == domain 2  score: -1.0 bits;  conditional E-value: 5.7\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkkke 72 \n",
            "                   +++  ++ C +     v++++ +\n",
            "  CAH8650391.1 203 TFMMCRYPCRHMllAYVKRRNLT 225\n",
            "                   58999999998853345555432 PP\n",
            "\n",
            ">> CAH8650356.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.2   3.1   0.00045      0.17       1      39 [.      23      61 ..      23     112 .. 0.68\n",
            "   2 ?   -1.0   1.1         6   2.3e+03      52      72 ..     225     247 ..     213     277 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.2 bits;  conditional E-value: 0.00045\n",
            "      ZSWIM3_N  1 melgstFktfeeFkklfkeykkekklkfsikdcksvkfa 39\n",
            "                  m l++ F  +++F++ +ke++k+++++++  +++ +k  \n",
            "  CAH8650356.1 23 MILNKRFVLWDDFESALKEFQKTTYTRYIHTESRLLKDV 61\n",
            "                  67899************************9999876543 PP\n",
            "\n",
            "  == domain 2  score: -1.0 bits;  conditional E-value: 6\n",
            "      ZSWIM3_N  52 kyyqvkfvCvrg..qevrkkkke 72 \n",
            "                   +++  ++ C +     v++++ +\n",
            "  CAH8650356.1 225 TFMMCRYPCRHMllAYVKRRNLT 247\n",
            "                   68999999999853345666433 PP\n",
            "\n",
            ">> CAH8461025.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.6   1.9   0.00067      0.26      17     102 .]      16      95 ..       1      95 [. 0.61\n",
            "   2 ?   -1.3   1.2       7.4   2.8e+03      70      79 ..     178     187 ..     113     210 .. 0.58\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.6 bits;  conditional E-value: 0.00067\n",
            "      ZSWIM3_N  17 fkeykkekklkfsikdcksvkfankklgtairedv..kyyqvkfvCvrgqevrkk....kkeskl..CeAylvLqasekldrLvikelnekHnh 102\n",
            "                   +k ++k ++++++  +++ +            +dv  ky+ v f C+ g++ +      kk sk   C+ ++   +  ++++ vik+ n  Hnh\n",
            "  CAH8461025.1  16 LKDFHKMTHTHYVHAESRLL------------KDVryKYLFVVFRCTFGRKRKSGlnvnKKPSKFlnCQSMFR--VRLEGQQYVIKSYNMSHNH 95 \n",
            "                   55555555555554444443............34411577789999999866544666444555444777765..5567999***********9 PP\n",
            "\n",
            "  == domain 2  score: -1.3 bits;  conditional E-value: 7.4\n",
            "      ZSWIM3_N  70 kkesklCeAy 79 \n",
            "                   ++ +++C+ +\n",
            "  CAH8461025.1 178 QRLQRMCKQQ 187\n",
            "                   2333334444 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (102 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       823  (0.0536156); expected 307.0 (0.02)\n",
            "Passed bias filter:                      306  (0.0199349); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        55  (0.00358306); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                        40  (0.00260586); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):              40  [number of targets reported over threshold]\n",
            "# CPU time: 0.33u 0.01s 00:00:00.34 Elapsed: 00:00:00.40\n",
            "# Mc/sec: 2646.95\n",
            "//\n",
            "Query:       ZSWIM4-8_C  [M=173]\n",
            "Accession:   PF21055.2\n",
            "Description: ZSWIM4-8, C-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    1.8e-45  154.6   0.0    2.2e-22   79.4   0.0    2.8  2  CAH8610270.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.1e-10   41.4   4.5    8.2e-05   22.3   0.1    3.6  4  CAH8635152.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8610270.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   72.1   0.0   5.4e-24   4.1e-20       1      92 [.    2063    2154 ..    2063    2183 .. 0.89\n",
            "   2 !   79.4   0.0   2.9e-26   2.2e-22      96     173 .]    2220    2304 ..    2198    2304 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 72.1 bits;  conditional E-value: 5.4e-24\n",
            "    ZSWIM4-8_C    1 qlftiarymekrgyllrAyrvallaletlalrvnddnhpaiedvkWllalaksLGkseLqklivlvvksvqcafvLsdilleaalslarsnp 92  \n",
            "                    qlf++arym+++g + rA+  al a + + l++ +++hp+++dv W++ l ++LG   Lq++++ v+++++c+++L+dil+++       + \n",
            "  CAH8610270.1 2063 QLFSLARYMDNCGWTWRAFPFALHATRLFVLSTTQESHPVGNDVLWACLLGHRLGPGALQEILNHVIRNIHCPTLLTDILHRCRSIPCGIHV 2154\n",
            "                    89*********************************************************************************876655444 PP\n",
            "\n",
            "  == domain 2  score: 79.4 bits;  conditional E-value: 2.9e-26\n",
            "    ZSWIM4-8_C   96 s..a....lslrsaplkqLleaaiqayvqtihsrLahisprhYedFieflskAretFll.apdGslqFkellqslkrlkkgkkkL 173 \n",
            "                       +    lsl+++plk Ll+aai+ +v ++h rLa+ispr+Y++F++fl++Ar+tF+l  pdG lqF++l++ l+++++gk+kL\n",
            "  CAH8610270.1 2220 VsgCtgkmLSLDRPPLKGLLDAAINGFVMATHMRLANISPRQYAEFVDFLTRARDTFHLlRPDGPLQFRSLVECLRQTYRGKRKL 2304\n",
            "                    1342444589***********************************************85279*********************98 PP\n",
            "\n",
            ">> CAH8635152.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -4.0   0.1       1.3   9.7e+03      63      89 ..    1163    1188 ..    1158    1216 .. 0.65\n",
            "   2 ?   -0.8   0.1      0.13   9.8e+02      94     166 ..    1947    2024 ..    1920    2028 .. 0.75\n",
            "   3 !   22.3   0.1   1.1e-08   8.2e-05      25      90 ..    2700    2765 ..    2687    2791 .. 0.83\n",
            "   4 !   18.5   0.0   1.5e-07    0.0012     101     170 ..    3033    3106 ..    2998    3109 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -4.0 bits;  conditional E-value: 1.3\n",
            "    ZSWIM4-8_C   63 ivlvvksvqcafvLsdilleaalslar 89  \n",
            "                    + l + +++c+ v+s i  ea + +++\n",
            "  CAH8635152.1 1163 CFLPI-HLACPHVISSIRSEAKQLASN 1188\n",
            "                    55544.578999999999988877744 PP\n",
            "\n",
            "  == domain 2  score: -0.8 bits;  conditional E-value: 0.13\n",
            "    ZSWIM4-8_C   94 lvsalslrsaplkqLleaaiqayvqtihsrLahisprhYedFieflsk.....AretFllapdGslqFkellqslkrl 166 \n",
            "                    l+ ++ l+ + l + le++ +   +t+   L  +  + Y+   ++l +     Ar  ++l +  +  F++ l+sl  +\n",
            "  CAH8635152.1 1947 LSLCTALKPQELYRALEQCGEHSSNTLERGLLCVERSAYQSSHDILPEiyffiARSWYTLYHSTNKEFEQALNSLESF 2024\n",
            "                    223678888999999999999999999999999999999988888754344449*****************9999765 PP\n",
            "\n",
            "  == domain 3  score: 22.3 bits;  conditional E-value: 1.1e-08\n",
            "    ZSWIM4-8_C   25 aletlalrvnddnhpaiedvkWllalaksLGkseLqklivlvvksvqcafvLsdilleaalslars 90  \n",
            "                    + +t + ++n ++++++  + W+l +a sLG   +++ + lv++ v+c+ ++++i  ++  ++ + \n",
            "  CAH8635152.1 2700 STTTTTTNTNTNSKSCSAHILWTLDVASSLGPLAVNEYCDLVLQCVNCPLLMEQITSKVIDYFEKY 2765\n",
            "                    556677788999*******************************************99988777553 PP\n",
            "\n",
            "  == domain 4  score: 18.5 bits;  conditional E-value: 1.5e-07\n",
            "    ZSWIM4-8_C  101 rsaplkqLleaaiqayvqtihsrLahi..sprhYedFieflskAretFllap..dGslqFkellqslkrlkkgk 170 \n",
            "                     ++++++L++ +++ + + i +rL+ i  s++++++F++++ kA ++ l +p  d  l++++ll +++r +k+ \n",
            "  CAH8635152.1 3033 GRDVIENLINHTHTLFQKYIDQRLQYIgqSQAEWDEFVDLILKAYNVHLSMPaiDCGLHWNNLLTRIRRHQKCS 3106\n",
            "                    56789*********************9667789*************997655348899**************96 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (173 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       502  (0.0327036); expected 307.0 (0.02)\n",
            "Passed bias filter:                      326  (0.0212378); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        23  (0.00149837); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000130293); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.24u 0.00s 00:00:00.24 Elapsed: 00:00:00.20\n",
            "# Mc/sec: 8635.73\n",
            "//\n",
            "Query:       ZT_dimer  [M=79]\n",
            "Accession:   PF16916.10\n",
            "Description: Dimerisation domain of Zinc Transporter\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    2.9e-09   36.7   0.0    7.4e-09   35.4   0.0    1.7  1  CAH8532776.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-09   36.7   0.0    7.4e-09   35.4   0.0    1.7  1  CAH8532787.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0045   16.9   5.8     0.0055   16.6   0.1    2.3  2  CAH8547921.1  unnamed protein product [Schistosoma intercalat\n",
            "     0.0051   16.7   5.7     0.0061   16.4   0.1    2.3  2  CAH8547888.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8532776.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   35.4   0.0   1.9e-12   7.4e-09       5      77 ..     466     538 ..     462     539 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 35.4 bits;  conditional E-value: 1.9e-12\n",
            "                   HHHHHHHHHTT.-TT-EEEEEEEEEECTTEEEEEEEEEE-TTS-HHHHHHHHHHHHHHHHHH-TTEEEEEEEEE CS\n",
            "      ZT_dimer   5 eeeeIreiies.vpgvkgvhdlrtRksGnkifvdlhievdgdlsvkeaHeiadevedrireefpevedvliHve 77 \n",
            "                    +++I  ++++ +p +++ + +r+ ++G++ +v+++i++++++ +keaH+i + ++++++ +++ ve+ ++H++\n",
            "  CAH8532776.1 466 FIQKIAFVCLNyHPLIEQLDTIRAFHFGENFLVEVDIVLPKEMCLKEAHDIGEGLQKKLE-KLETVERAFVHLD 538\n",
            "                   678888888776999********************************************6.788********87 PP\n",
            "\n",
            ">> CAH8532787.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   35.4   0.0   1.9e-12   7.4e-09       5      77 ..     466     538 ..     462     539 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 35.4 bits;  conditional E-value: 1.9e-12\n",
            "                   HHHHHHHHHTT.-TT-EEEEEEEEEECTTEEEEEEEEEE-TTS-HHHHHHHHHHHHHHHHHH-TTEEEEEEEEE CS\n",
            "      ZT_dimer   5 eeeeIreiies.vpgvkgvhdlrtRksGnkifvdlhievdgdlsvkeaHeiadevedrireefpevedvliHve 77 \n",
            "                    +++I  ++++ +p +++ + +r+ ++G++ +v+++i++++++ +keaH+i + ++++++ +++ ve+ ++H++\n",
            "  CAH8532787.1 466 FIQKIAFVCLNyHPLIEQLDTIRAFHFGENFLVEVDIVLPKEMCLKEAHDIGEGLQKKLE-KLETVERAFVHLD 538\n",
            "                   678888888776999********************************************6.788********87 PP\n",
            "\n",
            ">> CAH8547921.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.9   1.7     0.055   2.1e+02      50      76 ..     199     225 ..     185     226 .. 0.76\n",
            "   2 !   16.6   0.1   1.4e-06    0.0055       7      77 ..     289     356 ..     287     358 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.9 bits;  conditional E-value: 0.055\n",
            "                   HHHHHHHHHHHHHHHH-TTEEEEEEEE CS\n",
            "      ZT_dimer  50 eaHeiadevedrireefpevedvliHv 76 \n",
            "                   ++Hei ++++++  +    v+  +iHv\n",
            "  CAH8547921.1 199 HSHEINHDIHKEFNRPNITVRAAFIHV 225\n",
            "                   688888888888887776777777776 PP\n",
            "\n",
            "  == domain 2  score: 16.6 bits;  conditional E-value: 1.4e-06\n",
            "                   HHHHHHHTT-TT-EEEEEEEEEECT.TEEEEEEEEEE-TTS-HHHHHHHHHHHHHHHHHH-TTEEEEEEEEE CS\n",
            "      ZT_dimer   7 eeIreiiesvpgvkgvhdlrtRksG.nkifvdlhievdgdlsvkeaHeiadevedrireefpevedvliHve 77 \n",
            "                   +e+++ +++++gvk++h+l++ +   nk++v++h+ +++d   + a ei ++  + +++++   +dv+i +e\n",
            "  CAH8547921.1 289 NEVKNSLMNINGVKEIHNLHMWSLTtNKTAVSVHLAIEND---SNAQEILKQANHLLKQRYLA-HDVTIQLE 356\n",
            "                   578999999*************7666**************...9*****************66.89***987 PP\n",
            "\n",
            ">> CAH8547888.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.8   1.7     0.059   2.3e+02      50      76 ..     223     249 ..     209     250 .. 0.76\n",
            "   2 !   16.4   0.1   1.6e-06    0.0061       7      77 ..     313     380 ..     311     382 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.8 bits;  conditional E-value: 0.059\n",
            "                   HHHHHHHHHHHHHHHH-TTEEEEEEEE CS\n",
            "      ZT_dimer  50 eaHeiadevedrireefpevedvliHv 76 \n",
            "                   ++Hei ++++++  +    v+  +iHv\n",
            "  CAH8547888.1 223 HSHEINHDIHKEFNRPNITVRAAFIHV 249\n",
            "                   688888888888887776777777776 PP\n",
            "\n",
            "  == domain 2  score: 16.4 bits;  conditional E-value: 1.6e-06\n",
            "                   HHHHHHHTT-TT-EEEEEEEEEECT.TEEEEEEEEEE-TTS-HHHHHHHHHHHHHHHHHH-TTEEEEEEEEE CS\n",
            "      ZT_dimer   7 eeIreiiesvpgvkgvhdlrtRksG.nkifvdlhievdgdlsvkeaHeiadevedrireefpevedvliHve 77 \n",
            "                   +e+++ +++++gvk++h+l++ +   nk++v++h+ +++d   + a ei ++  + +++++   +dv+i +e\n",
            "  CAH8547888.1 313 NEVKNSLMNINGVKEIHNLHMWSLTtNKTAVSVHLAIEND---SNAQEILKQANHLLKQRYLA-HDVTIQLE 380\n",
            "                   578999999*************7666**************...9*****************66.89***987 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (79 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       488  (0.0317915); expected 307.0 (0.02)\n",
            "Passed bias filter:                      376  (0.0244951); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        20  (0.00130293); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         4  (0.000260586); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               4  [number of targets reported over threshold]\n",
            "# CPU time: 0.19u 0.01s 00:00:00.20 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 4774.71\n",
            "//\n",
            "Query:       ZU5  [M=97]\n",
            "Accession:   PF00791.25\n",
            "Description: ZU5 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    3.5e-36  123.1   0.5    1.7e-27   95.3   0.0    2.5  2  CAH8579544.1  unnamed protein product [Schistosoma intercalat\n",
            "    5.1e-36  122.6   0.5    2.1e-27   95.0   0.0    2.5  2  CAH8579530.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.2e-27   95.8   0.0    1.4e-18   66.7   0.0    2.6  2  CAH8618401.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.2e-27   95.8   0.0    1.3e-18   66.8   0.0    2.5  2  CAH8618414.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.3e-27   95.7   0.0    1.4e-18   66.7   0.0    2.6  2  CAH8618426.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.5e-27   95.5   0.0    1.7e-18   66.5   0.0    2.7  2  CAH8618387.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.8e-27   95.2   0.0    1.7e-18   66.4   0.0    2.7  2  CAH8618375.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.8e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618350.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.8e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618209.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.8e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618274.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618262.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618363.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618222.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-27   95.2   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618116.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-27   95.1   0.0    1.9e-18   66.3   0.0    2.8  2  CAH8618288.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.3e-27   94.9   0.0    2.2e-18   66.1   0.0    2.8  2  CAH8618249.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.3e-27   94.9   0.0    2.2e-18   66.1   0.0    2.8  2  CAH8618327.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.4e-27   94.8   0.0    2.2e-18   66.1   0.0    2.8  2  CAH8618194.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.4e-27   94.8   0.0    2.2e-18   66.1   0.0    2.8  2  CAH8618338.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-27   94.6   0.0    2.3e-18   66.0   0.0    2.7  2  CAH8618185.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-27   94.5   0.0    2.4e-18   66.0   0.0    2.8  2  CAH8618235.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.9e-27   94.5   0.0    2.3e-18   66.0   0.0    2.7  2  CAH8618172.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.3e-27   94.4   0.0    2.5e-18   65.9   0.0    2.8  2  CAH8618145.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.5e-27   94.3   0.0    2.6e-18   65.8   0.0    2.8  2  CAH8618131.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.5e-27   94.3   0.0    2.6e-18   65.8   0.0    2.8  2  CAH8618300.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.5e-27   94.3   0.0    2.6e-18   65.8   0.0    2.8  2  CAH8618159.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.6e-27   94.3   0.0    2.6e-18   65.8   0.0    2.8  2  CAH8618314.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.6e-26   92.2   0.0    4.3e-26   90.8   0.0    1.8  1  CAH8633315.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.7e-26   92.1   0.0    4.6e-26   90.7   0.0    1.8  1  CAH8633278.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.7e-26   92.1   0.0    4.7e-26   90.7   0.0    1.8  1  CAH8633284.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.9e-26   91.9   0.0      5e-26   90.6   0.0    1.7  1  CAH8633300.1  unnamed protein product [Schistosoma intercalat\n",
            "      2e-26   91.8   0.0    5.3e-26   90.5   0.0    1.8  1  CAH8633263.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.3e-25   89.2   0.1    9.7e-17   60.8   0.0    3.0  2  CAH8624229.1  unnamed protein product [Schistosoma intercalat\n",
            "    7.2e-14   51.6   0.0    2.4e-13   49.9   0.0    1.9  1  CAH8525196.1  unnamed protein product [Schistosoma intercalat\n",
            "    7.4e-14   51.6   0.0    2.2e-13   50.1   0.0    1.8  1  CAH8525213.1  unnamed protein product [Schistosoma intercalat\n",
            "    7.5e-14   51.6   0.0    2.5e-13   49.9   0.0    1.9  1  CAH8525178.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8579544.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   95.3   0.0     4e-30   1.7e-27       1      97 []      39     136 ..      39     136 .. 0.98\n",
            "   2 !   25.7   0.2     2e-08   8.6e-06       2      79 ..     197     274 ..     196     288 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 95.3 bits;  conditional E-value: 4e-30\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++vd+rGg+++++ + g+++lvP+ a++++tri++++++ ++++ +p+l+++e+l+++++e+GP+  +f++pv+le+pH+asl+  ++e++++rs\n",
            "  CAH8579544.1  39 VSFLVDARGGSMRGSrHPGLRILVPPSAASAPTRITCRMLRPERTARPPQLNDCEGLACRIIELGPHPCRFNSPVVLEIPHFASLRGRQRELVVLRS 135\n",
            "                   689********************************************************************************************98 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8579544.1 136 D 136\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 25.7 bits;  conditional E-value: 2e-08\n",
            "           ZU5   2 savvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevp 79 \n",
            "                   s +v+s+Gg l+++   +v+ ++Pega+++  r+ +  a     + + +l ++ + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8579544.1 197 SSLVGSDGGVLSSTvVPQVQAVFPEGALQKRIRVGLQ-AQPIAPELVTRLFGNRVTVSPIVTLEPRRRKFHKPITLTIP 274\n",
            "                   679***********999*************7777776.44445555669*****************************9 PP\n",
            "\n",
            ">> CAH8579530.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   95.0   0.0     5e-30   2.1e-27       1      97 []     255     352 ..     255     352 .. 0.98\n",
            "   2 !   25.4   0.2   2.5e-08   1.1e-05       2      79 ..     413     490 ..     412     504 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 95.0 bits;  conditional E-value: 5e-30\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++vd+rGg+++++ + g+++lvP+ a++++tri++++++ ++++ +p+l+++e+l+++++e+GP+  +f++pv+le+pH+asl+  ++e++++rs\n",
            "  CAH8579530.1 255 VSFLVDARGGSMRGSrHPGLRILVPPSAASAPTRITCRMLRPERTARPPQLNDCEGLACRIIELGPHPCRFNSPVVLEIPHFASLRGRQRELVVLRS 351\n",
            "                   689********************************************************************************************98 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8579530.1 352 D 352\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 25.4 bits;  conditional E-value: 2.5e-08\n",
            "           ZU5   2 savvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevp 79 \n",
            "                   s +v+s+Gg l+++   +v+ ++Pega+++  r+ +  a     + + +l ++ + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8579530.1 413 SSLVGSDGGVLSSTvVPQVQAVFPEGALQKRIRVGLQ-AQPIAPELVTRLFGNRVTVSPIVTLEPRRRKFHKPITLTIP 490\n",
            "                   679***********999*************7777776.44445555669*****************************9 PP\n",
            "\n",
            ">> CAH8618401.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.7   0.0   3.3e-21   1.4e-18       1      97 []     358     455 ..     358     455 .. 0.98\n",
            "   2 !   26.0   0.3   1.7e-08   7.2e-06       4      79 ..     518     593 ..     516     602 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.7 bits;  conditional E-value: 3.3e-21\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++rs\n",
            "  CAH8618401.1 358 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVILRS 454\n",
            "                   5899**********9999****************************************************************************998 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8618401.1 455 D 455\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 26.0 bits;  conditional E-value: 1.7e-08\n",
            "           ZU5   4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79 \n",
            "                   +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618401.1 518 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 593\n",
            "                   67899*******999************95555..55566655555549999**************************8 PP\n",
            "\n",
            ">> CAH8618414.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.8   0.0   3.1e-21   1.3e-18       1      97 []     166     263 ..     166     263 .. 0.98\n",
            "   2 !   26.1   0.3   1.6e-08   6.8e-06       4      79 ..     326     401 ..     324     410 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.8 bits;  conditional E-value: 3.1e-21\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++rs\n",
            "  CAH8618414.1 166 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVILRS 262\n",
            "                   5899**********9999****************************************************************************998 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8618414.1 263 D 263\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 26.1 bits;  conditional E-value: 1.6e-08\n",
            "           ZU5   4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79 \n",
            "                   +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618414.1 326 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 401\n",
            "                   67899*******999************95555..55566655555549999**************************8 PP\n",
            "\n",
            ">> CAH8618426.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.7   0.0   3.3e-21   1.4e-18       1      97 []      32     129 ..      32     129 .. 0.98\n",
            "   2 !   25.9   0.3   1.7e-08   7.3e-06       4      79 ..     192     267 ..     190     276 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.7 bits;  conditional E-value: 3.3e-21\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++rs\n",
            "  CAH8618426.1  32 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVILRS 128\n",
            "                   5899**********9999****************************************************************************998 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8618426.1 129 D 129\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 25.9 bits;  conditional E-value: 1.7e-08\n",
            "           ZU5   4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79 \n",
            "                   +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618426.1 192 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 267\n",
            "                   67899*******999************95555..55566655555549999**************************8 PP\n",
            "\n",
            ">> CAH8618387.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.5   0.0   3.9e-21   1.7e-18       1      97 []     568     665 ..     568     665 .. 0.98\n",
            "   2 !   25.7   0.3     2e-08   8.5e-06       4      79 ..     728     803 ..     726     812 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.5 bits;  conditional E-value: 3.9e-21\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++rs\n",
            "  CAH8618387.1 568 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVILRS 664\n",
            "                   5899**********9999****************************************************************************998 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8618387.1 665 D 665\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 25.7 bits;  conditional E-value: 2e-08\n",
            "           ZU5   4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79 \n",
            "                   +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618387.1 728 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 803\n",
            "                   67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618375.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.4   0.0     4e-21   1.7e-18       1      97 []     851     948 ..     851     948 .. 0.98\n",
            "   2 !   25.7   0.3   2.1e-08   8.8e-06       4      79 ..    1011    1086 ..    1009    1095 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.4 bits;  conditional E-value: 4e-21\n",
            "           ZU5   1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkrs 96 \n",
            "                   +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++rs\n",
            "  CAH8618375.1 851 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVILRS 947\n",
            "                   5899**********9999****************************************************************************998 PP\n",
            "\n",
            "           ZU5  97 d 97 \n",
            "                   d\n",
            "  CAH8618375.1 948 D 948\n",
            "                   6 PP\n",
            "\n",
            "  == domain 2  score: 25.7 bits;  conditional E-value: 2.1e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618375.1 1011 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1086\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618350.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.6   0.3   2.2e-08   9.6e-06       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618350.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618350.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.2e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618350.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618209.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.6   0.3   2.2e-08   9.6e-06       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618209.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618209.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.2e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618209.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618274.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.6   0.3   2.2e-08   9.6e-06       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618274.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618274.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.2e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618274.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618262.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.6   0.3   2.3e-08   9.7e-06       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618262.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618262.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.3e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618262.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618363.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.6   0.3   2.3e-08   9.7e-06       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618363.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618363.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.3e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618363.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618222.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.6   0.3   2.3e-08   9.7e-06       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618222.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618222.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.3e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618222.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618116.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.4e-21   1.9e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.6   0.3   2.3e-08   9.7e-06       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618116.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618116.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.6 bits;  conditional E-value: 2.3e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618116.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618288.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.3   0.0   4.6e-21   1.9e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.5   0.3   2.3e-08   9.9e-06       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.3 bits;  conditional E-value: 4.6e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618288.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618288.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.5 bits;  conditional E-value: 2.3e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618288.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618249.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.1   0.0   5.1e-21   2.2e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.4   0.3   2.6e-08   1.1e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.1 bits;  conditional E-value: 5.1e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618249.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618249.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.4 bits;  conditional E-value: 2.6e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618249.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618327.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.1   0.0   5.1e-21   2.2e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.4   0.3   2.6e-08   1.1e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.1 bits;  conditional E-value: 5.1e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618327.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618327.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.4 bits;  conditional E-value: 2.6e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618327.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618194.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.1   0.0   5.1e-21   2.2e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.4   0.3   2.6e-08   1.1e-05       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.1 bits;  conditional E-value: 5.1e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618194.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618194.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.4 bits;  conditional E-value: 2.6e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618194.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618338.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.1   0.0   5.1e-21   2.2e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.4   0.3   2.6e-08   1.1e-05       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.1 bits;  conditional E-value: 5.1e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618338.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618338.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.4 bits;  conditional E-value: 2.6e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618338.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618185.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.0   0.0   5.4e-21   2.3e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.3   0.3   2.8e-08   1.2e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.0 bits;  conditional E-value: 5.4e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618185.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618185.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.3 bits;  conditional E-value: 2.8e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618185.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618235.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.0   0.0   5.6e-21   2.4e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.2   0.3   2.9e-08   1.2e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.0 bits;  conditional E-value: 5.6e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618235.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618235.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.2 bits;  conditional E-value: 2.9e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618235.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618172.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   66.0   0.0   5.5e-21   2.3e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.3   0.3   2.8e-08   1.2e-05       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 66.0 bits;  conditional E-value: 5.5e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618172.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618172.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.3 bits;  conditional E-value: 2.8e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618172.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618145.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.9   0.0     6e-21   2.5e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.2   0.3     3e-08   1.3e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.9 bits;  conditional E-value: 6e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618145.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618145.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.2 bits;  conditional E-value: 3e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618145.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618131.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.8   0.0   6.1e-21   2.6e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.1   0.3   3.1e-08   1.3e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.8 bits;  conditional E-value: 6.1e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618131.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618131.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.1 bits;  conditional E-value: 3.1e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618131.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618300.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.8   0.0   6.2e-21   2.6e-18       1      97 []     972    1069 ..     972    1069 .. 0.98\n",
            "   2 !   25.1   0.3   3.1e-08   1.3e-05       4      79 ..    1132    1207 ..    1130    1216 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.8 bits;  conditional E-value: 6.2e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618300.1  972 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1066\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618300.1 1067 RSD 1069\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.1 bits;  conditional E-value: 3.1e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618300.1 1132 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1207\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618159.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.8   0.0   6.2e-21   2.6e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.1   0.3   3.1e-08   1.3e-05       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.8 bits;  conditional E-value: 6.2e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618159.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618159.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.1 bits;  conditional E-value: 3.1e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618159.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8618314.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.8   0.0   6.2e-21   2.6e-18       1      97 []     986    1083 ..     986    1083 .. 0.98\n",
            "   2 !   25.1   0.3   3.1e-08   1.3e-05       4      79 ..    1146    1221 ..    1144    1230 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.8 bits;  conditional E-value: 6.2e-21\n",
            "           ZU5    1 asavvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94  \n",
            "                    +s++ d+rGg +      ++++++P++ i+ ++ri ++l+  ++ +s+p +++g++l+++++e+GP+g++f +p+  e+pH+a l+ +++ei+++\n",
            "  CAH8618314.1  986 VSFIADARGGLIQARrYPDLRIFIPPNVINGPLRIVCRLVHPDRMNSPPIMNDGDGLACRLLEIGPAGIRFASPILFEIPHYAFLNGKNREIVIL 1080\n",
            "                    5899**********9999****************************************************************************9 PP\n",
            "\n",
            "           ZU5   95 rsd 97  \n",
            "                    rsd\n",
            "  CAH8618314.1 1081 RSD 1083\n",
            "                    986 PP\n",
            "\n",
            "  == domain 2  score: 25.1 bits;  conditional E-value: 3.1e-08\n",
            "           ZU5    4 vvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslp.kleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    +++ +Gg+l+++   +v++ +P+ga+++  r+   l v+  + +l+ ++ +  + +sp+v++ P+  kf kp++l++p\n",
            "  CAH8618314.1 1146 LIGPEGGTLTSTvVPDVHVRFPQGALQKRIRV--GLQVHPVDHELVtRMLGPRVSVSPIVTIEPRRRKFHKPITLTIP 1221\n",
            "                    67899*******999************95555..55566655555549999*************************98 PP\n",
            "\n",
            ">> CAH8633315.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   90.8   0.0     1e-28   4.3e-26       3      94 ..     745     836 ..     743     838 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 90.8 bits;  conditional E-value: 1e-28\n",
            "           ZU5   3 avvdsrGgrlvlpntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94 \n",
            "                    +++++G ++ lp++g+ l++P+ga+++++ +e+ylav+++++++p+l +++tllspv++ GP gl+++kpv l++pHca+l++++w i+++\n",
            "  CAH8633315.1 745 KTISCDGDQIALPRSGIFLTIPQGALQSNSLVEVYLAVCREDKHRPTLGDHQTLLSPVIQFGPVGLNLLKPVLLSFPHCAELNQSNWLIRVL 836\n",
            "                   68999************************************************************************************996 PP\n",
            "\n",
            ">> CAH8633278.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   90.7   0.0   1.1e-28   4.6e-26       3      94 ..     727     818 ..     725     820 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 90.7 bits;  conditional E-value: 1.1e-28\n",
            "           ZU5   3 avvdsrGgrlvlpntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94 \n",
            "                    +++++G ++ lp++g+ l++P+ga+++++ +e+ylav+++++++p+l +++tllspv++ GP gl+++kpv l++pHca+l++++w i+++\n",
            "  CAH8633278.1 727 KTISCDGDQIALPRSGIFLTIPQGALQSNSLVEVYLAVCREDKHRPTLGDHQTLLSPVIQFGPVGLNLLKPVLLSFPHCAELNQSNWLIRVL 818\n",
            "                   68999************************************************************************************996 PP\n",
            "\n",
            ">> CAH8633284.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   90.7   0.0   1.1e-28   4.7e-26       3      94 ..     745     836 ..     743     838 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 90.7 bits;  conditional E-value: 1.1e-28\n",
            "           ZU5   3 avvdsrGgrlvlpntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94 \n",
            "                    +++++G ++ lp++g+ l++P+ga+++++ +e+ylav+++++++p+l +++tllspv++ GP gl+++kpv l++pHca+l++++w i+++\n",
            "  CAH8633284.1 745 KTISCDGDQIALPRSGIFLTIPQGALQSNSLVEVYLAVCREDKHRPTLGDHQTLLSPVIQFGPVGLNLLKPVLLSFPHCAELNQSNWLIRVL 836\n",
            "                   68999************************************************************************************996 PP\n",
            "\n",
            ">> CAH8633300.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   90.6   0.0   1.2e-28     5e-26       3      94 ..     879     970 ..     877     972 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 90.6 bits;  conditional E-value: 1.2e-28\n",
            "           ZU5   3 avvdsrGgrlvlpntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94 \n",
            "                    +++++G ++ lp++g+ l++P+ga+++++ +e+ylav+++++++p+l +++tllspv++ GP gl+++kpv l++pHca+l++++w i+++\n",
            "  CAH8633300.1 879 KTISCDGDQIALPRSGIFLTIPQGALQSNSLVEVYLAVCREDKHRPTLGDHQTLLSPVIQFGPVGLNLLKPVLLSFPHCAELNQSNWLIRVL 970\n",
            "                   68999************************************************************************************996 PP\n",
            "\n",
            ">> CAH8633263.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   90.5   0.0   1.2e-28   5.3e-26       3      94 ..     879     970 ..     877     972 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 90.5 bits;  conditional E-value: 1.2e-28\n",
            "           ZU5   3 avvdsrGgrlvlpntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislk 94 \n",
            "                    +++++G ++ lp++g+ l++P+ga+++++ +e+ylav+++++++p+l +++tllspv++ GP gl+++kpv l++pHca+l++++w i+++\n",
            "  CAH8633263.1 879 KTISCDGDQIALPRSGIFLTIPQGALQSNSLVEVYLAVCREDKHRPTLGDHQTLLSPVIQFGPVGLNLLKPVLLSFPHCAELNQSNWLIRVL 970\n",
            "                   68999************************************************************************************996 PP\n",
            "\n",
            ">> CAH8624229.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   60.8   0.0   2.3e-19   9.7e-17       2      97 .]    1225    1321 ..    1224    1321 .. 0.98\n",
            "   2 !   25.2   0.1     3e-08   1.3e-05       3      79 ..    1383    1459 ..    1381    1473 .. 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 60.8 bits;  conditional E-value: 2.3e-19\n",
            "           ZU5    2 savvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevpHcaslkseeweislkr 95  \n",
            "                    s+++d+rGg ++ + + g+++lvP++a + ++ri ++l++ +  +++p  ++g+ l+++++e+ P++++fs p+ +evpH+as+k  e+ei + r\n",
            "  CAH8624229.1 1225 SFIIDARGGLVEAQrRPGLRFLVPPNAPSGPLRIICRLLRPEAIDNPPIFNDGDCLACRIIEMNPNQMRFSLPILVEVPHIASIKGREREIIIVR 1319\n",
            "                    79************99*****************************************************************************99 PP\n",
            "\n",
            "           ZU5   96 sd 97  \n",
            "                    s+\n",
            "  CAH8624229.1 1320 SE 1321\n",
            "                    85 PP\n",
            "\n",
            "  == domain 2  score: 25.2 bits;  conditional E-value: 3e-08\n",
            "           ZU5    3 avvdsrGgrlvlp.ntgvsllvPegaipegtrieiylavlkkesslpkleegetllspvvecGPsglkfskpvilevp 79  \n",
            "                    a ++s+Gg ++++   +v+ ++P+g++++  ++ +   +  ++  + +l +g + +spvv++ P+  kf kp++l++p\n",
            "  CAH8624229.1 1383 AFIGSDGGIISSTvAPQVQAVFPPGSLQKRIKVGLQAQIIPNDV-INRLADGRVSVSPVVSIEPRRRKFHKPITLTIP 1459\n",
            "                    5789*********9***************988888877775554.559*****************************8 PP\n",
            "\n",
            ">> CAH8525196.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   49.9   0.0   5.6e-16   2.4e-13       4      81 ..    1064    1151 ..    1061    1165 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 49.9 bits;  conditional E-value: 5.6e-16\n",
            "           ZU5    4 vvdsrGgrlvlpntgvsllvPegaipegtr.ieiylavlkkesslp.........kleegetllspvvecGPsglkfskpvilevpHc 81  \n",
            "                     ++s+Gg l++p+++v l +Pegai e++  ++i++ v + e++l          +++++ +l+sp+v+cGP+gl+f kpv+l+vp +\n",
            "  CAH8525196.1 1064 EFGSQGGILEIPEHNVCLRIPEGAIYEEEGlQKIFIRVYESENELLidnlpcethQSSDKPILVSPMVMCGPRGLTFHKPVELTVPRY 1151\n",
            "                    599**********************9877515555556665554444555466668888999***********************976 PP\n",
            "\n",
            ">> CAH8525213.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   50.1   0.0   5.1e-16   2.2e-13       4      81 ..     984    1071 ..     981    1085 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 50.1 bits;  conditional E-value: 5.1e-16\n",
            "           ZU5    4 vvdsrGgrlvlpntgvsllvPegaipegtr.ieiylavlkkesslp.........kleegetllspvvecGPsglkfskpvilevpHc 81  \n",
            "                     ++s+Gg l++p+++v l +Pegai e++  ++i++ v + e++l          +++++ +l+sp+v+cGP+gl+f kpv+l+vp +\n",
            "  CAH8525213.1  984 EFGSQGGILEIPEHNVCLRIPEGAIYEEEGlQKIFIRVYESENELLidnlpcethQSSDKPILVSPMVMCGPRGLTFHKPVELTVPRY 1071\n",
            "                    599**********************9877515555556665554444555466668888999***********************976 PP\n",
            "\n",
            ">> CAH8525178.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   49.9   0.0   5.7e-16   2.5e-13       4      81 ..    1094    1181 ..    1091    1195 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 49.9 bits;  conditional E-value: 5.7e-16\n",
            "           ZU5    4 vvdsrGgrlvlpntgvsllvPegaipegtr.ieiylavlkkesslp.........kleegetllspvvecGPsglkfskpvilevpHc 81  \n",
            "                     ++s+Gg l++p+++v l +Pegai e++  ++i++ v + e++l          +++++ +l+sp+v+cGP+gl+f kpv+l+vp +\n",
            "  CAH8525178.1 1094 EFGSQGGILEIPEHNVCLRIPEGAIYEEEGlQKIFIRVYESENELLidnlpcethQSSDKPILVSPMVMCGPRGLTFHKPVELTVPRY 1181\n",
            "                    599**********************9877515555556665554444555466668888999***********************976 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (97 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       327  (0.0213029); expected 307.0 (0.02)\n",
            "Passed bias filter:                      278  (0.0181107); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        46  (0.00299674); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                        36  (0.00234528); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):              36  [number of targets reported over threshold]\n",
            "# CPU time: 0.26u 0.00s 00:00:00.26 Elapsed: 00:00:00.32\n",
            "# Mc/sec: 3091.41\n",
            "//\n",
            "Query:       ZUO1-like_ZHD  [M=104]\n",
            "Accession:   PF21884.1\n",
            "Description: Zuotin-like, zuotin homology domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    1.4e-30  105.8   3.3    1.4e-30  105.8   3.3    3.1  2  CAH8530336.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.7e-30  105.5   3.3    1.7e-30  105.5   3.3    3.2  2  CAH8530320.1  unnamed protein product [Schistosoma intercalat\n",
            "    4.1e-12   46.4   4.7    4.1e-12   46.4   4.7    4.4  5  CAH8595884.1  unnamed protein product [Schistosoma intercalat\n",
            "  ------ inclusion threshold ------\n",
            "      0.036   14.5   0.9      0.036   14.5   0.9    3.1  3  CAH8538736.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.064   13.7   0.9      0.064   13.7   0.9    4.1  4  CAH8538687.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.064   13.7   0.9      0.064   13.7   0.9    3.9  4  CAH8538722.1  unnamed protein product [Schistosoma intercalat\n",
            "      0.064   13.7   0.9      0.064   13.7   0.9    3.9  4  CAH8538753.1  unnamed protein product [Schistosoma intercalat\n",
            "        0.1   13.0   2.7       0.27   11.6   2.7    1.7  1  CAH8495830.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8530336.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  105.8   3.3   7.3e-34   1.4e-30       1     103 [.     140     242 ..     140     243 .. 0.99\n",
            "   2 ?  -11.5  15.2         8   1.5e+04      74      90 ..     285     301 ..     244     334 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 105.8 bits;  conditional E-value: 7.3e-34\n",
            "                    HHHHHHHHHHHHHTTBSSSS------TT--HHHHHHHHHHHHT--B----CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD   1 FyklfgpvferearfskkqpvpkLGdeesskeevekfYdfwynfdswreFeyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdra 96 \n",
            "                    F++  g++f r++r+skkqp p+LG++ ++ +ev kfYdfw ++d+ r+++yldee+ke++e+re +r ie++n+ er++r++ee + +r++v  a\n",
            "   CAH8530336.1 140 FFTSLGNFFYRKSRWSKKQPTPHLGNSLTDIDEVLKFYDFWEEYDTCRDYSYLDEEDKEKGEDRETRRAIERQNRIERARRRNEEVSNVRNIVLLA 235\n",
            "                    99********************************************************************************************** PP\n",
            "\n",
            "                    HHH-HHH CS\n",
            "  ZUO1-like_ZHD  97 lklDpri 103\n",
            "                     ++Dpri\n",
            "   CAH8530336.1 236 KENDPRI 242\n",
            "                    ******9 PP\n",
            "\n",
            "  == domain 2  score: -11.5 bits;  conditional E-value: 8\n",
            "                    HHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  74 nkaerkkrkkeenarlr 90 \n",
            "                     +a+ ++r+++e +r+r\n",
            "   CAH8530336.1 285 ARAASEERRRSEAERIR 301\n",
            "                    23333444444444444 PP\n",
            "\n",
            ">> CAH8530320.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  105.5   3.3   8.9e-34   1.7e-30       1     103 [.     140     242 ..     140     243 .. 0.99\n",
            "   2 ?  -11.1  14.5         8   1.5e+04      74      91 ..     285     302 ..     245     334 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 105.5 bits;  conditional E-value: 8.9e-34\n",
            "                    HHHHHHHHHHHHHTTBSSSS------TT--HHHHHHHHHHHHT--B----CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD   1 FyklfgpvferearfskkqpvpkLGdeesskeevekfYdfwynfdswreFeyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdra 96 \n",
            "                    F++  g++f r++r+skkqp p+LG++ ++ +ev kfYdfw ++d+ r+++yldee+ke++e+re +r ie++n+ er++r++ee + +r++v  a\n",
            "   CAH8530320.1 140 FFTSLGNFFYRKSRWSKKQPTPHLGNSLTDIDEVLKFYDFWEEYDTCRDYSYLDEEDKEKGEDRETRRAIERQNRIERARRRNEEVSNVRNIVLLA 235\n",
            "                    99********************************************************************************************** PP\n",
            "\n",
            "                    HHH-HHH CS\n",
            "  ZUO1-like_ZHD  97 lklDpri 103\n",
            "                     ++Dpri\n",
            "   CAH8530320.1 236 KENDPRI 242\n",
            "                    ******9 PP\n",
            "\n",
            "  == domain 2  score: -11.1 bits;  conditional E-value: 8\n",
            "                    HHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  74 nkaerkkrkkeenarlrk 91 \n",
            "                     +a+ ++r+++e +r+rk\n",
            "   CAH8530320.1 285 ARAASEERRRSEAERIRK 302\n",
            "                    233334444444444443 PP\n",
            "\n",
            ">> CAH8595884.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.8   0.0       4.4   8.4e+03       1      10 [.     112     121 ..     106     125 .. 0.71\n",
            "   2 !   46.4   4.7   2.2e-15   4.1e-12      13     103 ..     153     246 ..     150     247 .. 0.90\n",
            "   3 ?   -3.2   0.9       5.8   1.1e+04      80      98 ..     255     273 ..     248     282 .. 0.48\n",
            "   4 ?   -1.0   5.6       1.2   2.4e+03      53      86 ..     442     475 ..     415     490 .. 0.70\n",
            "   5 ?   -1.8   1.3       2.2   4.2e+03      64      83 ..     600     619 ..     591     622 .] 0.55\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.8 bits;  conditional E-value: 4.4\n",
            "                    HHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD   1 Fyklfgpvfe 10 \n",
            "                    Fy+++++vf+\n",
            "   CAH8595884.1 112 FYTVYAKVFA 121\n",
            "                    6777777775 PP\n",
            "\n",
            "  == domain 2  score: 46.4 bits;  conditional E-value: 2.2e-15\n",
            "                    HTTBSS..SS------TT--HHHH.HHHHHHHHT--B----CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHH CS\n",
            "  ZUO1-like_ZHD  13 arfskk..qpvpkLGdeesskeev.ekfYdfwynfdswreFeyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdralklDpri 103\n",
            "                    +++s++  ++ p +G+++s+++ev   fY fw  f++ +++++ +++++  a+sr+e+r +e +n++ r    ++ n+ +r+lv    k D+r+\n",
            "   CAH8595884.1 153 KKYSRRksRSYPPFGSSSSPYKEVvAPFYLFWEIFETKKTYTWVEKYDTRLADSRQERRAMETENNRMRMSAIRKRNEEIRQLVAYVKKRDKRV 246\n",
            "                    56775511569*********9987256*****************************************************************97 PP\n",
            "\n",
            "  == domain 3  score: -3.2 bits;  conditional E-value: 5.8\n",
            "                    HHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  80 krkkeenarlrklvdralk 98 \n",
            "                    +  ke ++r + l ++a +\n",
            "   CAH8595884.1 255 RIAKESQNRTKLLAEKARQ 273\n",
            "                    3334444444444444433 PP\n",
            "\n",
            "  == domain 4  score: -1.0 bits;  conditional E-value: 1.2\n",
            "                    C-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  53 ldeevkedaesreekryiekknkaerkkrkkeen 86 \n",
            "                    +++ +++d++++ + + +++ +kaer+++k+ e \n",
            "   CAH8595884.1 442 NNSMNNNDNNEQSNIKLTKRAKKAERQRKKQFEL 475\n",
            "                    4445566777777788888888888888887664 PP\n",
            "\n",
            "  == domain 5  score: -1.8 bits;  conditional E-value: 2.2\n",
            "                    HHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  64 reekryiekknkaerkkrkk 83 \n",
            "                      + + i ++nk++ k++kk\n",
            "   CAH8595884.1 600 SSNTKDISNENKKKCKEKKK 619\n",
            "                    44455555666666555555 PP\n",
            "\n",
            ">> CAH8538736.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.0  13.2      0.61   1.2e+03      45      89 ..      58     102 ..      52     108 .. 0.82\n",
            "   2 ?   -1.6   7.2       1.9   3.6e+03      50      79 ..     113     142 ..     101     148 .. 0.51\n",
            "   3 ?   14.5   0.9   1.9e-05     0.036      18      97 ..     183     264 ..     175     268 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.0 bits;  conditional E-value: 0.61\n",
            "                    -B----CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  45 dswreFeyldeevkedaesreekryiekknkaerkkrkkeenarl 89 \n",
            "                    +  r+ ++  +++ ++++s+ +k+ ++k+n+ +++krk++  +r \n",
            "   CAH8538736.1  58 KKLRHKNHSHHHHRSHKHSKRHKKETKKSNRLKKQKRKNKHDSRT 102\n",
            "                    5556667778888889999999*************9999988886 PP\n",
            "\n",
            "  == domain 2  score: -1.6 bits;  conditional E-value: 1.9\n",
            "                    -CCC-TTS-SSTSSHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  50 Feyldeevkedaesreekryiekknkaerk 79 \n",
            "                     + +de+++  ++ +++kr +e+k ++e++\n",
            "   CAH8538736.1 113 VDSEDEKKQFIKQMKDKKRELERKRAQEEE 142\n",
            "                    233334444444555555555555544443 PP\n",
            "\n",
            "  == domain 3  score: 14.5 bits;  conditional E-value: 1.9e-05\n",
            "                    SSS------TT--HHHHHHHHHHHHT--B---..-CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  18 kqpvpkLGdeesskeevekfYdfwynfdswre..Feyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdral 97 \n",
            "                     + +p+ G+   + ee+e+f +  y ++  r+   e     ++++  s +ekr +e+ n+aer+kr  + +a++++l++r l\n",
            "   CAH8538736.1 183 GKRIPRRGEIGLTPEEIESFEKEGYVMSGSRHrrMEAVRLRKENQIYSADEKRALEHFNHAERAKREAKLQAQFKALIKRKL 264\n",
            "                    567888899999999999999877766665550055556667777789*****************************99865 PP\n",
            "\n",
            ">> CAH8538687.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.8   2.1       2.1     4e+03      50      88 ..      48      86 ..      38     101 .. 0.80\n",
            "   2 ?   -0.7  12.8      0.98   1.9e+03      51      89 ..     162     200 ..     150     206 .. 0.75\n",
            "   3 ?   -2.3   7.5       3.2   6.1e+03      50      78 ..     211     239 ..     196     245 .. 0.49\n",
            "   4 ?   13.7   0.9   3.3e-05     0.064      18      97 ..     281     362 ..     273     366 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.8 bits;  conditional E-value: 2.1\n",
            "                   -CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD 50 Feyldeevkedaesreekryiekknkaerkkrkkeenar 88\n",
            "                   ++y+d+++ +++ s+ +   +  + k+e ++rk++e + \n",
            "   CAH8538687.1 48 HSYKDKHDLSESYSKSKSPSLSISHKNEGRHRKTNEFNH 86\n",
            "                   578899998888888888888888888888888877654 PP\n",
            "\n",
            "  == domain 2  score: -0.7 bits;  conditional E-value: 0.98\n",
            "                    CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  51 eyldeevkedaesreekryiekknkaerkkrkkeenarl 89 \n",
            "                    ++  +++ ++++s+ +k+ ++k+n+ +++krk++  +r \n",
            "   CAH8538687.1 162 NHSHHHHRSHKHSKRHKKETKKSNRLKKQKRKNKHASRT 200\n",
            "                    444566677788888999999999999999999998886 PP\n",
            "\n",
            "  == domain 3  score: -2.3 bits;  conditional E-value: 3.2\n",
            "                    -CCC-TTS-SSTSSHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  50 Feyldeevkedaesreekryiekknkaer 78 \n",
            "                     + +de+++  ++ +++kr +e+k ++e+\n",
            "   CAH8538687.1 211 VDSEDEKKQFIKQMKDKKRELERKRAQEE 239\n",
            "                    22333333333444555555555544443 PP\n",
            "\n",
            "  == domain 4  score: 13.7 bits;  conditional E-value: 3.3e-05\n",
            "                    SSS------TT--HHHHHHHHHHHHT--B---..-CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  18 kqpvpkLGdeesskeevekfYdfwynfdswre..Feyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdral 97 \n",
            "                     + +p+ G+   + ee+e+f +  y ++  r+   e     ++++  s +ekr +e+ n+aer+kr  + +a++++l++r l\n",
            "   CAH8538687.1 281 GKRIPRRGEIGLTPEEIESFEKEGYVMSGSRHrrMEAVRLRKENQIYSADEKRALEHFNHAERAKREAKLQAQFKALIKRKL 362\n",
            "                    567888888999999999998877666665550055556667777789*****************************99865 PP\n",
            "\n",
            ">> CAH8538722.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.0   1.0       2.6   4.9e+03      50      87 ..      48      85 ..      39     100 .. 0.75\n",
            "   2 ?   -0.7  13.1      0.95   1.8e+03      45      89 ..     156     200 ..     150     205 .. 0.82\n",
            "   3 ?   -1.6   6.5       1.9   3.7e+03      50      79 ..     211     240 ..     201     246 .. 0.52\n",
            "   4 ?   13.7   0.9   3.3e-05     0.064      18      97 ..     281     362 ..     273     366 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.0 bits;  conditional E-value: 2.6\n",
            "                   -CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD 50 Feyldeevkedaesreekryiekknkaerkkrkkeena 87\n",
            "                   ++y+d+++ +++ s+ +   +  + k+e  +rk++e +\n",
            "   CAH8538722.1 48 HSYKDKHDISESYSKSKSPSLSISHKNEGCHRKTNEFN 85\n",
            "                   57788888888888777777777777777777777655 PP\n",
            "\n",
            "  == domain 2  score: -0.7 bits;  conditional E-value: 0.95\n",
            "                    -B----CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  45 dswreFeyldeevkedaesreekryiekknkaerkkrkkeenarl 89 \n",
            "                    +  r+ ++  +++ ++++s+ +k+ ++k+n+ +++krk++  +r \n",
            "   CAH8538722.1 156 KKLRHKNHSHHHHRSHKHSKRHKKETKKSNRLKKQKRKNKHDSRT 200\n",
            "                    5556667777888889999999*************9999988886 PP\n",
            "\n",
            "  == domain 3  score: -1.6 bits;  conditional E-value: 1.9\n",
            "                    -CCC-TTS-SSTSSHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  50 Feyldeevkedaesreekryiekknkaerk 79 \n",
            "                     + +de+++  ++ +++kr +e+k ++e++\n",
            "   CAH8538722.1 211 VDSEDEKKQFIKQMKDKKRELERKRAQEEE 240\n",
            "                    333344444445555555555555555443 PP\n",
            "\n",
            "  == domain 4  score: 13.7 bits;  conditional E-value: 3.3e-05\n",
            "                    SSS------TT--HHHHHHHHHHHHT--B---..-CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  18 kqpvpkLGdeesskeevekfYdfwynfdswre..Feyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdral 97 \n",
            "                     + +p+ G+   + ee+e+f +  y ++  r+   e     ++++  s +ekr +e+ n+aer+kr  + +a++++l++r l\n",
            "   CAH8538722.1 281 GKRIPRRGEIGLTPEEIESFEKEGYVMSGSRHrrMEAVRLRKENQIYSADEKRALEHFNHAERAKREAKLQAQFKALIKRKL 362\n",
            "                    567888888999999999998877666665550055556667777789*****************************99865 PP\n",
            "\n",
            ">> CAH8538753.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.2   1.1       2.8   5.4e+03      50      87 ..      48      85 ..      39     101 .. 0.75\n",
            "   2 ?   -0.7  13.1      0.95   1.8e+03      45      89 ..     156     200 ..     150     205 .. 0.82\n",
            "   3 ?   -1.6   6.5       1.9   3.7e+03      50      79 ..     211     240 ..     201     246 .. 0.52\n",
            "   4 ?   13.7   0.9   3.3e-05     0.064      18      97 ..     281     362 ..     273     366 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.2 bits;  conditional E-value: 2.8\n",
            "                   -CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD 50 Feyldeevkedaesreekryiekknkaerkkrkkeena 87\n",
            "                   ++y+d+++ +++ s+ +   +  + k+e  +rk++e +\n",
            "   CAH8538753.1 48 HSYKDKHDISESYSKSKSPSLSISHKNEGCHRKTNEFN 85\n",
            "                   57788888888887777777777777777777776655 PP\n",
            "\n",
            "  == domain 2  score: -0.7 bits;  conditional E-value: 0.95\n",
            "                    -B----CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  45 dswreFeyldeevkedaesreekryiekknkaerkkrkkeenarl 89 \n",
            "                    +  r+ ++  +++ ++++s+ +k+ ++k+n+ +++krk++  +r \n",
            "   CAH8538753.1 156 KKLRHKNHSHHHHRSHKHSKRHKKETKKSNRLKKQKRKNKHDSRT 200\n",
            "                    5556667777888889999999*************9999988886 PP\n",
            "\n",
            "  == domain 3  score: -1.6 bits;  conditional E-value: 1.9\n",
            "                    -CCC-TTS-SSTSSHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  50 Feyldeevkedaesreekryiekknkaerk 79 \n",
            "                     + +de+++  ++ +++kr +e+k ++e++\n",
            "   CAH8538753.1 211 VDSEDEKKQFIKQMKDKKRELERKRAQEEE 240\n",
            "                    333344444445555555555555555443 PP\n",
            "\n",
            "  == domain 4  score: 13.7 bits;  conditional E-value: 3.3e-05\n",
            "                    SSS------TT--HHHHHHHHHHHHT--B---..-CCC-TTS-SSTSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  18 kqpvpkLGdeesskeevekfYdfwynfdswre..Feyldeevkedaesreekryiekknkaerkkrkkeenarlrklvdral 97 \n",
            "                     + +p+ G+   + ee+e+f +  y ++  r+   e     ++++  s +ekr +e+ n+aer+kr  + +a++++l++r l\n",
            "   CAH8538753.1 281 GKRIPRRGEIGLTPEEIESFEKEGYVMSGSRHrrMEAVRLRKENQIYSADEKRALEHFNHAERAKREAKLQAQFKALIKRKL 362\n",
            "                    567888888999999999998877666665550055556667777789*****************************99865 PP\n",
            "\n",
            ">> CAH8495830.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.6   2.7   0.00014      0.27      15      84 ..     677     750 ..     666     762 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.6 bits;  conditional E-value: 0.00014\n",
            "                    TBSSSS------TT--HHHHHHHH.HHHHT--B----CCC-T...TS-SSTSSHHHHHHHHHHHHHHHHHHHHH CS\n",
            "  ZUO1-like_ZHD  15 fskkqpvpkLGdeesskeevekfY.dfwynfdswreFeylde...evkedaesreekryiekknkaerkkrkke 84 \n",
            "                    f  + +  +LG+++ +++++ +++ +   +f+ w +F++ ++   e+++d++s+++k++ ++++++++ k k+e\n",
            "   CAH8495830.1 677 FRMTDSTTRLGSNKLEQQKILDYFkEELNDFEEWENFDTVSSvinEESSDTHSQNDKKIDNNNDHQNKVKSKNE 750\n",
            "                    667778899*****9999987655167999*******9986611144445555555555444444444444433 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (104 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1672  (0.108925); expected 307.0 (0.02)\n",
            "Passed bias filter:                      642  (0.0418241); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        58  (0.0037785); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000586319); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               8  [number of targets reported over threshold]\n",
            "# CPU time: 0.30u 0.00s 00:00:00.30 Elapsed: 00:00:00.33\n",
            "# Mc/sec: 3264.91\n",
            "//\n",
            "Query:       ZW10_C  [M=131]\n",
            "Accession:   PF20666.2\n",
            "Description: Centromere/kinetochore protein zw10, C-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    2.4e-12   46.7   1.3    8.2e-09   35.3   0.1    3.6  3  CAH8606163.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.2e-11   44.4   6.0    9.8e-09   35.0   0.1    4.3  4  CAH8606132.1  unnamed protein product [Schistosoma intercalat\n",
            "    3.2e-11   43.1   7.7      9e-09   35.2   0.2    4.3  5  CAH8606147.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8606163.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.5   0.0      0.37   1.9e+03      31      64 ..      46      80 ..      33      83 .. 0.75\n",
            "   2 !    8.0   0.0   0.00045       2.3       1      26 [.     535     560 ..     535     577 .. 0.91\n",
            "   3 !   35.3   0.1   1.6e-12   8.2e-09      41      94 ..     600     653 ..     584     684 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.5 bits;  conditional E-value: 0.37\n",
            "                  HHHHHHHHHHHHHHHHHHHH.HHHHHTTTTTTS-H CS\n",
            "        ZW10_C 31 aevaaklfltvrnvlqlyvd.vvPkkhkelleslp 64\n",
            "                   ++ ++l + v ++ + +++ +vP kh++  ++++\n",
            "  CAH8606163.1 46 TRCKNRLTQHVDDLWRRWIIlEVPSKHQNVSSKFN 80\n",
            "                  4677889999999988886548*****98777665 PP\n",
            "\n",
            "  == domain 2  score: 8.0 bits;  conditional E-value: 0.00045\n",
            "                   SS---EEEEHHHHHHHHHHHHHHHHH CS\n",
            "        ZW10_C   1 fsfprCrvSesakklmeLlektlkea 26 \n",
            "                   f fp C+vSe++  +m+ ++k+l++ \n",
            "  CAH8606163.1 535 FDFPDCMVSETVDLVMKQIYKILHDC 560\n",
            "                   789*******************9874 PP\n",
            "\n",
            "  == domain 3  score: 35.3 bits;  conditional E-value: 1.6e-12\n",
            "                   HHHHHHHHHHHHHHHTTTTTTS-HHHHHHHHHHHHHHHHHHHHHTGGGTTTS-T CS\n",
            "        ZW10_C  41 vrnvlqlyvdvvPkkhkelleslpqlsalfhNnCmYlshllltlgfefraelak 94 \n",
            "                   +  ++ l+++ vP+ h++ +++  + ++l++N+CmYl+h+ ltlg +   ++ +\n",
            "  CAH8606163.1 600 IPRLIMLFTSLVPTLHADVFQGDLRFAVLYYNDCMYLAHECLTLGERGLYKFVQ 653\n",
            "                   567899**************************************9765555544 PP\n",
            "\n",
            ">> CAH8606132.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.1   0.2     0.061   3.1e+02      52      80 ..      82     109 ..      78     130 .. 0.73\n",
            "   2 ?   -1.8   0.0      0.46   2.3e+03      31      64 ..     208     242 ..     196     245 .. 0.75\n",
            "   3 !    7.7   0.0   0.00054       2.8       1      26 [.     697     722 ..     697     739 .. 0.91\n",
            "   4 !   35.0   0.1   1.9e-12   9.8e-09      41      93 ..     762     814 ..     742     842 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.1 bits;  conditional E-value: 0.061\n",
            "                   HHHHTTTTTTS-HHHHHHHHHHHHHHHHH CS\n",
            "        ZW10_C  52 vPkkhkelleslpqlsalfhNnCmYlshl 80 \n",
            "                   v + h++ +++l +l   +h nC Yl +l\n",
            "  CAH8606132.1  82 VSNIHQNAVSQLQELLD-MHDNCSYLLEL 109\n",
            "                   66778888888888765.5899***9875 PP\n",
            "\n",
            "  == domain 2  score: -1.8 bits;  conditional E-value: 0.46\n",
            "                   HHHHHHHHHHHHHHHHHHHH.HHHHHTTTTTTS-H CS\n",
            "        ZW10_C  31 aevaaklfltvrnvlqlyvd.vvPkkhkelleslp 64 \n",
            "                    ++ ++l + v ++ + +++ +vP kh++  ++++\n",
            "  CAH8606132.1 208 TRCKNRLTQHVDDLWRRWIIlEVPSKHQNVSSKFN 242\n",
            "                   4677888999999988886548*****98777665 PP\n",
            "\n",
            "  == domain 3  score: 7.7 bits;  conditional E-value: 0.00054\n",
            "                   SS---EEEEHHHHHHHHHHHHHHHHH CS\n",
            "        ZW10_C   1 fsfprCrvSesakklmeLlektlkea 26 \n",
            "                   f fp C+vSe++  +m+ ++k+l++ \n",
            "  CAH8606132.1 697 FDFPDCMVSETVDLVMKQIYKILHDC 722\n",
            "                   789*******************9874 PP\n",
            "\n",
            "  == domain 4  score: 35.0 bits;  conditional E-value: 1.9e-12\n",
            "                   HHHHHHHHHHHHHHHTTTTTTS-HHHHHHHHHHHHHHHHHHHHHTGGGTTTS- CS\n",
            "        ZW10_C  41 vrnvlqlyvdvvPkkhkelleslpqlsalfhNnCmYlshllltlgfefraela 93 \n",
            "                   +  ++ l+++ vP+ h++ +++  + ++l++N+CmYl+h+ ltlg +   ++ \n",
            "  CAH8606132.1 762 IPRLIMLFTSLVPTLHADVFQGDLRFAVLYYNDCMYLAHECLTLGERGLYKFV 814\n",
            "                   567899**************************************976555554 PP\n",
            "\n",
            ">> CAH8606147.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.2   0.2     0.055   2.8e+02      52      80 ..      82     109 ..      78     130 .. 0.73\n",
            "   2 ?   -1.6   0.0      0.41   2.1e+03      31      64 ..     208     242 ..     196     245 .. 0.75\n",
            "   3 !    7.8   0.0   0.00049       2.5       1      26 [.     697     722 ..     697     739 .. 0.91\n",
            "   4 !   35.2   0.2   1.8e-12     9e-09      41      93 ..     762     814 ..     742     845 .. 0.84\n",
            "   5 ?   -3.6   0.0       1.7   8.8e+03     106     129 ..     859     882 ..     846     883 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.2 bits;  conditional E-value: 0.055\n",
            "                   HHHHTTTTTTS-HHHHHHHHHHHHHHHHH CS\n",
            "        ZW10_C  52 vPkkhkelleslpqlsalfhNnCmYlshl 80 \n",
            "                   v + h++ +++l +l   +h nC Yl +l\n",
            "  CAH8606147.1  82 VSNIHQNAVSQLQELLD-MHDNCSYLLEL 109\n",
            "                   66778888888888765.5899***9875 PP\n",
            "\n",
            "  == domain 2  score: -1.6 bits;  conditional E-value: 0.41\n",
            "                   HHHHHHHHHHHHHHHHHHHH.HHHHHTTTTTTS-H CS\n",
            "        ZW10_C  31 aevaaklfltvrnvlqlyvd.vvPkkhkelleslp 64 \n",
            "                    ++ ++l + v ++ + +++ +vP kh++  ++++\n",
            "  CAH8606147.1 208 TRCKNRLTQHVDDLWRRWIIlEVPSKHQNVSSKFN 242\n",
            "                   4677888999999988886548*****98777665 PP\n",
            "\n",
            "  == domain 3  score: 7.8 bits;  conditional E-value: 0.00049\n",
            "                   SS---EEEEHHHHHHHHHHHHHHHHH CS\n",
            "        ZW10_C   1 fsfprCrvSesakklmeLlektlkea 26 \n",
            "                   f fp C+vSe++  +m+ ++k+l++ \n",
            "  CAH8606147.1 697 FDFPDCMVSETVDLVMKQIYKILHDC 722\n",
            "                   789*******************9874 PP\n",
            "\n",
            "  == domain 4  score: 35.2 bits;  conditional E-value: 1.8e-12\n",
            "                   HHHHHHHHHHHHHHHTTTTTTS-HHHHHHHHHHHHHHHHHHHHHTGGGTTTS- CS\n",
            "        ZW10_C  41 vrnvlqlyvdvvPkkhkelleslpqlsalfhNnCmYlshllltlgfefraela 93 \n",
            "                   +  ++ l+++ vP+ h++ +++  + ++l++N+CmYl+h+ ltlg +   ++ \n",
            "  CAH8606147.1 762 IPRLIMLFTSLVPTLHADVFQGDLRFAVLYYNDCMYLAHECLTLGERGLYKFV 814\n",
            "                   567899**************************************976555554 PP\n",
            "\n",
            "  == domain 5  score: -3.6 bits;  conditional E-value: 1.7\n",
            "                   GHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "        ZW10_C 106 lvprlrelgeeclekqltlqrsel 129\n",
            "                   lvp lr+ g + l   l++ r+ l\n",
            "  CAH8606147.1 859 LVPHLRQSGVNNLLDHLRRERQFL 882\n",
            "                   799999999999999999988765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (131 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       592  (0.0385668); expected 307.0 (0.02)\n",
            "Passed bias filter:                      315  (0.0205212); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        18  (0.00117264); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         3  (0.00019544); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.24u 0.01s 00:00:00.25 Elapsed: 00:00:00.23\n",
            "# Mc/sec: 5800.73\n",
            "//\n",
            "Query:       ZW10_C2  [M=152]\n",
            "Accession:   PF22766.1\n",
            "Description: ZW10 C-terminal helical domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    1.1e-19   70.8   0.6    1.2e-18   67.4   0.1    3.1  2  CAH8606132.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.1e-19   70.8   0.8    9.7e-19   67.8   0.1    2.9  2  CAH8606163.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.4e-06   28.3   2.0      7e-06   26.0   0.4    2.9  2  CAH8606147.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8606132.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.9   0.0      0.26   1.3e+03      30      76 ..     449     495 ..     442     527 .. 0.61\n",
            "   2 !   67.4   0.1   2.4e-22   1.2e-18       2     152 .]     907    1075 ..     906    1075 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.9 bits;  conditional E-value: 0.26\n",
            "                   HHHHHHHHHHHHHHHHHT-----HHHHHHHHHHHHHHHHHSTTS--- CS\n",
            "       ZW10_C2  30 lgalleavieeivrkiltleDlsadeaeelaklikvvlekapkvlad 76 \n",
            "                    ++ l  v++e+   ++ l+ +++d+ + +  ++++ ++  + ++++\n",
            "  CAH8606132.1 449 SCNQLTKVFQELYTYLFGLQAIQSDNKRFIDIIVNNNIDSDSGMFNN 495\n",
            "                   45556666666666666666666666666666666655555555543 PP\n",
            "\n",
            "  == domain 2  score: 67.4 bits;  conditional E-value: 2.4e-22\n",
            "                    HHHHHHHHHHHHHHHHTTTS-HHHHHHHHHHHHHHHHHHHHHHHHT-----HHHHHHHHHHHHHHHH......HSTTS---SSSHHHHHHHHT-H CS\n",
            "       ZW10_C2    2 avrqvllqlkllakvwqevLpekvYkkslgalleavieeivrkiltleDlsadeaeelaklikvvle......kapkvladlseadenskvldev 90  \n",
            "                    a++ + ++l+++++ +  +Lp +vY +++g + + v +e+v+ il+l D+++ + +++  l++ + +      k ++v+++++ ++   k + + \n",
            "  CAH8606132.1  907 AIEDCCHHLHKISEALS-PLPRTVYFRAMGIMCNCVCTELVKTILKLIDITKSDCDNILILLESIKNtiveffKPSEVVHQPAAKESIIKSQHNY 1000\n",
            "                    7999**********996.7*************************************988887766551111334456666665554333333322 PP\n",
            "\n",
            "                    .........-TTHHHHHHHHHHTT-.-HHHHHH.HHTTTTSHHH..HHS-HHHHHHHHHHHS-S-HHHHHHHHH- CS\n",
            "       ZW10_C2   91 .........vrkwsklkelvellea.slqeitd.rWangkGpla..anfsrsEvknlirAlFqdterRaealaki 152 \n",
            "                              +++ +l+ ++++l+a sl ei d  W +g+Gpl    ++++sE++ li+Al+  +  R e+l k+\n",
            "  CAH8606132.1 1001 ntdelltrrCPDFHRLQGIISVLSAsSLAEIRDiLWNDGNGPLSieSHLTTSELRCLIKALYSSSSFRDELLQKL 1075\n",
            "                    45555999999************8637****9956*******9733689**********************9986 PP\n",
            "\n",
            ">> CAH8606163.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.6   0.0      0.21   1.1e+03      30      76 ..     287     333 ..     280     366 .. 0.61\n",
            "   2 !   67.8   0.1   1.9e-22   9.7e-19       2     152 .]     745     913 ..     744     913 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.6 bits;  conditional E-value: 0.21\n",
            "                   HHHHHHHHHHHHHHHHHT-----HHHHHHHHHHHHHHHHHSTTS--- CS\n",
            "       ZW10_C2  30 lgalleavieeivrkiltleDlsadeaeelaklikvvlekapkvlad 76 \n",
            "                    ++ l  v++e+   ++ l+ +++d+ + +  ++++ ++  + ++++\n",
            "  CAH8606163.1 287 SCNQLTKVFQELYTYLFGLQAIQSDNKRFIDIIVNNNIDSDSGMFNN 333\n",
            "                   45555666666666666666666666666665555555555555543 PP\n",
            "\n",
            "  == domain 2  score: 67.8 bits;  conditional E-value: 1.9e-22\n",
            "                   HHHHHHHHHHHHHHHHTTTS-HHHHHHHHHHHHHHHHHHHHHHHHT-----HHHHHHHHHHHHHHHH......HSTTS---SSSHHHHHHHHT-H.. CS\n",
            "       ZW10_C2   2 avrqvllqlkllakvwqevLpekvYkkslgalleavieeivrkiltleDlsadeaeelaklikvvle......kapkvladlseadenskvldev.. 90 \n",
            "                   a++ + ++l+++++ +  +Lp +vY +++g + + v +e+v+ il+l D+++ + +++  l++ + +      k ++v+++++ ++   k + +   \n",
            "  CAH8606163.1 745 AIEDCCHHLHKISEALS-PLPRTVYFRAMGIMCNCVCTELVKTILKLIDITKSDCDNILILLESIKNtiveffKPSEVVHQPAAKESIIKSQHNYnt 840\n",
            "                   7999**********996.7*************************************98888776655111133445666666555433333332245 PP\n",
            "\n",
            "                   .......-TTHHHHHHHHHHTT-.-HHHHHH.HHTTTTSHHH..HHS-HHHHHHHHHHHS-S-HHHHHHHHH- CS\n",
            "       ZW10_C2  91 .......vrkwsklkelvellea.slqeitd.rWangkGpla..anfsrsEvknlirAlFqdterRaealaki 152\n",
            "                           +++ +l+ ++++l+a sl ei d  W +g+Gpl    ++++sE++ li+Al+  +  R e+l k+\n",
            "  CAH8606163.1 841 delltrrCPDFHRLQGIISVLSAsSLAEIRDiLWNDGNGPLSieSHLTTSELRCLIKALYSSSSFRDELLQKL 913\n",
            "                   555999999************8637****9956*******9733689**********************9986 PP\n",
            "\n",
            ">> CAH8606147.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.7   0.0      0.23   1.2e+03      30      76 ..     449     495 ..     442     528 .. 0.61\n",
            "   2 !   26.0   0.4   1.4e-09     7e-06       2      66 ..     907     970 ..     906     979 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.7 bits;  conditional E-value: 0.23\n",
            "                   HHHHHHHHHHHHHHHHHT-----HHHHHHHHHHHHHHHHHSTTS--- CS\n",
            "       ZW10_C2  30 lgalleavieeivrkiltleDlsadeaeelaklikvvlekapkvlad 76 \n",
            "                    ++ l  v++e+   ++ l+ +++d+ + +  ++++ ++  + ++++\n",
            "  CAH8606147.1 449 SCNQLTKVFQELYTYLFGLQAIQSDNKRFIDIIVNNNIDSDSGMFNN 495\n",
            "                   45555666666666666666666666666665555555555555543 PP\n",
            "\n",
            "  == domain 2  score: 26.0 bits;  conditional E-value: 1.4e-09\n",
            "                   HHHHHHHHHHHHHHHHTTTS-HHHHHHHHHHHHHHHHHHHHHHHHT-----HHHHHHHHHHHHHH CS\n",
            "       ZW10_C2   2 avrqvllqlkllakvwqevLpekvYkkslgalleavieeivrkiltleDlsadeaeelaklikvv 66 \n",
            "                   a++ + ++l+++++ +  +Lp +vY +++g + + v +e+v+ il+l D+++ + +++  l++ +\n",
            "  CAH8606147.1 907 AIEDCCHHLHKISEALS-PLPRTVYFRAMGIMCNCVCTELVKTILKLIDITKSDCDNILILLESI 970\n",
            "                   7999**********996.7*************************************999988876 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (152 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       828  (0.0539414); expected 307.0 (0.02)\n",
            "Passed bias filter:                      451  (0.0293811); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        37  (0.00241042); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         3  (0.00019544); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.28u 0.00s 00:00:00.28 Elapsed: 00:00:00.27\n",
            "# Mc/sec: 5875.07\n",
            "//\n",
            "Query:       Zw10_middle  [M=230]\n",
            "Accession:   PF20665.2\n",
            "Description: Centromere/kinetochore protein zw10, middle domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (230 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       675  (0.0439739); expected 307.0 (0.02)\n",
            "Passed bias filter:                      263  (0.0171336); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        17  (0.00110749); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.32u 0.00s 00:00:00.32 Elapsed: 00:00:00.30\n",
            "# Mc/sec: 7834.00\n",
            "//\n",
            "Query:       Zw10_N  [M=93]\n",
            "Accession:   PF06248.18\n",
            "Description: Centromere/kinetochore Zw10 N-terminal\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.055   13.8   1.4       0.21   12.0   1.4    2.1  1  CAH8431581.1  unnamed protein product [Schistosoma intercalat\n",
            "        3.6    8.0   6.4        2.7    8.4   0.0    3.4  3  CAH8288436.1  unnamed protein product [Schistosoma intercalat\n",
            "        6.3    7.2   7.6        6.3    7.2   0.8    3.1  2  CAH8565081.1  unnamed protein product [Schistosoma intercalat\n",
            "        7.8    6.9   7.6        7.1    7.1   0.8    3.2  2  CAH8565097.1  unnamed protein product [Schistosoma intercalat\n",
            "        7.9    6.9   7.6        7.1    7.1   0.8    3.2  2  CAH8565051.1  unnamed protein product [Schistosoma intercalat\n",
            "          8    6.9   7.6        7.2    7.1   0.8    3.2  2  CAH8565065.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8431581.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   1.4   8.1e-05      0.21      13      54 ..     920     961 ..     912     977 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 8.1e-05\n",
            "                   HHHHHHHHT.--STTHHHHHHHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "        Zw10_N  13 snmivkkYs.eflellsstqklieqvsslskeisdllekveke 54 \n",
            "                   +n+i++++s ef+  ++  q+l+++++ l  +i+++ + ++++\n",
            "  CAH8431581.1 920 FNKIQNNHSmEFQRQHQRQQQLMDKTDRL-HDINEMSSSTNNS 961\n",
            "                   689************************98.5788888777766 PP\n",
            "\n",
            ">> CAH8288436.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   0.0     0.001       2.7      18      57 ..     132     173 ..     131     176 .. 0.93\n",
            "   2 ?   -0.2   0.2       0.5   1.3e+03      14      48 ..     254     284 ..     240     313 .. 0.54\n",
            "   3 ?   -0.7   0.5      0.73   1.9e+03      32      49 ..     533     550 ..     530     560 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.001\n",
            "                   HHHT..--STTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT CS\n",
            "        Zw10_N  18 kkYs..eflellsstqklieqvsslskeisdllekvekeila 57 \n",
            "                   k++s   f e++s+++++i++    s +i ++l+ v+++il+\n",
            "  CAH8288436.1 132 KHFSpdSFKESMSNCNNIISNNVVNSGDIANTLSGVNDNILR 173\n",
            "                   7899989******************************99875 PP\n",
            "\n",
            "  == domain 2  score: -0.2 bits;  conditional E-value: 0.5\n",
            "                   HHHHHHHT--STTHHHHHHHHHHHHHHHHHHHHHH CS\n",
            "        Zw10_N  14 nmivkkYseflellsstqklieqvsslskeisdll 48 \n",
            "                    + +k  s+f+++l+s+    ++v  +s   +d+ \n",
            "  CAH8288436.1 254 DYSSKLCSNFSNSLTSS----STVTLSSTACNDIY 284\n",
            "                   33344444555555554....44444444444443 PP\n",
            "\n",
            "  == domain 3  score: -0.7 bits;  conditional E-value: 0.73\n",
            "                   HHHHHHHHHHHHHHHHHH CS\n",
            "        Zw10_N  32 klieqvsslskeisdlle 49 \n",
            "                   +l++++ss+s e+s   +\n",
            "  CAH8288436.1 533 ELLSNTSSSSSEMSLSSN 550\n",
            "                   455555555555554333 PP\n",
            "\n",
            ">> CAH8565081.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.2   0.8    0.0024       6.3      26      60 ..     520     552 ..     512     561 .. 0.85\n",
            "   2 ?    1.8   1.0      0.12   3.1e+02      18      62 ..     982    1026 ..     977    1035 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.2 bits;  conditional E-value: 0.0024\n",
            "                   THHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SS CS\n",
            "        Zw10_N  26 llsstqklieqvsslskeisdllekvekeiladlr 60 \n",
            "                    ++ ++k +++++s+sk++  +l ++++e  ++++\n",
            "  CAH8565081.1 520 KHNDSSKYLSESRSVSKDVRSTLTCISEE--EAIQ 552\n",
            "                   689999*********************99..5555 PP\n",
            "\n",
            "  == domain 2  score: 1.8 bits;  conditional E-value: 0.12\n",
            "                    HHHT--STTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SSSS CS\n",
            "        Zw10_N   18 kkYseflellsstqklieqvsslskeisdllekvekeiladlrgs 62  \n",
            "                    +k+s+ ++++ +  ++ie v+ +    sd+l+ v+k++ +  ++s\n",
            "  CAH8565081.1  982 NKHSQQADNHHNDWDTIESVQHSEVSPSDVLQSVSKSGAQKSDSS 1026\n",
            "                    699********************9999*********988877776 PP\n",
            "\n",
            ">> CAH8565097.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.1   0.8    0.0028       7.1      26      60 ..     687     719 ..     679     728 .. 0.85\n",
            "   2 ?    1.6   1.0      0.14   3.5e+02      18      62 ..    1149    1193 ..    1144    1202 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.1 bits;  conditional E-value: 0.0028\n",
            "                   THHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SS CS\n",
            "        Zw10_N  26 llsstqklieqvsslskeisdllekvekeiladlr 60 \n",
            "                    ++ ++k +++++s+sk++  +l ++++e  ++++\n",
            "  CAH8565097.1 687 KHNDSSKYLSESRSVSKDVRSTLTCISEE--EAIQ 719\n",
            "                   689999*********************99..5555 PP\n",
            "\n",
            "  == domain 2  score: 1.6 bits;  conditional E-value: 0.14\n",
            "                    HHHT--STTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SSSS CS\n",
            "        Zw10_N   18 kkYseflellsstqklieqvsslskeisdllekvekeiladlrgs 62  \n",
            "                    +k+s+ ++++ +  ++ie v+ +    sd+l+ v+k++ +  ++s\n",
            "  CAH8565097.1 1149 NKHSQQADNHHNDWDTIESVQHSEVSPSDVLQSVSKSGAQKSDSS 1193\n",
            "                    699********************9999*********988877776 PP\n",
            "\n",
            ">> CAH8565051.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.1   0.8    0.0028       7.1      26      60 ..     687     719 ..     679     728 .. 0.85\n",
            "   2 ?    1.6   1.0      0.14   3.5e+02      18      62 ..    1149    1193 ..    1144    1202 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.1 bits;  conditional E-value: 0.0028\n",
            "                   THHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SS CS\n",
            "        Zw10_N  26 llsstqklieqvsslskeisdllekvekeiladlr 60 \n",
            "                    ++ ++k +++++s+sk++  +l ++++e  ++++\n",
            "  CAH8565051.1 687 KHNDSSKYLSESRSVSKDVRSTLTCISEE--EAIQ 719\n",
            "                   689999*********************99..5555 PP\n",
            "\n",
            "  == domain 2  score: 1.6 bits;  conditional E-value: 0.14\n",
            "                    HHHT--STTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SSSS CS\n",
            "        Zw10_N   18 kkYseflellsstqklieqvsslskeisdllekvekeiladlrgs 62  \n",
            "                    +k+s+ ++++ +  ++ie v+ +    sd+l+ v+k++ +  ++s\n",
            "  CAH8565051.1 1149 NKHSQQADNHHNDWDTIESVQHSEVSPSDVLQSVSKSGAQKSDSS 1193\n",
            "                    699********************9999*********988877776 PP\n",
            "\n",
            ">> CAH8565065.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.1   0.8    0.0028       7.2      26      60 ..     698     730 ..     690     739 .. 0.85\n",
            "   2 ?    1.6   1.0      0.14   3.6e+02      18      62 ..    1160    1204 ..    1155    1213 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.1 bits;  conditional E-value: 0.0028\n",
            "                   THHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SS CS\n",
            "        Zw10_N  26 llsstqklieqvsslskeisdllekvekeiladlr 60 \n",
            "                    ++ ++k +++++s+sk++  +l ++++e  ++++\n",
            "  CAH8565065.1 698 KHNDSSKYLSESRSVSKDVRSTLTCISEE--EAIQ 730\n",
            "                   689999*********************99..5555 PP\n",
            "\n",
            "  == domain 2  score: 1.6 bits;  conditional E-value: 0.14\n",
            "                    HHHT--STTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT-SSSS CS\n",
            "        Zw10_N   18 kkYseflellsstqklieqvsslskeisdllekvekeiladlrgs 62  \n",
            "                    +k+s+ ++++ +  ++ie v+ +    sd+l+ v+k++ +  ++s\n",
            "  CAH8565065.1 1160 NKHSQQADNHHNDWDTIESVQHSEVSPSDVLQSVSKSGAQKSDSS 1204\n",
            "                    699********************9999*********988877776 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (93 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1560  (0.101629); expected 307.0 (0.02)\n",
            "Passed bias filter:                      700  (0.0456026); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        60  (0.00390879); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000456026); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               6  [number of targets reported over threshold]\n",
            "# CPU time: 0.29u 0.01s 00:00:00.30 Elapsed: 00:00:00.24\n",
            "# Mc/sec: 4040.06\n",
            "//\n",
            "Query:       Zwilch  [M=568]\n",
            "Accession:   PF09817.14\n",
            "Description: RZZ complex, subunit zwilch\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (568 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       909  (0.0592182); expected 307.0 (0.02)\n",
            "Passed bias filter:                      197  (0.0128339); expected 307.0 (0.02)\n",
            "Passed Vit filter:                         8  (0.000521173); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.52u 0.00s 00:00:00.52 Elapsed: 00:00:00.43\n",
            "# Mc/sec: 13674.40\n",
            "//\n",
            "Query:       Zwint  [M=252]\n",
            "Accession:   PF15556.11\n",
            "Description: ZW10 interactor\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (252 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1310  (0.085342); expected 307.0 (0.02)\n",
            "Passed bias filter:                      357  (0.0232573); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        24  (0.00156352); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.38u 0.01s 00:00:00.39 Elapsed: 00:00:00.31\n",
            "# Mc/sec: 8370.88\n",
            "//\n",
            "Query:       ZYG-11_interact  [M=295]\n",
            "Accession:   PF05884.17\n",
            "Description: Interactor of ZYG-11\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "  ------ inclusion threshold ------\n",
            "        0.2   10.8   0.4        3.6    6.7   0.1    2.0  2  CAH8290470.1  unnamed protein product [Schistosoma intercalat\n",
            "        0.4    9.8   2.4       0.48    9.6   2.4    1.2  1  CAH8447319.1  unnamed protein product [Schistosoma intercalat\n",
            "        0.5    9.5   3.3          1    8.5   3.3    1.5  1  CAH8521968.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8290470.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.2   0.0     0.017        86      20      49 ..      82     111 ..      75     137 .. 0.82\n",
            "   2 ?    6.7   0.1    0.0007       3.6      33     120 ..     201     306 ..     170     308 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.2 bits;  conditional E-value: 0.017\n",
            "  ZYG-11_interact  20 knegseeeavirlvirtysklgrdleqyye 49 \n",
            "                      ++++s+++ +i+ v++t ++  + +++y  \n",
            "     CAH8290470.1  82 NVNNSQQNNSINSVVNTTNNINNHINNYCF 111\n",
            "                      57899********************99864 PP\n",
            "\n",
            "  == domain 2  score: 6.7 bits;  conditional E-value: 0.0007\n",
            "  ZYG-11_interact  33 virtysklgrdleqyye....................knlkklpeegerlleqllssifsqkskdsvnelvnalvdlsppeklsskniieiFal 106\n",
            "                      + r+  kl+++++  ++                    +n  +l +  e + +   ss+  q++ ++++   +  v++   ++lss +i + + +\n",
            "     CAH8290470.1 201 IHRENIKLSQEVQSTVDvgiqtltcsleeeeeelirqRNCTHLTQSKEITTNLCSSSV--QTYDENLKSTCDFQVQTDFNHNLSSIKIDQHYNN 292\n",
            "                      5555555666666665555544444444444566665777788877777777777777..9********************************* PP\n",
            "\n",
            "  ZYG-11_interact 107 ssilLlvaslssli 120\n",
            "                       s+ Ll  sl +l+\n",
            "     CAH8290470.1 293 RSLQLLNVSLIKLV 306\n",
            "                      ******99998776 PP\n",
            "\n",
            ">> CAH8447319.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.6   2.4   9.3e-05      0.48       9      77 ..      25     100 ..       8     133 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.6 bits;  conditional E-value: 9.3e-05\n",
            "  ZYG-11_interact   9 gealsditeavknegseeeavirlvirtysklgrdleqyye......knlkklpeegerlleqllssi.fsqkskd 77 \n",
            "                       e+l++i +++k+e    +  i+ +++++ k  +++++         k+ +   ++ ++  ++l+s i  +qk +d\n",
            "     CAH8447319.1  25 KEKLKNILSSTKTECNSLKSQIESATEELRKVNKEIQDIQIkkklliKKQAIRRQREQKAQKDLYSLIsNKQKIED 100\n",
            "                      5677888999999999999999999999999999999954444344322222233333444456665542222222 PP\n",
            "\n",
            ">> CAH8521968.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   3.3    0.0002         1      13      77 ..     216     279 ..     203     294 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0002\n",
            "  ZYG-11_interact  13 sditeavknegseeeavirlvir.tysklgrdleqyyeknlkklpeegerlleqllssifsqkskd 77 \n",
            "                      + ++e  kn +++eea ++ +++ + +k+ +d ++ y+k+lk ++e  e  ++++    + ++ +d\n",
            "     CAH8521968.1 216 EYLSEQLKNAQTNEEARLNRATTeKEEKYQQDTKEKYDKQLKIIQEINEYRVDEIER--RRKRLED 279\n",
            "                      4578999***********9987615689*******************9999999887..5555555 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (295 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                       737  (0.048013); expected 307.0 (0.02)\n",
            "Passed bias filter:                      424  (0.0276221); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        63  (0.00410423); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                         8  (0.000521173); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.46u 0.01s 00:00:00.47 Elapsed: 00:00:00.44\n",
            "# Mc/sec: 6956.55\n",
            "//\n",
            "Query:       ZZ  [M=45]\n",
            "Accession:   PF00569.22\n",
            "Description: Zinc finger, ZZ type\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence     Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------     -----------\n",
            "    1.6e-12   46.8   6.4    3.9e-12   45.6   6.4    1.6  1  CAH8639460.1  unnamed protein product [Schistosoma intercalat\n",
            "    1.8e-12   46.6   8.9    1.8e-12   46.6   8.9    3.6  3  CAH8530491.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.1e-12   46.4   8.9    2.1e-12   46.4   8.9    3.5  3  CAH8530475.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.1e-12   46.4   8.9    2.1e-12   46.4   8.9    3.5  3  CAH8530460.1  unnamed protein product [Schistosoma intercalat\n",
            "    7.8e-12   44.6  11.9    1.5e-11   43.7   8.7    2.9  2  CAH8539730.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.3e-10   39.9   8.9    5.2e-10   38.8   8.9    1.6  1  CAH8632423.1  unnamed protein product [Schistosoma intercalat\n",
            "    2.6e-10   39.7   8.9    6.1e-10   38.5   8.9    1.6  1  CAH8632406.1  unnamed protein product [Schistosoma intercalat\n",
            "    7.5e-09   35.1   6.3    7.5e-09   35.1   6.3    3.4  3  CAH8558415.1  unnamed protein product [Schistosoma intercalat\n",
            "    8.6e-09   34.9   6.3    8.6e-09   34.9   6.3    3.6  3  CAH8558400.1  unnamed protein product [Schistosoma intercalat\n",
            "    8.6e-09   34.9   6.3    8.6e-09   34.9   6.3    3.6  3  CAH8558384.1  unnamed protein product [Schistosoma intercalat\n",
            "    4.2e-08   32.7   6.1    4.2e-08   32.7   6.1    3.0  1  CAH8645219.1  unnamed protein product [Schistosoma intercalat\n",
            "    5.3e-05   22.7   4.9    0.00014   21.4   4.9    1.6  1  CAH8505257.1  unnamed protein product, partial [Schistosoma i\n",
            "    0.00043   19.8   7.9    0.00079   19.0   7.9    1.4  1  CAH8495974.1  unnamed protein product [Schistosoma intercalat\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> CAH8639460.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   45.6   6.4   3.3e-15   3.9e-12       4      44 ..    1300    1341 ..    1298    1342 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 45.6 bits;  conditional E-value: 3.3e-15\n",
            "                    EEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTCC CS\n",
            "            ZZ    4 kytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkhk 44  \n",
            "                       C +C + plig Ry+clrc ++D+Cq+Cf++ +++ +hk\n",
            "  CAH8639460.1 1300 HIRCSVCHNQPLIGLRYRCLRCLNFDMCQQCFFAgRTSRNHK 1341\n",
            "                    578**************************************8 PP\n",
            "\n",
            ">> CAH8530491.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.1   1.1      0.26   3.1e+02      23      37 ..     841     855 ..     840     858 .. 0.85\n",
            "   2 ?   -2.9   0.5       4.7   5.5e+03       5      19 ..     891     899 ..     887     904 .. 0.51\n",
            "   3 !   46.6   8.9   1.5e-15   1.8e-12       3      45 .]    1327    1366 ..    1325    1366 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.1 bits;  conditional E-value: 0.26\n",
            "                   SSSTSSEECHHHHHT CS\n",
            "            ZZ  23 lrcsdYDLCqsCfst 37 \n",
            "                   l+c+ Y  C +Cfs+\n",
            "  CAH8530491.1 841 LVCDKYYQCEKCFSE 855\n",
            "                   689999999***987 PP\n",
            "\n",
            "  == domain 2  score: -2.9 bits;  conditional E-value: 4.7\n",
            "                   EBETTTSSBBECSCE CS\n",
            "            ZZ   5 ytCdgCseapligvR 19 \n",
            "                   ++C +C+       R\n",
            "  CAH8530491.1 891 VICVECGR------R 899\n",
            "                   44445544......3 PP\n",
            "\n",
            "  == domain 3  score: 46.6 bits;  conditional E-value: 1.5e-15\n",
            "                    SEEBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ    3 kkytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45  \n",
            "                    + y C++C++   i++R+hc  c++YDLC++C++t   ++h m\n",
            "  CAH8530491.1 1327 FMYNCNVCGA--QIETRWHCNECEEYDLCSRCYKT-ENHPHPM 1366\n",
            "                    679*******..***********************.8888876 PP\n",
            "\n",
            ">> CAH8530475.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.9   1.1       0.3   3.6e+02      23      37 ..     841     855 ..     840     858 .. 0.85\n",
            "   2 ?   -3.1   0.5       5.4   6.4e+03       5      19 ..     891     899 ..     887     904 .. 0.51\n",
            "   3 !   46.4   8.9   1.8e-15   2.1e-12       3      45 .]    1327    1366 ..    1325    1366 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.9 bits;  conditional E-value: 0.3\n",
            "                   SSSTSSEECHHHHHT CS\n",
            "            ZZ  23 lrcsdYDLCqsCfst 37 \n",
            "                   l+c+ Y  C +Cfs+\n",
            "  CAH8530475.1 841 LVCDKYYQCEKCFSE 855\n",
            "                   689999999***987 PP\n",
            "\n",
            "  == domain 2  score: -3.1 bits;  conditional E-value: 5.4\n",
            "                   EBETTTSSBBECSCE CS\n",
            "            ZZ   5 ytCdgCseapligvR 19 \n",
            "                   ++C +C+       R\n",
            "  CAH8530475.1 891 VICVECGR------R 899\n",
            "                   44445544......3 PP\n",
            "\n",
            "  == domain 3  score: 46.4 bits;  conditional E-value: 1.8e-15\n",
            "                    SEEBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ    3 kkytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45  \n",
            "                    + y C++C++   i++R+hc  c++YDLC++C++t   ++h m\n",
            "  CAH8530475.1 1327 FMYNCNVCGA--QIETRWHCNECEEYDLCSRCYKT-ENHPHPM 1366\n",
            "                    679*******..***********************.8888876 PP\n",
            "\n",
            ">> CAH8530460.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.9   1.1      0.31   3.6e+02      23      37 ..     841     855 ..     840     858 .. 0.85\n",
            "   2 ?   -3.2   0.5       5.5   6.4e+03       5      19 ..     891     899 ..     887     904 .. 0.51\n",
            "   3 !   46.4   8.9   1.8e-15   2.1e-12       3      45 .]    1327    1366 ..    1325    1366 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.9 bits;  conditional E-value: 0.31\n",
            "                   SSSTSSEECHHHHHT CS\n",
            "            ZZ  23 lrcsdYDLCqsCfst 37 \n",
            "                   l+c+ Y  C +Cfs+\n",
            "  CAH8530460.1 841 LVCDKYYQCEKCFSE 855\n",
            "                   689999999***987 PP\n",
            "\n",
            "  == domain 2  score: -3.2 bits;  conditional E-value: 5.5\n",
            "                   EBETTTSSBBECSCE CS\n",
            "            ZZ   5 ytCdgCseapligvR 19 \n",
            "                   ++C +C+       R\n",
            "  CAH8530460.1 891 VICVECGR------R 899\n",
            "                   44445544......3 PP\n",
            "\n",
            "  == domain 3  score: 46.4 bits;  conditional E-value: 1.8e-15\n",
            "                    SEEBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ    3 kkytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45  \n",
            "                    + y C++C++   i++R+hc  c++YDLC++C++t   ++h m\n",
            "  CAH8530460.1 1327 FMYNCNVCGA--QIETRWHCNECEEYDLCSRCYKT-ENHPHPM 1366\n",
            "                    679*******..***********************.8888876 PP\n",
            "\n",
            ">> CAH8539730.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   43.7   8.7   1.3e-14   1.5e-11       1      43 [.       4      47 ..       4      49 .. 0.93\n",
            "   2 ?    0.0   0.1      0.56   6.6e+02       4      12 ..      78      86 ..      76      92 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 43.7 bits;  conditional E-value: 1.3e-14\n",
            "                  SSSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTC CS\n",
            "            ZZ  1 ihkkytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkh 43\n",
            "                  +h+++tC+gCs   +   Ry+cl+c+dYDLC  Cf   + ++kh\n",
            "  CAH8539730.1  4 KHEGVTCNGCSRRDFRFRRYKCLICRDYDLCGTCFDNqQETEKH 47\n",
            "                  699*********999999*****************998888888 PP\n",
            "\n",
            "  == domain 2  score: 0.0 bits;  conditional E-value: 0.56\n",
            "                  EEBETTTSS CS\n",
            "            ZZ  4 kytCdgCse 12\n",
            "                   +tC +C++\n",
            "  CAH8539730.1 78 SFTCSVCGQ 86\n",
            "                  689****97 PP\n",
            "\n",
            ">> CAH8632423.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   38.8   8.9   4.4e-13   5.2e-10       2      37 ..     329     364 ..     328     372 .. 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 38.8 bits;  conditional E-value: 4.4e-13\n",
            "                   SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "            ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                   ++++ C+gC+  p+ g Ry+c+rc+ Y+LCq Cf  \n",
            "  CAH8632423.1 329 RHNVRCEGCKREPICGLRYKCTRCPHYNLCQDCFWI 364\n",
            "                   6899******************************65 PP\n",
            "\n",
            ">> CAH8632406.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   38.5   8.9   5.1e-13   6.1e-10       2      37 ..     329     364 ..     328     372 .. 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 38.5 bits;  conditional E-value: 5.1e-13\n",
            "                   SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "            ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                   ++++ C+gC+  p+ g Ry+c+rc+ Y+LCq Cf  \n",
            "  CAH8632406.1 329 RHNVRCEGCKREPICGLRYKCTRCPHYNLCQDCFWI 364\n",
            "                   6899******************************65 PP\n",
            "\n",
            ">> CAH8558415.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.3   0.3       1.4   1.7e+03      28      37 ..     351     360 ..     349     363 .. 0.84\n",
            "   2 ?   -3.0   0.4         5     6e+03       4      10 ..     420     426 ..     419     428 .. 0.82\n",
            "   3 !   35.1   6.3   6.3e-12   7.5e-09       5      45 .]     832     869 ..     830     869 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.3 bits;  conditional E-value: 1.4\n",
            "                   SEECHHHHHT CS\n",
            "            ZZ  28 YDLCqsCfst 37 \n",
            "                   Y  C +C+++\n",
            "  CAH8558415.1 351 YTVCERCYNE 360\n",
            "                   789****997 PP\n",
            "\n",
            "  == domain 2  score: -3.0 bits;  conditional E-value: 5\n",
            "                   EEBETTT CS\n",
            "            ZZ   4 kytCdgC 10 \n",
            "                   +++C+ C\n",
            "  CAH8558415.1 420 GFICNHC 426\n",
            "                   6899999 PP\n",
            "\n",
            "  == domain 3  score: 35.1 bits;  conditional E-value: 6.3e-12\n",
            "                   EBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ   5 ytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45 \n",
            "                   ytC++C +    ++ +hc +c +YDLC +C+ t   ++h m\n",
            "  CAH8558415.1 832 YTCNVCNA--QLETPWHCKQCIEYDLCPRCYET-ENHPHPM 869\n",
            "                   9*******..9**********************.8888876 PP\n",
            "\n",
            ">> CAH8558400.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.5   0.3       1.6   1.9e+03      28      37 ..     567     576 ..     565     579 .. 0.84\n",
            "   2 ?   -3.2   0.4       5.7   6.8e+03       4      10 ..     636     642 ..     635     644 .. 0.82\n",
            "   3 !   34.9   6.3   7.3e-12   8.6e-09       5      45 .]    1048    1085 ..    1046    1085 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.5 bits;  conditional E-value: 1.6\n",
            "                   SEECHHHHHT CS\n",
            "            ZZ  28 YDLCqsCfst 37 \n",
            "                   Y  C +C+++\n",
            "  CAH8558400.1 567 YTVCERCYNE 576\n",
            "                   789****997 PP\n",
            "\n",
            "  == domain 2  score: -3.2 bits;  conditional E-value: 5.7\n",
            "                   EEBETTT CS\n",
            "            ZZ   4 kytCdgC 10 \n",
            "                   +++C+ C\n",
            "  CAH8558400.1 636 GFICNHC 642\n",
            "                   6899999 PP\n",
            "\n",
            "  == domain 3  score: 34.9 bits;  conditional E-value: 7.3e-12\n",
            "                    EBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ    5 ytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45  \n",
            "                    ytC++C +    ++ +hc +c +YDLC +C+ t   ++h m\n",
            "  CAH8558400.1 1048 YTCNVCNA--QLETPWHCKQCIEYDLCPRCYET-ENHPHPM 1085\n",
            "                    9*******..9**********************.8888876 PP\n",
            "\n",
            ">> CAH8558384.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.5   0.3       1.6   1.9e+03      28      37 ..     572     581 ..     570     584 .. 0.84\n",
            "   2 ?   -3.2   0.4       5.7   6.8e+03       4      10 ..     641     647 ..     640     649 .. 0.82\n",
            "   3 !   34.9   6.3   7.3e-12   8.6e-09       5      45 .]    1053    1090 ..    1051    1090 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.5 bits;  conditional E-value: 1.6\n",
            "                   SEECHHHHHT CS\n",
            "            ZZ  28 YDLCqsCfst 37 \n",
            "                   Y  C +C+++\n",
            "  CAH8558384.1 572 YTVCERCYNE 581\n",
            "                   789****997 PP\n",
            "\n",
            "  == domain 2  score: -3.2 bits;  conditional E-value: 5.7\n",
            "                   EEBETTT CS\n",
            "            ZZ   4 kytCdgC 10 \n",
            "                   +++C+ C\n",
            "  CAH8558384.1 641 GFICNHC 647\n",
            "                   6899999 PP\n",
            "\n",
            "  == domain 3  score: 34.9 bits;  conditional E-value: 7.3e-12\n",
            "                    EBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ    5 ytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45  \n",
            "                    ytC++C +    ++ +hc +c +YDLC +C+ t   ++h m\n",
            "  CAH8558384.1 1053 YTCNVCNA--QLETPWHCKQCIEYDLCPRCYET-ENHPHPM 1090\n",
            "                    9*******..9**********************.8888876 PP\n",
            "\n",
            ">> CAH8645219.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   32.7   6.1   3.5e-11   4.2e-08       2      44 ..      92     135 ..      91     136 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 32.7 bits;  conditional E-value: 3.5e-11\n",
            "                   SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTCC CS\n",
            "            ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkhk 44 \n",
            "                   h + +C+ C e p+ g R++c++c  +DLC+ C+ + + +  h+\n",
            "  CAH8645219.1  92 HPGTICESCHESPIYGIRWKCVVCLSTDLCSTCYHAdKHSLAHQ 135\n",
            "                   66789******************************977766665 PP\n",
            "\n",
            ">> CAH8505257.1  unnamed protein product, partial [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   21.4   4.9   1.2e-07   0.00014       7      43 ..     135     171 ..     131     172 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 21.4 bits;  conditional E-value: 1.2e-07\n",
            "                   ETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTC CS\n",
            "            ZZ   7 CdgCseapligvRyhclrcsdYDLCqsCfsthkagkh 43 \n",
            "                   C  C +    g Ry cl c++  LC +Cf t ++++h\n",
            "  CAH8505257.1 135 CKLCEKFKWSGERYTCLFCPNIVLCRECFYTGYHKEH 171\n",
            "                   999999999**********************999888 PP\n",
            "\n",
            ">> CAH8495974.1  unnamed protein product [Schistosoma intercalatum]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.0   7.9   6.7e-07   0.00079       5      45 .]     144     184 ..     142     184 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.0 bits;  conditional E-value: 6.7e-07\n",
            "                   EBETTTSSBBECSCEEEESSSTSSEECHHHHHTGCBTTCCC CS\n",
            "            ZZ   5 ytCdgCseapligvRyhclrcsdYDLCqsCfsthkagkhkm 45 \n",
            "                    +C++C +    g R+ c++c+   LC  Cf+  ++ +h m\n",
            "  CAH8495974.1 144 LICNICNKNNWCGDRFTCVICPRVVLCPDCFKNNNHSEHPM 184\n",
            "                   68*******************************99999977 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (45 nodes)\n",
            "Target sequences:                      15350  (10474882 residues searched)\n",
            "Passed MSV filter:                      1239  (0.0807166); expected 307.0 (0.02)\n",
            "Passed bias filter:                      405  (0.0263844); expected 307.0 (0.02)\n",
            "Passed Vit filter:                        47  (0.00306189); expected 15.3 (0.001)\n",
            "Passed Fwd filter:                        15  (0.000977199); expected 0.2 (1e-05)\n",
            "Initial search space (Z):              15350  [actual number of targets]\n",
            "Domain search space  (domZ):              13  [number of targets reported over threshold]\n",
            "# CPU time: 0.22u 0.00s 00:00:00.22 Elapsed: 00:00:00.22\n",
            "# Mc/sec: 2113.39\n",
            "//\n",
            "[ok]\n"
          ]
        }
      ],
      "source": [
        "!hmmsearch --tblout test.out Pfam-A.hmm GCA_944470385.2_tdSchInte2.2_protein.faa"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "OrUXAmSqHXvw"
      },
      "source": [
        "и фильтр"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "CYrmpVOxsyMs"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "\n",
        "# выбрала рандомные из selected\n",
        "selected_domains = [\n",
        "    'PF00022', 'PF00046', 'PF00096', 'PF08210', 'PF05240', 'PF01388',\n",
        "    'PF00010', 'PF02752', 'PF13919', 'PF00004'\n",
        "]\n",
        "\n",
        "# Парсинг файла test.out\n",
        "hits = []\n",
        "with open('test.out') as f:\n",
        "    for line in f:\n",
        "        if line.startswith('#'):\n",
        "            continue\n",
        "        fields = line.strip().split()\n",
        "        protein_id = fields[0]\n",
        "        pfam_id = fields[3]\n",
        "        # print(pfam_id)\n",
        "        for id in selected_domains:\n",
        "            if id in pfam_id:\n",
        "              # print(1)\n",
        "              hits.append({'Protein': protein_id, 'Pfam': pfam_id})\n",
        "\n",
        "df = pd.DataFrame(hits)\n",
        "df.to_csv('epigenetic_genes.csv', index=False)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 423
        },
        "id": "dIw53k-Es6kH",
        "outputId": "3238b5f9-c789-49ed-f246-e15dff232754"
      },
      "outputs": [
        {
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 565,\n  \"fields\": [\n    {\n      \"column\": \"Protein\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 564,\n        \"samples\": [\n          \"CAH8567151.1\",\n          \"CAH8575424.1\",\n          \"CAH8552863.1\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"Pfam\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"num_unique_values\": 10,\n        \"samples\": [\n          \"PF00046.34\",\n          \"PF00022.24\",\n          \"PF02752.27\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}",
              "type": "dataframe",
              "variable_name": "df"
            },
            "text/html": [
              "\n",
              "  <div id=\"df-db81b3e3-5cc5-481a-9dfe-57d5f4398395\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Protein</th>\n",
              "      <th>Pfam</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>CAH8640559.1</td>\n",
              "      <td>PF00004.34</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>CAH8446709.1</td>\n",
              "      <td>PF00004.34</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>CAH8603513.1</td>\n",
              "      <td>PF00004.34</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>CAH8603497.1</td>\n",
              "      <td>PF00004.34</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>CAH8626915.1</td>\n",
              "      <td>PF00004.34</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>560</th>\n",
              "      <td>CAH8637121.1</td>\n",
              "      <td>PF00096.31</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>561</th>\n",
              "      <td>CAH8471300.1</td>\n",
              "      <td>PF00096.31</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>562</th>\n",
              "      <td>CAH8584730.1</td>\n",
              "      <td>PF00096.31</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>563</th>\n",
              "      <td>CAH8539730.1</td>\n",
              "      <td>PF00096.31</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>564</th>\n",
              "      <td>CAH8541810.1</td>\n",
              "      <td>PF00096.31</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>565 rows × 2 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-db81b3e3-5cc5-481a-9dfe-57d5f4398395')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-db81b3e3-5cc5-481a-9dfe-57d5f4398395 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-db81b3e3-5cc5-481a-9dfe-57d5f4398395');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "    <div id=\"df-9300834c-7aef-4960-95bf-577141a18f2d\">\n",
              "      <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-9300834c-7aef-4960-95bf-577141a18f2d')\"\n",
              "                title=\"Suggest charts\"\n",
              "                style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "      </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "      <script>\n",
              "        async function quickchart(key) {\n",
              "          const quickchartButtonEl =\n",
              "            document.querySelector('#' + key + ' button');\n",
              "          quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "          quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "          try {\n",
              "            const charts = await google.colab.kernel.invokeFunction(\n",
              "                'suggestCharts', [key], {});\n",
              "          } catch (error) {\n",
              "            console.error('Error during call to suggestCharts:', error);\n",
              "          }\n",
              "          quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "          quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "        }\n",
              "        (() => {\n",
              "          let quickchartButtonEl =\n",
              "            document.querySelector('#df-9300834c-7aef-4960-95bf-577141a18f2d button');\n",
              "          quickchartButtonEl.style.display =\n",
              "            google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "        })();\n",
              "      </script>\n",
              "    </div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "text/plain": [
              "          Protein        Pfam\n",
              "0    CAH8640559.1  PF00004.34\n",
              "1    CAH8446709.1  PF00004.34\n",
              "2    CAH8603513.1  PF00004.34\n",
              "3    CAH8603497.1  PF00004.34\n",
              "4    CAH8626915.1  PF00004.34\n",
              "..            ...         ...\n",
              "560  CAH8637121.1  PF00096.31\n",
              "561  CAH8471300.1  PF00096.31\n",
              "562  CAH8584730.1  PF00096.31\n",
              "563  CAH8539730.1  PF00096.31\n",
              "564  CAH8541810.1  PF00096.31\n",
              "\n",
              "[565 rows x 2 columns]"
            ]
          },
          "execution_count": 45,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "df"
      ]
    },
    {
      "attachments": {},
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "Поиск вторичных структур"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {},
      "outputs": [],
      "source": [
        "from Bio import SeqIO\n",
        "from Bio.Seq import Seq\n",
        "from Bio.Data import CodonTable\n",
        "from Bio import Entrez\n",
        "from Bio.SeqFeature import SeqFeature, FeatureLocation\n",
        "import pandas as pd"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {},
      "outputs": [],
      "source": [
        "scaffolds = {}\n",
        "for scaffold in SeqIO.parse('data/GCF_013758885.1_VT_AalbS3_pri_1.0_genomic.fna', format='fasta'):\n",
        "    scaffolds[scaffold.id] = scaffold"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Collecting pybedtools\n",
            "  Downloading pybedtools-0.12.0.tar.gz (12.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.6/12.6 MB\u001b[0m \u001b[31m1.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0ma \u001b[36m0:00:01\u001b[0m\n",
            "\u001b[?25h  Installing build dependencies ... \u001b[?25ldone\n",
            "\u001b[?25h  Getting requirements to build wheel ... \u001b[?25ldone\n",
            "\u001b[?25h  Preparing metadata (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25hRequirement already satisfied: numpy in /opt/homebrew/lib/python3.11/site-packages (from pybedtools) (1.26.0)\n",
            "Requirement already satisfied: pysam in /opt/homebrew/lib/python3.11/site-packages (from pybedtools) (0.22.1)\n",
            "Requirement already satisfied: pandas in /opt/homebrew/lib/python3.11/site-packages (from pybedtools) (2.2.2)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /Users/sofanelipovich/Library/Python/3.11/lib/python/site-packages (from pandas->pybedtools) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /opt/homebrew/lib/python3.11/site-packages (from pandas->pybedtools) (2023.3.post1)\n",
            "Requirement already satisfied: tzdata>=2022.7 in /opt/homebrew/lib/python3.11/site-packages (from pandas->pybedtools) (2023.3)\n",
            "Requirement already satisfied: six>=1.5 in /Users/sofanelipovich/Library/Python/3.11/lib/python/site-packages (from python-dateutil>=2.8.2->pandas->pybedtools) (1.16.0)\n",
            "Building wheels for collected packages: pybedtools\n",
            "  Building wheel for pybedtools (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25h  Created wheel for pybedtools: filename=pybedtools-0.12.0-cp311-cp311-macosx_13_0_arm64.whl size=12858820 sha256=00bf7be27ea11c3aafe4904735078c099f62b2957460f73ba2eeee7ba5b60adc\n",
            "  Stored in directory: /Users/sofanelipovich/Library/Caches/pip/wheels/95/62/57/e8c47c2a43884227b6644cf1d2dabf3162a0f40b9c1493c1f5\n",
            "Successfully built pybedtools\n",
            "Installing collected packages: pybedtools\n",
            "Successfully installed pybedtools-0.12.0\n"
          ]
        }
      ],
      "source": [
        "!pip install pybedtools"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {},
      "outputs": [],
      "source": [
        "import pybedtools\n",
        "import pandas as pd\n",
        "import re"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "5664\n",
            "5555\n"
          ]
        }
      ],
      "source": [
        "PQS = []\n",
        "PQS_minus = []\n",
        "pattern=\"(?:G{3,5}[ATGC]{1,7}){3,}G{3,5}\"\n",
        "pattern_minus=\"(?:C{3,5}[ATGC]{1,7}){3,}C{3,5}\"\n",
        "for scaffold, record in scaffolds.items():\n",
        "    name, sequence = record.id, str(record.seq)\n",
        "    PQS += [[name, m.start(),m.end(),m.group(0), name, '+'] for m in re.finditer(pattern,sequence,re.IGNORECASE)]\n",
        "    PQS_minus += [[name, m.start(),m.end(),m.group(0), name, '-'] for m in re.finditer(pattern_minus,sequence,re.IGNORECASE)]\n",
        "print(len(PQS))\n",
        "print(len(PQS_minus))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Chromosome</th>\n",
              "      <th>Start</th>\n",
              "      <th>End</th>\n",
              "      <th>Sequence</th>\n",
              "      <th>Name</th>\n",
              "      <th>Strand</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>37724</td>\n",
              "      <td>37745</td>\n",
              "      <td>GGGTGGGGGGAGAGGGAGGGG</td>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>+</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>41678</td>\n",
              "      <td>41702</td>\n",
              "      <td>GGGGAGTTTGGGAAGAAGGGTGGG</td>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>+</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>48274</td>\n",
              "      <td>48290</td>\n",
              "      <td>ggggggggaggggggg</td>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>+</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>86744</td>\n",
              "      <td>86761</td>\n",
              "      <td>gggaggggggggggggg</td>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>+</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>87959</td>\n",
              "      <td>88001</td>\n",
              "      <td>GGGGTGAGTGGGAAATGGGTccatgggggggggggggggggg</td>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>+</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11214</th>\n",
              "      <td>NC_050157.1</td>\n",
              "      <td>67952557</td>\n",
              "      <td>67952580</td>\n",
              "      <td>ccccccccccccccccccccccc</td>\n",
              "      <td>NC_050157.1</td>\n",
              "      <td>-</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11215</th>\n",
              "      <td>NW_023396103.1</td>\n",
              "      <td>8929</td>\n",
              "      <td>8953</td>\n",
              "      <td>cccttcggcccttgcgcccttccc</td>\n",
              "      <td>NW_023396103.1</td>\n",
              "      <td>-</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11216</th>\n",
              "      <td>NW_023396103.1</td>\n",
              "      <td>732534</td>\n",
              "      <td>732555</td>\n",
              "      <td>CCCACCCACTCTACCCTTCCC</td>\n",
              "      <td>NW_023396103.1</td>\n",
              "      <td>-</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11217</th>\n",
              "      <td>NW_023396104.1</td>\n",
              "      <td>190870</td>\n",
              "      <td>190922</td>\n",
              "      <td>CCCCGGCCCCCACCGTTTCCCCAGCCACCCCCTCCCGCTCCCGTAG...</td>\n",
              "      <td>NW_023396104.1</td>\n",
              "      <td>-</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>11218</th>\n",
              "      <td>NW_023396104.1</td>\n",
              "      <td>191827</td>\n",
              "      <td>191860</td>\n",
              "      <td>CCCACCAAACCCCACCCTGTTCCCCAAACCCCC</td>\n",
              "      <td>NW_023396104.1</td>\n",
              "      <td>-</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>11219 rows × 6 columns</p>\n",
              "</div>"
            ],
            "text/plain": [
              "           Chromosome     Start       End  \\\n",
              "0         NC_050155.1     37724     37745   \n",
              "1         NC_050155.1     41678     41702   \n",
              "2         NC_050155.1     48274     48290   \n",
              "3         NC_050155.1     86744     86761   \n",
              "4         NC_050155.1     87959     88001   \n",
              "...               ...       ...       ...   \n",
              "11214     NC_050157.1  67952557  67952580   \n",
              "11215  NW_023396103.1      8929      8953   \n",
              "11216  NW_023396103.1    732534    732555   \n",
              "11217  NW_023396104.1    190870    190922   \n",
              "11218  NW_023396104.1    191827    191860   \n",
              "\n",
              "                                                Sequence            Name  \\\n",
              "0                                  GGGTGGGGGGAGAGGGAGGGG     NC_050155.1   \n",
              "1                               GGGGAGTTTGGGAAGAAGGGTGGG     NC_050155.1   \n",
              "2                                       ggggggggaggggggg     NC_050155.1   \n",
              "3                                      gggaggggggggggggg     NC_050155.1   \n",
              "4             GGGGTGAGTGGGAAATGGGTccatgggggggggggggggggg     NC_050155.1   \n",
              "...                                                  ...             ...   \n",
              "11214                            ccccccccccccccccccccccc     NC_050157.1   \n",
              "11215                           cccttcggcccttgcgcccttccc  NW_023396103.1   \n",
              "11216                              CCCACCCACTCTACCCTTCCC  NW_023396103.1   \n",
              "11217  CCCCGGCCCCCACCGTTTCCCCAGCCACCCCCTCCCGCTCCCGTAG...  NW_023396104.1   \n",
              "11218                  CCCACCAAACCCCACCCTGTTCCCCAAACCCCC  NW_023396104.1   \n",
              "\n",
              "      Strand  \n",
              "0          +  \n",
              "1          +  \n",
              "2          +  \n",
              "3          +  \n",
              "4          +  \n",
              "...      ...  \n",
              "11214      -  \n",
              "11215      -  \n",
              "11216      -  \n",
              "11217      -  \n",
              "11218      -  \n",
              "\n",
              "[11219 rows x 6 columns]"
            ]
          },
          "execution_count": 7,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "quadrs = pd.DataFrame(PQS+PQS_minus, columns=['Chromosome', 'Start', 'End', 'Sequence', 'Name', 'Strand'])\n",
        "quadrs"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 9,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Collecting fuc\n",
            "  Downloading fuc-0.38.0-py3-none-any.whl.metadata (16 kB)\n",
            "Requirement already satisfied: biopython in /opt/homebrew/lib/python3.11/site-packages (from fuc) (1.83)\n",
            "Collecting lxml (from fuc)\n",
            "  Downloading lxml-5.4.0-cp311-cp311-macosx_10_9_universal2.whl.metadata (3.5 kB)\n",
            "Requirement already satisfied: matplotlib in /opt/homebrew/lib/python3.11/site-packages (from fuc) (3.8.0)\n",
            "Collecting matplotlib-venn (from fuc)\n",
            "  Downloading matplotlib-venn-1.1.2.tar.gz (40 kB)\n",
            "  Installing build dependencies ... \u001b[?25ldone\n",
            "\u001b[?25h  Getting requirements to build wheel ... \u001b[?25ldone\n",
            "\u001b[?25h  Installing backend dependencies ... \u001b[?25ldone\n",
            "\u001b[?25h  Preparing metadata (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25hRequirement already satisfied: numpy in /opt/homebrew/lib/python3.11/site-packages (from fuc) (1.26.0)\n",
            "Requirement already satisfied: pandas in /opt/homebrew/lib/python3.11/site-packages (from fuc) (2.2.2)\n",
            "Collecting pyranges (from fuc)\n",
            "  Downloading pyranges-0.1.4-py3-none-any.whl.metadata (3.7 kB)\n",
            "Requirement already satisfied: pysam in /opt/homebrew/lib/python3.11/site-packages (from fuc) (0.22.1)\n",
            "Requirement already satisfied: scipy in /opt/homebrew/lib/python3.11/site-packages (from fuc) (1.12.0)\n",
            "Requirement already satisfied: seaborn in /opt/homebrew/lib/python3.11/site-packages (from fuc) (0.13.0)\n",
            "Requirement already satisfied: statsmodels in /opt/homebrew/lib/python3.11/site-packages (from fuc) (0.14.4)\n",
            "Requirement already satisfied: contourpy>=1.0.1 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (1.1.1)\n",
            "Requirement already satisfied: cycler>=0.10 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (0.12.1)\n",
            "Requirement already satisfied: fonttools>=4.22.0 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (4.43.1)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (1.4.5)\n",
            "Requirement already satisfied: packaging>=20.0 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (23.2)\n",
            "Requirement already satisfied: pillow>=6.2.0 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (10.1.0)\n",
            "Requirement already satisfied: pyparsing>=2.3.1 in /opt/homebrew/lib/python3.11/site-packages (from matplotlib->fuc) (3.1.1)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /Users/sofanelipovich/Library/Python/3.11/lib/python/site-packages (from matplotlib->fuc) (2.8.2)\n",
            "Requirement already satisfied: six>=1.5 in /Users/sofanelipovich/Library/Python/3.11/lib/python/site-packages (from python-dateutil>=2.7->matplotlib->fuc) (1.16.0)\n",
            "Requirement already satisfied: pytz>=2020.1 in /opt/homebrew/lib/python3.11/site-packages (from pandas->fuc) (2023.3.post1)\n",
            "Requirement already satisfied: tzdata>=2022.7 in /opt/homebrew/lib/python3.11/site-packages (from pandas->fuc) (2023.3)\n",
            "Collecting ncls>=0.0.63 (from pyranges->fuc)\n",
            "  Downloading ncls-0.0.68.tar.gz (483 kB)\n",
            "  Installing build dependencies ... \u001b[?25ldone\n",
            "\u001b[?25h  Getting requirements to build wheel ... \u001b[?25ldone\n",
            "\u001b[?25h  Preparing metadata (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25hRequirement already satisfied: tabulate in /opt/homebrew/lib/python3.11/site-packages (from pyranges->fuc) (0.9.0)\n",
            "Collecting sorted_nearest>=0.0.33 (from pyranges->fuc)\n",
            "  Downloading sorted_nearest-0.0.39.tar.gz (1.2 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.2/1.2 MB\u001b[0m \u001b[31m10.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Installing build dependencies ... \u001b[?25ldone\n",
            "\u001b[?25h  Getting requirements to build wheel ... \u001b[?25ldone\n",
            "\u001b[?25h  Preparing metadata (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25hCollecting natsort (from pyranges->fuc)\n",
            "  Downloading natsort-8.4.0-py3-none-any.whl.metadata (21 kB)\n",
            "Requirement already satisfied: patsy>=0.5.6 in /opt/homebrew/lib/python3.11/site-packages (from statsmodels->fuc) (1.0.1)\n",
            "Downloading fuc-0.38.0-py3-none-any.whl (154 kB)\n",
            "Downloading lxml-5.4.0-cp311-cp311-macosx_10_9_universal2.whl (8.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m8.1/8.1 MB\u001b[0m \u001b[31m11.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0ma \u001b[36m0:00:01\u001b[0m\n",
            "\u001b[?25hDownloading pyranges-0.1.4-py3-none-any.whl (1.5 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.5/1.5 MB\u001b[0m \u001b[31m10.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading natsort-8.4.0-py3-none-any.whl (38 kB)\n",
            "Building wheels for collected packages: matplotlib-venn, ncls, sorted_nearest\n",
            "  Building wheel for matplotlib-venn (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25h  Created wheel for matplotlib-venn: filename=matplotlib_venn-1.1.2-py3-none-any.whl size=45298 sha256=468449d20f6bb6f296505f31e63b17008c8f79d84fe397bbb49a0d8582f0ef24\n",
            "  Stored in directory: /Users/sofanelipovich/Library/Caches/pip/wheels/c4/16/4a/93374cd955bbe0553a8ce74db7f0fbd5fcc876e19fb78bd601\n",
            "  Building wheel for ncls (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25h  Created wheel for ncls: filename=ncls-0.0.68-cp311-cp311-macosx_13_0_arm64.whl size=810654 sha256=bf33fbf1019bdc588b9d4ad5c2d78dc9d0d7fb90730eada9a1c690a517d99637\n",
            "  Stored in directory: /Users/sofanelipovich/Library/Caches/pip/wheels/18/d6/b2/1d6471d08c6fa78d03b46f18f9de2d3b83b53cebb023e72727\n",
            "  Building wheel for sorted_nearest (pyproject.toml) ... \u001b[?25ldone\n",
            "\u001b[?25h  Created wheel for sorted_nearest: filename=sorted_nearest-0.0.39-cp311-cp311-macosx_13_0_arm64.whl size=2035981 sha256=22c9449d8a2a36a740e22ee73bab4aeedade151cd7b7cd652d8f04621ddc969f\n",
            "  Stored in directory: /Users/sofanelipovich/Library/Caches/pip/wheels/60/14/d4/14a058aed24c248ed7d01b23ea9efbede6d2afe2f6a5ee7282\n",
            "Successfully built matplotlib-venn ncls sorted_nearest\n",
            "Installing collected packages: sorted_nearest, ncls, natsort, lxml, pyranges, matplotlib-venn, fuc\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m7/7\u001b[0m [fuc]\u001b[32m3/7\u001b[0m [lxml]\n",
            "\u001b[1A\u001b[2KSuccessfully installed fuc-0.38.0 lxml-5.4.0 matplotlib-venn-1.1.2 natsort-8.4.0 ncls-0.0.68 pyranges-0.1.4 sorted_nearest-0.0.39\n"
          ]
        }
      ],
      "source": [
        "!pip install fuc"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {},
      "outputs": [],
      "source": [
        "from fuc import pybed\n",
        "bf = pybed.BedFrame.from_frame(meta=[], data=quadrs)\n",
        "bf.to_file('quadruplex.bed')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {},
      "outputs": [],
      "source": [
        "!gunzip data/GCF_013758885.1_VT_AalbS3_pri_1.0_genomic.gff.gz"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {},
      "outputs": [],
      "source": [
        "annotation_file = 'data/GCF_013758885.1_VT_AalbS3_pri_1.0_genomic.gff'"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {},
      "outputs": [],
      "source": [
        "with open('./data/scaffolds_sizes.txt', 'w') as out:\n",
        "    for scaffold in scaffolds:\n",
        "        scaffold_name = scaffold\n",
        "        scaffold_length = len(scaffolds[scaffold].seq)\n",
        "        out.write(f\"{scaffold_name}\\t{scaffold_length}\\n\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {},
      "outputs": [
        {
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>scaffold</th>\n",
              "      <th>source</th>\n",
              "      <th>type</th>\n",
              "      <th>start</th>\n",
              "      <th>end</th>\n",
              "      <th>score</th>\n",
              "      <th>strand</th>\n",
              "      <th>phase</th>\n",
              "      <th>attributes</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>RefSeq</td>\n",
              "      <td>region</td>\n",
              "      <td>1</td>\n",
              "      <td>13047563</td>\n",
              "      <td>.</td>\n",
              "      <td>+</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=NC_050155.1:1..13047563;Dbxref=taxon:7167;N...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>Gnomon</td>\n",
              "      <td>gene</td>\n",
              "      <td>12555</td>\n",
              "      <td>14767</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=gene-LOC118466793;Dbxref=GeneID:118466793;N...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>Gnomon</td>\n",
              "      <td>lnc_RNA</td>\n",
              "      <td>12555</td>\n",
              "      <td>14767</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=rna-XR_004845662.1;Parent=gene-LOC118466793...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>Gnomon</td>\n",
              "      <td>exon</td>\n",
              "      <td>14726</td>\n",
              "      <td>14767</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=exon-XR_004845662.1-1;Parent=rna-XR_0048456...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>NC_050155.1</td>\n",
              "      <td>Gnomon</td>\n",
              "      <td>exon</td>\n",
              "      <td>12911</td>\n",
              "      <td>13230</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=exon-XR_004845662.1-2;Parent=rna-XR_0048456...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>376988</th>\n",
              "      <td>NW_023396105.1</td>\n",
              "      <td>cmsearch</td>\n",
              "      <td>gene</td>\n",
              "      <td>795561</td>\n",
              "      <td>795713</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=gene-LOC118468282;Dbxref=GeneID:118468282;N...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>376989</th>\n",
              "      <td>NW_023396105.1</td>\n",
              "      <td>cmsearch</td>\n",
              "      <td>rRNA</td>\n",
              "      <td>795561</td>\n",
              "      <td>795713</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=rna-XR_004845886.1;Parent=gene-LOC118468282...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>376990</th>\n",
              "      <td>NW_023396105.1</td>\n",
              "      <td>cmsearch</td>\n",
              "      <td>exon</td>\n",
              "      <td>795561</td>\n",
              "      <td>795713</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=exon-XR_004845886.1-1;Parent=rna-XR_0048458...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>376991</th>\n",
              "      <td>NW_023396105.1</td>\n",
              "      <td>cmsearch</td>\n",
              "      <td>pseudogene</td>\n",
              "      <td>800108</td>\n",
              "      <td>803511</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=gene-LOC118468251;Dbxref=GeneID:118468251;N...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>376992</th>\n",
              "      <td>NW_023396105.1</td>\n",
              "      <td>cmsearch</td>\n",
              "      <td>exon</td>\n",
              "      <td>800108</td>\n",
              "      <td>803511</td>\n",
              "      <td>.</td>\n",
              "      <td>-</td>\n",
              "      <td>.</td>\n",
              "      <td>ID=id-LOC118468251;Parent=gene-LOC118468251;Db...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>376993 rows × 9 columns</p>\n",
              "</div>"
            ],
            "text/plain": [
              "              scaffold    source        type   start       end score strand  \\\n",
              "0          NC_050155.1    RefSeq      region       1  13047563     .      +   \n",
              "1          NC_050155.1    Gnomon        gene   12555     14767     .      -   \n",
              "2          NC_050155.1    Gnomon     lnc_RNA   12555     14767     .      -   \n",
              "3          NC_050155.1    Gnomon        exon   14726     14767     .      -   \n",
              "4          NC_050155.1    Gnomon        exon   12911     13230     .      -   \n",
              "...                ...       ...         ...     ...       ...   ...    ...   \n",
              "376988  NW_023396105.1  cmsearch        gene  795561    795713     .      -   \n",
              "376989  NW_023396105.1  cmsearch        rRNA  795561    795713     .      -   \n",
              "376990  NW_023396105.1  cmsearch        exon  795561    795713     .      -   \n",
              "376991  NW_023396105.1  cmsearch  pseudogene  800108    803511     .      -   \n",
              "376992  NW_023396105.1  cmsearch        exon  800108    803511     .      -   \n",
              "\n",
              "       phase                                         attributes  \n",
              "0          .  ID=NC_050155.1:1..13047563;Dbxref=taxon:7167;N...  \n",
              "1          .  ID=gene-LOC118466793;Dbxref=GeneID:118466793;N...  \n",
              "2          .  ID=rna-XR_004845662.1;Parent=gene-LOC118466793...  \n",
              "3          .  ID=exon-XR_004845662.1-1;Parent=rna-XR_0048456...  \n",
              "4          .  ID=exon-XR_004845662.1-2;Parent=rna-XR_0048456...  \n",
              "...      ...                                                ...  \n",
              "376988     .  ID=gene-LOC118468282;Dbxref=GeneID:118468282;N...  \n",
              "376989     .  ID=rna-XR_004845886.1;Parent=gene-LOC118468282...  \n",
              "376990     .  ID=exon-XR_004845886.1-1;Parent=rna-XR_0048458...  \n",
              "376991     .  ID=gene-LOC118468251;Dbxref=GeneID:118468251;N...  \n",
              "376992     .  ID=id-LOC118468251;Parent=gene-LOC118468251;Db...  \n",
              "\n",
              "[376993 rows x 9 columns]"
            ]
          },
          "execution_count": 15,
          "metadata": {},
          "output_type": "execute_result"
        }
      ],
      "source": [
        "import pandas as pd\n",
        "annotation_df = pd.read_csv(annotation_file, sep='\\t', comment='#', header=None, names=[\n",
        "    \"scaffold\", \"source\", \"type\", \"start\", \"end\", \"score\", \"strand\", \"phase\", \"attributes\"\n",
        "])\n",
        "annotation_df"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 17,
      "metadata": {},
      "outputs": [],
      "source": [
        "def add_introns(annotation_df):\n",
        "    introns = []\n",
        "    for gene in annotation_df[annotation_df['type'] == 'gene'].itertuples():\n",
        "        gene_exons = annotation_df[(annotation_df['type'] == 'exon') &\n",
        "                                   (annotation_df['scaffold'] == gene.scaffold) &\n",
        "                                   (annotation_df['start'] >= gene.start) &\n",
        "                                   (annotation_df['end'] <= gene.end)]\n",
        "        gene_exons = gene_exons.sort_values(by='start')\n",
        "        previous_exon_end = None\n",
        "        for exon in gene_exons.itertuples():\n",
        "            if previous_exon_end is not None:\n",
        "                intron_start = previous_exon_end + 1\n",
        "                intron_end = exon.start - 1\n",
        "                if intron_start < intron_end:\n",
        "                    introns.append({\n",
        "                        \"scaffold\": gene.scaffold,\n",
        "                        \"source\": \"predicted\",\n",
        "                        \"type\": \"intron\",\n",
        "                        \"start\": intron_start,\n",
        "                        \"end\": intron_end,\n",
        "                        \"score\": \".\",\n",
        "                        \"strand\": gene.strand,\n",
        "                        \"phase\": \".\",\n",
        "                        \"attributes\": gene.attributes\n",
        "                    })\n",
        "            previous_exon_end = exon.end\n",
        "\n",
        "    introns_df = pd.DataFrame(introns)\n",
        "    return pd.concat([annotation_df, introns_df], ignore_index=True)\n",
        "\n",
        "annotation_intron_df = add_introns(annotation_df)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 18,
      "metadata": {},
      "outputs": [],
      "source": [
        "annotation_intron_df[['scaffold', 'start', 'end', 'type', 'scaffold', 'strand']].to_csv('annotation.bed', header=None, index=False, sep='\\t')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 19,
      "metadata": {},
      "outputs": [],
      "source": [
        "!awk '{if($1 !~ /^#/ ) print $1\"\\t\"$4-1\"\\t\"$5\"\\t\"$3\"\\t\"$9\"\\t\"$7}' data/GCF_013758885.1_VT_AalbS3_pri_1.0_genomic.gff > annotation.bed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 20,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "NC_050155.1\t0\t13047563\tregion\tID=NC_050155.1:1..13047563;Dbxref=taxon:7167;Name=X;chromosome=X;collection-date=Oct-2018;country=El\t+\n",
            "NC_050155.1\t12554\t14767\tgene\tID=gene-LOC118466793;Dbxref=GeneID:118466793;Name=LOC118466793;gbkey=Gene;gene=LOC118466793;gene_biotype=lncRNA\t-\n",
            "NC_050155.1\t12554\t14767\tlnc_RNA\tID=rna-XR_004845662.1;Parent=gene-LOC118466793;Dbxref=GeneID:118466793,Genbank:XR_004845662.1;Name=XR_004845662.1;gbkey=ncRNA;gene=LOC118466793;model_evidence=Supporting\t-\n",
            "NC_050155.1\t14725\t14767\texon\tID=exon-XR_004845662.1-1;Parent=rna-XR_004845662.1;Dbxref=GeneID:118466793,Genbank:XR_004845662.1;gbkey=ncRNA;gene=LOC118466793;product=uncharacterized\t-\n",
            "NC_050155.1\t12910\t13230\texon\tID=exon-XR_004845662.1-2;Parent=rna-XR_004845662.1;Dbxref=GeneID:118466793,Genbank:XR_004845662.1;gbkey=ncRNA;gene=LOC118466793;product=uncharacterized\t-\n",
            "NC_050155.1\t12554\t12596\texon\tID=exon-XR_004845662.1-3;Parent=rna-XR_004845662.1;Dbxref=GeneID:118466793,Genbank:XR_004845662.1;gbkey=ncRNA;gene=LOC118466793;product=uncharacterized\t-\n",
            "NC_050155.1\t14938\t19141\tgene\tID=gene-LOC118466830;Dbxref=GeneID:118466830;Name=LOC118466830;gbkey=Gene;gene=LOC118466830;gene_biotype=lncRNA\t-\n",
            "NC_050155.1\t14938\t19141\tlnc_RNA\tID=rna-XR_004845666.1;Parent=gene-LOC118466830;Dbxref=GeneID:118466830,Genbank:XR_004845666.1;Name=XR_004845666.1;gbkey=ncRNA;gene=LOC118466830;model_evidence=Supporting\t-\n",
            "NC_050155.1\t19099\t19141\texon\tID=exon-XR_004845666.1-1;Parent=rna-XR_004845666.1;Dbxref=GeneID:118466830,Genbank:XR_004845666.1;gbkey=ncRNA;gene=LOC118466830;product=uncharacterized\t-\n",
            "NC_050155.1\t15294\t15614\texon\tID=exon-XR_004845666.1-2;Parent=rna-XR_004845666.1;Dbxref=GeneID:118466830,Genbank:XR_004845666.1;gbkey=ncRNA;gene=LOC118466830;product=uncharacterized\t-\n"
          ]
        }
      ],
      "source": [
        "!head annotation.bed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 21,
      "metadata": {},
      "outputs": [],
      "source": [
        "!awk '$4 == \"exon\"' ./annotation.bed > exons.bed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 22,
      "metadata": {},
      "outputs": [],
      "source": [
        "!awk '$4 == \"intron\"' ./annotation.bed > introns.bed\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 23,
      "metadata": {},
      "outputs": [],
      "source": [
        "!awk 'BEGIN{OFS=\"\\t\"} {if ($4 == \"gene\") {print $1, $2, $3, $4, $5, $6}}' ./annotation.bed > genes.bed\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 35,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[34m==>\u001b[0m \u001b[1mAuto-updating Homebrew...\u001b[0m\n",
            "Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with\n",
            "HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/portable-ruby/portable-ruby/blobs/sha256:fd162df7a06190ee800a9e6afd28f4466d33548821a480ba043cd927b44d60f7\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring portable-ruby-3.4.4.arm64_big_sur.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mAuto-updated Homebrew!\u001b[0m\n",
            "Updated 2 taps (homebrew/core and homebrew/cask).\n",
            "\u001b[34m==>\u001b[0m \u001b[1mNew Formulae\u001b[0m\n",
            "ab-av1                                   libkeyfinder\n",
            "abpoa                                    libpg_query\n",
            "acme.sh                                  libpostal\n",
            "acronym                                  libpostal-rest\n",
            "act_runner                               libpq@16\n",
            "ad                                       libsvgtiny\n",
            "adapterremoval                           lima-additional-guestagents\n",
            "adaptivecpp                              lld@19\n",
            "add-determinism                          llvm@19\n",
            "addons-linter                            lol-html\n",
            "alejandra                                ltex-ls-plus\n",
            "algolia                                  ludusavi\n",
            "alive2                                   lume\n",
            "alloy                                    lzsa\n",
            "anubis                                   mac\n",
            "anyquery                                 malcontent\n",
            "apache-flink@1                           mani\n",
            "api-linter                               marmite\n",
            "aqtinstall                               martin\n",
            "aqua                                     mcp-inspector\n",
            "arelo                                    mcpm\n",
            "astroterm                                md2pdf\n",
            "async_simple                             mdq\n",
            "atac                                     mdz\n",
            "autocycler                               mender-artifact\n",
            "aws-lc                                   mender-cli\n",
            "azure-storage-blobs-cpp                  miniflux\n",
            "azure-storage-common-cpp                 miniprot\n",
            "babelfish                                mob\n",
            "bacon-ls                                 mongo-c-driver@1\n",
            "bagels                                   monocle\n",
            "bazel@7                                  moodle-dl\n",
            "beanquery                                mox\n",
            "behaviortree.cpp                         mpremote\n",
            "benchi                                   multi-gitter\n",
            "bender                                   mummer\n",
            "bento                                    nak\n",
            "binocle                                  nelm\n",
            "bkmr                                     nerdlog\n",
            "block-goose-cli                          netatalk\n",
            "blueprint-compiler                       netfetch\n",
            "boa                                      neverest\n",
            "bold                                     newsraft\n",
            "bombardier                               nip4\n",
            "bower-mail                               nmstatectl\n",
            "bpmnlint                                 node-red\n",
            "brename                                  nom\n",
            "breseq                                   nova\n",
            "bsc                                      nping\n",
            "btcli                                    nuitka\n",
            "buffrs                                   nx\n",
            "c2patool                                 odbc2parquet\n",
            "camlpdf                                  openapi-tui\n",
            "cargo-cache                              opkssh\n",
            "cargo-cyclonedx                          oranda\n",
            "cargo-dist                               org-formation\n",
            "cargo-flamegraph                         osx-trash\n",
            "cargo-hack                               otel-cli\n",
            "cargo-msrv                               oterm\n",
            "cargo-public-api                         otterdog\n",
            "cargo-shear                              ov\n",
            "cargo-shuttle                            oxen\n",
            "cargo-sort                               pangene\n",
            "cargo-spellcheck                         parallel-disk-usage\n",
            "cargo-update                             pdfly\n",
            "catgirl                                  pdtm\n",
            "cdncheck                                 perbase\n",
            "cf-terraforming                          percona-server@8.0\n",
            "cfnctl                                   percona-xtrabackup@8.0\n",
            "chaos-client                             periphery\n",
            "chart-releaser                           perl-dbd-mysql\n",
            "chawan                                   pgbackrest\n",
            "chdig                                    pgrx\n",
            "chiko                                    physunits\n",
            "clang-include-graph                      pieces-cli\n",
            "claude-squad                             pinact\n",
            "cloud-provider-kind                      pipelight\n",
            "cloudfoundry-cli                         pistache\n",
            "code2prompt                              pivy\n",
            "codeberg-cli                             pixd\n",
            "codesnap                                 pkgx\n",
            "codevis                                  pkl-lsp\n",
            "codex                                    planus\n",
            "comrak                                   pnpm@9\n",
            "concurrentqueue                          polaris\n",
            "cookcli                                  policy-engine\n",
            "cornelis                                 polypolish\n",
            "cot                                      postgresql-hll\n",
            "cpdf                                     precious\n",
            "cpp-lazy                                 preevy\n",
            "cram                                     prism-cli\n",
            "crd2pulumi                               prog8\n",
            "credo                                    projectable\n",
            "cspell                                   protobuf@29\n",
            "darklua                                  protoc-gen-doc\n",
            "dbg-macro                                proxyfor\n",
            "dblab                                    pscale\n",
            "decompose                                pulumictl\n",
            "desed                                    punktf\n",
            "diagram                                  pyp\n",
            "diary                                    qnm\n",
            "dish                                     qrkey\n",
            "dockcheck                                ra-multiplex\n",
            "dockerfilegraph                          ramalama\n",
            "dockerfmt                                rasterio\n",
            "dpic                                     rasusa\n",
            "dtsroll                                  ratarmount\n",
            "dud                                      ratify\n",
            "dvisvgm                                  rattler-index\n",
            "dvr-scan                                 readerwriterqueue\n",
            "dyff                                     readsb\n",
            "e2b                                      reckoner\n",
            "easyeda2kicad                            redocly-cli\n",
            "elf2uf2-rs                               restish\n",
            "elfio                                    reuse\n",
            "ente-cli                                 rhai\n",
            "erlang@27                                rink\n",
            "erlfmt                                   rmtrash\n",
            "evans                                    rna-star\n",
            "evil-helix                               rofi\n",
            "excalidraw-converter                     ropebwt3\n",
            "execline                                 rpds-py\n",
            "exomizer                                 ruby-lsp\n",
            "fabric-ai                                ruby@3.3\n",
            "falcosecurity-libs                       runitor\n",
            "fancy-cat                                rustic\n",
            "fastbuild                                rustywind\n",
            "fastga                                   s6-rc\n",
            "fastk                                    sacad\n",
            "fastly                                   samply\n",
            "fedify                                   sby\n",
            "feluda                                   scryer-prolog\n",
            "ferron                                   sdl3\n",
            "fiona                                    sdl3_image\n",
            "fish-lsp                                 sdl3_ttf\n",
            "flip-link                                semver\n",
            "flow-control                             sentry-cli\n",
            "flye                                     sentry-native\n",
            "fortitude                                sequoia-chameleon-gnupg\n",
            "fortran-stdlib                           sequoia-sqv\n",
            "foundry                                  serie\n",
            "foxglove-cli                             sexpect\n",
            "fpm                                      sfml@2\n",
            "fricas                                   shamrock\n",
            "ftxui                                    shepherd\n",
            "gama                                     showcert\n",
            "gamdl                                    shuttle\n",
            "garnet                                   sitefetch\n",
            "gauth                                    skalibs\n",
            "gcc@14                                   skani\n",
            "gcli                                     slumber\n",
            "gcr                                      smenu\n",
            "gdown                                    snowflake-cli\n",
            "gdtoolkit                                soft-serve\n",
            "geesefs                                  spice-server\n",
            "gersemi                                  sprocket\n",
            "gerust                                   sql-formatter\n",
            "ggh                                      sqruff\n",
            "gh-ost                                   sshx\n",
            "ghalint                                  statesmith\n",
            "ghc@9.10                                 static-web-server\n",
            "girara                                   streamrip\n",
            "git-graph                                stringtie\n",
            "git-ignore                               sttr\n",
            "git-mob                                  style-dictionary\n",
            "git-who                                  superhtml\n",
            "github-mcp-server                        sv2v\n",
            "glaze                                    swc\n",
            "globstar                                 swctl\n",
            "gnome-builder                            swiftly\n",
            "gnome-online-accounts                    sylph\n",
            "go-parquet-tools                         symfony-cli\n",
            "go@1.23                                  tabixpp\n",
            "goimports                                tach\n",
            "gomi                                     taskflow\n",
            "goshs                                    tdom\n",
            "gotz                                     technitium-dns\n",
            "gowall                                   technitium-library\n",
            "grafana-alloy                            termbg\n",
            "gtree                                    terraform-cleaner\n",
            "guichan                                  terraform-iam-policy-validator\n",
            "guile-fibers                             terraform-module-versions\n",
            "gurk                                     text-embeddings-inference\n",
            "gut                                      tf-summarize\n",
            "harper                                   tfcmt\n",
            "harsh                                    tfmcp\n",
            "havener                                  tfmv\n",
            "hcledit                                  tfocus\n",
            "hellwal                                  tfprovidercheck\n",
            "heroku                                   tfsort\n",
            "hevi                                     tgpt\n",
            "hexd                                     the-way\n",
            "hgrep                                    threatcl\n",
            "hishtory                                 timoni\n",
            "hjson                                    tiny\n",
            "hk                                       tinysparql\n",
            "hl                                       tldx\n",
            "html2markdown                            tmex\n",
            "htmlhint                                 tml\n",
            "hub-tool                                 tmuxai\n",
            "humanlog                                 todoist\n",
            "hyper-mcp                                toml-bombadil\n",
            "hypopg                                   trdsql\n",
            "i686-elf-grub                            trimal\n",
            "icann-rdap                               tsnet-serve\n",
            "iccmax                                   tsui\n",
            "icu4c@77                                 tun2proxy\n",
            "identme                                  twitch-cli\n",
            "iguana                                   typeshare\n",
            "immich-go                                typioca\n",
            "infat                                    ubi\n",
            "infisical                                uhubctl\n",
            "intermodal                               umka-lang\n",
            "is-fast                                  umockdev\n",
            "jaguar                                   unciv\n",
            "jenkins-cli                              undercutf1\n",
            "jira-cli                                 unoserver\n",
            "jjui                                     urx\n",
            "json2hcl                                 veccore\n",
            "jsrepo                                   vfkit\n",
            "jupytext                                 vgt\n",
            "jwt-cli                                  visidata\n",
            "jwt-hack                                 vscli\n",
            "kafkactl                                 vue-language-server\n",
            "kapp                                     wcstools\n",
            "kargo                                    webdav\n",
            "kbld                                     wfa2-lib\n",
            "keeper-commander                         wrkflw\n",
            "keyutils                                 x-cmd\n",
            "kirimase                                 x86_64-elf-grub\n",
            "koji                                     xan\n",
            "kraken2                                  xc\n",
            "krep                                     xeyes\n",
            "ktexttemplate                            xk6\n",
            "ktop                                     xlsclients\n",
            "kube-bench                               xml2rfc\n",
            "kubectl-ai                               xmq\n",
            "kubernetes-cli@1.32                      xprop\n",
            "languagetool-rust                        xtermcontrol\n",
            "largetifftools                           xtl\n",
            "lavat                                    xwininfo\n",
            "lazyjj                                   yalantinglibs\n",
            "lazyjournal                              yamlfix\n",
            "lazysql                                  yeet\n",
            "ldcli                                    yices2\n",
            "leetcode-cli                             yoke\n",
            "leetgo                                   yor\n",
            "legba                                    yozefu\n",
            "libbsc                                   ytt\n",
            "libcdio-paranoia                         yutu\n",
            "libcotp                                  zigup\n",
            "libesedb                                 zimfw\n",
            "libformfactor                            zlib-ng-compat\n",
            "libgit2@1.8                              zlib-rs\n",
            "libgoa                                   zns\n",
            "libgudev                                 zsh-history-enquirer\n",
            "libheinz\n",
            "\u001b[34m==>\u001b[0m \u001b[1mNew Casks\u001b[0m\n",
            "5ire                                     font-wdxl-lubrifont-jp-n\n",
            "ace-studio                               font-wdxl-lubrifont-sc\n",
            "advanced-renamer                         font-wdxl-lubrifont-tc\n",
            "agent-tars                               font-webdings\n",
            "agentkube                                font-winky-rough\n",
            "alcove                                   font-winky-sans\n",
            "alienator88-sentinel                     fontcreator\n",
            "alifix                                   freelens\n",
            "alisma                                   fruit-screensaver\n",
            "aloha-browser                            fuse-t\n",
            "amiberry                                 gg\n",
            "antinote                                 ghostty\n",
            "appexindexer                             ghostty@tip\n",
            "aqua-voice                               gologin\n",
            "atv-remote                               gpt4all\n",
            "autogram                                 granola\n",
            "automounterhelper                        grayjay\n",
            "aw-edid-editor                           green-go-control\n",
            "ayugram                                  hamrs-pro\n",
            "azookey                                  hedy\n",
            "badgeify                                 helio\n",
            "bambu-connect                            helpwire-operator\n",
            "batfi                                    highlight\n",
            "beutl                                    hy-rpe2\n",
            "billy-frontier                           ijhttp\n",
            "bindiff                                  ik-product-manager\n",
            "bison-wallet                             imaging-edge-webcam\n",
            "blankie                                  inmusic-software-center\n",
            "block-goose                              irpf2025\n",
            "bookmacster@beta                         istatistica-core\n",
            "brilliant                                istherenet\n",
            "browser-actions                          jetdrive-toolbox\n",
            "burp-suite-professional@early-adopter    jottacloud\n",
            "burp-suite@early-adopter                 jumpcloud-password-manager\n",
            "candy-crisis                             k6-studio\n",
            "captainplugins                           kate\n",
            "charles@4                                kilohearts-installer\n",
            "chatwise                                 kunkun\n",
            "cherry-studio                            langflow\n",
            "chime@alpha                              leader-key\n",
            "chordpotion                              linearmouse@beta\n",
            "clipbook                                 little-snitch@nightly\n",
            "cloudflare-warp@beta                     liviable\n",
            "cloudpouch                               lobehub\n",
            "clover-chord-systems                     longbridge-pro\n",
            "cmpxat                                   losslessswitcher\n",
            "comfyui                                  luanti\n",
            "companion                                lyric-fever\n",
            "companion-satellite                      macai\n",
            "companion@beta                           macskk\n",
            "consul                                   meru\n",
            "container                                mihomo-party\n",
            "coterm                                   mindmaster-cn\n",
            "cpu-info                                 mitti\n",
            "dana-dex                                 mixing-station\n",
            "dante-controller                         moment\n",
            "dante-via                                mouseless\n",
            "deepchat                                 mouseless@preview\n",
            "desktime                                 moves\n",
            "display-pilot                            mozilla-vpn\n",
            "dockfix                                  music-presence\n",
            "earnapp                                  muteme\n",
            "elemental                                nao\n",
            "elemental@6                              ndi-tools\n",
            "eufymake-studio                          nethlink\n",
            "excire-foto                              nexonplug\n",
            "fastdmg                                  nextcloud-talk\n",
            "feather                                  notion-mail\n",
            "firezone                                 nvidia-nsight-compute\n",
            "flashspace                               obscura-vpn\n",
            "flowvision                               onlook\n",
            "focu                                     open-eid\n",
            "foks                                     opencloud\n",
            "font-42dot-sans                          opensuperwhisper\n",
            "font-adwaita                             opera-air\n",
            "font-adwaita-mono-nerd-font              oracle-jdk-javadoc@21\n",
            "font-ancizar-sans                        outerbase-studio\n",
            "font-ancizar-serif                       ovice\n",
            "font-annotation-mono                     pairpods\n",
            "font-aporetic                            pale-moon\n",
            "font-asta-sans                           pareto-security\n",
            "font-atkinson-hyperlegible-mono          passepartout\n",
            "font-atkinson-hyperlegible-next          pastenow\n",
            "font-atkynson-mono-nerd-font             pdl\n",
            "font-big-shoulders                       peninsula\n",
            "font-big-shoulders-inline                pinwheel\n",
            "font-big-shoulders-stencil               precize\n",
            "font-bitcount                            profit\n",
            "font-bitcount-grid-double                qobuz-downloader\n",
            "font-bitcount-grid-single                qt-design-studio\n",
            "font-bitcount-prop-double                radiola\n",
            "font-bitcount-prop-single                rave\n",
            "font-bitcount-prop-single-ink            readest\n",
            "font-bitcount-single                     realvnc-connect\n",
            "font-bitcount-single-ink                 repo-prompt\n",
            "font-bizter                              restapia\n",
            "font-boldonse                            revisionist\n",
            "font-brass-mono                          rivet\n",
            "font-bytesized                           satdump\n",
            "font-charis                              sc-menu\n",
            "font-comic-relief                        serial-studio\n",
            "font-coral-pixels                        simpledemviewer\n",
            "font-epunda-sans                         slidepad\n",
            "font-epunda-slab                         slideshower\n",
            "font-exile                               sokim\n",
            "font-fairfax-hd                          soundanchor\n",
            "font-formudpgothic                       soundsource@test\n",
            "font-fzhei-b01                           sparsity\n",
            "font-fzxiheii-z08                        spectrolite\n",
            "font-gd-highwaygothicja                  stability-matrix\n",
            "font-gentium                             startupfolder\n",
            "font-gentium-book                        status\n",
            "font-harmonyos-sans                      stratoshark\n",
            "font-harmonyos-sans-naskh-arabic         stratoshark@dev\n",
            "font-harmonyos-sans-sc                   structuredlogviewer\n",
            "font-harmonyos-sans-tc                   swift-shift\n",
            "font-huninn                              tal-drum\n",
            "font-jetbrains-maple-mono                tartelet\n",
            "font-jetbrains-maple-mono-nf             tdr-molotok\n",
            "font-juisee                              tdr-prism\n",
            "font-juisee-nf                           teleport@16\n",
            "font-kreative-square                     termora\n",
            "font-kumar-one-outline                   textgrabber2\n",
            "font-leland                              thebrowsercompany-dia\n",
            "font-libertinus-math                     thelowtechguys-cling\n",
            "font-libertinus-mono                     thumbhost3mf\n",
            "font-liter                               timescribe\n",
            "font-lxgw-marker-gothic                  tiny-shield\n",
            "font-lxgw-wenkai-gb-lite                 tm-error-logger\n",
            "font-m-plus-rounded-1c                   tourbox-console\n",
            "font-manufacturing-consent               tpvirtual\n",
            "font-maple-mono-cn                       trae\n",
            "font-maple-mono-nf-cn                    trae-cn\n",
            "font-maple-mono-normal                   triliumnext-notes\n",
            "font-maple-mono-normal-cn                turbotax-2024\n",
            "font-maple-mono-normal-nf                ua-midi-control\n",
            "font-maple-mono-normal-nf-cn             ui-tars\n",
            "font-matangi                             valhalla-freq-echo\n",
            "font-menbere                             valhalla-space-modulator\n",
            "font-monomakh                            valhalla-supermassive\n",
            "font-nanum-square                        veracrypt-fuse-t\n",
            "font-nanum-square-neo                    vernier-spectral-analysis\n",
            "font-nanum-square-round                  versatility\n",
            "font-noto-sans-sunuwar                   vesktop\n",
            "font-noto-serif-dives-akuru              vezer\n",
            "font-parastoo                            viables\n",
            "font-pixel-code                          vibemeter\n",
            "font-playpen-sans-arabic                 vimy\n",
            "font-playpen-sans-deva                   voiceink\n",
            "font-playpen-sans-hebrew                 voicenotes\n",
            "font-playpen-sans-thai                   void\n",
            "font-pochaevsk                           voiden\n",
            "font-ponomar                             warp@preview\n",
            "font-pretendard-gov                      warsaw\n",
            "font-savate                              widgettoggler\n",
            "font-sf-mono-nerd-font-ligaturized       windowkeys\n",
            "font-shafarik                            windsurf@next\n",
            "font-simple-icons                        witsy\n",
            "font-source-han-code-jp                  wrkspace\n",
            "font-special-gothic                      yaak@beta\n",
            "font-special-gothic-condensed-one        zen-privacy\n",
            "font-special-gothic-expanded-one         zoho-cliq\n",
            "font-uoqmunthenkhung                     zoo-design-studio\n",
            "\n",
            "You have \u001b[1m91\u001b[0m outdated formulae installed.\n",
            "\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/bedtools/manifests/2.31.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for bedtools: \u001b[32mxz\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/xz/manifests/5.8.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mxz\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/xz/blobs/sha256:82fef9b66eea967\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mbedtools\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/bedtools/blobs/sha256:31c196dcf\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for bedtools: \u001b[32mxz\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling bedtools dependency: \u001b[32mxz\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/xz/manifests/5.8.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/86a115cc1d43ff8a480fd907f812e70a403e1675d8a7223f61bbb08cbd2adc27--xz-5.8.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring xz--5.8.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/xz/5.8.1: 96 files, 2.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mbedtools\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring bedtools--2.31.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/bedtools/2.31.1: 43 files, 1.6MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup bedtools`...\u001b[0m\n",
            "Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.\n",
            "Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).\n",
            "\u001b[34m==>\u001b[0m \u001b[1mUpgrading 17 dependents of upgraded formulae:\u001b[0m\n",
            "Disable this behaviour by setting HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK.\n",
            "Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).\n",
            "little-cms2 2.15 -> 2.17, aom 3.7.0 -> 3.12.1, boost 1.83.0 -> 1.88.0, gcc 13.2.0 -> 15.1.0, jpeg-xl 0.8.2_1 -> 0.11.1_1, leptonica 1.83.1 -> 1.85.0, ffmpeg 6.0_1 -> 7.1.1_3, libarchive 3.7.2 -> 3.8.1, libtiff 4.6.0 -> 4.7.0, libzip 1.9.2 -> 1.11.4, openjpeg 2.5.0_1 -> 2.5.3, python@3.11 3.11.6 -> 3.11.13, qemu 8.1.2 -> 10.0.2_1, tesseract 5.3.3 -> 5.5.1, tldr 1.5.0 -> 1.6.1, webp 1.3.2 -> 1.5.0, zstd 1.5.5 -> 1.5.7\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/zstd/manifests/1.5.7\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for zstd: \u001b[32mlz4\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/lz4/manifests/1.10.0-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlz4\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/lz4/blobs/sha256:03119aa78b7a96\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mzstd\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/zstd/blobs/sha256:56faac95df2a5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libarchive/manifests/3.8.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibarchive\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libarchive/blobs/sha256:be0595b\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libtiff/manifests/4.7.0-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for libtiff: \u001b[32mjpeg-turbo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg-turbo/manifests/3.1.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mjpeg-turbo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg-turbo/blobs/sha256:1f62cf2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibtiff\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libtiff/blobs/sha256:b8952fb7bf\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/little-cms2/manifests/2.17\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlittle-cms2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/little-cms2/blobs/sha256:a4edff\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/webp/manifests/1.5.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for webp: \u001b[32mgiflib\u001b[39m and \u001b[32mlibpng\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/giflib/manifests/5.2.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgiflib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/giflib/blobs/sha256:f0c469da58f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libpng/manifests/1.6.49\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibpng\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libpng/blobs/sha256:92a708784c3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mwebp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/webp/blobs/sha256:a0ce313237703\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg-xl/manifests/0.11.1_1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for jpeg-xl: \u001b[32mhighway\u001b[39m, \u001b[32mimath\u001b[39m, \u001b[32mlibdeflate\u001b[39m and \u001b[32mopenexr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/highway/manifests/1.2.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mhighway\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/highway/blobs/sha256:26b4d20fb4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/imath/manifests/3.1.12\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mimath\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/imath/blobs/sha256:b8fb42984770\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libdeflate/manifests/1.24\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibdeflate\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libdeflate/blobs/sha256:66411c4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openexr/manifests/3.3.4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mopenexr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openexr/blobs/sha256:d5e969d9d4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mjpeg-xl\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg-xl/blobs/sha256:d4fc2bbf1f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/aom/manifests/3.12.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for aom: \u001b[32mlibvmaf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libvmaf/manifests/3.0.0-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibvmaf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libvmaf/blobs/sha256:ca7a55e4ab\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32maom\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/aom/blobs/sha256:cf9e2b2e861a14\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/boost/manifests/1.88.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for boost: \u001b[32micu4c@77\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/icu4c/77/manifests/77.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32micu4c@77\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/icu4c/77/blobs/sha256:425a38eb7\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mboost\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/boost/blobs/sha256:65d89e7f5967\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gcc/manifests/15.1.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for gcc: \u001b[32misl\u001b[39m and \u001b[32mmpfr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/isl/manifests/0.27\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32misl\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/isl/blobs/sha256:7d74b1abc36c01\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpfr/manifests/4.2.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mmpfr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpfr/blobs/sha256:12528a52c96fb\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgcc\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gcc/blobs/sha256:47bb2181ba7679\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openjpeg/manifests/2.5.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mopenjpeg\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openjpeg/blobs/sha256:10a4d7d2b\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/leptonica/manifests/1.85.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mleptonica\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/leptonica/blobs/sha256:c63d4257\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/tesseract/manifests/5.5.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for tesseract: \u001b[32mfreetype\u001b[39m, \u001b[32mfontconfig\u001b[39m, \u001b[32mpcre2\u001b[39m, \u001b[32mlibunistring\u001b[39m, \u001b[32mgettext\u001b[39m, \u001b[32mglib\u001b[39m, \u001b[32mxorgproto\u001b[39m, \u001b[32mlibxau\u001b[39m, \u001b[32mlibxdmcp\u001b[39m, \u001b[32mlibxcb\u001b[39m, \u001b[32mlibx11\u001b[39m, \u001b[32mlibxext\u001b[39m, \u001b[32mlibxrender\u001b[39m, \u001b[32mpixman\u001b[39m, \u001b[32mcairo\u001b[39m, \u001b[32mharfbuzz\u001b[39m, \u001b[32mfribidi\u001b[39m and \u001b[32mpango\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/freetype/manifests/2.13.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfreetype\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/freetype/blobs/sha256:47122c7d0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fontconfig/manifests/2.16.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfontconfig\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fontconfig/blobs/sha256:9af1bb1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pcre2/manifests/10.45\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpcre2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pcre2/blobs/sha256:a780535fb2f4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libunistring/manifests/1.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibunistring\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libunistring/blobs/sha256:b0d7a\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gettext/manifests/0.25\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgettext\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gettext/blobs/sha256:af2c9962b3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/glib/manifests/2.84.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mglib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/glib/blobs/sha256:c504fd9493eae\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/xorgproto/manifests/2024.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mxorgproto\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/xorgproto/blobs/sha256:3307601f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxau/manifests/1.0.12\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxau\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxau/blobs/sha256:7c0325743bb\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxdmcp/manifests/1.1.5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxdmcp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxdmcp/blobs/sha256:5c459f7ce\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxcb/manifests/1.17.0-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxcb\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxcb/blobs/sha256:3de506a3b5f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libx11/manifests/1.8.12\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibx11\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libx11/blobs/sha256:693416ac252\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxext/manifests/1.3.6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxext\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxext/blobs/sha256:fb6852f038\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxrender/manifests/0.9.12\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxrender\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxrender/blobs/sha256:0d5c91b\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pixman/manifests/0.46.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpixman\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pixman/blobs/sha256:d95aaa640c1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/cairo/manifests/1.18.4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mcairo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/cairo/blobs/sha256:8ea2169c9cc4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/harfbuzz/manifests/11.2.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mharfbuzz\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/harfbuzz/blobs/sha256:f3f560cff\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fribidi/manifests/1.0.16\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfribidi\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fribidi/blobs/sha256:334c412359\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pango/manifests/1.56.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpango\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pango/blobs/sha256:4d619b288285\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mtesseract\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/tesseract/blobs/sha256:89427030\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ffmpeg/manifests/7.1.1_3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for ffmpeg: \u001b[32mdav1d\u001b[39m, \u001b[32mfrei0r\u001b[39m, \u001b[32mca-certificates\u001b[39m, \u001b[32mlibidn2\u001b[39m, \u001b[32mlibtasn1\u001b[39m, \u001b[32mnettle\u001b[39m, \u001b[32mp11-kit\u001b[39m, \u001b[32mopenssl@3\u001b[39m, \u001b[32mlibnghttp2\u001b[39m, \u001b[32munbound\u001b[39m, \u001b[32mgnutls\u001b[39m, \u001b[32mlibunibreak\u001b[39m, \u001b[32mlibass\u001b[39m, \u001b[32mcjson\u001b[39m, \u001b[32mlibmicrohttpd\u001b[39m, \u001b[32mmbedtls\u001b[39m, \u001b[32mlibrist\u001b[39m, \u001b[32mlibssh\u001b[39m, \u001b[32mlibvpx\u001b[39m, \u001b[32mopus\u001b[39m, \u001b[32mrav1e\u001b[39m, \u001b[32mflac\u001b[39m, \u001b[32mmpg123\u001b[39m, \u001b[32mlibsndfile\u001b[39m, \u001b[32mrubberband\u001b[39m, \u001b[32msdl2\u001b[39m, \u001b[32msnappy\u001b[39m, \u001b[32msrt\u001b[39m, \u001b[32msvt-av1\u001b[39m, \u001b[32mx265\u001b[39m and \u001b[32mlibsodium\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dav1d/manifests/1.5.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mdav1d\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dav1d/blobs/sha256:554aac9fa65b\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/frei0r/manifests/2.3.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfrei0r\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/frei0r/blobs/sha256:e1432fd9d7d\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ca-certificates/manifests/2025-\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mca-certificates\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ca-certificates/blobs/sha256:dd\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libidn2/manifests/2.3.8\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibidn2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libidn2/blobs/sha256:d0d933dad3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libtasn1/manifests/4.20.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibtasn1\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libtasn1/blobs/sha256:b202c24f2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/nettle/manifests/3.10.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mnettle\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/nettle/blobs/sha256:10243a93746\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/p11-kit/manifests/0.25.5-2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mp11-kit\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/p11-kit/blobs/sha256:e3bc89565c\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openssl/3/manifests/3.5.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mopenssl@3\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openssl/3/blobs/sha256:e74364d5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libnghttp2/manifests/1.65.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibnghttp2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libnghttp2/blobs/sha256:8ae17c6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/unbound/manifests/1.23.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32munbound\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/unbound/blobs/sha256:aa011b012f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gnutls/manifests/3.8.9\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgnutls\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gnutls/blobs/sha256:449610f6f2c\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libunibreak/manifests/6.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibunibreak\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libunibreak/blobs/sha256:66f187\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libass/manifests/0.17.4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibass\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libass/blobs/sha256:b7b2a447ccc\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/cjson/manifests/1.7.18\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mcjson\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/cjson/blobs/sha256:f8cd3c29957e\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libmicrohttpd/manifests/1.0.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibmicrohttpd\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libmicrohttpd/blobs/sha256:a99e\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mbedtls/manifests/3.6.3.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mmbedtls\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mbedtls/blobs/sha256:f38f21dd53\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/librist/manifests/0.2.11\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibrist\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/librist/blobs/sha256:891def5dc7\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libssh/manifests/0.11.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibssh\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libssh/blobs/sha256:9de46491d93\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libvpx/manifests/1.15.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibvpx\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libvpx/blobs/sha256:8fc83875291\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/opus/manifests/1.5.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mopus\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/opus/blobs/sha256:d53715a8e666c\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rav1e/manifests/0.8.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mrav1e\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rav1e/blobs/sha256:9b2e4642fa87\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/flac/manifests/1.5.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mflac\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/flac/blobs/sha256:52c6aaad15049\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpg123/manifests/1.33.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mmpg123\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpg123/blobs/sha256:b38b8d614bf\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsndfile/manifests/1.2.2_1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibsndfile\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsndfile/blobs/sha256:a8be24e\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rubberband/manifests/4.0.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mrubberband\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rubberband/blobs/sha256:066c931\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/sdl2/manifests/2.32.8\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32msdl2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/sdl2/blobs/sha256:b709f4f2bb61a\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/snappy/manifests/1.2.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32msnappy\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/snappy/blobs/sha256:9e4594baee5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/srt/manifests/1.5.4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32msrt\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/srt/blobs/sha256:806fa132d70ecf\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/svt-av1/manifests/3.0.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32msvt-av1\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/svt-av1/blobs/sha256:3c65e6a356\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/x265/manifests/4.1-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mx265\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/x265/blobs/sha256:aa56445287b97\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsodium/manifests/1.0.20\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibsodium\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsodium/blobs/sha256:21b83b95\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mffmpeg\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ffmpeg/blobs/sha256:2c66c6ac49d\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libzip/manifests/1.11.4\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibzip\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libzip/blobs/sha256:65696504b92\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/python/3.11/manifests/3.11.13\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for python@3.11: \u001b[32mmpdecimal\u001b[39m, \u001b[32mreadline\u001b[39m and \u001b[32msqlite\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpdecimal/manifests/4.0.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mmpdecimal\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpdecimal/blobs/sha256:e7641186\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/readline/manifests/8.2.13\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mreadline\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/readline/blobs/sha256:57580f6ff\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/sqlite/manifests/3.50.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32msqlite\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/sqlite/blobs/sha256:c0cec07b3de\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpython@3.11\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/python/3.11/blobs/sha256:f41eae\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/qemu/manifests/10.0.2_1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching dependencies for qemu: \u001b[32mcapstone\u001b[39m, \u001b[32mdtc\u001b[39m, \u001b[32mlibslirp\u001b[39m, \u001b[32mlibusb\u001b[39m, \u001b[32mncurses\u001b[39m, \u001b[32mfaac\u001b[39m, \u001b[32mfaad2\u001b[39m, \u001b[32mfdk-aac\u001b[39m, \u001b[32mgdk-pixbuf\u001b[39m, \u001b[32mgraphene\u001b[39m, \u001b[32mdbus\u001b[39m, \u001b[32mlibxfixes\u001b[39m, \u001b[32mlibxi\u001b[39m, \u001b[32mlibxtst\u001b[39m, \u001b[32mat-spi2-core\u001b[39m, \u001b[32mgsettings-desktop-schemas\u001b[39m, \u001b[32mhicolor-icon-theme\u001b[39m, \u001b[32mlibepoxy\u001b[39m, \u001b[32mgtk+3\u001b[39m, \u001b[32mgtk4\u001b[39m, \u001b[32mjson-glib\u001b[39m, \u001b[32mlibnice\u001b[39m, \u001b[32mlibshout\u001b[39m, \u001b[32mglib-networking\u001b[39m, \u001b[32mlibpsl\u001b[39m, \u001b[32mlibsoup\u001b[39m, \u001b[32mlibusrsctp\u001b[39m, \u001b[32morc\u001b[39m, \u001b[32mpkgconf\u001b[39m, \u001b[32mexpat\u001b[39m, \u001b[32mpython@3.13\u001b[39m, \u001b[32mgobject-introspection\u001b[39m, \u001b[32mpy3cairo\u001b[39m, \u001b[32mpygobject3\u001b[39m, \u001b[32mrtmpdump\u001b[39m, \u001b[32msrtp\u001b[39m, \u001b[32mutf8cpp\u001b[39m, \u001b[32mtaglib\u001b[39m, \u001b[32mlibcuefile\u001b[39m, \u001b[32mlibreplaygain\u001b[39m, \u001b[32mmusepack\u001b[39m, \u001b[32mgstreamer\u001b[39m, \u001b[32mjpeg\u001b[39m and \u001b[32mspice-server\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/capstone/manifests/5.0.6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mcapstone\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/capstone/blobs/sha256:9c34130c8\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dtc/manifests/1.7.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mdtc\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dtc/blobs/sha256:0bc0e5c7f5681e\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libslirp/manifests/4.9.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibslirp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libslirp/blobs/sha256:4fc26d783\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libusb/manifests/1.0.29\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibusb\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libusb/blobs/sha256:c5c93e938ae\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ncurses/manifests/6.5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mncurses\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ncurses/blobs/sha256:b795629833\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/faac/manifests/1.31.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfaac\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/faac/blobs/sha256:1a0d7a2fc58b6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/faad2/manifests/2.11.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfaad2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/faad2/blobs/sha256:de416959deef\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fdk-aac/manifests/2.0.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mfdk-aac\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fdk-aac/blobs/sha256:51653e0466\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gdk-pixbuf/manifests/2.42.12_1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgdk-pixbuf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gdk-pixbuf/blobs/sha256:f3b25a9\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/graphene/manifests/1.10.8\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgraphene\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/graphene/blobs/sha256:49970a521\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dbus/manifests/1.16.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mdbus\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dbus/blobs/sha256:55db2782d19df\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxfixes/manifests/6.0.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxfixes\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxfixes/blobs/sha256:b087b60e\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxi/manifests/1.8.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxi\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxi/blobs/sha256:e21e8ceae47d\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxtst/manifests/1.2.5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibxtst\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxtst/blobs/sha256:ee96653ea1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/at-spi2-core/manifests/2.56.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mat-spi2-core\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/at-spi2-core/blobs/sha256:cc162\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gsettings-desktop-schemas/manif\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgsettings-desktop-schemas\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gsettings-desktop-schemas/blobs\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/hicolor-icon-theme/manifests/0.\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mhicolor-icon-theme\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/hicolor-icon-theme/blobs/sha256\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libepoxy/manifests/1.5.10\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibepoxy\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libepoxy/blobs/sha256:52aeeb179\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gtkx3/manifests/3.24.43-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgtk+3\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gtkx3/blobs/sha256:b2cfc5ed7e88\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gtk4/manifests/4.18.6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgtk4\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gtk4/blobs/sha256:1bbdc7c5e7503\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/json-glib/manifests/1.10.6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mjson-glib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/json-glib/blobs/sha256:589f8ee0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libnice/manifests/0.1.22-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibnice\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libnice/blobs/sha256:14745e9ade\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libshout/manifests/2.4.6_1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibshout\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libshout/blobs/sha256:9231d4d78\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/glib-networking/manifests/2.80.\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mglib-networking\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/glib-networking/blobs/sha256:f9\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libpsl/manifests/0.21.5_2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibpsl\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libpsl/blobs/sha256:0514d77bc12\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsoup/manifests/3.6.5\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibsoup\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsoup/blobs/sha256:7af02d6ee6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libusrsctp/manifests/0.9.5.0_1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibusrsctp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libusrsctp/blobs/sha256:27f5964\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/orc/manifests/0.4.41\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32morc\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/orc/blobs/sha256:fede6fc3e9d0b0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pkgconf/manifests/2.4.3-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpkgconf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pkgconf/blobs/sha256:2df66bd11b\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/expat/manifests/2.7.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mexpat\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/expat/blobs/sha256:55d44623dacc\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/python/3.13/manifests/3.13.5-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpython@3.13\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/python/3.13/blobs/sha256:16a255\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gobject-introspection/manifests\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgobject-introspection\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gobject-introspection/blobs/sha\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/py3cairo/manifests/1.28.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpy3cairo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/py3cairo/blobs/sha256:923bd6b36\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pygobject3/manifests/3.52.3\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mpygobject3\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pygobject3/blobs/sha256:45ccc44\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rtmpdump/manifests/2.4-20151223\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mrtmpdump\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rtmpdump/blobs/sha256:1dec5a57d\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/srtp/manifests/2.7.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32msrtp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/srtp/blobs/sha256:ffda680f84d2f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/utf8cpp/manifests/4.0.6\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mutf8cpp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/utf8cpp/blobs/sha256:bd37e8f689\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/taglib/manifests/2.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mtaglib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/taglib/blobs/sha256:5fa5bc0e5f1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libcuefile/manifests/r475-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibcuefile\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libcuefile/blobs/sha256:9329d10\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libreplaygain/manifests/r475-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mlibreplaygain\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libreplaygain/blobs/sha256:a8c8\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/musepack/manifests/r475-1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mmusepack\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/musepack/blobs/sha256:72634f1ba\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gstreamer/manifests/1.26.2\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mgstreamer\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gstreamer/blobs/sha256:2860d106\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg/manifests/9f\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mjpeg\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg/blobs/sha256:3492c054e815c\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/spice-server/manifests/0.16.0\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mspice-server\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/spice-server/blobs/sha256:027a1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mqemu\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/qemu/blobs/sha256:129c97a6fcadf\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[33mWarning:\u001b[0m tldr has been deprecated because it is not maintained upstream! It will be disabled on 2025-10-24.\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/tldr/manifests/1.6.1\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mFetching \u001b[32mtldr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/tldr/blobs/sha256:32223909bb788\u001b[0m\n",
            "######################################################################### 100.0%\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mzstd\u001b[39m\u001b[0m\n",
            "  1.5.5 -> 1.5.7 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for zstd: \u001b[32mlz4\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling zstd dependency: \u001b[32mlz4\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/lz4/manifests/1.10.0-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/8e11e90eb21a06e0f199af9d80e011e3693c77dd353b2477579d95c8471a5802--lz4-1.10.0-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring lz4--1.10.0.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/lz4/1.10.0: 24 files, 714.2KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mzstd\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring zstd--1.5.7.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/zstd/1.5.7: 32 files, 2.2MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup zstd`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/zstd/1.5.5... (31 files, 2.1MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mlibarchive\u001b[39m\u001b[0m\n",
            "  3.7.2 -> 3.8.1 \n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libarchive--3.8.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libarchive/3.8.1: 65 files, 3.9MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup libarchive`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/libarchive/3.7.2... (64 files, 3.9MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mlibtiff\u001b[39m\u001b[0m\n",
            "  4.6.0 -> 4.7.0 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for libtiff: \u001b[32mjpeg-turbo\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling libtiff dependency: \u001b[32mjpeg-turbo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg-turbo/manifests/3.1.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/c854f6cf182be1d17fc08db542e1a659f58691d8ccc481348465e3a038dbe73b--jpeg-turbo-3.1.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring jpeg-turbo--3.1.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/jpeg-turbo/3.1.1: 47 files, 3.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mlibtiff\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libtiff--4.7.0.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libtiff/4.7.0: 486 files, 8.5MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup libtiff`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/libtiff/4.6.0... (414 files, 6.7MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mlittle-cms2\u001b[39m\u001b[0m\n",
            "  2.15 -> 2.17 \n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring little-cms2--2.17.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/little-cms2/2.17: 23 files, 1.4MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup little-cms2`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/little-cms2/2.15... (21 files, 1.4MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mwebp\u001b[39m\u001b[0m\n",
            "  1.3.2 -> 1.5.0 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for webp: \u001b[32mgiflib\u001b[39m and \u001b[32mlibpng\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling webp dependency: \u001b[32mgiflib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/giflib/manifests/5.2.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/283773c4d2db4fe867419d7eea6811a6417889d78fad8871041c07f49b22d2a1--giflib-5.2.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring giflib--5.2.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/giflib/5.2.2: 20 files, 547.0KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling webp dependency: \u001b[32mlibpng\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libpng/manifests/1.6.49\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/e5b4a9bf70c7e7d587f8d51454aac4583dd0c94374830a805dbd043e88f8ab31--libpng-1.6.49.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libpng--1.6.49.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libpng/1.6.49: 28 files, 1.3MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mwebp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring webp--1.5.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/webp/1.5.0: 64 files, 2.5MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup webp`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/webp/1.3.2... (63 files, 2.3MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mjpeg-xl\u001b[39m\u001b[0m\n",
            "  0.8.2_1 -> 0.11.1_1 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for jpeg-xl: \u001b[32mhighway\u001b[39m, \u001b[32mimath\u001b[39m, \u001b[32mlibdeflate\u001b[39m and \u001b[32mopenexr\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling jpeg-xl dependency: \u001b[32mhighway\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/highway/manifests/1.2.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/8cb1608f21fba449c4e34b0a4912e3d73b6f2fa2577c916720799b5057207452--highway-1.2.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring highway--1.2.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/highway/1.2.0: 75 files, 4.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling jpeg-xl dependency: \u001b[32mimath\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/imath/manifests/3.1.12\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/6afdf93c912565e828ad2386867db9e96b58441f29b901492560bcde9e8a22ad--imath-3.1.12.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring imath--3.1.12.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/imath/3.1.12: 50 files, 947KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling jpeg-xl dependency: \u001b[32mlibdeflate\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libdeflate/manifests/1.24\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/f25944af7be6e49fb1ab535f6d86ab11b6414cd8d6fca2fec1817aff2268c451--libdeflate-1.24.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libdeflate--1.24.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libdeflate/1.24: 17 files, 325.5KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling jpeg-xl dependency: \u001b[32mopenexr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openexr/manifests/3.3.4\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/b25bbda6441cbbae3a0c9a840abc4e9768244d9f4eb755ba4d203d3aea5d6fed--openexr-3.3.4.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring openexr--3.3.4.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/openexr/3.3.4: 210 files, 4.8MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mjpeg-xl\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring jpeg-xl--0.11.1_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/jpeg-xl/0.11.1_1: 64 files, 31.9MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup jpeg-xl`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/jpeg-xl/0.8.2_1... (43 files, 10.8MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32maom\u001b[39m\u001b[0m\n",
            "  3.7.0 -> 3.12.1 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for aom: \u001b[32mlibvmaf\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling aom dependency: \u001b[32mlibvmaf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libvmaf/manifests/3.0.0-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/3e1a82ebf20a6bd20eace7c0c56b52b096c4fa892466fa0a9e7be486389e0974--libvmaf-3.0.0-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libvmaf--3.0.0.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libvmaf/3.0.0: 235 files, 7.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32maom\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring aom--3.12.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/aom/3.12.1: 24 files, 9.8MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup aom`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/aom/3.7.0... (23 files, 9MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mboost\u001b[39m\u001b[0m\n",
            "  1.83.0 -> 1.88.0 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for boost: \u001b[32micu4c@77\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling boost dependency: \u001b[32micu4c@77\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/icu4c/77/manifests/77.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/35ec2c3adb94255ba47424a4334da9616d97e1ec6d59ed907b368535018bed0a--icu4c@77-77.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring icu4c@77--77.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/icu4c@77/77.1: 277 files, 81.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mboost\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring boost--1.88.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/boost/1.88.0: 16,266 files, 341.9MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup boost`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/boost/1.83.0... (16,038 files, 508.4MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mgcc\u001b[39m\u001b[0m\n",
            "  13.2.0 -> 15.1.0 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for gcc: \u001b[32misl\u001b[39m and \u001b[32mmpfr\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling gcc dependency: \u001b[32misl\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/isl/manifests/0.27\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/40b1c5526f95db33208143fa79887179e758121659d8877597f553e6e6188879--isl-0.27.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring isl--0.27.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/isl/0.27: 74 files, 7.6MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling gcc dependency: \u001b[32mmpfr\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpfr/manifests/4.2.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/991b389ea6f1daa5dc48fd5250bcc5b85e827eb18b8ad4acd89419f2ace47d88--mpfr-4.2.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring mpfr--4.2.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/mpfr/4.2.2: 31 files, 3.1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mgcc\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gcc--15.1.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://formulae.brew.sh/api/formula.jws.json\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gcc/15.1.0: 1,957 files, 492.3MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup gcc`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/gcc/13.2.0... (1,489 files, 383.0MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mopenjpeg\u001b[39m\u001b[0m\n",
            "  2.5.0_1 -> 2.5.3 \n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring openjpeg--2.5.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/openjpeg/2.5.3: 543 files, 14.5MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup openjpeg`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/openjpeg/2.5.0_1... (536 files, 13.8MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mleptonica\u001b[39m\u001b[0m\n",
            "  1.83.1 -> 1.85.0 \n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring leptonica--1.85.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/leptonica/1.85.0: 57 files, 7.0MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup leptonica`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/leptonica/1.83.1... (56 files, 6.8MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mtesseract\u001b[39m\u001b[0m\n",
            "  5.3.3 -> 5.5.1 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for tesseract: \u001b[32mfreetype\u001b[39m, \u001b[32mfontconfig\u001b[39m, \u001b[32mpcre2\u001b[39m, \u001b[32mlibunistring\u001b[39m, \u001b[32mgettext\u001b[39m, \u001b[32mglib\u001b[39m, \u001b[32mxorgproto\u001b[39m, \u001b[32mlibxau\u001b[39m, \u001b[32mlibxdmcp\u001b[39m, \u001b[32mlibxcb\u001b[39m, \u001b[32mlibx11\u001b[39m, \u001b[32mlibxext\u001b[39m, \u001b[32mlibxrender\u001b[39m, \u001b[32mpixman\u001b[39m, \u001b[32mcairo\u001b[39m, \u001b[32mharfbuzz\u001b[39m, \u001b[32mfribidi\u001b[39m and \u001b[32mpango\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mfreetype\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/freetype/manifests/2.13.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/4c4d14d574a7af9a5a0220cd5012c698bd87b75a3d5e39a6a6c34b4c411aab52--freetype-2.13.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring freetype--2.13.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/freetype/2.13.3: 68 files, 2.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mfontconfig\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fontconfig/manifests/2.16.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/6897fed6b85e0f537cb8774ec21346308111caaf26c136f33fdc95302d361fe7--fontconfig-2.16.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring fontconfig--2.16.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRegenerating font cache, this may take a while\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/fontconfig/2.16.0/bin/fc-cache -frv\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/fontconfig/2.16.0: 92 files, 1.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mpcre2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pcre2/manifests/10.45\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/bbac938545583185faba88567f1a952d7cc0b825820a2980533f7b3550dc31ad--pcre2-10.45.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring pcre2--10.45.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/pcre2/10.45: 242 files, 6.7MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibunistring\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libunistring/manifests/1.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/a570da63bc1839c7e217f203abd54d4d873ebd6b99f6e88994d0e79e2ebe987c--libunistring-1.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libunistring--1.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libunistring/1.3: 59 files, 5.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mgettext\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gettext/manifests/0.25\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/344607fc5b91bb0c1287d07bb445cc40cc465a163a52e12eed3cc5cd60498f78--gettext-0.25.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gettext--0.25.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gettext/0.25: 2,418 files, 27.8MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mglib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/glib/manifests/2.84.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/88d63b55ddeb4b28ab9c7944cbfcc7dc19d04bd0effc32c2e9623ce50a6cd51f--glib-2.84.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring glib--2.84.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/glib/2.84.3: 504 files, 36.6MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mxorgproto\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/xorgproto/manifests/2024.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/d89d69340366d1d015e7214520f64081e2d82d7955fb28c6ea5f6539dcb610b2--xorgproto-2024.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring xorgproto--2024.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/xorgproto/2024.1: 268 files, 3.9MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibxau\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxau/manifests/1.0.12\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/6335be202d59f96fdee2e4a02f93d8ee87dfcfdbaa4ed132b8fe8c3e3651e27c--libxau-1.0.12.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxau--1.0.12.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxau/1.0.12: 21 files, 134.3KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibxdmcp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxdmcp/manifests/1.1.5\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/8713e53f73d0be64780dbea522b0ee07027e6cc7828ff1a4fa0a6596f14cfe25--libxdmcp-1.1.5.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxdmcp--1.1.5.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxdmcp/1.1.5: 12 files, 140.0KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibxcb\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxcb/manifests/1.17.0-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/a17e9d0e04a0013a5be592875b012c409023664b3df084b8051766331d213ff5--libxcb-1.17.0-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxcb--1.17.0.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxcb/1.17.0: 2,498 files, 7.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibx11\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libx11/manifests/1.8.12\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/8510ddcf75e7c4313fe4f8b0999d667b22f2ae0b850bc80ba53becb609335934--libx11-1.8.12.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libx11--1.8.12.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libx11/1.8.12: 1,043 files, 7MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibxext\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxext/manifests/1.3.6\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/fedb843acbfcf682fb451396914bfc096f9ea2748376d4c43ce2af7ce1b766b7--libxext-1.3.6.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxext--1.3.6.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxext/1.3.6: 88 files, 458.5KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mlibxrender\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxrender/manifests/0.9.12\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/152fb22ff88a23b62841dcfa14aac21018e1d736c072d732185f6139ef922e1c--libxrender-0.9.12.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxrender--0.9.12.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxrender/0.9.12: 13 files, 225.4KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mpixman\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pixman/manifests/0.46.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/c608060dd44c7a853d8ac82d8641c6911fcc8d7ba73a07435e0216d210b15842--pixman-0.46.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring pixman--0.46.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/pixman/0.46.2: 10 files, 678.9KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mcairo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/cairo/manifests/1.18.4\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/795ef2e2e2a57e6baba0fc9ceedb58980e8ec718aba0170b9441a0b25442cfbf--cairo-1.18.4.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring cairo--1.18.4.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/cairo/1.18.4: 53 files, 2.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mharfbuzz\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/harfbuzz/manifests/11.2.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/541d81ca7e8e8b678fb64d9f758603fe525bfc219f8fb4782fd91fc81f9465aa--harfbuzz-11.2.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring harfbuzz--11.2.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/harfbuzz/11.2.1: 77 files, 10.3MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mfribidi\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fribidi/manifests/1.0.16\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/9b4a81e7d1c2fe7c2d8ed4194565744f65bab0238c00f7f3f521f5aec2b8697f--fribidi-1.0.16.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring fribidi--1.0.16.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/fribidi/1.0.16: 68 files, 567.9KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling tesseract dependency: \u001b[32mpango\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pango/manifests/1.56.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/6f0e0f9ffd4d853b6b710a1ab143ecf1dbe833bc2505437fffb38d1a28fef526--pango-1.56.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring pango--1.56.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/pango/1.56.3: 69 files, 3.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mtesseract\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring tesseract--5.5.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/tesseract/5.5.1: 75 files, 33.2MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup tesseract`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/tesseract/5.3.3... (73 files, 32.6MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mffmpeg\u001b[39m\u001b[0m\n",
            "  6.0_1 -> 7.1.1_3 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for ffmpeg: \u001b[32mdav1d\u001b[39m, \u001b[32mfrei0r\u001b[39m, \u001b[32mca-certificates\u001b[39m, \u001b[32mlibidn2\u001b[39m, \u001b[32mlibtasn1\u001b[39m, \u001b[32mnettle\u001b[39m, \u001b[32mp11-kit\u001b[39m, \u001b[32mopenssl@3\u001b[39m, \u001b[32mlibnghttp2\u001b[39m, \u001b[32munbound\u001b[39m, \u001b[32mgnutls\u001b[39m, \u001b[32mlibunibreak\u001b[39m, \u001b[32mlibass\u001b[39m, \u001b[32mcjson\u001b[39m, \u001b[32mlibmicrohttpd\u001b[39m, \u001b[32mmbedtls\u001b[39m, \u001b[32mlibrist\u001b[39m, \u001b[32mlibssh\u001b[39m, \u001b[32mlibvpx\u001b[39m, \u001b[32mopus\u001b[39m, \u001b[32mrav1e\u001b[39m, \u001b[32mflac\u001b[39m, \u001b[32mmpg123\u001b[39m, \u001b[32mlibsndfile\u001b[39m, \u001b[32mrubberband\u001b[39m, \u001b[32msdl2\u001b[39m, \u001b[32msnappy\u001b[39m, \u001b[32msrt\u001b[39m, \u001b[32msvt-av1\u001b[39m, \u001b[32mx265\u001b[39m and \u001b[32mlibsodium\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mdav1d\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dav1d/manifests/1.5.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/76ce8a8dffcdb3883423d0fa5ff3dbd810a979e8ef55ee544728d3cf5625c656--dav1d-1.5.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring dav1d--1.5.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/dav1d/1.5.1: 16 files, 911.8KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mfrei0r\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/frei0r/manifests/2.3.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/f9266f5e6cb521cdce245d9ea3bf63ef75e12bea0ab3a45f107cbe0d01f58c60--frei0r-2.3.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring frei0r--2.3.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/frei0r/2.3.3: 157 files, 5.6MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mca-certificates\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ca-certificates/manifests/2025-\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/bc18acc15e0abddc102f828b57a29cfdbec1b6b002db37ad12bad9dbf0e9d12f--ca-certificates-2025-05-20.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring ca-certificates--2025-05-20.all.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRegenerating CA certificate bundle from keychain, this may take a while...\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/ca-certificates/2025-05-20: 4 files, 225.8KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibidn2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libidn2/manifests/2.3.8\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/f30f50fbde4bff9a71de54d684e482d7da3432656d680b97441163c6e5665468--libidn2-2.3.8.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libidn2--2.3.8.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libidn2/2.3.8: 80 files, 939.3KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibtasn1\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libtasn1/manifests/4.20.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/0d8e273dbfaade0247db22d4d5abf7e8d4d6c379f7912adc81c60e5d4418e8ca--libtasn1-4.20.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libtasn1--4.20.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libtasn1/4.20.0: 63 files, 689.9KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mnettle\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/nettle/manifests/3.10.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/3a897a4b5866bef2225eb637641c992e5903a0748ec913a0747a253dcee0c4d8--nettle-3.10.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring nettle--3.10.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/nettle/3.10.1: 96 files, 2.8MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mp11-kit\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/p11-kit/manifests/0.25.5-2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/771d90d8ac09d26012036319b49012b7c2ef8349c2e6899855425e29397bd7bc--p11-kit-0.25.5-2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring p11-kit--0.25.5.arm64_ventura.bottle.2.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/p11-kit/0.25.5: 31 files, 4.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mopenssl@3\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/openssl/3/manifests/3.5.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/d3c8243df37f69247721b1d8a6c51e1a8afd6664b832423921ffd99bb2bb381a--openssl@3-3.5.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring openssl@3--3.5.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/openssl@3/3.5.0: 7,563 files, 35.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibnghttp2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libnghttp2/manifests/1.65.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/5a72fb9f57dd728c5d6d30917849ae15111c3b440595f5694738ec9434c12427--libnghttp2-1.65.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libnghttp2--1.65.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libnghttp2/1.65.0: 14 files, 767.7KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32munbound\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/unbound/manifests/1.23.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/672e504e9852a6f9eb2fe7aa59481a05dcfa825003b59b7e28d0e6db4b66a266--unbound-1.23.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring unbound--1.23.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/unbound/1.23.0: 59 files, 6.1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mgnutls\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gnutls/manifests/3.8.9\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/aa9854d92d160da79cf6db83ae63d51f5c7d13682658fa43a335e4108af40362--gnutls-3.8.9.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gnutls--3.8.9.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gnutls/3.8.9: 1,296 files, 11MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibunibreak\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libunibreak/manifests/6.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/0ce1b2ced7836a56fa520c73ee31abcb81d0433729fe597e94ef03dd9d8a89f8--libunibreak-6.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libunibreak--6.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libunibreak/6.1: 19 files, 336.9KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibass\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libass/manifests/0.17.4\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/536891b34cc358f01c13f52a3e53ecaae3d5674e8d178736bb59ffeb622f6ce6--libass-0.17.4.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libass--0.17.4.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libass/0.17.4: 12 files, 618.9KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mcjson\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/cjson/manifests/1.7.18\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/3284084a143535a014f36b2aa7c948c6a28417940feba238b9425e4715e26a2b--cjson-1.7.18.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring cjson--1.7.18.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/cjson/1.7.18: 24 files, 257.3KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibmicrohttpd\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libmicrohttpd/manifests/1.0.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/01cc839b4ae42fa110aac833bd4c4ea38e803b45edd8f86da3d92d2f48922661--libmicrohttpd-1.0.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libmicrohttpd--1.0.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libmicrohttpd/1.0.1: 26 files, 1.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mmbedtls\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mbedtls/manifests/3.6.3.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/89560f085ce20375fc033e94245172c1ad9b9c31a4a1195ecec9f2f5f0b3ba46--mbedtls-3.6.3.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring mbedtls--3.6.3.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/mbedtls/3.6.3.1: 198 files, 13.3MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibrist\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/librist/manifests/0.2.11\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/a2325d2a172aadd20de7dbffb5d9ba60ff1144a0f11f4ee120b1df815f02e59b--librist-0.2.11.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring librist--0.2.11.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/librist/0.2.11: 29 files, 817.2KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibssh\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libssh/manifests/0.11.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/3591f41514c0ebe698a36753dce13e71fa33363993429a04af0da249f1eea04c--libssh-0.11.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libssh--0.11.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libssh/0.11.1: 25 files, 1.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibvpx\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libvpx/manifests/1.15.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/8339b291c068f0a2d760ab7abbffd80cc3c407611afdd6bf41f26bc0d19aaad6--libvpx-1.15.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libvpx--1.15.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libvpx/1.15.2: 22 files, 4.1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mopus\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/opus/manifests/1.5.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/ab70a7e8383f42047a27c0638675dbd9c3229997a01eed45d76f174664a4b607--opus-1.5.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring opus--1.5.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/opus/1.5.2: 16 files, 1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mrav1e\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rav1e/manifests/0.8.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/f59fec7d446af4f6e923227501c8bcb62c633e50876b69f91d76a629a29b087e--rav1e-0.8.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring rav1e--0.8.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/rav1e/0.8.1: 15 files, 48.7MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mflac\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/flac/manifests/1.5.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/089760ef9c022b15fbe3b3eaf6297c3ced28db6633ea0c868a569ae651efb325--flac-1.5.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring flac--1.5.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/flac/1.5.0: 40 files, 1.9MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mmpg123\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpg123/manifests/1.33.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/28fd5a6c98b6bcf1e4089972e626c78ed01f4b398817dc80bab5392ef996bf2a--mpg123-1.33.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring mpg123--1.33.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/mpg123/1.33.0: 34 files, 2.0MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibsndfile\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsndfile/manifests/1.2.2_1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/2e97439a3a58c856a00043feceb29f382febc8542b8a0c0a31aa202b2c93ea08--libsndfile-1.2.2_1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libsndfile--1.2.2_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libsndfile/1.2.2_1: 55 files, 1.8MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mrubberband\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rubberband/manifests/4.0.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/bc9f458d122d070584e4df5d712d01aaa5ba968f72342eb12ff5ec4cc8bb97bb--rubberband-4.0.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring rubberband--4.0.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/rubberband/4.0.0: 15 files, 2.0MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32msdl2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/sdl2/manifests/2.32.8\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/0bf42fc560ec7c95ad8dabc90400d56ed0376a39a0a0dad68786eb7bce49359b--sdl2-2.32.8.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring sdl2--2.32.8.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/sdl2/2.32.8: 94 files, 6.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32msnappy\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/snappy/manifests/1.2.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/df557ce14572532f881fab983b2733f9309e751f8eb27fed71262a67f828c09d--snappy-1.2.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring snappy--1.2.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/snappy/1.2.2: 19 files, 174.3KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32msrt\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/srt/manifests/1.5.4\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/33ca2d2d20854c6aeeb9dbcfb98a9721584a29e8e65bc2dcd6764f2792a2af27--srt-1.5.4.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring srt--1.5.4.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/srt/1.5.4: 21 files, 4.7MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32msvt-av1\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/svt-av1/manifests/3.0.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/08b707c1ceca9182539fadceb56d5017860f66e31092634bd65d4ee476c6f02d--svt-av1-3.0.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring svt-av1--3.0.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/svt-av1/3.0.2: 18 files, 3.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mx265\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/x265/manifests/4.1-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/af3d0898a21be2f4d792a707ba01ec1ede8578aca0b60918ba5497abf81d0931--x265-4.1-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring x265--4.1.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/x265/4.1: 12 files, 11.3MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling ffmpeg dependency: \u001b[32mlibsodium\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsodium/manifests/1.0.20\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/a9a9a2e1207e214070682a14f6470fb686cbb6ba7c24c2c747c0ca0663f42557--libsodium-1.0.20.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libsodium--1.0.20.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libsodium/1.0.20: 78 files, 888.7KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mffmpeg\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring ffmpeg--7.1.1_3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/ffmpeg/7.1.1_3: 287 files, 52.4MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup ffmpeg`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/ffmpeg/6.0_1... (284 files, 49.3MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mlibzip\u001b[39m\u001b[0m\n",
            "  1.9.2 -> 1.11.4 \n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libzip--1.11.4.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libzip/1.11.4: 161 files, 934.6KB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup libzip`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/libzip/1.9.2... (145 files, 851.7KB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mpython@3.11\u001b[39m\u001b[0m\n",
            "  3.11.6 -> 3.11.13 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for python@3.11: \u001b[32mmpdecimal\u001b[39m, \u001b[32mreadline\u001b[39m and \u001b[32msqlite\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling python@3.11 dependency: \u001b[32mmpdecimal\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/mpdecimal/manifests/4.0.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/dbbf60721dc54b6215f6c0988496331d4110a2a358da867a1129cd84b8166b31--mpdecimal-4.0.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring mpdecimal--4.0.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/mpdecimal/4.0.1: 22 files, 641.1KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling python@3.11 dependency: \u001b[32mreadline\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/readline/manifests/8.2.13\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/aa1afa38997a2866d91c81fdda8b36d436cd4ea7a82aed07d13c83c56eb3460e--readline-8.2.13.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring readline--8.2.13.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/readline/8.2.13: 51 files, 1.7MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling python@3.11 dependency: \u001b[32msqlite\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/sqlite/manifests/3.50.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/a40fe78724db12f47a20638d2ac82e7057f6f1f425ea185cb180dd97df08cb2e--sqlite-3.50.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring sqlite--3.50.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/sqlite/3.50.1: 13 files, 4.9MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mpython@3.11\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring python@3.11--3.11.13.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/python@3.11/3.11.13/bin/python3.11 -Im ensurepip\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/python@3.11/3.11.13/bin/python3.11 -Im pip install -v -\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mCaveats\u001b[0m\n",
            "Python is installed as\n",
            "  /opt/homebrew/bin/python3.11\n",
            "\n",
            "Unversioned and major-versioned symlinks `python`, `python3`, `python-config`, `python3-config`, `pip`, `pip3`, etc. pointing to\n",
            "`python3.11`, `python3.11-config`, `pip3.11` etc., respectively, are installed into\n",
            "  /opt/homebrew/opt/python@3.11/libexec/bin\n",
            "\n",
            "You can install Python packages with\n",
            "  pip3.11 install <package>\n",
            "They will install into the site-package directory\n",
            "  /opt/homebrew/lib/python3.11/site-packages\n",
            "\n",
            "tkinter is no longer included with this formula, but it is available separately:\n",
            "  brew install python-tk@3.11\n",
            "\n",
            "gdbm (`dbm.gnu`) is no longer included in this formula, but it is available separately:\n",
            "  brew install python-gdbm@3.11\n",
            "`dbm.ndbm` changed database backends in Homebrew Python 3.11.\n",
            "If you need to read a database from a previous Homebrew Python created via `dbm.ndbm`,\n",
            "you'll need to read your database using the older version of Homebrew Python and convert to another format.\n",
            "`dbm` still defaults to `dbm.gnu` when it is installed.\n",
            "\n",
            "If you do not need a specific version of Python, and always want Homebrew's `python3` in your PATH:\n",
            "  brew install python3\n",
            "\n",
            "For more information about Homebrew and Python, see: https://docs.brew.sh/Homebrew-and-Python\n",
            "\u001b[34m==>\u001b[0m \u001b[1mSummary\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/python@3.11/3.11.13: 3,306 files, 62.1MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup python@3.11`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/python@3.11/3.11.6... (3,399 files, 64.7MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mqemu\u001b[39m\u001b[0m\n",
            "  8.1.2 -> 10.0.2_1 \n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling dependencies for qemu: \u001b[32mcapstone\u001b[39m, \u001b[32mdtc\u001b[39m, \u001b[32mlibslirp\u001b[39m, \u001b[32mlibusb\u001b[39m, \u001b[32mncurses\u001b[39m, \u001b[32mfaac\u001b[39m, \u001b[32mfaad2\u001b[39m, \u001b[32mfdk-aac\u001b[39m, \u001b[32mgdk-pixbuf\u001b[39m, \u001b[32mgraphene\u001b[39m, \u001b[32mdbus\u001b[39m, \u001b[32mlibxfixes\u001b[39m, \u001b[32mlibxi\u001b[39m, \u001b[32mlibxtst\u001b[39m, \u001b[32mat-spi2-core\u001b[39m, \u001b[32mgsettings-desktop-schemas\u001b[39m, \u001b[32mhicolor-icon-theme\u001b[39m, \u001b[32mlibepoxy\u001b[39m, \u001b[32mgtk+3\u001b[39m, \u001b[32mgtk4\u001b[39m, \u001b[32mjson-glib\u001b[39m, \u001b[32mlibnice\u001b[39m, \u001b[32mlibshout\u001b[39m, \u001b[32mglib-networking\u001b[39m, \u001b[32mlibpsl\u001b[39m, \u001b[32mlibsoup\u001b[39m, \u001b[32mlibusrsctp\u001b[39m, \u001b[32morc\u001b[39m, \u001b[32mpkgconf\u001b[39m, \u001b[32mexpat\u001b[39m, \u001b[32mpython@3.13\u001b[39m, \u001b[32mgobject-introspection\u001b[39m, \u001b[32mpy3cairo\u001b[39m, \u001b[32mpygobject3\u001b[39m, \u001b[32mrtmpdump\u001b[39m, \u001b[32msrtp\u001b[39m, \u001b[32mutf8cpp\u001b[39m, \u001b[32mtaglib\u001b[39m, \u001b[32mlibcuefile\u001b[39m, \u001b[32mlibreplaygain\u001b[39m, \u001b[32mmusepack\u001b[39m, \u001b[32mgstreamer\u001b[39m, \u001b[32mjpeg\u001b[39m and \u001b[32mspice-server\u001b[39m\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mcapstone\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/capstone/manifests/5.0.6\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/e37f1aeaac48c7927bf63b0f77f747803f569a0d7001d01cc360d43059a32b53--capstone-5.0.6.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring capstone--5.0.6.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/capstone/5.0.6: 31 files, 22.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mdtc\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dtc/manifests/1.7.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/ed84da36a88c21945d4e425f140280305e493c750a27bc01887a737260f06105--dtc-1.7.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring dtc--1.7.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/dtc/1.7.2: 19 files, 627.0KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibslirp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libslirp/manifests/4.9.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/26c6da5c12d5f11ca5b8e2bb40dce3760c180c0e64c5bb6fb4ba4f31ad3c8e63--libslirp-4.9.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libslirp--4.9.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libslirp/4.9.1: 12 files, 411.0KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibusb\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libusb/manifests/1.0.29\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/7ab485d71ab6f3a78260ff61832769532c417c8194763e4225a8625077441be3--libusb-1.0.29.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libusb--1.0.29.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libusb/1.0.29: 23 files, 630.2KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mncurses\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/ncurses/manifests/6.5\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/b2f8ce4e08b15832435170efd265346225ee388d5baab3d5c50e3a77c5673b5a--ncurses-6.5.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring ncurses--6.5.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/ncurses/6.5: 4,052 files, 9.9MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mfaac\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/faac/manifests/1.31.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/2dc3ba40ae0e8e99b523e860037908f838777845d341f6b8efacab2202a3fea2--faac-1.31.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring faac--1.31.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/faac/1.31.1: 20 files, 453.0KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mfaad2\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/faad2/manifests/2.11.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/c9e8286e699dd2e9db9b4b06e7ed0db2135a4794a3f605704ff6f5ca29e37335--faad2-2.11.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring faad2--2.11.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/faad2/2.11.2: 18 files, 660KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mfdk-aac\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/fdk-aac/manifests/2.0.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/e55fa5244a07bf7923afebeb7a427146f03403fefb1ed888149fbd842808d340--fdk-aac-2.0.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring fdk-aac--2.0.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/fdk-aac/2.0.3: 16 files, 2.8MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgdk-pixbuf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gdk-pixbuf/manifests/2.42.12_1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/2eeb910ea69bc28d9dc0b3b121819ff59ea92fa20bab68c59b6a7a7d5af0708b--gdk-pixbuf-2.42.12_1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gdk-pixbuf--2.42.12_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/gdk-pixbuf/2.42.12_1/bin/gdk-pixbuf-query-loaders --upd\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gdk-pixbuf/2.42.12_1: 152 files, 4.0MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgraphene\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/graphene/manifests/1.10.8\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/95f4a8d4908c92f35e43a695c0bf8ef50b0f53030c942e4f226e8a79e3530ae6--graphene-1.10.8.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring graphene--1.10.8.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/graphene/1.10.8: 37 files, 1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mdbus\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/dbus/manifests/1.16.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/4d7addc05476f068c217c8e9d9c3a43fd366359d34bd4ad056d0de7568cd8847--dbus-1.16.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring dbus--1.16.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/dbus/1.16.2/bin/dbus-uuidgen --ensure=/opt/homebrew/var\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/dbus/1.16.2: 78 files, 2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibxfixes\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxfixes/manifests/6.0.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/b78eeb464b4c153eb6e75a04ba240b4972cdc642e77e9c5085e186630c2c4862--libxfixes-6.0.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxfixes--6.0.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxfixes/6.0.1: 13 files, 147.1KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibxi\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxi/manifests/1.8.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/089cbc546059463acb4a311e70491ca9665a8c04ab7683bc3c0803ee15b4d698--libxi-1.8.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxi--1.8.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxi/1.8.2: 86 files, 701.4KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibxtst\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libxtst/manifests/1.2.5\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/255b29f533b7064a6e692de37b3d9a41d73033506bec841e9900dbf2eb337a4c--libxtst-1.2.5.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libxtst--1.2.5.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libxtst/1.2.5: 23 files, 195.2KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mat-spi2-core\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/at-spi2-core/manifests/2.56.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/d5b3b2b2e639d506917ced078e2bfe9cd6f120787535f890904bfdd46d0e4850--at-spi2-core-2.56.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring at-spi2-core--2.56.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/at-spi2-core/2.56.2: 201 files, 4.3MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgsettings-desktop-schemas\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gsettings-desktop-schemas/manif\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/b50977acc6e572c3fef03f9e0d5a91fb31313e24dce8cea5c0cedcc626a07b3b--gsettings-desktop-schemas-48.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gsettings-desktop-schemas--48.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/opt/glib/bin/glib-compile-schemas /opt/homebrew/share/glib-2.0\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gsettings-desktop-schemas/48.0: 107 files, 5.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mhicolor-icon-theme\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/hicolor-icon-theme/manifests/0.\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/70835c74d449476f69c535127dcf54e7bc83475b0e28f15db78ccf6c2dfe3e7a--hicolor-icon-theme-0.18.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring hicolor-icon-theme--0.18.all.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/hicolor-icon-theme/0.18: 8 files, 80.7KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibepoxy\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libepoxy/manifests/1.5.10\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/fd91321dc6032305294ad3c7c3851bc80f1505f18512e4753036c509ba4b4e2c--libepoxy-1.5.10.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libepoxy--1.5.10.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libepoxy/1.5.10: 11 files, 2.6MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgtk+3\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gtkx3/manifests/3.24.43-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/2793cc81b701202c0a29a0bf3795aacf78e5f207783723237eb9442f6f284b53--gtk+3-3.24.43-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gtk+3--3.24.43.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/opt/glib/bin/glib-compile-schemas /opt/homebrew/share/glib-2.0\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/gtk+3/3.24.43/bin/gtk3-update-icon-cache -f -t /opt/hom\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/gtk+3/3.24.43/bin/gtk-query-immodules-3.0 > /opt/homebr\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gtk+3/3.24.43: 749 files, 56.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgtk4\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gtk4/manifests/4.18.6\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/edb8846643251c1fc42d48cd53c07a3891798abff4093527df65e0c50dfdd4e4--gtk4-4.18.6.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gtk4--4.18.6.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/opt/glib/bin/glib-compile-schemas /opt/homebrew/share/glib-2.0\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/Cellar/gtk4/4.18.6/bin/gtk4-update-icon-cache -f -t /opt/homeb\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/opt/glib/bin/gio-querymodules /opt/homebrew/lib/gtk-4.0/4.0.0/\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gtk4/4.18.6: 596 files, 60.3MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mjson-glib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/json-glib/manifests/1.10.6\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/a1a3897a05adbf64ce3cdd0d152252cc5b3db3e3f9b4dd659c684d72d9db5c35--json-glib-1.10.6.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring json-glib--1.10.6.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/json-glib/1.10.6: 121 files, 2.0MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibnice\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libnice/manifests/0.1.22-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/0c71ce9e479bf4dc51b9be5370473933683aa4170f22dbbdbef6d14fc108e2c1--libnice-0.1.22-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libnice--0.1.22.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libnice/0.1.22: 32 files, 826.6KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibshout\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libshout/manifests/2.4.6_1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/6bb6a35d9b7a23b52b0860746a0a10ca5a393f46d1506f563eb13d99235fac67--libshout-2.4.6_1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libshout--2.4.6_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libshout/2.4.6_1: 20 files, 445.2KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mglib-networking\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/glib-networking/manifests/2.80.\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/4f89b09c95b5e0b739677181793c68f87d7b371b8544cfd78f0b0be13534b4e5--glib-networking-2.80.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring glib-networking--2.80.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1m/opt/homebrew/opt/glib/bin/gio-querymodules /opt/homebrew/lib/gio/modules\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/glib-networking/2.80.1: 79 files, 767.6KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibpsl\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libpsl/manifests/0.21.5_2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/7211d5e3f08d33357735f48740173a11b19ad59f8b7af7923e891cb10e7659c7--libpsl-0.21.5_2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libpsl--0.21.5_2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libpsl/0.21.5_2: 16 files, 239.5KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibsoup\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libsoup/manifests/3.6.5\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/f6dd92b944c05bd8e89974eb26f0c2e835b1ade34602f772cda417fec3022496--libsoup-3.6.5.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libsoup--3.6.5.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libsoup/3.6.5: 125 files, 2.1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibusrsctp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libusrsctp/manifests/0.9.5.0_1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/90949db17b404e45eee4c247dbde57ba0301da4ced2ba512dc1e990bcba10fc4--libusrsctp-0.9.5.0_1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libusrsctp--0.9.5.0_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libusrsctp/0.9.5.0_1: 10 files, 521KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32morc\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/orc/manifests/0.4.41\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/fab5595fff526eb3a6777f1e0ea97c3c28e3a86d58413591195edd125667ca35--orc-0.4.41.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring orc--0.4.41.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/orc/0.4.41: 47 files, 1.1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mpkgconf\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pkgconf/manifests/2.4.3-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/f938254e85413ef0d18283c9c932823434b88c0c790e04e7534e2a3e923d0588--pkgconf-2.4.3-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring pkgconf--2.4.3.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/pkgconf/2.4.3: 27 files, 487KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mexpat\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/expat/manifests/2.7.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/09efb27f4cd66decc2e0f393e7f9292c729064e5c939cd8191f81fc000c3947a--expat-2.7.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring expat--2.7.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/expat/2.7.1: 23 files, 684.8KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mpython@3.13\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/python/3.13/manifests/3.13.5-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/6f054ddb0b7ceefefd45be898b4aee4dd0b8e38c8e43c6d5370347f18bf28259--python@3.13-3.13.5-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring python@3.13--3.13.5.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "\u001b[33mWarning:\u001b[0m These files were overwritten during the `brew link` step:\n",
            "/opt/homebrew/bin/pip3\n",
            "\n",
            "They have been backed up to: /Users/sofanelipovich/Library/Caches/Homebrew/Backup\n",
            "\u001b[34m==>\u001b[0m \u001b[1mSummary\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/python@3.13/3.13.5: 3,605 files, 67.0MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgobject-introspection\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gobject-introspection/manifests\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/0b119f9265755f50a70c208d5b43d904ec6ac896c934c70b2b5992c86d1a5ba4--gobject-introspection-1.84.0_1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gobject-introspection--1.84.0_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gobject-introspection/1.84.0_1: 783 files, 7.4MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mpy3cairo\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/py3cairo/manifests/1.28.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/3c16d50ad77dd43bb02779327703cae5f831127bb5442362fb5b4ec8f405cba2--py3cairo-1.28.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring py3cairo--1.28.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/py3cairo/1.28.0: 22 files, 1MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mpygobject3\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/pygobject3/manifests/3.52.3\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/597dd463c4b332f9475e1fc7faac944b028fd4adb4fe26afa219113a4cfacb87--pygobject3-3.52.3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring pygobject3--3.52.3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/pygobject3/3.52.3: 70 files, 1.7MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mrtmpdump\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/rtmpdump/manifests/2.4-20151223\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/b40cb4d6b301aad8ce26f9deb74a74c0d98d1868d19881f9bff55b81d6f11e50--rtmpdump-2.4-20151223_3.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring rtmpdump--2.4-20151223_3.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/rtmpdump/2.4-20151223_3: 21 files, 652.8KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32msrtp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/srtp/manifests/2.7.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/852a0a6f14822b4b07b6f60e0f6be33ae1e95b2e94a697aa004be69bdc996eaa--srtp-2.7.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring srtp--2.7.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/srtp/2.7.0: 15 files, 466.8KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mutf8cpp\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/utf8cpp/manifests/4.0.6\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/8136a52227e6e2e57272c2548f9d863433cb84db1334b7f0dcdeeff61372c657--utf8cpp-4.0.6.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring utf8cpp--4.0.6.all.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/utf8cpp/4.0.6: 15 files, 145.4KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mtaglib\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/taglib/manifests/2.1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/2c62daf4c5ef648e7766ae2c8a4d7414dad4de1efa7fd9c2232cf792cfe787ce--taglib-2.1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring taglib--2.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/taglib/2.1: 138 files, 2.0MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibcuefile\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libcuefile/manifests/r475-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/bcfd5f9c5e66525827bd2c1024f9dbee05db47868b3077b3e19178fe9e25fd91--libcuefile-r475-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libcuefile--r475.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libcuefile/r475: 13 files, 255.6KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mlibreplaygain\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/libreplaygain/manifests/r475-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/c2a362fb72ade5ac1b1d01f398f0b9cc9a1197d1d40629bbd063e5e49ecb1353--libreplaygain-r475-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring libreplaygain--r475.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/libreplaygain/r475: 8 files, 69.4KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mmusepack\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/musepack/manifests/r475-1\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/38024792b77872be2f125d871addc33da905cbbd9e388df3b01eec29cd6aa4ae--musepack-r475-1.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring musepack--r475.arm64_ventura.bottle.1.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/musepack/r475: 27 files, 757.7KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mgstreamer\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/gstreamer/manifests/1.26.2\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/e3a14cadf65adec95babc0ff85bd74320923c8f0903991e145458f7fd0c452e1--gstreamer-1.26.2.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring gstreamer--1.26.2.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/gstreamer/1.26.2: 1,360 files, 210.2MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mjpeg\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/jpeg/manifests/9f\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/89d881b2a8ef2cada874252595f3f23968d07fc6a69c7d9af6f04cd1617c3656--jpeg-9f.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring jpeg--9f.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/jpeg/9f: 22 files, 903.4KB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling qemu dependency: \u001b[32mspice-server\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mDownloading https://ghcr.io/v2/homebrew/core/spice-server/manifests/0.16.0\u001b[0m\n",
            "Already downloaded: /Users/sofanelipovich/Library/Caches/Homebrew/downloads/ba299e2da06b7881d9e22394cc7b95a75c846261d7bfb343484c40d17ef92206--spice-server-0.16.0.bottle_manifest.json\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring spice-server--0.16.0.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/spice-server/0.16.0: 20 files, 1.5MB\n",
            "\u001b[32m==>\u001b[0m \u001b[1mInstalling \u001b[32mqemu\u001b[39m\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring qemu--10.0.2_1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/qemu/10.0.2_1: 169 files, 675.3MB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup qemu`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/qemu/8.1.2... (162 files, 534.1MB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mUpgrading \u001b[32mtldr\u001b[39m\u001b[0m\n",
            "  1.5.0 -> 1.6.1 \n",
            "\u001b[34m==>\u001b[0m \u001b[1mPouring tldr--1.6.1.arm64_ventura.bottle.tar.gz\u001b[0m\n",
            "🍺  /opt/homebrew/Cellar/tldr/1.6.1: 11 files, 111.6KB\n",
            "\u001b[34m==>\u001b[0m \u001b[1mRunning `brew cleanup tldr`...\u001b[0m\n",
            "Removing: /opt/homebrew/Cellar/tldr/1.5.0... (10 files, 103.4KB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mChecking for dependents of upgraded formulae...\u001b[0m\n",
            "\u001b[34m==>\u001b[0m \u001b[1mNo broken dependents found!\u001b[0m\n",
            "\u001b[32m==>\u001b[0m \u001b[1m`brew cleanup` has not been run in the last 30 days, running now...\u001b[0m\n",
            "Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.\n",
            "Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).\n",
            "Removing: /opt/homebrew/Cellar/ca-certificates/2023-08-22... (3 files, 221.6KB)\n",
            "Removing: /opt/homebrew/Cellar/cairo/1.18.0... (52 files, 2.1MB)\n",
            "Removing: /opt/homebrew/Cellar/capstone/5.0.1... (30 files, 22.3MB)\n",
            "Removing: /opt/homebrew/Cellar/cjson/1.7.16... (23 files, 254KB)\n",
            "Removing: /opt/homebrew/Cellar/dav1d/1.2.1... (15 files, 910KB)\n",
            "Removing: /opt/homebrew/Cellar/dtc/1.7.0... (18 files, 619.2KB)\n",
            "Removing: /opt/homebrew/Cellar/flac/1.4.3... (284 files, 6.9MB)\n",
            "Removing: /opt/homebrew/Cellar/fontconfig/2.14.2... (88 files, 2.4MB)\n",
            "Removing: /opt/homebrew/Cellar/freetype/2.13.2... (67 files, 2.4MB)\n",
            "Removing: /opt/homebrew/Cellar/frei0r/2.3.1... (153 files, 5.5MB)\n",
            "Removing: /opt/homebrew/Cellar/fribidi/1.0.13... (67 files, 733.3KB)\n",
            "Removing: /opt/homebrew/Cellar/gettext/0.21.1... (1,983 files, 20.9MB)\n",
            "Removing: /opt/homebrew/Cellar/gettext/0.22.3... (2,040 files, 21.9MB)\n",
            "Removing: /opt/homebrew/Cellar/giflib/5.2.1... (19 files, 540.2KB)\n",
            "Removing: /opt/homebrew/Cellar/glib/2.78.0... (456 files, 22.4MB)\n",
            "Removing: /opt/homebrew/Cellar/glib/2.78.1... (456 files, 22.4MB)\n",
            "Removing: /opt/homebrew/Cellar/gnutls/3.8.2... (1,290 files, 10.8MB)\n",
            "Removing: /opt/homebrew/Cellar/harfbuzz/8.3.0... (76 files, 9.5MB)\n",
            "Removing: /opt/homebrew/Cellar/highway/1.0.7... (65 files, 3.3MB)\n",
            "Removing: /opt/homebrew/Cellar/icu4c/73.2... (268 files, 80.1MB)\n",
            "Removing: /opt/homebrew/Cellar/imath/3.1.9... (49 files, 935.6KB)\n",
            "Removing: /opt/homebrew/Cellar/isl/0.26... (73 files, 7.6MB)\n",
            "Removing: /opt/homebrew/Cellar/jpeg-turbo/3.0.0... (44 files, 3.4MB)\n",
            "Removing: /opt/homebrew/Cellar/libass/0.17.1... (11 files, 532.7KB)\n",
            "Removing: /opt/homebrew/Cellar/libidn2/2.3.4_1... (79 files, 1MB)\n",
            "Removing: /opt/homebrew/Cellar/libnghttp2/1.58.0... (13 files, 739.2KB)\n",
            "Removing: /Users/sofanelipovich/Library/Caches/Homebrew/libomp_bottle_manifest--19.1.5... (9KB)\n",
            "Removing: /Users/sofanelipovich/Library/Caches/Homebrew/libomp--19.1.5... (573.3KB)\n",
            "Removing: /opt/homebrew/Cellar/libpng/1.6.40... (27 files, 1.3MB)\n",
            "Removing: /opt/homebrew/Cellar/librist/0.2.7_4... (28 files, 737.4KB)\n",
            "Removing: /opt/homebrew/Cellar/libslirp/4.7.0... (11 files, 385KB)\n",
            "Removing: /opt/homebrew/Cellar/libsndfile/1.2.2... (53 files, 1MB)\n",
            "Removing: /opt/homebrew/Cellar/libsodium/1.0.19... (77 files, 1MB)\n",
            "Removing: /opt/homebrew/Cellar/libssh/0.10.5_1... (23 files, 1.3MB)\n",
            "Removing: /opt/homebrew/Cellar/libtasn1/4.19.0... (61 files, 717.8KB)\n",
            "Removing: /opt/homebrew/Cellar/libunibreak/5.1... (17 files, 328.7KB)\n",
            "Removing: /opt/homebrew/Cellar/libunistring/1.1... (56 files, 5.0MB)\n",
            "Removing: /opt/homebrew/Cellar/libusb/1.0.26... (22 files, 595.0KB)\n",
            "Removing: /opt/homebrew/Cellar/libvmaf/2.3.1... (234 files, 7MB)\n",
            "Removing: /opt/homebrew/Cellar/libvpx/1.13.1... (20 files, 3.7MB)\n",
            "Removing: /opt/homebrew/Cellar/libx11/1.8.7... (1,042 files, 7.0MB)\n",
            "Removing: /opt/homebrew/Cellar/libxau/1.0.11... (21 files, 123.5KB)\n",
            "Removing: /opt/homebrew/Cellar/libxcb/1.16... (2,491 files, 7.5MB)\n",
            "Removing: /opt/homebrew/Cellar/libxdmcp/1.1.4... (11 files, 130.4KB)\n",
            "Removing: /opt/homebrew/Cellar/libxext/1.3.5... (87 files, 446KB)\n",
            "Removing: /opt/homebrew/Cellar/libxrender/0.9.11... (12 files, 214.1KB)\n",
            "Removing: /opt/homebrew/Cellar/lz4/1.9.4... (22 files, 681.5KB)\n",
            "Removing: /opt/homebrew/Cellar/mbedtls/3.5.1... (192 files, 12MB)\n",
            "Removing: /opt/homebrew/Cellar/mpdecimal/2.5.1... (71 files, 2.2MB)\n",
            "Removing: /opt/homebrew/Cellar/mpfr/4.2.0-p12... (30 files, 3MB)\n",
            "Removing: /opt/homebrew/Cellar/mpg123/1.32.3... (33 files, 2.0MB)\n",
            "Removing: /opt/homebrew/Cellar/ncurses/6.4... (4,001 files, 9.7MB)\n",
            "Removing: /opt/homebrew/Cellar/nettle/3.9.1... (94 files, 2.7MB)\n",
            "Removing: /opt/homebrew/Cellar/openexr/3.2.1... (204 files, 7.8MB)\n",
            "Removing: /opt/homebrew/Cellar/openssl@3/3.1.4... (6,496 files, 28.4MB)\n",
            "Removing: /opt/homebrew/Cellar/opus/1.4... (15 files, 979.6KB)\n",
            "Removing: /opt/homebrew/Cellar/p11-kit/0.25.3... (28 files, 4.2MB)\n",
            "Removing: /opt/homebrew/Cellar/pango/1.50.14... (68 files, 3.3MB)\n",
            "Removing: /opt/homebrew/Cellar/pcre2/10.42... (230 files, 6.2MB)\n",
            "Removing: /opt/homebrew/Cellar/pixman/0.42.2... (11 files, 1.2MB)\n",
            "Removing: /opt/homebrew/Cellar/rav1e/0.6.6... (14 files, 140.2MB)\n",
            "Removing: /opt/homebrew/Cellar/readline/8.2.1... (50 files, 1.7MB)\n",
            "Removing: /opt/homebrew/Cellar/rubberband/3.3.0... (13 files, 1.7MB)\n",
            "Removing: /opt/homebrew/Cellar/sdl2/2.28.5... (93 files, 6.4MB)\n",
            "Removing: /opt/homebrew/Cellar/snappy/1.1.10... (18 files, 164.2KB)\n",
            "Removing: /opt/homebrew/Cellar/sqlite/3.43.1... (11 files, 4.7MB)\n",
            "Removing: /opt/homebrew/Cellar/srt/1.5.3... (20 files, 4.4MB)\n",
            "Removing: /opt/homebrew/Cellar/svt-av1/1.7.0... (24 files, 3.5MB)\n",
            "Removing: /opt/homebrew/Cellar/unbound/1.19.0... (58 files, 5.9MB)\n",
            "Removing: /opt/homebrew/Cellar/x265/3.5... (11 files, 6.6MB)\n",
            "Removing: /opt/homebrew/Cellar/xorgproto/2023.2... (267 files, 3.9MB)\n",
            "Removing: /opt/homebrew/Cellar/xz/5.4.4... (163 files, 2.6MB)\n",
            "Removing: /Users/sofanelipovich/Library/Caches/Homebrew/portable-ruby-3.3.6.arm64_big_sur.bottle.tar.gz... (11.2MB)\n",
            "Removing: /opt/homebrew/lib/python3.11/site-packages/__pycache__/google_auth_httplib2.cpython-311.pyc... (11.4KB)\n",
            "Removing: /opt/homebrew/lib/python3.11/site-packages/__pycache__/isympy.cpython-311.pyc... (11KB)\n",
            "Removing: /opt/homebrew/lib/python3.11/site-packages/__pycache__/py.cpython-311.pyc... (416B)\n",
            "Removing: /opt/homebrew/lib/python3.11/site-packages/__pycache__/retrying.cpython-311.pyc... (13.3KB)\n",
            "Removing: /opt/homebrew/lib/python3.11/site-packages/__pycache__/threadpoolctl.cpython-311.pyc... (60.2KB)\n",
            "Removing: /opt/homebrew/lib/python3.11/site-packages/__pycache__/typing_extensions.cpython-311.pyc... (147.9KB)\n",
            "\u001b[32m==>\u001b[0m \u001b[1mCaveats\u001b[0m\n",
            "zsh completions have been installed to:\n",
            "  /opt/homebrew/share/zsh/site-functions\n",
            "\u001b[34m==>\u001b[0m \u001b[1mpython@3.11\u001b[0m\n",
            "Python is installed as\n",
            "  /opt/homebrew/bin/python3.11\n",
            "\n",
            "Unversioned and major-versioned symlinks `python`, `python3`, `python-config`, `python3-config`, `pip`, `pip3`, etc. pointing to\n",
            "`python3.11`, `python3.11-config`, `pip3.11` etc., respectively, are installed into\n",
            "  /opt/homebrew/opt/python@3.11/libexec/bin\n",
            "\n",
            "You can install Python packages with\n",
            "  pip3.11 install <package>\n",
            "They will install into the site-package directory\n",
            "  /opt/homebrew/lib/python3.11/site-packages\n",
            "\n",
            "tkinter is no longer included with this formula, but it is available separately:\n",
            "  brew install python-tk@3.11\n",
            "\n",
            "gdbm (`dbm.gnu`) is no longer included in this formula, but it is available separately:\n",
            "  brew install python-gdbm@3.11\n",
            "`dbm.ndbm` changed database backends in Homebrew Python 3.11.\n",
            "If you need to read a database from a previous Homebrew Python created via `dbm.ndbm`,\n",
            "you'll need to read your database using the older version of Homebrew Python and convert to another format.\n",
            "`dbm` still defaults to `dbm.gnu` when it is installed.\n",
            "\n",
            "If you do not need a specific version of Python, and always want Homebrew's `python3` in your PATH:\n",
            "  brew install python3\n",
            "\n",
            "For more information about Homebrew and Python, see: https://docs.brew.sh/Homebrew-and-Python\n"
          ]
        }
      ],
      "source": [
        "!brew install bedtools"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 36,
      "metadata": {},
      "outputs": [],
      "source": [
        "!bedtools complement -i genes.bed -g ./data/scaffolds_sizes.txt > intergenic.bed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 37,
      "metadata": {},
      "outputs": [],
      "source": [
        "!bedtools flank -i genes.bed -g ./datascaffolds_sizes.txt -l 1000 -r 0 -s > promoters.bed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 38,
      "metadata": {},
      "outputs": [],
      "source": [
        "!bedtools flank -i genes.bed -g ./data/scaffolds_sizes.txt -l 0 -r 200 -s  > downstream.bed"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 39,
      "metadata": {},
      "outputs": [],
      "source": [
        "!bedtools intersect -a quadruplex.bed -b exons.bed -wa -s > quadr_in_exons.bed\n",
        "!bedtools intersect -a quadruplex.bed -b introns.bed -wa -s > quadr_in_introns.bed\n",
        "!bedtools intersect -a quadruplex.bed -b promoters.bed -wa -s > quadr_in_promoters.bed\n",
        "!bedtools intersect -a quadruplex.bed -b downstream.bed -wa -s > quadr_in_downstream.bed\n",
        "!bedtools intersect -a quadruplex.bed -b intergenic.bed -wa > quadr_in_intergenic.bed"
      ]
    },
    {
      "attachments": {},
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "Пересечения с квадруплексами"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 55,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "851\n",
            "0\n",
            "162\n",
            "29\n",
            "2167\n",
            "5760\n"
          ]
        }
      ],
      "source": [
        "!cat quadr_in_exons.bed | wc -l\n",
        "\n",
        "!cat quadr_in_introns.bed | wc -l\n",
        "\n",
        "!cat quadr_in_promoters.bed | wc -l\n",
        "\n",
        "!cat quadr_in_downstream.bed | wc -l\n",
        "\n",
        "!cat quadr_in_intergenic.bed | wc -l\n",
        "\n",
        "!cat quadruplex.bed | wc -l"
      ]
    },
    {
      "attachments": {},
      "cell_type": "markdown",
      "metadata": {},
      "source": [
        "Количество участков с квадруплексами\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 50,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "851\n",
            "181291\n"
          ]
        }
      ],
      "source": [
        "!bedtools intersect -a quadruplex.bed -b exons.bed -wb -s > exons_with_quadruplex_all.bed\n",
        "!sort -u exons_with_quadruplex_all.bed > exons_with_quadr.bed\n",
        "!cat exons_with_quadr.bed | wc -l\n",
        "!cat exons.bed | wc -l"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 51,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "0\n",
            "0\n"
          ]
        }
      ],
      "source": [
        "!bedtools intersect -a quadruplex.bed -b introns.bed -wb -s > introns_with_quadruplex_all.bed\n",
        "!sort -u introns_with_quadruplex_all.bed > introns_with_quadr.bed\n",
        "!cat introns_with_quadr.bed | wc -l\n",
        "!cat introns.bed | wc -l"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 56,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "162\n",
            "12765\n"
          ]
        }
      ],
      "source": [
        "!bedtools intersect -a quadruplex.bed -b promoters.bed -wb -s > promoters_with_quadruplex_all.bed\n",
        "!sort -u promoters_with_quadruplex_all.bed > promoters_with_quadr.bed\n",
        "!cat promoters_with_quadr.bed | wc -l\n",
        "!cat promoters.bed | wc -l"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 57,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "29\n",
            "12765\n"
          ]
        }
      ],
      "source": [
        "!bedtools intersect -a quadruplex.bed -b downstream.bed -wb -s > downstream_with_quadruplex_all.bed\n",
        "!sort -u downstream_with_quadruplex_all.bed > downstream_with_quadr.bed\n",
        "!cat downstream_with_quadr.bed | wc -l\n",
        "!cat downstream.bed | wc -l\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 58,
      "metadata": {},
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "2167\n",
            "10265\n"
          ]
        }
      ],
      "source": [
        "!bedtools intersect -a quadruplex.bed -b intergenic.bed -wb > intergenic_with_quadruplex_all.bed\n",
        "!sort -u intergenic_with_quadruplex_all.bed > intergenic_with_quadr.bed\n",
        "!cat intergenic_with_quadr.bed | wc -l\n",
        "!cat intergenic.bed | wc -l"
      ]
    }
  ],
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.11.6"
    },
    "vscode": {
      "interpreter": {
        "hash": "b0fa6594d8f4cbf19f97940f81e996739fb7646882a419484c72d19e05852a7e"
      }
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}
